{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/swat08mx/CONVEX/blob/main/AMR_NER_NLP.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "hoHRlg66-_dI"
      },
      "outputs": [],
      "source": [
        "import pandas as pd\n",
        "import tensorflow as tf\n",
        "import matplotlib.pyplot as plt\n",
        "import torch\n",
        "import time\n",
        "import numpy as np\n",
        "import torch.nn as nn\n",
        "import torch.nn.functional as F\n",
        "\n",
        "from sklearn.model_selection import train_test_split\n",
        "from sklearn.metrics import multilabel_confusion_matrix, confusion_matrix\n",
        "import seaborn as sn\n",
        "import torch.optim as optim\n",
        "from torch.utils.data import TensorDataset\n",
        "from torch.utils.data import DataLoader\n",
        "from gensim.models import KeyedVectors, word2vec\n",
        "import gensim as gs\n",
        "from sklearn.manifold import TSNE\n",
        "import json\n",
        "from spacy.training import offsets_to_biluo_tags\n",
        "from spacy.lang.en import English\n",
        "# from nltk import pos_tag, word_tokenize, sent_tokenize, wordpunct_tokenize\n",
        "# nltk.download()"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "data = pd.read_json(\"project_9_dataset.jsonl\", lines=True)\n",
        "data"
      ],
      "metadata": {
        "id": "g7rR-0cc_D7Y",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 423
        },
        "outputId": "bfe99693-6a04-4b8c-f58c-32f290a87d19"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "        id                                               text  \\\n",
              "0      685  Current limitations in the understanding and c...   \n",
              "1      686  Without addressing gaps in identified areas, s...   \n",
              "2     1893                 Thirty-eight clones were detected.   \n",
              "3      670  Several examples of the implementation of DART...   \n",
              "4      688  Specifically, antimicrobial use in human, anim...   \n",
              "...    ...                                                ...   \n",
              "2332  2329  In this review, we summarize the status of res...   \n",
              "2333  2473  Here, we survey the current literature focussi...   \n",
              "2334  2276  The results showed that the number of related ...   \n",
              "2335  1071  We also describe the current state of developm...   \n",
              "2336   973  We also highlight research about the phenotypi...   \n",
              "\n",
              "                                            annotations meta  \\\n",
              "0     [{'label': 35, 'start_offset': 159, 'end_offse...   {}   \n",
              "1                                                    []   {}   \n",
              "2                                                    []   {}   \n",
              "3     [{'label': 37, 'start_offset': 42, 'end_offset...   {}   \n",
              "4                                                    []   {}   \n",
              "...                                                 ...  ...   \n",
              "2332  [{'label': 33, 'start_offset': 240, 'end_offse...   {}   \n",
              "2333                                                 []   {}   \n",
              "2334                                                 []   {}   \n",
              "2335                                                 []   {}   \n",
              "2336  [{'label': 32, 'start_offset': 94, 'end_offset...   {}   \n",
              "\n",
              "     annotation_approver comments  \n",
              "0               shantanu       []  \n",
              "1               shantanu       []  \n",
              "2               shantanu       []  \n",
              "3               shantanu       []  \n",
              "4               shantanu       []  \n",
              "...                  ...      ...  \n",
              "2332            shantanu       []  \n",
              "2333            shantanu       []  \n",
              "2334            shantanu       []  \n",
              "2335            shantanu       []  \n",
              "2336            shantanu       []  \n",
              "\n",
              "[2337 rows x 6 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-0041a1f4-a081-4d95-a2bc-fd545cb39ed1\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>id</th>\n",
              "      <th>text</th>\n",
              "      <th>annotations</th>\n",
              "      <th>meta</th>\n",
              "      <th>annotation_approver</th>\n",
              "      <th>comments</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>685</td>\n",
              "      <td>Current limitations in the understanding and c...</td>\n",
              "      <td>[{'label': 35, 'start_offset': 159, 'end_offse...</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>686</td>\n",
              "      <td>Without addressing gaps in identified areas, s...</td>\n",
              "      <td>[]</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>1893</td>\n",
              "      <td>Thirty-eight clones were detected.</td>\n",
              "      <td>[]</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>670</td>\n",
              "      <td>Several examples of the implementation of DART...</td>\n",
              "      <td>[{'label': 37, 'start_offset': 42, 'end_offset...</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>688</td>\n",
              "      <td>Specifically, antimicrobial use in human, anim...</td>\n",
              "      <td>[]</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2332</th>\n",
              "      <td>2329</td>\n",
              "      <td>In this review, we summarize the status of res...</td>\n",
              "      <td>[{'label': 33, 'start_offset': 240, 'end_offse...</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2333</th>\n",
              "      <td>2473</td>\n",
              "      <td>Here, we survey the current literature focussi...</td>\n",
              "      <td>[]</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2334</th>\n",
              "      <td>2276</td>\n",
              "      <td>The results showed that the number of related ...</td>\n",
              "      <td>[]</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2335</th>\n",
              "      <td>1071</td>\n",
              "      <td>We also describe the current state of developm...</td>\n",
              "      <td>[]</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2336</th>\n",
              "      <td>973</td>\n",
              "      <td>We also highlight research about the phenotypi...</td>\n",
              "      <td>[{'label': 32, 'start_offset': 94, 'end_offset...</td>\n",
              "      <td>{}</td>\n",
              "      <td>shantanu</td>\n",
              "      <td>[]</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>2337 rows × 6 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-0041a1f4-a081-4d95-a2bc-fd545cb39ed1')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-0041a1f4-a081-4d95-a2bc-fd545cb39ed1 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-0041a1f4-a081-4d95-a2bc-fd545cb39ed1');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-7cef6280-817e-42a3-bdae-7046b95fbe7f\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-7cef6280-817e-42a3-bdae-7046b95fbe7f')\"\n",
              "            title=\"Suggest charts.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-7cef6280-817e-42a3-bdae-7046b95fbe7f button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ]
          },
          "metadata": {},
          "execution_count": 62
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "## code for making lists of datas and storing info\n",
        "text=[]\n",
        "start=[]\n",
        "end=[]\n",
        "label=[]\n",
        "final=[]\n",
        "\n",
        "for i in range(len(data)-1):\n",
        "  if data['annotations'][i]!=[]:\n",
        "    if len(data['annotations'][i])>0:\n",
        "      for j in range(len(data['annotations'][i])):\n",
        "        text.append(data['text'][i])\n",
        "        start.append(data['annotations'][i][j]['start_offset'])\n",
        "        end.append(data['annotations'][i][j]['end_offset'])\n",
        "        label.append(data['annotations'][i][j]['label'])\n",
        "  else:\n",
        "    text.append(data['text'][i])\n",
        "    start.append(\"\")\n",
        "    end.append(\"\")\n",
        "    label.append(\"\")"
      ],
      "metadata": {
        "id": "dbpeRRV_Bo7f"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "Label Parser"
      ],
      "metadata": {
        "id": "TGnisjwLFvIe"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "labels_data = pd.read_json(\"project_9_labels.json\")\n",
        "dat=36\n",
        "for i in range(len(labels_data)-1):\n",
        "  if labels_data['id'][i]==dat:\n",
        "    dat=labels_data['text'][i]\n",
        "\n",
        "#label swapper code\n",
        "for i in range(len(label)):\n",
        "  for j in range(len(labels_data)):\n",
        "    if label[i]==labels_data['id'][j]:\n",
        "      label[i]=labels_data['text'][j]"
      ],
      "metadata": {
        "id": "1izivgEkFkPC"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "##### bug fix code for space errors\n",
        "for i in range(len(text)):\n",
        "  try:\n",
        "    start_new = int(start[i])\n",
        "    end_new = int(end[i])\n",
        "    print(f\"The {i} line is : {text[i][start_new:end_new]}\")\n",
        "    var=text[i][start_new:end_new]\n",
        "    bef=len(var)\n",
        "    print(bef)\n",
        "    var_one=var.strip()\n",
        "    print(var_one)\n",
        "    aft=len(var_one)\n",
        "    if bef!=aft:\n",
        "      if var[-1]==\" \":\n",
        "        end[i]-=1\n",
        "      else:\n",
        "        start[i]+=1\n",
        "    end_new=end[i]\n",
        "    start_new=start[i]\n",
        "    print(len(text[i][start_new:end_new]))\n",
        "    print(\"-----------------------------------------------\")\n",
        "  except:\n",
        "    pass"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "1m3xznvEEk7k",
        "outputId": "aac8ba36-9671-493d-f637-a581bb396e4e"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "The 1583 line is : Spn\n",
            "3\n",
            "Spn\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 1584 line is : Mega1. IV or Mega2. IVc\n",
            "23\n",
            "Mega1. IV or Mega2. IVc\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 1585 line is : Vibrio\n",
            "6\n",
            "Vibrio\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1586 line is : biochemical tests\n",
            "17\n",
            "biochemical tests\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 1587 line is : molecular detection\n",
            "19\n",
            "molecular detection\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 1588 line is : pyrH\n",
            "4\n",
            "pyrH\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 1590 line is : norfloxacin\n",
            "11\n",
            "norfloxacin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1591 line is : penicillin G\n",
            "12\n",
            "penicillin G\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1604 line is : MRSA\n",
            "4\n",
            "MRSA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 1605 line is : (XY)2RRRF(YX)2-NH2\n",
            "18\n",
            "(XY)2RRRF(YX)2-NH2\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 1606 line is : tetracycline\n",
            "12\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1607 line is : tet[L]\n",
            "6\n",
            "tet[L]\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1608 line is : tet[M]\n",
            "6\n",
            "tet[M]\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1609 line is : tet[K]\n",
            "6\n",
            "tet[K]\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1610 line is : tetA[P]\n",
            "7\n",
            "tetA[P]\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 1611 line is : aminoglycoside\n",
            "14\n",
            "aminoglycoside\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1612 line is : aadA2\n",
            "5\n",
            "aadA2\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1613 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1614 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1615 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1616 line is : AS-48\n",
            "5\n",
            "AS-48\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1617 line is : ntimicrobial peptide lysozyme\n",
            "29\n",
            "ntimicrobial peptide lysozyme\n",
            "29\n",
            "-----------------------------------------------\n",
            "The 1618 line is : scn gene\n",
            "8\n",
            "scn gene\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 1619 line is : enterotoxin genes\n",
            "17\n",
            "enterotoxin genes\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 1620 line is : scn gene\n",
            "8\n",
            "scn gene\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 1621 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1622 line is : MRSA\n",
            "4\n",
            "MRSA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 1623 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1624 line is : PCR\n",
            "3\n",
            "PCR\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 1625 line is : tetracycline\n",
            "12\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1626 line is : lincomycin\n",
            "10\n",
            "lincomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1627 line is : chloramphenicol\n",
            "15\n",
            "chloramphenicol\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 1628 line is : aminoglycoside\n",
            "14\n",
            "aminoglycoside\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1629 line is : colistin\n",
            "8\n",
            "colistin\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 1630 line is : vancomycin\n",
            "10\n",
            "vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1631 line is : carbapenem\n",
            "10\n",
            "carbapenem\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1643 line is : scn virulence marker\n",
            "20\n",
            "scn virulence marker\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 1644 line is : virulence marker edinB\n",
            "22\n",
            "virulence marker edinB\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 1645 line is : enterotoxin genes\n",
            "17\n",
            "enterotoxin genes\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 1655 line is : Microreact\n",
            "10\n",
            "Microreact\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1657 line is : cephalosporin\n",
            "13\n",
            "cephalosporin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1658 line is : Neisseria gonorrhoeae \n",
            "22\n",
            "Neisseria gonorrhoeae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1659 line is :  monotherapy\n",
            "12\n",
            "monotherapy\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1660 line is : ST840\n",
            "5\n",
            "ST840\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1661 line is : ST841\n",
            "5\n",
            "ST841\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1662 line is : ST842\n",
            "5\n",
            "ST842\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1663 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1664 line is : N. gonorrhoeae\n",
            "14\n",
            "N. gonorrhoeae\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1665 line is : methicillin-resistant coagulase-negative staphylococci (MRCoNS)\n",
            "63\n",
            "methicillin-resistant coagulase-negative staphylococci (MRCoNS)\n",
            "63\n",
            "-----------------------------------------------\n",
            "The 1666 line is : MRCoNS\n",
            "6\n",
            "MRCoNS\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1667 line is : staphylococcal\n",
            "14\n",
            "staphylococcal\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1668 line is : staphylococci\n",
            "13\n",
            "staphylococci\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1669 line is :  MRSE\n",
            "5\n",
            "MRSE\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 1670 line is :  S. aureus\n",
            "10\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1671 line is : staphylococci\n",
            "13\n",
            "staphylococci\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1672 line is : staphylococci\n",
            "13\n",
            "staphylococci\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1673 line is : Klebsiella pneumoniae\n",
            "21\n",
            "Klebsiella pneumoniae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1674 line is : Serratia marcescens\n",
            "19\n",
            "Serratia marcescens\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 1675 line is : Chlamydomonas reinhardtii (Cr-SPs)\n",
            "34\n",
            "Chlamydomonas reinhardtii (Cr-SPs)\n",
            "34\n",
            "-----------------------------------------------\n",
            "The 1676 line is : Streptococcus pneumoniae\n",
            "24\n",
            "Streptococcus pneumoniae\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 1677 line is : Cr-SPs\n",
            "6\n",
            "Cr-SPs\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1678 line is : Serratia marcescens\n",
            "19\n",
            "Serratia marcescens\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 1679 line is : Klebsiella pneumoniae\n",
            "21\n",
            "Klebsiella pneumoniae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1680 line is : Cr\n",
            "2\n",
            "Cr\n",
            "2\n",
            "-----------------------------------------------\n",
            "The 1682 line is : Cr-SPs\n",
            "6\n",
            "Cr-SPs\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1683 line is : Cr-SPs\n",
            "6\n",
            "Cr-SPs\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1684 line is : PCV13\n",
            "5\n",
            "PCV13\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1685 line is : co-trimoxazole\n",
            "14\n",
            "co-trimoxazole\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1686 line is : multilocus sequence typing (MLST) \n",
            "34\n",
            "multilocus sequence typing (MLST)\n",
            "33\n",
            "-----------------------------------------------\n",
            "The 1687 line is : staphylococci\n",
            "13\n",
            "staphylococci\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1688 line is : staphylococci\n",
            "13\n",
            "staphylococci\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1689 line is : chloramphenicol\n",
            "15\n",
            "chloramphenicol\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 1690 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1707 line is : Pasteurellaceae\n",
            "15\n",
            "Pasteurellaceae\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 1708 line is : Gallibacterium\n",
            "14\n",
            "Gallibacterium\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1709 line is : aadA23\n",
            "6\n",
            "aadA23\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1710 line is : blaCARB-8\n",
            "9\n",
            "blaCARB-8\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1711 line is : tet(Y)\n",
            "6\n",
            "tet(Y)\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1712 line is : qnrD1\n",
            "5\n",
            "qnrD1\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1713 line is : transposases\n",
            "12\n",
            "transposases\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1714 line is : plasmids\n",
            "8\n",
            "plasmids\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 1715 line is : Single-nucleotide polymorphism genotyping\n",
            "41\n",
            "Single-nucleotide polymorphism genotyping\n",
            "41\n",
            "-----------------------------------------------\n",
            "The 1716 line is : G. anatis\n",
            "9\n",
            "G. anatis\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1717 line is : G. anatis\n",
            "9\n",
            "G. anatis\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1726 line is : cephalosporin-resistant strains\n",
            "31\n",
            "cephalosporin-resistant strains\n",
            "31\n",
            "-----------------------------------------------\n",
            "The 1747 line is : Enterococcal\n",
            "12\n",
            "Enterococcal\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1748 line is : staphylococcal\n",
            "14\n",
            "staphylococcal\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1749 line is : penicillin-binding proteins (PBPs)\n",
            "34\n",
            "penicillin-binding proteins (PBPs)\n",
            "34\n",
            "-----------------------------------------------\n",
            "The 1750 line is : enterococci\n",
            "11\n",
            "enterococci\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1751 line is : staphylococci\n",
            "13\n",
            "staphylococci\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1752 line is : Vancomycin \n",
            "11\n",
            "Vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1753 line is : glycopeptide \n",
            "13\n",
            "glycopeptide\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1754 line is : Pseudomonas aeruginosa\n",
            "22\n",
            "Pseudomonas aeruginosa\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 1755 line is : Klebsiella pneumoniae\n",
            "21\n",
            "Klebsiella pneumoniae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1756 line is : Acinetobacter baumannii\n",
            "23\n",
            "Acinetobacter baumannii\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 1757 line is : antibiotics\n",
            "11\n",
            "antibiotics\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1758 line is : phospholipids\n",
            "13\n",
            "phospholipids\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1759 line is : lipopolysaccharide\n",
            "18\n",
            "lipopolysaccharide\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 1760 line is : lipoproteins\n",
            "12\n",
            "lipoproteins\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1761 line is : β-barrel proteins\n",
            "17\n",
            "β-barrel proteins\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 1762 line is :  Lol, BAM, and Lpt pathways.\n",
            "28\n",
            "Lol, BAM, and Lpt pathways.\n",
            "27\n",
            "-----------------------------------------------\n",
            "The 1765 line is : aminoglycosides\n",
            "15\n",
            "aminoglycosides\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 1766 line is : aztreonam\n",
            "9\n",
            "aztreonam\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1767 line is : cephalosporins\n",
            "14\n",
            "cephalosporins\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1768 line is : clindamycin\n",
            "11\n",
            "clindamycin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1769 line is : imipenem\n",
            "8\n",
            "imipenem\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 1770 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1771 line is :  trimethoprim-sulfamethoxazole\n",
            "30\n",
            "trimethoprim-sulfamethoxazole\n",
            "29\n",
            "-----------------------------------------------\n",
            "The 1772 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1773 line is : aminoglycosides\n",
            "15\n",
            "aminoglycosides\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 1774 line is : chloramphenicol\n",
            "15\n",
            "chloramphenicol\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 1775 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1776 line is : tetracycline\n",
            "12\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1777 line is : vancomycin\n",
            "10\n",
            "vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1778 line is : enterococcal \n",
            "13\n",
            "enterococcal\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1792 line is : linezolid\n",
            "9\n",
            "linezolid\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1793 line is : beta-lactams\n",
            "12\n",
            "beta-lactams\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1794 line is : glycopeptides\n",
            "13\n",
            "glycopeptides\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1795 line is : enterococci\n",
            "11\n",
            "enterococci\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1796 line is : Gram-positive pathogens\n",
            "23\n",
            "Gram-positive pathogens\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 1797 line is : optrA\n",
            "5\n",
            "optrA\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1798 line is : cfr\n",
            "3\n",
            "cfr\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 1799 line is : Penicillin\n",
            "10\n",
            "Penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1800 line is : Streptococcus pneumoniae\n",
            "24\n",
            "Streptococcus pneumoniae\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 1801 line is : enterococci\n",
            "11\n",
            "enterococci\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1802 line is : oxacillin\n",
            "9\n",
            "oxacillin\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1803 line is : vancomycin\n",
            "10\n",
            "vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1804 line is : MRSA\n",
            "4\n",
            "MRSA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 1805 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1806 line is : E. faecalis\n",
            "11\n",
            "E. faecalis\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1807 line is : E. faecium\n",
            "10\n",
            "E. faecium\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1809 line is : gatifloxacin\n",
            "12\n",
            "gatifloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1810 line is : moxifloxacin\n",
            "12\n",
            "moxifloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1811 line is : gemifloxacin\n",
            "12\n",
            "gemifloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1812 line is : gatifloxacin\n",
            "12\n",
            "gatifloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1813 line is : beta-lactams\n",
            "12\n",
            "beta-lactams\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1814 line is : cephalosporins\n",
            "14\n",
            "cephalosporins\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1815 line is : carbapenems\n",
            "11\n",
            "carbapenems\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1816 line is : ceftriaxone\n",
            "11\n",
            "ceftriaxone\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1817 line is : ceftazidime\n",
            "11\n",
            "ceftazidime\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1818 line is : beta-lactamase\n",
            "14\n",
            "beta-lactamase\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1819 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1820 line is : cefepime\n",
            "8\n",
            "cefepime\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 1821 line is : meropenem\n",
            "9\n",
            "meropenem\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1823 line is :  Malassezia spp\n",
            "15\n",
            "Malassezia spp\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1824 line is :  Pseudomonas spp\n",
            "16\n",
            "Pseudomonas spp\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 1825 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1826 line is : cefoxitin\n",
            "9\n",
            "cefoxitin\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1827 line is : methicillin\n",
            "11\n",
            "methicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1828 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1829 line is : ampicillin\n",
            "10\n",
            "ampicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1830 line is : Enterococcus faecalis\n",
            "21\n",
            "Enterococcus faecalis\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1831 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1832 line is : nucleic acid-mediated amplification technologies\n",
            "48\n",
            "nucleic acid-mediated amplification technologies\n",
            "48\n",
            "-----------------------------------------------\n",
            "The 1833 line is : proteomic and other \"omic\" methodologies\n",
            "40\n",
            "proteomic and other \"omic\" methodologies\n",
            "40\n",
            "-----------------------------------------------\n",
            "The 1834 line is : sequencing\n",
            "10\n",
            "sequencing\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1838 line is : SENTRY\n",
            "6\n",
            "SENTRY\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 1842 line is : cefinase paper discs\n",
            "20\n",
            "cefinase paper discs\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 1843 line is : β-lactamase enzymes\n",
            "19\n",
            "β-lactamase enzymes\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 1844 line is : Staphylococcus spp.\n",
            "19\n",
            "Staphylococcus spp.\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 1859 line is : efflux pump inhibitor\n",
            "21\n",
            "efflux pump inhibitor\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1860 line is : siderophore\n",
            "11\n",
            "siderophore\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1861 line is : A. baumannii\n",
            "12\n",
            "A. baumannii\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1862 line is : disc diffusion\n",
            "14\n",
            "disc diffusion\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1863 line is :  agar dilution\n",
            "14\n",
            "agar dilution\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1864 line is : fluoroquinolones\n",
            "16\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 1865 line is : aminoglycosides\n",
            "15\n",
            "aminoglycosides\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 1866 line is : tobramycin\n",
            "10\n",
            "tobramycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1867 line is : Gram-negative bacteria\n",
            "22\n",
            "Gram-negative bacteria\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 1868 line is : Klebsiella spp\n",
            "14\n",
            "Klebsiella spp\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1869 line is : Staphylococcus spp\n",
            "18\n",
            "Staphylococcus spp\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 1870 line is : Escherichia coli\n",
            "16\n",
            "Escherichia coli\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 1871 line is : Staphylococcus spp\n",
            "18\n",
            "Staphylococcus spp\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 1872 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 1873 line is : ampicillin\n",
            "10\n",
            "ampicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1874 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 1875 line is : ceftazidime\n",
            "11\n",
            "ceftazidime\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1876 line is : florfenicol\n",
            "11\n",
            "florfenicol\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1877 line is : amikacin\n",
            "8\n",
            "amikacin\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 1878 line is : meropenem\n",
            "9\n",
            "meropenem\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1879 line is : polymyxin\n",
            "9\n",
            "polymyxin\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1880 line is : Staphylococcus spp\n",
            "18\n",
            "Staphylococcus spp\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 1881 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1882 line is : rimethoprim-sulfamethoxazole\n",
            "28\n",
            "rimethoprim-sulfamethoxazole\n",
            "28\n",
            "-----------------------------------------------\n",
            "The 1883 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1884 line is : vancomycin\n",
            "10\n",
            "vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1885 line is : nitrofurantoin\n",
            "14\n",
            "nitrofurantoin\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1886 line is : Escherichia coli\n",
            "16\n",
            "Escherichia coli\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 1887 line is : Klebsiella pneumoniae\n",
            "21\n",
            "Klebsiella pneumoniae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1888 line is : cefotaxime\n",
            "10\n",
            "cefotaxime\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1889 line is :  Acinetobacter baumannii\n",
            "24\n",
            "Acinetobacter baumannii\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 1890 line is : imipenem\n",
            "8\n",
            "imipenem\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 1891 line is : meropenem\n",
            "9\n",
            "meropenem\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1898 line is :  Gram-negative bacteria\n",
            "23\n",
            "Gram-negative bacteria\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 1910 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1914 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1915 line is : bovine intramammary infections \n",
            "31\n",
            "bovine intramammary infections\n",
            "30\n",
            "-----------------------------------------------\n",
            "The 1916 line is : S. typhi\n",
            "8\n",
            "S. typhi\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 1917 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1918 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1919 line is : clindamycin\n",
            "11\n",
            "clindamycin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1920 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1921 line is : gentamycin\n",
            "10\n",
            "gentamycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1922 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1923 line is : zoonotic salmonellas \n",
            "21\n",
            "zoonotic salmonellas\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 1924 line is : Ceftiofur\n",
            "9\n",
            "Ceftiofur\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1925 line is : cephalotin\n",
            "10\n",
            "cephalotin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1926 line is : AgNPs\n",
            "5\n",
            "AgNPs\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1927 line is : AgNPs\n",
            "5\n",
            "AgNPs\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 1928 line is : green synthesis of AgNPs\n",
            "24\n",
            "green synthesis of AgNPs\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 1929 line is : Salmonella spp\n",
            "14\n",
            "Salmonella spp\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1930 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1931 line is : Salmonella typhimurium\n",
            "22\n",
            "Salmonella typhimurium\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 1932 line is : gyrA gene\n",
            "9\n",
            "gyrA gene\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1933 line is : PCR LightCycler-based gyrA mutation assay\n",
            "41\n",
            "PCR LightCycler-based gyrA mutation assay\n",
            "41\n",
            "-----------------------------------------------\n",
            "The 1936 line is : Salmonella typhi\n",
            "16\n",
            "Salmonella typhi\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 1956 line is : Campylobacter jejuni\n",
            "20\n",
            "Campylobacter jejuni\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 1957 line is : Campylobacter coli\n",
            "18\n",
            "Campylobacter coli\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 1958 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1959 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1960 line is : macrolides\n",
            "10\n",
            "macrolides\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1961 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1962 line is : fluoroquinolones\n",
            "16\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 1963 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1964 line is : E. faecalis\n",
            "11\n",
            "E. faecalis\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1965 line is :  microdilution method\n",
            "21\n",
            "microdilution method\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 1967 line is : Tetracyclines \n",
            "14\n",
            "Tetracyclines\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1969 line is : Campylobacter \n",
            "14\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1982 line is : Methicillin\n",
            "11\n",
            "Methicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1983 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 1985 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1986 line is : β-lactamase cefotaximase\n",
            "24\n",
            "β-lactamase cefotaximase\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 1987 line is : 16S ribosomal RNA\n",
            "17\n",
            "16S ribosomal RNA\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 1988 line is : immunoassays\n",
            "12\n",
            "immunoassays\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 1989 line is : K. pneumoniae\n",
            "13\n",
            "K. pneumoniae\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1990 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 1991 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 1992 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 1993 line is : MRSA\n",
            "4\n",
            "MRSA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 1994 line is : vancomycin\n",
            "10\n",
            "vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 1995 line is : MRSA isolation\n",
            "14\n",
            "MRSA isolation\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 1996 line is : MRSA\n",
            "4\n",
            "MRSA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 1997 line is : MRSA\n",
            "4\n",
            "MRSA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 1998 line is : Dalbavancin\n",
            "11\n",
            "Dalbavancin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 1999 line is : tigecycline \n",
            "12\n",
            "tigecycline\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2000 line is : ceftobiprole\n",
            "12\n",
            "ceftobiprole\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2001 line is : beta-lactams/beta-lactamase\n",
            "27\n",
            "beta-lactams/beta-lactamase\n",
            "27\n",
            "-----------------------------------------------\n",
            "The 2002 line is : quinolones\n",
            "10\n",
            "quinolones\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2003 line is : cephalosporins\n",
            "14\n",
            "cephalosporins\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2004 line is : aminoglycosides\n",
            "15\n",
            "aminoglycosides\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2007 line is : Acinetobacter baumannii\n",
            "23\n",
            "Acinetobacter baumannii\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 2008 line is : carbapenem \n",
            "11\n",
            "carbapenem\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2012 line is : electrochemical immunoassays (EC-IA)\n",
            "36\n",
            "electrochemical immunoassays (EC-IA)\n",
            "36\n",
            "-----------------------------------------------\n",
            "The 2013 line is : mass spectral immunoassays (MS-IA)\n",
            "34\n",
            "mass spectral immunoassays (MS-IA)\n",
            "34\n",
            "-----------------------------------------------\n",
            "The 2014 line is : signal ion emission reactive release amplification (SIERRA)\n",
            "59\n",
            "signal ion emission reactive release amplification (SIERRA)\n",
            "59\n",
            "-----------------------------------------------\n",
            "The 2016 line is : Escherichia coli\n",
            "16\n",
            "Escherichia coli\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2017 line is : Pseudomonas aeruginosa\n",
            "22\n",
            "Pseudomonas aeruginosa\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 2018 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2019 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2020 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2021 line is : Klebsiella pneumoniae\n",
            "21\n",
            "Klebsiella pneumoniae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2022 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2031 line is : Neisseria gonorrhoeae\n",
            "21\n",
            "Neisseria gonorrhoeae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2032 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2033 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2034 line is : P. stuartii\n",
            "11\n",
            "P. stuartii\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2035 line is :  P. aeruginosa\n",
            "14\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2036 line is : Enterobacteriaceae\n",
            "18\n",
            "Enterobacteriaceae\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 2037 line is : A. baumannii\n",
            "12\n",
            "A. baumannii\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2038 line is : imipenem\n",
            "8\n",
            "imipenem\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2039 line is : beta-lactams\n",
            "12\n",
            "beta-lactams\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2040 line is : staphylococci\n",
            "13\n",
            "staphylococci\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2042 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2043 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2044 line is : oxacillin\n",
            "9\n",
            "oxacillin\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2045 line is : oxacillin\n",
            "9\n",
            "oxacillin\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2046 line is : Staphylococcus haemolyticus\n",
            "27\n",
            "Staphylococcus haemolyticus\n",
            "27\n",
            "-----------------------------------------------\n",
            "The 2047 line is : Staphylococcus epidermidis\n",
            "26\n",
            "Staphylococcus epidermidis\n",
            "26\n",
            "-----------------------------------------------\n",
            "The 2048 line is : TCM\n",
            "3\n",
            "TCM\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 2050 line is : Imipenem\n",
            "8\n",
            "Imipenem\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2051 line is : quinolone\n",
            "9\n",
            "quinolone\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2052 line is : staphylococci\n",
            "13\n",
            "staphylococci\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2053 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2054 line is : Clostridioides difficile\n",
            "24\n",
            "Clostridioides difficile\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 2055 line is : Coxiella burnetii\n",
            "17\n",
            "Coxiella burnetii\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 2056 line is : C. difficile\n",
            "12\n",
            "C. difficile\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2057 line is : pefloxacin\n",
            "10\n",
            "pefloxacin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2058 line is : CDI therapies\n",
            "13\n",
            "CDI therapies\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2059 line is : C. difficile\n",
            "12\n",
            "C. difficile\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2060 line is : ceftozolane\n",
            "11\n",
            "ceftozolane\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2061 line is : tazobactam\n",
            "10\n",
            "tazobactam\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2062 line is : carbapenems\n",
            "11\n",
            "carbapenems\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2063 line is : cephalosporins\n",
            "14\n",
            "cephalosporins\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2064 line is : ceftazidime\n",
            "11\n",
            "ceftazidime\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2065 line is : avibactam\n",
            "9\n",
            "avibactam\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2066 line is : plazomicin\n",
            "10\n",
            "plazomicin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2068 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2069 line is : N. gonorrhoeae\n",
            "14\n",
            "N. gonorrhoeae\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2070 line is :  Acinetobacter baumannii\n",
            "24\n",
            "Acinetobacter baumannii\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 2071 line is : Enterobacteriaceae\n",
            "18\n",
            "Enterobacteriaceae\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 2072 line is : imipenem\n",
            "8\n",
            "imipenem\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2073 line is : Pseudomonas aeruginosa\n",
            "22\n",
            "Pseudomonas aeruginosa\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 2076 line is : Enterococcus faecalis\n",
            "21\n",
            "Enterococcus faecalis\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2077 line is : ciprofloxacin \n",
            "14\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2078 line is : ciprofloxacin \n",
            "14\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2091 line is : quinolones\n",
            "10\n",
            "quinolones\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2092 line is : carbapenems\n",
            "11\n",
            "carbapenems\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2093 line is : Pseudomonas aeruginosa\n",
            "22\n",
            "Pseudomonas aeruginosa\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 2094 line is : ofloxacin\n",
            "9\n",
            "ofloxacin\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2095 line is : ceftazidime\n",
            "11\n",
            "ceftazidime\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2096 line is : imipenem\n",
            "8\n",
            "imipenem\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2097 line is : Fluoroquinolones \n",
            "17\n",
            "Fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2098 line is : quinolones\n",
            "10\n",
            "quinolones\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2099 line is : fluoroquinolones\n",
            "16\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2100 line is : DNA gyrase\n",
            "10\n",
            "DNA gyrase\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2101 line is : topoisomerase IV \n",
            "17\n",
            "topoisomerase IV\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2102 line is :  P. aeruginosa \n",
            "15\n",
            "P. aeruginosa\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2103 line is : quinolones \n",
            "11\n",
            "quinolones\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2104 line is : quinolone\n",
            "9\n",
            "quinolone\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2105 line is : macrolide\n",
            "9\n",
            "macrolide\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2106 line is : C. coli\n",
            "7\n",
            "C. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2107 line is : Campylobacter jejuni\n",
            "20\n",
            "Campylobacter jejuni\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 2108 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2109 line is : biofilms\n",
            "8\n",
            "biofilms\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2110 line is : multidrug-resistant (MDR)\n",
            "25\n",
            "multidrug-resistant (MDR)\n",
            "25\n",
            "-----------------------------------------------\n",
            "The 2111 line is : Polymyxins\n",
            "10\n",
            "Polymyxins\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2112 line is : fluoroquinolones\n",
            "16\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2113 line is : gram-positive\n",
            "13\n",
            "gram-positive\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2114 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2115 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2116 line is : S. pneumoniae\n",
            "13\n",
            "S. pneumoniae\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2117 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2118 line is : MRSA\n",
            "4\n",
            "MRSA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2119 line is : N. gonorrhoeae\n",
            "14\n",
            "N. gonorrhoeae\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2120 line is : guanidinium\n",
            "11\n",
            "guanidinium\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2121 line is : porin-forming protein\n",
            "21\n",
            "porin-forming protein\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2123 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2124 line is : meropenem (MEPM)\n",
            "16\n",
            "meropenem (MEPM)\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2126 line is : fluoroquinolones \n",
            "17\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2140 line is : rifampin \n",
            "9\n",
            "rifampin\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2141 line is : isoniazid (INH)\n",
            "15\n",
            "isoniazid (INH)\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2142 line is : katG\n",
            "4\n",
            "katG\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2143 line is : INH\n",
            "3\n",
            "INH\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 2144 line is : INH\n",
            "3\n",
            "INH\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 2145 line is : inhA\n",
            "4\n",
            "inhA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2146 line is : beta-ketoacyl-acyl carrier protein synthase\n",
            "43\n",
            "beta-ketoacyl-acyl carrier protein synthase\n",
            "43\n",
            "-----------------------------------------------\n",
            "The 2147 line is : kasA\n",
            "4\n",
            "kasA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2148 line is : Shigella spp\n",
            "12\n",
            "Shigella spp\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2149 line is : Shigella flexneri\n",
            "17\n",
            "Shigella flexneri\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 2150 line is : Shigella sonnei\n",
            "15\n",
            "Shigella sonnei\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2151 line is : Streptomycin\n",
            "12\n",
            "Streptomycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2152 line is : ampicillin\n",
            "10\n",
            "ampicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2153 line is :  tetracycline\n",
            "13\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2154 line is : trimethoprim\n",
            "12\n",
            "trimethoprim\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2155 line is : S. flexneri\n",
            "11\n",
            "S. flexneri\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2156 line is :  trimethoprim\n",
            "13\n",
            "trimethoprim\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2157 line is :  oxa\n",
            "4\n",
            "oxa\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 2158 line is : Dihydrofolate reductase 1 genes (dfrA1)\n",
            "39\n",
            "Dihydrofolate reductase 1 genes (dfrA1)\n",
            "39\n",
            "-----------------------------------------------\n",
            "The 2159 line is : gyrA\n",
            "4\n",
            "gyrA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2160 line is : Chloramphenicol\n",
            "15\n",
            "Chloramphenicol\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2161 line is : embB gene\n",
            "9\n",
            "embB gene\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2162 line is : Ethambutol \n",
            "11\n",
            "Ethambutol\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2163 line is : macrolide\n",
            "9\n",
            "macrolide\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2164 line is : quinolone\n",
            "9\n",
            "quinolone\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2165 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2166 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2167 line is : macrolides\n",
            "10\n",
            "macrolides\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2168 line is : fluoroquinolones\n",
            "16\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2169 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2170 line is : Kanamycin\n",
            "9\n",
            "Kanamycin\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2171 line is : rrs\n",
            "3\n",
            "rrs\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 2172 line is : 16S rRNA\n",
            "8\n",
            "16S rRNA\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2173 line is : levofloxacin\n",
            "12\n",
            "levofloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2174 line is : OH-CATH30\n",
            "9\n",
            "OH-CATH30\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2175 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2176 line is : keratitis\n",
            "9\n",
            "keratitis\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2177 line is : cefoperazone\n",
            "12\n",
            "cefoperazone\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2178 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2179 line is : gentamicin\n",
            "10\n",
            "gentamicin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2180 line is : microdilution checkerboard assay\n",
            "32\n",
            "microdilution checkerboard assay\n",
            "32\n",
            "-----------------------------------------------\n",
            "The 2181 line is : levofloxacin\n",
            "12\n",
            "levofloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2182 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2183 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2184 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2185 line is :  OH-CATH30\n",
            "10\n",
            "OH-CATH30\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2186 line is : OH-CATH30\n",
            "9\n",
            "OH-CATH30\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2187 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2188 line is : levofloxacin\n",
            "12\n",
            "levofloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2189 line is : P. aeruginosa keratitis\n",
            "23\n",
            "P. aeruginosa keratitis\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 2193 line is : pncA\n",
            "4\n",
            "pncA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2194 line is : pyrazinamide \n",
            "13\n",
            "pyrazinamide\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2195 line is : pyrazinamidase \n",
            "15\n",
            "pyrazinamidase\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2196 line is : fluoroquinolone\n",
            "15\n",
            "fluoroquinolone\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2207 line is : gyrase\n",
            "6\n",
            "gyrase\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2208 line is : fluoroquinolones\n",
            "16\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2209 line is : nalidixic acid\n",
            "14\n",
            "nalidixic acid\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2210 line is : fluoroquinolone\n",
            "15\n",
            "fluoroquinolone\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2211 line is : Escherichia coli Q2\n",
            "19\n",
            "Escherichia coli Q2\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 2212 line is : norfloxacin\n",
            "11\n",
            "norfloxacin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2213 line is : fluoroquinolones\n",
            "16\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2214 line is : nalidixic acid\n",
            "14\n",
            "nalidixic acid\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2215 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2216 line is : fluoroquinolones\n",
            "16\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2217 line is : nalidixic acid\n",
            "14\n",
            "nalidixic acid\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2218 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2219 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2220 line is : norfloxacin\n",
            "11\n",
            "norfloxacin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2221 line is : E. coli Q1\n",
            "10\n",
            "E. coli Q1\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2222 line is : E. coli Q2\n",
            "10\n",
            "E. coli Q2\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2223 line is : quinolone\n",
            "9\n",
            "quinolone\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2224 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2225 line is : tosufloxacin\n",
            "12\n",
            "tosufloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2226 line is : tosufloxacin\n",
            "12\n",
            "tosufloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2227 line is : tosufloxacin\n",
            "12\n",
            "tosufloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2228 line is : proton-pump inhibitor\n",
            "21\n",
            "proton-pump inhibitor\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2229 line is : quinolone\n",
            "9\n",
            "quinolone\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2230 line is : minimum inhibitory concentration (MIC)\n",
            "38\n",
            "minimum inhibitory concentration (MIC)\n",
            "38\n",
            "-----------------------------------------------\n",
            "The 2231 line is : Salmonella\n",
            "10\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2239 line is : Shigella flexneri 2a strain\n",
            "27\n",
            "Shigella flexneri 2a strain\n",
            "27\n",
            "-----------------------------------------------\n",
            "The 2255 line is : drug-resistant pathogens\n",
            "24\n",
            "drug-resistant pathogens\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 2256 line is : enterococci\n",
            "11\n",
            "enterococci\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2257 line is : Enterococcus faecalis\n",
            "21\n",
            "Enterococcus faecalis\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2258 line is : Enterococcus faecium\n",
            "20\n",
            "Enterococcus faecium\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 2259 line is : opportunistic bacterial pathogens\n",
            "33\n",
            "opportunistic bacterial pathogens\n",
            "33\n",
            "-----------------------------------------------\n",
            "The 2260 line is : Global Antimicrobial Resistance Surveillance System\n",
            "51\n",
            "Global Antimicrobial Resistance Surveillance System\n",
            "51\n",
            "-----------------------------------------------\n",
            "The 2261 line is : Enterococci\n",
            "11\n",
            "Enterococci\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2270 line is : carbapenem-resistant Acinetobacter spp.\n",
            "39\n",
            "carbapenem-resistant Acinetobacter spp.\n",
            "39\n",
            "-----------------------------------------------\n",
            "The 2271 line is : carbapenem-resistant Escherichia coli\n",
            "37\n",
            "carbapenem-resistant Escherichia coli\n",
            "37\n",
            "-----------------------------------------------\n",
            "The 2289 line is : Staphylococcus\n",
            "14\n",
            "Staphylococcus\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2291 line is : chemotherapy\n",
            "12\n",
            "chemotherapy\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2292 line is : dalbavancin\n",
            "11\n",
            "dalbavancin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2293 line is : oritavancin\n",
            "11\n",
            "oritavancin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2294 line is : phenotypic tests \n",
            "17\n",
            "phenotypic tests\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2295 line is : gram-positive cocci\n",
            "19\n",
            "gram-positive cocci\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 2296 line is : Streptococcus spp\n",
            "17\n",
            "Streptococcus spp\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 2297 line is : Staphylococcus spp\n",
            "18\n",
            "Staphylococcus spp\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 2298 line is : Staphylococcus spp\n",
            "18\n",
            "Staphylococcus spp\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 2301 line is :  P. aeruginosa\n",
            "14\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2322 line is : Salmonella\n",
            "10\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2323 line is : ceftriaxone\n",
            "11\n",
            "ceftriaxone\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2324 line is : Neisseria gonorrhoeae\n",
            "21\n",
            "Neisseria gonorrhoeae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2325 line is : FC428\n",
            "5\n",
            "FC428\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 2326 line is : ceftriaxone\n",
            "11\n",
            "ceftriaxone\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2327 line is : ß-lactam\n",
            "8\n",
            "ß-lactam\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2328 line is : predicted no effect concentrations for resistance (PNECRs)\n",
            "58\n",
            "predicted no effect concentrations for resistance (PNECRs)\n",
            "58\n",
            "-----------------------------------------------\n",
            "The 2329 line is : predicted no effect concentrations for persistence (PNECPs)\n",
            "59\n",
            "predicted no effect concentrations for persistence (PNECPs)\n",
            "59\n",
            "-----------------------------------------------\n",
            "The 2350 line is : doxycycline\n",
            "11\n",
            "doxycycline\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2351 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2352 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2353 line is : carbapenemase-producing organisms (CPO).\n",
            "40\n",
            "carbapenemase-producing organisms (CPO).\n",
            "40\n",
            "-----------------------------------------------\n",
            "The 2354 line is : anti-infective therapy\n",
            "22\n",
            "anti-infective therapy\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 2355 line is : carbapenem\n",
            "10\n",
            "carbapenem\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2356 line is : COVID-19\n",
            "8\n",
            "COVID-19\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2357 line is : Doxycycline\n",
            "11\n",
            "Doxycycline\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2358 line is : doxycycline\n",
            "11\n",
            "doxycycline\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2359 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2360 line is : Enterococcus\n",
            "12\n",
            "Enterococcus\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2361 line is : doxycycline\n",
            "11\n",
            "doxycycline\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2362 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2363 line is : doxycycline\n",
            "11\n",
            "doxycycline\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2364 line is : Erythromycin\n",
            "12\n",
            "Erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2368 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2369 line is :  E. coli\n",
            "8\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2370 line is : Gram-negative organisms\n",
            "23\n",
            "Gram-negative organisms\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 2373 line is : extended-spectrum ß-lactamase-producing Escherichia coli\n",
            "56\n",
            "extended-spectrum ß-lactamase-producing Escherichia coli\n",
            "56\n",
            "-----------------------------------------------\n",
            "The 2374 line is : carbapenem-resistance genes (CRG)\n",
            "33\n",
            "carbapenem-resistance genes (CRG)\n",
            "33\n",
            "-----------------------------------------------\n",
            "The 2388 line is : enteropathogen antimicrobial resistance\n",
            "39\n",
            "enteropathogen antimicrobial resistance\n",
            "39\n",
            "-----------------------------------------------\n",
            "The 2389 line is :  enteropathogens\n",
            "16\n",
            "enteropathogens\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2390 line is : fluoroquinolone\n",
            "15\n",
            "fluoroquinolone\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2392 line is : fluoroquinolones\n",
            "16\n",
            "fluoroquinolones\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2393 line is : C. jejuni \n",
            "10\n",
            "C. jejuni\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2394 line is : Salmonella\n",
            "10\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2395 line is : Salmonella Typhimurium.\n",
            "23\n",
            "Salmonella Typhimurium.\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 2396 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2397 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2398 line is : Salmonella \n",
            "11\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2399 line is : Salmonella \n",
            "11\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2400 line is : antibiotic-resistant bacteria\n",
            "29\n",
            "antibiotic-resistant bacteria\n",
            "29\n",
            "-----------------------------------------------\n",
            "The 2401 line is : renal parenchyma\n",
            "16\n",
            "renal parenchyma\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2402 line is : bacteraemia\n",
            "11\n",
            "bacteraemia\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2405 line is : Campylobacter spp.\n",
            "18\n",
            "Campylobacter spp.\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 2406 line is : Salmonella spp.\n",
            "15\n",
            "Salmonella spp.\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2421 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2422 line is : Staphylococcus \n",
            "15\n",
            "Staphylococcus\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2423 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2424 line is : Staphylococcus\n",
            "14\n",
            "Staphylococcus\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2425 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2426 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2427 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2428 line is : rifampicin\n",
            "10\n",
            "rifampicin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2429 line is : Escherichia coli O157:H7\n",
            "24\n",
            "Escherichia coli O157:H7\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 2430 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2431 line is : Ampicillin \n",
            "11\n",
            "Ampicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2432 line is : plasmid transformation\n",
            "22\n",
            "plasmid transformation\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 2433 line is : ampP strepC strains\n",
            "19\n",
            "ampP strepC strains\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 2434 line is : ampP\n",
            "4\n",
            "ampP\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2435 line is : Staphylococcus\n",
            "14\n",
            "Staphylococcus\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2436 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2438 line is : ampP\n",
            "4\n",
            "ampP\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2439 line is : ampC\n",
            "4\n",
            "ampC\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2440 line is : strepC\n",
            "6\n",
            "strepC\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2441 line is : Staphylococcus\n",
            "14\n",
            "Staphylococcus\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2442 line is : Corynebacterium \n",
            "16\n",
            "Corynebacterium\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2443 line is : Streptococcus viridans\n",
            "22\n",
            "Streptococcus viridans\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 2444 line is : tobramycin\n",
            "10\n",
            "tobramycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2451 line is : ampicillin \n",
            "11\n",
            "ampicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2452 line is : ampC\n",
            "4\n",
            "ampC\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2453 line is : streptomycin \n",
            "13\n",
            "streptomycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2454 line is : strepC\n",
            "6\n",
            "strepC\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2455 line is : ampicillin \n",
            "11\n",
            "ampicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2456 line is : streptomycin \n",
            "13\n",
            "streptomycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2457 line is : ampC \n",
            "5\n",
            "ampC\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2458 line is : strepC\n",
            "6\n",
            "strepC\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2471 line is : Metagenomics\n",
            "12\n",
            "Metagenomics\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2472 line is :  antibiotic resistant genes (ARGs)\n",
            "34\n",
            "antibiotic resistant genes (ARGs)\n",
            "33\n",
            "-----------------------------------------------\n",
            "The 2473 line is : bacteriostatic\n",
            "14\n",
            "bacteriostatic\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2474 line is : bacteriolytic\n",
            "13\n",
            "bacteriolytic\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2475 line is : bacteriolytic\n",
            "13\n",
            "bacteriolytic\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2476 line is : bacteriostatic\n",
            "14\n",
            "bacteriostatic\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2477 line is : antibiotics\n",
            "11\n",
            "antibiotics\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2479 line is : enterobacteriaceae\n",
            "18\n",
            "enterobacteriaceae\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 2498 line is : IncL \n",
            "5\n",
            "IncL\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2499 line is : blaOXA-48\n",
            "9\n",
            "blaOXA-48\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2500 line is : ermA\n",
            "4\n",
            "ermA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2501 line is : msrA\n",
            "4\n",
            "msrA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2502 line is : E. faecalis\n",
            "11\n",
            "E. faecalis\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2503 line is : aac(6')-Ie-aph(2'')-Ia\n",
            "22\n",
            "aac(6')-Ie-aph(2'')-Ia\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 2504 line is : aph(3')-IIIa\n",
            "12\n",
            "aph(3')-IIIa\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2505 line is : β-lactam-β-lactamase inhibitor \n",
            "31\n",
            "β-lactam-β-lactamase inhibitor\n",
            "30\n",
            "-----------------------------------------------\n",
            "The 2506 line is : vanA\n",
            "4\n",
            "vanA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2507 line is : ermB\n",
            "4\n",
            "ermB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2508 line is : ermC\n",
            "4\n",
            "ermC\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2509 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2511 line is : multidrug-resistant\n",
            "19\n",
            "multidrug-resistant\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 2512 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2513 line is : cephalosporin\n",
            "13\n",
            "cephalosporin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2514 line is : TEM penicillinases \n",
            "19\n",
            "TEM penicillinases\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 2515 line is : CTX-M extended-spectrum β-lactamases\n",
            "36\n",
            "CTX-M extended-spectrum β-lactamases\n",
            "36\n",
            "-----------------------------------------------\n",
            "The 2516 line is : β-lactamases\n",
            "12\n",
            "β-lactamases\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2517 line is : carbapenems\n",
            "11\n",
            "carbapenems\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2518 line is : carbapenemase\n",
            "13\n",
            "carbapenemase\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2519 line is : OXA-48\n",
            "6\n",
            "OXA-48\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2520 line is : OXA-48\n",
            "6\n",
            "OXA-48\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2521 line is : OXA-48\n",
            "6\n",
            "OXA-48\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2522 line is : OXA-48\n",
            "6\n",
            "OXA-48\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2523 line is : carbapenems\n",
            "11\n",
            "carbapenems\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2524 line is : Vancomycin\n",
            "10\n",
            "Vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2525 line is : carbapenem\n",
            "10\n",
            "carbapenem\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2526 line is : Enterobacterales \n",
            "17\n",
            "Enterobacterales\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2527 line is : Enterobacterales\n",
            "16\n",
            "Enterobacterales\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2528 line is : OXA-48-like β-lactamases\n",
            "24\n",
            "OXA-48-like β-lactamases\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 2550 line is : Salmonella\n",
            "10\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2551 line is : Salmonella\n",
            "10\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2552 line is : Salmonella\n",
            "10\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2553 line is : Salmonella\n",
            "10\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2554 line is : PubMed\n",
            "6\n",
            "PubMed\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2555 line is : MEDLINE\n",
            "7\n",
            "MEDLINE\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2556 line is : imipenem-relebactam nonsusceptible Pseudomonas aeruginosa\n",
            "57\n",
            "imipenem-relebactam nonsusceptible Pseudomonas aeruginosa\n",
            "57\n",
            "-----------------------------------------------\n",
            "The 2557 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2558 line is : Klebsiella\n",
            "10\n",
            "Klebsiella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2559 line is : Enterococci\n",
            "11\n",
            "Enterococci\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2560 line is : MexAB\n",
            "5\n",
            "MexAB\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 2561 line is : MexEF\n",
            "5\n",
            "MexEF\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 2562 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2563 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2564 line is : Amoxicillin\n",
            "11\n",
            "Amoxicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2565 line is : Amoxyclav\n",
            "9\n",
            "Amoxyclav\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2566 line is : Co-trimoxazole\n",
            "14\n",
            "Co-trimoxazole\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2567 line is : Ceftriaxone\n",
            "11\n",
            "Ceftriaxone\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2568 line is : Moxifloxacin\n",
            "12\n",
            "Moxifloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2569 line is : Cefixime\n",
            "8\n",
            "Cefixime\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2570 line is : Ciprofloxacin\n",
            "13\n",
            "Ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2571 line is : Cefuroxime\n",
            "10\n",
            "Cefuroxime\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2572 line is : Ceftazidim\n",
            "10\n",
            "Ceftazidim\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2573 line is : Nalidixic acid\n",
            "14\n",
            "Nalidixic acid\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2574 line is : Cephotaxime\n",
            "11\n",
            "Cephotaxime\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2575 line is :  macrophages\n",
            "12\n",
            "macrophages\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2576 line is : Quinolone\n",
            "9\n",
            "Quinolone\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2577 line is : FupA/B\n",
            "6\n",
            "FupA/B\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2578 line is : CIP\n",
            "3\n",
            "CIP\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 2579 line is : Francisella \n",
            "12\n",
            "Francisella\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2587 line is : Salmonella\n",
            "10\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2604 line is : nosocomial\n",
            "10\n",
            "nosocomial\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2606 line is : bacteriocins\n",
            "12\n",
            "bacteriocins\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2629 line is : Campylobacter jejuni\n",
            "20\n",
            "Campylobacter jejuni\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 2630 line is : C. jejuni\n",
            "9\n",
            "C. jejuni\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2631 line is : Campylobacter spp\n",
            "17\n",
            "Campylobacter spp\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 2632 line is : macrolide\n",
            "9\n",
            "macrolide\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2633 line is : macrolide\n",
            "9\n",
            "macrolide\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2634 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2635 line is : C. jejuni\n",
            "9\n",
            "C. jejuni\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2636 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2637 line is : C. jejuni\n",
            "9\n",
            "C. jejuni\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2638 line is : C. jejuni\n",
            "9\n",
            "C. jejuni\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2639 line is : C. jejuni\n",
            "9\n",
            "C. jejuni\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2640 line is : azithromycin\n",
            "12\n",
            "azithromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2642 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2643 line is : azithromycin\n",
            "12\n",
            "azithromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2644 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2645 line is : clavulanic acid\n",
            "15\n",
            "clavulanic acid\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2646 line is : Campylobacter Amoxicillin\n",
            "25\n",
            "Campylobacter Amoxicillin\n",
            "25\n",
            "-----------------------------------------------\n",
            "The 2647 line is : macrolide\n",
            "9\n",
            "macrolide\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 2648 line is : Kirby-Bauer disc diffusion test.\n",
            "32\n",
            "Kirby-Bauer disc diffusion test.\n",
            "32\n",
            "-----------------------------------------------\n",
            "The 2649 line is : APEC\n",
            "4\n",
            "APEC\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2650 line is : AMR Clinical Reference Center\n",
            "29\n",
            "AMR Clinical Reference Center\n",
            "29\n",
            "-----------------------------------------------\n",
            "The 2651 line is : APEC\n",
            "4\n",
            "APEC\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2652 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2653 line is : amoxicillin\n",
            "11\n",
            "amoxicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2654 line is : tetracycline\n",
            "12\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2655 line is : doxycycline\n",
            "11\n",
            "doxycycline\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2656 line is : AMR CRC\n",
            "7\n",
            "AMR CRC\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2657 line is : Salmonella \n",
            "11\n",
            "Salmonella\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2659 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2661 line is : quinolone- and macrolide-resistant Campylobacter\n",
            "48\n",
            "quinolone- and macrolide-resistant Campylobacter\n",
            "48\n",
            "-----------------------------------------------\n",
            "The 2663 line is : macrolides \n",
            "11\n",
            "macrolides\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2664 line is : Campylobacter \n",
            "14\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2665 line is : macrolides\n",
            "10\n",
            "macrolides\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2666 line is : Campylobacter \n",
            "14\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2667 line is : amoxicillin\n",
            "11\n",
            "amoxicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2668 line is : clavulanic acid\n",
            "15\n",
            "clavulanic acid\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2688 line is : cefixime MIC\n",
            "12\n",
            "cefixime MIC\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2689 line is : NG-MAST ST1407\n",
            "14\n",
            "NG-MAST ST1407\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2690 line is : mosaic penA-34\n",
            "14\n",
            "mosaic penA-34\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2691 line is : cefixime\n",
            "8\n",
            "cefixime\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 2693 line is : benzylpenicillin\n",
            "16\n",
            "benzylpenicillin\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2694 line is : AMR testing\n",
            "11\n",
            "AMR testing\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2695 line is : NG-MAST\n",
            "7\n",
            "NG-MAST\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2696 line is : sequencing of the AMR determinant penA\n",
            "38\n",
            "sequencing of the AMR determinant penA\n",
            "38\n",
            "-----------------------------------------------\n",
            "The 2697 line is : tetracycline\n",
            "12\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2698 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2699 line is : N. gonorrhoeae \n",
            "15\n",
            "N. gonorrhoeae\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2700 line is : CARB-X\n",
            "6\n",
            "CARB-X\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2701 line is :  BARDA\n",
            "6\n",
            "BARDA\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 2702 line is : Mobile genetic elements (MGEs)\n",
            "30\n",
            "Mobile genetic elements (MGEs)\n",
            "30\n",
            "-----------------------------------------------\n",
            "The 2703 line is : Escherichia coli\n",
            "16\n",
            "Escherichia coli\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2704 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2705 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2706 line is :  N. gonorrhoeae multiantigen sequence typing (NG-MAST) \n",
            "55\n",
            "N. gonorrhoeae multiantigen sequence typing (NG-MAST)\n",
            "54\n",
            "-----------------------------------------------\n",
            "The 2707 line is : gonococcal\n",
            "10\n",
            "gonococcal\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2708 line is : gonococcal\n",
            "10\n",
            "gonococcal\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 2709 line is : NG-MAST\n",
            "7\n",
            "NG-MAST\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2710 line is : AMR testing\n",
            "11\n",
            "AMR testing\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2711 line is : penA\n",
            "4\n",
            "penA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2734 line is : matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) \n",
            "71\n",
            "matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF)\n",
            "70\n",
            "-----------------------------------------------\n",
            "The 2735 line is : spectrometry,\n",
            "13\n",
            "spectrometry,\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2736 line is :  Fourier transform infrared (FTIR) spectroscopy\n",
            "47\n",
            "Fourier transform infrared (FTIR) spectroscopy\n",
            "46\n",
            "-----------------------------------------------\n",
            "The 2737 line is : microfluidics technology\n",
            "24\n",
            "microfluidics technology\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 2738 line is : whole genome sequencing (WGS)\n",
            "29\n",
            "whole genome sequencing (WGS)\n",
            "29\n",
            "-----------------------------------------------\n",
            "The 2767 line is : horizontal gene transfer\n",
            "24\n",
            "horizontal gene transfer\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 2800 line is :  environmental resistomes \n",
            "26\n",
            "environmental resistomes\n",
            "25\n",
            "-----------------------------------------------\n",
            "The 2801 line is : zoonotic pathogens\n",
            "18\n",
            "zoonotic pathogens\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 2852 line is : mAbs\n",
            "4\n",
            "mAbs\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2853 line is : Klebsiella pneumoniae\n",
            "21\n",
            "Klebsiella pneumoniae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2854 line is : Mycobacterium tuberculosis\n",
            "26\n",
            "Mycobacterium tuberculosis\n",
            "26\n",
            "-----------------------------------------------\n",
            "The 2855 line is : Salmonella enterica\n",
            "19\n",
            "Salmonella enterica\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 2856 line is : K. pneumoniae\n",
            "13\n",
            "K. pneumoniae\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 2857 line is : M. tuberculosis\n",
            "15\n",
            "M. tuberculosis\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2858 line is : S. enterica\n",
            "11\n",
            "S. enterica\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2863 line is : Klebsiella pneumoniae\n",
            "21\n",
            "Klebsiella pneumoniae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 2864 line is : Mycobacterium tuberculosis\n",
            "26\n",
            "Mycobacterium tuberculosis\n",
            "26\n",
            "-----------------------------------------------\n",
            "The 2865 line is : Salmonella enterica\n",
            "19\n",
            "Salmonella enterica\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 2871 line is : mAbs\n",
            "4\n",
            "mAbs\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2872 line is : mAbs\n",
            "4\n",
            "mAbs\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 2922 line is : RSA-AMR\n",
            "7\n",
            "RSA-AMR\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2923 line is : model learning.\n",
            "15\n",
            "model learning.\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 2924 line is : Listeria monocytogenes\n",
            "22\n",
            "Listeria monocytogenes\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 2925 line is : listeriosis\n",
            "11\n",
            "listeriosis\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 2926 line is :  relative solvent accessibility (RSA) \n",
            "38\n",
            "relative solvent accessibility (RSA)\n",
            "37\n",
            "-----------------------------------------------\n",
            "The 2927 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 2928 line is : \"Salmonella spp.\"\n",
            "17\n",
            "\"Salmonella spp.\"\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 2954 line is :  AMR surveillance\n",
            "17\n",
            "AMR surveillance\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2955 line is : Clostridioides difficile\n",
            "24\n",
            "Clostridioides difficile\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 2956 line is : C. difficile \n",
            "13\n",
            "C. difficile\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 2957 line is : C. difficile .\n",
            "14\n",
            "C. difficile .\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 2958 line is : MDR\n",
            "3\n",
            "MDR\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 2959 line is : L. monocytogenes\n",
            "16\n",
            "L. monocytogenes\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2960 line is : National Pathogen Detection Isolate Browser (NPDIB)\n",
            "51\n",
            "National Pathogen Detection Isolate Browser (NPDIB)\n",
            "51\n",
            "-----------------------------------------------\n",
            "The 2961 line is : L. monocytogenes\n",
            "16\n",
            "L. monocytogenes\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2962 line is : L. monocytogenes\n",
            "16\n",
            "L. monocytogenes\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 2963 line is : Hierarchical clustering \n",
            "24\n",
            "Hierarchical clustering\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 2964 line is :  tetM-positive transposon \n",
            "26\n",
            "tetM-positive transposon\n",
            "25\n",
            "-----------------------------------------------\n",
            "The 2965 line is : Tn6944\n",
            "6\n",
            "Tn6944\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 2966 line is :  fosX,\n",
            "6\n",
            "fosX,\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 2967 line is :  lin\n",
            "4\n",
            "lin\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 2968 line is : abc-f\n",
            "5\n",
            "abc-f\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 2969 line is :  tet(M) \n",
            "8\n",
            "tet(M)\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3017 line is : Escherichia coli\n",
            "16\n",
            "Escherichia coli\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3018 line is : Staphylococcus spp.\n",
            "19\n",
            "Staphylococcus spp.\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 3019 line is : Salmonella spp.\n",
            "15\n",
            "Salmonella spp.\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3020 line is : PATRIC database\n",
            "15\n",
            "PATRIC database\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3021 line is : prophylaxis\n",
            "11\n",
            "prophylaxis\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3031 line is : Escherichia coli\n",
            "16\n",
            "Escherichia coli\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3032 line is : Klebsiella pneumoniae\n",
            "21\n",
            "Klebsiella pneumoniae\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3033 line is : Enterobacteriaceae \n",
            "19\n",
            "Enterobacteriaceae\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 3042 line is : Campylobacter \n",
            "14\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3049 line is :  colorimetric-based microfluidic sensors\n",
            "40\n",
            "colorimetric-based microfluidic sensors\n",
            "39\n",
            "-----------------------------------------------\n",
            "The 3050 line is : IoT-assisted AMR surveillance\n",
            "29\n",
            "IoT-assisted AMR surveillance\n",
            "29\n",
            "-----------------------------------------------\n",
            "The 3051 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3052 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3053 line is : PATRIC\n",
            "6\n",
            "PATRIC\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 3054 line is : RAST\n",
            "4\n",
            "RAST\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3063 line is : HAIs\n",
            "4\n",
            "HAIs\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3066 line is : phages\n",
            "6\n",
            "phages\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 3070 line is : multidrug-resistant pathogens\n",
            "29\n",
            "multidrug-resistant pathogens\n",
            "29\n",
            "-----------------------------------------------\n",
            "The 3074 line is : antimicrobial stewardship\n",
            "25\n",
            "antimicrobial stewardship\n",
            "25\n",
            "-----------------------------------------------\n",
            "The 3075 line is : AMR surveillance\n",
            "16\n",
            "AMR surveillance\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3077 line is : S. canis\n",
            "8\n",
            "S. canis\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3080 line is : non-antibiotic therapies\n",
            "24\n",
            "non-antibiotic therapies\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 3081 line is : fluoroquinolone\n",
            "15\n",
            "fluoroquinolone\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3083 line is : outer membrane vesicles (OMV)\n",
            "29\n",
            "outer membrane vesicles (OMV)\n",
            "29\n",
            "-----------------------------------------------\n",
            "The 3084 line is : Gram-negative bacteria\n",
            "22\n",
            "Gram-negative bacteria\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 3085 line is : antigen repertoire\n",
            "18\n",
            "antigen repertoire\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 3086 line is : Antimicrobial stewardship (AMS)\n",
            "31\n",
            "Antimicrobial stewardship (AMS)\n",
            "31\n",
            "-----------------------------------------------\n",
            "The 3089 line is : KpI (K. pneumoniae)\n",
            "19\n",
            "KpI (K. pneumoniae)\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 3092 line is : antimicrobial susceptibility test (AST)\n",
            "39\n",
            "antimicrobial susceptibility test (AST)\n",
            "39\n",
            "-----------------------------------------------\n",
            "The 3093 line is : disk diffusion tests\n",
            "20\n",
            "disk diffusion tests\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 3094 line is : automated quantitative AST methods\n",
            "34\n",
            "automated quantitative AST methods\n",
            "34\n",
            "-----------------------------------------------\n",
            "The 3095 line is : advanced molecular methods\n",
            "26\n",
            "advanced molecular methods\n",
            "26\n",
            "-----------------------------------------------\n",
            "The 3098 line is : B. fragilis\n",
            "11\n",
            "B. fragilis\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3099 line is : whole-genome sequencing\n",
            "23\n",
            "whole-genome sequencing\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 3100 line is : macrolide-resistant enterococci\n",
            "31\n",
            "macrolide-resistant enterococci\n",
            "31\n",
            "-----------------------------------------------\n",
            "The 3101 line is : staphylococcal strains\n",
            "22\n",
            "staphylococcal strains\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 3102 line is : staphylococcal species\n",
            "22\n",
            "staphylococcal species\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 3103 line is : aadA\n",
            "4\n",
            "aadA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3104 line is : aph(3'')\n",
            "8\n",
            "aph(3'')\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3105 line is : aph(3'')-Ib\n",
            "11\n",
            "aph(3'')-Ib\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3106 line is : aph(6)-I\n",
            "8\n",
            "aph(6)-I\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3107 line is : aph(6)-Id\n",
            "9\n",
            "aph(6)-Id\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3108 line is : bla\n",
            "3\n",
            "bla\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3109 line is : blaCMY\n",
            "6\n",
            "blaCMY\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 3110 line is : tet\n",
            "3\n",
            "tet\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3111 line is : tet(A)\n",
            "6\n",
            "tet(A)\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 3112 line is : sul2\n",
            "4\n",
            "sul2\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3113 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3114 line is : multi-drug resistant E. coli\n",
            "28\n",
            "multi-drug resistant E. coli\n",
            "28\n",
            "-----------------------------------------------\n",
            "The 3116 line is : WWTP sewage treatment.\n",
            "22\n",
            "WWTP sewage treatment.\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 3118 line is : Campylobacter spp.\n",
            "18\n",
            "Campylobacter spp.\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 3119 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3120 line is : Gram-positive pathogen\n",
            "22\n",
            "Gram-positive pathogen\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 3122 line is : Vibrio spp.\n",
            "11\n",
            "Vibrio spp.\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3123 line is : pyrH gene\n",
            "9\n",
            "pyrH gene\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3124 line is : mef(E)\n",
            "6\n",
            "mef(E)\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 3125 line is : mel\n",
            "3\n",
            "mel\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3126 line is : spxB\n",
            "4\n",
            "spxB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3127 line is : autolysin (lytA) gene\n",
            "21\n",
            "autolysin (lytA) gene\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3128 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3129 line is : colistin\n",
            "8\n",
            "colistin\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3130 line is : carbapenem\n",
            "10\n",
            "carbapenem\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3131 line is : IMP\n",
            "3\n",
            "IMP\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3132 line is : OXA-48\n",
            "6\n",
            "OXA-48\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 3133 line is : NDM\n",
            "3\n",
            "NDM\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3134 line is : KPC\n",
            "3\n",
            "KPC\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3135 line is : mcr-1\n",
            "5\n",
            "mcr-1\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3136 line is : mcr-2\n",
            "5\n",
            "mcr-2\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3137 line is : mcr-3\n",
            "5\n",
            "mcr-3\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3138 line is : mcr-4\n",
            "5\n",
            "mcr-4\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3139 line is : mcr-5\n",
            "5\n",
            "mcr-5\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3140 line is : vanB \n",
            "5\n",
            "vanB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3141 line is : vancomycin\n",
            "10\n",
            "vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3142 line is : mecA \n",
            "5\n",
            "mecA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3143 line is : SCCmec IV and V\n",
            "15\n",
            "SCCmec IV and V\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3146 line is : enterococcal\n",
            "12\n",
            "enterococcal\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3148 line is : Salmonella spp.\n",
            "15\n",
            "Salmonella spp.\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3149 line is : Multiple resistance\n",
            "19\n",
            "Multiple resistance\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 3150 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3151 line is : Salmonella virchow\n",
            "18\n",
            "Salmonella virchow\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 3152 line is : Salmonella hadar\n",
            "16\n",
            "Salmonella hadar\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3156 line is :  Gram-negative organisms\n",
            "24\n",
            "Gram-negative organisms\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 3159 line is : MRSA\n",
            "4\n",
            "MRSA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3162 line is : enterococci \n",
            "12\n",
            "enterococci\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3163 line is : enrofloxacin \n",
            "13\n",
            "enrofloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3164 line is : cefotaxime \n",
            "11\n",
            "cefotaxime\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3165 line is : penicillin \n",
            "11\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3167 line is : Enterococci\n",
            "11\n",
            "Enterococci\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3171 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3172 line is : Erythromycin\n",
            "12\n",
            "Erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3173 line is : vanA\n",
            "4\n",
            "vanA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3174 line is : vanB \n",
            "5\n",
            "vanB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3175 line is : vanC1 \n",
            "6\n",
            "vanC1\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3176 line is : ermB \n",
            "5\n",
            "ermB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3177 line is : ermC \n",
            "5\n",
            "ermC\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3178 line is : Resistome \n",
            "10\n",
            "Resistome\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3179 line is : macrophages\n",
            "11\n",
            "macrophages\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3181 line is : Campylobacter \n",
            "14\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3182 line is : ciprofloxacin (CIP)\n",
            "19\n",
            "ciprofloxacin (CIP)\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 3183 line is : nalidixic acid (NAL)\n",
            "20\n",
            "nalidixic acid (NAL)\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 3184 line is : erythromycin (ERY)\n",
            "18\n",
            "erythromycin (ERY)\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 3185 line is : azithromycin (AZM)\n",
            "18\n",
            "azithromycin (AZM)\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 3186 line is : tetracycline (TE)\n",
            "17\n",
            "tetracycline (TE)\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 3187 line is : gentamicin (GM)\n",
            "15\n",
            "gentamicin (GM)\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3188 line is : ampicillin (AMP)\n",
            "16\n",
            "ampicillin (AMP)\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3189 line is : amoxicillin\n",
            "11\n",
            "amoxicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3190 line is : clavulanic acid (AMC)\n",
            "21\n",
            "clavulanic acid (AMC)\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3191 line is : ceftriaxone (CRO)\n",
            "17\n",
            "ceftriaxone (CRO)\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 3192 line is : chloramphenicol (C)\n",
            "19\n",
            "chloramphenicol (C)\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 3193 line is : trimethoprim-sulfamethoxazole (TMS)\n",
            "35\n",
            "trimethoprim-sulfamethoxazole (TMS)\n",
            "35\n",
            "-----------------------------------------------\n",
            "The 3195 line is : tetracycline\n",
            "12\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3196 line is : benzylpenicillin\n",
            "16\n",
            "benzylpenicillin\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3197 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3198 line is : cefixime MIC\n",
            "12\n",
            "cefixime MIC\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3199 line is : tetracycline\n",
            "12\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3200 line is : benzylpenicillin \n",
            "17\n",
            "benzylpenicillin\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3201 line is : ciprofloxacin \n",
            "14\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3202 line is : cefixime \n",
            "9\n",
            "cefixime\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3203 line is : NG-MAST\n",
            "7\n",
            "NG-MAST\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3212 line is : PATRIC database\n",
            "15\n",
            "PATRIC database\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3214 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3215 line is : Staphylococcus epidermidis\n",
            "26\n",
            "Staphylococcus epidermidis\n",
            "26\n",
            "-----------------------------------------------\n",
            "The 3216 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3217 line is : Transcriptomic analysis\n",
            "23\n",
            "Transcriptomic analysis\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 3218 line is : Staphylococcus\n",
            "14\n",
            "Staphylococcus\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3219 line is : streptococcus\n",
            "13\n",
            "streptococcus\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3220 line is : mecA\n",
            "4\n",
            "mecA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3221 line is : enrofloxacin\n",
            "12\n",
            "enrofloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3222 line is : gentamicin\n",
            "10\n",
            "gentamicin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3223 line is : ARG-ANNOT\n",
            "9\n",
            "ARG-ANNOT\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3224 line is : CARD\n",
            "4\n",
            "CARD\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3225 line is : SRST2\n",
            "5\n",
            "SRST2\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3226 line is : MEGARes\n",
            "7\n",
            "MEGARes\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3227 line is : Genefinder\n",
            "10\n",
            "Genefinder\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3228 line is : ARIBA\n",
            "5\n",
            "ARIBA\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3229 line is : KmerResistance\n",
            "14\n",
            "KmerResistance\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3230 line is : AMRFinder\n",
            "9\n",
            "AMRFinder\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3231 line is : ResFinder\n",
            "9\n",
            "ResFinder\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3232 line is : monocytogenes \n",
            "14\n",
            "monocytogenes\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3233 line is : anti-antimicrobial resistance approach\n",
            "38\n",
            "anti-antimicrobial resistance approach\n",
            "38\n",
            "-----------------------------------------------\n",
            "The 3234 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3235 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3236 line is : methicillin\n",
            "11\n",
            "methicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3237 line is : methicillin\n",
            "11\n",
            "methicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3238 line is : gyrA\n",
            "4\n",
            "gyrA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3239 line is : gyrB\n",
            "4\n",
            "gyrB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3240 line is : parC\n",
            "4\n",
            "parC\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3241 line is : parE\n",
            "4\n",
            "parE\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3242 line is : fluoroquinolone\n",
            "15\n",
            "fluoroquinolone\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3243 line is : staphylococcal\n",
            "14\n",
            "staphylococcal\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3244 line is : beta-lactamases\n",
            "15\n",
            "beta-lactamases\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3245 line is : acetylases\n",
            "10\n",
            "acetylases\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3246 line is : adenylases\n",
            "10\n",
            "adenylases\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3247 line is : phosphorylases\n",
            "14\n",
            "phosphorylases\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3249 line is : A. baumannii\n",
            "12\n",
            "A. baumannii\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3250 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3251 line is : K. pneumoniae\n",
            "13\n",
            "K. pneumoniae\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3252 line is : methicillin\n",
            "11\n",
            "methicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3253 line is : S. aureus\n",
            "9\n",
            "S. aureus\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3254 line is : P. aeruginosa\n",
            "13\n",
            "P. aeruginosa\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3255 line is : cecal ligation\n",
            "14\n",
            "cecal ligation\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3256 line is : A. baumannii\n",
            "12\n",
            "A. baumannii\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3257 line is :  trimethoprim\n",
            "13\n",
            "trimethoprim\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3258 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3259 line is : AMR\n",
            "3\n",
            "AMR\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3260 line is : pneumococcal conjugate vaccines (PCVs)\n",
            "38\n",
            "pneumococcal conjugate vaccines (PCVs)\n",
            "38\n",
            "-----------------------------------------------\n",
            "The 3261 line is : β-lactams\n",
            "9\n",
            "β-lactams\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3262 line is :  tetracyclines\n",
            "14\n",
            "tetracyclines\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3263 line is :  macrolide lincosamide \n",
            "23\n",
            "macrolide lincosamide\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 3264 line is : streptogramin \n",
            "14\n",
            "streptogramin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3265 line is : quinolones\n",
            "10\n",
            "quinolones\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3266 line is : sulfonamides\n",
            "12\n",
            "sulfonamides\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3267 line is : aminoglycosides\n",
            "15\n",
            "aminoglycosides\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3268 line is : efflux pump.\n",
            "12\n",
            "efflux pump.\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3269 line is : quinupristin/dalfopristin (Synercid)\n",
            "36\n",
            "quinupristin/dalfopristin (Synercid)\n",
            "36\n",
            "-----------------------------------------------\n",
            "The 3270 line is : vernimicin\n",
            "10\n",
            "vernimicin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3271 line is : SCH 27899 (Ziracin)\n",
            "19\n",
            "SCH 27899 (Ziracin)\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 3272 line is : linezolid (Zyvax)\n",
            "17\n",
            "linezolid (Zyvax)\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 3273 line is : doxycycline\n",
            "11\n",
            "doxycycline\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3274 line is : lactic acid (LA)\n",
            "16\n",
            "lactic acid (LA)\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3275 line is : ampicillin\n",
            "10\n",
            "ampicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3276 line is : streptomycin\n",
            "12\n",
            "streptomycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3277 line is : macrolides\n",
            "10\n",
            "macrolides\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3278 line is : tetracyclines\n",
            "13\n",
            "tetracyclines\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3279 line is : sulfonamides\n",
            "12\n",
            "sulfonamides\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3280 line is : carbapenem\n",
            "10\n",
            "carbapenem\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3281 line is : carbapenem\n",
            "10\n",
            "carbapenem\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3282 line is : Escherichia coli\n",
            "16\n",
            "Escherichia coli\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3283 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3284 line is : fluoroquinolone\n",
            "15\n",
            "fluoroquinolone\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3285 line is : mobile genetic elements \n",
            "24\n",
            "mobile genetic elements\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 3286 line is : metal resistance genes (BMRGs) \n",
            "31\n",
            "metal resistance genes (BMRGs)\n",
            "30\n",
            "-----------------------------------------------\n",
            "The 3287 line is : Campylobacter\n",
            "13\n",
            "Campylobacter\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3288 line is : salmonellas\n",
            "11\n",
            "salmonellas\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3289 line is : clindamycin\n",
            "11\n",
            "clindamycin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3290 line is : gentamycin\n",
            "10\n",
            "gentamycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3291 line is : oxacillin\n",
            "9\n",
            "oxacillin\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3293 line is : Salmonella enterica\n",
            "19\n",
            "Salmonella enterica\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 3294 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3295 line is : Shigella\n",
            "8\n",
            "Shigella\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3296 line is : fosA\n",
            "4\n",
            "fosA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3297 line is : oqxB\n",
            "4\n",
            "oqxB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3298 line is : ble\n",
            "3\n",
            "ble\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3299 line is : floR\n",
            "4\n",
            "floR\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3300 line is : fosA7\n",
            "5\n",
            "fosA7\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3301 line is : mcr-9.1\n",
            "7\n",
            "mcr-9.1\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3302 line is : aadA1\n",
            "5\n",
            "aadA1\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3303 line is : aadA2\n",
            "5\n",
            "aadA2\n",
            "5\n",
            "-----------------------------------------------\n",
            "The 3304 line is : ant(2\")-Ia\n",
            "10\n",
            "ant(2\")-Ia\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3305 line is : aph(3\")-Ib\n",
            "10\n",
            "aph(3\")-Ib\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3306 line is : aph(3')-Ia\n",
            "10\n",
            "aph(3')-Ia\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3307 line is : aph(6)-Id\n",
            "9\n",
            "aph(6)-Id\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3308 line is : blaTEM-1\n",
            "8\n",
            "blaTEM-1\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3309 line is : qacEdelta1\n",
            "10\n",
            "qacEdelta1\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3310 line is : sul1\n",
            "4\n",
            "sul1\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3311 line is : sul2\n",
            "4\n",
            "sul2\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3312 line is : tet(A)\n",
            "6\n",
            "tet(A)\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 3313 line is : tet(B)\n",
            "6\n",
            "tet(B)\n",
            "6\n",
            "-----------------------------------------------\n",
            "The 3314 line is : carbapenem\n",
            "10\n",
            "carbapenem\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3315 line is : beta-lactamase (ESBL and BL) genes\n",
            "34\n",
            "beta-lactamase (ESBL and BL) genes\n",
            "34\n",
            "-----------------------------------------------\n",
            "The 3316 line is : horizontal gene transfer (HGT)\n",
            "30\n",
            "horizontal gene transfer (HGT)\n",
            "30\n",
            "-----------------------------------------------\n",
            "The 3317 line is :  G. anatis \n",
            "11\n",
            "G. anatis\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3318 line is : antimicrobial therapy.\n",
            "22\n",
            "antimicrobial therapy.\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 3320 line is : ARG\n",
            "3\n",
            "ARG\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3321 line is : ARB\n",
            "3\n",
            "ARB\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3322 line is : ARG\n",
            "3\n",
            "ARG\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3323 line is : ARB\n",
            "3\n",
            "ARB\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3324 line is : ARG\n",
            "3\n",
            "ARG\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3325 line is : ARB\n",
            "3\n",
            "ARB\n",
            "3\n",
            "-----------------------------------------------\n",
            "The 3326 line is : metagenomic sequencing \n",
            "23\n",
            "metagenomic sequencing\n",
            "22\n",
            "-----------------------------------------------\n",
            "The 3327 line is : fluorescence-activated cell sorting\n",
            "35\n",
            "fluorescence-activated cell sorting\n",
            "35\n",
            "-----------------------------------------------\n",
            "The 3328 line is : antimicrobial therapy\n",
            "21\n",
            "antimicrobial therapy\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3329 line is : Bacteroides fragilis\n",
            "20\n",
            "Bacteroides fragilis\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 3330 line is :  gyrA\n",
            "5\n",
            "gyrA\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3331 line is : quinolone\n",
            "9\n",
            "quinolone\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3332 line is : active efflux mechanism\n",
            "23\n",
            "active efflux mechanism\n",
            "23\n",
            "-----------------------------------------------\n",
            "The 3333 line is : tosufloxacin\n",
            "12\n",
            "tosufloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3334 line is : amoxicillin\n",
            "11\n",
            "amoxicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3335 line is : ampicillin\n",
            "10\n",
            "ampicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3336 line is : tetracycline\n",
            "12\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3337 line is : doxycycline\n",
            "11\n",
            "doxycycline\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3338 line is : apramycin\n",
            "9\n",
            "apramycin\n",
            "9\n",
            "-----------------------------------------------\n",
            "The 3339 line is : neomycin\n",
            "8\n",
            "neomycin\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3340 line is : enrofloxacin\n",
            "12\n",
            "enrofloxacin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3341 line is : flumequine\n",
            "10\n",
            "flumequine\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3342 line is : co-trimoxazole\n",
            "14\n",
            "co-trimoxazole\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3343 line is : florfenicol\n",
            "11\n",
            "florfenicol\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3344 line is : lincospectin\n",
            "12\n",
            "lincospectin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3345 line is : amoxicillin\n",
            "11\n",
            "amoxicillin\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3346 line is : ampicillin\n",
            "10\n",
            "ampicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3347 line is : neomycin\n",
            "8\n",
            "neomycin\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3348 line is : flumequine\n",
            "10\n",
            "flumequine\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3349 line is : co-trimoxazole\n",
            "14\n",
            "co-trimoxazole\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3350 line is : florfenicol\n",
            "11\n",
            "florfenicol\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3351 line is : lincospectin\n",
            "12\n",
            "lincospectin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3352 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3353 line is : E. coli\n",
            "7\n",
            "E. coli\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3354 line is : pneumococcal conjugate vaccines\n",
            "31\n",
            "pneumococcal conjugate vaccines\n",
            "31\n",
            "-----------------------------------------------\n",
            "The 3355 line is : Vibrios\n",
            "7\n",
            "Vibrios\n",
            "7\n",
            "-----------------------------------------------\n",
            "The 3356 line is : plasmids\n",
            "8\n",
            "plasmids\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3357 line is : integrating conjugative elements\n",
            "32\n",
            "integrating conjugative elements\n",
            "32\n",
            "-----------------------------------------------\n",
            "The 3358 line is : superintegron\n",
            "13\n",
            "superintegron\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3359 line is : transposable elements\n",
            "21\n",
            "transposable elements\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3360 line is : insertion sequences\n",
            "19\n",
            "insertion sequences\n",
            "19\n",
            "-----------------------------------------------\n",
            "The 3361 line is : β-lactamase\n",
            "11\n",
            "β-lactamase\n",
            "11\n",
            "-----------------------------------------------\n",
            "The 3362 line is :  nalidixic acid\n",
            "15\n",
            "nalidixic acid\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3363 line is : bacteriophage therapy\n",
            "21\n",
            "bacteriophage therapy\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3364 line is : combination therapies\n",
            "21\n",
            "combination therapies\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3365 line is :  antimicrobial adjuvants therapy\n",
            "32\n",
            "antimicrobial adjuvants therapy\n",
            "31\n",
            "-----------------------------------------------\n",
            "The 3366 line is : nanotechnology\n",
            "14\n",
            "nanotechnology\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3367 line is : K. quasipneumoniae\n",
            "18\n",
            "K. quasipneumoniae\n",
            "18\n",
            "-----------------------------------------------\n",
            "The 3368 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3369 line is : penicillin\n",
            "10\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3370 line is : Staphylococcus aureus\n",
            "21\n",
            "Staphylococcus aureus\n",
            "21\n",
            "-----------------------------------------------\n",
            "The 3371 line is : Staphylococcus\n",
            "14\n",
            "Staphylococcus\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3372 line is : vancomycin\n",
            "10\n",
            "vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3373 line is : neomycin\n",
            "8\n",
            "neomycin\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3374 line is : chloramphenicol\n",
            "15\n",
            "chloramphenicol\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3375 line is : Sequencing\n",
            "10\n",
            "Sequencing\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3376 line is :  penicillin\n",
            "11\n",
            "penicillin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3377 line is :  amoxicillin \n",
            "13\n",
            "amoxicillin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3378 line is :  antimicrobial chemotherapy.\n",
            "28\n",
            "antimicrobial chemotherapy.\n",
            "27\n",
            "-----------------------------------------------\n",
            "The 3379 line is : microbiota dysbiosis\n",
            "20\n",
            "microbiota dysbiosis\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 3380 line is : horizontal gene transfer\n",
            "24\n",
            "horizontal gene transfer\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 3381 line is : integrative conjugative elements (ICE)\n",
            "38\n",
            "integrative conjugative elements (ICE)\n",
            "38\n",
            "-----------------------------------------------\n",
            "The 3382 line is : taphylococcus aureus, Neisseria gonorrhoeae, and Enterobacteriaceae.\n",
            "68\n",
            "taphylococcus aureus, Neisseria gonorrhoeae, and Enterobacteriaceae.\n",
            "68\n",
            "-----------------------------------------------\n",
            "The 3383 line is : Enterococcus spp\n",
            "16\n",
            "Enterococcus spp\n",
            "16\n",
            "-----------------------------------------------\n",
            "The 3384 line is : tetracycline\n",
            "12\n",
            "tetracycline\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3385 line is : erythromycin\n",
            "12\n",
            "erythromycin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3386 line is : ciprofloxacin\n",
            "13\n",
            "ciprofloxacin\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3387 line is : chloramphenicol\n",
            "15\n",
            "chloramphenicol\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3388 line is : quinupristin\n",
            "12\n",
            "quinupristin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3389 line is : dalfopristin\n",
            "12\n",
            "dalfopristin\n",
            "12\n",
            "-----------------------------------------------\n",
            "The 3390 line is : nitrofurantoin\n",
            "14\n",
            "nitrofurantoin\n",
            "14\n",
            "-----------------------------------------------\n",
            "The 3391 line is : aminoglycosides\n",
            "15\n",
            "aminoglycosides\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3393 line is : polymerase chain reaction (PCR)\n",
            "31\n",
            "polymerase chain reaction (PCR)\n",
            "31\n",
            "-----------------------------------------------\n",
            "The 3394 line is : isothermal amplification\n",
            "24\n",
            "isothermal amplification\n",
            "24\n",
            "-----------------------------------------------\n",
            "The 3395 line is : high-throughput sequencing\n",
            "26\n",
            "high-throughput sequencing\n",
            "26\n",
            "-----------------------------------------------\n",
            "The 3396 line is : gene microarray\n",
            "15\n",
            "gene microarray\n",
            "15\n",
            "-----------------------------------------------\n",
            "The 3397 line is : mass spectrometry\n",
            "17\n",
            "mass spectrometry\n",
            "17\n",
            "-----------------------------------------------\n",
            "The 3398 line is :  Van resistance genes\n",
            "21\n",
            "Van resistance genes\n",
            "20\n",
            "-----------------------------------------------\n",
            "The 3399 line is : vancomycin\n",
            "10\n",
            "vancomycin\n",
            "10\n",
            "-----------------------------------------------\n",
            "The 3400 line is : analytical ultracentrifugation\n",
            "30\n",
            "analytical ultracentrifugation\n",
            "30\n",
            "-----------------------------------------------\n",
            "The 3401 line is : circular dichroism spectroscopy\n",
            "31\n",
            "circular dichroism spectroscopy\n",
            "31\n",
            "-----------------------------------------------\n",
            "The 3402 line is : VanS receptor\n",
            "13\n",
            "VanS receptor\n",
            "13\n",
            "-----------------------------------------------\n",
            "The 3406 line is : extended-spectrum β-lactamase-producing Enterobacteriaceae\n",
            "58\n",
            "extended-spectrum β-lactamase-producing Enterobacteriaceae\n",
            "58\n",
            "-----------------------------------------------\n",
            "The 3407 line is : carbapenemase-producing Enterobacteriaceae\n",
            "42\n",
            "carbapenemase-producing Enterobacteriaceae\n",
            "42\n",
            "-----------------------------------------------\n",
            "The 3416 line is : mgrB \n",
            "5\n",
            "mgrB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3417 line is : mgrB \n",
            "5\n",
            "mgrB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3418 line is : mgrB \n",
            "5\n",
            "mgrB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3419 line is : mgrB\n",
            "4\n",
            "mgrB\n",
            "4\n",
            "-----------------------------------------------\n",
            "The 3421 line is : S. typhi\n",
            "8\n",
            "S. typhi\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3422 line is : S. typhi\n",
            "8\n",
            "S. typhi\n",
            "8\n",
            "-----------------------------------------------\n",
            "The 3423 line is : S. typhi\n",
            "8\n",
            "S. typhi\n",
            "8\n",
            "-----------------------------------------------\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "for i in range(len(text)):\n",
        "  print(\"{}: [({}, {}, {}, {})]\".format(i,start[i],end[i],label[i],text[i]))"
      ],
      "metadata": {
        "id": "flBQZLi45A5B",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "7f880c78-cd2c-4154-a05c-5c0b94ffd378"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "0: [(159, 182, Techniques, Current limitations in the understanding and control of antimicrobial resistance (AMR) in Canada are described through a comprehensive review focusing on: (1) treatment optimization; (2) surveillance of antimicrobial use and AMR; and (3) prevention of transmission of AMR.)]\n",
            "1: [(, , , Without addressing gaps in identified areas, sustained progress in AMR mitigation is unlikely. Expert opinions and perspectives contributed to prioritizing identified gaps.)]\n",
            "2: [(, , , Thirty-eight clones were detected.)]\n",
            "3: [(42, 51, Others, Several examples of the implementation of DART 2020 are outlined here.)]\n",
            "4: [(, , , Specifically, antimicrobial use in human, animal, crop, and environmental sectors cannot be regarded as independent; therefore, a One Health approach is needed in AMR research and understanding, current surveillance efforts, and policy.)]\n",
            "5: [(, , , Antimicrobial resistance (AMR) is tracked most closely in clinical settings and high-income countries.)]\n",
            "6: [(, , , Discussions regarding addressing described knowledge gaps are separated into four categories: (1) further research; (2) increased capacity/resources; (3) increased prescriber/end-user knowledge; and (4) policy development/enforcement.)]\n",
            "7: [(, , , In urban and food production settings, top-down and infrastructure-dependent interventions against AMR that require strong regulatory oversight are less likely to curtail transmission when used alone and should be combined with bottom-up AMR-containment approaches.)]\n",
            "8: [(, , , We observe that the power of genomics to expose transmission channels and hotspots is largely unharnessed, and that existing and upcoming technological innovations need to be exploited towards containing AMR in low- and middle-income settings.)]\n",
            "9: [(, , , Antimicrobial Resistance (AMR) causes more than a million deaths globally per year due to infections incurable with currently available antibiotics.)]\n",
            "10: [(, , , Failing to effectively address AMR will have significant negative consequences for Canadians and the Canadian economy.)]\n",
            "11: [(, , , There is thus considerable global interest in promoting a One Health view of AMR to enable a more realistic understanding of its ecology.)]\n",
            "12: [(, , , Canada is behind on allocation of required funding and nationally coordinated AMR mitigation strategies relative to other high-income countries.)]\n",
            "13: [(, , , A Pan-Canadian AMR action plan and development of a new governance model is pending. Recent AMR-specific funding commitments are significant but fall short while distribution of funds indicate a siloed approach.)]\n",
            "14: [(, , , Canada could initiate progress towards AMR mitigation through incorporation within the scope of budget allocations intended for COVID-19 recovery and mitigation efforts.)]\n",
            "15: [(, , , Using Canada as an example, this review emphasizes the importance and necessity of a One Health approach for understanding and mitigating AMR.)]\n",
            "16: [(, , , However, resistant organisms thrive globally and are transmitted to and from healthy humans, animals and the environment, particularly in many low- and middle-income settings.)]\n",
            "17: [(, , , This review highlights the research and increased capacity and resources to generate new knowledge and implement recommendations needed to address all identified gaps, including economic, social, and environmental considerations.)]\n",
            "18: [(, , , The overall public health and clinical significance of these transmission opportunities remain to be completely clarified.)]\n",
            "19: [(, , , More prescriber/end-user knowledge and policy development/enforcement are needed, but must be informed by realistic recommendations, with input from all relevant stakeholders.)]\n",
            "20: [(, , , For most knowledge gaps, important next steps are uncertain. In conclusion, identified knowledge gaps underlined the need for AMR policy decisions to be considered in a One Health framework, while highlighting critical needs to achieve realistic and meaningful progress.)]\n",
            "21: [(, , , In reality, AMR surveillance outside hospitals remains insufficient and it has been very challenging to convincingly document transmission at the interfaces between clinical specimens and other niches.)]\n",
            "22: [(, , , In this Review, we describe AMR and its transmission in low- and middle-income-country settings, emphasizing high-risk transmission points such as urban settings and food-animal handling.)]\n",
            "23: [(, , , The One Health approach may help to ensure food safety by anticipating, preventing, detecting, and controlling diseases that spread between animals, humans, and the environment.)]\n",
            "24: [(29, 48, Techniques, We report the use of a novel pain-relief therapy for FMD.)]\n",
            "25: [(, , , The overall distribution of non-vaccine, PCV7 (4, 6B, 9V, 14, 18C, 19F and 23F) and additional PCV13 serotypes (1, 3, 5, 6A, 19A and 7F) did not change significantly before and after PCV introduction in the Central region (Fisher's test P value 0.27) despite an increase in the proportion of non-vaccine serotypes to 40 % (n=6) in the post-PCV introduction period compared to 21.9 % (n=14).)]\n",
            "26: [(, , , In this context, the development of innovative, versatile, and effective alternatives to control bacterial infections in order to assure comprehensive food microbial safety is becoming an urgent issue.)]\n",
            "27: [(, , , Bacteriophages (phages), viruses of bacteria, have gained significance in the last years due to the request for new effective antimicrobials for the treatment of bacterial diseases, along with many other applications, including biotechnology and food safety.)]\n",
            "28: [(108, 114, Microbes, Regarding the necessary decrease in the use of antibiotics, results taken from the literature indicate that phages are also promising tools to help to address this issue.)]\n",
            "29: [(0, 14, Microbes, Bacteriophages (phages)-viruses that infect and kill bacteria-represent a promising tool for combatting the global threat of antibiotic-resistant pathogens.)]\n",
            "30: [(72, 79, Antibiotic-Resistant Bacteria, This review revealed that 18 out of 64 districts in Bangladesh reported E. coli in poultry, having a higher prevalence (combined prevalence: 69.3%, 95% confidence interval, CI: 67.3-71%).)]\n",
            "31: [(, , , The altered composition of gut microbiota during sepsis is profoundly associated with a loss of commensal bacteria and an overgrowth of potentially pathogenic bacteria, especially AMR bacteria.)]\n",
            "32: [(23, 30, Antibiotic-Resistant Bacteria, This review found that E. coli isolates showed resistance to 14 antimicrobial classes and 45 different antimicrobial agents, including the last-line (reserve group) antibiotics and banned antimicrobial categories for the treatment of infections in agricultural animals.)]\n",
            "33: [(57, 82, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "34: [(156, 171, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "35: [(183, 196, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "36: [(211, 222, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "37: [(237, 248, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "38: [(263, 273, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "39: [(286, 296, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "40: [(309, 321, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "41: [(336, 347, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "42: [(363, 374, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "43: [(390, 401, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "44: [(414, 428, Antibiotics, Phenotypic resistance of E. coli against penicillins and beta-lactamase inhibitors (20.2%-100%), cephalosporins (1.9%-100%), fluoroquinolones (5.98%-100%), aminoglycosides (6%-100%), tetracyclines (17.7%-100%), carbapenems (13.6%-72.7%), macrolides (11.8%-100%), polymyxins (7.9%-100%), phenicols (20%-97.2%), sulfa drugs (44.7%-100%), cephamycins (21.4%-48.8%), nitrofurans (21.4%-63.2%), monobactams (1.2%), and glycylcyclines (2.3%) was recorded in the last decades in Bangladesh.)]\n",
            "45: [(55, 67, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "46: [(160, 168, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "47: [(69, 76, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "48: [(78, 89, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "49: [(104, 115, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "50: [(91, 102, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "51: [(117, 126, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "52: [(128, 138, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "53: [(140, 147, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "54: [(153, 157, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "55: [(170, 174, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "56: [(179, 183, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "57: [(186, 202, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "58: [(220, 233, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "59: [(258, 270, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "60: [(289, 301, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "61: [(311, 326, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "62: [(335, 347, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "63: [(357, 367, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "64: [(380, 392, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "65: [(405, 420, Antibiotics, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "66: [(455, 462, Antibiotic-Resistant Bacteria, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "67: [(213, 217, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "68: [(204, 208, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "69: [(235, 239, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "70: [(251, 255, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "71: [(241, 245, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "72: [(272, 276, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "73: [(281, 286, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "74: [(328, 332, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "75: [(303, 308, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "76: [(349, 354, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "77: [(369, 377, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "78: [(394, 398, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "79: [(432, 436, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "80: [(422, 427, Antibiotic-Resistant Genes, Twenty-four different AMR genes encoding resistance to beta-lactams (bla TEM, bla CTX-M-1, bla CTX-M-2, bla CTX-M-9, bla OXA-1, bla OXA-47, bla SHV, and CITM), colistin (mcr1 and mcr3), fluoroquinolones (qnrB and qnrS), tetracyclines (tetA, tetB, and tetC), sulfonamides (sulI and sulII), trimethoprim (dfrA1), aminoglycosides (rmtB), streptomycin (aadA1), gentamicin (aac-3-IV), erythromycin (ereA), and chloramphenicol (catA1 and cmlA) were detected in E. coli isolates.)]\n",
            "81: [(129, 136, Microbes, We suggest a regular antimicrobial surveillance program with a strong One Health approach to lessen the hazardous effects of AMR E. coli in poultry industries in Bangladesh.)]\n",
            "82: [(21, 55, Others, We suggest a regular antimicrobial surveillance program with a strong One Health approach to lessen the hazardous effects of AMR E. coli in poultry industries in Bangladesh.)]\n",
            "83: [(, , , Antimicrobial resistance (AMR) is a significant public health issue in Bangladesh like many other developing countries where data on resistance trends are scarce.)]\n",
            "84: [(53, 69, Antibiotic-Resistant Bacteria, Moreover, the existence of multidrug-resistant (MDR) Escherichia coli exerts an ominous effect on the poultry sector.)]\n",
            "85: [(, , , To assist future phage-based real applications, the pending issues and main challenges to be addressed shortly by future studies are also taken into account.)]\n",
            "86: [(, , , The enrichment and diversity of gut microbiota play an important role in sepsis, but the role of gut microbiota composition and early-life colonization in sepsis and septic shock has not yet been characterized.)]\n",
            "87: [(14, 20, Techniques, Following the PRISMA guidelines, a total of 17 published scientific articles were selected for this systematic review.)]\n",
            "88: [(, , , The impact of gut microbiota diversity on host immunological disorders and future treatments of inflammatory diseases are not yet fully elucidated.)]\n",
            "89: [(, , , Further, the association between the microbiota and immune development in sepsis remains unknown, and the underlying mechanisms are not well understood.)]\n",
            "90: [(, , , Moreover, the prevalence ranged from 24.3% to 100%.)]\n",
            "91: [(, , , Disruptions of gut microbiota diversity are directly associated with susceptibility to sepsis and a higher risk of adverse outcomes.)]\n",
            "92: [(27, 38, Antibiotic-Resistant Bacteria, Also, 14 articles reported MDR E. coli in poultry, including a 100% MDR in nine articles and a 92.7% (95% CI: 91.2-94%) combined percentage of MDR E. coli isolates.)]\n",
            "93: [(143, 154, Antibiotic-Resistant Bacteria, Also, 14 articles reported MDR E. coli in poultry, including a 100% MDR in nine articles and a 92.7% (95% CI: 91.2-94%) combined percentage of MDR E. coli isolates.)]\n",
            "94: [(, , , The presence of MDR E. coli and their corresponding resistance genes in poultry and poultry environments is an alarming issue for all health communities in Bangladesh.)]\n",
            "95: [(, , , Indiscriminate antimicrobial use is the most important driver for AMR.)]\n",
            "96: [(, , , Antimicrobial resistance (AMR) is a serious global health problem associated with high mortality in the era of modern medicine.)]\n",
            "97: [(, , , The predisposing factors are inadequate veterinary healthcare, monitoring and regulatory services, intervention of excessive informal animal health service providers, and farmers' knowledge gap on drugs, and AMR which have resulted in the misuse and overuse of antibiotics, ultimate in the evolution of antibiotic-resistant bacteria and genes in all types of animal farming settings of Bangladesh.)]\n",
            "98: [(, , , Moreover, in the absence of an effective antibiotic, medical and surgical interventions can highly become a risk.)]\n",
            "99: [(, , , At present, preserving the power of existing antibiotics by prudent use and curtailing spread of pathogens by infection prevention and control (biosecurity) in both humans and animals are the best available options to defer AMR crisis.)]\n",
            "100: [(, , , This review provides an overview of various promising, potential and under investigative strategies as alternatives to antibiotics.)]\n",
            "101: [(, , , Currently, several novel approaches explored by scientists have shown promising results pertaining to their antimicrobial activity against pathogens.)]\n",
            "102: [(, , , In this article, the authors have summarized various novel therapeutic options explored to contain AMR with special attention to the mechanism of action, advantages, and disadvantages of different approaches.)]\n",
            "103: [(, , , Antimicrobial resistance (AMR), which is a major challenge for global healthcare, emerging because of several reasons including overpopulation, increased global migration and selection pressure due to enhanced use of antibiotics.)]\n",
            "104: [(, , , International travelers are at risk for acquiring resistant STIs with limited therapeutic options.)]\n",
            "105: [(, , , This has led to critical challenges when treating infections by multi-drug resistant (MDR) pathogens, which often display high mortality and morbidity rates.)]\n",
            "106: [(118, 134, Techniques, Besides the conventional approaches, some novel approaches such as the use of antimicrobial peptides, bacteriophages, immunomodulation, host-directed therapy and antibodies have shown really promising potentials.)]\n",
            "107: [(136, 157, Techniques, Besides the conventional approaches, some novel approaches such as the use of antimicrobial peptides, bacteriophages, immunomodulation, host-directed therapy and antibodies have shown really promising potentials.)]\n",
            "108: [(94, 105, Antibiotics, With the advent of the global antimicrobial resistance (AMR) crisis, our arsenal of effective antibiotics is diminishing.)]\n",
            "109: [(, , , Several studies have confirmed that a mutual association between gut microbiota and the host is important for the metabolism of essential nutrients for the organism, for gut development, and for the maturation and development of a fully functional immune system.)]\n",
            "110: [(198, 210, Techniques, Therefore, understanding the gut microbiota diversity, composition, and function during various inflammatory conditions and sepsis may provide a comprehensive knowledge of the mechanisms behind the pathogenesis of gut-derived infection in diseases and the design of new treatment options (e.g., probiotics or fecal microbiota transplantation).)]\n",
            "111: [(18, 38, Others, These alternative antibacterial agents could be a key element in our fight against AMR and provide desperately needed options, to veterinary and medical clinicians alike.)]\n",
            "112: [(117, 142, Others, Overuse of antibiotics in humans, agriculture, and animal husbandry has resulted in the emergence of a wide range of multidrug-resistant (MDR) pathogens which are difficult to treat.)]\n",
            "113: [(140, 154, Microbes, Meanwhile, to reduce dependence on antibiotics, other alternatives such as vaccines, antibodies, pattern recognition receptors, probiotics, bacteriophages, peptides, phytochemicals, metals, and antimicrobial enzymes are being explored.)]\n",
            "114: [(128, 138, Microbes, Meanwhile, to reduce dependence on antibiotics, other alternatives such as vaccines, antibodies, pattern recognition receptors, probiotics, bacteriophages, peptides, phytochemicals, metals, and antimicrobial enzymes are being explored.)]\n",
            "115: [(85, 95, Antibiotics, Meanwhile, to reduce dependence on antibiotics, other alternatives such as vaccines, antibodies, pattern recognition receptors, probiotics, bacteriophages, peptides, phytochemicals, metals, and antimicrobial enzymes are being explored.)]\n",
            "116: [(97, 126, Techniques, Meanwhile, to reduce dependence on antibiotics, other alternatives such as vaccines, antibodies, pattern recognition receptors, probiotics, bacteriophages, peptides, phytochemicals, metals, and antimicrobial enzymes are being explored.)]\n",
            "117: [(194, 215, Others, Meanwhile, to reduce dependence on antibiotics, other alternatives such as vaccines, antibodies, pattern recognition receptors, probiotics, bacteriophages, peptides, phytochemicals, metals, and antimicrobial enzymes are being explored.)]\n",
            "118: [(156, 189, Others, Meanwhile, to reduce dependence on antibiotics, other alternatives such as vaccines, antibodies, pattern recognition receptors, probiotics, bacteriophages, peptides, phytochemicals, metals, and antimicrobial enzymes are being explored.)]\n",
            "119: [(87, 100, Antibiotic-Resistant Bacteria, Hence it is high time for the development of new strategies to fight against dangerous MDR pathogens.)]\n",
            "120: [(, , , Additionally, persistent failure to develop and commercialize new antibiotics has created the scarcity of effective anti-infective drugs.)]\n",
            "121: [(, , , Thus, there is an urgent need for a new class of antimicrobials and other novel approaches to curb the menace of AMR.)]\n",
            "122: [(, , , An international action plan and multisectoral cooperative approach are therefore needed to combat and halt AMR dissemination.)]\n",
            "123: [(, , , Critical to these efforts are enhanced research programs aimed at identifying new antimicrobial agents, as well as the use of advanced biotechnology tools to develop alternative antimicrobial approaches.)]\n",
            "124: [(, , , The widespread use and misuse of antibiotics in human and veterinary medicine, compounded by the lack of novel classes of antibiotic in the pharmaceutical pipeline, has left a hole in our antibiotic armamentarium.)]\n",
            "125: [(, , , Thus, alternatives to traditional antibiotics are being investigated, including two major groups of antibacterial agents, which have been extensively studied, phytochemicals and metals.)]\n",
            "126: [(, , , Within these groups, there are several subclasses of compound/elements, including polyphenols and metal nanoparticles, which could be used to complement traditional antibiotics, either to increase their potency or extend their spectrum of activity.)]\n",
            "127: [(, , , Alone or in combination, these antibacterial agents have been shown to be effective against a vast array of human and animal bacterial pathogens, including those resistant to licensed antibacterials.)]\n",
            "128: [(, , , Emerging evidence displays an important role of gut microbiota and their derived metabolites in modulating the host mucosal immune response and determining the susceptibility to, as well as outcomes of sepsis.)]\n",
            "129: [(, , , Antibiotics once regarded as magic bullets are no more considered so.)]\n",
            "130: [(, , , Antimicrobial resistance (AMR) is a burgeoning challenge of global priority, warranting immediate action to prevent the explosion of multidrug-resistant (MDR) pathogens.)]\n",
            "131: [(, , , Apart from their use to treat bacterial infections in humans, antibiotics also play an important role in animal husbandry.)]\n",
            "132: [(, , , Ingestion of food or water contaminated with pathogenic bacteria may cause serious diseases.)]\n",
            "133: [(, , , With limited antibiotic options, alternate strategies are required to overcome AMR.)]\n",
            "134: [(, , , Antibiotics have always been considered as one of the most relevant discoveries of the twentieth century.)]\n",
            "135: [(119, 144, Others, This review highlights the magnitude of the AMR in the pediatric population and calls for the urgent implementation of antimicrobial stewardship programs to save the remaining antimicrobials.)]\n",
            "136: [(69, 85, Antibiotic-Resistant Bacteria, We have recently reported the acquisition of multidrug resistance in Escherichia coli after exposure to gut-relevant concentrations of the antipsychotic quetiapine.)]\n",
            "137: [(34, 39, Microbes, Thus, there is still a paucity of phage-related studies in this region, including those focused on therapeutic applications, clinical trials, biotechnology, and biocontrol.)]\n",
            "138: [(161, 171, Techniques, Thus, there is still a paucity of phage-related studies in this region, including those focused on therapeutic applications, clinical trials, biotechnology, and biocontrol.)]\n",
            "139: [(127, 148, Techniques, This article highlights the importance of bacteriophage research and development and discusses the potential implementation of phage-based therapies in Saudi Arabia.)]\n",
            "140: [(0, 22, Techniques, Bloodstream infections (BSI) due to multidrug-resistant organisms, especially from pediatric intensive care units (PICU), are being increasingly reported across the world.)]\n",
            "141: [(36, 65, Antibiotic-Resistant Bacteria, Bloodstream infections (BSI) due to multidrug-resistant organisms, especially from pediatric intensive care units (PICU), are being increasingly reported across the world.)]\n",
            "142: [(, , , Since BSI is associated with high mortality, it is essential to treat these infections early with appropriate antibiotics.)]\n",
            "143: [(74, 82, Microbes, In this review on etiology and its antibiogram in community acquired BSI, S. typhi followed by S. paratyphi A were the major bacterial isolates.)]\n",
            "144: [(95, 109, Microbes, In this review on etiology and its antibiogram in community acquired BSI, S. typhi followed by S. paratyphi A were the major bacterial isolates.)]\n",
            "145: [(0, 9, Antibiotics, Quinolone resistance of more than 90% in Salmonella is now reported from all over India. Ceftriaxone remains the drug of choice for enteric fever due to its 100% susceptibility.)]\n",
            "146: [(41, 51, Microbes, Quinolone resistance of more than 90% in Salmonella is now reported from all over India. Ceftriaxone remains the drug of choice for enteric fever due to its 100% susceptibility.)]\n",
            "147: [(89, 100, Antibiotics, Quinolone resistance of more than 90% in Salmonella is now reported from all over India. Ceftriaxone remains the drug of choice for enteric fever due to its 100% susceptibility.)]\n",
            "148: [(0, 20, Techniques, Passive immunization through oral, nasal and topical administration of egg yolk-derived IgY antibodies from immunized chickens were recently shown to be effective for treating bacterial infections in animals and humans.)]\n",
            "149: [(51, 71, Microbes, In PICU there is an emergence of candidemia due to non-albicans candida which are now predominant isolates at few centers.)]\n",
            "150: [(0, 12, Techniques, Immunization of chickens with specific antigens offers the possibility of creating specific antibodies targeting a wide range of antibiotic-resistant bacteria.)]\n",
            "151: [(45, 66, Microbes, BSI due to gram-negative bacteria, mostly by Klebseilla pneumoniae and gram-positive cocci (S. aureus) are the other major pathogens commonly observed in BSI from PICU.)]\n",
            "152: [(85, 102, Microbes, BSI due to gram-negative bacteria, mostly by Klebseilla pneumoniae and gram-positive cocci (S. aureus) are the other major pathogens commonly observed in BSI from PICU.)]\n",
            "153: [(144, 158, Others, In this review, we describe the growing global problem of antimicrobial resistance and highlight the promising potential of the use of egg yolk IgY antibodies for the treatment of bacterial infections, particularly those listed in the World Health Organization priority list.)]\n",
            "154: [(89, 99, Antibiotics, There is a high prevalence of antimicrobial resistance to commonly used antibiotics like ampicillin (94.9%-90.7%), cefotaxime (92.4%-71.4%), piperacillin-tazobactum (31.2%-27.5%) and levofloxacin (42.4%-39.8%).)]\n",
            "155: [(115, 125, Antibiotics, There is a high prevalence of antimicrobial resistance to commonly used antibiotics like ampicillin (94.9%-90.7%), cefotaxime (92.4%-71.4%), piperacillin-tazobactum (31.2%-27.5%) and levofloxacin (42.4%-39.8%).)]\n",
            "156: [(141, 164, Antibiotics, There is a high prevalence of antimicrobial resistance to commonly used antibiotics like ampicillin (94.9%-90.7%), cefotaxime (92.4%-71.4%), piperacillin-tazobactum (31.2%-27.5%) and levofloxacin (42.4%-39.8%).)]\n",
            "157: [(183, 195, Antibiotics, There is a high prevalence of antimicrobial resistance to commonly used antibiotics like ampicillin (94.9%-90.7%), cefotaxime (92.4%-71.4%), piperacillin-tazobactum (31.2%-27.5%) and levofloxacin (42.4%-39.8%).)]\n",
            "158: [(14, 25, Antibiotics, Resistance to carbapenems, primarily due to blaNDM is seen in all the centers and the rate varies between 1%- 79% with K. pneumoniae and A. baumannii showing the maximum resistance.)]\n",
            "159: [(44, 50, Antibiotic-Resistant Genes, Resistance to carbapenems, primarily due to blaNDM is seen in all the centers and the rate varies between 1%- 79% with K. pneumoniae and A. baumannii showing the maximum resistance.)]\n",
            "160: [(119, 132, Microbes, Resistance to carbapenems, primarily due to blaNDM is seen in all the centers and the rate varies between 1%- 79% with K. pneumoniae and A. baumannii showing the maximum resistance.)]\n",
            "161: [(137, 149, Microbes, Resistance to carbapenems, primarily due to blaNDM is seen in all the centers and the rate varies between 1%- 79% with K. pneumoniae and A. baumannii showing the maximum resistance.)]\n",
            "162: [(, , , In this context, the identification of new potential antimicrobial targets and/or the identification of new chemical entities as antimicrobial drugs are in great demand.)]\n",
            "163: [(, , , Psychotropic drugs have long been known to possess antimicrobial activity against several groups of microorganisms.)]\n",
            "164: [(, , , To date, among the many possible approaches used to deal with antibiotic resistance is the use of antibiotic adjuvants that hit bacterial non-essential targets.)]\n",
            "165: [(, , , In this review, the author focuses on the discovery of antibiotic adjuvants and on new tools to study and reduce the prevalence of resistant bacterial infections.)]\n",
            "166: [(, , , Rooted in the mechanism of action of antibiotics and subject to bacterial evolution, antibiotic resistance is difficult and perhaps impossible to overcome.)]\n",
            "167: [(, , , Phages and their potential applications have been extensively studied in Europe and the United States (US) for decades.)]\n",
            "168: [(, , , Nevertheless, strategies can be used to minimize the emergence and impact of resistance. Antibiotic adjuvants offer one such approach.)]\n",
            "169: [(, , , However, although health authorities in the Gulf Health Council (GHC), including Saudi Arabia, have developed an action plan to combat AMR, phage research in the Middle East has lagged behind global scientific efforts.)]\n",
            "170: [(, , , These are compounds that have little or no antibiotic activity themselves but act to block resistance or otherwise enhance antibiotic action.)]\n",
            "171: [(, , , Antibiotic adjuvants are therefore delivered in combination with antibiotics and can be divided into two groups: Class I agents that act on the pathogen, and Class II agents that act on the host.)]\n",
            "172: [(, , , Adjuvants offer a means to both suppress the emergence of resistance and rescue the activity of existing drugs, offering an orthogonal strategy complimentary to new antibiotic discovery VIDEO ABSTRACT.)]\n",
            "173: [(, , , Antibiotic resistant bacteria are a growing threat to global health security.)]\n",
            "174: [(, , , Whilst the emergence of antimicrobial resistance (AMR) is a natural phenomenon, it is also driven by antibiotic exposure in health care, agriculture, and the environment.)]\n",
            "175: [(, , , Antibiotic pressure and inappropriate use of antibiotics are important factors which drive resistance.)]\n",
            "176: [(, , , Surveillance of etiology and emerging antimicrobial resistance (AMR) is considered an important step in the formulation of antibiotic policy for early treatment and judicious use of antibiotics.)]\n",
            "177: [(, , , Antimicrobial resistance (AMR) has become a global clinical problem in recent years.)]\n",
            "178: [(, , , AMR has led to depletion of the antibiotic pipeline and developing new antibiotics is extremely challenging due to technical and financial issues and also resistance emerges as soon any new antibiotic is introduced.)]\n",
            "179: [(, , , Hence we propose it plays a key role in disseminating AMR genes from environmental microbes to clinically important pathogens.)]\n",
            "180: [(, , , With the discovery of antibiotics, infections were not a deadly problem for clinicians as they used to be.)]\n",
            "181: [(, , , Antibiotic use in severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) patients during the COVID-19 pandemic has exceeded the incidence of bacterial coinfections and secondary infections, suggesting inappropriate and excessive prescribing.)]\n",
            "182: [(, , , However, worldwide AMR comes with the overuse/misuse of antibiotics and the spread of resistance is deteriorated by a multitude of mobile genetic elements and relevant resistant genes.)]\n",
            "183: [(, , , Even in settings with established antimicrobial stewardship (AMS) programmes, there were weaknesses exposed regarding appropriate antibiotic use in the context of the pandemic.)]\n",
            "184: [(, , , Moreover, antimicrobial resistance (AMR) surveillance and AMS have been deprioritised with diversion of health system resources to the pandemic response.)]\n",
            "185: [(, , , This experience highlights deficiencies in AMR containment and mitigation strategies that require urgent attention from clinical and scientific communities.)]\n",
            "186: [(, , , The ubiquitous use of microplastics and their release into the environment especially the water bodies by anthropogenic/industrial activities are the major resources for microplastic contamination.)]\n",
            "187: [(4, 17, Microbes, The bacteriophage (or phage for short) has been used as an antibacterial agent for over a century but was abandoned in most countries after the discovery and broad use of antibiotics.)]\n",
            "188: [(175, 187, Techniques, Nowadays, phage-based technology has been developed for many purposes, including those beyond the framework of antibacterial treatment, such as to suppress viruses by phages, gene therapy, vaccine development, etc.)]\n",
            "189: [(31, 61, Techniques, New hope comes in the shape of antibiotic combination therapy, which although used by mother nature, is under-explored and could provide the solution to AMR.)]\n",
            "190: [(226, 229, Antibiotic-Resistant Genes, In addition to being a significant clinical problem in its own right, K. pneumoniae is the species within which several new AMR genes were first discovered before spreading to other pathogens (e.g. carbapenem-resistance genes KPC, OXA-48 and NDM-1).)]\n",
            "191: [(231, 237, Antibiotic-Resistant Genes, In addition to being a significant clinical problem in its own right, K. pneumoniae is the species within which several new AMR genes were first discovered before spreading to other pathogens (e.g. carbapenem-resistance genes KPC, OXA-48 and NDM-1).)]\n",
            "192: [(242, 247, Antibiotic-Resistant Genes, In addition to being a significant clinical problem in its own right, K. pneumoniae is the species within which several new AMR genes were first discovered before spreading to other pathogens (e.g. carbapenem-resistance genes KPC, OXA-48 and NDM-1).)]\n",
            "193: [(7, 21, Microbes, Whilst K. pneumoniae's contribution to the overall AMR crisis is impossible to quantify, current evidence suggests it has a wider ecological distribution, significantly more varied DNA composition, greater AMR gene diversity and a higher plasmid burden than other Gram negative opportunists.)]\n",
            "194: [(, , , Long-term global surveillance of clinical and societal antibiotic use and resistance trends is required to prepare for subsequent changes in AMR epidemiology, while ensuring uninterrupted supply chains and preventing drug shortages and stock outs.)]\n",
            "195: [(, , , These interventions present implementation challenges in resource-constrained settings, making a case for implementation research on AMR.)]\n",
            "196: [(, , , Knowledge and support for these practices will come from internationally coordinated, targeted research on AMR, supporting the preparation for future challenges from emerging AMR in the context of the current COVID-19 pandemic or future pandemics.)]\n",
            "197: [(, , , Considering these observations, this review attempts to establish if a relationship between psychotropics and AMR in microorganisms has been defined in the scientific literature.)]\n",
            "198: [(, , , Microplastics are emerging as the hubs as well as effective carriers of these microbial pathogens beside their AMR-genes (ARGs) in marine, freshwater, sewage/wastewater, and urban river ecosystems.)]\n",
            "199: [(, , , The rapid progress recently made in molecular biology and genetic engineering has accelerated the generation of phage-related products with superior therapeutic potentials against bacterial infection.)]\n",
            "200: [(, , , These drug resistant bacteria interact with microplastics forming synthetic plastispheres, the ideal niche for biofilm formations which in turn facilitates the transfer of ARGs via horizontal gene transfer and further escalates the occurrence and levels of AMR.)]\n",
            "201: [(, , , Microplastic-associated AMR is an emerging threat for human health and healthcare besides being a challenge for the research community for effective management/address of this menace.)]\n",
            "202: [(, , , Here, we highlighted the current progress in phage engineering technology and its application in modern medicine.)]\n",
            "203: [(, , , In this review, we encompass the increasing prevalence of microplastics in environment, emphasizing mainly on water environments, how they act as centers and vectors of microbial pathogens with their associated bacterial assemblage compositions and ultimately lead to AMR.)]\n",
            "204: [(142, 163, Techniques, The worldwide emergence and high prevalence of antimicrobial-resistant (AMR) bacteria have led to a revival of interest in the long-forgotten antibacterial therapy with phages (phage therapy) as an alternative approach to combatting AMR bacteria.)]\n",
            "205: [(177, 190, Techniques, The worldwide emergence and high prevalence of antimicrobial-resistant (AMR) bacteria have led to a revival of interest in the long-forgotten antibacterial therapy with phages (phage therapy) as an alternative approach to combatting AMR bacteria.)]\n",
            "206: [(176, 191, Techniques, The worldwide emergence and high prevalence of antimicrobial-resistant (AMR) bacteria have led to a revival of interest in the long-forgotten antibacterial therapy with phages (phage therapy) as an alternative approach to combatting AMR bacteria.)]\n",
            "207: [(, , , Klebsiella pneumoniae is an opportunistic bacterial pathogen known for its high frequency and diversity of antimicrobial resistance (AMR) genes.)]\n",
            "208: [(, , , It further discusses the mechanistic insights on how microplastics act as hosts of biofilms (creating the plastisphere).)]\n",
            "209: [(, , , We have also presented the modern toolbox used for microplastic-biofilm analyses. A review on potential strategies for addressing microplastic-associated AMR is given with recent success stories, challenges and future prospects.)]\n",
            "210: [(, , , However, with increased urbanization, industrialization, and heavy discharge of anthropogenic waste harboring antibiotics, heavy metals, pesticides, antibiotic resistance genes (ARGs), antibiotic resistant bacteria (ARB), urban rivers are becoming major reservoirs of ARGs and a hotspot for accelerated selection of ARB.)]\n",
            "211: [(, , , This will help combat AMR in the pathogenic Neisseria by: (i) determining the contemporary AMR profile of commensal Neisseria, (ii) correlating AMR phenotypes with known and novel genetic determinants, (iii) qualifying and quantifying horizontal gene transfer (HGT) for AMR determinants, and (iv) expanding commensal Neisseria genomic databases, perhaps leading to the identification of new drug and vaccine targets.)]\n",
            "212: [(, , , Most pathogens have developed an intrinsic capacity to thrive by developing resistance to antimicrobial compounds utilized in treatment.)]\n",
            "213: [(, , , Antimicrobial resistance arises when microbial agents such as bacteria, viruses, fungi, and parasites alter their behaviour to make current conventional medicines inefficient.)]\n",
            "214: [(, , , The proposed modification to established Neisseria collection protocols could transform our ability to address AMR N. gonorrhoeae, while requiring minor modifications to current surveillance practices.)]\n",
            "215: [(117, 127, Microbes, Substantial effort and financial support are being spent on identifying resistance mechanisms circulating within the gonococcal population.)]\n",
            "216: [(89, 110, Microbes, IMPORTANCE Contemporary increases in the prevalence of antimicrobial resistance (AMR) in Neisseria gonorrhoeae populations is a direct threat to global public health and the effective treatment of gonorrhea.)]\n",
            "217: [(71, 80, Antibiotic-Resistant Bacteria, However, these surveys often overlook a known source of resistance for gonococci-the commensal Neisseria.)]\n",
            "218: [(66, 78, Antibiotic-Resistant Bacteria, C. difficile ribotype (RT) 017 has emerged as the major strain of C. difficile in Asia, where antimicrobial use is poorly regulated.)]\n",
            "219: [(0, 30, Antibiotic-Resistant Bacteria, C. difficile ribotype (RT) 017 has emerged as the major strain of C. difficile in Asia, where antimicrobial use is poorly regulated.)]\n",
            "220: [(106, 119, Techniques, Here, we propose the expansion of established gonococcal surveillance programs to integrate a collection, AMR profiling, and genomic sequencing pipeline for commensal species.)]\n",
            "221: [(125, 143, Techniques, Here, we propose the expansion of established gonococcal surveillance programs to integrate a collection, AMR profiling, and genomic sequencing pipeline for commensal species.)]\n",
            "222: [(125, 136, Antibiotics, This strain has also caused CDI outbreaks around the world for almost 30 years. Many of these outbreaks were associated with clindamycin and fluoroquinolone resistance.)]\n",
            "223: [(157, 166, Microbes, This proposed expansion will enhance the field's ability to identify resistance in and from nonpathogenic reservoirs and anticipate AMR trends in pathogenic Neisseria.)]\n",
            "224: [(53, 65, Antibiotic-Resistant Bacteria, To address these topics, reports of outbreaks due to C. difficile RT 017, epidemiologic studies with antimicrobial susceptibility results, studies on resistance mechanisms found in C. difficile and related publications available through Pubmed until September 2019 were collated and the findings discussed.)]\n",
            "225: [(181, 193, Antibiotic-Resistant Bacteria, To address these topics, reports of outbreaks due to C. difficile RT 017, epidemiologic studies with antimicrobial susceptibility results, studies on resistance mechanisms found in C. difficile and related publications available through Pubmed until September 2019 were collated and the findings discussed.)]\n",
            "226: [(, , , Expert opinion: Primary prevention is the key to control CDI.)]\n",
            "227: [(52, 73, Microbes, Antimicrobial resistance (AMR) is widespread within Neisseria gonorrhoeae populations.)]\n",
            "228: [(155, 158, Techniques, Recent work has highlighted the importance of commensal Neisseria (cN) as a source of AMR for their pathogenic relatives through horizontal gene transfer (HGT) of AMR alleles, such as mosaic penicillin binding protein 2 (penA), multiple transferable efflux pump (mtr), and DNA gyrase subunit A (gyrA) which impact beta-lactam, azithromycin, and ciprofloxacin susceptibility, respectively.)]\n",
            "229: [(191, 201, Antibiotics, Recent work has highlighted the importance of commensal Neisseria (cN) as a source of AMR for their pathogenic relatives through horizontal gene transfer (HGT) of AMR alleles, such as mosaic penicillin binding protein 2 (penA), multiple transferable efflux pump (mtr), and DNA gyrase subunit A (gyrA) which impact beta-lactam, azithromycin, and ciprofloxacin susceptibility, respectively.)]\n",
            "230: [(221, 225, Gene/Proteins, Recent work has highlighted the importance of commensal Neisseria (cN) as a source of AMR for their pathogenic relatives through horizontal gene transfer (HGT) of AMR alleles, such as mosaic penicillin binding protein 2 (penA), multiple transferable efflux pump (mtr), and DNA gyrase subunit A (gyrA) which impact beta-lactam, azithromycin, and ciprofloxacin susceptibility, respectively.)]\n",
            "231: [(263, 266, Others, Recent work has highlighted the importance of commensal Neisseria (cN) as a source of AMR for their pathogenic relatives through horizontal gene transfer (HGT) of AMR alleles, such as mosaic penicillin binding protein 2 (penA), multiple transferable efflux pump (mtr), and DNA gyrase subunit A (gyrA) which impact beta-lactam, azithromycin, and ciprofloxacin susceptibility, respectively.)]\n",
            "232: [(295, 299, Others, Recent work has highlighted the importance of commensal Neisseria (cN) as a source of AMR for their pathogenic relatives through horizontal gene transfer (HGT) of AMR alleles, such as mosaic penicillin binding protein 2 (penA), multiple transferable efflux pump (mtr), and DNA gyrase subunit A (gyrA) which impact beta-lactam, azithromycin, and ciprofloxacin susceptibility, respectively.)]\n",
            "233: [(345, 358, Antibiotics, Recent work has highlighted the importance of commensal Neisseria (cN) as a source of AMR for their pathogenic relatives through horizontal gene transfer (HGT) of AMR alleles, such as mosaic penicillin binding protein 2 (penA), multiple transferable efflux pump (mtr), and DNA gyrase subunit A (gyrA) which impact beta-lactam, azithromycin, and ciprofloxacin susceptibility, respectively.)]\n",
            "234: [(327, 339, Antibiotics, Recent work has highlighted the importance of commensal Neisseria (cN) as a source of AMR for their pathogenic relatives through horizontal gene transfer (HGT) of AMR alleles, such as mosaic penicillin binding protein 2 (penA), multiple transferable efflux pump (mtr), and DNA gyrase subunit A (gyrA) which impact beta-lactam, azithromycin, and ciprofloxacin susceptibility, respectively.)]\n",
            "235: [(314, 325, Antibiotics, Recent work has highlighted the importance of commensal Neisseria (cN) as a source of AMR for their pathogenic relatives through horizontal gene transfer (HGT) of AMR alleles, such as mosaic penicillin binding protein 2 (penA), multiple transferable efflux pump (mtr), and DNA gyrase subunit A (gyrA) which impact beta-lactam, azithromycin, and ciprofloxacin susceptibility, respectively.)]\n",
            "236: [(416, 432, Gene/Proteins, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "237: [(200, 211, Antibiotics, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "238: [(222, 243, Microbes, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "239: [(252, 262, Antibiotics, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "240: [(273, 285, Microbes, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "241: [(293, 303, Antibiotics, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "242: [(314, 338, Microbes, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "243: [(365, 376, Gene/Proteins, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "244: [(443, 461, Microbes, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "245: [(463, 476, Gene/Proteins, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "246: [(487, 505, Microbes, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "247: [(539, 562, Microbes, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "248: [(567, 589, Microbes, This review provides an overview of the current situation, mechanism, epidemiology, detection methods and clinical treatment for antimicrobial resistant genes in clinical important bacteria including methicillin-resistant Staphylococcus aureus (MRSA), vancomycin-resistant Enterococcus (VRE), penicillin-resistant Streptococcus pneumoniae (PRSP), extended-spectrum β-lactamase-producing Enterobacteriaceae, acquired AmpC β-lactamase-producing Enterobacteriaceae, carbapenemase-producing Enterobacteriaceae (CPE), multidrug-resistant (MDR) Acinetobacter baumannii and Pseudomonas aeruginosa.)]\n",
            "249: [(246, 282, Techniques, We summarize the current understanding of commensal Neisseria as an AMR reservoir, and call to increase research on commensal Neisseria species, through expanding established gonococcal surveillance programs to include the collection, isolation, antimicrobial resistance phenotyping, and whole-genome sequencing (WGS) of commensal isolates.)]\n",
            "250: [(288, 317, Techniques, We summarize the current understanding of commensal Neisseria as an AMR reservoir, and call to increase research on commensal Neisseria species, through expanding established gonococcal surveillance programs to include the collection, isolation, antimicrobial resistance phenotyping, and whole-genome sequencing (WGS) of commensal isolates.)]\n",
            "251: [(242, 257, Antibiotic-Resistant Bacteria, Vaccines against infectious agents such as Streptococcus pneumoniae and Haemophilus influenzae have already been shown to reduce tolerance to antimicrobial medications; however, vaccines against some antimicrobial-resistant pathogens such as Vibrio cholerae, Salmonella typhi, Escherichia coli, nosocomial infections, and pulmonary and diarrheal disease viruses require more research and development.)]\n",
            "252: [(43, 67, Antibiotic-Resistant Bacteria, Vaccines against infectious agents such as Streptococcus pneumoniae and Haemophilus influenzae have already been shown to reduce tolerance to antimicrobial medications; however, vaccines against some antimicrobial-resistant pathogens such as Vibrio cholerae, Salmonella typhi, Escherichia coli, nosocomial infections, and pulmonary and diarrheal disease viruses require more research and development.)]\n",
            "253: [(72, 94, Antibiotic-Resistant Bacteria, Vaccines against infectious agents such as Streptococcus pneumoniae and Haemophilus influenzae have already been shown to reduce tolerance to antimicrobial medications; however, vaccines against some antimicrobial-resistant pathogens such as Vibrio cholerae, Salmonella typhi, Escherichia coli, nosocomial infections, and pulmonary and diarrheal disease viruses require more research and development.)]\n",
            "254: [(277, 293, Antibiotic-Resistant Bacteria, Vaccines against infectious agents such as Streptococcus pneumoniae and Haemophilus influenzae have already been shown to reduce tolerance to antimicrobial medications; however, vaccines against some antimicrobial-resistant pathogens such as Vibrio cholerae, Salmonella typhi, Escherichia coli, nosocomial infections, and pulmonary and diarrheal disease viruses require more research and development.)]\n",
            "255: [(259, 275, Antibiotic-Resistant Bacteria, Vaccines against infectious agents such as Streptococcus pneumoniae and Haemophilus influenzae have already been shown to reduce tolerance to antimicrobial medications; however, vaccines against some antimicrobial-resistant pathogens such as Vibrio cholerae, Salmonella typhi, Escherichia coli, nosocomial infections, and pulmonary and diarrheal disease viruses require more research and development.)]\n",
            "256: [(98, 119, Antibiotic-Resistant Bacteria, Introduction: Antimicrobial resistance (AMR) played an important role in the initial outbreaks of Clostridium difficile infection (CDI) in the 1970s.)]\n",
            "257: [(, , , AMR and selective pressure is likely to be responsible for the success of this RT and may drive future outbreaks.)]\n",
            "258: [(, , , Introduction: Anti-Microbial Resistance (AMR) is a pandemic which threatens modern medicine.)]\n",
            "259: [(88, 100, Antibiotic-Resistant Bacteria, Areas covered: This narrative review summarizes the prevalence and mechanisms of AMR in C. difficile RT 017 and transmission of these AMR mechanisms.)]\n",
            "260: [(101, 107, Antibiotic-Resistant Genes, Areas covered: This narrative review summarizes the prevalence and mechanisms of AMR in C. difficile RT 017 and transmission of these AMR mechanisms.)]\n",
            "261: [(, , , There is a lack of effective drug treatment due to the slow pace, high cost and low achievable sales prices of new antibiotic monotherapies.)]\n",
            "262: [(143, 163, Techniques, Commensal Neisseria and pathogenic Neisseria frequently share DNA through horizontal gene transfer, which has played a large role in rendering antibiotic therapies ineffective in pathogenic Neisseria populations.)]\n",
            "263: [(, , , This should be achieved by developing antimicrobial stewardship in medical, veterinary and agricultural practices.)]\n",
            "264: [(, , , AMR is the key factor that drives CDI outbreaks, and methods for the early detection of AMR can facilitate the control of outbreaks.)]\n",
            "265: [(, , , We have examined the current thinking and evidence on resistance reduction using combination therapies, with a review on toxicity and drug-drug antagonism.)]\n",
            "266: [(, , , Expert opinion: Antibiotic combination therapy, exploiting synergies, old-drug rejuvenation and resistance reduction could provide the solution to AMR.)]\n",
            "267: [(, , , Recently, the AMR Clinical Reference Center (AMR CRC) in Japan estimated the disease burden of AMR in the form of number of deaths attributed to blood-stream infections caused by antimicrobial-resistant organisms.)]\n",
            "268: [(, , , The number of pharmaceutical companies in this area is likely to expand, bringing promising combinations to the bedside, to save millions of lives worldwide.)]\n",
            "269: [(, , , However, a more extensive and precise assessment is needed to understand the disease burden of AMR more clearly and enable us to compare these indicators with those published by other countries.)]\n",
            "270: [(, , , Antimicrobial resistance threatens a resurgence of life-threatening bacterial infections and the potential demise of many aspects of modern medicine.)]\n",
            "271: [(, , , Cassini and colleagues from the World Health Organization estimated the disease burden of AMR in the European Union as disability-adjusted life years (DALYs) in 2018.)]\n",
            "272: [(, , , Despite intensive drug discovery efforts, no new classes of antibiotics have been developed into new medicines for decades, in large part owing to the stringent chemical, biological and pharmacological requisites for effective antibiotic drugs.)]\n",
            "273: [(, , , Whole genome sequence analysis of clinical and environmental V. cholerae strains revealed that the genome of most of the recent isolates harbour integrating conjugative elements (ICEs), plasmids, superintegron, transposable elements and insertion sequences, which are the key carriers of genetic traits encoding antimicrobial resistance function.)]\n",
            "274: [(, , , Combinations of antibiotics and of antibiotics with non-antibiotic activity-enhancing compounds offer a productive strategy to address the widespread emergence of antibiotic-resistant strains.)]\n",
            "275: [(, , , Here, we present an overview of the present insights on the emergence and mechanisms of AMR in V. cholerae.)]\n",
            "276: [(, , , At present, AMR CRC is attempting to establish such data for examination at the national level - a challenging but worthwhile task.)]\n",
            "277: [(, , , For terrestrial animal production, potential sources consist of feed, humans, water, air/dust, soil, wildlife, rodents, arthropods and equipment.)]\n",
            "278: [(, , , The role of food-producing environments in the emergence and spread of antimicrobial resistance (AMR) in EU plant-based food production, terrestrial animals (poultry, cattle and pigs) and aquaculture was assessed.)]\n",
            "279: [(4, 53, Others, The German Antibiotic Resistance Strategy (DART 2020) is based on the main objectives of the global action plan and was revised and published in 2015.)]\n",
            "280: [(55, 70, Antibiotic-Resistant Bacteria, The unique genetic makeup and remarkable competency of Vibrio cholerae are the key factors that help the cholera pathogen adapt rapidly to adverse environmental conditions and resist the detrimental effect of antimicrobial agents.)]\n",
            "281: [(137, 140, Others, In the last few decades, V. cholerae that causes acute watery diarrhoeal disease cholera has emerged as a notorious multidrug resistant (MDR) enteric pathogen.)]\n",
            "282: [(81, 88, Microbes, In the last few decades, V. cholerae that causes acute watery diarrhoeal disease cholera has emerged as a notorious multidrug resistant (MDR) enteric pathogen.)]\n",
            "283: [(44, 71, Others, The government of Japan established its own National Action Plan on AMR in 2016; however, Japan's AMR countermeasures are still in the developmental stage.)]\n",
            "284: [(, , , Antimicrobial resistance (AMR) is a global public-health emergency, which threatens the advances made by modern medical care over the past century.)]\n",
            "285: [(, , , Among the various sources and transmission routes identified, fertilisers of faecal origin, irrigation and surface water for plant-based food and water for aquaculture were considered of major importance.)]\n",
            "286: [(, , , In this review, we highlight the mechanisms of resistance and the genomic epidemiology of these organisms, and the impact of AMR.)]\n",
            "287: [(, , , Among those, evidence was found for introduction with feed and humans, for the other sources, the importance could not be assessed.)]\n",
            "288: [(55, 66, Antibiotic-Resistant Bacteria, Different antimicrobial resistance genes identified in V. cholerae can contribute in antibiotic resistance by facilitating one of the following three mechanisms; (i) reduced permeability or active efflux of the antibiotics, (ii) alteration of the antibiotic targets by introducing post-transcriptional/translational modifications and (iii) hydrolysis or chemical modification of antibiotics.)]\n",
            "289: [(, , , Areas covered: We performed a search of Pubmed and Medline using the keywords 'combination therapy', 'antimicrobial resistance' for articles between 1930 and 2019, as supplemented with other relevant references to our knowledge.)]\n",
            "290: [(227, 238, Antibiotic-Resistant Bacteria, Although chromosomal mutations can contribute to antimicrobial resistance (AMR), the frequent acquisition of extrachromosomal mobile genetic elements (MGEs) from closely/distantly related bacterial species are major players in V. cholerae drug resistance.)]\n",
            "291: [(, , , We have reviewed the theoretical considerations for combination development and examine the existing and future clinical indications of combination therapies.)]\n",
            "292: [(, , , Antimicrobial resistance (AMR) is currently a global health threat.)]\n",
            "293: [(, , , We have discussed the potential of antibiotic combinations to provide therapeutic synergy, rejuvenating the effectiveness of old antibiotics to which the bacteria had developed resistance previously.)]\n",
            "294: [(, , , Many countries have issued their own national action plans following the publication of the Global Action Plan on AMR by the World Health Organization.)]\n",
            "295: [(, , , Objective: Antimicrobial resistance (AMR) is a global threat to health and healthcare. In response to the growing AMR burden, research funding also increased.)]\n",
            "296: [(, , , Where do we stand in our fight against antimicrobial resistance (AMR)? Many antimicrobials may lose their clinical efficacy, particularly due to the rise of multidrug-resistant (MDR) and extended drug-resistant (XDR) pathogens, including bacteria, fungi, and parasites.)]\n",
            "297: [(, , , However, a comprehensive overview of the research output, including conceptual, temporal, and geographical trends, is missing.)]\n",
            "298: [(, , , We need weapons against them all. Society must come together against these pathogens, just like we did against HIV, cancer, and heart disease.)]\n",
            "299: [(, , , This opinion piece is, first and foremost, a call to arms for all partners involved in the war against AMR.)]\n",
            "300: [(, , , In recent times, the decreased incline of pharmaceutical industries toward research and development of newer effective antibiotics to fight this MDR pathogens have further fuelled the scarcity of antibiotics, thus the number of antibiotics in the pipeline is extremely limited.)]\n",
            "301: [(, , , A topic network was established and topic trends were analysed by frequency, proportion, and importance over time and space.)]\n",
            "302: [(, , , So, how do we win the war against AMR? We must learn from the past, so we can act in the present, to save the future.)]\n",
            "303: [(, , , Results: In total, 88 topics were identified in 158,616 articles from 166 countries. AMR publications increased by 450% between 1999 and 2018, emphasizing the vibrancy of the field.)]\n",
            "304: [(, , , Antimicrobial resistance (AMR) is increasingly perceived as a global health problem.)]\n",
            "305: [(, , , Prominent topics in 2018 were Strategies for emerging resistances and diseases, Nanoparticles, and Stewardship.)]\n",
            "306: [(, , , China and India were growing contributors in recent years, following the United States of America as overall lead contributor.)]\n",
            "307: [(, , , To tackle AMR effectively, a multisectoral one health approach is needed.)]\n",
            "308: [(, , , These factors, combined with the human, animal, and environmental interactions that underlie AMR dissemination at a population level, make for a highly complex landscape.)]\n",
            "309: [(, , , The Global Action Plan on AMR, which has been developed by the World Health Organization (WHO), in close collaboration with the Food and Agriculture Organization of the United Nations (FAO) and the World Organisation for Animal Health (OIE) is considered a blueprint to combat AMR.)]\n",
            "310: [(41, 82, Antibiotic-Resistant Bacteria, Geographical trends showed prominence of Multidrug-resistant tuberculosis (MDR-TB) in the WHO African Region, corresponding with the MDR-TB burden.)]\n",
            "311: [(133, 139, Antibiotic-Resistant Bacteria, Geographical trends showed prominence of Multidrug-resistant tuberculosis (MDR-TB) in the WHO African Region, corresponding with the MDR-TB burden.)]\n",
            "312: [(28, 38, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "313: [(40, 47, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "314: [(49, 56, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "315: [(58, 69, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "316: [(76, 87, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "317: [(89, 92, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "318: [(94, 98, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "319: [(100, 104, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "320: [(106, 109, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "321: [(114, 119, Antibiotic-Resistant Genes, Among highest priority ARGs bla CTX -M, bla VIM, bla NDM, bla OXA -48-like, bla OXA -23, mcr, armA, vanA, cfr and optrA were reported.)]\n",
            "322: [(, , , Control measures are urgently needed to slow the trajectory of AMR but are hampered by an incomplete understanding of the interplay between pathogens, AMR encoding genes, and mobile genetic elements at a microbial level.)]\n",
            "323: [(0, 23, Techniques, Whole-genome sequencing (WGS) and, more recently, metagenomic analyses have greatly enhanced our understanding of these processes, and these approaches are informing mitigation strategies for how we better understand and control AMR.)]\n",
            "324: [(163, 175, Antibiotic-Resistant Genes, Potential AMR-specific interventions are in the early stages of development. Many data gaps relating to sources and relevance of transmission routes, diversity of ARB and ARGs, effectiveness of mitigation measures were identified.)]\n",
            "325: [(25, 39, Techniques, This review explores how WGS techniques have advanced global, national, and local AMR surveillance, and how this improved understanding is being applied to inform solutions, such as novel diagnostic methods that allow antimicrobial use to be optimised and vaccination strategies for better controlling AMR.)]\n",
            "326: [(27, 42, Techniques, Therefore, this study uses topic modelling, a machine learning approach, to reveal the scientific evolution of AMR research and its trends, and provides an interactive user interface for further analyses.)]\n",
            "327: [(46, 71, Techniques, Therefore, this study uses topic modelling, a machine learning approach, to reveal the scientific evolution of AMR research and its trends, and provides an interactive user interface for further analyses.)]\n",
            "328: [(9, 41, Techniques, Methods: Structural topic modelling (STM) was applied on a text corpus resulting from a PubMed query comprising AMR articles (1999-2018).)]\n",
            "329: [(, , , Member states endorsed the action plan during the World Health Assembly 2015 and committed themselves to develop national action plans on AMR.)]\n",
            "330: [(, , , Conclusion: This study provides a comprehensive overview of the AMR research output thereby revealing the AMR research response to the increased AMR burden.)]\n",
            "331: [(, , , Both the results and the publicly available interactive database serve as a base to inform and optimise future research.)]\n",
            "332: [(, , , Antimicrobial resistance (AMR) is a major global public health threat, which has been largely driven by the excessive use of antimicrobials.)]\n",
            "333: [(, , , These highest priority bacteria and genes were identified in different sources, at primary and post-harvest level, particularly faeces/manure, soil and water.)]\n",
            "334: [(, , , Proper implementation of good hygiene practices, biosecurity and food safety management systems is very important.)]\n",
            "335: [(59, 69, Antibiotics, Several ARB of highest priority for public health, such as carbapenem or extended-spectrum cephalosporin and/or fluoroquinolone-resistant Enterobacterales (including Salmonella enterica), fluoroquinolone-resistant Campylobacter spp., methicillin-resistant Staphylococcus aureus and glycopeptide-resistant Enterococcus faecium and E. faecalis were identified.)]\n",
            "336: [(73, 154, Antibiotic-Resistant Bacteria, Several ARB of highest priority for public health, such as carbapenem or extended-spectrum cephalosporin and/or fluoroquinolone-resistant Enterobacterales (including Salmonella enterica), fluoroquinolone-resistant Campylobacter spp., methicillin-resistant Staphylococcus aureus and glycopeptide-resistant Enterococcus faecium and E. faecalis were identified.)]\n",
            "337: [(166, 185, Antibiotic-Resistant Bacteria, Several ARB of highest priority for public health, such as carbapenem or extended-spectrum cephalosporin and/or fluoroquinolone-resistant Enterobacterales (including Salmonella enterica), fluoroquinolone-resistant Campylobacter spp., methicillin-resistant Staphylococcus aureus and glycopeptide-resistant Enterococcus faecium and E. faecalis were identified.)]\n",
            "338: [(, , , Representative epidemiological and attribution studies on AMR and its effective control in food production environments at EU level, linked to One Health and environmental initiatives, are urgently required.)]\n",
            "339: [(, , , We highlight some future opportunities for AMR control informed by genomic sequencing, along with the remaining challenges that must be overcome to fully realise the potential of WGS approaches for international AMR control.)]\n",
            "340: [(, , , Recent findings: AMR is diverse, and currently Gram-negative multidrug-resistant organisms (MDROs) are the main challenge in ICUs worldwide.)]\n",
            "341: [(, , , The availability of bioinformatics tools and online accessible databases has been a prerequisite for this.)]\n",
            "342: [(, , , ICU patients are at a high risk of infection with MDRO because often multiple risk factors are present.)]\n",
            "343: [(, , , We conducted a scientific literature review and here we present a description of examples of available tools and databases for antimicrobial resistance (AMR) detection and provide future perspectives and recommendations.)]\n",
            "344: [(, , , However, nonpathogenic commensal species are rarely characterized. Here, we propose that surveillance of the universally carried commensal Neisseria may play the role of the \"canary in the coal mine,\" and reveal circulating known and novel antimicrobial resistance determinants transferable to pathogenic Neisseria.)]\n",
            "345: [(, , , Summary: AMR is a reality in most ICUs around the world, but the extent of the problem is clearly highly variable.)]\n",
            "346: [(, , , Infection prevention and control as well as appropriate antimicrobial use are the cornerstones to turn the tide.)]\n",
            "347: [(, , , Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "348: [(, , , Antimicrobial resistance (AMR) is a threat to animal and human health.)]\n",
            "349: [(, , , As a consequence, each tool has strengths and limitations in sensitivity and specificity of detection of AMR determinants and in application, which for some of the tools have been highlighted in benchmarking exercises and scientific articles.)]\n",
            "350: [(, , , The identified tools are either available at public genome data centers, from GitHub or can be run locally.)]\n",
            "351: [(0, 4, Techniques, NCBI and European Nucleotide Archive (ENA) provide possibilities for online submission of both sequencing and accompanying phenotypic antimicrobial susceptibility data, allowing for other researchers to further analyze data, and develop and test new tools.)]\n",
            "352: [(9, 42, Techniques, NCBI and European Nucleotide Archive (ENA) provide possibilities for online submission of both sequencing and accompanying phenotypic antimicrobial susceptibility data, allowing for other researchers to further analyze data, and develop and test new tools.)]\n",
            "353: [(90, 117, Techniques, We conclude by suggesting that priorities be expanded to include greater consideration of host-microbial interactions that dictate AMR, as well as anthropogenic and environmental systems that promote dissemination of AMR.)]\n",
            "354: [(147, 186, Others, We conclude by suggesting that priorities be expanded to include greater consideration of host-microbial interactions that dictate AMR, as well as anthropogenic and environmental systems that promote dissemination of AMR.)]\n",
            "355: [(87, 96, Antibiotic-Resistant Genes, The overall presence of all resistance genes in a specific environment is defined as a resistome.)]\n",
            "356: [(30, 69, Techniques, Traditional themes such as in vitro antibiotic susceptibility testing and its clinical value, AMR evolution in persistent Pseudomonas aeruginosa infection and the impact of biofilm on AMR were discussed.)]\n",
            "357: [(48, 75, Techniques, The recent advancements in rapid and affordable DNA sequencing technologies have revolutionized diagnostic microbiology and microbial surveillance.)]\n",
            "358: [(124, 146, Techniques, The recent advancements in rapid and affordable DNA sequencing technologies have revolutionized diagnostic microbiology and microbial surveillance.)]\n",
            "359: [(40, 45, Microbes, Geographical variation in prevalence of MDROs is substantial, and local epidemiology should be considered to assess the current threat of AMR.)]\n",
            "360: [(99, 123, Others, At least 47 freely accessible bioinformatics resources for detection of AMR determinants in DNA or amino acid sequence data have been developed to date.)]\n",
            "361: [(72, 95, Others, At least 47 freely accessible bioinformatics resources for detection of AMR determinants in DNA or amino acid sequence data have been developed to date.)]\n",
            "362: [(48, 57, Techniques, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder.)]\n",
            "363: [(59, 63, Techniques, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder.)]\n",
            "364: [(65, 70, Techniques, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder.)]\n",
            "365: [(72, 79, Techniques, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder.)]\n",
            "366: [(81, 91, Techniques, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder.)]\n",
            "367: [(93, 98, Techniques, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder.)]\n",
            "368: [(100, 115, Techniques, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder.)]\n",
            "369: [(116, 125, Techniques, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder.)]\n",
            "370: [(131, 140, Techniques, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder.)]\n",
            "371: [(, , , Accordingly, stakeholders in human health and livestock production have implemented antimicrobial stewardship programs aimed at reducing use.)]\n",
            "372: [(, , , Thus far, these efforts have yielded variable impacts on AMR.)]\n",
            "373: [(, , , The advancement in whole genome sequencing and the application of online tools for real-time detection of AMR determinants are essential to identify control and prevention strategies to combat the increasing threat of AMR.)]\n",
            "374: [(, , , Furthermore, scientific advances are prompting an expansion and more nuanced appreciation of the many nonantibiotic factors that drive AMR, as well as how these factors vary across systems, geographies, and contexts.)]\n",
            "375: [(, , , Accessible tools and DNA sequence data are expanding, which will allow establishing global pathogen surveillance and AMR tracking based on genomics.)]\n",
            "376: [(, , , Given these trends, we propose a framework to prioritize AMR interventions.)]\n",
            "377: [(, , , There is however, a need for standardization of pipelines and databases as well as phenotypic predictions based on the data.)]\n",
            "378: [(, , , We use this framework to evaluate the impact of interventions that focus on antimicrobial use.)]\n",
            "379: [(, , , Cities that are densely populated are reservoirs of antibiotic resistant genes (ARGs).)]\n",
            "380: [(, , , Spatial proximity of surfaces and different hygienic conditions leads to the transfer of antibiotic resistant bacteria (ARB) within urban environments.)]\n",
            "381: [(, , , Keynote lectures provided an update on the latest information regarding the phenomenon of antimicrobial resistance (AMR) in cystic fibrosis (CF).)]\n",
            "382: [(, , , Built environments, public transportation, green spaces, and citizens' behaviors all support persistence and transfer of antimicrobial resistances (AMR).)]\n",
            "383: [(, , , Purpose of review: Antimicrobial resistance (AMR) is increasing in ICUs around the world, but the prevalence is variable.)]\n",
            "384: [(, , , We will review recent literature and try to answer the question whether this is a myth or a new reality, as well as discuss challenges and potential solutions.)]\n",
            "385: [(, , , In addition, the report gives an overview on very recent AMR-related topics that include an ecological view of AMR in CF lung, referred to as resistome, and novel anti-infective approaches in preclinical or early clinical research such as antibiofilm drugs and bacteriophages.)]\n",
            "386: [(, , , The discovery of novel antimicrobial agents is urgently needed to overcome the challenges created by AMR.)]\n",
            "387: [(, , , In this review we gather existing knowledge on S. canis, describing its epidemiology in animals and humans and present information on virulence factors, classification schemes and AMR prevalence.)]\n",
            "388: [(, , , EOs and their constituents that hold pro-oxidant properties can induce oxidative stress by producing reactive oxygen species (ROS), leading to biological damage in target cells.)]\n",
            "389: [(, , , Even more so, it is an attempt to highlight the positives in a seemingly long line of failures, and to identify the current set of challenges we must work on.)]\n",
            "390: [(, , , In contrast, the antioxidant properties scavenge free radicals through offsetting ROS. Both pro-oxidant and antioxidant activities in EOs represent a promising strategy to tackle AMR.)]\n",
            "391: [(, , , Thus, this review aimed to discuss how pro-oxidants and antioxidants in EOs may contribute to the mitigation of AMR and provided a detailed description of the challenges and limitations of utilizing them as a means to combat AMR.)]\n",
            "392: [(, , , Background: Training is needed to increase awareness and understanding of the complex problem of antimicrobial resistance (AMR) among professionals.)]\n",
            "393: [(, , , However, AMR capacity building often does not stretch beyond the biomedical sciences, limiting interdisciplinary collaboration.)]\n",
            "394: [(, , , We also report the range of known AMR phenotypes and the emergence of new mechanisms of resistance.)]\n",
            "395: [(0, 19, Microbes, Streptococcus canis may be regarded as a neglected pathogen of one health concern. Further research is needed for its better understanding and effective control.)]\n",
            "396: [(146, 164, Others, The UN General Assembly has listed AMR among issues that need urgent address; subsequently FAO/OIE/WHO tripartite took lead and jointly developed Global Action Plan (GAP) to tackle the problem.)]\n",
            "397: [(91, 102, Others, The UN General Assembly has listed AMR among issues that need urgent address; subsequently FAO/OIE/WHO tripartite took lead and jointly developed Global Action Plan (GAP) to tackle the problem.)]\n",
            "398: [(97, 112, Techniques, In the social science domain, institution and policy elements are most frequently covered, while transformations are the least covered.)]\n",
            "399: [(0, 19, Microbes, Streptococcus canis is a multi-host pathogen that causes disease of varying severity in a wide range of mammals, including humans.)]\n",
            "400: [(130, 138, Microbes, Dogs and cats appear to be the primary hosts and may play a role in transmitting infection to humans. The broader epidemiology of S. canis, however, is still poorly understood, as are its virulence mechanisms, antimicrobial resistance (AMR) and population structure.)]\n",
            "401: [(189, 200, Others, Natural products such as plant extracts and essential oils (EOs) have been viewed as potential candidates to combat AMR due to their complex chemistry that carries inherent pro-oxidant and antioxidant properties.)]\n",
            "402: [(173, 184, Others, Natural products such as plant extracts and essential oils (EOs) have been viewed as potential candidates to combat AMR due to their complex chemistry that carries inherent pro-oxidant and antioxidant properties.)]\n",
            "403: [(44, 64, Others, Natural products such as plant extracts and essential oils (EOs) have been viewed as potential candidates to combat AMR due to their complex chemistry that carries inherent pro-oxidant and antioxidant properties.)]\n",
            "404: [(142, 171, Others, We describe the main ecological niches of S. canis in companion animals, discuss potential risk factors for infection in humans and propose a multi-host transmission cycle.)]\n",
            "405: [(42, 50, Microbes, We describe the main ecological niches of S. canis in companion animals, discuss potential risk factors for infection in humans and propose a multi-host transmission cycle.)]\n",
            "406: [(34, 42, Microbes, We show that current knowledge on S. canis virulence determinants is limited and sometimes contradictory.)]\n",
            "407: [(64, 72, Antibiotic-Resistant Bacteria, We illustrate the different typing systems proposed to classify S. canis.)]\n",
            "408: [(, , , Methods: Twenty-eight training courses covering social dimensions of AMR were identified via a survey (n = 133), interviews (n = 6) and an additional internet search.)]\n",
            "409: [(, , , Antimicrobial Resistance (AMR) is already affecting human and animal health sectors negatively.)]\n",
            "410: [(, , , General characteristics, quality and social science relevance indicators were extracted and analysed for each of these training courses.)]\n",
            "411: [(, , , Results: Because only 57% of the analysed training courses were fully focused on AMR, AMR was usually superficially covered, focusing on the biomedical basics and just mentioning some social aspects without using social science theories or experts.)]\n",
            "412: [(38, 69, Techniques, States and Governments have developed AMR National Action Plans (NAP), Policies and Communication Strategies.)]\n",
            "413: [(, , , Only 3 of the 28 training courses covered AMR primarily from a social science perspective, while only 14% of the educators involved had social science expertise.)]\n",
            "414: [(, , , In health communication realms AMR is a distinctive and novel phenomenon. Consequently, communication theories and principles on this issue have not been tested adequately; even as literature around AMR is growing.)]\n",
            "415: [(, , , Biomedical dimensions of AMR were covered twice as much as the social science dimensions.)]\n",
            "416: [(, , , While there are lessons that can be learned from previous and ongoing health communication campaigns targeting various audiences and coined around behavior change strategies; AMR is certainly a unique and maiden case. Kenya developed and is implementing its AMR - NAP.)]\n",
            "417: [(, , , Kenya is among few countries that have developed and implemented an AMR Communication Strategy with the goal to create awareness and subsequently change audience behavior.)]\n",
            "418: [(, , , This paper presents a review of available literature, reports, Kenya experiences and theories that can be applied to AMR communications.)]\n",
            "419: [(, , , It illustrates the gaps, opportunities and challenges and proposes strategies that can be applied at the moment as more literature on AMR communication is collected through research and documentation of country experiences.)]\n",
            "420: [(, , , Our understanding and our ability to control the rise of AMR in an urban setting is linked to our knowledge of the network connecting urban reservoirs and the environment.)]\n",
            "421: [(, , , The rapid evolution of antimicrobial resistance (AMR) has remained a major public health issue, reducing the efficacy of antibiotics and increasing the difficulty of treating infections.)]\n",
            "422: [(, , , It is imperative for antimicrobial stewardship to be more widely implemented to achieve better control of the AMR phenomenon.)]\n",
            "423: [(, , , The specific needs to be addressed and barriers to be overcome, in particular, in low- and middle-income countries in order to implement antimicrobial stewardship practices in agrifood systems are being identified.)]\n",
            "424: [(, , , Solutions should focus on reducing the risk of cross-transmission in the ICU through strict infection prevention and control practices and reducing exposure to antimicrobials as the major contributor to the development of AMR.)]\n",
            "425: [(, , , As a global community, it is essential that we now move beyond discussing the AMR problem and focus on implementing solutions.)]\n",
            "426: [(, , , This review focuses on therapeutic options for AMR mechanisms of resistance in bacteria that can be overcome by combinational therapy and testing methods for synergy.)]\n",
            "427: [(, , , Together, these will contribute to the control of AMR and also bring us closer to the achievement of multiple sustainable development goals.)]\n",
            "428: [(, , , The integration of diverse approaches may provide information that is imperative in mitigating the threat of AMR.)]\n",
            "429: [(, , , With increasing reports on antimicrobial resistance (AMR) in humans, animals and the environment, we are at risk of returning to a pre-antibiotic era.)]\n",
            "430: [(46, 64, Techniques, Classical approaches for AMR diagnostics like phenotypic methods, biochemical assays, and molecular techniques are cumbersome and resource-intensive and involve a long turnaround time to yield confirmatory results.)]\n",
            "431: [(66, 84, Techniques, Classical approaches for AMR diagnostics like phenotypic methods, biochemical assays, and molecular techniques are cumbersome and resource-intensive and involve a long turnaround time to yield confirmatory results.)]\n",
            "432: [(0, 24, Antibiotic-Resistant Bacteria, Streptococcus pneumoniae is a major cause of pneumonia, meningitis, and septicaemia worldwide.)]\n",
            "433: [(154, 166, Antibiotic-Resistant Bacteria, Here, we discuss a variety of evolutionary mechanisms at the host, pathogen, and environmental levels that may contribute to changes in the prevalence of pneumococcal AMR in the post-vaccination era.)]\n",
            "434: [(0, 37, Others, The Food and Agriculture Organization (FAO) of the United Nations plays an important role in promoting and facilitating antimicrobial stewardship.)]\n",
            "435: [(65, 77, Microbes, The relative importance of these factors may vary by population, pneumococcal lineage, geography, and time, leading to the complex relationship between vaccination, antibiotic use, and AMR.)]\n",
            "436: [(152, 163, Techniques, The relative importance of these factors may vary by population, pneumococcal lineage, geography, and time, leading to the complex relationship between vaccination, antibiotic use, and AMR.)]\n",
            "437: [(151, 165, Antibiotic-Resistant Bacteria, Antimicrobial resistance (AMR) has become a major concern for healthcare workers due to the emergence of new variants of resistant markers, especially carbapenemases.)]\n",
            "438: [(0, 32, Techniques, Combinational antibiotic therapy is one of the best and easiest approaches to handle the current situation of AMR. Although some antibiotic combinations are already in clinical use, they remain to be studied in detail.)]\n",
            "439: [(13, 16, Others, For example, FAO is developing a platform to collect data on AMR in animals and antimicrobial use (AMU) in plants (InFARM), working on a campaign to reduce the need to use antimicrobials, studying the use of alternatives to the use of antimicrobials (especially those used for growth promotion) and actively promoting the implementation of the Codex Alimentarius AMR standards.)]\n",
            "440: [(, , , The rampant and inappropriate use of antibiotics due to the lack of adequate and timely diagnosis is a leading cause behind AMR evolution.)]\n",
            "441: [(, , , Therefore, AMR is recognized as one of the major global health threats of this century. Antibiotics are used extensively in farming systems to treat and prevent infections in food animals or to increase their growth.)]\n",
            "442: [(, , , Unfortunately, populations with poor economic status and those residing in densely populated areas are the most affected ones, frequently leading to emergence of AMR pathogens.)]\n",
            "443: [(, , , Besides the risk of a transfer of AMR between the human and the animal sector, there is another yet largely overlooked sector in the One Health triad.)]\n",
            "444: [(, , , In contrast, recent emergence of nanotechnology-assisted approaches helps to overcome challenges in classical approaches and offer simpler, more sensitive, faster, and more affordable solutions for AMR diagnostics.)]\n",
            "445: [(, , , Nanomaterial platforms (metallic, quantum-dot, carbon-based, upconversion, etc.), nanoparticle-based rapid point-of-care platforms, nano-biosensors (optical, mechanical, electrochemical), microfluidic-assisted devices, and importantly, nanotheranostic devices for diagnostics with treatment of AMR infections are examples of rapidly growing nanotechnology approaches used for AMR management.)]\n",
            "446: [(, , , This review comprehensively summarizes the past 10 years of research progress on nanotechnology approaches for AMR diagnostics and for estimating antimicrobial susceptibility against commonly used antibiotics.)]\n",
            "447: [(, , , Conclusions: There is a clear gap in educational resources on AMR, but moreover for social scientists wanting to engage in AMR, or for non-social scientists wanting to learn about the social dimensions of AMR from an interdisciplinary perspective.)]\n",
            "448: [(, , , This review also highlights several bottlenecks in nanotechnology approaches that need to be addressed prior to considering their translation to clinics.)]\n",
            "449: [(, , , This gap needs to be bridged if we want social sciences to become a relevant partner in the struggle against AMR.)]\n",
            "450: [(, , , Antimicrobial resistance (AMR) has been clearly identified as a major global health challenge.)]\n",
            "451: [(, , , It is a leading cause of human deaths and also has a toll on animals, plants, and the environment.)]\n",
            "452: [(, , , Pneumococcal antimicrobial resistance (AMR) has been highlighted by the WHO as an important public health concern, with emerging serotypes showing resistance to multiple antibiotics.)]\n",
            "453: [(, , , Despite the considerable socio-economic impacts, the level of awareness of the problem remains woefully inadequate, and antimicrobials are not generally recognized as a global common good, one that everyone has a role and responsibility to conserve.)]\n",
            "454: [(, , , Articles were screened using strict inclusion/exclusion criteria. AST data was extracted, with medians and ranges of resistance rates calculated for specific bug-drug combinations.)]\n",
            "455: [(, , , Therefore, methods to effectively monitor memory B cells and long-lived plasma cells and subsequently prevent DSA production are key to reducing the adverse effects of AMR.)]\n",
            "456: [(, , , Results: Twenty-four papers were included for final analysis, with 20 describing isolates from human populations.)]\n",
            "457: [(, , , Therefore, this review mainly summarizes the production pathways of memory B cells and long-lived plasma cells and provides suggestions for the prevention of AMR after transplantation.)]\n",
            "458: [(, , , Healthcare-associated infections (HAIs) affect up to 15% of all hospitalized patients, representing a global concern.)]\n",
            "459: [(, , , Major causes include the persistent microbial contamination of hospital environment, and the growing antimicrobial-resistance (AMR) of HAI-associated microbes.)]\n",
            "460: [(, , , To support this approach, several AMR databases and gene identification algorithms have been recently developed.)]\n",
            "461: [(, , , The recently implemented national AMR surveillance systems will begin to address this data gap.)]\n",
            "462: [(, , , Antimicrobial resistance (AMR) circulates through humans, animals, and the environments, requiring a One Health approach.)]\n",
            "463: [(, , , The control of contamination has been so far addressed by the use of chemical-based sanitation procedures, which however have limitations, as testified by the persistence of contamination itself and by the growing AMR of hospital microbes.)]\n",
            "464: [(, , , Recently, urban sewage has increasingly been suggested as a hotspot for AMR even in high-income countries (HICs), where the water sanitation and hygiene infrastructure are well-developed.)]\n",
            "465: [(, , , Here we review the results collected by a microbial-based sanitation system, inspired by the microbiome balance principles, in obtaining more effective control of microbial contamination and AMR.)]\n",
            "466: [(, , , To understand the current status of AMR in wastewater in a HIC, we reviewed the epidemiological studies on AMR in the sewage environment in Japan from the published literature.)]\n",
            "467: [(, , , Soil resistomes are augmented by the application of manure from treated livestock.)]\n",
            "468: [(76, 111, Techniques, Currently, no recognized effective treatments are available, especially for chronic antibody-mediated rejection (CAMR).)]\n",
            "469: [(0, 32, Gene/Proteins, Donor-specific antibodies (DSAs) secreted by long-lived plasma cells and memory B cells are acknowledged as biomarkers of AMR.)]\n",
            "470: [(41, 64, Techniques, Nevertheless, it may be too late for the DSA routine examination production since DSAs may have binded to graft vascular endothelial cells through complement-dependent or complement-independent pathways.)]\n",
            "471: [(147, 167, Techniques, Nevertheless, it may be too late for the DSA routine examination production since DSAs may have binded to graft vascular endothelial cells through complement-dependent or complement-independent pathways.)]\n",
            "472: [(171, 202, Techniques, Nevertheless, it may be too late for the DSA routine examination production since DSAs may have binded to graft vascular endothelial cells through complement-dependent or complement-independent pathways.)]\n",
            "473: [(0, 16, Microbes, Escherichia coli was the most commonly described organism, with median resistance rates from human isolates of 92.8% (n=6 articles), 46.4% (n=4), 55.4% (n=8), and 46.4% (n=5) to ampicillin, 3rd generation cephalosporins, fluoroquinolones, and gentamicin respectively.)]\n",
            "474: [(178, 188, Antibiotics, Escherichia coli was the most commonly described organism, with median resistance rates from human isolates of 92.8% (n=6 articles), 46.4% (n=4), 55.4% (n=8), and 46.4% (n=5) to ampicillin, 3rd generation cephalosporins, fluoroquinolones, and gentamicin respectively.)]\n",
            "475: [(205, 219, Antibiotics, Escherichia coli was the most commonly described organism, with median resistance rates from human isolates of 92.8% (n=6 articles), 46.4% (n=4), 55.4% (n=8), and 46.4% (n=5) to ampicillin, 3rd generation cephalosporins, fluoroquinolones, and gentamicin respectively.)]\n",
            "476: [(243, 253, Antibiotics, Escherichia coli was the most commonly described organism, with median resistance rates from human isolates of 92.8% (n=6 articles), 46.4% (n=4), 55.4% (n=8), and 46.4% (n=5) to ampicillin, 3rd generation cephalosporins, fluoroquinolones, and gentamicin respectively.)]\n",
            "477: [(221, 237, Antibiotics, Escherichia coli was the most commonly described organism, with median resistance rates from human isolates of 92.8% (n=6 articles), 46.4% (n=4), 55.4% (n=8), and 46.4% (n=5) to ampicillin, 3rd generation cephalosporins, fluoroquinolones, and gentamicin respectively.)]\n",
            "478: [(58, 73, Antibiotic-Resistant Bacteria, Conclusions: Whilst resistance rates are high for several GLASS organisms, there were insufficient data to draw robust conclusions about the AMR situation in Cambodia.)]\n",
            "479: [(, , , Our review showed that a wide variety of clinically important antibiotic-resistant bacteria (ARB), antibiotic resistance genes (ARGs), and antimicrobial residues are present in human wastewater in Japan.)]\n",
            "480: [(, , , The control of infectious diseases has always been a top medical priority.)]\n",
            "481: [(, , , Their concentrations are lower than in low- and middle-income countries (LMICs) and are further reduced by sewage treatment plants (STPs) before discharge.)]\n",
            "482: [(, , , For years during the so-called antibiotic era, we enjoyed prolonged life expectancy and the benefits of superior pathogen control.)]\n",
            "483: [(, , , Human-dominated ecosystems such as agricultural soils are a major sink for antibiotics and AMR originating from livestock farming.)]\n",
            "484: [(, , , Nevertheless, the remaining ARB and ARGs could be an important source of AMR contamination in river water.)]\n",
            "485: [(, , , This review summarizes current knowledge on the prevalence of AMR at the interface of animal and agricultural production and discusses the potential implications for human health.)]\n",
            "486: [(, , , Furthermore, hospital effluence may be an important reservoir of clinically important ARB.)]\n",
            "487: [(, , , The high concentration of antimicrobial agents commonly prescribed in Japan may contribute to the selection and dissemination of AMR within wastewater.)]\n",
            "488: [(, , , Subsequent transfer of AMR into plant microbiomes may likely play a critical role in human exposure to antibiotic resistance in the environment.)]\n",
            "489: [(, , , Our review shows the importance of both monitoring for AMR and antimicrobials in human wastewater and efforts to reduce their contamination load in wastewater.)]\n",
            "490: [(, , , Based on the knowledge that is currently available we advocate that more attention should be paid to the role of environmental resistomes in the AMR crisis.)]\n",
            "491: [(, , , Antibody-mediated rejection (AMR) has a strongly negative impact on long-term renal allograft survival.)]\n",
            "492: [(, , , Objectives: Following the launch of the Global Antimicrobial Resistance Surveillance System (GLASS), antimicrobial resistance (AMR) rates in many countries remain poorly described.)]\n",
            "493: [(, , , This review provides an overview of published AMR data from Cambodia in the context of recently initiated national human and food-animal surveillance.)]\n",
            "494: [(, , , Methods: PubMed and the Cochrane Database of Systematic Reviews were searched for articles published from 2000 to 2018, which reported antimicrobial susceptibility testing (AST) data for GLASS specific organisms isolated from Cambodia.)]\n",
            "495: [(, , , In the last century, life expectancy has increased considerably, thanks to the introduction of antibiotics, hygiene and vaccines that have contributed to the cure and prevention of many infectious diseases.)]\n",
            "496: [(, , , Antimicrobial resistance (AMR) refers to microorganisms becoming resistant to antibiotics that were designed and expected to kill them.)]\n",
            "497: [(, , , The need of the hour is to address the grave threat of AMR, which is loitering silently; if not the mankind will endure more affliction hereafter.)]\n",
            "498: [(, , , To understand the dynamics of antimicrobial resistance (AMR), in a One-Health perspective, surveillance play an important role.)]\n",
            "499: [(, , , However, immediately after the introduction of these novel drugs, microorganisms started to become resistant through different strategies.)]\n",
            "500: [(, , , Monitoring systems already exist in the human health and livestock sectors, but there are no environmental monitoring programs.)]\n",
            "501: [(, , , Although resistance mechanisms were already present before antibiotic introduction, their large-scale use and mis-use have increased the number of resistant microorganisms.)]\n",
            "502: [(, , , Therefore there is an urgent need to initiate environmental AMR monitoring programs nationally and globally, which will complement existing systems in different sectors.)]\n",
            "503: [(, , , However, environmental programs should not only identify anthropogenic influences and levels of AMR, but they should also allow for identification of transmissions to and from human and animal populations.)]\n",
            "504: [(, , , Today, antimicrobial resistance (AMR) remains one of the major global concerns to be addressed and only global efforts could help in finding a solution.)]\n",
            "505: [(, , , In terms of magnitude the economic impact of AMR is estimated to be comparable to that of climate global change in 2030.)]\n",
            "506: [(, , , Antibiotics have made it possible to treat bacterial infections such as meningitis and bacteraemia that, prior to their introduction, were untreatable and consequently fatal.)]\n",
            "507: [(, , , Consequently, there is a shortage of antibiotics to treat Multidrug Resistance (MDR) and Extensively Drug Resistance (XDR) patients.)]\n",
            "508: [(, , , Unfortunately, in recent decades overuse and misuse of antibiotics as well as social and economic factors have accelerated the spread of antibiotic-resistant bacteria, making drug treatment ineffective.)]\n",
            "509: [(81, 85, Gene/Proteins, Among non-antibiotic strategies, vaccines and therapeutic monoclonal antibodies (mAbs) play a strategic role.)]\n",
            "510: [(, , , The primary cause of AMR is the vast array of anthropogenic disturbances in natural microfauna brought about by the extensive use of antibiotics.)]\n",
            "511: [(, , , This surge in antibiotic usage has escalated the residual antibiotics reaching Wastewater Treatment Plants (WWTPs) from pharmaceutical companies, health care centers, and domestic settings.)]\n",
            "512: [(11, 32, Techniques, The era of antimicrobial therapy started in the nineteenth century with the identification of chemical compounds with antimicrobial properties.)]\n",
            "513: [(73, 89, Antibiotic-Resistant Bacteria, In the current review we therefore propose using antimicrobial resistant Escherichia coli as indicators for monitoring occurrence and levels of AMR in the environment, including wildlife.)]\n",
            "514: [(, , , Currently, at least 700,000 people worldwide die each year due to antimicrobial resistance (AMR).)]\n",
            "515: [(, , , In this review, we will summarize the evolution and the mechanisms of antibiotic resistance, and the impact of AMR on life expectancy and economics.)]\n",
            "516: [(, , , The devastating failure of the medical system, agriculture and pharmaceutical companies and the general population to appreciate and safeguard these benefits is now leading us into a grim post-antibiotic era.)]\n",
            "517: [(, , , Antimicrobial resistance (AMR) is emerging as a severe concern due to the escalating instances of resistant human pathogens encountered by health workers.)]\n",
            "518: [(, , , Prior to the COVID-19 pandemic, AMR was recognised by the World Health Organization as the central priority area with growing public awareness of the threat AMR now presents.)]\n",
            "519: [(, , , The Review on Antimicrobial Resistance, a project commissioned by the UK government, predicted that the death toll of AMR could be one person every 3 seconds, amounting to 10 million deaths per year by 2050.)]\n",
            "520: [(, , , Coronavirus Disease of 2019 (COVID-19) has crashed antibiotic stewardship and single-handedly increased the global usage of antibiotics, Personal Protective Equipment (PPE), and biocide, causing a ripple effect in the existing global AMR problem.)]\n",
            "521: [(, , , This review aims to raise awareness of the evergrowing extensiveness of antimicrobial resistance and identify major sources of this adversity, focusing on agriculture's role in this problem and its solutions.)]\n",
            "522: [(, , , KEYPOINTS: • Widespread development of antibiotic resistance is a major global health risk.)]\n",
            "523: [(, , , Ultimately the natural water bodies receiving their effluents will have higher concentrations of emerging contaminants as the WWTPs cannot remove the Pharmaceuticals and Personal Care Products (PPCPs) completely.)]\n",
            "524: [(, , , Antibiotic resistance is abundant in agricultural produce, soil, food, water, air and probiotics.)]\n",
            "525: [(, , , Furthermore, increased biocides usage will increase AMR by co-resistance, and increasing plastics will turn into microplastics and get converted to plastisphere, which will further enhance its propagation.)]\n",
            "526: [(, , , New approaches are being developed to control and reduce antimicrobial resistance.)]\n",
            "527: [(, , , Therefore, it is crucial to curb antibiotic usage, implement antibiotic stewardship dynamically; and, ameliorate the present condition of WWTPs to remove residual PPCPs efficiently.)]\n",
            "528: [(, , , Members of both classes interact with a number of components in the cells of these bacteria, so the cellular targets are also considered.)]\n",
            "529: [(, , , Expression of efflux systems is regulated by GntR-like (S. aureus NorG), MarR-like (MgrA, MepR) regulators or a two-component signal transduction system (TCS) (S. aureus ArlSR).)]\n",
            "530: [(, , , No AMR genotype was identified in any strains belonging to the cryptic clades. C. difficile from Australia/New Zealand had the lowest AMR prevalence compared to strains from Asia, Europe and North America (P<0.0001). Based on the phylogenetic clade, AMR prevalence was higher in clades 2 (84.3 %), 4 (81.5 %) and 5 (64.8 %) compared to other clades (collectively 26.9 %) (P<0.0001). MDR prevalence was highest in clade 4 (61.6 %) which was over three times higher than in clade 2, the clade with the second-highest MDR prevalence (18.3 %).)]\n",
            "531: [(, , , Resistance to vancomycin occurs through modification of the D-Ala-D-Ala target in the cell wall peptidoglycan and removal of high affinity precursors, or by target protection via cell wall thickening.)]\n",
            "532: [(, , , Of the six Van resistance types (VanA-E, VanG), the VanA resistance type is considered in this review, including its regulation by the VanSR TCS.)]\n",
            "533: [(83, 104, Antibiotic-Resistant Bacteria, In this review, we discuss mechanisms of resistance identified in bacterial agents Staphylococcus aureus and the enterococci towards two priority classes of antibiotics-the fluoroquinolones and the glycopeptides.)]\n",
            "534: [(112, 124, Microbes, In this review, we discuss mechanisms of resistance identified in bacterial agents Staphylococcus aureus and the enterococci towards two priority classes of antibiotics-the fluoroquinolones and the glycopeptides.)]\n",
            "535: [(173, 189, Antibiotics, In this review, we discuss mechanisms of resistance identified in bacterial agents Staphylococcus aureus and the enterococci towards two priority classes of antibiotics-the fluoroquinolones and the glycopeptides.)]\n",
            "536: [(198, 212, Antibiotics, In this review, we discuss mechanisms of resistance identified in bacterial agents Staphylococcus aureus and the enterococci towards two priority classes of antibiotics-the fluoroquinolones and the glycopeptides.)]\n",
            "537: [(, , , An important transfer route into the aquatic environment is the urban water cycle.)]\n",
            "538: [(186, 207, Antibiotic-Resistant Bacteria, In February 2017, in light of increasing antibiotic resistance, the WHO published a list of pathogens that includes the pathogens designated by the acronym ESKAPE (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) to which were given the highest \"priority status\" since they represent the great threat to humans.)]\n",
            "539: [(209, 230, Antibiotic-Resistant Bacteria, In February 2017, in light of increasing antibiotic resistance, the WHO published a list of pathogens that includes the pathogens designated by the acronym ESKAPE (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) to which were given the highest \"priority status\" since they represent the great threat to humans.)]\n",
            "540: [(232, 255, Antibiotic-Resistant Bacteria, In February 2017, in light of increasing antibiotic resistance, the WHO published a list of pathogens that includes the pathogens designated by the acronym ESKAPE (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) to which were given the highest \"priority status\" since they represent the great threat to humans.)]\n",
            "541: [(257, 279, Antibiotic-Resistant Bacteria, In February 2017, in light of increasing antibiotic resistance, the WHO published a list of pathogens that includes the pathogens designated by the acronym ESKAPE (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) to which were given the highest \"priority status\" since they represent the great threat to humans.)]\n",
            "542: [(285, 305, Antibiotic-Resistant Bacteria, In February 2017, in light of increasing antibiotic resistance, the WHO published a list of pathogens that includes the pathogens designated by the acronym ESKAPE (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) to which were given the highest \"priority status\" since they represent the great threat to humans.)]\n",
            "543: [(164, 184, Antibiotic-Resistant Bacteria, In February 2017, in light of increasing antibiotic resistance, the WHO published a list of pathogens that includes the pathogens designated by the acronym ESKAPE (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) to which were given the highest \"priority status\" since they represent the great threat to humans.)]\n",
            "544: [(, , , In this paper the occurrence and transport of antibiotic microbial resistance in the urban water cycle are critically reviewed.)]\n",
            "545: [(156, 175, Techniques, A more efficient removal of antibiotic microbial resistance abundances from wastewater effluents can be achieved by advanced treatment processes, including membrane filtration, ozonation, UV-irradiation or chlorination, to levels typically observed in urban surface water or low impacted surface water.)]\n",
            "546: [(177, 186, Techniques, A more efficient removal of antibiotic microbial resistance abundances from wastewater effluents can be achieved by advanced treatment processes, including membrane filtration, ozonation, UV-irradiation or chlorination, to levels typically observed in urban surface water or low impacted surface water.)]\n",
            "547: [(187, 202, Techniques, A more efficient removal of antibiotic microbial resistance abundances from wastewater effluents can be achieved by advanced treatment processes, including membrane filtration, ozonation, UV-irradiation or chlorination, to levels typically observed in urban surface water or low impacted surface water.)]\n",
            "548: [(206, 218, Techniques, A more efficient removal of antibiotic microbial resistance abundances from wastewater effluents can be achieved by advanced treatment processes, including membrane filtration, ozonation, UV-irradiation or chlorination, to levels typically observed in urban surface water or low impacted surface water.)]\n",
            "549: [(34, 58, Antibiotic-Resistant Bacteria, Antimicrobial resistance (AMR) in Clostridioides difficile remains a significant threat to global healthcare systems, not just for the treatment of C.difficile infection (CDI), but as a reservoir of AMR genes that could be potentially transferred to other pathogens.)]\n",
            "550: [(148, 159, Antibiotic-Resistant Bacteria, Antimicrobial resistance (AMR) in Clostridioides difficile remains a significant threat to global healthcare systems, not just for the treatment of C.difficile infection (CDI), but as a reservoir of AMR genes that could be potentially transferred to other pathogens.)]\n",
            "551: [(0, 5, Antibiotic-Resistant Bacteria, ExPEC strains are versatile bacteria that can cause urinary tract, bloodstream, prostate, and other infections at non-intestinal sites.)]\n",
            "552: [(82, 99, Others, The mechanisms of resistance for several antimicrobials such as metronidazole and MLSB-class agents are only beginning to be elucidated, and increasingly, there is evidence that previously unconsidered mechanisms such as plasmid-mediated resistance may play an important role in AMR in this bacterium.)]\n",
            "553: [(64, 77, Antibiotics, The mechanisms of resistance for several antimicrobials such as metronidazole and MLSB-class agents are only beginning to be elucidated, and increasingly, there is evidence that previously unconsidered mechanisms such as plasmid-mediated resistance may play an important role in AMR in this bacterium.)]\n",
            "554: [(221, 248, Techniques, The mechanisms of resistance for several antimicrobials such as metronidazole and MLSB-class agents are only beginning to be elucidated, and increasingly, there is evidence that previously unconsidered mechanisms such as plasmid-mediated resistance may play an important role in AMR in this bacterium.)]\n",
            "555: [(39, 51, Antibiotic-Resistant Bacteria, In this review, the genetics of AMR in C. difficile will be described, along with a discussion of the factors contributing to the difficulty in clearly determining the true burden of AMR in C. difficile and how it affects the treatment of CDI.)]\n",
            "556: [(190, 202, Antibiotic-Resistant Bacteria, In this review, the genetics of AMR in C. difficile will be described, along with a discussion of the factors contributing to the difficulty in clearly determining the true burden of AMR in C. difficile and how it affects the treatment of CDI.)]\n",
            "557: [(0, 15, Antibiotics, Fluoroquinolone resistance mechanisms include efflux pumps (MepA, NorA, NorB, NorC, MdeA, LmrS or SdrM in S. aureus and EfmA or EfrAB in the enterococci) for removal of fluoroquinolone from the intracellular environment of bacterial cells and/or protection of the gyrase and topoisomerase IV target sites in Enterococcus faecalis by Qnr-like proteins.)]\n",
            "558: [(46, 58, Techniques, Fluoroquinolone resistance mechanisms include efflux pumps (MepA, NorA, NorB, NorC, MdeA, LmrS or SdrM in S. aureus and EfmA or EfrAB in the enterococci) for removal of fluoroquinolone from the intracellular environment of bacterial cells and/or protection of the gyrase and topoisomerase IV target sites in Enterococcus faecalis by Qnr-like proteins.)]\n",
            "559: [(106, 115, Antibiotic-Resistant Bacteria, Fluoroquinolone resistance mechanisms include efflux pumps (MepA, NorA, NorB, NorC, MdeA, LmrS or SdrM in S. aureus and EfmA or EfrAB in the enterococci) for removal of fluoroquinolone from the intracellular environment of bacterial cells and/or protection of the gyrase and topoisomerase IV target sites in Enterococcus faecalis by Qnr-like proteins.)]\n",
            "560: [(141, 152, Antibiotic-Resistant Bacteria, Fluoroquinolone resistance mechanisms include efflux pumps (MepA, NorA, NorB, NorC, MdeA, LmrS or SdrM in S. aureus and EfmA or EfrAB in the enterococci) for removal of fluoroquinolone from the intracellular environment of bacterial cells and/or protection of the gyrase and topoisomerase IV target sites in Enterococcus faecalis by Qnr-like proteins.)]\n",
            "561: [(308, 329, Antibiotic-Resistant Bacteria, Fluoroquinolone resistance mechanisms include efflux pumps (MepA, NorA, NorB, NorC, MdeA, LmrS or SdrM in S. aureus and EfmA or EfrAB in the enterococci) for removal of fluoroquinolone from the intracellular environment of bacterial cells and/or protection of the gyrase and topoisomerase IV target sites in Enterococcus faecalis by Qnr-like proteins.)]\n",
            "562: [(333, 336, Gene/Proteins, Fluoroquinolone resistance mechanisms include efflux pumps (MepA, NorA, NorB, NorC, MdeA, LmrS or SdrM in S. aureus and EfmA or EfrAB in the enterococci) for removal of fluoroquinolone from the intracellular environment of bacterial cells and/or protection of the gyrase and topoisomerase IV target sites in Enterococcus faecalis by Qnr-like proteins.)]\n",
            "563: [(41, 53, Antibiotics, Resistance to the glycopeptide antibiotic teicoplanin occurs via efflux regulated by the TcaR regulator in S. aureus.)]\n",
            "564: [(65, 71, Techniques, Resistance to the glycopeptide antibiotic teicoplanin occurs via efflux regulated by the TcaR regulator in S. aureus.)]\n",
            "565: [(89, 103, Antibiotic-Resistant Genes, Resistance to the glycopeptide antibiotic teicoplanin occurs via efflux regulated by the TcaR regulator in S. aureus.)]\n",
            "566: [(, , , The World Health Organization (WHO) has identified the spread of antibiotic resistance as one of the major risks to global public health.)]\n",
            "567: [(, , , Without new and better treatments, the World Health Organization (WHO) predicts that this number could rise to 10 million by 2050, highlighting a health concern not of secondary importance.)]\n",
            "568: [(, , , Understanding the resistance mechanisms of these bacteria is a key step in the development of new antimicrobial drugs to tackle drug-resistant bacteria.)]\n",
            "569: [(, , , The presence of antibiotic resistance in low impacted surface water is being discussed to determine background antibiotic resistance levels, which might serve as a reference for treatment targets in the absence of health-based threshold levels.)]\n",
            "570: [(, , , In this review, both the mode of action and the mechanisms of resistance of commonly used antimicrobials will be examined.)]\n",
            "571: [(, , , Different biological, physical and disinfection/oxidation processes employed in wastewater treatment and their efficacy regarding their removal of antibiotic resistant bacteria and antibiotic resistance geness (ARGs) were evaluated.)]\n",
            "572: [(, , , It also discusses the current state of AMR in the most critical resistant bacteria as determined by the WHO's global priority pathogens list.)]\n",
            "573: [(, , , Antimicrobial resistance (AMR) is a crucial multifactorial and complex global problem and Bangladesh poses a regional and global threat with a high degree of antibiotic resistance.)]\n",
            "574: [(, , , Bangladesh is implementing the National Action Plan (NAP) for containing AMR in human, animal, and environment sectors through \"One Health\" approach where the Department of Livestock Services (DLS) is the mandated body to implement NAP strategies in the animal health sector of the country.)]\n",
            "575: [(, , , Although the routine application of antimicrobials in the livestock industry has largely contributed to the health and productivity, it correspondingly plays a significant role in the evolution of different pathogenic bacterial strains having multidrug resistance (MDR) properties.)]\n",
            "576: [(, , , This review presents a \"snapshot\" of the predisposing factors, and current situations of AMR along with the weakness and strength of DLS to contain the problem in animal farming practices in Bangladesh.)]\n",
            "577: [(, , , In the present review, resistance monitoring data and risk assessment identified several direct and/or indirect predisposing factors to be potentially associated with AMR development in the animal health sector of Bangladesh.)]\n",
            "578: [(, , , MDR bacteria with extreme resistance against antibiotics recommended to use in both animals and humans have been reported and been being a potential public health hazard in Bangladesh.)]\n",
            "579: [(, , , Moreover, various PVL-positive CA-MRSA clones have disseminated in Japan, whereas the USA300 LV/J clone evolved in that country. Our study provides important information regarding MRSA infection control.)]\n",
            "580: [(, , , Execution of extensive AMR surveillance in veterinary practices and awareness-building programs for stakeholders along with the strengthening of the capacity of DLS are recommended for effective containment of AMR emergence and dissemination in the animal health sector of Bangladesh)]\n",
            "581: [(, , , Antibiotic resistance is spreading rapidly around the world and seriously impeding efforts to control microbial infections.)]\n",
            "582: [(, , , This concept pays special attention to the increasing spread and dissemination of antibiotic-resistant bacteria, which are considered one of the most important environment-related human and animal health hazards.)]\n",
            "583: [(, , , Antimicrobial resistance (AMR) is a global crisis for human public health which threatens the effective prevention and control of ever-increasing infectious diseases.)]\n",
            "584: [(, , , Although nucleic acid testing is widely deployed for the detection of antibiotic resistant bacteria, the current techniques-mainly based on polymerase chain reaction (PCR)-are time-consuming and laborious.)]\n",
            "585: [(, , , There is an urgent need to develop new strategies to control bacterial infections and the spread of antimicrobial resistance (AMR).)]\n",
            "586: [(, , , Meanwhile, the pipeline of novel antibiotics development stagnates, which prompts scientists and pharmacists to develop unconventional antimicrobials.)]\n",
            "587: [(, , , Consequently, integrating nurse-density in future AMR-policies could be beneficial.)]\n",
            "588: [(11, 21, Techniques, Engineered CRISPR-Cas systems are reported to effectively kill bacteria or even revert bacterial resistance to antibiotics (resensitizing bacterial cells to antibiotics).)]\n",
            "589: [(56, 62, Techniques, Strategies for combating antimicrobial resistance using CRISPR (i.e., Cas9, Cas12, Cas13, and Cas14) can be of great significance in detecting bacteria and their resistance to antibiotics.)]\n",
            "590: [(70, 74, Techniques, Strategies for combating antimicrobial resistance using CRISPR (i.e., Cas9, Cas12, Cas13, and Cas14) can be of great significance in detecting bacteria and their resistance to antibiotics.)]\n",
            "591: [(76, 81, Techniques, Strategies for combating antimicrobial resistance using CRISPR (i.e., Cas9, Cas12, Cas13, and Cas14) can be of great significance in detecting bacteria and their resistance to antibiotics.)]\n",
            "592: [(83, 88, Techniques, Strategies for combating antimicrobial resistance using CRISPR (i.e., Cas9, Cas12, Cas13, and Cas14) can be of great significance in detecting bacteria and their resistance to antibiotics.)]\n",
            "593: [(94, 99, Techniques, Strategies for combating antimicrobial resistance using CRISPR (i.e., Cas9, Cas12, Cas13, and Cas14) can be of great significance in detecting bacteria and their resistance to antibiotics.)]\n",
            "594: [(11, 21, Antibiotics, As of now, endolysins have been applied to diverse aspects. In this review, we comprehensively introduce the structures and activities of endolysins and summarize the latest application progress of recombinant endolysins in the fields of medical treatment, pathogen diagnosis, food safety, and agriculture.)]\n",
            "595: [(34, 55, Antibiotic-Resistant Bacteria, This review focuses on the AMR of Staphylococcus aureus, a major gram-positive pathogen, which has the ability to acquire resistance to antimicrobials.)]\n",
            "596: [(29, 36, Microbes, As a general classification, E. coli can be distinguished based on the ability to cause infection of the gastrointestinal system (IPEC) or outside of it (ExPEC).)]\n",
            "597: [(105, 135, Others, As a general classification, E. coli can be distinguished based on the ability to cause infection of the gastrointestinal system (IPEC) or outside of it (ExPEC).)]\n",
            "598: [(25, 35, Antibiotic-Resistant Bacteria, The resistance factors of S. aureus are frequently found on mobile elements, including plasmids and transposons.)]\n",
            "599: [(31, 38, Antibiotic-Resistant Bacteria, Among the different pathogens, E. coli are becoming of great importance, and it has been suggested that ExPEC may harbor resistance genes that may be transferred to pathogenic or opportunistic bacteria.)]\n",
            "600: [(104, 109, Antibiotic-Resistant Bacteria, Among the different pathogens, E. coli are becoming of great importance, and it has been suggested that ExPEC may harbor resistance genes that may be transferred to pathogenic or opportunistic bacteria.)]\n",
            "601: [(26, 37, Antibiotics, Furthermore, outbreaks of methicillin-resistant S. aureus (MRSA) in hospitals and communities have led us to study infection controls, including the antiseptic susceptibility evaluation and molecular epidemiology of MRSA.)]\n",
            "602: [(52, 66, Gene/Proteins, Various antiseptic resistance determinants, such as qacA/B and smr, were identified on plasmids and characterized.)]\n",
            "603: [(82, 104, Techniques, We demonstrated that the plasmid-mediated efflux pump QacB variant QacIII confers fluoroquinolone efflux ability to S. aureus.)]\n",
            "604: [(25, 53, Techniques, We demonstrated that the plasmid-mediated efflux pump QacB variant QacIII confers fluoroquinolone efflux ability to S. aureus.)]\n",
            "605: [(116, 125, Antibiotic-Resistant Bacteria, We demonstrated that the plasmid-mediated efflux pump QacB variant QacIII confers fluoroquinolone efflux ability to S. aureus.)]\n",
            "606: [(54, 58, Gene/Proteins, We demonstrated that the plasmid-mediated efflux pump QacB variant QacIII confers fluoroquinolone efflux ability to S. aureus.)]\n",
            "607: [(67, 73, Gene/Proteins, We demonstrated that the plasmid-mediated efflux pump QacB variant QacIII confers fluoroquinolone efflux ability to S. aureus.)]\n",
            "608: [(44, 77, Antibiotic-Resistant Bacteria, Studies on MRSA epidemiology had shown that community-acquired MRSA (CA-MRSA) was disseminated into hospitals and that an increased use of alcohol-based rubs could reduce the incidence of MRSA infections in such institutions.)]\n",
            "609: [(11, 15, Antibiotic-Resistant Bacteria, Studies on MRSA epidemiology had shown that community-acquired MRSA (CA-MRSA) was disseminated into hospitals and that an increased use of alcohol-based rubs could reduce the incidence of MRSA infections in such institutions.)]\n",
            "610: [(188, 192, Antibiotic-Resistant Bacteria, Studies on MRSA epidemiology had shown that community-acquired MRSA (CA-MRSA) was disseminated into hospitals and that an increased use of alcohol-based rubs could reduce the incidence of MRSA infections in such institutions.)]\n",
            "611: [(27, 34, Microbes, Additionally, the study of CA-MRSA collected from communities and hospitals showed an increase in Panton-Valentine leucocidin (PVL)-positive CA-MRSA, causing severe skin and soft tissue infections.)]\n",
            "612: [(98, 148, Antibiotic-Resistant Bacteria, Additionally, the study of CA-MRSA collected from communities and hospitals showed an increase in Panton-Valentine leucocidin (PVL)-positive CA-MRSA, causing severe skin and soft tissue infections.)]\n",
            "613: [(14, 40, Antibiotic-Resistant Bacteria, The advent of pandrug-resistant bacteria makes most, if not all, available antibiotics invalid.)]\n",
            "614: [(4, 14, Techniques, The CRISPR-Cas system is an adaptive immune system found in many prokaryotes that presents attractive opportunities to target and edit nucleic acids with high precision and reliability.)]\n",
            "615: [(, , , This review discusses the structures, mechanisms, and detection methods of CRISPR-Cas systems and how these systems can be engineered for the rapid and reliable detection of bacteria using various approaches, with a particular focus on nanoparticles.)]\n",
            "616: [(, , , Antimicrobial resistance (AMR) is a serious problem worldwide.)]\n",
            "617: [(67, 77, Techniques, In addition, we summarize the most recent advances in applying the CRISPR-Cas system for virulence modulation of bacterial infections and combating antimicrobial resistance.)]\n",
            "618: [(, , , We searched for the AMR determinants of various bacteria isolated from clinical settings and studied their resistance mechanisms and molecular epidemiology.)]\n",
            "619: [(, , , The One Health approach emphasizes the importance of antimicrobial resistance (AMR) as a major concern both in public health and in food animal production systems.)]\n",
            "620: [(130, 139, Antibiotic-Resistant Bacteria, We determined the complete DNA sequence of the tetracycline-resistance plasmid and found that the inducible expression of tetK in S. aureus was regulated by a post-transcriptional attenuation mechanism.)]\n",
            "621: [(122, 126, Antibiotic-Resistant Genes, We determined the complete DNA sequence of the tetracycline-resistance plasmid and found that the inducible expression of tetK in S. aureus was regulated by a post-transcriptional attenuation mechanism.)]\n",
            "622: [(47, 78, Antibiotic-Resistant Genes, We determined the complete DNA sequence of the tetracycline-resistance plasmid and found that the inducible expression of tetK in S. aureus was regulated by a post-transcriptional attenuation mechanism.)]\n",
            "623: [(, , , This review aims to describe pathotype characteristics of ExPEC to increase our knowledge of these bacteria and, consequently, to increase our chances to control them and reduce the risk for AMR, following a One Health approach.)]\n",
            "624: [(, , , Antimicrobial resistance (AMR) has become a global public health issue and antibiotic agents have lagged behind the rise in bacterial resistance.)]\n",
            "625: [(, , , Traditional antimicrobial treatments consist of drugs which target different essential functions in pathogens.)]\n",
            "626: [(, , , Nevertheless, bacteria continue to evolve new mechanisms to evade this drug-mediated killing with surprising speed on the deployment of each new drug and antibiotic worldwide, a phenomenon called antimicrobial resistance (AMR).)]\n",
            "627: [(, , , Nowadays, AMR represents a critical health threat, for which new medical interventions are urgently needed.)]\n",
            "628: [(, , , Using the \"one health\" approach, this review summarizes the phage products used in plant, food, animal, and human health.)]\n",
            "629: [(, , , Thus, FAO provides multi-pronged support for nations to improve antimicrobial stewardship through programs to strengthen governance, increase awareness, develop and enhance AMR surveillance, and implement best practices related to antimicrobial resistance in agrifood systems.)]\n",
            "630: [(, , , In addition, the advantages and disadvantages and future perspectives for the development of phage therapy as an antibiotic alternative to combat AMR are also discussed in this review.)]\n",
            "631: [(, , , The spread of COVID-19 serves as a reminder of the might of microbes in the era of modern medicine.)]\n",
            "632: [(, , , For years, another threat has preoccupied infectious disease experts and public health officials alike: rising antimicrobial resistance (AMR).)]\n",
            "633: [(, , , Antibiotics in dermatology are often used for prolonged courses, with significant potential for microbiome alteration and antibiotic-related adverse effects.)]\n",
            "634: [(, , , A dry antimicrobial pipeline is threatening regression to a preantibiotic era.)]\n",
            "635: [(, , , Resistance is exceeding stewardship efforts as well as the rates of new drug development and approval in the market.)]\n",
            "636: [(27, 43, Microbes, In this review, we present Escherichia coli and Pseudomonas aeruginosa as specific examples of key AMR threats caused by Gram-negative bacteria and we discuss the current status of mAbs and vaccine approaches under development as well as how knowledge on OMV could benefit antigen discovery strategies.)]\n",
            "637: [(121, 143, Microbes, In this review, we present Escherichia coli and Pseudomonas aeruginosa as specific examples of key AMR threats caused by Gram-negative bacteria and we discuss the current status of mAbs and vaccine approaches under development as well as how knowledge on OMV could benefit antigen discovery strategies.)]\n",
            "638: [(48, 70, Antibiotic-Resistant Bacteria, In this review, we present Escherichia coli and Pseudomonas aeruginosa as specific examples of key AMR threats caused by Gram-negative bacteria and we discuss the current status of mAbs and vaccine approaches under development as well as how knowledge on OMV could benefit antigen discovery strategies.)]\n",
            "639: [(12, 27, Antibiotic-Resistant Bacteria, Non-typhoid Salmonella spp. is one of the leading foodborne pathogens which infect humans worldwide and is associated with food and livestock.)]\n",
            "640: [(0, 19, Antibiotic-Resistant Bacteria, Salmonella enterica remains an important foodborne pathogen in all regions of the world, with Typhimurium as one of the most frequent serotypes causing foodborne disease.)]\n",
            "641: [(94, 105, Antibiotic-Resistant Bacteria, Salmonella enterica remains an important foodborne pathogen in all regions of the world, with Typhimurium as one of the most frequent serotypes causing foodborne disease.)]\n",
            "642: [(56, 90, Antibiotic-Resistant Bacteria, In the present review, we highlighted data published on antimicrobial resistant Salmonella species and serovars associated with livestock and food chain animals.)]\n",
            "643: [(231, 241, Microbes, Plasmids, transposons, and insertion sequence elements mediate dissemination of not only AMR genes but also genes for resistance to heavy metals and disinfectants, thus limiting the therapeutic options for treatment and control of Salmonella.)]\n",
            "644: [(66, 74, Antibiotic-Resistant Genes, Genes for resistance to colistin (mcr-1 to mcr-9) and carbapenem (blaVIM-1, blaDNM-1, and blaNDM-5) have been detected from poultry, pig, and human Salmonella isolates, indicating food animal-associated AMR which is a threat to human public health.)]\n",
            "645: [(76, 84, Antibiotic-Resistant Genes, Genes for resistance to colistin (mcr-1 to mcr-9) and carbapenem (blaVIM-1, blaDNM-1, and blaNDM-5) have been detected from poultry, pig, and human Salmonella isolates, indicating food animal-associated AMR which is a threat to human public health.)]\n",
            "646: [(90, 98, Antibiotic-Resistant Genes, Genes for resistance to colistin (mcr-1 to mcr-9) and carbapenem (blaVIM-1, blaDNM-1, and blaNDM-5) have been detected from poultry, pig, and human Salmonella isolates, indicating food animal-associated AMR which is a threat to human public health.)]\n",
            "647: [(24, 32, Antibiotics, Genes for resistance to colistin (mcr-1 to mcr-9) and carbapenem (blaVIM-1, blaDNM-1, and blaNDM-5) have been detected from poultry, pig, and human Salmonella isolates, indicating food animal-associated AMR which is a threat to human public health.)]\n",
            "648: [(54, 64, Antibiotics, Genes for resistance to colistin (mcr-1 to mcr-9) and carbapenem (blaVIM-1, blaDNM-1, and blaNDM-5) have been detected from poultry, pig, and human Salmonella isolates, indicating food animal-associated AMR which is a threat to human public health.)]\n",
            "649: [(52, 58, Microbes, We are searching for a new method to combat AMR and phages are viruses that can effectively fight bacterial infections, which have renewed interest as antibiotic alternatives with their specificity.)]\n",
            "650: [(6, 11, Microbes, Large phage products have been produced in recent years to fight AMR.)]\n",
            "651: [(3, 7, Others, By 2050, it is estimated that the leading cause of death will be through untreatable AMR pathogens.)]\n",
            "652: [(38, 42, Antibiotics, For the discovery of new vaccines and mAbs, the search for effective antigens that are able to raise protective immune responses is a challenging undertaking.)]\n",
            "653: [(, , , The rising trend of antimicrobial resistance (AMR) by foodborne bacteria is a public health concern as these pathogens are easily transmitted to humans through the food chain.)]\n",
            "654: [(, , , While the consequences of resistance may seem far removed from daily clinical practice, awareness of AMR is essential to dermatological care given that dermatologists prescribe more antibiotics per physician than other providers.)]\n",
            "655: [(225, 235, Microbes, Due to the lack of discovery of new antibiotics and the pressure exerted by antimicrobial resistance in the pharmaceutical industry, this review aimed to address the issue of antibiotic use in livestock which leads to AMR in Salmonella.)]\n",
            "656: [(, , , Through this review we hope to contribute to efforts of bringing the crisis of AMR to the forefront of daily dermatological practice.)]\n",
            "657: [(, , , Much attention was given to resistance to carbapenems and colistin which are the last-line antibiotics used in cases of multi drug resistant bacterial infections.)]\n",
            "658: [(46, 56, Antibiotics, The importance of genomic characterization of carbapenem and colistin resistant Salmonella in determining the relationship between human clinical isolates and food animal isolates was also discussed in this review.)]\n",
            "659: [(61, 69, Antibiotics, The importance of genomic characterization of carbapenem and colistin resistant Salmonella in determining the relationship between human clinical isolates and food animal isolates was also discussed in this review.)]\n",
            "660: [(80, 90, Antibiotic-Resistant Bacteria, The importance of genomic characterization of carbapenem and colistin resistant Salmonella in determining the relationship between human clinical isolates and food animal isolates was also discussed in this review.)]\n",
            "661: [(, , , In this context of rapidly increasing multidrug-resistance worldwide and a diminishingly effective antimicrobial arsenal to tackle resistant strains.)]\n",
            "662: [(, , , ExPEC infections are now a serious public health threat worldwide.)]\n",
            "663: [(, , , Genotyping, plasmid characterization, and phylogenetic analysis is important in understanding the epidemiology of livestock-related Salmonella so that measures of preventing foodborne threats to humans can be improved.)]\n",
            "664: [(, , , This article discusses our current knowledge of the global ecology, epidemiology, transmission, bacterial adaptation, and evolution of this emerging Salmonella serotype.)]\n",
            "665: [(, , , Interestingly, heavy metals are known to also select for antimicrobial resistance.)]\n",
            "666: [(, , , This zoonotic organism has historically been developed into a biological weapon. For this Tier 1, Category A select agent, it is important to expand our understanding of its mechanisms of antibiotic resistance (AMR).)]\n",
            "667: [(, , , Antimicrobial resistance (AMR) is a serious threat to global public health.)]\n",
            "668: [(, , , It increases morbidity and mortality, and is associated with high economic costs due to its health care burden.)]\n",
            "669: [(, , , The hospital environment represents in fact a reservoir of potential pathogens, continuously spread by healthcare personnel, visiting persons and hospitalized patients.)]\n",
            "670: [(, , , It is possible to artificially introduce plasmids with AMR markers for cloning and gene expression purposes.)]\n",
            "671: [(, , , In this review, we survey both the experimental research on AMR in Francisella and bioinformatic databases which contain genomic and proteomic data.)]\n",
            "672: [(, , , We explore both the genetic determinants of intrinsic AMR and naturally acquired or engineered antimicrobial resistance as well as phenotypic resistance in Francisella.)]\n",
            "673: [(, , , ASP has been very successful in promoting antimicrobials' appropriate use by implementing evidence-based interventions.)]\n",
            "674: [(31, 43, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "675: [(45, 56, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "676: [(58, 69, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "677: [(71, 86, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "678: [(88, 100, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "679: [(102, 112, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "680: [(114, 124, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "681: [(126, 134, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "682: [(136, 148, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "683: [(154, 170, Antibiotics, Herein we survey resistance to beta-lactams, monobactams, carbapenems, aminoglycosides, tetracycline, polymyxins, macrolides, rifampin, fosmidomycin, and fluoroquinolones.)]\n",
            "684: [(120, 131, Microbes, We discuss newly developed methods of testing antibiotics against Francisella which involve the intracellular nature of Francisella infection and may better reflect the eventual clinical outcomes for new antibiotic compounds.)]\n",
            "685: [(61, 72, Antibiotic-Resistant Bacteria, Understanding the genetically encoded determinants of AMR in Francisella is key to optimizing the treatment of patients and potentially developing new antimicrobials for this dangerous intracellular pathogen.)]\n",
            "686: [(0, 23, Antibiotic-Resistant Bacteria, Acinetobacter baumannii has become increasingly resistant to leading antimicrobial agents since the 1970s.)]\n",
            "687: [(127, 141, Antibiotic-Resistant Bacteria, However, the past two decades have seen a rapid worldwide emergence of a new Salmonella serotype, namely monophasic variant of S. Typhimurium, whose antigenic formula is 1,4,[5],12:i:-. It has become one of the 2-5 most common Salmonella serotypes responsible for animal and human infections in different regions.)]\n",
            "688: [(54, 68, Antibiotic-Resistant Bacteria, The unexpected link to swine has escalated monophasic S. Typhimurium infections to the status of a global public health emergency.)]\n",
            "689: [(0, 5, Antibiotic-Resistant Bacteria, MDRAb gained media attention after being identified in American soldiers returning from Iraq and treated in US military facilities, where it was termed \"Iraqibacter.\")]\n",
            "690: [(31, 60, Techniques, The large-scale application of whole genome sequencing (WGS) in the last 10 years has revealed the phylogenetic associations of the bacterium and its antimicrobial resistance (AMR) genes.)]\n",
            "691: [(21, 43, Microbes, Tularemia, caused by Francisella tularensis, is endemic to the northern hemisphere.)]\n",
            "692: [(127, 138, Antibiotic-Resistant Bacteria, In this review we highlight the potential causes of antimicrobial resistance by heavy metals, with a focus on its emergence in A. baumanni in war zones.)]\n",
            "693: [(0, 11, Microbes, Francisella is unlike many Gram-negative organisms in that it does not have significant plasmid mobility, and does not express AMR mechanisms on plasmids; thus plasmid-mediated resistance does not occur naturally.)]\n",
            "694: [(16, 41, Others, Infections with multidrug-resistant (MDR) bacteria also have substantial implications on clinical and economic outcomes.)]\n",
            "695: [(, , , Moreover, increased indiscriminate use of antibiotics during the COVID-19 pandemic will heighten bacterial resistance and ultimately lead to more deaths.)]\n",
            "696: [(92, 125, Others, This review highlights AMR's scale and consequences, the importance, and implications of an antimicrobial stewardship program (ASP) to fight resistance and protect global health.)]\n",
            "697: [(, , , The \"One Health\" approach, a holistic and multisectoral approach, is also needed to address AMR's rising threat. AMS practices, principles, and interventions are critical steps towards containing and mitigating AMR.)]\n",
            "698: [(, , , Evidence-based policies must guide the \"One Health\" approach, vaccination protocols, health professionals' education, and the public's awareness about AMR.)]\n",
            "699: [(, , , In this study, we aim to synthesize some evidence on the impacts that coronavirus disease 2019 (COVID-19) is having on the epidemiology of antimicrobial resistance (AMR) in Africa since it was declared a global pandemic by the WHO in March 2020.)]\n",
            "700: [(, , , Increased resistance appears linked to armed conflicts, notably since widespread media stories amplified clinical reports in the wake of the American invasion of Iraq in 2003.)]\n",
            "701: [(, , , Antimicrobial resistance is usually assumed to arise through selection pressure exerted by antimicrobial treatment, particularly where treatment is inadequate, as in the case of low dosing, substandard antimicrobial agents, or shortened treatment course.)]\n",
            "702: [(, , , The global epidemic of monophasic S. 1,4,[5],12:i:- has mainly been characterized by an increase in multidrug-resistant S. 1,4,[5],12:i:- isolated in Europe since 1997.)]\n",
            "703: [(56, 97, Antibiotic-Resistant Bacteria, Recently attention has focused on an emerging pathogen, multi-drug resistant A. baumannii (MDRAb).)]\n",
            "704: [(9, 14, Antibiotic-Resistant Bacteria, However, MDRAb is strongly associated in the literature with war injuries that are heavily contaminated by both environmental debris and shrapnel from weapons.)]\n",
            "705: [(, , , Local and global transmission reconstructed by WGS have shown that different clones have emerged following multiple independent events worldwide, and have elucidated the role of this zoonotic pathogen in the spread of AMR.)]\n",
            "706: [(, , , Both may harbor substantial amounts of toxic heavy metals.)]\n",
            "707: [(, , , In fact, Antarctica is often perceived as the last pristine continent on Earth.)]\n",
            "708: [(, , , Antimicrobial resistance (AMR) with the ability to thwart clinical therapies and escalate mortality rates is emerging as one of the most pressing global health and environmental concerns.)]\n",
            "709: [(, , , Such remote regions, which are assumed to have very low levels of AMR due to limited human activity, represent potential model environments to understand the mechanisms and interactions underpinning the early stages of evolution, de novo development, acquisition and transmission of AMR.)]\n",
            "710: [(, , , Urban rivers as an important subsystem of the environment offer galore of ecological services which benefit the city dwellers.)]\n",
            "711: [(, , , Due to the recognition of this crisis, concerted efforts for resource mobilization and global cooperation are needed to tackle it.)]\n",
            "712: [(, , , Antarctica, with its defined zones of human colonisation (centred around scientific research stations) and large populations of migratory birds and animals, also has great potential with regard to mapping and understanding the spread of early-stage zoonotic interactions.)]\n",
            "713: [(, , , However, to date, studies of AMR in Antarctica are limited.)]\n",
            "714: [(, , , The integration of diverse approaches may provide information that is imperative in mitigating the threat of AMR.)]\n",
            "715: [(, , , Here, we survey the current literature focussing on the following: i) Dissection of human-introduced AMR versus naturally occurring AMR, based on the premise that multiple drug resistance and resistance to synthetic antibiotics not yet found in nature are the results of human contamination)]\n",
            "716: [(, , , In addition, urban rivers also act as important vectors for AMR spread.)]\n",
            "717: [(, , , Their study could be considered an important milestone in terms of its thoroughness. If we hope to estimate the disease burden of AMR in a more precise manner, age-stratified patient data is needed in conjunction with a surveillance report.)]\n",
            "718: [(, , , ii) The potential role of endemic wildlife in AMR spread There is clear evidence for greater concentrations of AMR around research stations, and although data show reverse zoonosis of the characteristic human gut bacteria to endemic wildlife, AMR within birds and seals appears to be very low, albeit on limited samplings.)]\n",
            "719: [(, , , This is mainly due to the direct exposure of humans and animals to the heavily contaminated river water and high mobility of organisms (aquatic animals, pathogenic, non-pathogenic bacteria) as well as the genetic elements including ARGs and mobile genetic elements (MGEs) in the river.)]\n",
            "720: [(, , , Furthermore, areas where there is little, to no, human activity still appear to be free from anthropogenically introduced AMR.)]\n",
            "721: [(, , , However, in spite of recent advocacy for comprehensive research programs aimed to investigate the occurrence, extent and major drivers of AMR in urban rivers globally, such studies are missing largely.)]\n",
            "722: [(84, 113, Antibiotic-Resistant Bacteria, In this article, after briefly reviewing the most important and clinically relevant multi-drug-resistant bacteria and their main resistance mechanisms, we describe the emerging antimicrobial options for both MDR Gram-positive cocci and Gram-negative bacilli, including recently marketed agents, molecules just approved or under evaluation and rediscovered older antibiotics that have regained importance due to their antimicrobial spectrum.)]\n",
            "723: [(75, 86, Antibiotics, Specifically, emerging options for Gram-positive cocci we reviewed include ceftaroline, ceftobiprole, tedizolid, dalbavancin, and fosfomycin.)]\n",
            "724: [(88, 100, Antibiotics, Specifically, emerging options for Gram-positive cocci we reviewed include ceftaroline, ceftobiprole, tedizolid, dalbavancin, and fosfomycin.)]\n",
            "725: [(102, 111, Antibiotics, Specifically, emerging options for Gram-positive cocci we reviewed include ceftaroline, ceftobiprole, tedizolid, dalbavancin, and fosfomycin.)]\n",
            "726: [(113, 124, Antibiotics, Specifically, emerging options for Gram-positive cocci we reviewed include ceftaroline, ceftobiprole, tedizolid, dalbavancin, and fosfomycin.)]\n",
            "727: [(130, 140, Antibiotics, Specifically, emerging options for Gram-positive cocci we reviewed include ceftaroline, ceftobiprole, tedizolid, dalbavancin, and fosfomycin.)]\n",
            "728: [(49, 54, Microbes, Specifically, emerging options for Gram-positive cocci we reviewed include ceftaroline, ceftobiprole, tedizolid, dalbavancin, and fosfomycin.)]\n",
            "729: [(45, 52, Microbes, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "730: [(243, 254, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "731: [(227, 237, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "732: [(213, 225, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "733: [(146, 165, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "734: [(167, 186, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "735: [(201, 211, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "736: [(188, 199, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "737: [(123, 144, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "738: [(100, 121, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "739: [(76, 98, Antibiotics, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "740: [(31, 44, Microbes, Emerging treatment options for Gram-negative bacilli we considered comprise ceftolozane-tazobactam, ceftazidime-avibactam, meropenem-vaborbactam, imipenem-relebactam, aztreonam-avibactam, minocycline, fosfomycin, eravacycline, plazomicin, and cefiderocol.)]\n",
            "741: [(6, 10, Others, These ARGs in urban rivers have the potential of being transferred to clinically important pathogens.)]\n",
            "742: [(, , , The development of such standards alongside a centralised reporting system would provide baseline data to feedback directly into wastewater treatment policies for the Antarctic Treaty Area to help preserve this relatively pristine environment.)]\n",
            "743: [(, , , Antimicrobial resistance (AMR) remains one of the top public health issues of global concern.)]\n",
            "744: [(, , , Among the most important strategies for AMR control there is the correct and appropriate use of antibiotics, including those available for the treatment of AMR pathogens.)]\n",
            "745: [(, , , A scoping review was undertaken by collecting and curating relevant resources from peer-reviewed articles and also from the gray literature.)]\n",
            "746: [(, , , Mixed approaches of extracting data (qualitative and quantitative) were employed in synthesizing evidence, as suggested by the Health Evidence Network.)]\n",
            "747: [(, , , A model constructed based on the synthesis of early evidence available on the effects of factors linked to COVID-19 in impacting the evolution of AMR in Africa predicted that, in cumulative terms, those factors favoring the evolution of AMR outpace those disfavoring it by no less than three folds.)]\n",
            "748: [(, , , COVID-19 is likely fueling the evolution of AMR almost unhindered in Africa.)]\n",
            "749: [(, , , An exciting scenario is opening today with the long awaited growing availability of novel molecules for the treatment of AMR bacteria.)]\n",
            "750: [(, , , Knowledge of mechanisms of action and resistance patterns allows physicians to increasingly drive antimicrobial treatment towards a precision medicine approach.)]\n",
            "751: [(, , , Although the rapid spread of antimicrobial resistance (AMR), particularly in relation to clinical settings, is causing concern in many regions of the globe, remote, extreme environments, such as Antarctica, are thought to be relatively free from the negative impact of human activities.)]\n",
            "752: [(, , , Strict adherence to antimicrobial stewardship practices will allow us to preserve the emerging antimicrobials for our future.)]\n",
            "753: [(, , , Although antimicrobials have traditionally been used to treat infections and improve health outcomes, resistance to commonly used antimicrobials has posed a major challenge.)]\n",
            "754: [(, , , We discuss the identification of small molecule chemotypes active against Mtb and subsequent structure-activity relationships (SAR) and MoA deconvolution studies.)]\n",
            "755: [(, , , An estimated 700,000 deaths occur globally every year as a result of infections caused by antimicrobial-resistant pathogens.)]\n",
            "756: [(, , , Antimicrobial resistance (AMR) also contributes directly to the decline in the global economy.)]\n",
            "757: [(, , , We also discuss our efforts on drug repositioning toward addressing DR-TB.)]\n",
            "758: [(, , , In 2019, sub-Saharan Africa (SSA) had the highest mortality rate (23.5 deaths per 100,000) attributable to AMR compared to other regions.)]\n",
            "759: [(, , , Although antibiotics remain a first-line treatment, non-antibiotic therapies such as prophylactic vaccines and therapeutic monoclonal antibodies (mAbs) are increasingly interesting alternatives to limit the spread of such antibiotic resistant microorganisms.)]\n",
            "760: [(, , , \"Drug resistance is an unavoidable consequence of the use of drugs; however, the emergence of multi-drug resistance can be managed by accurate diagnosis and tailor-made regimens.)]\n",
            "761: [(, , , The SSA region is disproportionately affected by AMR, in part owing to the prevailing high levels of poverty, which result in a high burden of infectious diseases, poor regulation of antimicrobial use, and a lack of alternatives to ineffective antimicrobials.)]\n",
            "762: [(309, 329, Techniques, This is followed by a discussion on a chemical series identified by whole-cell cross-screening against Mtb, for which MoA deconvolution studies revealed a pathway that explained the lack of in vivo efficacy in a mouse model of TB and reiterated the importance of selecting an appropriate growth medium during phenotypic screening.)]\n",
            "763: [(68, 94, Techniques, This is followed by a discussion on a chemical series identified by whole-cell cross-screening against Mtb, for which MoA deconvolution studies revealed a pathway that explained the lack of in vivo efficacy in a mouse model of TB and reiterated the importance of selecting an appropriate growth medium during phenotypic screening.)]\n",
            "764: [(, , , The global increase in drug-resistant strains of various bacterial pathogens prompted the World Health Organization (WHO) to develop a priority list of AMR pathogens.)]\n",
            "765: [(, , , This review encompasses the issues of AMR, major drivers and their vital roles in the evolution and spread of ARB with an emphasis on sources and hotspots of diverse ARGs in urban rivers contributing to co-occurrence of ARGs and MGEs.)]\n",
            "766: [(, , , \"Antimicrobial resistance (AMR), is one of the most paramount health perils that has emerged in the 21st century.)]\n",
            "767: [(0, 26, Microbes, Mycobacterium tuberculosis (Mtb), an acid-fast bacillus that causes tuberculosis (TB), merits being one of the highest priority pathogens on this list since drug-resistant TB (DR-TB) accounts for ∼29% of deaths attributable to AMR.)]\n",
            "768: [(68, 80, Microbes, Mycobacterium tuberculosis (Mtb), an acid-fast bacillus that causes tuberculosis (TB), merits being one of the highest priority pathogens on this list since drug-resistant TB (DR-TB) accounts for ∼29% of deaths attributable to AMR.)]\n",
            "769: [(114, 143, Antibiotic-Resistant Bacteria, However, despite the availability of drug regimes, the current cure rate for multi-drug-resistant TB (MDR-TB) and extensively drug-resistant TB (XDR-TB) treatment regimens is 50% and 30%, respectively.)]\n",
            "770: [(77, 109, Antibiotic-Resistant Bacteria, However, despite the availability of drug regimes, the current cure rate for multi-drug-resistant TB (MDR-TB) and extensively drug-resistant TB (XDR-TB) treatment regimens is 50% and 30%, respectively.)]\n",
            "771: [(, , , We highlight our research aimed at identifying novel small molecule leads and associated targets against TB toward contributing to the global TB drug discovery and development pipeline.)]\n",
            "772: [(94, 117, Techniques, Our work mainly involves screening of various small molecule chemical libraries in phenotypic whole-cell based assays to identify hits for medicinal chemistry optimization, with attendant deconvolution of the MoA.)]\n",
            "773: [(141, 146, Others, In the concluding section, we preview some promising future directions and the challenges inherent in advancing the drug pipeline to address DR-TB.)]\n",
            "774: [(, , , A perspective review of current and emerging strategies with potentials of combating AMR in urban river ecosystems including advanced water treatment methodologies and floating islands or constructed wetlands.)]\n",
            "775: [(, , , Antimicrobial resistance (AMR) represents one of the most important human- and animal health-threatening issues worldwide.)]\n",
            "776: [(, , , In recent years, funded collaborative efforts of researchers from academia, not-for-profit virtual R&D organizations and industry have resulted in the continuous growth of the TB drug discovery and development pipeline.)]\n",
            "777: [(, , , Bacterial capability to face antimicrobial compounds is an ancient feature, enabling bacterial survival over time and the dynamic surrounding.)]\n",
            "778: [(62, 73, Antibiotics, This has so far led to the accelerated regulatory approval of bedaquiline and delamanid for the treatment of DR-TB.)]\n",
            "779: [(78, 87, Antibiotics, This has so far led to the accelerated regulatory approval of bedaquiline and delamanid for the treatment of DR-TB.)]\n",
            "780: [(, , , The mechanisms responsible for both innate and acquired AMR are thoroughly investigated.)]\n",
            "781: [(, , , It is to be noted that these regimens are administered over a long duration and have a serious side effect profile. Coupled with poor patient adherence, this has led to further acquisition of drug resistance and treatment failure.)]\n",
            "782: [(, , , Commonly, AMR traits are included in mobilizable genetic elements enabling the homogeneous diffusion of the AMR traits pool between the ecosystems of diverse sectors, such as human medicine, veterinary medicine, and the environment.)]\n",
            "783: [(, , , There is therefore an urgent need to develop new TB drugs with novel mechanism of actions (MoAs) and associated regimens.)]\n",
            "784: [(, , , Thus, a coordinated multisectoral approach, such as One-Health, provides a detailed comprehensive picture of the AMR onset and diffusion.)]\n",
            "785: [(, , , This Account recapitulates drug resistance in TB, existing challenges in addressing DR-TB, new drugs and regimens in development, and potential ways to treat DR-TB.)]\n",
            "786: [(, , , Following a general revision of the molecular mechanisms responsible for both innate and acquired AMR, the present manuscript focuses on reviewing the contribution of veterinary medicine to the overall issue of AMR.)]\n",
            "787: [(, , , The main sources of AMR amenable to veterinary medicine are described, driving the attention towards the indissoluble cross-talk existing between the diverse ecosystems and sectors and their cumulative cooperation to this warning phenomenon.)]\n",
            "788: [(, , , While these problems are becoming bigger, the development of new antimicrobials has remained stagnant, and some essential antimicrobials for treatment may be depleted in the near future.)]\n",
            "789: [(, , , This is a significant problem that may jeopardize the sustainability of healthcare itself.)]\n",
            "790: [(, , , Although some antibiotic combinations are already in clinical use, they remain to be studied in detail.)]\n",
            "791: [(, , , In Japan, the National Action Plan on Antimicrobial Resistance was established in April 2016.)]\n",
            "792: [(, , , Based on the five strategic objectives set out in the World Health Organization's Global Action Plan on Antimicrobial Resistance, and by adding a 6th objective of international cooperation as a Japanese unique goal, the goals, strategies and specific actions of each of these six fields are presented.)]\n",
            "793: [(, , , The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 was first reported in Wuhan, China in December 2019 and is associated with high levels of morbidity and mortality.)]\n",
            "794: [(, , , This Action Plan is significantly characterized by setting numerical targets as a performance index to be achieved by 2020.)]\n",
            "795: [(, , , The global action plan as a strategy for prevention and combating AMR has been adopted by most countries, but fewer countries are able to fully implement country-specific action plans, and several challenges exist in many settings.)]\n",
            "796: [(, , , Various types of bacterial and fungal infections occur in patients with COVID-19 with some resistant to antimicrobials that are associated with significantly worse outcomes and deaths.)]\n",
            "797: [(, , , Antimicrobial resistance (AMR) is of increasing global concern.)]\n",
            "798: [(, , , Besides, antimicrobial-resistant (AMR) co-infections are responsible for clinically significant mortality in past pandemics.)]\n",
            "799: [(, , , Human mobility is one factor that has recently been associated with AMR, though the extent of its impact has not yet been well established due to the limited availability of rigorous data.)]\n",
            "800: [(, , , These metabolic processes can be targeted with the aim of re-sensitizing resistant pathogens to antibiotic treatments.)]\n",
            "801: [(, , , This review examines the existing literature regarding various types of human mobility including short-term travelers, forcibly displaced persons, migrant populations, and their association with global rates of AMR.)]\n",
            "802: [(0, 32, Techniques, Combinational antibiotic therapy is one of the best and easiest approaches to handle the current situation of AMR.)]\n",
            "803: [(112, 133, Techniques, This review focuses on therapeutic options for AMR mechanisms of resistance in bacteria that can be overcome by combinational therapy and testing methods for synergy.)]\n",
            "804: [(, , , These complex interrelated determinants intersect with AMR in current and past pandemics and could amplify the potential of a future antimicrobial resistance pandemic.)]\n",
            "805: [(, , , The emergence of antimicrobial resistance (AMR) in bacterial pathogens represents a global health threat.)]\n",
            "806: [(, , , Therefore, global concerted interventions targeted at all levels of society to reduce the use/misuse of antimicrobials and disrupt these multifaceted, interrelated, and interdependent factors are urgently needed.)]\n",
            "807: [(, , , The metabolic state of bacteria is associated with a range of genetic and phenotypic resistance mechanisms.)]\n",
            "808: [(, , , This paper leverages prior research to describe complex major determinants of antimicrobial resistance and provides fresh insights into possible intervention strategies to tackle antimicrobial resistance including in the current and future pandemics.)]\n",
            "809: [(, , , This review provides an overview of the roles of metabolic processes that are associated with AMR mechanisms, including energy production, cell wall synthesis, cell-cell communication, and bacterial growth.)]\n",
            "810: [(, , , The use of antibiotics has enabled the successful treatment of bacterial infections, saving the lives and improving the health of many patients worldwide.)]\n",
            "811: [(, , , However, the emergence and spread of antimicrobial resistance (AMR) has been highlighted as a global threat by different health organizations, and pathogens resistant to antimicrobials cause substantial morbidity and death.)]\n",
            "812: [(, , , We discuss how state-of-the-art metabolomics approaches can be used for comprehensive analysis of microbial AMR-related metabolism, which may facilitate the discovery of novel drug targets and treatment strategies.)]\n",
            "813: [(, , , As resistance to multiple drugs increases, novel and effective therapies as well as prevention strategies are needed.)]\n",
            "814: [(, , , It is a well-known fact that microorganisms are developing resistance to antimicrobial drugs present in the market that is known as antimicrobial resistance (AMR).)]\n",
            "815: [(, , , In this Review, we discuss evidence that vaccines can have a major role in fighting AMR.)]\n",
            "816: [(, , , This resistance in microbes is a great matter of concern among the scientific fraternity.)]\n",
            "817: [(, , , Vaccines are used prophylactically, decreasing the number of infectious disease cases, and thus antibiotic use and the emergence and spread of AMR.)]\n",
            "818: [(, , , This review article focuses on antibiotics and their respective resistant microbes, factors that cause resistance among microbes, and consequences of AMR at global as well as Indian scenario.)]\n",
            "819: [(, , , This article would be a helpful resource in nutshell for making the ground for discovery of new antibiotics that will be more effective toward microbes.)]\n",
            "820: [(, , , Antimicrobial-resistant bacteria that are spreading all over the world have caused significant problems.)]\n",
            "821: [(, , , Antimicrobial resistance (AMR) has become a major concern for healthcare workers due to the emergence of new variants of resistant markers, especially carbapenemases.)]\n",
            "822: [(, , , The studies covered 30,060 drug-resistant isolates from 26 identified bacterial species.)]\n",
            "823: [(, , , Most were enteric, accounting for 65% of the identified species and 92% of all documented isolates.)]\n",
            "824: [(, , , High-income countries were more likely to be recipient nations for AMR originating from middle- and low-income countries.)]\n",
            "825: [(, , , The most common origin of travellers with resistant bacteria was Asia, covering 36% of the total isolates.)]\n",
            "826: [(, , , Klebsiella pneumoniae is considered an opportunistic pathogen, constituting an ongoing health concern for immunocompromised patients, the elderly, and neonates.)]\n",
            "827: [(, , , Reports on the isolation of K. pneumoniae from other sources are increasing, many of which express multidrug-resistant (MDR) phenotypes.)]\n",
            "828: [(, , , Medical tourism was twice as likely to be associated with multidrug-resistant organisms than general travel.)]\n",
            "829: [(, , , There is increasing evidence that human movement facilitates the global spread of resistant bacteria and antimicrobial resistance (AMR) genes.)]\n",
            "830: [(, , , Repeated use of antibiotics, usually administered in the feed or drinking water, may also result in the selection of resistant bacteria in animal feces, able to transfer their antimicrobial-resistance genes (ARG), residing on mobile elements, to other microorganisms, including human pathogens.)]\n",
            "831: [(, , , International travel is a vehicle for the transmission of antimicrobial resistance globally.)]\n",
            "832: [(, , , In 2016, the World Health Organization highlighted the global burden of 'curable' STIs, estimating 376 million new infections of gonorrhoea, chlamydia, syphilis and trichomoniasis annually, with considerable geographic variation in both the burden of disease and prevalence of resistance.)]\n",
            "833: [(, , , Health systems should identify recent travellers to ensure that adequate precautions are taken.)]\n",
            "834: [(0, 37, Antibiotic-Resistant Bacteria, Pneumococcal antimicrobial resistance (AMR) has been highlighted by the WHO as an important public health concern, with emerging serotypes showing resistance to multiple antibiotics.)]\n",
            "835: [(154, 166, Antibiotic-Resistant Bacteria, Here, we discuss a variety of evolutionary mechanisms at the host, pathogen, and environmental levels that may contribute to changes in the prevalence of pneumococcal AMR in the post-vaccination era.)]\n",
            "836: [(100, 113, Antibiotic-Resistant Bacteria, In this review, the prevalence, reservoirs, AMR, Raman spectroscopy detection, and pathogenicity of K. pneumoniae are summarized and various extraintestinal infection pathways are further narrated to extend our understanding of its adaptation and survival ability in reservoirs, and associated disease risks.)]\n",
            "837: [(49, 77, Techniques, In this review, the prevalence, reservoirs, AMR, Raman spectroscopy detection, and pathogenicity of K. pneumoniae are summarized and various extraintestinal infection pathways are further narrated to extend our understanding of its adaptation and survival ability in reservoirs, and associated disease risks.)]\n",
            "838: [(42, 60, Microbes, Multi-drug and extensively drug resistant gonorrhoea strains have been associated with international spread, particularly in travelers returning from Southeast Asia.)]\n",
            "839: [(0, 9, Microbes, Chlamydia is the most common bacterial STI worldwide.)]\n",
            "840: [(131, 144, Antibiotics, Although in vitro resistance has been reported, surveillance data suggest that clinically significant resistance to macrolides and tetracyclines is rare.)]\n",
            "841: [(116, 126, Antibiotics, Although in vitro resistance has been reported, surveillance data suggest that clinically significant resistance to macrolides and tetracyclines is rare.)]\n",
            "842: [(0, 9, Antibiotics, Macrolide resistance in syphilis is now endemic in much of the world but there is no documented penicillin resistance, which remains first-line therapy.)]\n",
            "843: [(0, 14, Others, Trichomoniasis is the most common non-viral STI worldwide.)]\n",
            "844: [(0, 12, Antibiotics, Beta-lactams and quinolones were the most documented drug-resistant organisms, accounting for 35% and 31% of the overall drug resistance, respectively.)]\n",
            "845: [(17, 27, Antibiotics, Beta-lactams and quinolones were the most documented drug-resistant organisms, accounting for 35% and 31% of the overall drug resistance, respectively.)]\n",
            "846: [(64, 77, Antibiotics, Although clinical failure after treatment occurs, resistance to metronidazole is thought to be uncommon.)]\n",
            "847: [(162, 178, Antibiotics, Mycoplasma genitalium exhibits intrinsic resistance to many antibiotics, and the prevalence of resistance to both first- and second-line regimens (macrolides and fluoroquinolones) is increasing worldwide, with limited alternative therapeutic options.)]\n",
            "848: [(0, 21, Microbes, Mycoplasma genitalium exhibits intrinsic resistance to many antibiotics, and the prevalence of resistance to both first- and second-line regimens (macrolides and fluoroquinolones) is increasing worldwide, with limited alternative therapeutic options.)]\n",
            "849: [(147, 157, Antibiotics, Mycoplasma genitalium exhibits intrinsic resistance to many antibiotics, and the prevalence of resistance to both first- and second-line regimens (macrolides and fluoroquinolones) is increasing worldwide, with limited alternative therapeutic options.)]\n",
            "850: [(, , , Increasing reports are describing detections along the food chain, suggesting the possibility exists that this could be a hitherto unexplored reservoir for this opportunistic bacterial pathogen.)]\n",
            "851: [(, , , Expression of MDR phenotypes elaborated by these bacteria is due to the nature of various plasmids carrying antimicrobial resistance (AMR)-encoding genes, and is a challenge to animal, environmental, and human health alike.)]\n",
            "852: [(126, 136, Antibiotic-Resistant Bacteria, Raman spectroscopy has the potential to provide for the rapid identification and screening of antimicrobial susceptibility of Klebsiella isolates.)]\n",
            "853: [(, , , Moreover, hypervirulent isolates linked with extraintestinal infections express phenotypes that may support their niche adaptation.)]\n",
            "854: [(65, 77, Antibiotic-Resistant Bacteria, The relative importance of these factors may vary by population, pneumococcal lineage, geography, and time, leading to the complex relationship between vaccination, antibiotic use, and AMR.)]\n",
            "855: [(, , , International travel facilitates the spread of drug-resistant infections, including sexually transmitted infections (STIs).)]\n",
            "856: [(, , , We systematically reviewed the literature on the impact of travel on the dissemination of AMR.)]\n",
            "857: [(, , , Travelers' risk of contracting and transmitting drug-resistant STIs depends in part on their geographic exposure.)]\n",
            "858: [(, , , We searched the databases Medline, EMBASE and SCOPUS from database inception until the end of June 2019.)]\n",
            "859: [(, , , In this review, we describe the epidemiology of antimicrobial resistance (AMR) and the management of these four common STIs and Mycoplasma genitalium, an increasingly recognized cause of non-gonococcal urethritis.)]\n",
            "860: [(16, 29, Antibiotic-Resistant Bacteria, Infections with K. pneumoniae can be transmitted through contaminated food or water and can be associated with community-acquired infections or between persons and animals involved in hospital-acquired infections.)]\n",
            "861: [(0, 24, Microbes, Streptococcus pneumoniae is a major cause of pneumonia, meningitis, and septicaemia worldwide.)]\n",
            "862: [(, , , Of the 3052 titles identified, 2253 articles passed the initial screening, of which 238 met the inclusion criteria.)]\n",
            "863: [(, , , Use of sludge in agriculture, improper discard of livestock animals and aquaculture industry are considered AMR contributors in other countries but Indian data regarding these are lacking.)]\n",
            "864: [(, , , These include inconsistencies in ARG reporting units, lack of ARG/ARB monitoring surrogates, lack of a standardised protocol for determining ARG removal via wastewater treatments, and the inability to support appropriate risk assessment.)]\n",
            "865: [(, , , Efforts to combat AMR have been initiated by the Indian health authorities but are still at preliminary stages.)]\n",
            "866: [(, , , This is due to a lack of standard monitoring targets and agreed threshold values, and paucity of information on the ARG-pathogen host relationship and risk management.)]\n",
            "867: [(, , , Keeping in view the challenges unique to India, future directions are proposed.)]\n",
            "868: [(, , , These research gaps need to be addressed and research findings need to be transformed into practical guidance for WWTP operators to enable effective progress towards mitigating the evolution and spread of AMR.)]\n",
            "869: [(, , , The emergence and global epidemic of antimicrobial resistance (AMR) poses a serious threat to global public health in recent years.)]\n",
            "870: [(, , , India has one of the highest rates of resistance to antimicrobial agents used both in humans and food animals.)]\n",
            "871: [(, , , As the Royal Society for Biology (RSB) was forming 10 years ago, antimicrobial resistance (AMR) was being heralded as the next threat with a magnitude on a par with global warming.)]\n",
            "872: [(, , , AMR genes are shared between bacterial pathogens mainly via horizontal gene transfer (HGT) on mobile genetic elements (MGEs), thereby accelerating the spread of antimicrobial resistance (AMR) and increasing the burden of drug resistance.)]\n",
            "873: [(, , , Just a few years later, in 2016, Jim O'Neill's report was published laying out recommendations for tackling drug-resistant infections globally.)]\n",
            "874: [(, , , There is an urgent need to develop new strategies to control bacterial infections and the spread of antimicrobial resistance.)]\n",
            "875: [(, , , Infections of mild or moderate severity caused by community-acquired MRSA can be treated with cotrimoxazole (trimethoprim/sulfamethoxazole), doxycycline or clindamycin when susceptibility results are available, while severe community-acquired or hospital-acquired MRSA infections should be managed with glycopeptides, linezolide or daptomycin.)]\n",
            "876: [(, , , The clustered regularly interspaced short palindromic repeats (CRISPR) and CRISPR-associated proteins (Cas) are an RNA-guided adaptive immune system in prokaryotes that recognizes and defends against invasive genetic elements such as phages and plasmids.)]\n",
            "877: [(98, 108, Techniques, Because of its specifically target and cleave DNA sequences encoding antibiotic resistance genes, CRISPR/Cas system has been developed into a new gene-editing tool for the prevention and control of bacterial drug resistance.)]\n",
            "878: [(0, 10, Techniques, CRISPR-Cas plays a potentially important role in controlling horizontal gene transfer and limiting the spread of antibiotic resistance.)]\n",
            "879: [(73, 91, Techniques, In this review, we will introduce the structure and working mechanism of CRISPR-Cas systems, followed by delivery strategies, and then focus on the relationship between antimicrobial resistance and CRISPR-Cas.)]\n",
            "880: [(198, 208, Techniques, In this review, we will introduce the structure and working mechanism of CRISPR-Cas systems, followed by delivery strategies, and then focus on the relationship between antimicrobial resistance and CRISPR-Cas.)]\n",
            "881: [(, , , As no vaccinations are currently available for these STIs, and pre-exposure prophylaxis is an area of active study with limited data, condom use is critical for prevention.)]\n",
            "882: [(, , , Travel medicine providers should incorporate STI risk reduction counselling, with an emphasis on condom use, into the routine pre-travel consultation.)]\n",
            "883: [(, , , Antimicrobial resistance (AMR) continues to pose a significant public health problem in terms of mortality and economic loss.)]\n",
            "884: [(, , , Moreover, the challenges and prospects of this research field are discussed, thereby providing a reference for the prevention and control of the spread of antibiotic resistance.)]\n",
            "885: [(, , , Health authorities of several countries including India have formulated action plans for its containment.)]\n",
            "886: [(, , , Antimicrobial resistance (AMR) is a growing threat to human and animal health.)]\n",
            "887: [(, , , In this fight against AMR, it is important to realize the contribution by all the following four spheres: humans, animals, food and environment.)]\n",
            "888: [(, , , Progress in molecular biology has revealed new and significant challenges for AMR mitigation given the immense diversity of antibiotic resistance genes (ARGs), the complexity of ARG transfer, and the broad range of omnipresent factors contributing to AMR.)]\n",
            "889: [(, , , This review incorporates all the spheres of One Health concept from the Indian perspective.)]\n",
            "890: [(, , , Municipal, hospital and abattoir wastewater are collected and treated in wastewater treatment plants (WWTPs), where the presence of diverse selection pressures together with a highly concentrated consortium of pathogenic/commensal microbes create favourable conditions for the transfer of ARGs and proliferation of antibiotic resistant bacteria (ARB).)]\n",
            "891: [(, , , The rapid emergence of antibiotic resistant pathogens of clinical and veterinary significance over the past 80 years has re-defined the role of WWTPs as a focal point in the fight against AMR.)]\n",
            "892: [(, , , The environment, especially the water bodies, have also reported the presence of resistant organisms or their genes.)]\n",
            "893: [(, , , By reviewing the occurrence of ARGs in wastewater and sludge and the current technologies used to quantify ARGs and identify ARB, this paper provides a research roadmap to address existing challenges in AMR control via wastewater treatment.)]\n",
            "894: [(, , , Specific socio-economic and cultural factors prevalent in India make the containment of resistance more challenging.)]\n",
            "895: [(, , , Injudicious use of antimicrobials and inadequate treatment of waste waters are important drivers of AMR in India.)]\n",
            "896: [(, , , Data and information provided in this review highlight significant knowledge gaps.)]\n",
            "897: [(, , , To combat the serious threat of rising AMR, it is crucial to develop strategies for robust surveillance, use of new/novel pharmaceuticals, and prevention of antibiotic misuse.)]\n",
            "898: [(, , , Antimicrobial resistance (AMR) plasmids have been recognized as important vectors for efficient spread of AMR phenotypes.)]\n",
            "899: [(, , , Where are we now, and what are the challenges ahead? As a slow burner, how will the impact of AMR compare against the recent rapid devastation of the COVID-19 pandemic, and how can we channel some of the good things that come from it (like the awareness and technique of effective hand hygiene) to help us combat AMR speedily and definitively?)]\n",
            "900: [(, , , The food reservoir includes both food-producing animals and food products, and a huge diversity of AMR plasmids have been reported in this sector.)]\n",
            "901: [(, , , Three phylogroups were identified based on nucleotide differences.)]\n",
            "902: [(, , , The spread of antimicrobial resistance (AMR) is a rapidly growing threat to humankind on both regional and global scales.)]\n",
            "903: [(, , , Based on molecular typing methods and/or whole genome sequencing approaches, certain AMR genes/plasmids combinations were found more frequently in food compared to other settings.)]\n",
            "904: [(, , , However, the food source of a definite AMR plasmid is highly complex to confirm due to cross-sectorial transfers and international spread of AMR plasmids.)]\n",
            "905: [(66, 119, Techniques, Presently, AMR sensing technologies have significantly progressed phenotypic antimicrobial susceptibility testing (AST) and genotypic antimicrobial resistance gene (ARG) detection in human healthcare.)]\n",
            "906: [(123, 169, Antibiotic-Resistant Genes, Presently, AMR sensing technologies have significantly progressed phenotypic antimicrobial susceptibility testing (AST) and genotypic antimicrobial resistance gene (ARG) detection in human healthcare.)]\n",
            "907: [(207, 221, Antibiotics, For risk assessment purposes related to human health, AMR plasmids found in food and bearing genes conferring resistances to critically important antibiotics in human medicine - such as to extended-spectrum cephalosporins, carbapenems or colistin - have been under specific scrutiny these last years.)]\n",
            "908: [(223, 234, Antibiotics, For risk assessment purposes related to human health, AMR plasmids found in food and bearing genes conferring resistances to critically important antibiotics in human medicine - such as to extended-spectrum cephalosporins, carbapenems or colistin - have been under specific scrutiny these last years.)]\n",
            "909: [(238, 246, Antibiotics, For risk assessment purposes related to human health, AMR plasmids found in food and bearing genes conferring resistances to critically important antibiotics in human medicine - such as to extended-spectrum cephalosporins, carbapenems or colistin - have been under specific scrutiny these last years.)]\n",
            "910: [(, , , Those plasmids are often multidrug resistant and their dissemination can be driven by the selective pressure exerted by any of the antibiotics concerned.)]\n",
            "911: [(, , , This systematic review aimed to establish whether antimicrobial resistance (AMR) occurs following prolonged use of antimicrobial hand hygiene (HH) products, and, if so, in what magnitude.)]\n",
            "912: [(48, 62, Microbes, Fish and other seafood can be contaminated with Vibrio species, natural inhabitants of the marine, estuarine, and freshwater environment.)]\n",
            "913: [(72, 96, Antibiotic-Resistant Bacteria, Pathogenic Vibrios of major public health concerns are Vibrio cholerae, Vibrio parahaemolyticus, and Vibrio vulnificus.)]\n",
            "914: [(55, 71, Antibiotic-Resistant Bacteria, Pathogenic Vibrios of major public health concerns are Vibrio cholerae, Vibrio parahaemolyticus, and Vibrio vulnificus.)]\n",
            "915: [(101, 118, Antibiotic-Resistant Bacteria, Pathogenic Vibrios of major public health concerns are Vibrio cholerae, Vibrio parahaemolyticus, and Vibrio vulnificus.)]\n",
            "916: [(40, 65, Antibiotics, Approximately 10 % showed resistance to macrolides (azithromycin), for which genetic determinants are less clear.)]\n",
            "917: [(38, 60, Techniques, Quality assessment was done using the Newcastle-Ottawa Scale (NOS). Of 339 full-text articles assessed for eligibility, only four heterogeneous United States (US) studies conducted in the period between 1986 and 2015 were found eligible, and included.)]\n",
            "918: [(11, 22, Antibiotics, Currently, doxycycline, azithromycin, or ciprofloxacin are commonly used for V. cholerae, and doxycycline or quinolone are administered for V. parahaemolyticus, whereas doxycycline and a third-generation cephalosporin are recommended for V. vulnificus as initial treatment regimen.)]\n",
            "919: [(24, 36, Antibiotics, Currently, doxycycline, azithromycin, or ciprofloxacin are commonly used for V. cholerae, and doxycycline or quinolone are administered for V. parahaemolyticus, whereas doxycycline and a third-generation cephalosporin are recommended for V. vulnificus as initial treatment regimen.)]\n",
            "920: [(40, 54, Antibiotics, Currently, doxycycline, azithromycin, or ciprofloxacin are commonly used for V. cholerae, and doxycycline or quinolone are administered for V. parahaemolyticus, whereas doxycycline and a third-generation cephalosporin are recommended for V. vulnificus as initial treatment regimen.)]\n",
            "921: [(203, 217, Antibiotics, Currently, doxycycline, azithromycin, or ciprofloxacin are commonly used for V. cholerae, and doxycycline or quinolone are administered for V. parahaemolyticus, whereas doxycycline and a third-generation cephalosporin are recommended for V. vulnificus as initial treatment regimen.)]\n",
            "922: [(51, 58, Microbes, The emergence of antimicrobial resistance (AMR) in Vibrios is increasingly common across the globe and a decrease in the effectiveness of commonly available antibiotics poses a global threat to public health.)]\n",
            "923: [(107, 137, Techniques, These mobile genetic elements are highly dynamic and could potentially propagate to other bacteria through horizontal gene transfer (HGT).)]\n",
            "924: [(, , , For effective AMR management, an evolution of innovative sensing technologies is needed for tackling the unique challenges of interconnected AMR across various and different health domains.)]\n",
            "925: [(, , , This review comprehensively discusses the modern state-of-play for innovative commercial and emerging AMR sensing technologies, including sequencing, microfluidic, and miniaturized point-of-need platforms.)]\n",
            "926: [(, , , Also, AMR plasmids carry numerous other genes conferring vital properties to the bacterial cell and are recurrently subjected to evolutionary steps such as hybrid plasmids, making the epidemiology of AMR plasmids in food a moving picture.)]\n",
            "927: [(, , , With a unique view toward the future of One Health, we also provide our perspectives and outlook on the constantly changing landscape of AMR sensing technologies beyond the human health domain.)]\n",
            "928: [(, , , Foodborne illness caused by pathogenic Vibrios is generally associated with the consumption of raw or undercooked seafood.)]\n",
            "929: [(, , , Key bibliographic databases were searched to locate items on HH use and AMR development from database inception to December 2020.)]\n",
            "930: [(, , , Records were screened and full texts of all potentially eligible articles were retrieved and checked for inclusion.)]\n",
            "931: [(, , , The following data from the included studies were abstracted: type of HH product used, including the name of antimicrobial agent, study setting, country, study year, duration of use and development of AMR including the organisms involved.)]\n",
            "932: [(, , , Common symptoms of foodborne Vibrio infection include watery diarrhea, stomach cramping, nausea, vomiting, fever, and chills.)]\n",
            "933: [(, , , Administration of oral or intravenous rehydration salts solution is the mainstay for the management of cholera, and antibiotics are also used to shorten the duration of diarrhea and to limit further transmission of the disease.)]\n",
            "934: [(, , , One hospital-based study showed evidence of AMR following long term use of HH products, two studies conducted in household settings showed no evidence of AMR, and another experimental study showed partial evidence of AMR.)]\n",
            "935: [(, , , The overall certainty of the evidence was moderate.)]\n",
            "936: [(, , , Antimicrobial resistance (AMR) is a crucial and emerging multifactorial \"One Health\" problem involving human and animal health, agriculture, aquaculture, and environment; and posing a potential public health hazard globally.)]\n",
            "937: [(, , , The containment of AMR justifies effective surveillance programs to explicate the magnitude of the problem across the contributing sectors.)]\n",
            "938: [(, , , Antibiotic resistance poses an important threat to global public health and has become a challenge to modern medicine.)]\n",
            "939: [(, , , An AMR surveillance program should have a \"top management\" for fund mobilization, planning, formulating, and multilateral coordinating of the surveillance activities.)]\n",
            "940: [(, , , The occurrence of antibiotic-resistant bacteria in a broad range of foods has led to a growing concern about the impact that food may have as a reservoir of antibiotic resistance genes.)]\n",
            "941: [(, , , The top management should identify competent participating laboratories to form a network comprising a reference laboratory and an adequate number of sentinel laboratories.)]\n",
            "942: [(, , , Considering Minas Frescal Cheese (MFC)-a typical Brazilian white soft cheese-and its economic and cultural values, in this study, medically relevant antimicrobial-resistance genetic markers (AR genes) were screened, and the occurrence of integrons were evaluated in manufactured MFC using culture-independent approaches.)]\n",
            "943: [(, , , The responsibilities of the reference laboratory include the development of standardized test methods for ensuring quality and homogeneity of surveillance activities, providing training to the laboratory personnel, and in-depth AMR characterization.)]\n",
            "944: [(, , , The sentinel laboratories will take the responsibilities of receiving samples, isolation and identification of microbes, and initial AMR characterization.)]\n",
            "945: [(33, 45, Techniques, Several other markers, including efflux pumps and aminoglycosides-resistance were distributed among brands.)]\n",
            "946: [(50, 65, Antibiotics, Several other markers, including efflux pumps and aminoglycosides-resistance were distributed among brands.)]\n",
            "947: [(23, 39, Antibiotic-Resistant Bacteria, From 2007 to 2009, 104 Escherichia coli isolates were obtained from four teachers and 38 children in a child care center.)]\n",
            "948: [(39, 56, Antibiotic-Resistant Bacteria, Of these, those caused by diarrheagenic Escherichia coli (DEC) strains present substantial morbidity and mortality.)]\n",
            "949: [(9, 15, Antibiotics, In 2009, cephem-resistant isolates were detected in children as well as a teacher.)]\n",
            "950: [(13, 32, Techniques, Pulsed-field gel electrophoresis revealed 100% similarity for four isolates in 2007 and one isolate in 2008, and also similarity among seven isolates carrying the virulence gene (CNF1).)]\n",
            "951: [(17, 28, Techniques, Laboratory-based AMR testing and characterization is the key component of an AMR surveillance program.)]\n",
            "952: [(10, 33, Techniques, Through a fingerprinting analysis, the tested MFCs were brand-clustered, indicating reproducibility along the production chain.)]\n",
            "953: [(95, 104, Antibiotics, A common core of resistance markers in all brands evaluated and related antimicrobials such as β-lactams, tetracyclines, quinolones, and sulfonamide was detected.)]\n",
            "954: [(106, 119, Antibiotics, A common core of resistance markers in all brands evaluated and related antimicrobials such as β-lactams, tetracyclines, quinolones, and sulfonamide was detected.)]\n",
            "955: [(121, 131, Antibiotics, A common core of resistance markers in all brands evaluated and related antimicrobials such as β-lactams, tetracyclines, quinolones, and sulfonamide was detected.)]\n",
            "956: [(137, 148, Antibiotics, A common core of resistance markers in all brands evaluated and related antimicrobials such as β-lactams, tetracyclines, quinolones, and sulfonamide was detected.)]\n",
            "957: [(, , , Standard guidelines set by Clinical Laboratory Standards Institute or the European Committee on Antimicrobial Susceptibility Testing, should be followed to bring about conformity and harmonization in the AST procedures.)]\n",
            "958: [(, , , Class 1 and 2 integrons were observed, respectively, in 77% and 97% of the samples.)]\n",
            "959: [(, , , AMR surveillance program in animals is eventually similar to that in human health with the exception is that veterinary antibiotics and veterinary pathogens should be given preference here.)]\n",
            "960: [(, , , The presence of AR genes is of special interest due to their clinical relevance.)]\n",
            "961: [(, , , Hence, the review study was envisaged to look deep into the structure of the AMR surveillance program with significance on laboratory-based AMR testing and characterization methods.)]\n",
            "962: [(, , , Taken together, the data may suggest that the production chain of MFC might contribute to the spread of putative drug-resistant bacteria, which could greatly impact human health.)]\n",
            "963: [(, , , Several reports describe antimicrobial-resistance transfer among children and the community in outbreak situations, but transfer between a child and a care giver has not been examined in child care facilities under normal circumstances.)]\n",
            "964: [(, , , Furthermore, detection of class 1 and class 2 integrons in MFC has led to discussions about resistance gene spread in this traditional cheese, providing evidence of potential horizontal transfer of AR genes to human gut microbiota.)]\n",
            "965: [(, , , We investigated the transfer of antimicrobial-resistance genes, resistant bacteria, or both among healthy children and teachers.)]\n",
            "966: [(, , , Water contamination by pathogenic bacteria is a global public health problem.)]\n",
            "967: [(, , , Contamination of surface water utilized to irrigate food products, or for human consumption, causes outbreaks of foodborne and waterborne disease.)]\n",
            "968: [(, , , Twenty-six cephem-resistant isolates were obtained from children in 2007 and 2008.)]\n",
            "969: [(, , , The aim of this study was, therefore, to investigate the microbiological quality of surface water and the presence of DEC strains in different water bodies.)]\n",
            "970: [(, , , Nalidixic acid-resistant isolates from the same teacher for 3 years showed low similarity (<50%) to each other.)]\n",
            "971: [(, , , A total of 472 water samples were collected from irrigation canal, dam, river, and dike water bodies from January through December 2015 in Sinaloa, a State located in Northwestern Mexico.)]\n",
            "972: [(, , , However, an isolate from a teacher in 2007 and another from a child in 2008 showed high similarity (87%).)]\n",
            "973: [(, , , One thousand two hundred eighty-three patients were included in the surveillance.)]\n",
            "974: [(, , , One hundred forty-seven HAIs were detected with a cumulative incidence of 9.2 per 100 patients 4-year period and an incidence rate of 17.4 per 1000 patient days.)]\n",
            "975: [(, , , Fifty-six out of 1283 patients were affected by at least one episode of ICU-acquired pneumonia, and 72.7% of these were associated with intubation.)]\n",
            "976: [(, , , ICU-acquired bloodstream infections (BSIs) occurred in 4.4% of patients and 89.5% were catheter-related.)]\n",
            "977: [(, , , ICU-acquired urinary tract infections (UTIs) occurred in 1% of patients, with 84.6% of the episodes being associated with the use of an urinary catheter.)]\n",
            "978: [(34, 44, Antibiotics, Among the Gram-negative bacteria, carbapenem resistance was found in 91.6% of Acinetobacter baumannii and 28.5% of Klebsiella pneumoniae isolates.)]\n",
            "979: [(78, 101, Microbes, Among the Gram-negative bacteria, carbapenem resistance was found in 91.6% of Acinetobacter baumannii and 28.5% of Klebsiella pneumoniae isolates.)]\n",
            "980: [(115, 136, Microbes, Among the Gram-negative bacteria, carbapenem resistance was found in 91.6% of Acinetobacter baumannii and 28.5% of Klebsiella pneumoniae isolates.)]\n",
            "981: [(, , , The majority of HAIs in the ICU studied were associated with the use of invasive devices.)]\n",
            "982: [(72, 83, Others, Positive correlation between ARG and BMI, caloric intake, and intake of xenobiotics, was observed for obese individuals.)]\n",
            "983: [(376, 401, Others, A common theory for this behavior is the loss of gravity-driven convection processes in the orbital environment, resulting in both reduction of extracellular nutrient availability and the accumulation of bacterial byproducts near the cell. To further characterize the responses, this study investigated the transcriptomic response of Escherichia coli to both microgravity and antibiotic concentration.)]\n",
            "984: [(116, 126, Antibiotics, E. coli was grown aboard International Space Station in the presence of increasing concentrations of the antibiotic gentamicin with identical ground controls conducted on Earth.)]\n",
            "985: [(, , , This study reinforces the hypothesis that obese individuals may harbour an altered gut microbiota, if compared to eutrophic.)]\n",
            "986: [(27, 37, Antibiotics, Furthermore, the increased xenobiotic intake associated to obesity may play an important role in the antimicrobial resistance phenomenon.)]\n",
            "987: [(67, 91, Microbes, Our studies demonstrated that 47.0% (222/472) of samples contained thermotolerant coliforms above permissive levels whereas E. coli strains were isolated from 43.6% (206/472).)]\n",
            "988: [(124, 131, Microbes, Our studies demonstrated that 47.0% (222/472) of samples contained thermotolerant coliforms above permissive levels whereas E. coli strains were isolated from 43.6% (206/472).)]\n",
            "989: [(12, 19, Microbes, Among these E. coli isolates, DEC strains were identified in 14% (29/206) of samples including in irrigation canal (26/29) and river water (3/29) collected from the northern (83%) and central area (17%). Isolated DEC strains were classified as enteroaggregative E. coli (EAEC) 34.4% (10/29), enteropathogenic E. coli (EPEC) 31.0% (9/29), diffuse adherent E. coli (DAEC) 27.5% (8/29), and enterotoxigenic E. coli (ETEC) 6.8% (2/29).)]\n",
            "990: [(30, 41, Microbes, Among these E. coli isolates, DEC strains were identified in 14% (29/206) of samples including in irrigation canal (26/29) and river water (3/29) collected from the northern (83%) and central area (17%). Isolated DEC strains were classified as enteroaggregative E. coli (EAEC) 34.4% (10/29), enteropathogenic E. coli (EPEC) 31.0% (9/29), diffuse adherent E. coli (DAEC) 27.5% (8/29), and enterotoxigenic E. coli (ETEC) 6.8% (2/29).)]\n",
            "991: [(244, 276, Microbes, Among these E. coli isolates, DEC strains were identified in 14% (29/206) of samples including in irrigation canal (26/29) and river water (3/29) collected from the northern (83%) and central area (17%). Isolated DEC strains were classified as enteroaggregative E. coli (EAEC) 34.4% (10/29), enteropathogenic E. coli (EPEC) 31.0% (9/29), diffuse adherent E. coli (DAEC) 27.5% (8/29), and enterotoxigenic E. coli (ETEC) 6.8% (2/29).)]\n",
            "992: [(292, 323, Microbes, Among these E. coli isolates, DEC strains were identified in 14% (29/206) of samples including in irrigation canal (26/29) and river water (3/29) collected from the northern (83%) and central area (17%). Isolated DEC strains were classified as enteroaggregative E. coli (EAEC) 34.4% (10/29), enteropathogenic E. coli (EPEC) 31.0% (9/29), diffuse adherent E. coli (DAEC) 27.5% (8/29), and enterotoxigenic E. coli (ETEC) 6.8% (2/29).)]\n",
            "993: [(338, 369, Microbes, Among these E. coli isolates, DEC strains were identified in 14% (29/206) of samples including in irrigation canal (26/29) and river water (3/29) collected from the northern (83%) and central area (17%). Isolated DEC strains were classified as enteroaggregative E. coli (EAEC) 34.4% (10/29), enteropathogenic E. coli (EPEC) 31.0% (9/29), diffuse adherent E. coli (DAEC) 27.5% (8/29), and enterotoxigenic E. coli (ETEC) 6.8% (2/29).)]\n",
            "994: [(388, 418, Microbes, Among these E. coli isolates, DEC strains were identified in 14% (29/206) of samples including in irrigation canal (26/29) and river water (3/29) collected from the northern (83%) and central area (17%). Isolated DEC strains were classified as enteroaggregative E. coli (EAEC) 34.4% (10/29), enteropathogenic E. coli (EPEC) 31.0% (9/29), diffuse adherent E. coli (DAEC) 27.5% (8/29), and enterotoxigenic E. coli (ETEC) 6.8% (2/29).)]\n",
            "995: [(112, 123, Techniques, Demographics, intrinsic and extrinsic risk factors, information regarding infection and isolated pathogens with antibiogram results were collected.)]\n",
            "996: [(26, 37, Microbes, Moreover, 90% of isolated DEC strains exhibited resistance to at least one commonly prescribed antibiotic in Mexico whereas 17% were multi-drug resistant.)]\n",
            "997: [(31, 34, Microbes, In conclusion, the presence of DEC strains in surface water represents a potential source for human infection, and thus routine monitoring of DEC in surface water and other indirect affected areas should be considered at northwestern Mexico.)]\n",
            "998: [(142, 145, Microbes, In conclusion, the presence of DEC strains in surface water represents a potential source for human infection, and thus routine monitoring of DEC in surface water and other indirect affected areas should be considered at northwestern Mexico.)]\n",
            "999: [(55, 101, Others, The aim of this study was to compare the occurrence of antimicrobial resistance genetic markers (ARG) in faecal specimens of eutrophic, overweight and obese individuals, and their correlation with xenobiotic intake and gut bacteria density.)]\n",
            "1000: [(265, 276, Others, Methods: This was a cross-sectional case-controlled study including 72 adult participants with no record of intestinal or systemic diseases, or recent use of antimicrobials, grouped as eutrophic, overweight, or obese. Anthropometric profile, eating habits and oral xenobiotics intake were recorded.)]\n",
            "1001: [(93, 134, Techniques, Faecal metagenomic DNA was used to screen for ARG by PCR, and to measure bacterial groups by fluorescence in situ hybridization (FISH). Student's t and Wilcoxon tests were used to compare means and differences in ARG detection (95% confidence intervals).)]\n",
            "1002: [(154, 163, Antibiotics, The pattern of antimicrobial-resistance in the isolates showed, among the Gram-positive bacteria, that 66.6% and 16.6% of Staphylococcus epidermidis were oxacillin and teicoplanin resistant, respectively.)]\n",
            "1003: [(168, 179, Antibiotics, The pattern of antimicrobial-resistance in the isolates showed, among the Gram-positive bacteria, that 66.6% and 16.6% of Staphylococcus epidermidis were oxacillin and teicoplanin resistant, respectively.)]\n",
            "1004: [(122, 148, Antibiotic-Resistant Bacteria, The pattern of antimicrobial-resistance in the isolates showed, among the Gram-positive bacteria, that 66.6% and 16.6% of Staphylococcus epidermidis were oxacillin and teicoplanin resistant, respectively.)]\n",
            "1005: [(, , , Results: Increase in abdominal circumference, waist circumference, hip, waist-hip ratio, BMI, carbohydrate, fibres, and total calorie intakes were different from eutrophic to obese participants.)]\n",
            "1006: [(, , , Habitual use of antihypertensive and anti-inflammatory drugs, antacids, and artificial sweeteners were associated mainly with obesity and overweight.)]\n",
            "1007: [(, , , Since a significant proportion of these HAIs are considered preventable, reinforcement of the evidence-based preventive procedures are needed.)]\n",
            "1008: [(, , , Bacteria grown in space experiments under microgravity conditions have been found to undergo unique physiological responses, ranging from modified cell morphology and growth dynamics to a putative increased tolerance to antibiotics.)]\n",
            "1009: [(, , , Relationships among ARG detection and bacteria densities were also different.)]\n",
            "1010: [(, , , The overweight individuals display a transitional gut microbiota which seems to be between eutrophic and obese.)]\n",
            "1011: [(, , , The study aimed to evaluate the distribution of healthcare-associated infections (HAIs), the incidence rates and device utilization ratio (DUR) of device-associated infections (DAIs), as well as the distribution and patterns of antimicrobial resistance of the responsible pathogens.)]\n",
            "1012: [(, , , Eligible patients who were admitted to an adult Intensive Care Unit (ICU) from May 1, 2013 to December 31, 2016 were included in the surveillance.)]\n",
            "1013: [(, , , Objective NPS MedicineWise aims to ensure that medicines are prescribed and used in a manner consistent with current evidence-based best practice.)]\n",
            "1014: [(, , , However, de novo assemblies from short reads alone are often fractured due to repeat regions and the presence of multiple copies of identical IS elements.)]\n",
            "1015: [(, , , A series of nationwide educational and advertising interventions for general practitioners and consumers were implemented in Australia between 2009 and 2015 with the aim of reducing antibiotic prescriptions for upper respiratory tract infections (URTIs).)]\n",
            "1016: [(, , , Identification of plasmids in clinical isolates can aid in the surveillance of the dissemination of AMR, and comprehensive sequence databases support microbiome and metagenomic studies.)]\n",
            "1017: [(, , , The work described in this paper quantifies the change in antibiotic dispensing following these interventions.)]\n",
            "1018: [(, , , We tested several short-read, hybrid and long-lead assembly pipelines by assembling the type strain B. fragilis CCUG4856T (=ATCC25285=NCTC9343) with Illumina short reads and long reads generated by Oxford Nanopore Technologies (ONT) MinION sequencing.)]\n",
            "1019: [(, , , Methods Antibiotic dispensing data between 2004 and 2015 were obtained from a national claims database.)]\n",
            "1020: [(, , , Hybrid assembly with Unicycler, using quality filtered Illumina reads and Filtlong filtered and Canu-corrected ONT reads, produced the assembly of highest quality.)]\n",
            "1021: [(, , , This approach was then applied to six clinical multidrug-resistant B. fragilis isolates and, with minimal manual finishing of chromosomal assemblies of three isolates, complete, circular assemblies of all isolates were produced.)]\n",
            "1022: [(, , , A Bayesian structural time series model was used to forecast a series of antibiotic dispensing volumes expected to have occurred if the interventions had not taken place.)]\n",
            "1023: [(, , , This study investigated the effects and potential mechanisms of urban PM2.5 on the horizontal transfer of ARGs between opportunistic Escherichia coli (E. coli) strains.)]\n",
            "1024: [(, , , These were compared with the volumes that were actually observed to estimate the intervention effect.)]\n",
            "1025: [(, , , Eleven circular, putative plasmids were identified in the six assemblies, of which only three corresponded to a known cultured Bacteroides plasmid.)]\n",
            "1026: [(, , , Results On average, 126,536 fewer antibiotics were dispensed each month since the intervention programs began in 2009 (95% Bayesian credible interval = 71,580-181,490).)]\n",
            "1027: [(82, 86, Antibiotic-Resistant Genes, This study highlights the effect of PM2.5 on promoting the horizontal transfer of ARGs and elucidates the mechanism of the antimicrobial-resistance risks posed by urban PM2.5.)]\n",
            "1028: [(147, 176, Techniques, Complete IS elements could be identified upstream of AMR genes; however, there was not complete correlation between the absence of IS elements and antimicrobial susceptibility.)]\n",
            "1029: [(, , , Conclusions Continual educational intervention programs that emphasise the judicious use of antibiotics may effectively reduce inappropriate prescribing of antibiotics for the treatment of URTIs at a national level.)]\n",
            "1030: [(, , , Fine particulate matter (PM2.5) and antimicrobial resistance are two major threats to public health worldwide.)]\n",
            "1031: [(, , , Current air pollution studies rely heavily on the assessment of PM2.5 chemistry and toxicity.)]\n",
            "1032: [(, , , However, whether and how PM2.5 affects the proliferation and transfer of antimicrobial resistance genes (ARGs) in various environments has remained unanswered.)]\n",
            "1033: [(, , , Here we show that within 49 h of being cultured, E. coli adapted to grow at higher antibiotic concentrations in space compared to Earth, and demonstrated consistent changes in expression of 63 genes in response to an increase in drug concentration in both environments, including specific responses related to oxidative stress and starvation response.)]\n",
            "1034: [(, , , The results showed that urban PM2.5 samples collected from Xi'an (XA), Shanghai (SH), and Shijiazhuang (SJZ) in China induced location- and concentration-dependent promotion of conjugative transfer frequencies compared to the control group.)]\n",
            "1035: [(, , , Additionally, we find 50 stress-response genes upregulated in response to the microgravity when compared directly to the equivalent concentration in the ground control.)]\n",
            "1036: [(, , , The relevant mechanisms were also explored, including the formation of intracellular reactive oxygen species (ROS) and the subsequent induction of oxidative stress, SOS response, changes in membrane permeability, and alternations in mRNA expression of genes involved in horizontal transfer.)]\n",
            "1037: [(, , , We conclude that the increased antibiotic tolerance in microgravity may be attributed not only to diminished transport processes, but also to a resultant antibiotic cross-resistance response conferred by an overlapping effect of stress response genes.)]\n",
            "1038: [(, , , Our data suggest that direct stresses of nutrient starvation and acid-shock conveyed by the microgravity environment can incidentally upregulate stress response pathways related to antibiotic stress and in doing so contribute to the increased antibiotic stress tolerance observed for bacteria in space experiments.)]\n",
            "1039: [(, , , These findings are of great value in understanding the transmission of antimicrobial resistance in various environments and provide valuable information for re-evaluating air quality assessment practices.)]\n",
            "1040: [(, , , These results provide insights into the ability of bacteria to adapt under extreme stress conditions and potential strategies to prevent antimicrobial-resistance in space and on Earth.)]\n",
            "1041: [(, , , This method allowed for the detection of novel STs, as well as co-colonization, with a predominance of non-PCV13 serotypes in this cohort.)]\n",
            "1042: [(, , , Forty-eight resistance genes, as well as mutations associated with resistance were detected, but the correlation with phenotypic non-susceptibility was lower than expected.)]\n",
            "1043: [(, , , The role of European fish farms in the spread of antimicrobial-resistance in the environment and food chain, as well as possible sources of their contamination by clinically relevant antimicrobial-resistance bacteria is scarcely known.)]\n",
            "1044: [(, , , This study aimed to assess the contribution of Portuguese rural trout farms on dispersion of Enterococcus with antimicrobial-resistance and putative virulence genes in the environment and food chain, as well as to identify farms contamination sources.)]\n",
            "1045: [(, , , Antimicrobial-resistance rates were similar among upstream, within and downstream trout tank samples (P>0.05), positioning water-supplying aquacultures as a source of multidrug-resistant (MDR) strains.)]\n",
            "1046: [(30, 44, Antibiotic-Resistant Bacteria, Nevertheless, predominance of MDR E. faecium in feed, trout tanks and trout comparing to upstream samples, suggests feed as an additional aquaculture contamination source.)]\n",
            "1047: [(19, 29, Antibiotic-Resistant Bacteria, The observation of E. faecium and E. faecalis susceptible to ampicillin and gentamicin by clinical breakpoints but with low-levels of resistance to those antimicrobials by ECOFFs breakpoints is of concern, as they might evolve throughout secondary genetic events to resistance levels with human clinical impact.)]\n",
            "1048: [(34, 45, Antibiotic-Resistant Bacteria, The observation of E. faecium and E. faecalis susceptible to ampicillin and gentamicin by clinical breakpoints but with low-levels of resistance to those antimicrobials by ECOFFs breakpoints is of concern, as they might evolve throughout secondary genetic events to resistance levels with human clinical impact.)]\n",
            "1049: [(61, 71, Antibiotics, The observation of E. faecium and E. faecalis susceptible to ampicillin and gentamicin by clinical breakpoints but with low-levels of resistance to those antimicrobials by ECOFFs breakpoints is of concern, as they might evolve throughout secondary genetic events to resistance levels with human clinical impact.)]\n",
            "1050: [(76, 86, Antibiotics, The observation of E. faecium and E. faecalis susceptible to ampicillin and gentamicin by clinical breakpoints but with low-levels of resistance to those antimicrobials by ECOFFs breakpoints is of concern, as they might evolve throughout secondary genetic events to resistance levels with human clinical impact.)]\n",
            "1051: [(192, 228, Techniques, These were selected on the basis of changes in serotype and antibiogram over time. NP samples underwent short-term enrichment for streptococci prior to total nucleic acid extraction and whole metagenome shotgun sequencing (WMGS).)]\n",
            "1052: [(158, 181, Techniques, These were selected on the basis of changes in serotype and antibiogram over time. NP samples underwent short-term enrichment for streptococci prior to total nucleic acid extraction and whole metagenome shotgun sequencing (WMGS).)]\n",
            "1053: [(83, 85, Microbes, These were selected on the basis of changes in serotype and antibiogram over time. NP samples underwent short-term enrichment for streptococci prior to total nucleic acid extraction and whole metagenome shotgun sequencing (WMGS).)]\n",
            "1054: [(111, 173, Techniques, Pneumococcal contigs were aligned to the Antibiotic Resistance Gene-ANNOTation database of acquired AMR genes. In silico pneumococcal capsular and multilocus sequence typing were performed.)]\n",
            "1055: [(41, 87, Techniques, Pneumococcal contigs were aligned to the Antibiotic Resistance Gene-ANNOTation database of acquired AMR genes. In silico pneumococcal capsular and multilocus sequence typing were performed.)]\n",
            "1056: [(153, 163, Microbes, Macrolides are critically important antimicrobials in human health and used to treat some foodborne bacterial diseases, such as Campylobacter jejuni and Salmonella.)]\n",
            "1057: [(128, 148, Antibiotic-Resistant Bacteria, Macrolides are critically important antimicrobials in human health and used to treat some foodborne bacterial diseases, such as Campylobacter jejuni and Salmonella.)]\n",
            "1058: [(85, 92, Antibiotics, We conducted a systematic review and meta-analysis to evaluate the impact of feeding tylosin to cattle on phenotypic and genotypic resistance in several potential zoonotic enteric bacteria: Enterococcus species, Escherichia coli, Salmonella enterica subspecies enterica, and Campylobacter species.)]\n",
            "1059: [(190, 202, Microbes, We conducted a systematic review and meta-analysis to evaluate the impact of feeding tylosin to cattle on phenotypic and genotypic resistance in several potential zoonotic enteric bacteria: Enterococcus species, Escherichia coli, Salmonella enterica subspecies enterica, and Campylobacter species.)]\n",
            "1060: [(230, 249, Antibiotic-Resistant Bacteria, We conducted a systematic review and meta-analysis to evaluate the impact of feeding tylosin to cattle on phenotypic and genotypic resistance in several potential zoonotic enteric bacteria: Enterococcus species, Escherichia coli, Salmonella enterica subspecies enterica, and Campylobacter species.)]\n",
            "1061: [(274, 288, Antibiotic-Resistant Bacteria, We conducted a systematic review and meta-analysis to evaluate the impact of feeding tylosin to cattle on phenotypic and genotypic resistance in several potential zoonotic enteric bacteria: Enterococcus species, Escherichia coli, Salmonella enterica subspecies enterica, and Campylobacter species.)]\n",
            "1062: [(212, 228, Antibiotic-Resistant Bacteria, We conducted a systematic review and meta-analysis to evaluate the impact of feeding tylosin to cattle on phenotypic and genotypic resistance in several potential zoonotic enteric bacteria: Enterococcus species, Escherichia coli, Salmonella enterica subspecies enterica, and Campylobacter species.)]\n",
            "1063: [(, , , This review was registered with PROSPERO (#CRD42018085949).)]\n",
            "1064: [(33, 45, Microbes, We also assessed the presence of Enterococcus with low-levels of antimicrobial-resistance using epidemiological cut-offs (ECOFFs).)]\n",
            "1065: [(56, 78, Antibiotic-Resistant Bacteria, There remains a significant proportion of deaths due to pneumococcal pneumonia in infants from low- and middle-income countries despite the marginal global declines recorded in the past decade.)]\n",
            "1066: [(, , , As a proof of concept, 196 NP samples were retrieved from a subset of 23 infants to explore strain-level pneumococcal colonization patterns and associated antimicrobial-resistance determinants.)]\n",
            "1067: [(37, 49, Microbes, They included major human and animal Enterococcus lineages, suggesting human and non-aquatic animal origins.)]\n",
            "1068: [(, , , The results highlight the need to define the maximum acceptance level of antimicrobial-resistance genes/bacteria to assess water quality and to monitor antimicrobial-resistance strains on feed, essential requirements to maintain a sustainable aquaculture production.)]\n",
            "1069: [(, , , Reads were assembled and aligned to pneumococcal reference genomes for the extraction of pneumococcal and non-pneumococcal bacterial reads.)]\n",
            "1070: [(0, 7, Antibiotics, Tylosin is a commonly used in-feed antimicrobial and is approved in several countries to reduce the incidence of liver abscesses in beef cattle.)]\n",
            "1071: [(, , , Results: Of the 196 samples sequenced, 174 had corresponding positive cultures for pneumococci, of which, 152 were assigned an in silico serotype. Metagenomic sequencing detected a single pneumococcal serotype in 85% (129/152), and co-colonization in 15% (23/152) of the samples.)]\n",
            "1072: [(, , , Feeding tylosin could select for resistant enteric bacteria in cattle, which could contaminate beef products at slaughter and potentially cause foodborne illness.)]\n",
            "1073: [(, , , Twenty-two different pneumococcal serotypes were identified, with 15B/15C and 16F being the most common non-PCV13 serotypes, while 23F and 19A were the most common PCV13 serotypes.)]\n",
            "1074: [(112, 116, Antibiotic-Resistant Genes, Twenty-six different sequence types (STs), including four novel STs were identified in silico. Mutations in the folA and folP genes, associated with cotrimoxazole resistance, were detected in 89% (87/98) of cotrimoxazole-non-susceptible pneumococci, as well as in the pbp1a and pbp2x genes, in penicillin non-susceptible ST705215B/15C isolates.)]\n",
            "1075: [(121, 125, Antibiotic-Resistant Genes, Twenty-six different sequence types (STs), including four novel STs were identified in silico. Mutations in the folA and folP genes, associated with cotrimoxazole resistance, were detected in 89% (87/98) of cotrimoxazole-non-susceptible pneumococci, as well as in the pbp1a and pbp2x genes, in penicillin non-susceptible ST705215B/15C isolates.)]\n",
            "1076: [(149, 162, Antibiotics, Twenty-six different sequence types (STs), including four novel STs were identified in silico. Mutations in the folA and folP genes, associated with cotrimoxazole resistance, were detected in 89% (87/98) of cotrimoxazole-non-susceptible pneumococci, as well as in the pbp1a and pbp2x genes, in penicillin non-susceptible ST705215B/15C isolates.)]\n",
            "1077: [(207, 248, Antibiotic-Resistant Bacteria, Twenty-six different sequence types (STs), including four novel STs were identified in silico. Mutations in the folA and folP genes, associated with cotrimoxazole resistance, were detected in 89% (87/98) of cotrimoxazole-non-susceptible pneumococci, as well as in the pbp1a and pbp2x genes, in penicillin non-susceptible ST705215B/15C isolates.)]\n",
            "1078: [(268, 273, Antibiotic-Resistant Genes, Twenty-six different sequence types (STs), including four novel STs were identified in silico. Mutations in the folA and folP genes, associated with cotrimoxazole resistance, were detected in 89% (87/98) of cotrimoxazole-non-susceptible pneumococci, as well as in the pbp1a and pbp2x genes, in penicillin non-susceptible ST705215B/15C isolates.)]\n",
            "1079: [(278, 283, Antibiotic-Resistant Genes, Twenty-six different sequence types (STs), including four novel STs were identified in silico. Mutations in the folA and folP genes, associated with cotrimoxazole resistance, were detected in 89% (87/98) of cotrimoxazole-non-susceptible pneumococci, as well as in the pbp1a and pbp2x genes, in penicillin non-susceptible ST705215B/15C isolates.)]\n",
            "1080: [(294, 334, Antibiotic-Resistant Bacteria, Twenty-six different sequence types (STs), including four novel STs were identified in silico. Mutations in the folA and folP genes, associated with cotrimoxazole resistance, were detected in 89% (87/98) of cotrimoxazole-non-susceptible pneumococci, as well as in the pbp1a and pbp2x genes, in penicillin non-susceptible ST705215B/15C isolates.)]\n",
            "1081: [(248, 254, Antibiotics, Monitoring changes in pneumococcal carriage is key to understanding vaccination-induced shifts in the ecology of carriage, patterns of antimicrobial resistance, and impact on health. We longitudinally investigated pneumococcal carriage dynamics in PCV-13 vaccinated infants by collecting nasopharyngeal (NP) samples at 2-weekly intervals from birth through the first year of life from 137 infants.)]\n",
            "1082: [(9, 12, Others, Multiple MDR clones carrying copper tolerance (tcrB/cueO), putative virulence or other genes often associated with clinical strains (e.g. E. faecium with IS16/ptsD/sgrA) were observed, some in distinct samples (e.g. upstream and within trout tanks).)]\n",
            "1083: [(138, 148, Microbes, Multiple MDR clones carrying copper tolerance (tcrB/cueO), putative virulence or other genes often associated with clinical strains (e.g. E. faecium with IS16/ptsD/sgrA) were observed, some in distinct samples (e.g. upstream and within trout tanks).)]\n",
            "1084: [(, , , Conclusions: Metagenomic sequencing of NP samples is a valuable culture-independent technique for a detailed evaluation of the pneumococcal component and resistome of the NP microbiome.)]\n",
            "1085: [(, , , Eleven databases were searched for primary research studies that fed tylosin at approved doses to feedlot cattle and tested bacteria of interest for phenotypic or genotypic resistance.)]\n",
            "1086: [(, , , Samples were collected during the period between 2009 and 2010.)]\n",
            "1087: [(, , , This study suggests that green anoles may play an important role of the infection of S. enterica on this island.)]\n",
            "1088: [(, , , Attention is needed from the aspect of public and ecological health.)]\n",
            "1089: [(, , , Nasopharyngeal (NP) colonization by Streptococcus pneumoniae (pneumococcus) precedes the development of respiratory tract infection.)]\n",
            "1090: [(136, 143, Antibiotics, Semi-quantitative analyses of other bacteria-antimicrobial combinations revealed mixed results, but many comparisons found no effect of tylosin administration.)]\n",
            "1091: [(16, 51, Antibiotic-Resistant Bacteria, Colonization by antimicrobial-resistant pneumococci, especially in infants, is a major public health concern.)]\n",
            "1092: [(64, 75, Microbes, We longitudinally investigated antimicrobial-resistance amongst pneumococci colonizing the nasopharynx of South African infants immunized with the 13-valent pneumococcal conjugate vaccine (PCV13).)]\n",
            "1093: [(147, 187, Antibiotics, We longitudinally investigated antimicrobial-resistance amongst pneumococci colonizing the nasopharynx of South African infants immunized with the 13-valent pneumococcal conjugate vaccine (PCV13).)]\n",
            "1094: [(189, 194, Antibiotics, We longitudinally investigated antimicrobial-resistance amongst pneumococci colonizing the nasopharynx of South African infants immunized with the 13-valent pneumococcal conjugate vaccine (PCV13).)]\n",
            "1095: [(, , , Methods: NP swabs were collected every second week from birth through the first year of life from 137 infants.)]\n",
            "1096: [(0, 11, Microbes, Pneumococci were identified and serotyped using conventional microbiological techniques, and their antibiotic susceptibility profiles determined by disk diffusion and E-test.)]\n",
            "1097: [(61, 87, Techniques, Pneumococci were identified and serotyped using conventional microbiological techniques, and their antibiotic susceptibility profiles determined by disk diffusion and E-test.)]\n",
            "1098: [(148, 162, Techniques, Pneumococci were identified and serotyped using conventional microbiological techniques, and their antibiotic susceptibility profiles determined by disk diffusion and E-test.)]\n",
            "1099: [(167, 173, Techniques, Pneumococci were identified and serotyped using conventional microbiological techniques, and their antibiotic susceptibility profiles determined by disk diffusion and E-test.)]\n",
            "1100: [(44, 92, Techniques, While AMR isolate data are available in the NCBI Pathogen Detection Isolates Browser (NPDIB) database, few researches have been performed to compare antimicrobial resistance detected in environmental and clinical isolates.)]\n",
            "1101: [(52, 57, Antibiotics, Results: All infants were immunized with 3 doses of PCV13. 1520 pneumococci (760 non-repeat) isolates were recovered from 137 infants; including non-typeable (n = 99), PCV13 (n = 133) and non-PCV13 serotypes (n = 528).)]\n",
            "1102: [(64, 75, Microbes, Results: All infants were immunized with 3 doses of PCV13. 1520 pneumococci (760 non-repeat) isolates were recovered from 137 infants; including non-typeable (n = 99), PCV13 (n = 133) and non-PCV13 serotypes (n = 528).)]\n",
            "1103: [(18, 28, Antibiotics, The prevalence of penicillin, erythromycin, and cotrimoxazole non-susceptibility was 19% (95% CI 17-22%) (3% fully resistant), 18% (95% CI 15-21%) (14% fully resistant), and 45% (95% CI 42-49%) (36% fully resistant), respectively.)]\n",
            "1104: [(30, 42, Antibiotics, The prevalence of penicillin, erythromycin, and cotrimoxazole non-susceptibility was 19% (95% CI 17-22%) (3% fully resistant), 18% (95% CI 15-21%) (14% fully resistant), and 45% (95% CI 42-49%) (36% fully resistant), respectively.)]\n",
            "1105: [(48, 61, Antibiotics, The prevalence of penicillin, erythromycin, and cotrimoxazole non-susceptibility was 19% (95% CI 17-22%) (3% fully resistant), 18% (95% CI 15-21%) (14% fully resistant), and 45% (95% CI 42-49%) (36% fully resistant), respectively.)]\n",
            "1106: [(0, 10, Antibiotics, Ampicillin, ceftriaxone, gentamicin, tetracycline, and cefoxitin were the antimicrobials which got the most resistance in both settings.)]\n",
            "1107: [(12, 23, Antibiotics, Ampicillin, ceftriaxone, gentamicin, tetracycline, and cefoxitin were the antimicrobials which got the most resistance in both settings.)]\n",
            "1108: [(25, 35, Antibiotics, Ampicillin, ceftriaxone, gentamicin, tetracycline, and cefoxitin were the antimicrobials which got the most resistance in both settings.)]\n",
            "1109: [(37, 49, Antibiotics, Ampicillin, ceftriaxone, gentamicin, tetracycline, and cefoxitin were the antimicrobials which got the most resistance in both settings.)]\n",
            "1110: [(55, 64, Antibiotics, Ampicillin, ceftriaxone, gentamicin, tetracycline, and cefoxitin were the antimicrobials which got the most resistance in both settings.)]\n",
            "1111: [(22, 32, Antibiotics, Non-susceptibility to penicillin, erythromycin, and cotrimoxazole remained relatively constant through the first year of life (X 2 test for trend: p = 0.184, p = 0.171, and p = 0.572, respectively).)]\n",
            "1112: [(34, 46, Antibiotics, Non-susceptibility to penicillin, erythromycin, and cotrimoxazole remained relatively constant through the first year of life (X 2 test for trend: p = 0.184, p = 0.171, and p = 0.572, respectively).)]\n",
            "1113: [(52, 65, Antibiotics, Non-susceptibility to penicillin, erythromycin, and cotrimoxazole remained relatively constant through the first year of life (X 2 test for trend: p = 0.184, p = 0.171, and p = 0.572, respectively).)]\n",
            "1114: [(87, 99, Microbes, We screened 1,626 citations and identified 13 studies that met the inclusion criteria. Enterococcus species were tested in seven studies, Escherichia coli was isolated in five studies, three studies reported on Salmonella, and two studies reported on Campylobacter species.)]\n",
            "1115: [(138, 154, Microbes, We screened 1,626 citations and identified 13 studies that met the inclusion criteria. Enterococcus species were tested in seven studies, Escherichia coli was isolated in five studies, three studies reported on Salmonella, and two studies reported on Campylobacter species.)]\n",
            "1116: [(251, 264, Microbes, We screened 1,626 citations and identified 13 studies that met the inclusion criteria. Enterococcus species were tested in seven studies, Escherichia coli was isolated in five studies, three studies reported on Salmonella, and two studies reported on Campylobacter species.)]\n",
            "1117: [(211, 221, Microbes, We screened 1,626 citations and identified 13 studies that met the inclusion criteria. Enterococcus species were tested in seven studies, Escherichia coli was isolated in five studies, three studies reported on Salmonella, and two studies reported on Campylobacter species.)]\n",
            "1118: [(34, 70, Techniques, Most studies relied on phenotypic antimicrobial susceptibility testing and seven also reported resistance gene testing.)]\n",
            "1119: [(95, 118, Techniques, Most studies relied on phenotypic antimicrobial susceptibility testing and seven also reported resistance gene testing.)]\n",
            "1120: [(18, 29, Antibiotic-Resistant Bacteria, The resistance of S. enterica of these samples against 12 common antimicrobial agents was also determined.)]\n",
            "1121: [(34, 46, Antibiotics, A random-effects meta-analyses of erythromycin-resistant enterococci from four studies had significant residual heterogeneity.)]\n",
            "1122: [(57, 68, Microbes, A random-effects meta-analyses of erythromycin-resistant enterococci from four studies had significant residual heterogeneity.)]\n",
            "1123: [(30, 49, Others, Salmonella strains, including serovar Oranienburg and Aberdeen, were detected from the large intestines of 30.4% of 79 green anole samples.)]\n",
            "1124: [(54, 62, Others, Salmonella strains, including serovar Oranienburg and Aberdeen, were detected from the large intestines of 30.4% of 79 green anole samples.)]\n",
            "1125: [(0, 10, Microbes, Salmonella strains, including serovar Oranienburg and Aberdeen, were detected from the large intestines of 30.4% of 79 green anole samples.)]\n",
            "1126: [(55, 62, Antibiotics, Only two studies were available for a meta-analysis of tylosin-resistant enterococci. A semi-quantitative analysis demonstrated an increase in macrolide-resistant enterococci after long durations of tylosin administration (>100 days).)]\n",
            "1127: [(73, 84, Microbes, Only two studies were available for a meta-analysis of tylosin-resistant enterococci. A semi-quantitative analysis demonstrated an increase in macrolide-resistant enterococci after long durations of tylosin administration (>100 days).)]\n",
            "1128: [(143, 152, Antibiotics, Only two studies were available for a meta-analysis of tylosin-resistant enterococci. A semi-quantitative analysis demonstrated an increase in macrolide-resistant enterococci after long durations of tylosin administration (>100 days).)]\n",
            "1129: [(163, 174, Microbes, Only two studies were available for a meta-analysis of tylosin-resistant enterococci. A semi-quantitative analysis demonstrated an increase in macrolide-resistant enterococci after long durations of tylosin administration (>100 days).)]\n",
            "1130: [(199, 206, Antibiotics, Only two studies were available for a meta-analysis of tylosin-resistant enterococci. A semi-quantitative analysis demonstrated an increase in macrolide-resistant enterococci after long durations of tylosin administration (>100 days).)]\n",
            "1131: [(37, 52, Antibiotics, And 37.5% of which were resistant to Oxytetracycline.)]\n",
            "1132: [(89, 96, Antibiotics, However, about half of these no-effect comparisons did not record the cumulative days of tylosin administration or the time since the last dose.)]\n",
            "1133: [(88, 97, Antibiotics, When fed at approved dosages for typical durations, tylosin increases the proportion of macrolide-resistant enterococci in the cattle gastrointestinal tract, which could pose a zoonotic risk to human beef consumers.)]\n",
            "1134: [(52, 59, Antibiotics, When fed at approved dosages for typical durations, tylosin increases the proportion of macrolide-resistant enterococci in the cattle gastrointestinal tract, which could pose a zoonotic risk to human beef consumers.)]\n",
            "1135: [(, , , There may also be an impact on other bacteria and other antimicrobial resistances but additional details or data are needed to strengthen these comparisons.)]\n",
            "1136: [(, , , We encourage authors of antimicrobial-resistance studies to follow reporting guidelines and publish details of all comparisons to strengthen future meta-analyses.)]\n",
            "1137: [(, , , Antimicrobial resistance (AMR) has become an urgent public health issue, as pathogens are becoming increasingly resistant to commonly used antimicrobials.)]\n",
            "1138: [(, , , To address this, this work conducted the first multivariate statistical analysis of antimicrobial-resistance pathogens detected in NPDIB clinical and environmental isolates for the US from 2013 to 2018.)]\n",
            "1139: [(, , , The highly occurring AMR genes and pathogens were identified for both clinical and environmental settings, and the historical profiles of those genes and pathogens were then compared for the two settings.)]\n",
            "1140: [(, , , The predominant penicillin-non-susceptible serotypes included 19A, 19F, 15B/15C, 15A, and 21, while susceptible serotypes included 23A, 34, and 17A.)]\n",
            "1141: [(, , , The historical profiles of these genes, pathogens, and antimicrobials indicated that higher occurrence frequencies generally took place earlier in the environmental setting than in the clinical setting.)]\n",
            "1142: [(78, 83, Others, Multidrug-resistance (MDR) was observed in 9% (95% CI 7-11%) of the isolates. PCV13 serotypes were more likely to be non-susceptible, compared to non-PCV13 serotypes, to penicillin (26% vs. 16%, p = 0.007), erythromycin (23% vs. 15%, p = 0.027) and cotrimoxazole (62% vs. 41%, p < 0.001).)]\n",
            "1143: [(170, 180, Antibiotics, Multidrug-resistance (MDR) was observed in 9% (95% CI 7-11%) of the isolates. PCV13 serotypes were more likely to be non-susceptible, compared to non-PCV13 serotypes, to penicillin (26% vs. 16%, p = 0.007), erythromycin (23% vs. 15%, p = 0.027) and cotrimoxazole (62% vs. 41%, p < 0.001).)]\n",
            "1144: [(207, 219, Antibiotics, Multidrug-resistance (MDR) was observed in 9% (95% CI 7-11%) of the isolates. PCV13 serotypes were more likely to be non-susceptible, compared to non-PCV13 serotypes, to penicillin (26% vs. 16%, p = 0.007), erythromycin (23% vs. 15%, p = 0.027) and cotrimoxazole (62% vs. 41%, p < 0.001).)]\n",
            "1145: [(249, 262, Antibiotics, Multidrug-resistance (MDR) was observed in 9% (95% CI 7-11%) of the isolates. PCV13 serotypes were more likely to be non-susceptible, compared to non-PCV13 serotypes, to penicillin (26% vs. 16%, p = 0.007), erythromycin (23% vs. 15%, p = 0.027) and cotrimoxazole (62% vs. 41%, p < 0.001).)]\n",
            "1146: [(, , , We investigated the prevalence of Salmonellaenterica and its antimicrobial resistance from 79 green anoles, the invasive alien species inhabits Haha-jima of the Ogasawara archipelago.)]\n",
            "1147: [(, , , Conclusion: Prevalence of NP carriage with antibiotic-non-susceptible pneumococci was relatively constant throughout the first year of life.)]\n",
            "1148: [(, , , PCV13 serotypes were more commonly non-susceptible to penicillin, erythromycin, and cotrimoxazole.)]\n",
            "1149: [(, , , In 48 (77%) strains analyzed, 107 different plasmids were detected, and only some of them showed similarities with plasmids found previously.)]\n",
            "1150: [(, , , In total, seven different antimicrobial-resistance genes carried by plasmids were identified.)]\n",
            "1151: [(, , , Five of the CoNS staphylococci carried plasmids identical with either those of other CoNS species tested, or a well characterized Staphylococcus aureus strain COL, suggesting plasmid dissemination through horizontal transfer.)]\n",
            "1152: [(, , , To demonstrate the possibility of horizontal transfer, we performed electroporation of four resistance plasmids among Staphylococcus epidermidis, Staphylococcus petrasii, and coagulase-positive S. aureus strains.)]\n",
            "1153: [(, , , This results in negative financial impacts on livelihoods plus risks of food safety and development of antimicrobial resistance.)]\n",
            "1154: [(, , , Plasmids were transferred unchanged, were stably maintained in recipient strains, and expressed resistance genes.)]\n",
            "1155: [(43, 64, Microbes, The first extensively drug resistant (XDR) Neisseria gonorrhoeae strain with high resistance to the extended-spectrum cephalosporin ceftriaxone was identified in 2009 in Japan, but no other strain with this antimicrobial-resistance profile has been reported since.)]\n",
            "1156: [(118, 143, Antibiotics, The first extensively drug resistant (XDR) Neisseria gonorrhoeae strain with high resistance to the extended-spectrum cephalosporin ceftriaxone was identified in 2009 in Japan, but no other strain with this antimicrobial-resistance profile has been reported since.)]\n",
            "1157: [(147, 162, Antibiotics, We analysed the whole-genome sequence data and the antimicrobial-susceptibility testing results of 204 strains sampled in a region where the first XDR ceftriaxone-resistant N. gonorrhoeae was isolated, complemented with 67 additional genomes from other time frames and locations within Japan.)]\n",
            "1158: [(173, 187, Microbes, We analysed the whole-genome sequence data and the antimicrobial-susceptibility testing results of 204 strains sampled in a region where the first XDR ceftriaxone-resistant N. gonorrhoeae was isolated, complemented with 67 additional genomes from other time frames and locations within Japan.)]\n",
            "1159: [(119, 130, Antibiotics, Strains resistant to ceftriaxone were not found, but we discovered a sequence type (ST)7363 sub-lineage susceptible to ceftriaxone and cefixime in which the mosaic penA allele responsible for reduced susceptibility had reverted to a susceptible allele by recombination.)]\n",
            "1160: [(135, 143, Antibiotics, Strains resistant to ceftriaxone were not found, but we discovered a sequence type (ST)7363 sub-lineage susceptible to ceftriaxone and cefixime in which the mosaic penA allele responsible for reduced susceptibility had reverted to a susceptible allele by recombination.)]\n",
            "1161: [(83, 91, Antibiotics, Strains resistant to ceftriaxone were not found, but we discovered a sequence type (ST)7363 sub-lineage susceptible to ceftriaxone and cefixime in which the mosaic penA allele responsible for reduced susceptibility had reverted to a susceptible allele by recombination.)]\n",
            "1162: [(70, 83, Antibiotics, Approximately 85 % of isolates showed resistance to fluoroquinolones (ciprofloxacin) explained by linked amino acid substitutions at positions 91 and 95 of GyrA with 99 % sensitivity and 100 % specificity.)]\n",
            "1163: [(52, 68, Antibiotics, Approximately 85 % of isolates showed resistance to fluoroquinolones (ciprofloxacin) explained by linked amino acid substitutions at positions 91 and 95 of GyrA with 99 % sensitivity and 100 % specificity.)]\n",
            "1164: [(156, 160, Gene/Proteins, Approximately 85 % of isolates showed resistance to fluoroquinolones (ciprofloxacin) explained by linked amino acid substitutions at positions 91 and 95 of GyrA with 99 % sensitivity and 100 % specificity.)]\n",
            "1165: [(9, 19, Antibiotics, Overall, penicillin or erythromycin-non-susceptible pneumococci were carried for a shorter duration than susceptible pneumococci [penicillin (mean days, 18 vs. 21, p = 0.013) and erythromycin (mean days, 18 vs. 21, p = 0.035)].)]\n",
            "1166: [(23, 35, Antibiotics, Overall, penicillin or erythromycin-non-susceptible pneumococci were carried for a shorter duration than susceptible pneumococci [penicillin (mean days, 18 vs. 21, p = 0.013) and erythromycin (mean days, 18 vs. 21, p = 0.035)].)]\n",
            "1167: [(179, 191, Antibiotics, Overall, penicillin or erythromycin-non-susceptible pneumococci were carried for a shorter duration than susceptible pneumococci [penicillin (mean days, 18 vs. 21, p = 0.013) and erythromycin (mean days, 18 vs. 21, p = 0.035)].)]\n",
            "1168: [(0, 39, Microbes, Coagulase-negative staphylococci (CoNS) are an important cause of human and animal diseases.)]\n",
            "1169: [(89, 97, Microbes, Therefore, they are assumed to serve as a reservoir of resistance genes often located on plasmids.)]\n",
            "1170: [(73, 77, Microbes, In this study, we analyzed plasmid content in 62 strains belonging to 10 CoNS species of human and veterinary origin.)]\n",
            "1171: [(0, 10, Antibiotics, Penicillin or erythromycin-non-susceptible pneumococci were carried for a shorter duration than penicillin or erythromycin-susceptible pneumococci.)]\n",
            "1172: [(14, 26, Antibiotics, Penicillin or erythromycin-non-susceptible pneumococci were carried for a shorter duration than penicillin or erythromycin-susceptible pneumococci.)]\n",
            "1173: [(43, 54, Microbes, Penicillin or erythromycin-non-susceptible pneumococci were carried for a shorter duration than penicillin or erythromycin-susceptible pneumococci.)]\n",
            "1174: [(96, 106, Antibiotics, Penicillin or erythromycin-non-susceptible pneumococci were carried for a shorter duration than penicillin or erythromycin-susceptible pneumococci.)]\n",
            "1175: [(110, 122, Antibiotics, Penicillin or erythromycin-non-susceptible pneumococci were carried for a shorter duration than penicillin or erythromycin-susceptible pneumococci.)]\n",
            "1176: [(135, 146, Microbes, Penicillin or erythromycin-non-susceptible pneumococci were carried for a shorter duration than penicillin or erythromycin-susceptible pneumococci.)]\n",
            "1177: [(0, 22, Others, Foot-and-mouth disease (FMD) causes negative impacts on global food security, the livestock trade, national economies, and farming-family livelihoods, particularly in resource-poor developing countries with inadequate biosecurity and low levels of vaccination from inadequate veterinary services.)]\n",
            "1178: [(93, 96, Others, As smallholder farmers have limited understanding of disease-risk management, their focus in FMD outbreaks is on accessing clinically effective therapies.)]\n",
            "1179: [(167, 194, Others, However, most are provided inappropriate traditional treatments and/or topical or parenteral antibiotics, often delivered by paraveterinarians inadequately trained in antimicrobial custodianship.)]\n",
            "1180: [(82, 104, Antibiotics, However, most are provided inappropriate traditional treatments and/or topical or parenteral antibiotics, often delivered by paraveterinarians inadequately trained in antimicrobial custodianship.)]\n",
            "1181: [(, , , Methods: Clinical examinations in an outbreak of suspected FMD in April 2019 in Muang Khay village in Luang Prabang province, Laos confirmed signs and lesions of severe, subacute, ulcerative glossitis and interdigital dermatitis, typical of FMD.)]\n",
            "1182: [(, , , However, surveillance to date has been based on phenotypic methods and sequence typing, not genome sequencing.)]\n",
            "1183: [(, , , Results: Treatment with PRP resulted in immediate improvement in demeanor and locomotion, and no adverse events were observed.)]\n",
            "1184: [(, , , Therefore, little is known about the local population structure at the genomic level, and how resistance determinants and lineages are distributed and evolve.)]\n",
            "1185: [(, , , On follow-up interview, all owners confirmed that their animals were eating within 2 days and lesions had healed within 5 days.)]\n",
            "1186: [(, , , Having experienced the positive clinical impacts of PRP on affected animals, these and surrounding farmers were keen to purchase the PRP for future use.)]\n",
            "1187: [(, , , The veterinary authorities rapidly registered the PRP for FMD therapy in Laos due to the observed efficacy.)]\n",
            "1188: [(, , , Discussion: These findings suggest a potential paradigm shift from treating FMD with expensive antimicrobials, which risks antimicrobial resistance, to a new, less expensive therapeutic approach that reduces animal suffering and may motivate farmers to report disease to access treatment.)]\n",
            "1189: [(, , , Use of the PRP is suggested as an innovation that may improve future FMD management, particularly in developing countries.)]\n",
            "1190: [(, , , Within individual infants carrying the same serotype longitudinally, changes in antibiotic susceptibility were observed over time in 45% (61/137) of infants and these changes were predominantly for penicillin (76%, 79/104).)]\n",
            "1191: [(, , , Treatment of these diseases is complicated by their common antimicrobial resistance, caused by overuse of antibiotics in hospital and veterinary environment.)]\n",
            "1192: [(, , , However, these pathogens gain resistance to the antimicrobials that are commonly used to treat them.)]\n",
            "1193: [(, , , Typically, antimicrobial resistance is caused by mechanisms encoded by multiple antimicrobial-resistance genes.)]\n",
            "1194: [(, , , These are carried through pathogens found in foods such as meats.)]\n",
            "1195: [(129, 173, Techniques, Therefore, we perform the first multivariate statistical analysis of the antimicrobial-resistance gene data provided in the NCBI Pathogen Detection Isolates Browser database, covering six states that are geographically either in close proximity to one another (i.e., Pennsylvania (PA), Maryland (MD), and New York (NY)) or far (i.e., New Mexico (NM), Minnesota (MN), and California (CA)).)]\n",
            "1196: [(, , , It is, thus, important to study the genes that are most related to antimicrobial resistance, the pathogens, and the meats carrying antimicrobial-resistance genes.)]\n",
            "1197: [(, , , This information can be further used to correlate the antimicrobial-resistance genes found in humans for improving human health.)]\n",
            "1198: [(, , , Isolates were collected from ten swine farms and one slaughterhouse.)]\n",
            "1199: [(, , , Hundreds of multidimensional data points were projected onto a two-dimensional space that was specified by the first and second principal components, which were then categorized with a hierarchical clustering approach.)]\n",
            "1200: [(, , , Isolates presented high similarity based on comparison of DNA sequences (minimum of 89.6%), and sequence variation grouped according to serotype.)]\n",
            "1201: [(135, 137, Antibiotic-Resistant Genes, While geographically close states like PA, MD and NY share more similar antimicrobial-resistance genes, geographically far states like NM, MN, and CA also contain most of these common antimicrobial-resistance genes.)]\n",
            "1202: [(139, 141, Antibiotic-Resistant Genes, While geographically close states like PA, MD and NY share more similar antimicrobial-resistance genes, geographically far states like NM, MN, and CA also contain most of these common antimicrobial-resistance genes.)]\n",
            "1203: [(147, 149, Antibiotic-Resistant Genes, While geographically close states like PA, MD and NY share more similar antimicrobial-resistance genes, geographically far states like NM, MN, and CA also contain most of these common antimicrobial-resistance genes.)]\n",
            "1204: [(72, 90, Antibiotic-Resistant Bacteria, This study aimed at characterizing seven novel environmental strains of K. quasipneumoniae using a genomic approach which was misidentified by phenotypic methods in a previous batch of 27 species thought to be K. pneumoniae.)]\n",
            "1205: [(210, 223, Antibiotic-Resistant Bacteria, This study aimed at characterizing seven novel environmental strains of K. quasipneumoniae using a genomic approach which was misidentified by phenotypic methods in a previous batch of 27 species thought to be K. pneumoniae.)]\n",
            "1206: [(30, 35, Gene/Proteins, Six strains were positive for mcr-1. At the time of collection, the sampled farms were adding ciprofloxacin to feed and this may have contributed to the high prevalence of resistance to this antibiotic.)]\n",
            "1207: [(94, 107, Gene/Proteins, Six strains were positive for mcr-1. At the time of collection, the sampled farms were adding ciprofloxacin to feed and this may have contributed to the high prevalence of resistance to this antibiotic.)]\n",
            "1208: [(103, 109, Gene/Proteins, Furthermore, we revealed different evolutionary paths of the two major lineages: single acquisition of penA X in the ST7363-associated lineage, followed by multiple independent acquisitions of the penA X and XXXIV in the ST1901-associated lineage.)]\n",
            "1209: [(197, 203, Gene/Proteins, Furthermore, we revealed different evolutionary paths of the two major lineages: single acquisition of penA X in the ST7363-associated lineage, followed by multiple independent acquisitions of the penA X and XXXIV in the ST1901-associated lineage.)]\n",
            "1210: [(117, 123, Antibiotic-Resistant Bacteria, Furthermore, we revealed different evolutionary paths of the two major lineages: single acquisition of penA X in the ST7363-associated lineage, followed by multiple independent acquisitions of the penA X and XXXIV in the ST1901-associated lineage.)]\n",
            "1211: [(221, 227, Antibiotic-Resistant Bacteria, Furthermore, we revealed different evolutionary paths of the two major lineages: single acquisition of penA X in the ST7363-associated lineage, followed by multiple independent acquisitions of the penA X and XXXIV in the ST1901-associated lineage.)]\n",
            "1212: [(43, 53, Antibiotics, All isolates were resistant to ampicillin, cephalexin, and amoxicillin-clavulanic acid and harbored the fosA, bla OKP types, oqxB, and oqxA genes.)]\n",
            "1213: [(59, 86, Antibiotics, All isolates were resistant to ampicillin, cephalexin, and amoxicillin-clavulanic acid and harbored the fosA, bla OKP types, oqxB, and oqxA genes.)]\n",
            "1214: [(31, 41, Antibiotics, All isolates were resistant to ampicillin, cephalexin, and amoxicillin-clavulanic acid and harbored the fosA, bla OKP types, oqxB, and oqxA genes.)]\n",
            "1215: [(104, 108, Antibiotic-Resistant Genes, All isolates were resistant to ampicillin, cephalexin, and amoxicillin-clavulanic acid and harbored the fosA, bla OKP types, oqxB, and oqxA genes.)]\n",
            "1216: [(110, 123, Antibiotic-Resistant Genes, All isolates were resistant to ampicillin, cephalexin, and amoxicillin-clavulanic acid and harbored the fosA, bla OKP types, oqxB, and oqxA genes.)]\n",
            "1217: [(125, 129, Antibiotic-Resistant Genes, All isolates were resistant to ampicillin, cephalexin, and amoxicillin-clavulanic acid and harbored the fosA, bla OKP types, oqxB, and oqxA genes.)]\n",
            "1218: [(135, 139, Antibiotic-Resistant Genes, All isolates were resistant to ampicillin, cephalexin, and amoxicillin-clavulanic acid and harbored the fosA, bla OKP types, oqxB, and oqxA genes.)]\n",
            "1219: [(65, 74, Gene/Proteins, One isolate additionally harbored a gene cassettes consisting of bla SHV-1, bla OXA-1, aac(6')-Ib-cr, catB genes.)]\n",
            "1220: [(76, 85, Gene/Proteins, One isolate additionally harbored a gene cassettes consisting of bla SHV-1, bla OXA-1, aac(6')-Ib-cr, catB genes.)]\n",
            "1221: [(87, 100, Gene/Proteins, One isolate additionally harbored a gene cassettes consisting of bla SHV-1, bla OXA-1, aac(6')-Ib-cr, catB genes.)]\n",
            "1222: [(102, 106, Gene/Proteins, One isolate additionally harbored a gene cassettes consisting of bla SHV-1, bla OXA-1, aac(6')-Ib-cr, catB genes.)]\n",
            "1223: [(4, 18, Antibiotics, The aminoglycoside-modifying enzyme gene aph(3\")-Ia was bracketed by two insertion elements.)]\n",
            "1224: [(29, 36, Gene/Proteins, Plasmid analyses showed that IncFIBK was the most prevalent plasmid, circulating in six isolates, while one isolate exhibited seven different plasmids.)]\n",
            "1225: [(113, 117, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1226: [(134, 138, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1227: [(140, 144, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1228: [(207, 211, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1229: [(213, 217, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1230: [(219, 223, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1231: [(225, 229, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1232: [(231, 235, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1233: [(241, 245, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1234: [(50, 67, Techniques, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1235: [(69, 87, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1236: [(89, 111, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1237: [(120, 132, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1238: [(147, 171, Gene/Proteins, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1239: [(173, 182, Others, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1240: [(188, 195, Others, The isolates have virulence genes responsible for capsule formation, lipopolysaccharide, iron uptake aerobactin (iutA), salmochelins (iroE, iroN), enterobactin siderophore, adherence, and biofilm formation (mrkA, mrkB, mrkC, mrkD, mrkF, and mrkH).)]\n",
            "1241: [(63, 73, Microbes, Pork products are important sources of foodborne non-typhoidal Salmonella in Brazil where antibiotics are commonly used throughout the pork production process and this has the potential to selectively favor antibiotic-resistant strains.)]\n",
            "1242: [(59, 70, Microbes, We characterized the genotypic and phenotypic diversity of S. enterica isolates (n = 41) that were isolated in Brazil.)]\n",
            "1243: [(71, 102, Microbes, Whole-genome sequencing and in silico serotyping demonstrated that the S. enterica serovar Typhimurium was the most common serotype (n = 17), but eight additional servoars were identified.)]\n",
            "1244: [(, , , Eight multilocus sequence types were identified with ST19 being most common (n = 21). Several plasmids replicons were detected, with Col (RNAI) the most abundant (n = 30), followed by IncR (n = 22), IncI1 (n = 10) and IncA/C2 (n = 10).)]\n",
            "1245: [(, , , One potential reason for this spread of antimicrobial-resistance genes beyond the geographic limitation is that animal meats like chicken and turkey act as the carriers for the nationwide spread of these genes.)]\n",
            "1246: [(97, 109, Antibiotics, Minimum inhibitory concentration assays showed that the principle resistance phenotypes were for streptomycin (90.2%), tetracycline (87.8%), ampicillin (80.5%), chloramphenicol (70.7%) and ciprofloxacin (51.2%).)]\n",
            "1247: [(119, 131, Antibiotics, Minimum inhibitory concentration assays showed that the principle resistance phenotypes were for streptomycin (90.2%), tetracycline (87.8%), ampicillin (80.5%), chloramphenicol (70.7%) and ciprofloxacin (51.2%).)]\n",
            "1248: [(141, 151, Antibiotics, Minimum inhibitory concentration assays showed that the principle resistance phenotypes were for streptomycin (90.2%), tetracycline (87.8%), ampicillin (80.5%), chloramphenicol (70.7%) and ciprofloxacin (51.2%).)]\n",
            "1249: [(161, 176, Antibiotics, Minimum inhibitory concentration assays showed that the principle resistance phenotypes were for streptomycin (90.2%), tetracycline (87.8%), ampicillin (80.5%), chloramphenicol (70.7%) and ciprofloxacin (51.2%).)]\n",
            "1250: [(189, 202, Antibiotics, Minimum inhibitory concentration assays showed that the principle resistance phenotypes were for streptomycin (90.2%), tetracycline (87.8%), ampicillin (80.5%), chloramphenicol (70.7%) and ciprofloxacin (51.2%).)]\n",
            "1251: [(, , , The emergence of bacterial pathogens in environmental hosts represents a major risk to public health.)]\n",
            "1252: [(53, 67, Antibiotics, Only two isolates were resistant to third-generation cephalosporins and no isolates were resistant to two tested carbapenems.)]\n",
            "1253: [(, , , Twenty-six unique antimicrobial-resistance genes were identified with blaTEM-1A and blaTEM-1B likely responsible for most beta-lactam resistance and floR responsible for most chloramphenicol resistance.)]\n",
            "1254: [(, , , Whole-genome sequencing was performed using the Illumina platform, and the generated raw reads were de novo assembled.)]\n",
            "1255: [(, , , Comparative genomic, resistome, virulome, mobilome, and phylogeny were then investigated using dierent bioinformatics tools.)]\n",
            "1256: [(31, 49, Antibiotic-Resistant Bacteria, Six strains were identified as K. quasipneumoniae subsp similipneumoniae and one as K. quasipneumoniae subsp. quasipneumoniae.)]\n",
            "1257: [(56, 72, Antibiotic-Resistant Bacteria, Six strains were identified as K. quasipneumoniae subsp similipneumoniae and one as K. quasipneumoniae subsp. quasipneumoniae.)]\n",
            "1258: [(84, 102, Antibiotic-Resistant Bacteria, Six strains were identified as K. quasipneumoniae subsp similipneumoniae and one as K. quasipneumoniae subsp. quasipneumoniae.)]\n",
            "1259: [(110, 125, Antibiotic-Resistant Bacteria, Six strains were identified as K. quasipneumoniae subsp similipneumoniae and one as K. quasipneumoniae subsp. quasipneumoniae.)]\n",
            "1260: [(, , , Our study provides a detailed picture of the distribution of resistance determinants and disentangles the evolution of the two major lineages spreading worldwide.)]\n",
            "1261: [(, , , Foodborne pathogens cause thousands of illnesses across the US each year.)]\n",
            "1262: [(, , , A total of 108 262 urine E. coli isolates from a public hospital-based laboratory (N = 34 103) and a private community-based laboratory (N = 74 159) in a single health district between 2007-2019 were analysed.)]\n",
            "1263: [(, , , Antimicrobial resistance is a growing global problem that will need a multinational collaborative effort to overcome this serious challenge.)]\n",
            "1264: [(, , , Linear regression was used to identify significance of change in AMR rates in both laboratories independently and detect any significant interaction of each setting in proportional change over the study period.)]\n",
            "1265: [(, , , Similar AMR trends were detected among urinary E. coli isolates in private community-based laboratory and public hospital-based laboratory settings over 12 y. AMR rates were consistently higher in the public hospital-based setting.)]\n",
            "1266: [(, , , Statistical analysis revealed a significant association between risk factors and occurrence of ARGs (p ≤ .05).)]\n",
            "1267: [(, , , Significant risk factors include the last treatment and history of antimicrobial administration, breed of horses, use of horses, type of diet fed for horses, practice management and history of last illness.)]\n",
            "1268: [(, , , All other antibiotics showed a significant increase in AMR rates over time in both settings.)]\n",
            "1269: [(, , , Tetracycline-resistance gene (tetA) was 23 times more likely to be found in the Arabian and local breeds of horses compared to English and warmblood breed.)]\n",
            "1270: [(, , , AMR rates in both the private community-based laboratory and public hospital-based laboratory settings increased over time and were consistently higher in the public hospital-based laboratory setting.)]\n",
            "1271: [(, , , TetA is also 8 times more likely to be found in horses that were fed a natural diet compared to other horses that were fed manufactured/ processed feed.)]\n",
            "1272: [(, , , Since private laboratories handle the vast majority of pathology volumes in community outpatient settings in Australia, interventions incorporating the community-based laboratory setting are critical to addressing AMR in the community.)]\n",
            "1273: [(66, 87, Antibiotic-Resistant Bacteria, Colonization of food-producing animals by antimicrobial-resistant Staphylococcus aureus, especially methicillin-resistant S. aureus (MRSA), has become a serious public health problem worldwide. In the current study, clonal diversities of livestock-associated S. aureus isolates collected from broiler farms, slaughterhouses, and retail chicken meat were examined.)]\n",
            "1274: [(100, 111, Antibiotics, Colonization of food-producing animals by antimicrobial-resistant Staphylococcus aureus, especially methicillin-resistant S. aureus (MRSA), has become a serious public health problem worldwide. In the current study, clonal diversities of livestock-associated S. aureus isolates collected from broiler farms, slaughterhouses, and retail chicken meat were examined.)]\n",
            "1275: [(122, 131, Antibiotic-Resistant Bacteria, Colonization of food-producing animals by antimicrobial-resistant Staphylococcus aureus, especially methicillin-resistant S. aureus (MRSA), has become a serious public health problem worldwide. In the current study, clonal diversities of livestock-associated S. aureus isolates collected from broiler farms, slaughterhouses, and retail chicken meat were examined.)]\n",
            "1276: [(259, 268, Antibiotic-Resistant Bacteria, Colonization of food-producing animals by antimicrobial-resistant Staphylococcus aureus, especially methicillin-resistant S. aureus (MRSA), has become a serious public health problem worldwide. In the current study, clonal diversities of livestock-associated S. aureus isolates collected from broiler farms, slaughterhouses, and retail chicken meat were examined.)]\n",
            "1277: [(15, 19, Antibiotic-Resistant Bacteria, In contrast to MSSA, 2 dominant genetic lineages of MRSA (ST692-SCCmecV with t2249 spa type, and ST188-SCCmecIVa with spa type t189) were found in healthy broilers.)]\n",
            "1278: [(52, 56, Antibiotic-Resistant Bacteria, In contrast to MSSA, 2 dominant genetic lineages of MRSA (ST692-SCCmecV with t2249 spa type, and ST188-SCCmecIVa with spa type t189) were found in healthy broilers.)]\n",
            "1279: [(58, 71, Gene/Proteins, In contrast to MSSA, 2 dominant genetic lineages of MRSA (ST692-SCCmecV with t2249 spa type, and ST188-SCCmecIVa with spa type t189) were found in healthy broilers.)]\n",
            "1280: [(77, 91, Gene/Proteins, In contrast to MSSA, 2 dominant genetic lineages of MRSA (ST692-SCCmecV with t2249 spa type, and ST188-SCCmecIVa with spa type t189) were found in healthy broilers.)]\n",
            "1281: [(97, 112, Gene/Proteins, In contrast to MSSA, 2 dominant genetic lineages of MRSA (ST692-SCCmecV with t2249 spa type, and ST188-SCCmecIVa with spa type t189) were found in healthy broilers.)]\n",
            "1282: [(118, 131, Gene/Proteins, In contrast to MSSA, 2 dominant genetic lineages of MRSA (ST692-SCCmecV with t2249 spa type, and ST188-SCCmecIVa with spa type t189) were found in healthy broilers.)]\n",
            "1283: [(23, 28, Antibiotics, The high prevalence of ST692 and ST188 in healthy broilers is associated with high levels of multiple antimicrobial-resistance phenotypes, particularly fluoroquinolone resistance.)]\n",
            "1284: [(33, 38, Antibiotics, The high prevalence of ST692 and ST188 in healthy broilers is associated with high levels of multiple antimicrobial-resistance phenotypes, particularly fluoroquinolone resistance.)]\n",
            "1285: [(152, 167, Antibiotics, The high prevalence of ST692 and ST188 in healthy broilers is associated with high levels of multiple antimicrobial-resistance phenotypes, particularly fluoroquinolone resistance.)]\n",
            "1286: [(197, 208, Antibiotic-Resistant Bacteria, The high number of multidrug resistant Salmonella and the presence of multiple resistant genes and plasmids emphasize the diversity of Salmonella in the studied pork chain, specially from serotype Typhimurium.)]\n",
            "1287: [(123, 134, Antibiotics, All fluoroquinolone-resistant isolates carried double point mutations in gyrA (S84L) and parC (S80F), regardless of STs or methicillin resistance.)]\n",
            "1288: [(73, 84, Antibiotic-Resistant Genes, All fluoroquinolone-resistant isolates carried double point mutations in gyrA (S84L) and parC (S80F), regardless of STs or methicillin resistance.)]\n",
            "1289: [(89, 100, Antibiotic-Resistant Genes, All fluoroquinolone-resistant isolates carried double point mutations in gyrA (S84L) and parC (S80F), regardless of STs or methicillin resistance.)]\n",
            "1290: [(4, 19, Antibiotics, All fluoroquinolone-resistant isolates carried double point mutations in gyrA (S84L) and parC (S80F), regardless of STs or methicillin resistance.)]\n",
            "1291: [(158, 174, Antibiotic-Resistant Bacteria, The aim of the study is to investigate the potential risk factors associated with the prevalence and distribution of antimicrobial-resistance genes (ARGs) of Escherichia coli isolates obtained from equine fecal samples.)]\n",
            "1292: [(95, 104, Antibiotic-Resistant Bacteria, These results provide important insights into the genetic diversity of antimicrobial-resistant S. aureus strains associated with the chicken meat production chain, including healthy broilers, in Korea.)]\n",
            "1293: [(227, 259, Techniques, In total, 103 E. coli isolates were collected from the 118 horse fecal samples. Genes that are responsible for resistance to β-lactams, tetracyclines, aminoglycosides, and trimethoprim were detected using uniplex and multiplex polymerase chain reaction (PCR).)]\n",
            "1294: [(172, 184, Antibiotics, In total, 103 E. coli isolates were collected from the 118 horse fecal samples. Genes that are responsible for resistance to β-lactams, tetracyclines, aminoglycosides, and trimethoprim were detected using uniplex and multiplex polymerase chain reaction (PCR).)]\n",
            "1295: [(158, 170, Antibiotics, The prevalence of antimicrobial resistance was significantly higher in hospitalized horses compared to non-hospitalized ones (p ≤ .05), particularly against (trimethoprim, doxycycline, oxytetracycline, and amoxicillin-clavulanic acid).)]\n",
            "1296: [(172, 183, Antibiotics, The prevalence of antimicrobial resistance was significantly higher in hospitalized horses compared to non-hospitalized ones (p ≤ .05), particularly against (trimethoprim, doxycycline, oxytetracycline, and amoxicillin-clavulanic acid).)]\n",
            "1297: [(185, 200, Antibiotics, The prevalence of antimicrobial resistance was significantly higher in hospitalized horses compared to non-hospitalized ones (p ≤ .05), particularly against (trimethoprim, doxycycline, oxytetracycline, and amoxicillin-clavulanic acid).)]\n",
            "1298: [(206, 233, Antibiotics, The prevalence of antimicrobial resistance was significantly higher in hospitalized horses compared to non-hospitalized ones (p ≤ .05), particularly against (trimethoprim, doxycycline, oxytetracycline, and amoxicillin-clavulanic acid).)]\n",
            "1299: [(86, 90, Gene/Proteins, The most prevalent antimicrobial-resistant genes were aminoglycoside resistant genes (strA, strB, and aadA) in percentages; 89%, 85%, and 84%, respectively.)]\n",
            "1300: [(92, 96, Gene/Proteins, The most prevalent antimicrobial-resistant genes were aminoglycoside resistant genes (strA, strB, and aadA) in percentages; 89%, 85%, and 84%, respectively.)]\n",
            "1301: [(102, 106, Gene/Proteins, The most prevalent antimicrobial-resistant genes were aminoglycoside resistant genes (strA, strB, and aadA) in percentages; 89%, 85%, and 84%, respectively.)]\n",
            "1302: [(0, 10, Antibiotics, Ampicillin was the only antibiotic for which the E. coli resistance trend did not significantly change over the time period in either laboratory setting.)]\n",
            "1303: [(49, 56, Microbes, Ampicillin was the only antibiotic for which the E. coli resistance trend did not significantly change over the time period in either laboratory setting.)]\n",
            "1304: [(112, 125, Antibiotics, Notably, only the ST188 lineage carried an additional third mutation in gyrB (D494N), correlating with enhanced ciprofloxacin minimum inhibitory concentration values versus the strains with double mutations.)]\n",
            "1305: [(72, 84, Antibiotic-Resistant Genes, Notably, only the ST188 lineage carried an additional third mutation in gyrB (D494N), correlating with enhanced ciprofloxacin minimum inhibitory concentration values versus the strains with double mutations.)]\n",
            "1306: [(18, 23, Antibiotic-Resistant Bacteria, Notably, only the ST188 lineage carried an additional third mutation in gyrB (D494N), correlating with enhanced ciprofloxacin minimum inhibitory concentration values versus the strains with double mutations.)]\n",
            "1307: [(, , , Efforts to monitor and combat antimicrobial resistance (AMR) are typically focused on the hospital-based laboratory setting.)]\n",
            "1308: [(, , , One hundred and eighteen horses from different geographical locations and management systems were enrolled in the study and a questionnaire containing information about each individual horse was designed and filled.)]\n",
            "1309: [(42, 46, Antibiotic-Resistant Bacteria, Fifteen different sequence types (STs) of MSSA strains displaying a relatively high degree of genetic diversity were detected in broiler farms, slaughterhouses, and retail chicken meat.)]\n",
            "1310: [(, , , The enrolled horses belonged to 2 main categories (Hospitalized horses (n = 31), and Non hospitalized horses (n = 87)).)]\n",
            "1311: [(, , , The aim of this study was to longitudinally examine and compare trends in AMR among urine Escherichia coli isolates from a private community-based laboratory and a public hospital-based laboratory in an Australian local health district.)]\n",
            "1312: [(, , , Residual antibiotics in the sewage tank were also measured.)]\n",
            "1313: [(, , , Limited data is available on the prevalence and antimicrobial resistance of extended spectrum (ESBL) and AmpC β-lactamases Escherichia coli in sheep and goats in Jordan.)]\n",
            "1314: [(, , , This study determined the molecular prevalence and antimicrobial-resistance of ESBL and AmpC β-lactamases E. coli in 155 sheep and goat flocks across Jordan by testing 948 milk samples.)]\n",
            "1315: [(, , , By using game theory to choose which protein characteristics to use in our machine learning model, we can predict AMR protein sequences for Gram-negative bacteria with an accuracy ranging from 93% to 99%.)]\n",
            "1316: [(, , , The samples were enriched in MacConkey broth, and then plated on MacConkey agar with cefotaxime.)]\n",
            "1317: [(, , , The bacterial composition of new sewage tanks was comparable with that of old tanks within 1 month after relocation and was instantly affected by excrement.)]\n",
            "1318: [(, , , The bacterial composition of sewage tanks in the old and new buildings, containing rooms where seriously ill patients were treated, was similar.)]\n",
            "1319: [(, , , Selection on CHROMagar ESBL allowed detection of EPOs harboring bla CTX-M and carbapenemase genes in all sewage tanks.)]\n",
            "1320: [(, , , One of the sewage Escherichia coli strain comprising ST393 harboring bla CTX-M-27 corresponded to the clinical isolates based on core genome analysis.)]\n",
            "1321: [(29, 33, Antibiotics, A high percentage (>59 %) of ESBL and AmpC β-lactamases E. coli resisted sulfamethoxazole/trimethoprim, tetracycline and nalidixic acid.)]\n",
            "1322: [(38, 55, Antibiotics, A high percentage (>59 %) of ESBL and AmpC β-lactamases E. coli resisted sulfamethoxazole/trimethoprim, tetracycline and nalidixic acid.)]\n",
            "1323: [(73, 89, Antibiotics, A high percentage (>59 %) of ESBL and AmpC β-lactamases E. coli resisted sulfamethoxazole/trimethoprim, tetracycline and nalidixic acid.)]\n",
            "1324: [(90, 102, Antibiotics, A high percentage (>59 %) of ESBL and AmpC β-lactamases E. coli resisted sulfamethoxazole/trimethoprim, tetracycline and nalidixic acid.)]\n",
            "1325: [(104, 116, Antibiotics, A high percentage (>59 %) of ESBL and AmpC β-lactamases E. coli resisted sulfamethoxazole/trimethoprim, tetracycline and nalidixic acid.)]\n",
            "1326: [(121, 135, Antibiotics, A high percentage (>59 %) of ESBL and AmpC β-lactamases E. coli resisted sulfamethoxazole/trimethoprim, tetracycline and nalidixic acid.)]\n",
            "1327: [(43, 51, Antibiotic-Resistant Genes, About 52.2 % of the isolates harbored both blaCTX-M and blaTEM.)]\n",
            "1328: [(56, 62, Antibiotic-Resistant Genes, About 52.2 % of the isolates harbored both blaCTX-M and blaTEM.)]\n",
            "1329: [(110, 116, Techniques, In order to obtain similar classification results, identity thresholds as low as 53% were required when using BLASTp.)]\n",
            "1330: [(24, 31, Antibiotic-Resistant Bacteria, Antimicrobial-resistant E. coli was isolated from different life cycle stages and ARG carriage quantified.)]\n",
            "1331: [(82, 85, Antibiotic-Resistant Genes, Antimicrobial-resistant E. coli was isolated from different life cycle stages and ARG carriage quantified.)]\n",
            "1332: [(13, 20, Antibiotic-Resistant Bacteria, The ingested E. coli persisted throughout the fly life cycle, and ARG carriage was maintained at a constant level in the housefly microbiota.)]\n",
            "1333: [(144, 151, Antibiotic-Resistant Bacteria, To clarify the transmission of ARB from flies to livestock, 30-day-old chickens were inoculated with maggots containing antimicrobial-resistant E. coli.)]\n",
            "1334: [(93, 100, Antibiotic-Resistant Bacteria, Based on the quantification of bacteria isolated from cecal samples, antimicrobial-resistant E. coli persisted in these chickens for at least 16 days.)]\n",
            "1335: [(77, 88, Gene/Proteins, Furthermore, we performed a comparative genome analysis of extended spectrum β-lactamase (ESBL)-producing organisms (EPOs) from hospital sewage and clinical samples.)]\n",
            "1336: [(89, 122, Gene/Proteins, Furthermore, we performed a comparative genome analysis of extended spectrum β-lactamase (ESBL)-producing organisms (EPOs) from hospital sewage and clinical samples.)]\n",
            "1337: [(79, 102, Techniques, Metagenomic analysis was performed on the hospital sewage samples, followed by whole genome sequencing of EPOs.)]\n",
            "1338: [(0, 20, Techniques, Metagenomic analysis was performed on the hospital sewage samples, followed by whole genome sequencing of EPOs.)]\n",
            "1339: [(15, 20, Antibiotic-Resistant Genes, The presence of ESBL and AmpC genes in the E. coli isolates was determined by PCR and the resistance toward critically important antimicrobials was tested by disc diffusion.)]\n",
            "1340: [(25, 29, Antibiotic-Resistant Genes, The presence of ESBL and AmpC genes in the E. coli isolates was determined by PCR and the resistance toward critically important antimicrobials was tested by disc diffusion.)]\n",
            "1341: [(15, 22, Antibiotic-Resistant Bacteria, In total, 1016 E. coli isolates were isolated from the cefotaxime supplemented MacConkey, 382 isolates harbored ESBL genes and 54 harbored blaCMY.)]\n",
            "1342: [(112, 116, Antibiotic-Resistant Genes, In total, 1016 E. coli isolates were isolated from the cefotaxime supplemented MacConkey, 382 isolates harbored ESBL genes and 54 harbored blaCMY.)]\n",
            "1343: [(139, 145, Antibiotic-Resistant Genes, In total, 1016 E. coli isolates were isolated from the cefotaxime supplemented MacConkey, 382 isolates harbored ESBL genes and 54 harbored blaCMY.)]\n",
            "1344: [(55, 65, Antibiotics, In total, 1016 E. coli isolates were isolated from the cefotaxime supplemented MacConkey, 382 isolates harbored ESBL genes and 54 harbored blaCMY.)]\n",
            "1345: [(18, 26, Gene/Proteins, The prevalence of blaCTX-M, blaTEM, blaCMY and blaSHVE. coli in the milk samples were 33.5 %, 31.7 %, 5.7 %. and 1.1 %, respectively.)]\n",
            "1346: [(47, 60, Gene/Proteins, The prevalence of blaCTX-M, blaTEM, blaCMY and blaSHVE. coli in the milk samples were 33.5 %, 31.7 %, 5.7 %. and 1.1 %, respectively.)]\n",
            "1347: [(28, 34, Gene/Proteins, The prevalence of blaCTX-M, blaTEM, blaCMY and blaSHVE. coli in the milk samples were 33.5 %, 31.7 %, 5.7 %. and 1.1 %, respectively.)]\n",
            "1348: [(36, 42, Gene/Proteins, The prevalence of blaCTX-M, blaTEM, blaCMY and blaSHVE. coli in the milk samples were 33.5 %, 31.7 %, 5.7 %. and 1.1 %, respectively.)]\n",
            "1349: [(24, 36, Antibiotics, Moreover, the levels of levofloxacin and clarithromycin in the hospital sewage were 0.0325 and 0.0135 µg/mL, respectively.)]\n",
            "1350: [(41, 55, Antibiotics, Moreover, the levels of levofloxacin and clarithromycin in the hospital sewage were 0.0325 and 0.0135 µg/mL, respectively.)]\n",
            "1351: [(38, 49, Gene/Proteins, At the flock level, 30.3 % flocks had β-lactamase E. coli, specifically 25.2 %, 20.7 %, 5.2 % and 2.6 % had blaCTX-M, blaTEM, blaCMY and blaSHV E. coli, respectively.)]\n",
            "1352: [(50, 57, Microbes, At the flock level, 30.3 % flocks had β-lactamase E. coli, specifically 25.2 %, 20.7 %, 5.2 % and 2.6 % had blaCTX-M, blaTEM, blaCMY and blaSHV E. coli, respectively.)]\n",
            "1353: [(108, 116, Gene/Proteins, At the flock level, 30.3 % flocks had β-lactamase E. coli, specifically 25.2 %, 20.7 %, 5.2 % and 2.6 % had blaCTX-M, blaTEM, blaCMY and blaSHV E. coli, respectively.)]\n",
            "1354: [(118, 124, Gene/Proteins, At the flock level, 30.3 % flocks had β-lactamase E. coli, specifically 25.2 %, 20.7 %, 5.2 % and 2.6 % had blaCTX-M, blaTEM, blaCMY and blaSHV E. coli, respectively.)]\n",
            "1355: [(126, 132, Gene/Proteins, At the flock level, 30.3 % flocks had β-lactamase E. coli, specifically 25.2 %, 20.7 %, 5.2 % and 2.6 % had blaCTX-M, blaTEM, blaCMY and blaSHV E. coli, respectively.)]\n",
            "1356: [(137, 143, Gene/Proteins, At the flock level, 30.3 % flocks had β-lactamase E. coli, specifically 25.2 %, 20.7 %, 5.2 % and 2.6 % had blaCTX-M, blaTEM, blaCMY and blaSHV E. coli, respectively.)]\n",
            "1357: [(144, 151, Microbes, At the flock level, 30.3 % flocks had β-lactamase E. coli, specifically 25.2 %, 20.7 %, 5.2 % and 2.6 % had blaCTX-M, blaTEM, blaCMY and blaSHV E. coli, respectively.)]\n",
            "1358: [(0, 23, Microbes, Campylobacter enteritis is the leading cause of gastroenteritis in humans worldwide including Bangladesh.)]\n",
            "1359: [(, , , With the availability of whole genome sequences, best-hit methods can be used to identify AMR genes by differentiating unknown sequences with known AMR sequences in existing online repositories.)]\n",
            "1360: [(, , , The bacterial flora in the sewage tank was rapidly affected, especially by the ward with seriously ill patients.)]\n",
            "1361: [(, , , Nevertheless, these methods may not perform well when identifying resistance genes with sequences having low sequence identity with known sequences.)]\n",
            "1362: [(, , , AMR monitoring of hospital sewage may help detect carriers prior to nosocomial ARB-associated outbreaks and control the outbreaks.)]\n",
            "1363: [(, , , We present a machine learning approach that uses protein sequences, with sequence identity ranging between 10% and 90%, as an alternative to conventional DNA sequence alignment-based approaches to identify putative AMR genes in Gram-negative bacteria.)]\n",
            "1364: [(, , , Flies play an important role as vectors in the transmission of antimicrobial-resistant bacteria (ARB) and are hypothesized to transfer ARB between internal and external livestock housing areas.)]\n",
            "1365: [(, , , The aim of this study was to understand the role that flies may play in the maintenance of ARB in the farm environment.)]\n",
            "1366: [(, , , Excrement containing antimicrobial-resistant bacteria (ARB) is discharged from the hospital sewage through wastewater treatment plants (WWTP) into rivers, increasing the antimicrobial resistance (AMR) burden on the environment.)]\n",
            "1367: [(, , , We illustrate the contamination of hospital sewage tanks with ARB harboring antimicrobial resistance genes (ARGs) using comprehensive metagenomic sequencing.)]\n",
            "1368: [(, , , During the study period, we moved to a new hospital building constructed for renovation.)]\n",
            "1369: [(, , , Therefore, we investigated the difference in bacterial flora in the sewage tanks for each building with different departments, and the change in bacterial flora over time in new sewage tanks.)]\n",
            "1370: [(, , , These results suggest that flies act as a reservoir of ARB throughout their life cycle and may therefore be involved in the maintenance and circulation of ARB in the farm environment.)]\n",
            "1371: [(, , , Preferential ARG occurrence, and potential transmission, between characteristically similar biomes indicate strong ecological boundaries.)]\n",
            "1372: [(, , , Our results provide a high-resolution global map of ARG distribution and importantly, identify checkpoint biomes wherein interventions aimed at disrupting ARGs dissemination are likely to be most effective in reducing dissemination and in the long term, the ARG global burden.)]\n",
            "1373: [(, , , Here, we provide new evidence showing that AMPs do not stimulate or enhance bacterial DNA recombination in the surviving fractions.)]\n",
            "1374: [(, , , Recombination accelerates evolution of antibiotic resistance. Our findings have implications for our understanding of host-microbe interactions, the evolution of innate immune defences, and shed new light on the dynamic of antimicrobial-resistance evolution.)]\n",
            "1375: [(, , , The most common STs were ST320, ST81, ST271, ST876, and ST3173.)]\n",
            "1376: [(30, 34, Gene/Proteins, Overall, 93.5 % and 96.3 % of ESBL and AmpC E. coli were resistant to ≥1 another antimicrobial class and 44.5 % and 44.4 % were resistant to ≥3 another antimicrobial class, respectively.)]\n",
            "1377: [(39, 43, Gene/Proteins, Overall, 93.5 % and 96.3 % of ESBL and AmpC E. coli were resistant to ≥1 another antimicrobial class and 44.5 % and 44.4 % were resistant to ≥3 another antimicrobial class, respectively.)]\n",
            "1378: [(44, 51, Microbes, Overall, 93.5 % and 96.3 % of ESBL and AmpC E. coli were resistant to ≥1 another antimicrobial class and 44.5 % and 44.4 % were resistant to ≥3 another antimicrobial class, respectively.)]\n",
            "1379: [(80, 84, Gene/Proteins, This study shows that sheep and goat milk is a reservoir of multidrug resistant ESBL and AmpC β-lactamases E. coli, thus good hygienic practices and judicious antimicrobials use are important in small ruminants' farming.)]\n",
            "1380: [(107, 114, Microbes, This study shows that sheep and goat milk is a reservoir of multidrug resistant ESBL and AmpC β-lactamases E. coli, thus good hygienic practices and judicious antimicrobials use are important in small ruminants' farming.)]\n",
            "1381: [(89, 106, Gene/Proteins, This study shows that sheep and goat milk is a reservoir of multidrug resistant ESBL and AmpC β-lactamases E. coli, thus good hygienic practices and judicious antimicrobials use are important in small ruminants' farming.)]\n",
            "1382: [(0, 24, Microbes, Streptococcus pneumoniae is an important human pathogen that can cause severe invasive pneumococcal diseases (IPDs).)]\n",
            "1383: [(57, 101, Others, The aim of this multicenter study was to investigate the serotype and sequence type (ST) distribution, antimicrobial susceptibility, and virulence of S. pneumoniae strains causing IPD in China.)]\n",
            "1384: [(150, 163, Antibiotic-Resistant Bacteria, The aim of this multicenter study was to investigate the serotype and sequence type (ST) distribution, antimicrobial susceptibility, and virulence of S. pneumoniae strains causing IPD in China.)]\n",
            "1385: [(6, 9, Antibiotic-Resistant Genes, These ARGs were most frequent in human gut, oral and skin biomes, followed by anthropogenic (wastewater, bioreactor, compost, food), and natural biomes (freshwater, marine, sediment).)]\n",
            "1386: [(0, 5, Gene/Proteins, Mcr-9 was the most prevalent mcr gene, spatially and temporally; blaOXA-233 and blaTEM-1 were the most prevalent CR and BL/ESBL genes, but blaGES-2 and blaTEM-116 showed the widest distribution.)]\n",
            "1387: [(65, 75, Gene/Proteins, Mcr-9 was the most prevalent mcr gene, spatially and temporally; blaOXA-233 and blaTEM-1 were the most prevalent CR and BL/ESBL genes, but blaGES-2 and blaTEM-116 showed the widest distribution.)]\n",
            "1388: [(80, 88, Gene/Proteins, Mcr-9 was the most prevalent mcr gene, spatially and temporally; blaOXA-233 and blaTEM-1 were the most prevalent CR and BL/ESBL genes, but blaGES-2 and blaTEM-116 showed the widest distribution.)]\n",
            "1389: [(113, 115, Gene/Proteins, Mcr-9 was the most prevalent mcr gene, spatially and temporally; blaOXA-233 and blaTEM-1 were the most prevalent CR and BL/ESBL genes, but blaGES-2 and blaTEM-116 showed the widest distribution.)]\n",
            "1390: [(120, 122, Gene/Proteins, Mcr-9 was the most prevalent mcr gene, spatially and temporally; blaOXA-233 and blaTEM-1 were the most prevalent CR and BL/ESBL genes, but blaGES-2 and blaTEM-116 showed the widest distribution.)]\n",
            "1391: [(123, 127, Gene/Proteins, Mcr-9 was the most prevalent mcr gene, spatially and temporally; blaOXA-233 and blaTEM-1 were the most prevalent CR and BL/ESBL genes, but blaGES-2 and blaTEM-116 showed the widest distribution.)]\n",
            "1392: [(139, 147, Gene/Proteins, Mcr-9 was the most prevalent mcr gene, spatially and temporally; blaOXA-233 and blaTEM-1 were the most prevalent CR and BL/ESBL genes, but blaGES-2 and blaTEM-116 showed the widest distribution.)]\n",
            "1393: [(152, 162, Gene/Proteins, Mcr-9 was the most prevalent mcr gene, spatially and temporally; blaOXA-233 and blaTEM-1 were the most prevalent CR and BL/ESBL genes, but blaGES-2 and blaTEM-116 showed the widest distribution.)]\n",
            "1394: [(32, 41, Antibiotics, All strains were susceptible to ertapenem, levofloxacin, moxifloxacin, linezolid, and vancomycin, but a very high proportion (>95%) was resistant to macrolides and clindamycin.)]\n",
            "1395: [(43, 55, Antibiotics, All strains were susceptible to ertapenem, levofloxacin, moxifloxacin, linezolid, and vancomycin, but a very high proportion (>95%) was resistant to macrolides and clindamycin.)]\n",
            "1396: [(57, 69, Antibiotics, All strains were susceptible to ertapenem, levofloxacin, moxifloxacin, linezolid, and vancomycin, but a very high proportion (>95%) was resistant to macrolides and clindamycin.)]\n",
            "1397: [(71, 80, Antibiotics, All strains were susceptible to ertapenem, levofloxacin, moxifloxacin, linezolid, and vancomycin, but a very high proportion (>95%) was resistant to macrolides and clindamycin.)]\n",
            "1398: [(86, 96, Antibiotics, All strains were susceptible to ertapenem, levofloxacin, moxifloxacin, linezolid, and vancomycin, but a very high proportion (>95%) was resistant to macrolides and clindamycin.)]\n",
            "1399: [(149, 159, Antibiotics, All strains were susceptible to ertapenem, levofloxacin, moxifloxacin, linezolid, and vancomycin, but a very high proportion (>95%) was resistant to macrolides and clindamycin.)]\n",
            "1400: [(164, 175, Antibiotics, All strains were susceptible to ertapenem, levofloxacin, moxifloxacin, linezolid, and vancomycin, but a very high proportion (>95%) was resistant to macrolides and clindamycin.)]\n",
            "1401: [(0, 19, Techniques, Redundancy analysis and Bayesian analysis showed ARG distribution was non-random and best-explained by potential host genera and biomes, followed by collection year, anthropogenic factors and collection countries.)]\n",
            "1402: [(24, 41, Techniques, Redundancy analysis and Bayesian analysis showed ARG distribution was non-random and best-explained by potential host genera and biomes, followed by collection year, anthropogenic factors and collection countries.)]\n",
            "1403: [(62, 72, Antibiotics, Based on the oral, meningitis and non-meningitis breakpoints, penicillin non-susceptible Streptococcus pneumoniae (PNSP) accounted for 67.7, 67.7 and 4.3% of the isolates, respectively.)]\n",
            "1404: [(89, 113, Microbes, Based on the oral, meningitis and non-meningitis breakpoints, penicillin non-susceptible Streptococcus pneumoniae (PNSP) accounted for 67.7, 67.7 and 4.3% of the isolates, respectively.)]\n",
            "1405: [(130, 133, Antibiotic-Resistant Genes, Serotype 3 strains were characterized by high virulence levels and low antimicrobial-resistance rates, while strains of serotypes 23F, 19F, 19A, and 14, exhibited low virulence and high resistance rates to antibiotics.)]\n",
            "1406: [(135, 138, Antibiotic-Resistant Genes, Serotype 3 strains were characterized by high virulence levels and low antimicrobial-resistance rates, while strains of serotypes 23F, 19F, 19A, and 14, exhibited low virulence and high resistance rates to antibiotics.)]\n",
            "1407: [(140, 143, Antibiotic-Resistant Genes, Serotype 3 strains were characterized by high virulence levels and low antimicrobial-resistance rates, while strains of serotypes 23F, 19F, 19A, and 14, exhibited low virulence and high resistance rates to antibiotics.)]\n",
            "1408: [(149, 151, Antibiotic-Resistant Genes, Serotype 3 strains were characterized by high virulence levels and low antimicrobial-resistance rates, while strains of serotypes 23F, 19F, 19A, and 14, exhibited low virulence and high resistance rates to antibiotics.)]\n",
            "1409: [(120, 133, Microbes, Capsular polysaccharide and non-capsular virulence factors were collectively responsible for the virulence diversity of S. pneumoniae strains.)]\n",
            "1410: [(9, 31, Gene/Proteins, Cationic antimicrobial peptides are ubiquitous immune effectors of multicellular organisms.)]\n",
            "1411: [(104, 117, Antibiotics, Conclusion: Our study provides a comprehensive insight into the epidemiology and virulence diversity of S. pneumoniae strains causing IPD in China.)]\n",
            "1412: [(134, 137, Others, Conclusion: Our study provides a comprehensive insight into the epidemiology and virulence diversity of S. pneumoniae strains causing IPD in China.)]\n",
            "1413: [(150, 157, Antibiotic-Resistant Bacteria, We previously reported, that in contrast to most of the classic antibiotics, cationic antimicrobial peptides (AMPs) do not increase mutation rates in E. coli.)]\n",
            "1414: [(0, 43, Microbes, Extraintestinal pathogenic Escherichia coli (ExPEC) associated infections are significant health concerns for both animals and humans.)]\n",
            "1415: [(0, 5, Microbes, ExPEC strains are associated with various infections in humans, i.e. urinary tract infections, meningitis, septicemia, and other infections.)]\n",
            "1416: [(82, 97, Others, Over the few years, several studies revealed, food animals act as a reservoir for ExPEC pathovars, but there is no information about the agricultural sector.)]\n",
            "1417: [(, , , However, global outbreaks of food-borne illnesses from contaminated food have made a significant concern for both public health and food safety.)]\n",
            "1418: [(, , , This review focuses on the reservoirs for ExPEC and their potential circulation between animals, humans, and environment.)]\n",
            "1419: [(, , , In this, we first report that the agricultural setting could be the reservoir of ExPEC and can play a role in disseminating antimicrobial-resistant ExPEC.)]\n",
            "1420: [(, , , A thorough understanding of ExPEC ecology, reservoirs, and transmission dynamics can significantly contribute to reducing the burden of ExPEC-associated infections.)]\n",
            "1421: [(, , , Overall, the study provides the important data on the current state of knowledge for different reservoirs with dynamic, dissemination, and transmission of antimicrobial-resistance ExPEC in animals, humans, and environment in the \"One-Health\" context.)]\n",
            "1422: [(, , , Methods: A total of 300 invasive S. pneumoniae isolates were included in this study. The serotype, ST, and antimicrobial susceptibility of the strains, were determined by the Quellung reaction, multi-locus sequence typing (MLST) and broth microdilution method, respectively.)]\n",
            "1423: [(, , , Understanding the myriad pathways by which antimicrobial-resistance genes (ARGs) spread across biomes is necessary to counteract the global menace of antimicrobial resistance.)]\n",
            "1424: [(177, 183, Techniques, The virulence level of the strains in the most prevalent serotypes was evaluated by a mouse sepsis model, and the expression level of well-known virulence genes was measured by RT-PCR.)]\n",
            "1425: [(, , , Results: The most common serotypes in this study were 23F, 19A, 19F, 3, and 14. The serotype coverages of PCV7, PCV10, PCV13, and PPV23 vaccines on the strain collection were 42.3, 45.3, 73.3 and 79.3%, respectively.)]\n",
            "1426: [(, , , The availability of high-quality genomes is essential in the fields of veterinary medicine and nutrition to unravel the biological role of key microbial members in the canine gut environment.)]\n",
            "1427: [(, , , Our aim was to evaluate nanopore long-read metagenomics and Hi-C (high-throughput chromosome conformation capture) proximity ligation to provide high-quality metagenome-assembled genomes (HQ MAGs) of the canine gut environment.)]\n",
            "1428: [(, , , Furthermore, a nurse-density-dependent moderation of the inter-species distribution balance was a likely pathway of the observed effect.)]\n",
            "1429: [(, , , By combining nanopore long-read metagenomics and Hi-C proximity ligation, we retrieved 27 HQ MAGs and 7 medium-quality MAGs of a faecal sample of a healthy dog.)]\n",
            "1430: [(, , , Canine MAGs (CanMAGs) improved genome contiguity of representatives from the animal and human MAG catalogues - short-read MAGs from public datasets - for the species they represented: they were more contiguous with complete ribosomal operons and at least 18 canonical tRNAs.)]\n",
            "1431: [(, , , This is the first attempt of quantifying potential nurse-density effects on antimicrobial-resistance at national level.)]\n",
            "1432: [(, , , Both canine-specific bacterial species and gut generalists inhabit the dog's gastrointestinal environment.)]\n",
            "1433: [(, , , Our results suggest an increase in CRGN by decreasing nurse-density.)]\n",
            "1434: [(, , , Nurse-density is thus a novel factor that might improve our understanding of the unbalanced CRGN-distribution among sub-European regions.)]\n",
            "1435: [(, , , Specifically, we assigned the mobilome functions and the associated mobile genetic elements to the bacterial host.)]\n",
            "1436: [(, , , The CanMAGs harboured 50 bacteriophages, providing novel bacterial-host information for eight viral clusters, and Hi-C proximity ligation data linked the six potential plasmids to their bacterial host.)]\n",
            "1437: [(96, 103, Techniques, At the functional level, carbohydrate transport and metabolism was the most variable within the CanMAGs, and mobilome function was abundant in some MAGs.)]\n",
            "1438: [(148, 152, Techniques, At the functional level, carbohydrate transport and metabolism was the most variable within the CanMAGs, and mobilome function was abundant in some MAGs.)]\n",
            "1439: [(0, 25, Techniques, Pearson's chi-square test was performed, and a p value of <0.05 was considered for the level of significance.)]\n",
            "1440: [(84, 101, Antibiotic-Resistant Bacteria, Nearly one in three diarrheal patients admitted in this hospital were infected with Campylobacter spp.)]\n",
            "1441: [(22, 39, Microbes, Overall prevalence of Campylobacter spp. was estimated to be 31.5% (104/330) that comprised the prevalence of C. jejuni, 21.8% (n = 72), and C. coli, 9.6% (n = 32).)]\n",
            "1442: [(110, 119, Antibiotic-Resistant Bacteria, Overall prevalence of Campylobacter spp. was estimated to be 31.5% (104/330) that comprised the prevalence of C. jejuni, 21.8% (n = 72), and C. coli, 9.6% (n = 32).)]\n",
            "1443: [(141, 148, Antibiotic-Resistant Bacteria, Overall prevalence of Campylobacter spp. was estimated to be 31.5% (104/330) that comprised the prevalence of C. jejuni, 21.8% (n = 72), and C. coli, 9.6% (n = 32).)]\n",
            "1444: [(53, 63, Antibiotics, The burden of antimicrobial-resistance, specifically carbapenem-resistance in gram-negative bacteria (CRGN), presents a serious public health threat worldwide.)]\n",
            "1445: [(78, 107, Microbes, The burden of antimicrobial-resistance, specifically carbapenem-resistance in gram-negative bacteria (CRGN), presents a serious public health threat worldwide.)]\n",
            "1446: [(29, 46, Antibiotic-Resistant Bacteria, The present study shows that Campylobacter spp. is most prevalent among the hospital-admitted diarrheal patients, and proper measures should be taken to reduce the burden focusing on the potential determinants.)]\n",
            "1447: [(22, 26, Others, Our modeling provided CRGN-estimations in two non-EARS-net countries (Switzerland and Turkey), which were almost equal to empirically estimated values (CAESAR-Network).)]\n",
            "1448: [(37, 41, Antibiotic-Resistant Bacteria, These observations were specific for CRGN, in contrast to other resistance types in the same species.)]\n",
            "1449: [(49, 61, Microbes, Most of them belonged to Firmicutes, followed by Bacteroidota and Proteobacteria. We also assembled one Actinobacteriota and one Fusobacteriota MAG. CanMAGs harboured antimicrobial-resistance genes (ARGs) and prophages and were linked to plasmids.)]\n",
            "1450: [(25, 35, Microbes, Most of them belonged to Firmicutes, followed by Bacteroidota and Proteobacteria. We also assembled one Actinobacteriota and one Fusobacteriota MAG. CanMAGs harboured antimicrobial-resistance genes (ARGs) and prophages and were linked to plasmids.)]\n",
            "1451: [(66, 80, Microbes, Most of them belonged to Firmicutes, followed by Bacteroidota and Proteobacteria. We also assembled one Actinobacteriota and one Fusobacteriota MAG. CanMAGs harboured antimicrobial-resistance genes (ARGs) and prophages and were linked to plasmids.)]\n",
            "1452: [(104, 120, Microbes, Most of them belonged to Firmicutes, followed by Bacteroidota and Proteobacteria. We also assembled one Actinobacteriota and one Fusobacteriota MAG. CanMAGs harboured antimicrobial-resistance genes (ARGs) and prophages and were linked to plasmids.)]\n",
            "1453: [(129, 147, Microbes, Most of them belonged to Firmicutes, followed by Bacteroidota and Proteobacteria. We also assembled one Actinobacteriota and one Fusobacteriota MAG. CanMAGs harboured antimicrobial-resistance genes (ARGs) and prophages and were linked to plasmids.)]\n",
            "1454: [(149, 156, Antibiotic-Resistant Genes, Most of them belonged to Firmicutes, followed by Bacteroidota and Proteobacteria. We also assembled one Actinobacteriota and one Fusobacteriota MAG. CanMAGs harboured antimicrobial-resistance genes (ARGs) and prophages and were linked to plasmids.)]\n",
            "1455: [(209, 218, Microbes, Most of them belonged to Firmicutes, followed by Bacteroidota and Proteobacteria. We also assembled one Actinobacteriota and one Fusobacteriota MAG. CanMAGs harboured antimicrobial-resistance genes (ARGs) and prophages and were linked to plasmids.)]\n",
            "1456: [(0, 3, Antibiotic-Resistant Genes, ARGs conferring resistance to tetracycline were most predominant within CanMAGs, followed by lincosamide and macrolide ones.)]\n",
            "1457: [(30, 42, Antibiotics, ARGs conferring resistance to tetracycline were most predominant within CanMAGs, followed by lincosamide and macrolide ones.)]\n",
            "1458: [(72, 79, Antibiotics, ARGs conferring resistance to tetracycline were most predominant within CanMAGs, followed by lincosamide and macrolide ones.)]\n",
            "1459: [(93, 104, Antibiotics, ARGs conferring resistance to tetracycline were most predominant within CanMAGs, followed by lincosamide and macrolide ones.)]\n",
            "1460: [(109, 118, Antibiotics, ARGs conferring resistance to tetracycline were most predominant within CanMAGs, followed by lincosamide and macrolide ones.)]\n",
            "1461: [(207, 220, Microbes, In this study, we evaluated a total of 330 clinical samples for the presence Campylobacter spp. via cultural and biochemical tests and molecular assays. Furthermore, antimicrobial susceptibility testing for Campylobacter species was accomplished by the standard agar disc diffusion technique against eight commercially available antimicrobial agents.)]\n",
            "1462: [(77, 95, Microbes, In this study, we evaluated a total of 330 clinical samples for the presence Campylobacter spp. via cultural and biochemical tests and molecular assays. Furthermore, antimicrobial susceptibility testing for Campylobacter species was accomplished by the standard agar disc diffusion technique against eight commercially available antimicrobial agents.)]\n",
            "1463: [(, , , A pretested semistructured questionnaire was used to capture the data on socioanthropological factors from the diarrheal patients.)]\n",
            "1464: [(, , , Long-read metagenomics and Hi-C proximity ligation are likely to become a comprehensive approach to HQ MAG discovery and assignment of extra-chromosomal elements to their bacterial host.)]\n",
            "1465: [(, , , This will provide essential information for studying the canine gut microbiome in veterinary medicine and animal nutrition.)]\n",
            "1466: [(, , , In Europe, Southern and Eastern countries (SEC) display a higher CRGN-prevalence as compared to Northern and Western countries (NWC).)]\n",
            "1467: [(142, 155, Antibiotic-Resistant Bacteria, Age, family level's personal hygiene, and involvement with animal husbandry were captured as potential determinants to be associated with the Campylobacter positive status.)]\n",
            "1468: [(, , , Since SEC also display lower nurse-density on average, we hypothesized that the occurrence of CRGN might correlate with nurse understaffing and therefore aimed at quantifying a potential independent effect of nurse-density on total CRGN in Europe.)]\n",
            "1469: [(70, 77, Antibiotic-Resistant Bacteria, Among the isolates, 27.3% (n = 20) of C. jejuni and 31.2% (n = 10) of C. coli demonstrated as multidrug-resistant (MDR) to three or more antimicrobial agents.)]\n",
            "1470: [(38, 47, Antibiotic-Resistant Bacteria, Among the isolates, 27.3% (n = 20) of C. jejuni and 31.2% (n = 10) of C. coli demonstrated as multidrug-resistant (MDR) to three or more antimicrobial agents.)]\n",
            "1471: [(, , , A 30-country cross-sectional study was conducted. Cumulative six-year CRGN-prevalence (2011-2016) in four gram-negative bacterial species was determined based on > 700 k clinical invasive isolates (EARS-net).)]\n",
            "1472: [(, , , We performed multivariable log-linear regression to provide estimations of the effect of nurse-density while adjusting to various health-system variables.)]\n",
            "1473: [(, , , The human gut microbiome has been extensively studied, yet the canine gut microbiome is still largely unknown.)]\n",
            "1474: [(, , , Multivariable analysis (adj.-R2 ~ 93%) suggested an average 0.4% [95%-CI 0.2-1.0%] CRGN-increase due to a decrement of one practicing nurse per week of hospital-stay of one population individual.)]\n",
            "1475: [(, , , In this research, we reconstructed the phylogenetic relationship based on whole-genome data of strains almost all over China, screened for resistance and virulence genes, and took core and pan genome analysis to perform a comprehensive molecular epidemiology study of S. aureus that is epidemic in China.)]\n",
            "1476: [(, , , Unfortunately, improper use of antibiotics leads to antibiotic resistance.)]\n",
            "1477: [(, , , Our results highlight that there are 4 major epidemic groups with different epidemiology phenotypes after phylogenetic categorization with exclusive genetic features in virulence genotypic profiles, antimicrobial-resistance genotypic profiles, and core and pangenome features, and we found key gene features involved in epidemic transmission and adaptive evolution.)]\n",
            "1478: [(, , , Our findings are critical in describing molecular characteristic profiles of S. aureus infection, which could update existing preventive measures and take appropriate strategies.)]\n",
            "1479: [(, , , Reciprocal, intimate relationships between the human microbiome and the host immune system are shaped by past microbial encounters and prepare the host for future ones.)]\n",
            "1480: [(, , , Antibiotics and other antimicrobials leave their mark on both the microbiome and host immunity.)]\n",
            "1481: [(, , , Antimicrobials alter the structure of the microbiota, expand the host-specific pool of antimicrobial-resistance genes and organisms, degrade the protective effects of the microbiota against invasion by pathogens, and may impair vaccine efficacy.)]\n",
            "1482: [(, , , Through these effects on the microbiome they may affect immune responses. Vaccines that exert protective or therapeutic effects against pathogens may reduce the use of antimicrobials, the development and spread of antimicrobial resistance, and the harmful impacts of these drugs on the microbiome.)]\n",
            "1483: [(, , , This article describes the intersections of immunity, microbiome and antimicrobial exposure, and the use of vaccines and other alternative strategies for the control and management of antimicrobial resistance.)]\n",
            "1484: [(134, 143, Antibiotic-Resistant Bacteria, However, there are few comprehensive genome-based molecular epidemiology studies to investigate the genotypic background of the major S. aureus clones that are epidemic in China.)]\n",
            "1485: [(0, 9, Antibiotics, Polymyxin resistance is a public health concern - present in humans, animals and the environment - caused by chromosomal-encoding or plasmid-encoding mechanisms.)]\n",
            "1486: [(109, 129, Techniques, Polymyxin resistance is a public health concern - present in humans, animals and the environment - caused by chromosomal-encoding or plasmid-encoding mechanisms.)]\n",
            "1487: [(133, 160, Techniques, Polymyxin resistance is a public health concern - present in humans, animals and the environment - caused by chromosomal-encoding or plasmid-encoding mechanisms.)]\n",
            "1488: [(11, 20, Antibiotic-Resistant Bacteria, Here, four S. aureus isolates that were recovered from hospital personnel were sequenced.)]\n",
            "1489: [(27, 31, Antibiotic-Resistant Bacteria, Chromosomal alterations in MgrB are frequently detected in Klebsiella spp., but not yet reported and characterised in Klebsiella variicola (K. variicola).)]\n",
            "1490: [(118, 138, Antibiotic-Resistant Bacteria, Chromosomal alterations in MgrB are frequently detected in Klebsiella spp., but not yet reported and characterised in Klebsiella variicola (K. variicola).)]\n",
            "1491: [(140, 152, Antibiotic-Resistant Bacteria, Chromosomal alterations in MgrB are frequently detected in Klebsiella spp., but not yet reported and characterised in Klebsiella variicola (K. variicola).)]\n",
            "1492: [(59, 73, Antibiotic-Resistant Bacteria, Chromosomal alterations in MgrB are frequently detected in Klebsiella spp., but not yet reported and characterised in Klebsiella variicola (K. variicola).)]\n",
            "1493: [(75, 84, Antibiotics, This study performed microbiological and genomic characterisation of three polymyxin-resistant K. variicola isolates (M14, M15 and M50) recovered from the microbiota of migratory birds in Brazil.)]\n",
            "1494: [(95, 107, Antibiotic-Resistant Bacteria, This study performed microbiological and genomic characterisation of three polymyxin-resistant K. variicola isolates (M14, M15 and M50) recovered from the microbiota of migratory birds in Brazil.)]\n",
            "1495: [(22, 31, Antibiotic-Resistant Bacteria, It was found that 332 S. aureus isolates were phylogenetically categorized into 4 major epidemic groups with different epidemiology phenotypes.)]\n",
            "1496: [(31, 40, Techniques, The isolates were submitted to SpeI-PFGE, broth microdilution and whole genome sequencing using Illumina MiSeq for analysis of genetic relatedness, sequence typing and detection of antimicrobial-resistance genes. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance only for polymyxins.)]\n",
            "1497: [(96, 110, Techniques, The isolates were submitted to SpeI-PFGE, broth microdilution and whole genome sequencing using Illumina MiSeq for analysis of genetic relatedness, sequence typing and detection of antimicrobial-resistance genes. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance only for polymyxins.)]\n",
            "1498: [(48, 61, Techniques, The isolates were submitted to SpeI-PFGE, broth microdilution and whole genome sequencing using Illumina MiSeq for analysis of genetic relatedness, sequence typing and detection of antimicrobial-resistance genes. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance only for polymyxins.)]\n",
            "1499: [(213, 225, Antibiotic-Resistant Bacteria, The isolates were submitted to SpeI-PFGE, broth microdilution and whole genome sequencing using Illumina MiSeq for analysis of genetic relatedness, sequence typing and detection of antimicrobial-resistance genes. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance only for polymyxins.)]\n",
            "1500: [(311, 321, Antibiotics, The isolates were submitted to SpeI-PFGE, broth microdilution and whole genome sequencing using Illumina MiSeq for analysis of genetic relatedness, sequence typing and detection of antimicrobial-resistance genes. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance only for polymyxins.)]\n",
            "1501: [(52, 61, Antibiotic-Resistant Bacteria, Moreover, a comparative core genome analysis of 332 S. aureus isolates indicated several key genes that contributed to differences in molecular epidemic characterization and promoted the adaptive evolutionary process of each group.)]\n",
            "1502: [(141, 150, Antibiotic-Resistant Bacteria, This study provides a comprehensive understanding of molecular epidemiological characteristics and adaptive evolutionary directions of major S. aureus clones that are epidemic in China.)]\n",
            "1503: [(0, 21, Antibiotic-Resistant Bacteria, Staphylococcus aureus is an important Gram-positive pathogen that is epidemic worldwide and causes various infections in humans and domestic animals.)]\n",
            "1504: [(38, 51, Microbes, Staphylococcus aureus is an important Gram-positive pathogen that is epidemic worldwide and causes various infections in humans and domestic animals.)]\n",
            "1505: [(47, 57, Antibiotic-Resistant Bacteria, Nevertheless, information on the occurrence of Vibrio spp. and antibiotic use in shrimp, particularly in Malaysia, is minimal.)]\n",
            "1506: [(, , , In combination with whole-genome sequencing (WGS) data of 328 S. aureus strains as references, we performed a comprehensive molecular epidemiology study to reveal the molecular epidemic characterization of S. aureus that is epidemic in China.)]\n",
            "1507: [(, , , Each group has exclusive features in virulence genotypic profiles, antimicrobial resistance genotypic profiles, core and pangenome features representing the differences involved in genetic features, evolutionary processes, and potential future evolutionary directions.)]\n",
            "1508: [(48, 53, Gene/Proteins, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1509: [(55, 60, Gene/Proteins, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1510: [(62, 67, Gene/Proteins, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1511: [(69, 74, Gene/Proteins, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1512: [(80, 84, Gene/Proteins, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1513: [(103, 109, Techniques, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1514: [(114, 120, Techniques, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1515: [(131, 174, Antibiotic-Resistant Bacteria, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1516: [(233, 245, Techniques, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1517: [(247, 259, Antibiotic-Resistant Bacteria, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1518: [(340, 350, Antibiotics, Sequences of chromosomal two-component systems (PmrAB, PhoPQ, RstAB, CrrAB) and MgrB were evaluated by blastN and blastP against a polymyxin-susceptible K. variicola (A58243), and mutations with biological effect were checked by the PROVEAN tool. K. variicola isolates belonged to two clones, and susceptibility tests showed resistance for polymyxins.)]\n",
            "1519: [(77, 86, Antibiotics, This study confirmed the role of a non-functional mgrB variant in conferring polymyxin resistance, stressing the role of this regulator in K. variicola and drawing attention to novel polymyxin resistance mechanisms emerging in wildlife.)]\n",
            "1520: [(139, 151, Antibiotic-Resistant Bacteria, This study confirmed the role of a non-functional mgrB variant in conferring polymyxin resistance, stressing the role of this regulator in K. variicola and drawing attention to novel polymyxin resistance mechanisms emerging in wildlife.)]\n",
            "1521: [(183, 192, Antibiotics, This study confirmed the role of a non-functional mgrB variant in conferring polymyxin resistance, stressing the role of this regulator in K. variicola and drawing attention to novel polymyxin resistance mechanisms emerging in wildlife.)]\n",
            "1522: [(50, 54, Antibiotic-Resistant Genes, This study confirmed the role of a non-functional mgrB variant in conferring polymyxin resistance, stressing the role of this regulator in K. variicola and drawing attention to novel polymyxin resistance mechanisms emerging in wildlife.)]\n",
            "1523: [(0, 6, Microbes, Vibrio is the most common bacterium associated with diseases in crustaceans.)]\n",
            "1524: [(9, 19, Antibiotic-Resistant Bacteria, In China, S. aureus is the most common Gram-positive pathogen that causes clinical infections.)]\n",
            "1525: [(, , , However, there has been relatively little research on comprehensive molecular epidemiology in China.)]\n",
            "1526: [(, , , Outbreaks of vibriosis pose a serious threat to shrimp production. Therefore, antibiotics are commonly used as preventative and therapeutic measures.)]\n",
            "1527: [(, , , The goal of this study was to investigate if the disruption of iron acquisition, or the antimicrobial-resistance activity of Mega, contributed to inhibiting the killing mechanism.)]\n",
            "1528: [(, , , Neither depletion of iron with 2,2'-dipyridyl-d8 (DP) nor incubating with a double knockout mutant SpnΔpiaAΔpiuA, inhibited killing of Sau.)]\n",
            "1529: [(, , , Shrimp were sampled from seven farms that were located in different geographical regions of Peninsular Malaysia.)]\n",
            "1530: [(, , , According to the observations, 85% of the shrimp were healthy, whereas 15% were unhealthy.)]\n",
            "1531: [(, , , An ex vivo recombination system was used to transfer Mega1. IV or Mega2. IVc to reference Spn strains, which was confirmed by whole genome sequencing, and recombinants TIGR4Mega2. IVc, D39Mega2. IVc, and D39Mega1. IV were delayed for killing Sau.)]\n",
            "1532: [(, , , We then compared Sau killing of selected Mega-carrying Spn strains when incubated with sub-inhibitory erythromycin (Mega-induced) or sub-inhibitory cefuroxime.)]\n",
            "1533: [(, , , The isolates were also tested for antibiotic susceptibility against 16 antibiotics and were subjected to plasmid profiling.)]\n",
            "1534: [(, , , Remarkably, killing of Sau was completely inhibited under the Mega-induced condition whereas incubation with cefuroxime did not interfere with killing.)]\n",
            "1535: [(, , , These findings support the use of surveillance data on the emerging patterns of antimicrobial-resistance and plasmid profiles of Vibrio spp. in shrimp farms.)]\n",
            "1536: [(, , , In fact, MRSA strains are frequently resistant to all β-lactam antibiotics, and co-resistances with other drugs such as macrolides, aminoglycosides, and lincosamides are usually reported, limiting the therapeutical options.)]\n",
            "1537: [(25, 44, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1538: [(129, 139, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1539: [(52, 63, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1540: [(70, 83, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1541: [(90, 100, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1542: [(107, 122, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1543: [(147, 163, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1544: [(170, 185, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1545: [(192, 205, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1546: [(212, 219, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1547: [(226, 236, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1548: [(243, 256, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1549: [(268, 279, Antibiotic-Resistant Bacteria, They were the following: V. parahaemolyticus (55%), V. communis (9%), V. campbellii (8%), V. owensii (7%), V. rotiferianus (5%), Vibrio spp. (4%), V. alginolyticus (3%), V. brasiliensis (2%), V. natriegens (2%), V. xuii (1%), V. harveyi (1%), V. hepatarius (0.4%) and P. damselae (3%).)]\n",
            "1550: [(66, 133, Gene/Proteins, It has been shown that Pseudomonas aeruginosa elastase (LasB) and Clostridium histolyticum (Hathewaya histolytica) collagenase (ColH) play a significant role in the infection process and thereby represent promising antivirulence targets.)]\n",
            "1551: [(23, 61, Gene/Proteins, It has been shown that Pseudomonas aeruginosa elastase (LasB) and Clostridium histolyticum (Hathewaya histolytica) collagenase (ColH) play a significant role in the infection process and thereby represent promising antivirulence targets.)]\n",
            "1552: [(22, 50, Gene/Proteins, Here, we report novel N-aryl-3-mercaptosuccinimide inhibitors that target both LasB and ColH, displaying potent activities in vitro and high selectivity for the bacterial over human metalloproteases.)]\n",
            "1553: [(79, 83, Gene/Proteins, Here, we report novel N-aryl-3-mercaptosuccinimide inhibitors that target both LasB and ColH, displaying potent activities in vitro and high selectivity for the bacterial over human metalloproteases.)]\n",
            "1554: [(88, 92, Gene/Proteins, Here, we report novel N-aryl-3-mercaptosuccinimide inhibitors that target both LasB and ColH, displaying potent activities in vitro and high selectivity for the bacterial over human metalloproteases.)]\n",
            "1555: [(182, 198, Gene/Proteins, Here, we report novel N-aryl-3-mercaptosuccinimide inhibitors that target both LasB and ColH, displaying potent activities in vitro and high selectivity for the bacterial over human metalloproteases.)]\n",
            "1556: [(29, 33, Gene/Proteins, Furthermore, the most active ColH inhibitor shows a significant reduction of collagen degradation in an ex vivo pig-skin model.)]\n",
            "1557: [(77, 85, Gene/Proteins, Furthermore, the most active ColH inhibitor shows a significant reduction of collagen degradation in an ex vivo pig-skin model.)]\n",
            "1558: [(, , , The findings from this study can be used to develop a better disease management strategy for shrimp farming.)]\n",
            "1559: [(45, 88, Antibiotic-Resistant Bacteria, The treatment and hospital-spread-control of methicillin-resistant Staphylococcus aureus (MRSA) is an important challenge since these bacteria are involved in a considerable number of nosocomial infections that are difficult to treat and produce prolonged hospitalization, thus also increasing the risk of death.)]\n",
            "1560: [(37, 58, Antibiotic-Resistant Bacteria, Streptococcus pneumoniae (Spn) kills Staphylococcus aureus (Sau) through a contact-dependent mechanism that is catalyzed by cations, including iron, to convert hydrogen peroxide (H2O2) to highly toxic hydroxyl radicals (•OH).)]\n",
            "1561: [(0, 24, Antibiotic-Resistant Bacteria, Streptococcus pneumoniae (Spn) kills Staphylococcus aureus (Sau) through a contact-dependent mechanism that is catalyzed by cations, including iron, to convert hydrogen peroxide (H2O2) to highly toxic hydroxyl radicals (•OH).)]\n",
            "1562: [(89, 92, Gene/Proteins, There are two well-characterized ABC transporters that contribute to the pool of iron in Spn, named Pia and Piu.)]\n",
            "1563: [(100, 103, Gene/Proteins, There are two well-characterized ABC transporters that contribute to the pool of iron in Spn, named Pia and Piu.)]\n",
            "1564: [(108, 111, Gene/Proteins, There are two well-characterized ABC transporters that contribute to the pool of iron in Spn, named Pia and Piu.)]\n",
            "1565: [(37, 43, Gene/Proteins, Some Spn strains have acquired genes mef(E)/mel encoding another ABC trasporter (Mega) that produces an inducible efflux pump for resistance to macrolides.)]\n",
            "1566: [(44, 47, Gene/Proteins, Some Spn strains have acquired genes mef(E)/mel encoding another ABC trasporter (Mega) that produces an inducible efflux pump for resistance to macrolides.)]\n",
            "1567: [(144, 154, Antibiotics, Some Spn strains have acquired genes mef(E)/mel encoding another ABC trasporter (Mega) that produces an inducible efflux pump for resistance to macrolides.)]\n",
            "1568: [(5, 8, Microbes, Some Spn strains have acquired genes mef(E)/mel encoding another ABC trasporter (Mega) that produces an inducible efflux pump for resistance to macrolides.)]\n",
            "1569: [(80, 89, Antibiotic-Resistant Bacteria, Here, we genotypically and phenotypically characterized 100 clinically isolated S. aureus for their resistance to 18 antibiotics (33% of them were OXA resistant MRSA) and ability to form biofilms.)]\n",
            "1570: [(161, 165, Antibiotic-Resistant Bacteria, Here, we genotypically and phenotypically characterized 100 clinically isolated S. aureus for their resistance to 18 antibiotics (33% of them were OXA resistant MRSA) and ability to form biofilms.)]\n",
            "1571: [(147, 150, Antibiotics, Here, we genotypically and phenotypically characterized 100 clinically isolated S. aureus for their resistance to 18 antibiotics (33% of them were OXA resistant MRSA) and ability to form biofilms.)]\n",
            "1572: [(3, 12, Antibiotics, In macrolide-resistant Spn clinical isolates the insertion of Mega class 1. IV and 2. IVc deleted the locus piaABCD and these strains were attenuated for intoxicating Sau.)]\n",
            "1573: [(23, 26, Microbes, In macrolide-resistant Spn clinical isolates the insertion of Mega class 1. IV and 2. IVc deleted the locus piaABCD and these strains were attenuated for intoxicating Sau.)]\n",
            "1574: [(108, 115, Gene/Proteins, In macrolide-resistant Spn clinical isolates the insertion of Mega class 1. IV and 2. IVc deleted the locus piaABCD and these strains were attenuated for intoxicating Sau.)]\n",
            "1575: [(167, 170, Gene/Proteins, In macrolide-resistant Spn clinical isolates the insertion of Mega class 1. IV and 2. IVc deleted the locus piaABCD and these strains were attenuated for intoxicating Sau.)]\n",
            "1576: [(62, 72, Gene/Proteins, In macrolide-resistant Spn clinical isolates the insertion of Mega class 1. IV and 2. IVc deleted the locus piaABCD and these strains were attenuated for intoxicating Sau.)]\n",
            "1577: [(76, 78, Gene/Proteins, In macrolide-resistant Spn clinical isolates the insertion of Mega class 1. IV and 2. IVc deleted the locus piaABCD and these strains were attenuated for intoxicating Sau.)]\n",
            "1578: [(86, 89, Gene/Proteins, In macrolide-resistant Spn clinical isolates the insertion of Mega class 1. IV and 2. IVc deleted the locus piaABCD and these strains were attenuated for intoxicating Sau.)]\n",
            "1579: [(61, 71, Microbes, This study aimed to provide information on the occurrence of Vibrio spp., its status of antibiotic resistance and the plasmid profiles of Vibrio spp. isolated from cultured shrimp in Peninsular Malaysia.)]\n",
            "1580: [(14, 24, Antibiotics, Resistance to vancomycin (34.0%, 98/288) and teicoplanin (18.8%, 98/288) was frequently observed in Enterococcus faecium strains.)]\n",
            "1581: [(45, 56, Antibiotics, Resistance to vancomycin (34.0%, 98/288) and teicoplanin (18.8%, 98/288) was frequently observed in Enterococcus faecium strains.)]\n",
            "1582: [(100, 120, Microbes, Resistance to vancomycin (34.0%, 98/288) and teicoplanin (18.8%, 98/288) was frequently observed in Enterococcus faecium strains.)]\n",
            "1583: [(9, 12, Antibiotic-Resistant Genes, Clinical Spn strains carrying Mega1. IV or Mega2. IVc showed a significant delay for killing Sau.)]\n",
            "1584: [(30, 53, Gene/Proteins, Clinical Spn strains carrying Mega1. IV or Mega2. IVc showed a significant delay for killing Sau.)]\n",
            "1585: [(30, 36, Microbes, Subsequently, 225 presumptive Vibrio isolates were subjected to biochemical tests and molecular detection using the pyrH gene.)]\n",
            "1586: [(64, 81, Techniques, Subsequently, 225 presumptive Vibrio isolates were subjected to biochemical tests and molecular detection using the pyrH gene.)]\n",
            "1587: [(86, 105, Techniques, Subsequently, 225 presumptive Vibrio isolates were subjected to biochemical tests and molecular detection using the pyrH gene.)]\n",
            "1588: [(116, 120, Gene/Proteins, Subsequently, 225 presumptive Vibrio isolates were subjected to biochemical tests and molecular detection using the pyrH gene.)]\n",
            "1589: [(, , , We demonstrated that erythromycin induction of Mega inhibits the •OH-mediated intoxication of Sau and that the inhibition occurred at the post-translational level suggesting that an imbalance of ions in the membrane inhibits these reactions.)]\n",
            "1590: [(120, 131, Antibiotics, Antibiotic susceptibility profiles revealed that all isolates were resistant to penicillin G (100%), but susceptible to norfloxacin (96%).)]\n",
            "1591: [(80, 92, Antibiotics, Antibiotic susceptibility profiles revealed that all isolates were resistant to penicillin G (100%), but susceptible to norfloxacin (96%).)]\n",
            "1592: [(, , , In light of the global antimicrobial-resistance crisis, there is an urgent need for novel bacterial targets and antibiotics with novel modes of action.)]\n",
            "1593: [(, , , Furthermore, 16% of the isolates revealed MAR of less than 0.2, while 84% were greater than 0.2.)]\n",
            "1594: [(, , , A total of 125 isolates harbored plasmids with molecular weights between 1.0 and above 10 kb, detected among the resistant isolates.)]\n",
            "1595: [(, , , The resistant isolates were mediated by both chromosomal and plasmid factors.)]\n",
            "1596: [(, , , Additionally, the inhibitors demonstrate no signs of cytotoxicity against selected human cell lines and in a zebrafish embryo toxicity model.)]\n",
            "1597: [(, , , To this must be added that the ability of these bacteria to form biofilms on hospital surfaces and devices confer high antibiotic resistance and favors horizontal gene transfer of genetic-resistant mobile elements, the spreading of infections, and relapses.)]\n",
            "1598: [(, , , From them, we selected 48 strains on the basis on genotype group, antimicrobial-resistance profile, and existing OXA resistance to be assayed against bacteriocin AS-48.)]\n",
            "1599: [(, , , Finally, we explored the effect of AS-48 on formed S. aureus biofilms, observing a reduction in S. aureus S-33 viability.)]\n",
            "1600: [(, , , Changes in the matrix structure of the biofilms as well as in the cell division process were observed with scanning electron microscopy in both S-33 and S-48 S. aureus strains.)]\n",
            "1601: [(, , , Antibiotics are routinely used in commercial poultry farms for the treatment of economically important bacterial diseases.)]\n",
            "1602: [(, , , Whereas food safety is not compromised by the presence of low amounts of S. aureus in sushi, this study shows that with regards to public health such foods may serve as vehicles for transmission of multidrug-resistant S. aureus and MRSA lineages.)]\n",
            "1603: [(, , , The emergence of multidrug-resistant bacteria has dramatically increased the lethality, level of resistance, and difficulty of treatment.)]\n",
            "1604: [(6, 10, Antibiotic-Resistant Bacteria, Three MRSA isolates were obtained, one from pork (ST398) and two from one sushi outlet (ST59).)]\n",
            "1605: [(112, 130, Gene/Proteins, In this study, a series of new antimicrobial peptides (AMPs) based on the β-hairpin structure with the template (XY)2RRRF(YX)2-NH2 (X: hydrophobic amino acids; Y: cationic amino acids) were synthesized; surprisingly, almost all of the new peptides have strong antibacterial activity and negligible hemolytic toxicity.)]\n",
            "1606: [(40, 52, Antibiotics, The highest positivity was obtained for tetracycline (tet[L], tet[M], tet[K], tetA[P]] and aminoglycoside (aadA2) ARG, confirming the predominant use of these antimicrobials in the veterinary practice and their potential to enhance the resistance patterns also in humans as a consequence of environmental contamination.)]\n",
            "1607: [(54, 60, Antibiotics, The highest positivity was obtained for tetracycline (tet[L], tet[M], tet[K], tetA[P]] and aminoglycoside (aadA2) ARG, confirming the predominant use of these antimicrobials in the veterinary practice and their potential to enhance the resistance patterns also in humans as a consequence of environmental contamination.)]\n",
            "1608: [(62, 68, Antibiotics, The highest positivity was obtained for tetracycline (tet[L], tet[M], tet[K], tetA[P]] and aminoglycoside (aadA2) ARG, confirming the predominant use of these antimicrobials in the veterinary practice and their potential to enhance the resistance patterns also in humans as a consequence of environmental contamination.)]\n",
            "1609: [(70, 76, Antibiotics, The highest positivity was obtained for tetracycline (tet[L], tet[M], tet[K], tetA[P]] and aminoglycoside (aadA2) ARG, confirming the predominant use of these antimicrobials in the veterinary practice and their potential to enhance the resistance patterns also in humans as a consequence of environmental contamination.)]\n",
            "1610: [(78, 85, Antibiotics, The highest positivity was obtained for tetracycline (tet[L], tet[M], tet[K], tetA[P]] and aminoglycoside (aadA2) ARG, confirming the predominant use of these antimicrobials in the veterinary practice and their potential to enhance the resistance patterns also in humans as a consequence of environmental contamination.)]\n",
            "1611: [(91, 105, Antibiotics, The highest positivity was obtained for tetracycline (tet[L], tet[M], tet[K], tetA[P]] and aminoglycoside (aadA2) ARG, confirming the predominant use of these antimicrobials in the veterinary practice and their potential to enhance the resistance patterns also in humans as a consequence of environmental contamination.)]\n",
            "1612: [(107, 112, Antibiotics, The highest positivity was obtained for tetracycline (tet[L], tet[M], tet[K], tetA[P]] and aminoglycoside (aadA2) ARG, confirming the predominant use of these antimicrobials in the veterinary practice and their potential to enhance the resistance patterns also in humans as a consequence of environmental contamination.)]\n",
            "1613: [(89, 110, Antibiotic-Resistant Bacteria, The mean counts of coliforms were 2.7 and 2.9 log CFU/g in sushi and pork, respectively. Staphylococcus aureus was isolated from seven sushi outlets (13 isolates) and two pork outlets (2 isolates) with average counts below 2 log CFU/g in all cases.)]\n",
            "1614: [(14, 23, Antibiotic-Resistant Bacteria, A total of 15 S. aureus isolates were further characterized. Six lineages of S. aureus were present, including ST398 (n = 5), ST25 (n = 4), ST15 (n = 2), ST59 (n = 2), ST8 (n = 1) and ST2631 (n = 1).)]\n",
            "1615: [(77, 86, Antibiotic-Resistant Bacteria, A total of 15 S. aureus isolates were further characterized. Six lineages of S. aureus were present, including ST398 (n = 5), ST25 (n = 4), ST15 (n = 2), ST59 (n = 2), ST8 (n = 1) and ST2631 (n = 1).)]\n",
            "1616: [(24, 29, Microbes, The results showed that AS-48 was active against all strains, regardless of their clinical source, genotype, antimicrobial resistance profile, or biofilm formation capacity, and this activity was enhanced in the presence of the antimicrobial peptide lysozyme.)]\n",
            "1617: [(229, 258, Gene/Proteins, The results showed that AS-48 was active against all strains, regardless of their clinical source, genotype, antimicrobial resistance profile, or biofilm formation capacity, and this activity was enhanced in the presence of the antimicrobial peptide lysozyme.)]\n",
            "1618: [(98, 106, Gene/Proteins, They were all resistant to at least three classes of antimicrobials and two of them contained the scn gene and enterotoxin genes.)]\n",
            "1619: [(111, 128, Gene/Proteins, They were all resistant to at least three classes of antimicrobials and two of them contained the scn gene and enterotoxin genes.)]\n",
            "1620: [(65, 73, Gene/Proteins, Twelve sushi isolates and one of the pork isolates contained the scn gene, indicating that they were of human origin.)]\n",
            "1621: [(98, 107, Microbes, This emphasizes the potential importance of transmission through foods of antimicrobial-resistant S. aureus including MRSA.)]\n",
            "1622: [(118, 122, Microbes, This emphasizes the potential importance of transmission through foods of antimicrobial-resistant S. aureus including MRSA.)]\n",
            "1623: [(20, 29, Microbes, We also showed that S. aureus exhibited geographical variation with regards to ST profiles, antimicrobial-resistance and virulence genes when comparing isolates from sushi products sold in Beijing and Copenhagen, Denmark.)]\n",
            "1624: [(36, 39, Techniques, In this study, single and multiplex PCR protocols were performed to detect tetracycline-, lincomycin-, chloramphenicol-, aminoglycoside-, colistin-, vancomycin-, and carbapenem-resistance genes, starting from 38 litter samples collected from 6 poultry and 2 turkey Italian flocks.)]\n",
            "1625: [(75, 87, Antibiotics, In this study, single and multiplex PCR protocols were performed to detect tetracycline-, lincomycin-, chloramphenicol-, aminoglycoside-, colistin-, vancomycin-, and carbapenem-resistance genes, starting from 38 litter samples collected from 6 poultry and 2 turkey Italian flocks.)]\n",
            "1626: [(90, 100, Antibiotics, In this study, single and multiplex PCR protocols were performed to detect tetracycline-, lincomycin-, chloramphenicol-, aminoglycoside-, colistin-, vancomycin-, and carbapenem-resistance genes, starting from 38 litter samples collected from 6 poultry and 2 turkey Italian flocks.)]\n",
            "1627: [(103, 118, Antibiotics, In this study, single and multiplex PCR protocols were performed to detect tetracycline-, lincomycin-, chloramphenicol-, aminoglycoside-, colistin-, vancomycin-, and carbapenem-resistance genes, starting from 38 litter samples collected from 6 poultry and 2 turkey Italian flocks.)]\n",
            "1628: [(121, 135, Antibiotics, In this study, single and multiplex PCR protocols were performed to detect tetracycline-, lincomycin-, chloramphenicol-, aminoglycoside-, colistin-, vancomycin-, and carbapenem-resistance genes, starting from 38 litter samples collected from 6 poultry and 2 turkey Italian flocks.)]\n",
            "1629: [(138, 146, Antibiotics, In this study, single and multiplex PCR protocols were performed to detect tetracycline-, lincomycin-, chloramphenicol-, aminoglycoside-, colistin-, vancomycin-, and carbapenem-resistance genes, starting from 38 litter samples collected from 6 poultry and 2 turkey Italian flocks.)]\n",
            "1630: [(149, 159, Antibiotics, In this study, single and multiplex PCR protocols were performed to detect tetracycline-, lincomycin-, chloramphenicol-, aminoglycoside-, colistin-, vancomycin-, and carbapenem-resistance genes, starting from 38 litter samples collected from 6 poultry and 2 turkey Italian flocks.)]\n",
            "1631: [(166, 176, Antibiotics, In this study, single and multiplex PCR protocols were performed to detect tetracycline-, lincomycin-, chloramphenicol-, aminoglycoside-, colistin-, vancomycin-, and carbapenem-resistance genes, starting from 38 litter samples collected from 6 poultry and 2 turkey Italian flocks.)]\n",
            "1632: [(, , , Particularly, the therapeutic index (TI) values of F(RI)2R and F(KW)2K reached up to 115.9 and 70.7, respectively.)]\n",
            "1633: [(, , , On the contrary, the dissemination by poultry of ARG for critically important antimicrobials seems to be of minor concern, suggesting a negligible environmental dissemination by these genes in the Italian poultry industry.)]\n",
            "1634: [(, , , In addition, they did not show induced drug resistance and inhibited the development of antibiotic resistance when combined and used with traditional antibiotics.)]\n",
            "1635: [(, , , Finally, the molecular screening performed in this study using a noninvasive sampling method represents a simple and rapid tool for monitoring the ARG patterns at the farm level.)]\n",
            "1636: [(, , , In addition, their antibacterial mechanism was preliminarily studied. Moreover, the new peptides F(RI)2R and F(KW)2K showed excellent performance in the pulmonary bacterial infection model and low toxicity in mice.)]\n",
            "1637: [(, , , Staphylococcus aureus is an opportunistic pathogen leading to food poisoning as well as human infections.)]\n",
            "1638: [(, , , In conclusion, F(RI)2R and F(KW)2K are considered new antimicrobial alternatives to address the antimicrobial-resistance crisis.)]\n",
            "1639: [(, , , The present study examined the prevalence and characterization of antimicrobial-resistant S. aureus in sushi from 42 outlets and in pork products from eight outlets in Beijing, China.)]\n",
            "1640: [(, , , The growth of respiratory diseases, as witnessed through the SARS and COVID-19 outbreaks, and antimicrobial-resistance together pose a serious threat to humanity.)]\n",
            "1641: [(, , , The total bacterial counts were between 3.0 and 8.9 log CFU/g (mean 5.5 ± 1.5 log CFU/g) in sushi products and 4.8 to 7.4 log CFU/g (mean 5.6 ± 0.8 log CFU/g) in pork products.)]\n",
            "1642: [(, , , One reason for antimicrobial resistance is formation of bacterial biofilms.)]\n",
            "1643: [(32, 52, Gene/Proteins, Thirteen isolates contained the scn virulence marker, whereas four and eight isolates contained the virulence marker edinB and enterotoxin genes, respectively.)]\n",
            "1644: [(100, 122, Gene/Proteins, Thirteen isolates contained the scn virulence marker, whereas four and eight isolates contained the virulence marker edinB and enterotoxin genes, respectively.)]\n",
            "1645: [(127, 144, Gene/Proteins, Thirteen isolates contained the scn virulence marker, whereas four and eight isolates contained the virulence marker edinB and enterotoxin genes, respectively.)]\n",
            "1646: [(, , , Characterization of antimicrobial resistance profiles documented resistances to ampicillin (n = 15), penicillin (n = 14), ceftazidime (n = 6), erythromycin (n = 4), tetracycline (n = 3), clindamycin (n = 3), and gentamicin (n = 1).)]\n",
            "1647: [(, , , Scanning electron microscopy analysis showed that Cr-SPs effectively altered the morphology of these bacterial cells and distorted the bacterial biofilms.)]\n",
            "1648: [(, , , The current study paves way towards developing Cr-SPs as a control strategy for treatment of respiratory tract infections.)]\n",
            "1649: [(, , , The phylogeny illustrated the diversity of strains associated with paediatric meningitis in West Africa and highlighted the existence of phylogeographical clustering, with isolates from the same sub-region clustering and sharing similar accessory genome content.)]\n",
            "1650: [(, , , Staphylococci are major hospital-associated pathogens, and the dissemination of methicillin-resistant staphylococci in hospitals poses a great challenge for managing hospital-acquired infections.)]\n",
            "1651: [(, , , Little is known about the dissemination of staphylococci recovered from the hospital environment and personnel in China.)]\n",
            "1652: [(, , , However, there was no discernible trend linking the presence of resistance genotypes with the vaccine introduction period or whether the strain was a vaccine or non-vaccine serotype.)]\n",
            "1653: [(, , , Resistance genotypes appeared to be conserved within selected sub-clades of the phylogenetic tree, suggesting clonal inheritance.)]\n",
            "1654: [(, , , We analysed whole-genome sequencing data from 185 S. pneumoniae isolates recovered from suspected paediatric meningitis cases as part of the World Health Organization (WHO) invasive bacterial diseases surveillance from 2010 to 2016.)]\n",
            "1655: [(37, 47, Techniques, This article contains data hosted by Microreact.)]\n",
            "1656: [(, , , The minimum bactericidal concentration was determined by culture at 1, 24 and 48 h.)]\n",
            "1657: [(13, 26, Antibiotics, The emerging cephalosporin-resistant Neisseria gonorrhoeae poses an urgent threat to the continued efficacy of the last-line monotherapy for gonorrhea.)]\n",
            "1658: [(37, 58, Antibiotic-Resistant Bacteria, The emerging cephalosporin-resistant Neisseria gonorrhoeae poses an urgent threat to the continued efficacy of the last-line monotherapy for gonorrhea.)]\n",
            "1659: [(125, 136, Techniques, The emerging cephalosporin-resistant Neisseria gonorrhoeae poses an urgent threat to the continued efficacy of the last-line monotherapy for gonorrhea.)]\n",
            "1660: [(42, 47, Gene/Proteins, Three new sequence types were assigned as ST840, ST841, and ST842. One (2%) frequently touched surface was contaminated by S. aureus, which suggested that environmental contamination occurred in the hospital in China.)]\n",
            "1661: [(49, 54, Gene/Proteins, Three new sequence types were assigned as ST840, ST841, and ST842. One (2%) frequently touched surface was contaminated by S. aureus, which suggested that environmental contamination occurred in the hospital in China.)]\n",
            "1662: [(60, 65, Gene/Proteins, Three new sequence types were assigned as ST840, ST841, and ST842. One (2%) frequently touched surface was contaminated by S. aureus, which suggested that environmental contamination occurred in the hospital in China.)]\n",
            "1663: [(123, 132, Microbes, Three new sequence types were assigned as ST840, ST841, and ST842. One (2%) frequently touched surface was contaminated by S. aureus, which suggested that environmental contamination occurred in the hospital in China.)]\n",
            "1664: [(152, 166, Antibiotic-Resistant Bacteria, Consequently, high-throughput, accurate, and reasonable molecular assays are urgently needed for strengthening antimicrobial-resistance surveillance in N. gonorrhoeae.)]\n",
            "1665: [(64, 127, Antibiotic-Resistant Bacteria, Nineteen (39%) frequently touched surfaces were contaminated by methicillin-resistant coagulase-negative staphylococci (MRCoNS), and 46% of HP carried MRCoNS.)]\n",
            "1666: [(151, 157, Antibiotic-Resistant Bacteria, Nineteen (39%) frequently touched surfaces were contaminated by methicillin-resistant coagulase-negative staphylococci (MRCoNS), and 46% of HP carried MRCoNS.)]\n",
            "1667: [(7, 21, Microbes, Varied staphylococcal species and antimicrobial-resistance rates were observed between staphylococci that were recovered from hospital personnel and frequently touched surfaces.)]\n",
            "1668: [(87, 100, Microbes, Varied staphylococcal species and antimicrobial-resistance rates were observed between staphylococci that were recovered from hospital personnel and frequently touched surfaces.)]\n",
            "1669: [(20, 24, Others, The transmission of MRSE and S. aureus between hospital personnel and frequently touched surfaces was detected.)]\n",
            "1670: [(29, 38, Antibiotic-Resistant Bacteria, The transmission of MRSE and S. aureus between hospital personnel and frequently touched surfaces was detected.)]\n",
            "1671: [(78, 91, Antibiotic-Resistant Bacteria, Hospital items and personnel may act as reservoirs of antimicrobial-resistant staphylococci, and cleaning strategies should be carried out to decrease the dissemination of antimicrobial-resistant staphylococci in hospitals in China.)]\n",
            "1672: [(196, 209, Antibiotic-Resistant Bacteria, Hospital items and personnel may act as reservoirs of antimicrobial-resistant staphylococci, and cleaning strategies should be carried out to decrease the dissemination of antimicrobial-resistant staphylococci in hospitals in China.)]\n",
            "1673: [(162, 183, Antibiotic-Resistant Bacteria, In this study the sulphated polysaccharides from green algae Chlamydomonas reinhardtii (Cr-SPs) is tested for its antibacterial and antibiofilm potential against Klebsiella pneumoniae and Serratia marcescens.)]\n",
            "1674: [(188, 207, Antibiotic-Resistant Bacteria, In this study the sulphated polysaccharides from green algae Chlamydomonas reinhardtii (Cr-SPs) is tested for its antibacterial and antibiofilm potential against Klebsiella pneumoniae and Serratia marcescens.)]\n",
            "1675: [(61, 95, Microbes, In this study the sulphated polysaccharides from green algae Chlamydomonas reinhardtii (Cr-SPs) is tested for its antibacterial and antibiofilm potential against Klebsiella pneumoniae and Serratia marcescens.)]\n",
            "1676: [(115, 139, Antibiotic-Resistant Bacteria, Despite contributing to the large disease burden in West Africa, little is known about the genomic epidemiology of Streptococcus pneumoniae which cause meningitis among children under 5 years old in the region.)]\n",
            "1677: [(64, 70, Gene/Proteins, Agar cup assay clearly indicated the antibacterial potential of Cr-SPs. Minimum inhibitory concentration (MIC50) of Cr-SPs against Klebsiella pneumoniae was found to be 850 µg/ml, and it is 800 µg/ml in Serratia marcescens.)]\n",
            "1678: [(203, 222, Microbes, Agar cup assay clearly indicated the antibacterial potential of Cr-SPs. Minimum inhibitory concentration (MIC50) of Cr-SPs against Klebsiella pneumoniae was found to be 850 µg/ml, and it is 800 µg/ml in Serratia marcescens.)]\n",
            "1679: [(131, 152, Antibiotic-Resistant Bacteria, Agar cup assay clearly indicated the antibacterial potential of Cr-SPs. Minimum inhibitory concentration (MIC50) of Cr-SPs against Klebsiella pneumoniae was found to be 850 µg/ml, and it is 800 µg/ml in Serratia marcescens.)]\n",
            "1680: [(116, 118, Gene/Proteins, Agar cup assay clearly indicated the antibacterial potential of Cr-SPs. Minimum inhibitory concentration (MIC50) of Cr-SPs against Klebsiella pneumoniae was found to be 850 µg/ml, and it is 800 µg/ml in Serratia marcescens.)]\n",
            "1681: [(, , , Time-kill and colony-forming ability assays suggest the concentration-dependent bactericidal potential of Cr-SPs. Cr-SPs showed 74-100% decrease in biofilm formation in a concentration-dependent manner by modifying the cell surface hydrophobic properties of these bacteria.)]\n",
            "1682: [(0, 6, Antibiotic-Resistant Bacteria, Cr-SPs have also distorted preformed-biofilms by their ability to interact and destroy the extra polymeric substance and eDNA of the matured biofilm.)]\n",
            "1683: [(85, 91, Gene/Proteins, Furthermore reduced protease, urease and prodigiosin pigment production suggest that Cr-SPs interferes the quorum sensing mechanism in these bacteria.)]\n",
            "1684: [(27, 32, Others, In the Western sub-region, PCV13 serotypes were more dominant among isolates from The Gambia following the introduction of PCV7, 81 % (n=17), compared to the pre-PCV period in neighbouring Senegal, 51 % (n=27).)]\n",
            "1685: [(91, 105, Antibiotics, Antibiotic-resistance genotypes known to confer resistance to penicillin, chloramphenicol, co-trimoxazole and tetracycline were detected across all sub-regions.)]\n",
            "1686: [(91, 124, Techniques, In this study, antimicrobial susceptibility tests, mecA gene detection, SCCmec typing, and multilocus sequence typing (MLST) were performed to clarify the molecular epidemiology of staphylococci in a large hospital in Tianjin, China.)]\n",
            "1687: [(181, 194, Antibiotic-Resistant Bacteria, In this study, antimicrobial susceptibility tests, mecA gene detection, SCCmec typing, and multilocus sequence typing (MLST) were performed to clarify the molecular epidemiology of staphylococci in a large hospital in Tianjin, China.)]\n",
            "1688: [(28, 41, Antibiotic-Resistant Bacteria, One hundred and ninety-five staphylococci were recovered, and 94% of isolates were resistant to at least one antibiotic.)]\n",
            "1689: [(109, 124, Antibiotics, Our data underscore the need for continued surveillance on the emergence of non-vaccine serotypes as well as chloramphenicol and penicillin resistance, as these antibiotics are likely still being used for empirical treatment in low-resource settings.)]\n",
            "1690: [(129, 139, Antibiotics, Our data underscore the need for continued surveillance on the emergence of non-vaccine serotypes as well as chloramphenicol and penicillin resistance, as these antibiotics are likely still being used for empirical treatment in low-resource settings.)]\n",
            "1691: [(, , , The phylogeny was reconstructed, accessory genome similarity was computed and antimicrobial-resistance patterns were inferred from the genome data and compared to phenotypic resistance from disc diffusion.)]\n",
            "1692: [(, , , We studied the changes in the distribution of serotypes pre- and post-pneumococcal conjugate vaccine (PCV) introduction in the Central and Western sub-regions separately.)]\n",
            "1693: [(, , , Subsequently, AMR transmission is affected by the standards of infection control, sanitation, access to clean water, access to assured quality antimicrobials and diagnostics, travel and migration.)]\n",
            "1694: [(, , , This review provides a brief description of AMR as a global concern and the possible contribution of different environmental drivers to the transmission of antibiotic-resistant bacteria or ARGs through various mechanisms.)]\n",
            "1695: [(, , , This method can be directly applied to clinical samples without cumbersome culture and successfully predicted all cephalosporin-resistant isolates (sensitivity: 100%).)]\n",
            "1696: [(, , , We also aim to highlight the key knowledge gaps that hinder environmental regulators and mitigation strategies in delivering environmental protection against AMR.)]\n",
            "1697: [(, , , Antimicrobial resistance is developing rapidly and threatens to outstrip the rate at which new antimicrobials are introduced.)]\n",
            "1698: [(, , , Coronavirus disease 2019 may have a complex long-term impact on antimicrobial resistance (AMR).)]\n",
            "1699: [(, , , Genetic recombination allows bacteria to rapidly disseminate genes encoding for antimicrobial resistance within and across species.)]\n",
            "1700: [(, , , Coordinated strategies at the individual, health-care and policy levels are urgently required to inform necessary actions to reduce the potential longer-term impact on AMR and on access to effective antimicrobials.)]\n",
            "1701: [(, , , Antimicrobial use creates a selective evolutionary pressure, which leads to further resistance.)]\n",
            "1702: [(, , , The mechanisms behind the development and spread of bacterial resistance to antimicrobial drugs are reviewed.)]\n",
            "1703: [(, , , Antimicrobial stewardship, best use, and infection prevention are the most effective ways to slow the spread and development of antimicrobial resistance.)]\n",
            "1704: [(, , , The chief mechanisms by which antimicrobials act are interference with nucleic acid synthesis, binding to ribosomes, and inhibition of cell-wall synthesis and folate metabolism.)]\n",
            "1705: [(, , , Various drivers are contributing factors to the spread of antibiotic-resistant bacteria and their ARGs either directly through antimicrobial drug use in health care, agriculture/livestock and the environment or antibiotic residues released from various domestic settings.)]\n",
            "1706: [(, , , Bacteria have evolved genetic and biochemical ways of resisting these antimicrobial actions.)]\n",
            "1707: [(166, 181, Microbes, Whole-genome sequencing revealed 24 different antimicrobial-resistance determinants, including genes not previously described in the Gallibacterium genus or even the Pasteurellaceae family, such as aadA23, blaCARB-8, tet(Y), and qnrD1.)]\n",
            "1708: [(133, 147, Microbes, Whole-genome sequencing revealed 24 different antimicrobial-resistance determinants, including genes not previously described in the Gallibacterium genus or even the Pasteurellaceae family, such as aadA23, blaCARB-8, tet(Y), and qnrD1.)]\n",
            "1709: [(198, 204, Gene/Proteins, Whole-genome sequencing revealed 24 different antimicrobial-resistance determinants, including genes not previously described in the Gallibacterium genus or even the Pasteurellaceae family, such as aadA23, blaCARB-8, tet(Y), and qnrD1.)]\n",
            "1710: [(206, 215, Gene/Proteins, Whole-genome sequencing revealed 24 different antimicrobial-resistance determinants, including genes not previously described in the Gallibacterium genus or even the Pasteurellaceae family, such as aadA23, blaCARB-8, tet(Y), and qnrD1.)]\n",
            "1711: [(217, 223, Gene/Proteins, Whole-genome sequencing revealed 24 different antimicrobial-resistance determinants, including genes not previously described in the Gallibacterium genus or even the Pasteurellaceae family, such as aadA23, blaCARB-8, tet(Y), and qnrD1.)]\n",
            "1712: [(229, 234, Gene/Proteins, Whole-genome sequencing revealed 24 different antimicrobial-resistance determinants, including genes not previously described in the Gallibacterium genus or even the Pasteurellaceae family, such as aadA23, blaCARB-8, tet(Y), and qnrD1.)]\n",
            "1713: [(83, 95, Others, Some resistance genes were closely linked in resistance gene cassettes with either transposases in close proximity or situated on putative mobile elements or predicted plasmids.)]\n",
            "1714: [(168, 176, Microbes, Some resistance genes were closely linked in resistance gene cassettes with either transposases in close proximity or situated on putative mobile elements or predicted plasmids.)]\n",
            "1715: [(0, 41, Techniques, Single-nucleotide polymorphism genotyping revealed large genetic variation between the G. anatis isolates, including isolates retrieved from the same farm.)]\n",
            "1716: [(87, 96, Microbes, Single-nucleotide polymorphism genotyping revealed large genetic variation between the G. anatis isolates, including isolates retrieved from the same farm.)]\n",
            "1717: [(0, 9, Microbes, G. anatis might play a hitherto unrecognized role as a respiratory pathogen and resistance gene reservoir in cattle and has unknown zoonotic potential.)]\n",
            "1718: [(, , , While antimicrobial agents are beneficial to combat infection, their widespread use contributes to the increase in and emergence of novel resistant microbes in virtually all environmental niches.)]\n",
            "1719: [(, , , Genetic mechanisms include mutation and acquisition of new DNA.)]\n",
            "1720: [(, , , The human microbiome is an important reservoir of AMR with initial exposure occurring in early life.)]\n",
            "1721: [(, , , In this study, we designed a high-throughput multiplex method that incorporates high-resolution melting technology and is based on a 6-codon assay (among the most parsimonious assays) developed following comprehensive and systematic reviews.)]\n",
            "1722: [(, , , Once seeded with AMR, commensal organisms may be key contributors to the dissemination of resistance due to the interconnectedness of microbial communities.)]\n",
            "1723: [(, , , The results showed that our method can precisely distinguish specific single-nucleotide polymorphisms in resistance-associated genes with a specificity and sensitivity of 100% and a detection limit as low as 10 copies per reaction.)]\n",
            "1724: [(, , , When acquired by pathogens however, AMR becomes a serious public health threat worldwide.)]\n",
            "1725: [(, , , Our ability to combat the threat of emerging resistance relies on accurate AMR detection methods and the development of therapeutics that function despite the presence of antimicrobial resistance.)]\n",
            "1726: [(206, 237, Antibiotic-Resistant Bacteria, The method presented here represents a technique for rapid testing of antimicrobial resistance and will serve as a valuable tool for tailor-made antimicrobial therapy and for monitoring the transmission of cephalosporin-resistant strains.)]\n",
            "1727: [(, , , Imprudent and overuse of clinically relevant antibiotics in agriculture, veterinary and medical sectors contribute to the global epidemic increase in antimicrobial resistance (AMR).)]\n",
            "1728: [(, , , Gallibacterium anatis is an opportunistic pathogen, previously associated with deaths in poultry, domestic birds, and occasionally humans.)]\n",
            "1729: [(, , , There is a growing concern among researchers and stakeholders that the environment acts as an AMR reservoir and plays a key role in the dissemination of antimicrobial resistance genes (ARGs).)]\n",
            "1730: [(, , , Collected isolates were multidrug-resistant to extensively drug-resistant, exhibiting resistance against 5-7 classes of antimicrobial drugs.)]\n",
            "1731: [(, , , Resistant micro-organisms and their resistance genes enter the soil, air, water and sediments through various routes or hotspots such as hospital wastewater, agricultural waste or wastewater treatment plants.)]\n",
            "1732: [(, , , Global mitigation strategies primarily involve the identification of high-risk environments that are responsible for the evolution and spread of resistance.)]\n",
            "1733: [(, , , There is no question that the widespread use, overuse, and misuse of antimicrobials during the last 80 years have been associated with the explosion of antimicrobial resistance.)]\n",
            "1734: [(, , , On the other hand, the molecular pathways behind the emergence of antimicrobial resistance in bacteria were present since ancient times.)]\n",
            "1735: [(, , , In this review, we summarize recent studies on the assembly systems of OM components and analyze studies for the development of inhibitors that target these systems.)]\n",
            "1736: [(, , , Some of these mechanisms are the ancestors of current resistance determinants.)]\n",
            "1737: [(, , , These analyses show that OM assembly machines have the potential to be a novel attractive drug target of Gram-negative bacteria.)]\n",
            "1738: [(, , , Evidently, there are plenty of putative resistance genes in the environment, however, we cannot yet predict which ones would be able to be expressed as phenotypes in pathogenic bacteria and cause clinical disease.)]\n",
            "1739: [(, , , In addition, in the presence of inhibitory and sub-inhibitory concentrations of antibiotics in natural habitats, one could assume that novel resistance mechanisms will arise against antimicrobial compounds.)]\n",
            "1740: [(, , , They can easily contaminate the food and the environment, entering the food chain.)]\n",
            "1741: [(, , , Moreover, Enterococcus is an important opportunistic pathogen, especially the species E. faecalis and E. faecium, causing a wide variety of infections.)]\n",
            "1742: [(, , , This review presents an overview of antimicrobial resistance mechanisms, and describes how these have evolved and how they continue to emerge.)]\n",
            "1743: [(, , , As antimicrobial strategies able to bypass the development of resistance are urgently needed, a better understanding of the critical factors that contribute to the persistence and spread of antimicrobial resistance may yield innovative perspectives on the design of such new therapeutic targets.)]\n",
            "1744: [(, , , The genetic plasticity of microbes allows them to adapt to stressors via genetic mutations, acquisition or sharing of genetic material and modulation of genetic expression leading to resistance to virtually any antimicrobial used in clinical practice.)]\n",
            "1745: [(, , , Areas covered: This is a comprehensive review that outlines major mechanisms of resistance in the most common hospital-associated pathogens including bacteria and fungi.)]\n",
            "1746: [(, , , Enterococci are natural inhabitants of the intestinal tract in humans and many animals, including food-producing and companion animals.)]\n",
            "1747: [(0, 12, Microbes, Enterococcal and staphylococcal resistance mechanisms are of particular importance clinically.)]\n",
            "1748: [(17, 31, Microbes, Enterococcal and staphylococcal resistance mechanisms are of particular importance clinically.)]\n",
            "1749: [(140, 174, Others, Staphylococcal resistance to penicillins is expressed as beta-lactamase production, secretion of novel beta-lactamases, expression of novel penicillin-binding proteins (PBPs) to which penicillins bind poorly, and increased production of or altered affinity to existing PBPs.)]\n",
            "1750: [(103, 114, Microbes, Of great concern is whether newly described glycopeptide resistance can be transferred clinically from enterococci to staphylococci.)]\n",
            "1751: [(118, 131, Microbes, Of great concern is whether newly described glycopeptide resistance can be transferred clinically from enterococci to staphylococci.)]\n",
            "1752: [(0, 10, Antibiotics, Vancomycin use is discouraged to limit the spread of glycopeptide resistance.)]\n",
            "1753: [(53, 65, Gene/Proteins, Vancomycin use is discouraged to limit the spread of glycopeptide resistance.)]\n",
            "1754: [(56, 78, Antibiotic-Resistant Bacteria, Gram-negative pathogens, such as Klebsiella pneumoniae, Pseudomonas aeruginosa, and Acinetobacter baumannii, pose a serious threat to public health worldwide, due to high rates of antibiotic resistance and the lack of development of novel antimicrobial agents targeting Gram-negative bacteria.)]\n",
            "1755: [(33, 54, Antibiotic-Resistant Bacteria, Gram-negative pathogens, such as Klebsiella pneumoniae, Pseudomonas aeruginosa, and Acinetobacter baumannii, pose a serious threat to public health worldwide, due to high rates of antibiotic resistance and the lack of development of novel antimicrobial agents targeting Gram-negative bacteria.)]\n",
            "1756: [(84, 107, Antibiotic-Resistant Bacteria, Gram-negative pathogens, such as Klebsiella pneumoniae, Pseudomonas aeruginosa, and Acinetobacter baumannii, pose a serious threat to public health worldwide, due to high rates of antibiotic resistance and the lack of development of novel antimicrobial agents targeting Gram-negative bacteria.)]\n",
            "1757: [(151, 162, Antibiotics, The outer membrane (OM) of Gram-negative bacteria is a unique architecture that acts as a potent permeability barrier against toxic molecules, such as antibiotics.)]\n",
            "1758: [(22, 35, Gene/Proteins, The OM is composed of phospholipids, lipopolysaccharide (LPS), outer membrane β-barrel proteins (OMP), and lipoproteins.)]\n",
            "1759: [(37, 55, Gene/Proteins, The OM is composed of phospholipids, lipopolysaccharide (LPS), outer membrane β-barrel proteins (OMP), and lipoproteins.)]\n",
            "1760: [(107, 119, Gene/Proteins, The OM is composed of phospholipids, lipopolysaccharide (LPS), outer membrane β-barrel proteins (OMP), and lipoproteins.)]\n",
            "1761: [(78, 95, Gene/Proteins, The OM is composed of phospholipids, lipopolysaccharide (LPS), outer membrane β-barrel proteins (OMP), and lipoproteins.)]\n",
            "1762: [(173, 200, Others, These components are synthesized in the cytoplasm or in the inner membrane, and are then selectively transported to the OM by the specific transport machines, including the Lol, BAM, and Lpt pathways.)]\n",
            "1763: [(, , , Widespread use of antibiotics, however, has led to the development of resistant organisms, particularly in the healthcare setting.)]\n",
            "1764: [(, , , Today, the clinician is often faced with pathogens carrying a cadre of resistance determinants that severely limit therapeutic options.)]\n",
            "1765: [(78, 93, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1766: [(95, 104, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1767: [(106, 120, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1768: [(122, 133, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1769: [(135, 143, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1770: [(145, 155, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1771: [(161, 190, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1772: [(277, 287, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1773: [(289, 304, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1774: [(306, 321, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1775: [(323, 335, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1776: [(337, 349, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1777: [(355, 365, Antibiotics, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1778: [(25, 37, Microbes, There are three types of enterococcal resistance: (1) intrinsic resistance to aminoglycosides, aztreonam, cephalosporins, clindamycin, imipenem, penicillin, and trimethoprim-sulfamethoxazole, (2) tolerance to all cell-wall-active antimicrobials, and (3) acquired resistance to penicillin, aminoglycosides, chloramphenicol, erythromycin, tetracycline, and vancomycin.)]\n",
            "1779: [(, , , Expert commentary: Understanding the genetic and biochemical mechanisms of such antimicrobial adaptation is crucial to tackling the rapid spread of resistance, can expose unconventional therapeutic targets to combat multidrug resistant pathogens and lead to more accurate prediction of antimicrobial susceptibility using rapid molecular diagnostics.)]\n",
            "1780: [(, , , Clinicians making treatment decisions based on the molecular basis of resistance may design therapeutic strategies that include de-escalation of broad spectrum antimicrobial usage, more focused therapies or combination therapies.)]\n",
            "1781: [(, , , These strategies are likely to improve patient outcomes and decrease the risk of resistance in hospital settings.)]\n",
            "1782: [(, , , Many mechanisms are responsible for the development and spread of antimicrobial resistance.)]\n",
            "1783: [(, , , In recent years, we have seen antimicrobial resistance rapidly emerge at a global scale and spread from one country to the other faster than previously thought.)]\n",
            "1784: [(, , , Superbugs and multidrug-resistant bacteria are endemic in many parts of the world.)]\n",
            "1785: [(, , , Emerging rates of multidrug resistance observed in almost 50% of the isolates may alert for a near future of challenging veterinary cases unresponsive to first-line antimicrobials.)]\n",
            "1786: [(, , , In addition, these results highlight a potential public health concern of multidrug resistant bacteria, given the proximity of pets and their owners.)]\n",
            "1787: [(, , , This study addressed central aspects of external otitis, providing microbiologists and clinicians updated information on the etiology and treatment of challenging cases of multidrug resistant bacteria.)]\n",
            "1788: [(, , , It also provides priceless surveillance value in monitoring resistant bacteria in small animals.)]\n",
            "1789: [(, , , Technologies based on the measurement of bacterial density in suspension have evolved marginally in accuracy and rapidity over the 20th century, but assays expanded for new combinations of bacteria and antimicrobials have been automated, and made amenable to high-throughput turn-around.)]\n",
            "1790: [(, , , This microorganism not only contains intrinsic resistance mechanisms to several antimicrobial agents, but also has the capacity to acquire new mechanisms of antimicrobial resistance.)]\n",
            "1791: [(, , , At the end of 2015, a global action plan on antimicrobial resistance (AMR) was proposed by the World Health Organization, and the Global AMR Surveillance System (GLASS) was subsequently initiated.)]\n",
            "1792: [(254, 263, Antibiotics, Moreover, resistance mechanisms for different classes of antimicrobials of clinical relevance are reviewed, as well as the epidemiology of multidrug-resistant enterococci of animal origin, with special attention given to beta-lactams, glycopeptides, and linezolid.)]\n",
            "1793: [(221, 233, Antibiotics, Moreover, resistance mechanisms for different classes of antimicrobials of clinical relevance are reviewed, as well as the epidemiology of multidrug-resistant enterococci of animal origin, with special attention given to beta-lactams, glycopeptides, and linezolid.)]\n",
            "1794: [(235, 248, Antibiotics, Moreover, resistance mechanisms for different classes of antimicrobials of clinical relevance are reviewed, as well as the epidemiology of multidrug-resistant enterococci of animal origin, with special attention given to beta-lactams, glycopeptides, and linezolid.)]\n",
            "1795: [(159, 170, Antibiotic-Resistant Bacteria, Moreover, resistance mechanisms for different classes of antimicrobials of clinical relevance are reviewed, as well as the epidemiology of multidrug-resistant enterococci of animal origin, with special attention given to beta-lactams, glycopeptides, and linezolid.)]\n",
            "1796: [(101, 124, Microbes, There is great public and professional concern related to antimicrobial resistance, especially among Gram-positive pathogens associated with high morbidity and mortality.)]\n",
            "1797: [(93, 98, Antibiotic-Resistant Genes, The emergence of new antimicrobial resistance genes in enterococci of animal origin, such as optrA and cfr, is highlighted.)]\n",
            "1798: [(103, 106, Antibiotic-Resistant Genes, The emergence of new antimicrobial resistance genes in enterococci of animal origin, such as optrA and cfr, is highlighted.)]\n",
            "1799: [(0, 10, Antibiotics, Penicillin-nonsusceptible Streptococcus pneumoniae, glycopeptide resistant enterococci, and oxacillin-resistant (MRSA) or vancomycin-intermediate (VISA) Staphylococcus aureus isolates continue to escalate in occurrence leading to the widespread use of empiric combination regimens.)]\n",
            "1800: [(26, 50, Antibiotic-Resistant Bacteria, Penicillin-nonsusceptible Streptococcus pneumoniae, glycopeptide resistant enterococci, and oxacillin-resistant (MRSA) or vancomycin-intermediate (VISA) Staphylococcus aureus isolates continue to escalate in occurrence leading to the widespread use of empiric combination regimens.)]\n",
            "1801: [(75, 86, Antibiotic-Resistant Bacteria, Penicillin-nonsusceptible Streptococcus pneumoniae, glycopeptide resistant enterococci, and oxacillin-resistant (MRSA) or vancomycin-intermediate (VISA) Staphylococcus aureus isolates continue to escalate in occurrence leading to the widespread use of empiric combination regimens.)]\n",
            "1802: [(92, 101, Antibiotics, Penicillin-nonsusceptible Streptococcus pneumoniae, glycopeptide resistant enterococci, and oxacillin-resistant (MRSA) or vancomycin-intermediate (VISA) Staphylococcus aureus isolates continue to escalate in occurrence leading to the widespread use of empiric combination regimens.)]\n",
            "1803: [(122, 132, Antibiotics, Penicillin-nonsusceptible Streptococcus pneumoniae, glycopeptide resistant enterococci, and oxacillin-resistant (MRSA) or vancomycin-intermediate (VISA) Staphylococcus aureus isolates continue to escalate in occurrence leading to the widespread use of empiric combination regimens.)]\n",
            "1804: [(113, 117, Antibiotic-Resistant Bacteria, Penicillin-nonsusceptible Streptococcus pneumoniae, glycopeptide resistant enterococci, and oxacillin-resistant (MRSA) or vancomycin-intermediate (VISA) Staphylococcus aureus isolates continue to escalate in occurrence leading to the widespread use of empiric combination regimens.)]\n",
            "1805: [(153, 174, Antibiotic-Resistant Bacteria, Penicillin-nonsusceptible Streptococcus pneumoniae, glycopeptide resistant enterococci, and oxacillin-resistant (MRSA) or vancomycin-intermediate (VISA) Staphylococcus aureus isolates continue to escalate in occurrence leading to the widespread use of empiric combination regimens.)]\n",
            "1806: [(59, 70, Antibiotic-Resistant Bacteria, The molecular epidemiology and the population structure of E. faecalis and E. faecium isolates in farm and companion animals is presented. Moreover, the types of plasmids that carry the antimicrobial resistance genes in enterococci of animal origin are reviewed.)]\n",
            "1807: [(75, 85, Antibiotic-Resistant Bacteria, The molecular epidemiology and the population structure of E. faecalis and E. faecium isolates in farm and companion animals is presented. Moreover, the types of plasmids that carry the antimicrobial resistance genes in enterococci of animal origin are reviewed.)]\n",
            "1808: [(, , , The etiology of otitis in dogs and cats is multifactorial and complex, involving bacterial and fungal pathogens.)]\n",
            "1809: [(22, 34, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1810: [(50, 62, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1811: [(36, 48, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1812: [(65, 77, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1813: [(177, 189, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1814: [(211, 225, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1815: [(227, 238, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1816: [(384, 395, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1817: [(397, 408, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1818: [(419, 433, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1819: [(446, 456, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1820: [(491, 499, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1821: [(504, 513, Antibiotics, Among the quinolones (gatifloxacin, gemifloxacin, moxifloxacin), gatifloxacin is closest to clinical use and appears safe based on initial trial reports. Several broad-spectrum beta-lactams (\"fourth-generation\" cephalosporins, carbapenems) are expected to be used with increasing frequency as a result of the emerging high rates of specific beta-lactamases that compromise the use of ceftriaxone, ceftazidime, and many beta-lactamase inhibition/ penicillin combinations. Among these agents, cefepime and meropenem are the most potent and broadest in clinical application in their respective classes.)]\n",
            "1822: [(, , , A pilot phase of Kor-GLASS was operated from May to December 2016 with six sentinel hospitals, and phase I of Kor-GLASS started in January 2017 with eight sentinel hospitals.)]\n",
            "1823: [(119, 133, Microbes, The presence of mixed cultures of Enterobacteriaceae, Pseudomonas spp. and Staphylococcus spp., and co-infections with Malassezia spp., emphasizes the polymicrobial etiology of external otitis in small animals.)]\n",
            "1824: [(54, 69, Microbes, The presence of mixed cultures of Enterobacteriaceae, Pseudomonas spp. and Staphylococcus spp., and co-infections with Malassezia spp., emphasizes the polymicrobial etiology of external otitis in small animals.)]\n",
            "1825: [(29, 50, Microbes, One-half (53.2%, 377/708) of Staphylococcus aureus blood strains exhibited resistance to cefoxitin, indicating methicillin-resistant S. aureus.)]\n",
            "1826: [(89, 98, Antibiotics, One-half (53.2%, 377/708) of Staphylococcus aureus blood strains exhibited resistance to cefoxitin, indicating methicillin-resistant S. aureus.)]\n",
            "1827: [(111, 122, Antibiotics, One-half (53.2%, 377/708) of Staphylococcus aureus blood strains exhibited resistance to cefoxitin, indicating methicillin-resistant S. aureus.)]\n",
            "1828: [(133, 142, Microbes, One-half (53.2%, 377/708) of Staphylococcus aureus blood strains exhibited resistance to cefoxitin, indicating methicillin-resistant S. aureus.)]\n",
            "1829: [(14, 24, Antibiotics, Resistance to ampicillin in Enterococcus faecalis blood strains was rare (0.6%, 1/175), while the resistance rate to penicillin was 26.3% (46/175).)]\n",
            "1830: [(28, 49, Microbes, Resistance to ampicillin in Enterococcus faecalis blood strains was rare (0.6%, 1/175), while the resistance rate to penicillin was 26.3% (46/175).)]\n",
            "1831: [(117, 127, Antibiotics, Resistance to ampicillin in Enterococcus faecalis blood strains was rare (0.6%, 1/175), while the resistance rate to penicillin was 26.3% (46/175).)]\n",
            "1832: [(67, 115, Techniques, Over the past 25 years, elevated AST rapidity has been provided by nucleic acid-mediated amplification technologies, proteomic and other \"omic\" methodologies, and the use of next-generation sequencing.)]\n",
            "1833: [(117, 157, Techniques, Over the past 25 years, elevated AST rapidity has been provided by nucleic acid-mediated amplification technologies, proteomic and other \"omic\" methodologies, and the use of next-generation sequencing.)]\n",
            "1834: [(190, 200, Techniques, Over the past 25 years, elevated AST rapidity has been provided by nucleic acid-mediated amplification technologies, proteomic and other \"omic\" methodologies, and the use of next-generation sequencing.)]\n",
            "1835: [(, , , As empiric antimicrobial prescription is a common practice when treating such cases, antimicrobial resistance may represent a complicating factor.)]\n",
            "1836: [(, , , Physicians must stay informed about drug development and antimicrobial resistance by using results from local surveillance programs.)]\n",
            "1837: [(, , , The aim of this study was to describe microbiological features and susceptibility profiles of pathogens associated with 142 cases of external otitis, comprising 138 dogs and 4 cats.)]\n",
            "1838: [(110, 116, Techniques, When these data are unavailable, physicians should consider the use of national or global monitoring systems (SENTRY) to direct empiric antimicrobial selection.)]\n",
            "1839: [(, , , The specimens were processed to identify bacterial and fungal etiologies following standard microbiological methods.)]\n",
            "1840: [(, , , Antimicrobial susceptibility was determined in vitro against 15 antibiotics and 3 antifungals.)]\n",
            "1841: [(, , , The Centers for Disease Control and Prevention of South Korea established a customized AMR surveillance system for South Korea, called Kor-GLASS, in early 2016.)]\n",
            "1842: [(99, 119, Techniques, Further, Staphylococcus spp. isolates were screened for the detection of β-lactamase enzymes using cefinase paper discs.)]\n",
            "1843: [(73, 92, Gene/Proteins, Further, Staphylococcus spp. isolates were screened for the detection of β-lactamase enzymes using cefinase paper discs.)]\n",
            "1844: [(9, 28, Microbes, Further, Staphylococcus spp. isolates were screened for the detection of β-lactamase enzymes using cefinase paper discs.)]\n",
            "1845: [(, , , Pseudomonas spp. and isolates from Enterobacteriaceae family were screened for colistin (Polymyxin E) resistance and for the mcr-1-mediated colistin resistance gene by PCR.)]\n",
            "1846: [(, , , Previous surveillance data for overestimated AMR due to duplicate isolation of drug-resistant pathogens were corrected and error-free AMR data were compared with those from other countries.)]\n",
            "1847: [(, , , These results suggested that multiple bacterial strains, many of which are multidrug-resistant, can cause UTIs in dogs.)]\n",
            "1848: [(, , , The resistance rates of Pseudomonas aeruginosa strains to imipenem and meropenem were 19.5% (29/149) and 18.1% (27/149), respectively.)]\n",
            "1849: [(, , , It is widely accepted that conjugation plays a major role in the spread of AMR in bacteria, and the focus of this review is therefore mainly on the evidence provided that antimicrobial treatment affects this process.)]\n",
            "1850: [(, , , Thus, basic antimicrobial susceptibility tests should be performed to provide guidance for the selection of first-line clinical therapeutics, and to help prevent the occurrence and spread of induced antimicrobial resistance.)]\n",
            "1851: [(, , , Resistance against nearly all antibiotics used clinically have been documented in bacteria.)]\n",
            "1852: [(, , , There is an ever-increasing danger caused by multidrug-resistant Gram-negative bacteria in both hospital and community settings.)]\n",
            "1853: [(, , , In Gram-negative bacteria, intrinsic resistance to currently available antibiotics is mainly due to overexpressed efflux pumps which are constitutively present and also presence of protective outer membrane.)]\n",
            "1854: [(, , , Combination therapy, i.e., use of two or more antibiotics, was thought to be an effective strategy because it took advantage of the additive effects of multiple antimicrobial mechanisms, lower risk of resistance development and lower mortality and improved clinical outcome.)]\n",
            "1855: [(, , , The high incidence of bacteremia caused by major AMR pathogens among hospitalized patients especially in intensive care units emphasized the importance of hospital infection control and the need to improve the crowded hospitalization system in South Korea.)]\n",
            "1856: [(, , , However, none of the benefits were seen in in vivo studies. Antibiotic hybrids are being used to challenge the growing drug resistance threat and increase the usefulness of current antibiotic arsenal.)]\n",
            "1857: [(, , , The selective pressures caused by increased use and misuse of antimicrobials in medicine and livestock production have accelerated the overall selection of resistant bacteria.)]\n",
            "1858: [(, , , Antibiotic hybrids are synthetic constructs of two molecules which are covalently linked.)]\n",
            "1859: [(63, 84, Others, These could be two antibiotics or antibiotic with an adjuvant (efflux pump inhibitor, siderophore, etc.) which increases the access of the antibiotics to the target.)]\n",
            "1860: [(86, 97, Others, These could be two antibiotics or antibiotic with an adjuvant (efflux pump inhibitor, siderophore, etc.) which increases the access of the antibiotics to the target.)]\n",
            "1861: [(36, 48, Antibiotic-Resistant Bacteria, Antimicrobial susceptibility of 150 A. baumannii isolates was assessed by disc diffusion and agar dilution methods.)]\n",
            "1862: [(74, 88, Techniques, Antimicrobial susceptibility of 150 A. baumannii isolates was assessed by disc diffusion and agar dilution methods.)]\n",
            "1863: [(93, 106, Techniques, Antimicrobial susceptibility of 150 A. baumannii isolates was assessed by disc diffusion and agar dilution methods.)]\n",
            "1864: [(47, 63, Antibiotics, Majority of the studies have been conducted on fluoroquinolones and aminoglycosides molecules.)]\n",
            "1865: [(68, 83, Antibiotics, Majority of the studies have been conducted on fluoroquinolones and aminoglycosides molecules.)]\n",
            "1866: [(15, 25, Antibiotics, The antibiotic tobramycin has the property to enhance the action of antimicrobial agents against which the multidrug-resistant Gram-negative bacteria were earlier resistant, and thus potentiating the action of legacy antibiotics.)]\n",
            "1867: [(127, 149, Microbes, The antibiotic tobramycin has the property to enhance the action of antimicrobial agents against which the multidrug-resistant Gram-negative bacteria were earlier resistant, and thus potentiating the action of legacy antibiotics.)]\n",
            "1868: [(138, 152, Microbes, The proportion of positive female samples was higher than that of males. The predominant Gram-negative bacteria were Escherichia coli and Klebsiella spp., while Staphylococcus spp. were the predominant Gram-positive bacteria.)]\n",
            "1869: [(161, 179, Microbes, The proportion of positive female samples was higher than that of males. The predominant Gram-negative bacteria were Escherichia coli and Klebsiella spp., while Staphylococcus spp. were the predominant Gram-positive bacteria.)]\n",
            "1870: [(117, 133, Antibiotic-Resistant Bacteria, The proportion of positive female samples was higher than that of males. The predominant Gram-negative bacteria were Escherichia coli and Klebsiella spp., while Staphylococcus spp. were the predominant Gram-positive bacteria.)]\n",
            "1871: [(105, 123, Microbes, Broth microdilution-based antimicrobial susceptibility testing showed that 39 % of E. coli and 51.5 % of Staphylococcus spp. isolates were multidrug-resistant.)]\n",
            "1872: [(83, 90, Antibiotic-Resistant Bacteria, Broth microdilution-based antimicrobial susceptibility testing showed that 39 % of E. coli and 51.5 % of Staphylococcus spp. isolates were multidrug-resistant.)]\n",
            "1873: [(66, 76, Antibiotics, Specifically, E. coli isolates showed high rates of resistance to ampicillin (40.5 %), ceftazidime (59.5 %), and florfenicol (42.9 %), but limited resistance to amikacin (2.38 %), meropenem (7.14 %), and polymyxin E (7.14 %).)]\n",
            "1874: [(14, 21, Antibiotic-Resistant Bacteria, Specifically, E. coli isolates showed high rates of resistance to ampicillin (40.5 %), ceftazidime (59.5 %), and florfenicol (42.9 %), but limited resistance to amikacin (2.38 %), meropenem (7.14 %), and polymyxin E (7.14 %).)]\n",
            "1875: [(87, 98, Antibiotics, Specifically, E. coli isolates showed high rates of resistance to ampicillin (40.5 %), ceftazidime (59.5 %), and florfenicol (42.9 %), but limited resistance to amikacin (2.38 %), meropenem (7.14 %), and polymyxin E (7.14 %).)]\n",
            "1876: [(113, 124, Antibiotics, Specifically, E. coli isolates showed high rates of resistance to ampicillin (40.5 %), ceftazidime (59.5 %), and florfenicol (42.9 %), but limited resistance to amikacin (2.38 %), meropenem (7.14 %), and polymyxin E (7.14 %).)]\n",
            "1877: [(161, 169, Antibiotics, Specifically, E. coli isolates showed high rates of resistance to ampicillin (40.5 %), ceftazidime (59.5 %), and florfenicol (42.9 %), but limited resistance to amikacin (2.38 %), meropenem (7.14 %), and polymyxin E (7.14 %).)]\n",
            "1878: [(180, 189, Antibiotics, Specifically, E. coli isolates showed high rates of resistance to ampicillin (40.5 %), ceftazidime (59.5 %), and florfenicol (42.9 %), but limited resistance to amikacin (2.38 %), meropenem (7.14 %), and polymyxin E (7.14 %).)]\n",
            "1879: [(204, 213, Antibiotics, Specifically, E. coli isolates showed high rates of resistance to ampicillin (40.5 %), ceftazidime (59.5 %), and florfenicol (42.9 %), but limited resistance to amikacin (2.38 %), meropenem (7.14 %), and polymyxin E (7.14 %).)]\n",
            "1880: [(15, 33, Microbes, In comparison, Staphylococcus spp. showed high rates of resistance to erythromycin (60.6 %), trimethoprim-sulfamethoxazole (54.6 %), and penicillin (45. 5 %), but low resistance rates to vancomycin (6.06 %) and nitrofurantoin (6.06 %).)]\n",
            "1881: [(70, 82, Antibiotics, In comparison, Staphylococcus spp. showed high rates of resistance to erythromycin (60.6 %), trimethoprim-sulfamethoxazole (54.6 %), and penicillin (45. 5 %), but low resistance rates to vancomycin (6.06 %) and nitrofurantoin (6.06 %).)]\n",
            "1882: [(94, 122, Antibiotics, In comparison, Staphylococcus spp. showed high rates of resistance to erythromycin (60.6 %), trimethoprim-sulfamethoxazole (54.6 %), and penicillin (45. 5 %), but low resistance rates to vancomycin (6.06 %) and nitrofurantoin (6.06 %).)]\n",
            "1883: [(137, 147, Antibiotics, In comparison, Staphylococcus spp. showed high rates of resistance to erythromycin (60.6 %), trimethoprim-sulfamethoxazole (54.6 %), and penicillin (45. 5 %), but low resistance rates to vancomycin (6.06 %) and nitrofurantoin (6.06 %).)]\n",
            "1884: [(187, 197, Antibiotics, In comparison, Staphylococcus spp. showed high rates of resistance to erythromycin (60.6 %), trimethoprim-sulfamethoxazole (54.6 %), and penicillin (45. 5 %), but low resistance rates to vancomycin (6.06 %) and nitrofurantoin (6.06 %).)]\n",
            "1885: [(211, 225, Antibiotics, In comparison, Staphylococcus spp. showed high rates of resistance to erythromycin (60.6 %), trimethoprim-sulfamethoxazole (54.6 %), and penicillin (45. 5 %), but low resistance rates to vancomycin (6.06 %) and nitrofurantoin (6.06 %).)]\n",
            "1886: [(23, 39, Microbes, The resistance rate of Escherichia coli strains to cefotaxime was 32.4% (574/1772), and that of Klebsiella pneumoniae strains was 26.1% (181/693).)]\n",
            "1887: [(96, 117, Microbes, The resistance rate of Escherichia coli strains to cefotaxime was 32.4% (574/1772), and that of Klebsiella pneumoniae strains was 26.1% (181/693).)]\n",
            "1888: [(51, 61, Antibiotics, The resistance rate of Escherichia coli strains to cefotaxime was 32.4% (574/1772), and that of Klebsiella pneumoniae strains was 26.1% (181/693).)]\n",
            "1889: [(23, 46, Antibiotic-Resistant Bacteria, And 92.1% (187/203) of Acinetobacter baumannii strains were resistant to both imipenem and meropenem.)]\n",
            "1890: [(78, 86, Antibiotics, And 92.1% (187/203) of Acinetobacter baumannii strains were resistant to both imipenem and meropenem.)]\n",
            "1891: [(91, 100, Antibiotics, And 92.1% (187/203) of Acinetobacter baumannii strains were resistant to both imipenem and meropenem.)]\n",
            "1892: [(, , , The proportions of bacterial species in the blood strains were similar to those in other Asian countries with similar lifestyles.)]\n",
            "1893: [(, , , Urinary tract infections (UTIs), many of which are caused by bacterial pathogens, are some of the most common infections in dogs.)]\n",
            "1894: [(, , , The concepts, developments and challenges in the future use of antibiotic hybrids are discussed here.)]\n",
            "1895: [(, , , To effectively treat UTIs, it is important to identify the predominant bacterial pathogens and their susceptibility to antimicrobial agents.)]\n",
            "1896: [(, , , In this study, we collected 326 samples from cases with UTIs or other urinary system diseases at the China Agricultural University Veterinary Teaching Hospital, Beijing, from 2016-2018.)]\n",
            "1897: [(, , , In total, 129 non-duplicate bacterial isolates were recovered from 103 clinical samples.)]\n",
            "1898: [(59, 81, Microbes, Antibiotic hybrids may have a role as the silver bullet in Gram-negative bacteria to overcome drug resistance as well as extend the spectrum of existing antibiotics.)]\n",
            "1899: [(, , , The emergence and spread of antimicrobial resistance (AMR) among pathogenic bacteria constitute an accelerating crisis for public health.)]\n",
            "1900: [(, , , Antimicrobials, besides antibacterial function, also result in undesirable effects in the microbial populations, including the stimulation of HGT.)]\n",
            "1901: [(, , , Pulsed-field gel electrophoresis (PFGE)-based typing identified 31 PFGE patterns among the 43 E. coli isolates.)]\n",
            "1902: [(, , , The main aim of this narrative review was to present an overview of the current knowledge of the impact of antimicrobials on HGT in bacteria, including the effects of transformation, transduction and conjugation, as well as other less well-studied mechanisms of HGT.)]\n",
            "1903: [(, , , Other mechanisms of HGT have so far been deemed less important in this respect; however, recent discoveries suggest their role may be larger than previously thought, and the review provides an update on the rather limited knowledge currently available regarding the impact of antimicrobial treatment on these processes as well.)]\n",
            "1904: [(, , , The studies conducted so far seem promising; still, a few factors like the precise mechanism of action of AgNPs, their synergistic interaction with biomolecules, and industrial scalability, need to be explored further till effective drug development using green synthesized AgNPs in healthcare systems against AMR is established.)]\n",
            "1905: [(, , , A conclusion from the review is that there is an urgent need to investigate the mechanisms of antimicrobial-induced HGT, since this will be critical for developing new strategies to combat the spread of AMR.)]\n",
            "1906: [(, , , For centuries, plants have been used for a wide variety of purposes, from treating infectious diseases to food preservation and perfume production.)]\n",
            "1907: [(, , , Presently, the increasing resistance of microorganisms to currently used antimicrobials in combination with the appearance of emerging diseases requires the urgent development of new, more effective drugs.)]\n",
            "1908: [(, , , In developed countries it is now increasingly accepted that for the most part such strains are zoonotic in origin and acquire their resistance in the food-animal host before onward transmission to humans through the food chain.)]\n",
            "1909: [(, , , In the present paper, the history, composition, and antimicrobial activities of the basil, oregano, and thyme essential oils are reviewed.)]\n",
            "1910: [(0, 21, Antibiotic-Resistant Bacteria, Staphylococcus aureus, one of the main contagious mastitis pathogens worldwide, is characterized for causing chronic intramammary infections that respond poorly to antimicrobial therapy, disseminating within the herd leading to high economic losses.)]\n",
            "1911: [(, , , Unprincipled use of antibiotics has led to antimicrobial resistance (AMR) against mostly available compounds, and has now become a major cause of concern for the scientific community.)]\n",
            "1912: [(, , , Plants, due to the large biological and structural diversity of their components, constitute a unique and renewable source for the discovery of new antibacterial, antifungal, and antiparasitic compounds.)]\n",
            "1913: [(, , , Genetic similarity amongst the isolates was evaluated by ERIC-PCR. Resistance of A. baumannii to carbapenems increased from 2% (1994-1996) to 73% (2004-2007).)]\n",
            "1914: [(109, 118, Antibiotic-Resistant Bacteria, The aim of this study was to determine the prevalence of phenotypic resistance to antimicrobial agents among S. aureus collected worldwide in the context of bovine intramammary infections between the years 1969-2020.)]\n",
            "1915: [(157, 187, Others, The aim of this study was to determine the prevalence of phenotypic resistance to antimicrobial agents among S. aureus collected worldwide in the context of bovine intramammary infections between the years 1969-2020.)]\n",
            "1916: [(63, 71, Antibiotic-Resistant Bacteria, Furthermore, an increasing number of multiresistant strains of S. typhi are now exhibiting decreased susceptibility to ciprofloxacin, with concomitant treatment failures.)]\n",
            "1917: [(119, 132, Antibiotics, Furthermore, an increasing number of multiresistant strains of S. typhi are now exhibiting decreased susceptibility to ciprofloxacin, with concomitant treatment failures.)]\n",
            "1918: [(66, 76, Antibiotics, The highest overall prevalence of resistant S. aureus was against penicillin (pestimate 0.451, CI95 % 0.415-0.487), followed by clindamycin, erythromycin, and gentamycin (p-estimate = 0.149, 0.085, and 0.069, respectively).)]\n",
            "1919: [(128, 139, Antibiotics, The highest overall prevalence of resistant S. aureus was against penicillin (pestimate 0.451, CI95 % 0.415-0.487), followed by clindamycin, erythromycin, and gentamycin (p-estimate = 0.149, 0.085, and 0.069, respectively).)]\n",
            "1920: [(141, 153, Antibiotics, The highest overall prevalence of resistant S. aureus was against penicillin (pestimate 0.451, CI95 % 0.415-0.487), followed by clindamycin, erythromycin, and gentamycin (p-estimate = 0.149, 0.085, and 0.069, respectively).)]\n",
            "1921: [(159, 169, Antibiotics, The highest overall prevalence of resistant S. aureus was against penicillin (pestimate 0.451, CI95 % 0.415-0.487), followed by clindamycin, erythromycin, and gentamycin (p-estimate = 0.149, 0.085, and 0.069, respectively).)]\n",
            "1922: [(44, 53, Antibiotic-Resistant Bacteria, The highest overall prevalence of resistant S. aureus was against penicillin (pestimate 0.451, CI95 % 0.415-0.487), followed by clindamycin, erythromycin, and gentamycin (p-estimate = 0.149, 0.085, and 0.069, respectively).)]\n",
            "1923: [(51, 71, Microbes, In developed countries antimicrobial resistance in zoonotic salmonellas has been attributed to the injudicious use of antimicrobials in food-producing animals.)]\n",
            "1924: [(0, 9, Antibiotic-Resistant Genes, Ceftiofur and cephalotin presented the lowest overall prevalence of antimicrobial resistance (AMR, p-estimate = 0.020 and 0.015, respectively).)]\n",
            "1925: [(14, 24, Antibiotic-Resistant Genes, Ceftiofur and cephalotin presented the lowest overall prevalence of antimicrobial resistance (AMR, p-estimate = 0.020 and 0.015, respectively).)]\n",
            "1926: [(32, 37, Techniques, Unlike traditional antibiotics, AgNPs exert their action by acting on multiple mechanisms, which make them potential candidates against AMR.)]\n",
            "1927: [(19, 24, Techniques, Green synthesis of AgNPs using various medicinal plants has demonstrated a broader spectrum of action against several microbes in a number of attempts.)]\n",
            "1928: [(149, 173, Techniques, The present paper provides an insight into the scientific studies that have elucidated the positive role of plant extracts/phytochemicals during the green synthesis of AgNPs and their future perspectives.)]\n",
            "1929: [(11, 25, Microbes, Strains of Salmonella spp. with resistance to antimicrobial drugs are now widespread in both developed and developing countries.)]\n",
            "1930: [(274, 287, Antibiotics, Of particular importance since the early 1990s has been a multiresistant strain of Salmonella typhimurium definitive phage type (DT) 104, displaying resistance to up to six commonly used antimicrobials, with about 15% of isolates also exhibiting decreased susceptibility to ciprofloxacin.)]\n",
            "1931: [(83, 105, Microbes, Of particular importance since the early 1990s has been a multiresistant strain of Salmonella typhimurium definitive phage type (DT) 104, displaying resistance to up to six commonly used antimicrobials, with about 15% of isolates also exhibiting decreased susceptibility to ciprofloxacin.)]\n",
            "1932: [(17, 26, Antibiotic-Resistant Genes, Mutations in the gyrA gene in such isolates have been characterised by a PCR LightCycler-based gyrA mutation assay, and at least four different mutations have been identified.)]\n",
            "1933: [(73, 114, Techniques, Mutations in the gyrA gene in such isolates have been characterised by a PCR LightCycler-based gyrA mutation assay, and at least four different mutations have been identified.)]\n",
            "1934: [(, , , Multiple resistance is also being found in other serotypes in several other European countries, and has been associated with treatment failures.)]\n",
            "1935: [(, , , A systematic review was performed according to Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA).)]\n",
            "1936: [(4, 20, Antibiotic-Resistant Bacteria, For Salmonella typhi, multiple drug resistance is now the norm in strains originating in the Indian subcontinent and south-east Asia.)]\n",
            "1937: [(, , , One hundred and fifty-five articles were eligible for quantitative review.)]\n",
            "1938: [(, , , Such multiresistant strains have been responsible for several epidemics and some of these have been associated with contaminated water supplies.)]\n",
            "1939: [(, , , Most of studies included in this meta-analysis were from Europe (88), followed by Asia (56), Latin America (39), Africa (32), North America (26), and Oceania (8).)]\n",
            "1940: [(, , , However, in the past decade, green synthesized silver nanoparticles (AgNPs) have received greater attention for the development of newer therapies as antimicrobials by virtue of their unique physico- chemical properties.)]\n",
            "1941: [(, , , While several studies have shown that mixed-species communities are more tolerant toward antimicrobials than their monospecies counterparts, only limited empirical data are currently available on how interspecies interactions influence resistance development.)]\n",
            "1942: [(, , , No differences in AMR were detected regarding the clinical origin of the isolates (subclinical vs clinical mastitis) for almost all antibiotics evaluated.)]\n",
            "1943: [(, , , The purpose of this review was to describe the research outlining the emergence of resistance in common STI pathogens and new strategies for their treatment and prevention.)]\n",
            "1944: [(, , , Antibiotic resistance is a global public health threat.)]\n",
            "1945: [(, , , We here propose a theoretic framework outlining the potential impact of interspecies social behavior on different aspects of resistance development.)]\n",
            "1946: [(, , , We identify factors by which interspecies interactions might influence resistance evolution and distinguish between their effect on (a) the emergence of a resistant mutant and (b) the spread of this resistance throughout the population.)]\n",
            "1947: [(, , , Water from human activities is collected at wastewater treatment plants where processes often do not sufficiently neutralize antibiotic resistant bacteria and genes, which are further shed into the local environment.)]\n",
            "1948: [(, , , Humans most often become infected by ingesting contaminated food, especially undercooked chicken, but also other sources of bacteria have been described.)]\n",
            "1949: [(, , , Our analysis indicates that considering the social life of bacteria is imperative to the rational design of more effective antibiotic treatment strategies with a minimal hazard for resistance development.)]\n",
            "1950: [(, , , This protocol outlines the steps to conduct a systematic review based on the Population, Exposure, Comparator and Outcome (PECO) framework, aiming at answering the question \"Are antimicrobial-resistant enterobacteriaceae and antimicrobial resistance genes present (O) in air and water samples (P) taken either near or downstream or downwind or down-gradient from wastewater treatment plants (E), as compared to air and water samples taken either further away or upstream or upwind or up-gradient from such wastewater treatment plant (C)?\")]\n",
            "1951: [(, , , Antimicrobial resistance is a growing problem and is a significant public health issue.)]\n",
            "1952: [(, , , Presence of antimicrobial-resistant bacteria and genes will be quantitatively measured by extracting their prevalence or concentration, depending on the reviewed study.)]\n",
            "1953: [(, , , An increasing number of organisms are developing resistance to many of the antimicrobial agents available for treatment of infections in both humans and animals.)]\n",
            "1954: [(, , , First, the antibacterial activity of single concentrations of HICA, calcium hydroxide solution or slurry, chlorhexidine digluconate or acetate was tested against Enterococcus faecalis with and without potential inhibitors: dentine powder (DP), hydroxyapatite or bovine serum albumin, in a low concentration of peptone water.)]\n",
            "1955: [(, , , These resistant organisms often result in greater disease severity, longer hospitalization, and increased care and treatment costs.)]\n",
            "1956: [(0, 20, Microbes, Campylobacter jejuni and Campylobacter coli are recognized as the most common causative agents of bacterial gastroenteritis in the world.)]\n",
            "1957: [(25, 43, Microbes, Campylobacter jejuni and Campylobacter coli are recognized as the most common causative agents of bacterial gastroenteritis in the world.)]\n",
            "1958: [(62, 75, Antibiotic-Resistant Bacteria, The most common antimicrobial agents used in the treatment of Campylobacter infections are macrolides, such as erythromycin, and fluoroquinolones, such as ciprofloxacin.)]\n",
            "1959: [(155, 168, Antibiotics, The most common antimicrobial agents used in the treatment of Campylobacter infections are macrolides, such as erythromycin, and fluoroquinolones, such as ciprofloxacin.)]\n",
            "1960: [(91, 101, Antibiotics, The most common antimicrobial agents used in the treatment of Campylobacter infections are macrolides, such as erythromycin, and fluoroquinolones, such as ciprofloxacin.)]\n",
            "1961: [(111, 123, Antibiotics, The most common antimicrobial agents used in the treatment of Campylobacter infections are macrolides, such as erythromycin, and fluoroquinolones, such as ciprofloxacin.)]\n",
            "1962: [(129, 145, Antibiotics, The most common antimicrobial agents used in the treatment of Campylobacter infections are macrolides, such as erythromycin, and fluoroquinolones, such as ciprofloxacin.)]\n",
            "1963: [(180, 193, Microbes, In this review, the patterns of emerging resistance to the antimicrobial agents useful in treatment of the disease are presented and the mechanisms of resistance to these drugs in Campylobacter are discussed.)]\n",
            "1964: [(186, 197, Antibiotic-Resistant Bacteria, Relative viable counts were determined by culture at 1, 24 and 48 h. In the second set of experiments, the activity of three concentrations of HICA was evaluated against two isolates of E. faecalis with and without potential inhibitors in nutrient-rich thioglycollate broth using a modification of a standard microdilution method.)]\n",
            "1965: [(309, 329, Techniques, Relative viable counts were determined by culture at 1, 24 and 48 h. In the second set of experiments, the activity of three concentrations of HICA was evaluated against two isolates of E. faecalis with and without potential inhibitors in nutrient-rich thioglycollate broth using a modification of a standard microdilution method.)]\n",
            "1966: [(, , , Campylobacteriosis is normally a self-limiting disease. Antimicrobial treatment is needed only in patients with more severe disease and in those who are immunologically compromised.)]\n",
            "1967: [(0, 13, Antibiotics, Tetracyclines have been suggested as an alternative choice in the treatment of clinical campylobacteriosis but in practice are not often used.)]\n",
            "1968: [(, , , However, during the past few decades an increasing number of resistant Campylobacter isolates have developed resistance to fluoroquinolones and other antimicrobials such as macrolides, aminoglycosides, and beta-lactams.)]\n",
            "1969: [(168, 181, Antibiotic-Resistant Bacteria, Trends in antimicrobial resistance have shown a clear correlation between use of antibiotics in the veterinary medicine and animal production and resistant isolates of Campylobacter in humans.)]\n",
            "1970: [(, , , Differences in the method for testing AMR (disc diffusion method vs minimum inhibitory concentration) and type of study design for monitoring AMR were detected underscoring the importance of these variables as critical factors to enable comparisons for evaluating emergence of AMR.)]\n",
            "1971: [(, , , Bacteria are social organisms that commonly live in dense communities surrounded by a multitude of other species.)]\n",
            "1972: [(, , , Antimicrobial resistance in sexually acquired infection (STI) pathogens is an important global public health threat.)]\n",
            "1973: [(, , , The competitive and cooperative interactions between these species not only shape the bacterial communities but also influence their susceptibility to antimicrobials.)]\n",
            "1974: [(, , , There is an urgent need for novel STI treatment and prevention strategies to tackle the rising incidence of STIs in high-income settings and the static progress in low- and middle-income settings over the past decade.)]\n",
            "1975: [(, , , Diabetic foot ulcers are often complicated by infection. Among pathogens, Staphylococcus aureus predominates.)]\n",
            "1976: [(, , , The prevalence of methicillin-resistant S. aureus (MRSA) in infected foot ulcers is 15-30% and there is an alarming trend for increase in many countries.)]\n",
            "1977: [(, , , The diagnostic platform presented here with fluidics and combined immunoassays allows for pathogen isolation within 5 min and identification in as little as 15 min to 1 h, to help guide the decision for additional testing, optimally only on positive samples, such as multiplexed or resistance gene assays.)]\n",
            "1978: [(, , , Infections due to multidrug-resistant (MDR) bacteria are burdened by high mortality rates.)]\n",
            "1979: [(, , , The development of new compounds to face the global threat of resistance is urgently needed.)]\n",
            "1980: [(, , , Combination regimens including \"old\" high-dose antimicrobials are currently limited by the risk of toxicity, resistance selection, and reduced efficacy.)]\n",
            "1981: [(, , , Following the Infectious Diseases Society of America call to develop 10 new antibacterials by 2020, new molecules are currently under development or have become available for use in clinical practice.)]\n",
            "1982: [(0, 11, Antibiotics, Methicillin-resistant Staphylococcus aureus bacteremia in patients in intensive care units is associated with significant morbidity and mortality.)]\n",
            "1983: [(22, 43, Antibiotic-Resistant Bacteria, Methicillin-resistant Staphylococcus aureus bacteremia in patients in intensive care units is associated with significant morbidity and mortality.)]\n",
            "1984: [(, , , Foreign-body sources of infection should be removed or replaced in the majority of cases, and debridement of infectious foci should be undertaken.)]\n",
            "1985: [(120, 129, Antibiotic-Resistant Bacteria, Particular attention should be given to evaluation for the presence of cardiac involvement because inadequately managed S. aureus endocarditis is life threatening.)]\n",
            "1986: [(257, 281, Antibiotic-Resistant Genes, This method preserves bacterial cells allowing extraction and amplification of 16S ribosomal RNA genes and antibiotic resistance genes, as was demonstrated through identification and quantitation of two strains of E. coli, resistant and nonresistant due to β-lactamase cefotaximase genes.)]\n",
            "1987: [(79, 96, Gene/Proteins, This method preserves bacterial cells allowing extraction and amplification of 16S ribosomal RNA genes and antibiotic resistance genes, as was demonstrated through identification and quantitation of two strains of E. coli, resistant and nonresistant due to β-lactamase cefotaximase genes.)]\n",
            "1988: [(28, 40, Techniques, The sensitivities for these immunoassays were 83, 95, and 92% for the prediction of S. aureus, P. aeruginosa, and E. coli or K. pneumoniae positive culture, respectively, while specificities were 85, 92, and 97%.)]\n",
            "1989: [(125, 138, Microbes, The sensitivities for these immunoassays were 83, 95, and 92% for the prediction of S. aureus, P. aeruginosa, and E. coli or K. pneumoniae positive culture, respectively, while specificities were 85, 92, and 97%.)]\n",
            "1990: [(114, 121, Antibiotic-Resistant Bacteria, The sensitivities for these immunoassays were 83, 95, and 92% for the prediction of S. aureus, P. aeruginosa, and E. coli or K. pneumoniae positive culture, respectively, while specificities were 85, 92, and 97%.)]\n",
            "1991: [(84, 93, Antibiotic-Resistant Bacteria, The sensitivities for these immunoassays were 83, 95, and 92% for the prediction of S. aureus, P. aeruginosa, and E. coli or K. pneumoniae positive culture, respectively, while specificities were 85, 92, and 97%.)]\n",
            "1992: [(95, 108, Antibiotic-Resistant Bacteria, The sensitivities for these immunoassays were 83, 95, and 92% for the prediction of S. aureus, P. aeruginosa, and E. coli or K. pneumoniae positive culture, respectively, while specificities were 85, 92, and 97%.)]\n",
            "1993: [(50, 54, Antibiotic-Resistant Bacteria, There are also data that recognize new strains of MRSA that are resistant to vancomycin.)]\n",
            "1994: [(77, 87, Antibiotics, There are also data that recognize new strains of MRSA that are resistant to vancomycin.)]\n",
            "1995: [(13, 27, Techniques, The risk for MRSA isolation increases in the presence of osteomyelitis, nasal carriage of MRSA, prior use of antibacterials or hospitalization, larger ulcer size and longer duration of the ulcer.)]\n",
            "1996: [(90, 94, Antibiotic-Resistant Bacteria, The risk for MRSA isolation increases in the presence of osteomyelitis, nasal carriage of MRSA, prior use of antibacterials or hospitalization, larger ulcer size and longer duration of the ulcer.)]\n",
            "1997: [(85, 89, Antibiotic-Resistant Bacteria, The need for amputation and surgical debridement increases in patients infected with MRSA.)]\n",
            "1998: [(0, 11, Antibiotics, Dalbavancin, tigecycline and ceftobiprole are newer promising antimicrobial agents active against MRSA that may also have a role in the treatment of foot infections if more data on their efficacy and safety become available.)]\n",
            "1999: [(13, 24, Antibiotics, Dalbavancin, tigecycline and ceftobiprole are newer promising antimicrobial agents active against MRSA that may also have a role in the treatment of foot infections if more data on their efficacy and safety become available.)]\n",
            "2000: [(29, 41, Antibiotics, Dalbavancin, tigecycline and ceftobiprole are newer promising antimicrobial agents active against MRSA that may also have a role in the treatment of foot infections if more data on their efficacy and safety become available.)]\n",
            "2001: [(172, 199, Antibiotics, We have reviewed safety characteristics and tolerability of old antimicrobials that are currently employed in combination regimens as well as new antimicrobials, including beta-lactams/beta-lactamase inhibitors, new cephalosporins, quinolones, and aminoglycosides.)]\n",
            "2002: [(232, 242, Antibiotics, We have reviewed safety characteristics and tolerability of old antimicrobials that are currently employed in combination regimens as well as new antimicrobials, including beta-lactams/beta-lactamase inhibitors, new cephalosporins, quinolones, and aminoglycosides.)]\n",
            "2003: [(216, 230, Antibiotics, We have reviewed safety characteristics and tolerability of old antimicrobials that are currently employed in combination regimens as well as new antimicrobials, including beta-lactams/beta-lactamase inhibitors, new cephalosporins, quinolones, and aminoglycosides.)]\n",
            "2004: [(248, 263, Antibiotics, We have reviewed safety characteristics and tolerability of old antimicrobials that are currently employed in combination regimens as well as new antimicrobials, including beta-lactams/beta-lactamase inhibitors, new cephalosporins, quinolones, and aminoglycosides.)]\n",
            "2005: [(, , , Empiric antibacterials are often employed though frequency of bacterial coinfection superinfection is debated and concerns raised about selection of bacterial antimicrobial resistance.)]\n",
            "2006: [(, , , Objectives were to determine frequency of culture positivity, risk factors for and outcomes of positive cultures, and timing of antimicrobial resistance development.)]\n",
            "2007: [(91, 114, Antibiotic-Resistant Bacteria, The results of the study suggest that the high prevalence of carbapenem resistance amongst Acinetobacter baumannii organisms in this institution is not caused by the spread of a predominant clone.)]\n",
            "2008: [(61, 71, Antibiotics, The results of the study suggest that the high prevalence of carbapenem resistance amongst Acinetobacter baumannii organisms in this institution is not caused by the spread of a predominant clone.)]\n",
            "2009: [(, , , The global prevalence of antibiotic-resistant bacteria has increased the risk of dangerous infections, requiring rapid diagnosis and treatment.)]\n",
            "2010: [(, , , This work reinforces the importance monitoring antimicrobial susceptibility rates.)]\n",
            "2011: [(, , , The standard method for diagnosis of bacterial infections remains dependent on slow culture-based methods, carried out in central laboratories, not easily extensible to rapid identification of organisms, and thus not optimal for timely treatments at the point-of-care (POC).)]\n",
            "2012: [(62, 98, Techniques, Here, we demonstrate rapid detection of bacteria by combining electrochemical immunoassays (EC-IA) for pathogen identification with confirmatory quantitative mass spectral immunoassays (MS-IA) based on signal ion emission reactive release amplification (SIERRA) nanoparticles with unique mass labels.)]\n",
            "2013: [(158, 192, Techniques, Here, we demonstrate rapid detection of bacteria by combining electrochemical immunoassays (EC-IA) for pathogen identification with confirmatory quantitative mass spectral immunoassays (MS-IA) based on signal ion emission reactive release amplification (SIERRA) nanoparticles with unique mass labels.)]\n",
            "2014: [(202, 261, Techniques, Here, we demonstrate rapid detection of bacteria by combining electrochemical immunoassays (EC-IA) for pathogen identification with confirmatory quantitative mass spectral immunoassays (MS-IA) based on signal ion emission reactive release amplification (SIERRA) nanoparticles with unique mass labels.)]\n",
            "2015: [(, , , Prompt clinical attention is essential to ensure good outcomes, including identification and management of the source of infection and any associated complications.)]\n",
            "2016: [(292, 308, Antibiotic-Resistant Bacteria, This diagnostic method uses compatible reagents for all involved assays and standard fluidics for automatic sample preparation at POC. EC-IA, based on alkaline phosphatase-conjugated pathogen-specific antibodies, quantified down to 104 bacteria per sample when testing Staphylococcus aureus, Escherichia coli, and Pseudomonas aeruginosa lysates.)]\n",
            "2017: [(314, 336, Antibiotic-Resistant Bacteria, This diagnostic method uses compatible reagents for all involved assays and standard fluidics for automatic sample preparation at POC. EC-IA, based on alkaline phosphatase-conjugated pathogen-specific antibodies, quantified down to 104 bacteria per sample when testing Staphylococcus aureus, Escherichia coli, and Pseudomonas aeruginosa lysates.)]\n",
            "2018: [(269, 290, Antibiotic-Resistant Bacteria, This diagnostic method uses compatible reagents for all involved assays and standard fluidics for automatic sample preparation at POC. EC-IA, based on alkaline phosphatase-conjugated pathogen-specific antibodies, quantified down to 104 bacteria per sample when testing Staphylococcus aureus, Escherichia coli, and Pseudomonas aeruginosa lysates.)]\n",
            "2019: [(94, 103, Antibiotic-Resistant Bacteria, EC-IA quantitation was also obtained for wound samples. The MS-IA using nanoparticles against S. aureus, E. coli, Klebsiella pneumoniae, and P. aeruginosa allowed selective quantitation of ∼105 bacteria per sample.)]\n",
            "2020: [(105, 112, Antibiotic-Resistant Bacteria, EC-IA quantitation was also obtained for wound samples. The MS-IA using nanoparticles against S. aureus, E. coli, Klebsiella pneumoniae, and P. aeruginosa allowed selective quantitation of ∼105 bacteria per sample.)]\n",
            "2021: [(114, 135, Antibiotic-Resistant Bacteria, EC-IA quantitation was also obtained for wound samples. The MS-IA using nanoparticles against S. aureus, E. coli, Klebsiella pneumoniae, and P. aeruginosa allowed selective quantitation of ∼105 bacteria per sample.)]\n",
            "2022: [(141, 154, Antibiotic-Resistant Bacteria, EC-IA quantitation was also obtained for wound samples. The MS-IA using nanoparticles against S. aureus, E. coli, Klebsiella pneumoniae, and P. aeruginosa allowed selective quantitation of ∼105 bacteria per sample.)]\n",
            "2023: [(, , , We evaluated sputum bacterial and fungal growth from 165 intubated COVID-19 pneumonia patients.)]\n",
            "2024: [(, , , Selection of an appropriate antibiotic regimen is also an essential factor for optimal management.)]\n",
            "2025: [(, , , Finally, the combined immunoassays were compared against culture using remnant deidentified patient urine samples.)]\n",
            "2026: [(, , , Particularly worrying is that declines in clinical success of firstline CDI antimicrobials coincide with the spread of strains that are more resistant to these drugs.)]\n",
            "2027: [(, , , Yet, there is still much to learn regarding the prevalence of genetic elements in clinical isolates, their molecular mechanisms, and the extent to which this information can be translated to develop molecular diagnostics that improve antimicrobial prescribing and antimicrobial stewardship approaches for CDI.)]\n",
            "2028: [(, , , Gram-negative bacteria are responsible for a large proportion of antimicrobial-resistant infections in humans and animals.)]\n",
            "2029: [(, , , Among this class of bacteria are also some of the most successful environmental organisms.)]\n",
            "2030: [(, , , Part of this success is their adaptability to a variety of different niches, their intrinsic resistance to antimicrobial drugs and their ability to rapidly acquire resistance mechanisms.)]\n",
            "2031: [(0, 21, Antibiotic-Resistant Bacteria, Neisseria gonorrhoeae is evolving into a superbug with resistance to previously and currently recommended antimicrobials for treatment of gonorrhea, which is a major public health concern globally.)]\n",
            "2032: [(109, 122, Antibiotics, After six years of use, 94%-100% of Enterobacteriaceae (except Providencia stuartii) remained susceptible to ciprofloxacin but susceptibility of P. stuartii, A. baumannii and P. aeruginosa declined from 100% to 46%, 66% and 84%, respectively.)]\n",
            "2033: [(175, 188, Antibiotic-Resistant Bacteria, After six years of use, 94%-100% of Enterobacteriaceae (except Providencia stuartii) remained susceptible to ciprofloxacin but susceptibility of P. stuartii, A. baumannii and P. aeruginosa declined from 100% to 46%, 66% and 84%, respectively.)]\n",
            "2034: [(145, 156, Antibiotic-Resistant Bacteria, After six years of use, 94%-100% of Enterobacteriaceae (except Providencia stuartii) remained susceptible to ciprofloxacin but susceptibility of P. stuartii, A. baumannii and P. aeruginosa declined from 100% to 46%, 66% and 84%, respectively.)]\n",
            "2035: [(175, 188, Antibiotic-Resistant Bacteria, After six years of use, 94%-100% of Enterobacteriaceae (except Providencia stuartii) remained susceptible to ciprofloxacin but susceptibility of P. stuartii, A. baumannii and P. aeruginosa declined from 100% to 46%, 66% and 84%, respectively.)]\n",
            "2036: [(36, 54, Antibiotics, After six years of use, 94%-100% of Enterobacteriaceae (except Providencia stuartii) remained susceptible to ciprofloxacin but susceptibility of P. stuartii, A. baumannii and P. aeruginosa declined from 100% to 46%, 66% and 84%, respectively.)]\n",
            "2037: [(158, 170, Antibiotic-Resistant Bacteria, After six years of use, 94%-100% of Enterobacteriaceae (except Providencia stuartii) remained susceptible to ciprofloxacin but susceptibility of P. stuartii, A. baumannii and P. aeruginosa declined from 100% to 46%, 66% and 84%, respectively.)]\n",
            "2038: [(69, 77, Antibiotics, Oxacillin-resistant staphylococci were considered to be resistant to imipenem and all beta-lactams.)]\n",
            "2039: [(86, 98, Antibiotics, Oxacillin-resistant staphylococci were considered to be resistant to imipenem and all beta-lactams.)]\n",
            "2040: [(20, 33, Antibiotic-Resistant Bacteria, Oxacillin-resistant staphylococci were considered to be resistant to imipenem and all beta-lactams.)]\n",
            "2041: [(, , , For thousands of years, traditional Chinese medicine (TCM) has provided natural and unique advantages in the treatment of infectious diseases.)]\n",
            "2042: [(90, 111, Antibiotic-Resistant Bacteria, There were no quinolone-resistant staphylococci observed in 1986, but susceptibilities of Staphylococcus aureus, Staphylococcus epidermidis and Staphylococcus haemolyticus to ciprofloxacin decreased to 85-93% for oxacillin-susceptible strains and to 7-39% for oxacillin-resistant strains.)]\n",
            "2043: [(175, 188, Antibiotics, There were no quinolone-resistant staphylococci observed in 1986, but susceptibilities of Staphylococcus aureus, Staphylococcus epidermidis and Staphylococcus haemolyticus to ciprofloxacin decreased to 85-93% for oxacillin-susceptible strains and to 7-39% for oxacillin-resistant strains.)]\n",
            "2044: [(260, 269, Antibiotics, There were no quinolone-resistant staphylococci observed in 1986, but susceptibilities of Staphylococcus aureus, Staphylococcus epidermidis and Staphylococcus haemolyticus to ciprofloxacin decreased to 85-93% for oxacillin-susceptible strains and to 7-39% for oxacillin-resistant strains.)]\n",
            "2045: [(213, 222, Antibiotics, There were no quinolone-resistant staphylococci observed in 1986, but susceptibilities of Staphylococcus aureus, Staphylococcus epidermidis and Staphylococcus haemolyticus to ciprofloxacin decreased to 85-93% for oxacillin-susceptible strains and to 7-39% for oxacillin-resistant strains.)]\n",
            "2046: [(144, 171, Antibiotic-Resistant Bacteria, There were no quinolone-resistant staphylococci observed in 1986, but susceptibilities of Staphylococcus aureus, Staphylococcus epidermidis and Staphylococcus haemolyticus to ciprofloxacin decreased to 85-93% for oxacillin-susceptible strains and to 7-39% for oxacillin-resistant strains.)]\n",
            "2047: [(113, 139, Antibiotic-Resistant Bacteria, There were no quinolone-resistant staphylococci observed in 1986, but susceptibilities of Staphylococcus aureus, Staphylococcus epidermidis and Staphylococcus haemolyticus to ciprofloxacin decreased to 85-93% for oxacillin-susceptible strains and to 7-39% for oxacillin-resistant strains.)]\n",
            "2048: [(54, 57, Techniques, Therefore, it is important to develop further and use TCM to treat clinical infections caused by drug-resistant bacteria.)]\n",
            "2049: [(, , , The results indicate a continuing need for resistance surveillance and rational use of antimicrobial agents.)]\n",
            "2050: [(0, 8, Antibiotics, Imipenem-quinolone cross-resistance was observed for staphylococci but not for P. aeruginosa.)]\n",
            "2051: [(9, 18, Antibiotics, Imipenem-quinolone cross-resistance was observed for staphylococci but not for P. aeruginosa.)]\n",
            "2052: [(53, 66, Microbes, Imipenem-quinolone cross-resistance was observed for staphylococci but not for P. aeruginosa.)]\n",
            "2053: [(79, 92, Microbes, Imipenem-quinolone cross-resistance was observed for staphylococci but not for P. aeruginosa.)]\n",
            "2054: [(45, 69, Antibiotic-Resistant Bacteria, The evolution of antimicrobial resistance in Clostridioides difficile has markedly shaped its epidemiology and detrimentally impacted patient care.)]\n",
            "2055: [(91, 108, Antibiotic-Resistant Bacteria, Mechanisms of resistance to quinolones were investigated in two in vitro DNA gyrase mutant Coxiella burnetii strains.)]\n",
            "2056: [(0, 12, Microbes, C. difficile exhibits resistance to multiple classes of antimicrobials, due to accumulation of horizontally acquired resistance genes and de novo mutations to drug targets.)]\n",
            "2057: [(16, 26, Antibiotics, Accumulation of pefloxacin was found to be lower in resistant strains than in susceptible strains.)]\n",
            "2058: [(145, 158, Techniques, Thus, this perspective discusses current understanding and knowledge gaps of antimicrobial resistance mechanisms in C. difficile, emphasizing on CDI therapies.)]\n",
            "2059: [(116, 128, Microbes, Thus, this perspective discusses current understanding and knowledge gaps of antimicrobial resistance mechanisms in C. difficile, emphasizing on CDI therapies.)]\n",
            "2060: [(66, 77, Antibiotics, Agents that are potentially active against MDR Gram-negatives are ceftozolane/tazobactam, new carbapenems and cephalosporins, the combination of avibactam with ceftazidime, and plazomicin.)]\n",
            "2061: [(78, 88, Antibiotics, Agents that are potentially active against MDR Gram-negatives are ceftozolane/tazobactam, new carbapenems and cephalosporins, the combination of avibactam with ceftazidime, and plazomicin.)]\n",
            "2062: [(94, 105, Antibiotics, Agents that are potentially active against MDR Gram-negatives are ceftozolane/tazobactam, new carbapenems and cephalosporins, the combination of avibactam with ceftazidime, and plazomicin.)]\n",
            "2063: [(110, 124, Antibiotics, Agents that are potentially active against MDR Gram-negatives are ceftozolane/tazobactam, new carbapenems and cephalosporins, the combination of avibactam with ceftazidime, and plazomicin.)]\n",
            "2064: [(160, 171, Antibiotics, Agents that are potentially active against MDR Gram-negatives are ceftozolane/tazobactam, new carbapenems and cephalosporins, the combination of avibactam with ceftazidime, and plazomicin.)]\n",
            "2065: [(145, 154, Antibiotics, Agents that are potentially active against MDR Gram-negatives are ceftozolane/tazobactam, new carbapenems and cephalosporins, the combination of avibactam with ceftazidime, and plazomicin.)]\n",
            "2066: [(177, 187, Antibiotics, Agents that are potentially active against MDR Gram-negatives are ceftozolane/tazobactam, new carbapenems and cephalosporins, the combination of avibactam with ceftazidime, and plazomicin.)]\n",
            "2067: [(, , , Further data from clinical trials and post-marketing studies for drugs targeting MDR pathogens are crucial to confirm their efficacy and safety.)]\n",
            "2068: [(71, 84, Antibiotics, We have continuously monitored the in-vitro activities of imipenem and ciprofloxacin against large numbers of non-fastidious clinical isolates.)]\n",
            "2069: [(64, 78, Antibiotic-Resistant Bacteria, By understanding the evolution, emergence, and spread of AMR in N. gonorrhoeae, including its molecular and phenotypic mechanisms, resistance to antimicrobials used clinically can be anticipated, future methods for genetic testing for AMR might permit region-specific and tailor-made antimicrobial therapy, and the design of novel antimicrobials to circumvent the resistance problems can be undertaken more rationally.)]\n",
            "2070: [(60, 83, Antibiotic-Resistant Bacteria, After eight years of use, 97-100% of Enterobacteriaceae and Acinetobacter baumannii remained susceptible to imipenem, but susceptibility of Pseudomonas aeruginosa declined from 100% to 91%.)]\n",
            "2071: [(37, 55, Antibiotic-Resistant Bacteria, After eight years of use, 97-100% of Enterobacteriaceae and Acinetobacter baumannii remained susceptible to imipenem, but susceptibility of Pseudomonas aeruginosa declined from 100% to 91%.)]\n",
            "2072: [(108, 116, Antibiotics, After eight years of use, 97-100% of Enterobacteriaceae and Acinetobacter baumannii remained susceptible to imipenem, but susceptibility of Pseudomonas aeruginosa declined from 100% to 91%.)]\n",
            "2073: [(140, 162, Antibiotic-Resistant Bacteria, After eight years of use, 97-100% of Enterobacteriaceae and Acinetobacter baumannii remained susceptible to imipenem, but susceptibility of Pseudomonas aeruginosa declined from 100% to 91%.)]\n",
            "2074: [(, , , This review focuses on the history and evolution of gonorrhea treatment regimens and emerging resistance to them, on genetic and phenotypic determinants of gonococcal resistance to previously and currently recommended antimicrobials, including biological costs or benefits; and on crucial actions and future advances necessary to detect and treat resistant gonococcal strains and, ultimately, retain gonorrhea as a treatable infection.)]\n",
            "2075: [(, , , In recent years, the extensive use of antibiotics worldwide has led to an increase in the number of drug-resistant bacterial strains, thus resulting in an increasingly severe degree of bacterial resistance.)]\n",
            "2076: [(0, 21, Antibiotic-Resistant Bacteria, Enterococcus faecalis has remained susceptible to imipenem and the modal MIC of ciprofloxacin has remained 1 mg/L; however, susceptibility to ciprofloxacin 2 mg/L decreased from 94% to 64%.)]\n",
            "2077: [(80, 93, Antibiotics, Enterococcus faecalis has remained susceptible to imipenem and the modal MIC of ciprofloxacin has remained 1 mg/L; however, susceptibility to ciprofloxacin 2 mg/L decreased from 94% to 64%.)]\n",
            "2078: [(142, 155, Antibiotics, Enterococcus faecalis has remained susceptible to imipenem and the modal MIC of ciprofloxacin has remained 1 mg/L; however, susceptibility to ciprofloxacin 2 mg/L decreased from 94% to 64%.)]\n",
            "2079: [(, , , A literature search was performed using PubMed, Web of Science, Google Scholar, and the China National Knowledge Infrastructure databases.)]\n",
            "2080: [(, , , The articles were analyzed to extract information on the antimicrobial effects of Chinese herbal medicines, compounded Chinese medicines, monomeric compounds of herbal origin, and the combined use of Chinese medicine and antimicrobial drugs and to determine the synergistic effect of the combination of Chinese medicine and antibiotics, as well as investigate the possibility of restoring the antibiotic sensitivity of drug-resistant strains.)]\n",
            "2081: [(, , , However, the molecular mechanisms underlying the survival of biofilm cells are still not completely understood. \\)]\n",
            "2082: [(, , , Pre-incubation with carbonyl cyanide m-chlorophenylhydrazone had no apparent effect on intracellular quinolone fluorescence, indicating that an active efflux mechanism is not responsible for the decrease of pefloxacin accumulation.)]\n",
            "2083: [(, , , No differences were detected between the strains after outer membrane protein analysis.)]\n",
            "2084: [(, , , Decreased accumulation was found in all strains at acidic pH. It is possible that, apart from the mutation-induced mechanism of resistance, a pH-related mechanism is involved in the drug accumulation process.)]\n",
            "2085: [(, , , As polymyxins resistance emerges, there is an urgent need to develop effective antimicrobial agents capable of mitigating MDR.)]\n",
            "2086: [(, , , Because of their potent broad spectrum activity and absence of transferable mechanism of resistance or inactivating enzymes, it was hoped that clinical resistance to this useful group of drugs would not occur.)]\n",
            "2087: [(, , , In this review we discuss three mechanisms that play an important role in biofilm survival: (i) biofilm-specific protection against oxidative stress; (ii) biofilm-specific expression of efflux pumps; and (iii) protection provided by matrix polysaccharides.)]\n",
            "2088: [(, , , In addition, we give an overview of the data that suggests that these mechanisms may not be independent.)]\n",
            "2089: [(, , , Twenty seven (90.0%) IPM-resistant strains showed cross resistant to panipenem (PAPM), and 12 strains (44.4%) out of the 27 strains showed high susceptibility to MEPM. P. aeruginosa becomes resistant to IPM and PAPM only by the decrease in the outer membrane permeability of these carbapenems.)]\n",
            "2090: [(, , , In contrast, P. aeruginosa becomes equally resistant to MEPM by concurrent occurrence of the increase in the efflux of the antibiotics and the decrease in the outer membrane permeability of the antibiotics.)]\n",
            "2091: [(18, 28, Antibiotics, Proper use of the quinolones is also proposed to prevent emergence of the bacterial resistance.)]\n",
            "2092: [(118, 129, Antibiotics, The susceptibility of 260 strains of Pseudomonas aeruginosa to several antibiotics and the mechanism of resistance to carbapenems were investigated.)]\n",
            "2093: [(37, 59, Antibiotic-Resistant Bacteria, The susceptibility of 260 strains of Pseudomonas aeruginosa to several antibiotics and the mechanism of resistance to carbapenems were investigated.)]\n",
            "2094: [(76, 85, Antibiotics, The number of strains of P. aeruginosa moderately resistant or resistant to ofloxacin, ceftazidime and imipenem (IPM) were 76 (29.2%), 31 (11.9%) and 30 (11.5%), respectively.)]\n",
            "2095: [(87, 98, Antibiotics, The number of strains of P. aeruginosa moderately resistant or resistant to ofloxacin, ceftazidime and imipenem (IPM) were 76 (29.2%), 31 (11.9%) and 30 (11.5%), respectively.)]\n",
            "2096: [(103, 111, Antibiotics, The number of strains of P. aeruginosa moderately resistant or resistant to ofloxacin, ceftazidime and imipenem (IPM) were 76 (29.2%), 31 (11.9%) and 30 (11.5%), respectively.)]\n",
            "2097: [(0, 16, Antibiotics, Fluoroquinolones have some of the properties of an 'ideal' anti-microbial agent.)]\n",
            "2098: [(105, 115, Antibiotics, However, over the years, due to intense selective pressure and relative lack of potency of the available quinolones against some strains, bacteria have evolved at least two mechanisms of resistance: (i) alteration of molecular targets, and (ii) reduction of drug accumulation. DNA gyrase and topoisomerase IV are the two molecular targets of fluoroquinolones.)]\n",
            "2099: [(342, 358, Antibiotics, However, over the years, due to intense selective pressure and relative lack of potency of the available quinolones against some strains, bacteria have evolved at least two mechanisms of resistance: (i) alteration of molecular targets, and (ii) reduction of drug accumulation. DNA gyrase and topoisomerase IV are the two molecular targets of fluoroquinolones.)]\n",
            "2100: [(277, 287, Gene/Proteins, However, over the years, due to intense selective pressure and relative lack of potency of the available quinolones against some strains, bacteria have evolved at least two mechanisms of resistance: (i) alteration of molecular targets, and (ii) reduction of drug accumulation. DNA gyrase and topoisomerase IV are the two molecular targets of fluoroquinolones.)]\n",
            "2101: [(292, 308, Gene/Proteins, However, over the years, due to intense selective pressure and relative lack of potency of the available quinolones against some strains, bacteria have evolved at least two mechanisms of resistance: (i) alteration of molecular targets, and (ii) reduction of drug accumulation. DNA gyrase and topoisomerase IV are the two molecular targets of fluoroquinolones.)]\n",
            "2102: [(68, 82, Antibiotic-Resistant Bacteria, The possibility that both mechanisms are taken place concurrently in P. aeruginosa is considered to be low, and it was also supported by the results of the present study.)]\n",
            "2103: [(52, 62, Antibiotics, An efflux pump effective in pumping out hydrophilic quinolones has been described.)]\n",
            "2104: [(157, 166, Antibiotics, The incidence of human Campylobacter jejuni and C. coli infections has increased markedly in many parts of the world in the last decade as has the number of quinolone-resistant and, to a lesser extent, macrolide-resistant Campylobacter strains causing infections.)]\n",
            "2105: [(202, 211, Antibiotics, The incidence of human Campylobacter jejuni and C. coli infections has increased markedly in many parts of the world in the last decade as has the number of quinolone-resistant and, to a lesser extent, macrolide-resistant Campylobacter strains causing infections.)]\n",
            "2106: [(48, 55, Antibiotic-Resistant Bacteria, The incidence of human Campylobacter jejuni and C. coli infections has increased markedly in many parts of the world in the last decade as has the number of quinolone-resistant and, to a lesser extent, macrolide-resistant Campylobacter strains causing infections.)]\n",
            "2107: [(23, 43, Antibiotic-Resistant Bacteria, The incidence of human Campylobacter jejuni and C. coli infections has increased markedly in many parts of the world in the last decade as has the number of quinolone-resistant and, to a lesser extent, macrolide-resistant Campylobacter strains causing infections.)]\n",
            "2108: [(222, 235, Antibiotic-Resistant Bacteria, The incidence of human Campylobacter jejuni and C. coli infections has increased markedly in many parts of the world in the last decade as has the number of quinolone-resistant and, to a lesser extent, macrolide-resistant Campylobacter strains causing infections.)]\n",
            "2109: [(76, 84, Others, We demonstrate that these mechanisms are found both in bacterial and fungal biofilms and are often surprisingly similar between distantly related organisms.)]\n",
            "2110: [(46, 71, Others, Polymyxins remain the last line treatment for multidrug-resistant (MDR) infections.)]\n",
            "2111: [(0, 10, Antibiotics, Polymyxins remain the last line treatment for multidrug-resistant (MDR) infections.)]\n",
            "2112: [(4, 20, Antibiotics, New fluoroquinolones have potent and broad antimicrobial activity and spectra, respectively, against gram-positive and -negative bacteria including P. aeruginosa.)]\n",
            "2113: [(101, 114, Microbes, New fluoroquinolones have potent and broad antimicrobial activity and spectra, respectively, against gram-positive and -negative bacteria including P. aeruginosa.)]\n",
            "2114: [(148, 161, Microbes, New fluoroquinolones have potent and broad antimicrobial activity and spectra, respectively, against gram-positive and -negative bacteria including P. aeruginosa.)]\n",
            "2115: [(169, 182, Microbes, As a result of their frequent use, bacterial resistance to the quinolones has gradually developed and limited their therapeutic efficacy in infections, especially, with P. aeruginosa, S. pneumoniae, S. aureus (especially MRSA), and N. gonorrhoeae.)]\n",
            "2116: [(184, 197, Microbes, As a result of their frequent use, bacterial resistance to the quinolones has gradually developed and limited their therapeutic efficacy in infections, especially, with P. aeruginosa, S. pneumoniae, S. aureus (especially MRSA), and N. gonorrhoeae.)]\n",
            "2117: [(199, 208, Microbes, As a result of their frequent use, bacterial resistance to the quinolones has gradually developed and limited their therapeutic efficacy in infections, especially, with P. aeruginosa, S. pneumoniae, S. aureus (especially MRSA), and N. gonorrhoeae.)]\n",
            "2118: [(221, 225, Antibiotic-Resistant Bacteria, As a result of their frequent use, bacterial resistance to the quinolones has gradually developed and limited their therapeutic efficacy in infections, especially, with P. aeruginosa, S. pneumoniae, S. aureus (especially MRSA), and N. gonorrhoeae.)]\n",
            "2119: [(232, 246, Microbes, As a result of their frequent use, bacterial resistance to the quinolones has gradually developed and limited their therapeutic efficacy in infections, especially, with P. aeruginosa, S. pneumoniae, S. aureus (especially MRSA), and N. gonorrhoeae.)]\n",
            "2120: [(30, 41, Others, Here, we report biodegradable guanidinium-functionalized polycarbonates with a distinctive mechanism that does not induce drug resistance.)]\n",
            "2121: [(221, 242, Gene/Proteins, Bacterial resistance to the quinolones probably results from : 1) mutations with chromosomal genes of DNA gyrase or DNA topoisomerase in E. coli and S. aureus, 2) decreased permeability of the cell envelope through OmpF, porin-forming protein, in gram-negative bacteria, and 3) activation of active efflux-mediated permeability through the cell membrane protein, either NorA in S. aureus or Opr in P. aeruginosa.)]\n",
            "2122: [(, , , These biodegradable synthetic macromolecules have been demonstrated to have broad spectrum in vivo antimicrobial activity, and have excellent potential as systemic antimicrobials against MDR infections.)]\n",
            "2123: [(63, 76, Antibiotic-Resistant Bacteria, There was no clear relationship between the drug resistance of P. aeruginosa and serum type.)]\n",
            "2124: [(77, 93, Antibiotics, Fourteen strains (46.6%) out of 30 IPM-resistant strains were susceptible to meropenem (MEPM).)]\n",
            "2125: [(, , , Mutations in specified regions (quinolone resistance-determining region) in genes coding for the gyrase and/or topoisomerase leads to clinical resistance.)]\n",
            "2126: [(6, 22, Antibiotics, Newer fluoroquinolones which recognize both molecular targets and have improved pharmacokinetic properties offer hope of higher potency, thereby reducing the probability of development of resistance.)]\n",
            "2127: [(, , , The formation of microbial biofilms is an important reason for failure of antimicrobial therapy.)]\n",
            "2128: [(, , , However, the topical application of AMPs for ocular bacterial infection has not been well investigated.)]\n",
            "2129: [(, , , Multidrug resistant strains arise by sequential accumulation of resistance mutations for individual drugs.)]\n",
            "2130: [(, , , The AMP OH-CATH30, which was identified in the king cobra, exhibits potent antimicrobial activity.)]\n",
            "2131: [(, , , Limited evidence exists indicating that some drug resistant strains with mutations that severely alter catalase-peroxidase activity are less virulent in animal models.)]\n",
            "2132: [(, , , In this study, we investigated the therapeutic potential of OH-CATH30 for Pseudomonas aeruginosa keratitis.)]\n",
            "2133: [(, , , A diverse array of strategies is available to assist in rapid detection of drug resistance-associated gene mutations.)]\n",
            "2134: [(, , , Although remarkable advances have been made, much remains to be learned about the molecular genetic basis of drug resistance in Mycobacterium tuberculosis.)]\n",
            "2135: [(, , , Knowledge of the molecular genetic basis of resistance to antituberculous agents has advanced rapidly since we reviewed this topic 3 years ago.)]\n",
            "2136: [(, , , It is reasonable to believe that development of new therapeutics based on knowledge obtained from the study of the molecular mechanisms of resistance will occur.)]\n",
            "2137: [(, , , Currently, infections caused by drug-resistant bacteria have become a new challenge in anti-infective treatment, seriously endangering public health.)]\n",
            "2138: [(, , , In our continuous effort to develop new antimicrobials, a series of novel honokiol/magnolol amphiphiles were prepared by mimicking the chemical structures and antibacterial properties of cationic antimicrobial peptides.)]\n",
            "2139: [(, , , Additionally, 5i displayed high biosafety and potent in vivo anti-infective potency in a murine sepsis model.)]\n",
            "2140: [(36, 44, Antibiotics, Virtually all isolates resistant to rifampin and related rifamycins have a mutation that alters the sequence of a 27-amino-acid region of the beta subunit of ribonucleic acid (RNA) polymerase.)]\n",
            "2141: [(14, 29, Antibiotics, Resistance to isoniazid (INH) is more complex. Many resistant organisms have mutations in the katG gene encoding catalase-peroxidase that result in altered enzyme structure.)]\n",
            "2142: [(94, 98, Gene/Proteins, Resistance to isoniazid (INH) is more complex. Many resistant organisms have mutations in the katG gene encoding catalase-peroxidase that result in altered enzyme structure.)]\n",
            "2143: [(70, 73, Antibiotics, These structural changes apparently result in decreased conversion of INH to a biologically active form.)]\n",
            "2144: [(5, 8, Antibiotics, Some INH-resistant organisms also have mutations in the inhA locus or a recently characterized gene (kasA) encoding a beta-ketoacyl-acyl carrier protein synthase.)]\n",
            "2145: [(56, 60, Antibiotics, Some INH-resistant organisms also have mutations in the inhA locus or a recently characterized gene (kasA) encoding a beta-ketoacyl-acyl carrier protein synthase.)]\n",
            "2146: [(118, 161, Gene/Proteins, Some INH-resistant organisms also have mutations in the inhA locus or a recently characterized gene (kasA) encoding a beta-ketoacyl-acyl carrier protein synthase.)]\n",
            "2147: [(101, 105, Antibiotic-Resistant Genes, Some INH-resistant organisms also have mutations in the inhA locus or a recently characterized gene (kasA) encoding a beta-ketoacyl-acyl carrier protein synthase.)]\n",
            "2148: [(19, 31, Microbes, Eighty isolates of Shigella spp. (37 Shigella flexneri and 43 Shigella sonnei) from patients with travellers' diarrhoea were studied.)]\n",
            "2149: [(37, 54, Antibiotic-Resistant Bacteria, Eighty isolates of Shigella spp. (37 Shigella flexneri and 43 Shigella sonnei) from patients with travellers' diarrhoea were studied.)]\n",
            "2150: [(62, 77, Antibiotic-Resistant Bacteria, Eighty isolates of Shigella spp. (37 Shigella flexneri and 43 Shigella sonnei) from patients with travellers' diarrhoea were studied.)]\n",
            "2151: [(0, 12, Antibiotics, Streptomycin resistance is due mainly to mutations in the 16S rRNA gene or the rpsL gene encoding ribosomal protein S12.)]\n",
            "2152: [(71, 81, Antibiotics, Susceptibility tests revealed high levels of resistance, especially to ampicillin (65%), tetracycline (78%) and trimethoprim (75%), and particularly among the S. flexneri isolates. Dihydrofolate reductase 1 genes (dfrA1) were prevalent among the trimethoprim-resistant isolates, while oxa genes predominated among the ampicillin-resistant isolates.)]\n",
            "2153: [(89, 101, Antibiotics, Susceptibility tests revealed high levels of resistance, especially to ampicillin (65%), tetracycline (78%) and trimethoprim (75%), and particularly among the S. flexneri isolates. Dihydrofolate reductase 1 genes (dfrA1) were prevalent among the trimethoprim-resistant isolates, while oxa genes predominated among the ampicillin-resistant isolates.)]\n",
            "2154: [(112, 124, Antibiotics, Susceptibility tests revealed high levels of resistance, especially to ampicillin (65%), tetracycline (78%) and trimethoprim (75%), and particularly among the S. flexneri isolates. Dihydrofolate reductase 1 genes (dfrA1) were prevalent among the trimethoprim-resistant isolates, while oxa genes predominated among the ampicillin-resistant isolates.)]\n",
            "2155: [(159, 170, Antibiotic-Resistant Bacteria, Susceptibility tests revealed high levels of resistance, especially to ampicillin (65%), tetracycline (78%) and trimethoprim (75%), and particularly among the S. flexneri isolates. Dihydrofolate reductase 1 genes (dfrA1) were prevalent among the trimethoprim-resistant isolates, while oxa genes predominated among the ampicillin-resistant isolates.)]\n",
            "2156: [(246, 258, Antibiotics, Susceptibility tests revealed high levels of resistance, especially to ampicillin (65%), tetracycline (78%) and trimethoprim (75%), and particularly among the S. flexneri isolates. Dihydrofolate reductase 1 genes (dfrA1) were prevalent among the trimethoprim-resistant isolates, while oxa genes predominated among the ampicillin-resistant isolates.)]\n",
            "2157: [(285, 288, Antibiotic-Resistant Genes, Susceptibility tests revealed high levels of resistance, especially to ampicillin (65%), tetracycline (78%) and trimethoprim (75%), and particularly among the S. flexneri isolates. Dihydrofolate reductase 1 genes (dfrA1) were prevalent among the trimethoprim-resistant isolates, while oxa genes predominated among the ampicillin-resistant isolates.)]\n",
            "2158: [(181, 220, Antibiotic-Resistant Genes, Susceptibility tests revealed high levels of resistance, especially to ampicillin (65%), tetracycline (78%) and trimethoprim (75%), and particularly among the S. flexneri isolates. Dihydrofolate reductase 1 genes (dfrA1) were prevalent among the trimethoprim-resistant isolates, while oxa genes predominated among the ampicillin-resistant isolates.)]\n",
            "2159: [(165, 169, Gene/Proteins, Chloramphenicol resistance was associated with production of chloramphenicol acetyltransferase, while nalidixic acid-resistant isolates had a single mutation in the gyrA gene.)]\n",
            "2160: [(0, 15, Antibiotics, Chloramphenicol resistance was associated with production of chloramphenicol acetyltransferase, while nalidixic acid-resistant isolates had a single mutation in the gyrA gene.)]\n",
            "2161: [(149, 158, Gene/Proteins, Ethambutol resistance in approximately 60% of organisms is due to amino acid replacements at position 306 of an arabinosyltransferase encoded by the embB gene.)]\n",
            "2162: [(0, 10, Antibiotics, Ethambutol resistance in approximately 60% of organisms is due to amino acid replacements at position 306 of an arabinosyltransferase encoded by the embB gene.)]\n",
            "2163: [(10, 19, Antibiotics, We review macrolide and quinolone resistance in Campylobacter and track resistance trends in human clinical isolates in relation to use of these agents in food animals. Susceptibility data suggest that erythromycin and other macrolides should remain the drugs of choice in most regions, with systematic surveillance and control measures maintained, but fluoroquinolones may now be of limited use in the empiric treatment of Campylobacter infections in many regions.)]\n",
            "2164: [(24, 33, Antibiotics, We review macrolide and quinolone resistance in Campylobacter and track resistance trends in human clinical isolates in relation to use of these agents in food animals. Susceptibility data suggest that erythromycin and other macrolides should remain the drugs of choice in most regions, with systematic surveillance and control measures maintained, but fluoroquinolones may now be of limited use in the empiric treatment of Campylobacter infections in many regions.)]\n",
            "2165: [(48, 61, Microbes, We review macrolide and quinolone resistance in Campylobacter and track resistance trends in human clinical isolates in relation to use of these agents in food animals. Susceptibility data suggest that erythromycin and other macrolides should remain the drugs of choice in most regions, with systematic surveillance and control measures maintained, but fluoroquinolones may now be of limited use in the empiric treatment of Campylobacter infections in many regions.)]\n",
            "2166: [(202, 214, Antibiotics, We review macrolide and quinolone resistance in Campylobacter and track resistance trends in human clinical isolates in relation to use of these agents in food animals. Susceptibility data suggest that erythromycin and other macrolides should remain the drugs of choice in most regions, with systematic surveillance and control measures maintained, but fluoroquinolones may now be of limited use in the empiric treatment of Campylobacter infections in many regions.)]\n",
            "2167: [(225, 235, Antibiotics, We review macrolide and quinolone resistance in Campylobacter and track resistance trends in human clinical isolates in relation to use of these agents in food animals. Susceptibility data suggest that erythromycin and other macrolides should remain the drugs of choice in most regions, with systematic surveillance and control measures maintained, but fluoroquinolones may now be of limited use in the empiric treatment of Campylobacter infections in many regions.)]\n",
            "2168: [(353, 369, Antibiotics, We review macrolide and quinolone resistance in Campylobacter and track resistance trends in human clinical isolates in relation to use of these agents in food animals. Susceptibility data suggest that erythromycin and other macrolides should remain the drugs of choice in most regions, with systematic surveillance and control measures maintained, but fluoroquinolones may now be of limited use in the empiric treatment of Campylobacter infections in many regions.)]\n",
            "2169: [(424, 437, Microbes, We review macrolide and quinolone resistance in Campylobacter and track resistance trends in human clinical isolates in relation to use of these agents in food animals. Susceptibility data suggest that erythromycin and other macrolides should remain the drugs of choice in most regions, with systematic surveillance and control measures maintained, but fluoroquinolones may now be of limited use in the empiric treatment of Campylobacter infections in many regions.)]\n",
            "2170: [(0, 9, Antibiotics, Kanamycin resistance is due to nucleotide substitutions in the rrs gene encoding 16S rRNA.)]\n",
            "2171: [(63, 66, Antibiotic-Resistant Genes, Kanamycin resistance is due to nucleotide substitutions in the rrs gene encoding 16S rRNA.)]\n",
            "2172: [(81, 89, Gene/Proteins, Kanamycin resistance is due to nucleotide substitutions in the rrs gene encoding 16S rRNA.)]\n",
            "2173: [(148, 160, Antibiotics, Ten isolates of P. aeruginosa from individuals with keratitis were susceptible to OH-CATH30 but not to cefoperazone, ciprofloxacin, gentamicin, and levofloxacin.)]\n",
            "2174: [(82, 91, Gene/Proteins, Ten isolates of P. aeruginosa from individuals with keratitis were susceptible to OH-CATH30 but not to cefoperazone, ciprofloxacin, gentamicin, and levofloxacin.)]\n",
            "2175: [(16, 29, Antibiotic-Resistant Bacteria, Ten isolates of P. aeruginosa from individuals with keratitis were susceptible to OH-CATH30 but not to cefoperazone, ciprofloxacin, gentamicin, and levofloxacin.)]\n",
            "2176: [(52, 61, Others, Ten isolates of P. aeruginosa from individuals with keratitis were susceptible to OH-CATH30 but not to cefoperazone, ciprofloxacin, gentamicin, and levofloxacin.)]\n",
            "2177: [(103, 115, Antibiotics, Ten isolates of P. aeruginosa from individuals with keratitis were susceptible to OH-CATH30 but not to cefoperazone, ciprofloxacin, gentamicin, and levofloxacin.)]\n",
            "2178: [(117, 130, Antibiotics, Ten isolates of P. aeruginosa from individuals with keratitis were susceptible to OH-CATH30 but not to cefoperazone, ciprofloxacin, gentamicin, and levofloxacin.)]\n",
            "2179: [(132, 142, Antibiotics, Ten isolates of P. aeruginosa from individuals with keratitis were susceptible to OH-CATH30 but not to cefoperazone, ciprofloxacin, gentamicin, and levofloxacin.)]\n",
            "2180: [(4, 36, Techniques, The microdilution checkerboard assay showed that OH-CATH30 exhibited synergistic activity with ciprofloxacin and levofloxacin against antibiotic-resistant P. aeruginosa.)]\n",
            "2181: [(113, 125, Antibiotics, The microdilution checkerboard assay showed that OH-CATH30 exhibited synergistic activity with ciprofloxacin and levofloxacin against antibiotic-resistant P. aeruginosa.)]\n",
            "2182: [(95, 108, Antibiotics, The microdilution checkerboard assay showed that OH-CATH30 exhibited synergistic activity with ciprofloxacin and levofloxacin against antibiotic-resistant P. aeruginosa.)]\n",
            "2183: [(155, 168, Antibiotic-Resistant Bacteria, The microdilution checkerboard assay showed that OH-CATH30 exhibited synergistic activity with ciprofloxacin and levofloxacin against antibiotic-resistant P. aeruginosa.)]\n",
            "2184: [(11, 24, Antibiotic-Resistant Bacteria, Meanwhile, P. aeruginosa did not develop resistance to OH-CATH30, even after exposure at 0.5× the MIC for up to 25 subcultures. Furthermore, treatment with OH-CATH30, alone or in combination with levofloxacin, significantly improved the clinical outcomes of rabbit keratitis induced by antibiotic-resistant P. aeruginosa.)]\n",
            "2185: [(55, 64, Antibiotics, Meanwhile, P. aeruginosa did not develop resistance to OH-CATH30, even after exposure at 0.5× the MIC for up to 25 subcultures. Furthermore, treatment with OH-CATH30, alone or in combination with levofloxacin, significantly improved the clinical outcomes of rabbit keratitis induced by antibiotic-resistant P. aeruginosa.)]\n",
            "2186: [(156, 165, Antibiotics, Meanwhile, P. aeruginosa did not develop resistance to OH-CATH30, even after exposure at 0.5× the MIC for up to 25 subcultures. Furthermore, treatment with OH-CATH30, alone or in combination with levofloxacin, significantly improved the clinical outcomes of rabbit keratitis induced by antibiotic-resistant P. aeruginosa.)]\n",
            "2187: [(307, 320, Antibiotic-Resistant Bacteria, Meanwhile, P. aeruginosa did not develop resistance to OH-CATH30, even after exposure at 0.5× the MIC for up to 25 subcultures. Furthermore, treatment with OH-CATH30, alone or in combination with levofloxacin, significantly improved the clinical outcomes of rabbit keratitis induced by antibiotic-resistant P. aeruginosa.)]\n",
            "2188: [(196, 208, Antibiotics, Meanwhile, P. aeruginosa did not develop resistance to OH-CATH30, even after exposure at 0.5× the MIC for up to 25 subcultures. Furthermore, treatment with OH-CATH30, alone or in combination with levofloxacin, significantly improved the clinical outcomes of rabbit keratitis induced by antibiotic-resistant P. aeruginosa.)]\n",
            "2189: [(114, 137, Antibiotic-Resistant Bacteria, Taken together, our data indicate that the topical application of OH-CATH30 is efficacious against drug-resistant P. aeruginosa keratitis.)]\n",
            "2190: [(, , , Moreover, 5i exhibited rapid bactericidal properties, low resistance frequency, and good capabilities of disrupting bacterial biofilms.)]\n",
            "2191: [(, , , Mechanism studies revealed that 5i destroyed bacterial cell membranes, resulting in bacterial death.)]\n",
            "2192: [(, , , Our study indicates that these honokiol/magnolol amphiphiles shed light on developing novel antibacterial agents, and 5i is a potential antibacterial candidate for combating MRSA infections.)]\n",
            "2193: [(98, 102, Antibiotic-Resistant Genes, Resistance to pyrazinamide in the great majority of organisms is caused by mutations in the gene (pncA) encoding pyrazinamidase that result in diminished enzyme activity.)]\n",
            "2194: [(14, 26, Antibiotics, Resistance to pyrazinamide in the great majority of organisms is caused by mutations in the gene (pncA) encoding pyrazinamidase that result in diminished enzyme activity.)]\n",
            "2195: [(113, 127, Antibiotics, Resistance to pyrazinamide in the great majority of organisms is caused by mutations in the gene (pncA) encoding pyrazinamidase that result in diminished enzyme activity.)]\n",
            "2196: [(74, 89, Antibiotics, Amino acid changes in the A subunit of deoxyribonucleic acid gyrase cause fluoroquinolone resistance in most organisms.)]\n",
            "2197: [(, , , The therapeutic potential of antimicrobial peptides (AMPs) has been evaluated in many infectious diseases.)]\n",
            "2198: [(, , , DNA sequencing revealed that two amino acid substitutions, namely, Ser (TCG)-83-->Leu (TTG) and Asp (GAC)-87-->Gly (GGC), had occurred in the gyrA of all 3 strains isolated from the patients.)]\n",
            "2199: [(, , , Bacteria and fungi continue to develop new ways to adapt and survive the lethal or biostatic effects of antimicrobials through myriad mechanisms.)]\n",
            "2200: [(, , , Novel antibiotic resistance genes such as lsa(C), erm(44), VCC-1, mcr-1, mcr-2, mcr-3, mcr-4, bla KLUC-3 and bla KLUC-4 were discovered through comparative genomics and further functional studies.)]\n",
            "2201: [(, , , As well, mutations in genes that hitherto were unknown to confer resistance to antimicrobials, such as trm, PP2C, rpsJ, HSC82, FKS2 and Rv2887, were shown by genomics and transcomplementation assays to mediate antimicrobial resistance in Acinetobacter baumannii, Staphylococcus aureus, Enterococcus faecium, Saccharomyces cerevisae, Candida glabrata and Mycobacterium tuberculosis, respectively.)]\n",
            "2202: [(, , , Thus, genomics, transcriptomics and metagenomics, coupled with functional studies are the future of antimicrobial resistance research and novel drug discovery or design.)]\n",
            "2203: [(, , , Furthermore, in addition to this dual mutation, the active efflux mechanism also appears to be associated with resistance in bacteria that have been exposed to tosufloxacin.)]\n",
            "2204: [(, , , Enteric fever (caused by Salmonella enterica) has been associated with poor hygiene and is endemic in the South-Asian countries.)]\n",
            "2205: [(, , , The primary target of fluoroquinolone antimicrobial agents is the A subunit of DNA gyrase.)]\n",
            "2206: [(, , , The increase in resistance to first line antimicrobials has been observed, while the emergence of multi/extremely drug resistance cases have been identified in several countries.)]\n",
            "2207: [(70, 76, Gene/Proteins, Resistance to fluoroquinolones is by means of mutations affecting the gyrase gene coding for the A subunit and mutations which affect cell permeability particularly involving porin proteins of the outer membrane.)]\n",
            "2208: [(14, 30, Antibiotics, Resistance to fluoroquinolones is by means of mutations affecting the gyrase gene coding for the A subunit and mutations which affect cell permeability particularly involving porin proteins of the outer membrane.)]\n",
            "2209: [(53, 67, Antibiotics, Two associated resistance mechanisms were found in a nalidixic acid-susceptible (4 micrograms/ml) but fluoroquinolone-resistant (8 to 16 micrograms/ml) strain of Escherichia coli Q2 selected under norfloxacin therapy.)]\n",
            "2210: [(102, 117, Antibiotics, Two associated resistance mechanisms were found in a nalidixic acid-susceptible (4 micrograms/ml) but fluoroquinolone-resistant (8 to 16 micrograms/ml) strain of Escherichia coli Q2 selected under norfloxacin therapy.)]\n",
            "2211: [(162, 181, Microbes, Two associated resistance mechanisms were found in a nalidixic acid-susceptible (4 micrograms/ml) but fluoroquinolone-resistant (8 to 16 micrograms/ml) strain of Escherichia coli Q2 selected under norfloxacin therapy.)]\n",
            "2212: [(197, 208, Antibiotics, Two associated resistance mechanisms were found in a nalidixic acid-susceptible (4 micrograms/ml) but fluoroquinolone-resistant (8 to 16 micrograms/ml) strain of Escherichia coli Q2 selected under norfloxacin therapy.)]\n",
            "2213: [(200, 216, Antibiotics, As compared with the susceptible E. coli Q1 isolated before treatment, changes in outer membrane proteins and lipopolysaccharides in Q2 were associated with a 1.5- to 3-fold decrease in the uptake of fluoroquinolones but not nalidixic acid.)]\n",
            "2214: [(225, 239, Antibiotics, As compared with the susceptible E. coli Q1 isolated before treatment, changes in outer membrane proteins and lipopolysaccharides in Q2 were associated with a 1.5- to 3-fold decrease in the uptake of fluoroquinolones but not nalidixic acid.)]\n",
            "2215: [(33, 40, Antibiotic-Resistant Bacteria, As compared with the susceptible E. coli Q1 isolated before treatment, changes in outer membrane proteins and lipopolysaccharides in Q2 were associated with a 1.5- to 3-fold decrease in the uptake of fluoroquinolones but not nalidixic acid.)]\n",
            "2216: [(148, 164, Antibiotics, A 50% inhibition of DNA synthesis in toluene-permeabilized cells of the resistant strain E. coli Q2 required up to 500-fold increased quantities of fluoroquinolones, whereas such inhibition was obtained in both E. coli Q1 and Q2 with similar amounts of nalidixic acid.)]\n",
            "2217: [(253, 267, Antibiotics, A 50% inhibition of DNA synthesis in toluene-permeabilized cells of the resistant strain E. coli Q2 required up to 500-fold increased quantities of fluoroquinolones, whereas such inhibition was obtained in both E. coli Q1 and Q2 with similar amounts of nalidixic acid.)]\n",
            "2218: [(89, 96, Antibiotic-Resistant Bacteria, A 50% inhibition of DNA synthesis in toluene-permeabilized cells of the resistant strain E. coli Q2 required up to 500-fold increased quantities of fluoroquinolones, whereas such inhibition was obtained in both E. coli Q1 and Q2 with similar amounts of nalidixic acid.)]\n",
            "2219: [(211, 218, Antibiotic-Resistant Bacteria, A 50% inhibition of DNA synthesis in toluene-permeabilized cells of the resistant strain E. coli Q2 required up to 500-fold increased quantities of fluoroquinolones, whereas such inhibition was obtained in both E. coli Q1 and Q2 with similar amounts of nalidixic acid.)]\n",
            "2220: [(29, 40, Antibiotics, Selection from E. coli Q1 on norfloxacin of one-step resistant mutants resembling E. coli Q2 was unsuccessful.)]\n",
            "2221: [(15, 25, Antibiotic-Resistant Bacteria, Selection from E. coli Q1 on norfloxacin of one-step resistant mutants resembling E. coli Q2 was unsuccessful.)]\n",
            "2222: [(82, 92, Antibiotic-Resistant Bacteria, Selection from E. coli Q1 on norfloxacin of one-step resistant mutants resembling E. coli Q2 was unsuccessful.)]\n",
            "2223: [(165, 174, Antibiotics, From these results we infer that a decrease in outer membrane permeability, associated with a peculiar alteration of the DNA gyrase, was responsible for the unusual quinolone resistance phenotype of E. coli Q2.)]\n",
            "2224: [(199, 206, Antibiotic-Resistant Bacteria, From these results we infer that a decrease in outer membrane permeability, associated with a peculiar alteration of the DNA gyrase, was responsible for the unusual quinolone resistance phenotype of E. coli Q2.)]\n",
            "2225: [(159, 171, Antibiotics, Examination of the accumulation of antibiotics in these 3 strains revealed that the strain isolated after tosufloxacin treatment had the highest resistance to tosufloxacin, and exhibited decreased accumulation of tosufloxacin in the bacterial cells discharged after 5 days of treatment with this antibiotic.)]\n",
            "2226: [(213, 225, Antibiotics, Examination of the accumulation of antibiotics in these 3 strains revealed that the strain isolated after tosufloxacin treatment had the highest resistance to tosufloxacin, and exhibited decreased accumulation of tosufloxacin in the bacterial cells discharged after 5 days of treatment with this antibiotic.)]\n",
            "2227: [(106, 118, Antibiotics, Examination of the accumulation of antibiotics in these 3 strains revealed that the strain isolated after tosufloxacin treatment had the highest resistance to tosufloxacin, and exhibited decreased accumulation of tosufloxacin in the bacterial cells discharged after 5 days of treatment with this antibiotic.)]\n",
            "2228: [(52, 73, Others, However, accumulation was restored by addition of a proton-pump inhibitor.)]\n",
            "2229: [(184, 193, Antibiotics, These results suggest that the strains isolated from the inpatient and the patient with secondary infection acquired resistance due to dual gyrA mutation induced by treatment with new quinolone antibiotics.)]\n",
            "2230: [(72, 110, Others, In several cases a close relationship to ID50 (inhibitory dose-50%) and minimum inhibitory concentration (MIC) has been shown for gram-negative bacteria, although this has not been regularly observed for gram-positive bacteria to date.)]\n",
            "2231: [(91, 101, Antibiotic-Resistant Bacteria, The objective of this study is to analyze the current trend of antimicrobial resistance in Salmonella isolates in Nepal, and to identify the status of multi- and extremely- drug resistant isolates.)]\n",
            "2232: [(, , , An energy dependent efflux occurs which is more active in brain heart infusion broth than nutrient broth and which, to date, has not been shown to contribute to inhibition of growth or bacterial lethality.)]\n",
            "2233: [(, , , Areas covered: In this review, the authors discuss the opportunities offered by modern omics approaches to address the challenge and the use of this approach in antimicrobial development.)]\n",
            "2234: [(, , , The extent of uptake by different gram-negative bacteria varies and may contribute in some cases to determination of the MIC.)]\n",
            "2235: [(, , , Specifically, the authors focus on the role of omics technologies and bioinformatics for the revelation of the effects of antimicrobials in a variety of microbial cellular processes, as well as the identification of potential cellular targets, the mechanisms of antimicrobial resistance, and the development of new antimicrobials.)]\n",
            "2236: [(, , , Combined target and permeability resistance has been reported but involves two or more mutational steps.)]\n",
            "2237: [(, , , Resistance during clinical treatment has been observed, but is most likely to be of low magnitude and to be detected in patients with significantly compromised host defenses.)]\n",
            "2238: [(, , , Data were collected through clinical record reviews. Data of all these 60 cases were analyzed for socio-demographic characteristics.)]\n",
            "2239: [(2, 29, Antibiotic-Resistant Bacteria, A Shigella flexneri 2a strain, which did not respond clinically or in laboratory tests to treatment with new quinolone derivatives, was isolated for the first time in Japan from a patient admitted for diarrhea to a Tokyo hospital.)]\n",
            "2240: [(, , , Expert opinion: Prevention of antimicrobial resistance does not only depend on rational drug design such as narrow-spectrum antimicrobials but on several factors.)]\n",
            "2241: [(, , , There is a critical need for more effective use of antimicrobials to both extend their shelf life and prevent resistance from arising.)]\n",
            "2242: [(, , , As treatment choices are limited due to the high-drug resistance rates, there is an increase in the health care cost, duration of hospital stay, morbidity and mortality rates.)]\n",
            "2243: [(, , , To assess the capacities of national infection prevention and control and antimicrobial stewardship programs, we analyzed data on bloodstream infections reported to the Global Antimicrobial Resistance Surveillance System during 2017-2019, data from 7 countries on nationally representative surveys of antimicrobial prescriptions, and data from 2 regional surveys.)]\n",
            "2244: [(, , , Significantly, antimicrobial tolerance, i.e., the ability to survive but not proliferate during antimicrobial exposure, has been shown to precede the development of bona fide antimicrobial resistance (AMR), sparking a renewed and rapidly increasing interest in this field.)]\n",
            "2245: [(, , , Understanding the true burden of antimicrobial resistance for a geographical location is important to guide effective empirical therapy.)]\n",
            "2246: [(, , , As a consequence, problematic infections for the first time are now being investigated for antimicrobial tolerance, with increasing reports demonstrating in-host evolution of antimicrobial tolerance.)]\n",
            "2247: [(, , , To have a national data, it is imperative to have a systemic data capturing across the country through surveillance studies.)]\n",
            "2248: [(, , , Tolerance has been identified in a wide array of bacterial species to all bactericidal antimicrobials.)]\n",
            "2249: [(, , , Very few surveillance studies have been conducted in India to generate national data on antimicrobial resistance.)]\n",
            "2250: [(, , , In Australia, the overuse or inappropriate prescribing of antimicrobials in health (human and animal) and agriculture is a concern that has increased over the years.)]\n",
            "2251: [(, , , Persistence of these infections in the clinic is of growing concern, particularly for the immunocompromised.)]\n",
            "2252: [(, , , This has given bacteria, fungi, parasites, and some viruses through exposure more opportunity to develop resistance.)]\n",
            "2253: [(, , , A process of artificial and natural selection, which favors microorganisms in developing the strongest natural defenses increasing prevalence of antimicrobial resistance (AMR).)]\n",
            "2254: [(, , , Nowadays, topical antimicrobials are divided into three groups: disinfectants, antiseptics, and antibiotics.)]\n",
            "2255: [(18, 42, Antibiotic-Resistant Bacteria, Infections due to drug-resistant pathogens are becoming difficult and a challenge to treat.)]\n",
            "2256: [(27, 38, Microbes, Of particular interest are enterococci, which contain the opportunistic bacterial pathogens Enterococcus faecalis and Enterococcus faecium.)]\n",
            "2257: [(92, 113, Microbes, Of particular interest are enterococci, which contain the opportunistic bacterial pathogens Enterococcus faecalis and Enterococcus faecium.)]\n",
            "2258: [(118, 138, Microbes, Of particular interest are enterococci, which contain the opportunistic bacterial pathogens Enterococcus faecalis and Enterococcus faecium.)]\n",
            "2259: [(58, 91, Microbes, Of particular interest are enterococci, which contain the opportunistic bacterial pathogens Enterococcus faecalis and Enterococcus faecium.)]\n",
            "2260: [(87, 138, Techniques, This review aims to report the cumulative antibiogram and the resistance mechanisms of Global Antimicrobial Resistance Surveillance System (GLASS) priority pathogens from India.)]\n",
            "2261: [(0, 11, Microbes, Enterococci are one of the leading causes of hospital-acquired infection and possess intrinsic tolerance to a number of antimicrobial classes.)]\n",
            "2262: [(, , , The use of topical antimicrobials is beneficial for infection control in wound care because wound infection is the major cause of delayed healing.)]\n",
            "2263: [(, , , Appropriate use of antimicrobials can be lifesaving to humans and animals, but inappropriate use needs to be closely monitored and acted on to promote improved safety and quality of care.)]\n",
            "2264: [(, , , The advantages of topical over systemic antimicrobials include a higher concentration at the target site, fewer systemic adverse effects, and a lower incidence of antimicrobial resistance.)]\n",
            "2265: [(, , , It is the opinion of the authors that the use of a multi-omics bioinformatics approach should become an integral part of antimicrobial drug discovery as well as in the prevention of antimicrobial resistance.)]\n",
            "2266: [(, , , Only antiseptics and antibiotics can be applied to living skin; therefore, this review will focus only on these groups.)]\n",
            "2267: [(, , , To better guide the regional response to antimicrobial resistance (AMR), we report the burden of AMR over time in countries in the World Health Organization Eastern Mediterranean Region.)]\n",
            "2268: [(, , , The advantages of each topical antimicrobial are well established; however, their disadvantages remain prominent.)]\n",
            "2269: [(, , , It is widely known that antiseptics show higher cytotoxicity and a broader spectrum of activity than antibiotics, whereas antibiotics show a higher probability of bacterial resistance development.)]\n",
            "2270: [(64, 103, Antibiotic-Resistant Bacteria, The median proportion of bloodstream infections was highest for carbapenem-resistant Acinetobacter spp. (70.3%) and lowest for carbapenem-resistant Escherichia coli (4.6%).)]\n",
            "2271: [(127, 164, Antibiotic-Resistant Bacteria, The median proportion of bloodstream infections was highest for carbapenem-resistant Acinetobacter spp. (70.3%) and lowest for carbapenem-resistant Escherichia coli (4.6%).)]\n",
            "2272: [(, , , Results of the regional assessments indicate that few countries have capacities for infection prevention and control and antimicrobial stewardship programs to prevent emergence and spread of AMR.)]\n",
            "2273: [(, , , Moreover, the antimicrobials that should be avoided in particular populations are also summarized in this review in order to increase awareness for antimicrobial selection in those populations.)]\n",
            "2274: [(, , , Overall, the magnitude of the problem and the limited capacity to respond emphasize the need for regional political leadership in addressing AMR.)]\n",
            "2275: [(, , , Antimicrobial resistance is a major concern globally.)]\n",
            "2276: [(, , , Bacteria have developed resistance against every antimicrobial in clinical use at an alarming rate.)]\n",
            "2277: [(, , , The aim of this work was to identify gram-positive bacteria and their respective resistance profiles of free-living capuchin monkeys.)]\n",
            "2278: [(, , , Moreover, this review discusses the current knowledge gaps that need to be filled to understand the potential role of LGPs in highly needed clinical scenarios and the ongoing clinical studies that aim to address these voids in the upcoming years.)]\n",
            "2279: [(, , , The increasing prevalence of antimicrobial-resistant bacterial infections has ushered in a major global public health crisis.)]\n",
            "2280: [(, , , Antimicrobial resistance is of global concern, and preserving the ability of many antimicrobials to kill disease-causing bacteria is likely to become more challenging over time.)]\n",
            "2281: [(, , , For this, 15 Sapajus nigritus were captured in a municipal urban park in the northern region of the state of Paraná, Brazil, and under pharmacological restraint, samples were collected with sterile swabs from the oral, rectal, ocular, nasal, and auricular regions.)]\n",
            "2282: [(, , , However, we are speeding up this process dramatically by using antibiotics too much or in the wrong way.)]\n",
            "2283: [(, , , We present one such framework, based on nested biological units of antimicrobial resistance, and describe established or innovative strategies targeting units.)]\n",
            "2284: [(, , , After isolation of the 22 gram-positive bacteria, each isolate was subjected to the catalase and coagulase tests for presumptive identification.)]\n",
            "2285: [(, , , The samples resistant to oxacillin were submitted to the PCR technique to search for the mecA gene.)]\n",
            "2286: [(, , , Infections due to Pseudomonas aeruginosa is a major health concern, especially hospital-acquired infections, in critically ill individuals.)]\n",
            "2287: [(, , , Antimicrobial resistance (AMR) increases the morbidity and mortality rates associated with pseudomonal infections.)]\n",
            "2288: [(, , , While they are approved by the Food and Drug Administration for acute bacterial skin and soft tissue infections, their pharmacological properties suggest a potential role of these agents for the treatment of deep-seated and severe infections, such as bloodstream and bone and joint infections.)]\n",
            "2289: [(51, 65, Microbes, Future studies should be conducted to identify the Staphylococcus species, the high rate of antimicrobial resistance of the monkeys in this study suggests that Grooming's behavior may be contributing to the sharing of the resistant microorganism among the members of this group of primates.)]\n",
            "2290: [(, , , This short report reviews and summarizes some of the efforts used in Australia to control AMR and antibiotic prescribing.)]\n",
            "2291: [(137, 149, Techniques, Therefore, this study aimed to determine the microbial spectrum of infections and resistance patterns of their isolates in patients with chemotherapy-induced neutropenic septic shock.)]\n",
            "2292: [(41, 52, Antibiotics, The long-acting lipoglycopeptides (LGPs) dalbavancin and oritavancin are semisynthetic antimicrobials with broad and potent activity against Gram-positive bacterial pathogens.)]\n",
            "2293: [(57, 68, Antibiotics, The long-acting lipoglycopeptides (LGPs) dalbavancin and oritavancin are semisynthetic antimicrobials with broad and potent activity against Gram-positive bacterial pathogens.)]\n",
            "2294: [(14, 30, Techniques, Subsequently, phenotypic tests for bacterial resistance were performed using the agar diffusion disc method.)]\n",
            "2295: [(10, 29, Microbes, Of the 22 gram-positive cocci of these two (9.09%) are Streptococcus spp. and twenty (90.91%) Staphylococcus spp.)]\n",
            "2296: [(55, 72, Microbes, Of the 22 gram-positive cocci of these two (9.09%) are Streptococcus spp. and twenty (90.91%) Staphylococcus spp.)]\n",
            "2297: [(94, 112, Microbes, Of the 22 gram-positive cocci of these two (9.09%) are Streptococcus spp. and twenty (90.91%) Staphylococcus spp.)]\n",
            "2298: [(6, 24, Microbes, Among Staphylococcus spp. three (13.64%) were coagulate-negative (CoNS) and seventeen (86.36%) coagulate-positive (CoPS).)]\n",
            "2299: [(, , , Inappropriate use includes prescribing antimicrobials when they are not necessary, prescribing the wrong type of antimicrobial and prescribing them for the incorrect duration.)]\n",
            "2300: [(, , , Judicious or restricted antimicrobial use in animal agriculture, aiming to confine the use for the treatment of infections, is the most commonly proposed solution to reduce selection pressure for resistant bacterial strains and resistance genes.)]\n",
            "2301: [(55, 68, Antibiotic-Resistant Bacteria, In this review, we aim to address two major aspects of P. aeruginosa.)]\n",
            "2302: [(, , , However, a multifaceted solution will likely be required to make acceptable progress in reducing antimicrobial resistance, due to other common environmental conditions maintaining antimicrobial resistance and limited executionary potential as human healthcare and agriculture will continue to rely heavily on antimicrobials in the foreseeable future.)]\n",
            "2303: [(, , , The first part of the review will focus on the burden of AMR and its prevailing mechanisms seen in India, while the second part will focus on the challenges and approaches in the management with special emphasis on the role of newer antimicrobial agents.)]\n",
            "2304: [(, , , Drawing parallels from systematic approaches to the management of infectious disease agents and biodiversity loss, we provide examples that a more comprehensive approach is required, targeting antimicrobial resistance in agroecosystems on multiple fronts simultaneously.)]\n",
            "2305: [(, , , The use of these antimicrobials is particularly appealing when prolonged therapy, early discharge, and avoidance of long-term intravascular catheter access are desirable or when multidrug-resistant bacteria are suspected.)]\n",
            "2306: [(, , , Some of the proposed strategies are already in use or ready to be implemented, while some require further research and discussion among scientists and policymakers.)]\n",
            "2307: [(, , , We envision that antimicrobial resistance mitigation strategies for animal agriculture combining multiple tools would constitute powerful ecosystem-level interventions necessary to mitigate antimicrobial resistance.)]\n",
            "2308: [(, , , This review describes the current evidence for the use of oritavancin and dalbavancin in the treatment of invasive infections, as well as the hurdles that are preventing their optimal use.)]\n",
            "2309: [(, , , Current antimicrobial treatment regimens were designed against bacterial strains that were fully susceptible to them.)]\n",
            "2310: [(, , , To expand the useable lifetime of existing antimicrobial drug classes by modifying treatment regimens, data are needed on the antimicrobial pharmacodynamics (PD) against strains with reduced susceptibility.)]\n",
            "2311: [(, , , In this study, we generated and mathematically modelled the PD of the fluoroquinolone ciprofloxacin and the cephalosporin ceftriaxone against non-typhoidal Salmonella enterica subsp. enterica strains with varying levels of acquired resistance.)]\n",
            "2312: [(, , , We included Salmonella strains across categories of reduced susceptibility to fluoroquinolones or cephalosporins reported to date, including isolates from human infections, food-animal products sold in retail, and food-animal production.)]\n",
            "2313: [(, , , Though generally measured environmental concentrations (MECs) of antibiotics in aquatic systems are significantly lower than point of therapeutic use concentrations, there is increasing evidence that suggests these concentrations may still enrich antimicrobial resistant bacteria.)]\n",
            "2314: [(, , , Respecting simple key principles of optimal antibiotic prescribing together with commitment to further research in this area from the pediatric community is essential to extend the lifeline of antibiotics for the most vulnerable patients without limiting access to antibiotics for those children who require treatment.)]\n",
            "2315: [(, , , The incidence of infections caused by bacteria that are resistant to antibiotics is constantly increasing.)]\n",
            "2316: [(, , , We generated PD data for each drug and strain via time-kill assay. Mathematical models were compared in their fit to represent the PD.)]\n",
            "2317: [(, , , In Europe alone, it has been estimated that each year about 33'000 deaths are attributable to such infections.)]\n",
            "2318: [(, , , The best-fit model's parameter values across the strain susceptibility categories were compared. The inhibitory baseline sigmoid Imax (or Emax) model was best fit for the PD of each antimicrobial against a majority of the strains.)]\n",
            "2319: [(, , , The COVID public health emergency has brought home how vulnerable we are to forces beyond our control.)]\n",
            "2320: [(, , , One important driver of antimicrobial resistance is the use and abuse of antibiotics in human medicine. Inappropriate prescribing of antibiotics is still very frequent: up to 50% of all antimicrobials prescribed in humans might be unnecessary and several studies show that at least 50% of antibiotic treatments are inadequate, depending on the setting.)]\n",
            "2321: [(, , , There were statistically significant differences in the PD parameter values across the strain susceptibility categories for each antimicrobial.)]\n",
            "2322: [(121, 131, Microbes, The results demonstrate predictable multiparameter changes in the PD of these first-line antimicrobials depending on the Salmonella strain's susceptibility phenotype and specific genes conferring reduced susceptibility.)]\n",
            "2323: [(25, 36, Antibiotics, In 2016, we identified a ceftriaxone-resistant Neisseria gonorrhoeae isolate in China.)]\n",
            "2324: [(47, 68, Microbes, In 2016, we identified a ceftriaxone-resistant Neisseria gonorrhoeae isolate in China.)]\n",
            "2325: [(57, 62, Antibiotic-Resistant Genes, The strain genotype was identical to the resistant clone FC428 that originated in Japan. Enhanced international collaborative surveillance programs are crucial to track the transmission of the ceftriaxone-resistant clones.)]\n",
            "2326: [(193, 204, Antibiotics, The strain genotype was identical to the resistant clone FC428 that originated in Japan. Enhanced international collaborative surveillance programs are crucial to track the transmission of the ceftriaxone-resistant clones.)]\n",
            "2327: [(37, 45, Gene/Proteins, Monotherapy with an anti-pseudomonal ß-lactam agent is recommended. Although many studies reported the microbial etiology of infections and resistant patterns of febrile neutropenia, the patients were not well characterized as having neutropenic septic shock.)]\n",
            "2328: [(116, 174, Others, We consolidate terminology used thus far into a single framework that could be adopted moving forward, by proposing predicted no effect concentrations for resistance (PNECRs) and predicted no effect concentrations for persistence (PNECPs) be determined in AMR risk assessment.)]\n",
            "2329: [(179, 238, Others, We consolidate terminology used thus far into a single framework that could be adopted moving forward, by proposing predicted no effect concentrations for resistance (PNECRs) and predicted no effect concentrations for persistence (PNECPs) be determined in AMR risk assessment.)]\n",
            "2330: [(, , , Importantly, measures to improve antibiotic prescribing and antibiotic stewardship programs should focus on facilitating clinical reasoning and improving prescribing environment in order to remove any barriers to good prescribing.)]\n",
            "2331: [(, , , The generated PD parameter estimates could be used to optimise treatment regimens against infections by strains with reduced susceptibility.)]\n",
            "2332: [(, , , Antibiotics and antimicrobials are used, misused and overused in human and veterinary medicine, animal husbandry and aquaculture.)]\n",
            "2333: [(, , , These compounds can persist in both human and animal waste and then enter the environment through a variety of mechanisms.)]\n",
            "2334: [(, , , Among patients with febrile neutropenia that developed after chemotherapy, high-risk patients, such as those having clinical instability or Multinational Association of Supportive Care in Cancer score of < 21, require hospitalization for intravenous empiric antibiotic therapy.)]\n",
            "2335: [(, , , In light of this evidence, a rigorous and standardised novel methodology needs to be developed which can perform environmental risk assessment (ERA) of antimicrobials in terms of their selective potential as well as their environmental impact, to ensure that diffuse and point source discharges are safe.)]\n",
            "2336: [(, , , This review summarises and critically appraises the current methodological approaches that study selection at below point of therapeutic use, or sub-inhibitory, concentrations of antibiotics.)]\n",
            "2337: [(, , , We collate and compare selective concentration data generated to date. We recommend how these data can be interpreted in line with current ERA guidelines; outlining and describing novel concepts unique to risk assessment of AMR (such as direct selection of AMR or increased persistence of AMR).)]\n",
            "2338: [(, , , Such a framework will contribute to antibiotic stewardship and by extension, protection of human health, food security and the global economy.)]\n",
            "2339: [(, , , Antimicrobial resistance threatens therapeutic options for human and animal bacterial diseases worldwide.)]\n",
            "2340: [(, , , MDR bacteria were prevalent in patients with chemotherapy-induced neutropenic septic shock.)]\n",
            "2341: [(, , , Therefore, piperacillin-tazobactam or carbapenem may be considered as empiric antibiotics if MDR bacteria are suspected to be causative agents.)]\n",
            "2342: [(, , , It is clear that a greater more complex pandemic will emerge through antimicrobial resistance (AMR) if we don't embrace the lessons from this one.)]\n",
            "2343: [(, , , Earlier diagnostic testing, in the right place at the right time, using data to drive better public health approaches, should be our response.)]\n",
            "2344: [(, , , Proactive diagnostics as a paradigm shift from reactive diagnostics are essential and should be factored into the design of future services.)]\n",
            "2345: [(, , , By building on and capitalizing on the lessons on how to handle data, deploy technology and understand the spread of disease, we will improve the response of laboratories, and improve patient outcomes, and through that improve public health.)]\n",
            "2346: [(, , , The digital tools of the future will be a huge enabler of this, and this will be made more effective though sharing of data seamlessly.)]\n",
            "2347: [(, , , Action needs to be taken now to ensure that this change is implemented at the pace required to tackle AMR.)]\n",
            "2348: [(, , , Many of the newer beta-lactam/beta-lactamase inhibitor combinations are not active against organisms carrying Class B metallo-enzymes, so differentiating organisms with Class A or D carbapenemases from those with Class B enzymes rapidly is critical.)]\n",
            "2349: [(, , , Antibiotic resistance is a threat to public health, and uncomplicated urinary tract infections (uUTIs) are an example of this concern.)]\n",
            "2350: [(81, 92, Antibiotics, Comparing isolates before and after treatment, a clear increase in resistance to doxycycline (12% against 21.74%) and erythromycin (20% against 39.13%) was observed.)]\n",
            "2351: [(118, 130, Antibiotics, Comparing isolates before and after treatment, a clear increase in resistance to doxycycline (12% against 21.74%) and erythromycin (20% against 39.13%) was observed.)]\n",
            "2352: [(45, 58, Antibiotics, In Asia, there was a substantial increase in ciprofloxacin resistance during 2008 to 2014 from 25% to more than 40%.)]\n",
            "2353: [(83, 123, Others, Such infections can be very difficult to treat, especially when they are caused by carbapenemase-producing organisms (CPO).)]\n",
            "2354: [(213, 235, Techniques, Detecting CRG directly in positive blood culture bottles or in syndromic panels coupled with bacterial identification are helpful when results are positive, however, even negative results can provide guidance for anti-infective therapy for key organism-drug combinations when linked to local epidemiology.)]\n",
            "2355: [(155, 165, Antibiotics, This perspective will focus on the reluctance of laboratories to use molecular tests as aids to developing therapeutic strategies for infections caused by carbapenem-resistant organisms and how to overcome that reluctance.)]\n",
            "2356: [(4, 12, Others, The COVID-19 pandemic has shown that the management of infectious diseases is of global importance.)]\n",
            "2357: [(0, 11, Antibiotics, Doxycycline is an antibiotic that, in addition to the classic antibacterial use, is also prescribed to fight parasitic diseases, like heartworm disease in dogs.)]\n",
            "2358: [(152, 163, Antibiotics, Despite the concern that the overuse of this antibiotic may decrease susceptibility of clinically important bacteria, the consequences of the prolonged doxycycline therapy in heartworm-infected dogs have never been studied before.)]\n",
            "2359: [(47, 68, Microbes, We have analyzed the impact of this therapy on Staphylococcus aureus and Enterococcus antimicrobial resistance. In this study, 17 heartworm-infected dogs (10 that had completed the doxycycline treatment and 7 dogs that had not yet begun) were included.)]\n",
            "2360: [(73, 85, Microbes, We have analyzed the impact of this therapy on Staphylococcus aureus and Enterococcus antimicrobial resistance. In this study, 17 heartworm-infected dogs (10 that had completed the doxycycline treatment and 7 dogs that had not yet begun) were included.)]\n",
            "2361: [(181, 192, Antibiotics, We have analyzed the impact of this therapy on Staphylococcus aureus and Enterococcus antimicrobial resistance. In this study, 17 heartworm-infected dogs (10 that had completed the doxycycline treatment and 7 dogs that had not yet begun) were included.)]\n",
            "2362: [(24, 45, Microbes, Twenty-four isolates of Staphylococcus aureus were obtained from two locations of each dog. After treatment, 73.3% of isolates were resistant to at least one antibiotic but only 22.2% of isolates before treatment.)]\n",
            "2363: [(8, 19, Antibiotics, Most of doxycycline resistant isolates were obtained from dogs that have received treatment. Erythromycin resistance or intermediate susceptibility was detected in 45.6% of isolates, most of them from dogs after treatment.)]\n",
            "2364: [(93, 105, Antibiotics, Most of doxycycline resistant isolates were obtained from dogs that have received treatment. Erythromycin resistance or intermediate susceptibility was detected in 45.6% of isolates, most of them from dogs after treatment.)]\n",
            "2365: [(, , , Before treatment, 32% of isolates were resistant at least to one antibiotic while after, this data increase up to 65%.)]\n",
            "2366: [(, , , This systematic review (International Prospective Register of Systematic Reviews [PROSPERO] ID: CRD42020156674) is the first to determine the prevalence of Escherichia coli resistance to fluoroquinolones in women with community-acquired uUTI.)]\n",
            "2367: [(, , , PubMed and Embase searches were conducted; 38 studies fulfilled eligibility criteria and were included in the systematic review.)]\n",
            "2368: [(15, 28, Antibiotics, Within Europe, ciprofloxacin resistance in E. coli isolates varied between countries and increased in some from 2006 to 2008 and 2014 to 2016, specifically in the United Kingdom (0.5% to 15.3%), Germany (8.7% to 15.1%), and Spain (22.9% to 30.8%), although methodologies and settings were often not comparable.)]\n",
            "2369: [(43, 50, Antibiotic-Resistant Bacteria, Within Europe, ciprofloxacin resistance in E. coli isolates varied between countries and increased in some from 2006 to 2008 and 2014 to 2016, specifically in the United Kingdom (0.5% to 15.3%), Germany (8.7% to 15.1%), and Spain (22.9% to 30.8%), although methodologies and settings were often not comparable.)]\n",
            "2370: [(41, 64, Microbes, Infections caused by multidrug-resistant Gram-negative organisms have become a global threat.)]\n",
            "2371: [(, , , Since infections caused by CPO tend to have worse outcomes than non-CPO infections, it is important to identify the type of carbapenemase present in the isolate or at least the Ambler Class (i.e., A, B, or D), to optimize therapy.)]\n",
            "2372: [(, , , In total, 109 bacteria were found in patients with neutropenic septic shock. Gram-negative bacteria were the predominant causative organisms (84, 77.1%).)]\n",
            "2373: [(81, 137, Antibiotic-Resistant Bacteria, Moreover, 33 microorganisms (30.3%) were multidrug-resistant (MDR) bacteria with extended-spectrum ß-lactamase-producing Escherichia coli (17, 50%) being the commonest.)]\n",
            "2374: [(50, 83, Antibiotic-Resistant Genes, Using molecular tests to detect and differentiate carbapenem-resistance genes (CRG) in bacterial isolates provides fast and actionable results, but utilization of these tests globally appears to be low.)]\n",
            "2375: [(, , , The most commonly affected sites in patients with MDR bacterial infections were the gastrointestinal tract (45%) and unknown (43.5%).)]\n",
            "2376: [(, , , Approximately 48.5% of MDR bacteria were resistant to cefepime but not to piperacillin-tazobactam or carbapenem.)]\n",
            "2377: [(, , , The workshop concluded that despite an increase in scientific evidence that the environment may play a significant role, especially in low-resource settings, the exact relative role of the environment is still unclear.)]\n",
            "2378: [(, , , We highlight features associated with bacteriophage therapy that make it an attractive treatment option for multidrug-resistant infections and also discuss some of the challenges that need to be considered in the design and execution of clinical trials directed at evaluating the efficacy of bacteriophage therapy in humans.)]\n",
            "2379: [(, , , Febrile urinary tract infection (UTI) is currently considered the most frequent cause of serious bacterial illness in children in the first 2 years of life.)]\n",
            "2380: [(, , , There are some uncertainties as to the potential for the propagation of particular resistance genes within wastewater treatment plants (WWTPs), but there is no doubt that the reduction in viable microbes (with or without resistance genes) available for transmission via the environment is one of the goals of human waste management.)]\n",
            "2381: [(, , , Although progress has been made in the past years with respect to quantifying environmental AMR transmission potential, still more data on the spread of environmental AMR within human communities is needed.)]\n",
            "2382: [(, , , To avoid this risk, an early effective antibiotic treatment is essential.)]\n",
            "2383: [(, , , Even though evidence on AMR in WWTPs has increased, the reduction in the emergence and spread of AMR by basic sanitation methods is yet unresolved.)]\n",
            "2384: [(, , , In order to contribute to the generation of harmonized One Health surveillance data, WHO has initiated an integrated One Health surveillance strategy that includes the environment.)]\n",
            "2385: [(, , , However, antibiotic resistance for UTI-related bacterial pathogens continuously increases, making recommendations rapidly outdated and the definition of the best empiric antibiotic therapy more difficult.)]\n",
            "2386: [(, , , The main challenge lies in rolling it out globally including to the poorest regions.)]\n",
            "2387: [(, , , Overall, the occurrence of Campylobacter spp. and Salmonella spp. was 5.2% (93/1785, CI95%: 4.2-6.2%) and 20.8% (371/1785, CI95%: 18.9-22.7%), respectively.)]\n",
            "2388: [(52, 91, Others, Medical providers must understand current trends of enteropathogen antimicrobial resistance (AMR) in service members (SMs) to inform proper, timely treatment options.)]\n",
            "2389: [(5, 20, Microbes, Both enteropathogens have been isolated from a diversity of wild birds in Northern Europe, but there is limited information about gulls as potential reservoirs in Southern Europe.)]\n",
            "2390: [(35, 50, Antibiotics, One study indicated a link between fluoroquinolone resistance and uUTI recurrence.)]\n",
            "2391: [(, , , This paper uses bibliometrics to examine publications in the field of ARB from 2010 to 2020 that were retrieved from the Web of Science (WOS).)]\n",
            "2392: [(78, 94, Antibiotics, These findings may have implications for the empirical treatment of uUTI with fluoroquinolones globally, but more data are needed to fully understand regional situations and impact patient management.)]\n",
            "2393: [(47, 56, Antibiotic-Resistant Bacteria, The most predominant Campylobacter species was C. jejuni (94.6%). A high diversity of Salmonella serovars was isolated and the most frequent were those also reported in human outbreaks, such as Salmonella Typhimurium.)]\n",
            "2394: [(86, 96, Antibiotic-Resistant Bacteria, The most predominant Campylobacter species was C. jejuni (94.6%). A high diversity of Salmonella serovars was isolated and the most frequent were those also reported in human outbreaks, such as Salmonella Typhimurium.)]\n",
            "2395: [(194, 217, Antibiotic-Resistant Bacteria, The most predominant Campylobacter species was C. jejuni (94.6%). A high diversity of Salmonella serovars was isolated and the most frequent were those also reported in human outbreaks, such as Salmonella Typhimurium.)]\n",
            "2396: [(21, 34, Antibiotic-Resistant Bacteria, A high proportion of Campylobacter and Salmonella isolates showed resistance to at least one antimicrobial agent (20.2% and 51.5%, respectively), while 19.2% of Salmonella isolates were multidrug-resistant. These results show the relevance of gulls as reservoirs of Campylobacter and Salmonella by maintaining and spreading these bacteria, including resistant and multidrug resistant strains, in the environment.)]\n",
            "2397: [(266, 279, Antibiotic-Resistant Bacteria, A high proportion of Campylobacter and Salmonella isolates showed resistance to at least one antimicrobial agent (20.2% and 51.5%, respectively), while 19.2% of Salmonella isolates were multidrug-resistant. These results show the relevance of gulls as reservoirs of Campylobacter and Salmonella by maintaining and spreading these bacteria, including resistant and multidrug resistant strains, in the environment.)]\n",
            "2398: [(161, 171, Antibiotic-Resistant Bacteria, A high proportion of Campylobacter and Salmonella isolates showed resistance to at least one antimicrobial agent (20.2% and 51.5%, respectively), while 19.2% of Salmonella isolates were multidrug-resistant. These results show the relevance of gulls as reservoirs of Campylobacter and Salmonella by maintaining and spreading these bacteria, including resistant and multidrug resistant strains, in the environment.)]\n",
            "2399: [(284, 294, Antibiotic-Resistant Bacteria, A high proportion of Campylobacter and Salmonella isolates showed resistance to at least one antimicrobial agent (20.2% and 51.5%, respectively), while 19.2% of Salmonella isolates were multidrug-resistant. These results show the relevance of gulls as reservoirs of Campylobacter and Salmonella by maintaining and spreading these bacteria, including resistant and multidrug resistant strains, in the environment.)]\n",
            "2400: [(11, 40, Others, Given many antibiotic-resistant bacteria (ARB) can be part of the normal gut flora, it can be assumed that for environmental transmission, the burden of fecal-oral transmission of AMR in a geographical area follows that of WASH-related infections.)]\n",
            "2401: [(47, 63, Others, UTI in paediatrics can irreversibly damage the renal parenchyma and lead to chronic renal insufficiency and related problems.)]\n",
            "2402: [(98, 109, Others, Moreover, prompt treatment is mandatory to improve the clinical condition of the patient, prevent bacteraemia, and avoid the risk of bacterial localization in other body sites.)]\n",
            "2403: [(, , , However, little is known of enteric pathogen profiles across the Military Health System (MHS).)]\n",
            "2404: [(, , , The primary objectives of this study were to identify gaps in enteric pathogen surveillance within the MHS, describe the epidemiology of AMR in enteric pathogens, and identify trends across the MHS both within the Continental United States (CONUS) and outside of the Continental United States (OCONUS).)]\n",
            "2405: [(0, 18, Antibiotic-Resistant Bacteria, Campylobacter spp. and Salmonella spp. are the two most frequent zoonotic bacteria involved in human enteric infections in the European Union.)]\n",
            "2406: [(23, 38, Antibiotic-Resistant Bacteria, Campylobacter spp. and Salmonella spp. are the two most frequent zoonotic bacteria involved in human enteric infections in the European Union.)]\n",
            "2407: [(, , , A broad sampling of fledglings from nine colonies of yellow-legged gull (Larus michahellis, N = 1222) and Audouin's gull (Larus audouinii, N = 563) has been conducted in Spain and Tunisia during the late chick-rearing period.)]\n",
            "2408: [(, , , The emergence and spread of antimicrobial resistance (AMR), including clinically relevant antimicrobial-resistant bacteria, genetic resistance elements, and antibiotic residues, presents a significant threat to human health.)]\n",
            "2409: [(, , , Reducing the incidence of infection by improving water, sanitation, and hygiene (WASH) is one of five objectives in the World Health Organization's (WHO) Global Action Plan on AMR.)]\n",
            "2410: [(, , , In September 2019, WHO and the Health-Related Water Microbiology specialist group (HRWM-SG) of the International Water Association (IWA) organized its third workshop on AMR, focusing on the following three main issues: environmental pathways of AMR transmission, environmental surveillance, and removal from human waste.)]\n",
            "2411: [(, , , Our results suggest that gulls can serve as sentinel species for antibiotic pressure in the environment.)]\n",
            "2412: [(, , , Variation in pathogen susceptibility to antibiotics is essential for the choice of an effective therapy. Moreover, proper identification of cases at increased risk of difficult-to-treat UTIs can reduce the risk of ineffective therapy. In this review, the problem of emerging antibiotic resistance among pathogens associated with the development of paediatric febrile UTIs and the best potential solutions to ensure the most effective therapy are discussed.)]\n",
            "2413: [(, , , The samples were inoculated in blood agar and chocolate agar.)]\n",
            "2414: [(, , , Tolerance to LA was significantly influenced by both the ABR profile of the strain and LA concentration.)]\n",
            "2415: [(, , , The results from this study indicate that E. coli O157:H7 strains with certain ABR profiles might be more tolerant to LA.)]\n",
            "2416: [(, , , Bacillus was resistant to oxacillin.)]\n",
            "2417: [(, , , The antibiotic resistance trend of coagulase-negative Staphylococcus showed that the resistance to rifampicin increased through the years 2012-2018.)]\n",
            "2418: [(, , , This review will discuss emerging medicinal chemistry approaches for addressing the global multidrug resistance in the 21st century.)]\n",
            "2419: [(, , , Moraxella was susceptible to most antibiotics.)]\n",
            "2420: [(, , , This descriptive cross-sectional study was conducted on patients admitted with UTI (n=60) at a tertiary level hospital in Dhaka, Bangladesh from March 2019 to September 2019.)]\n",
            "2421: [(38, 59, Antibiotic-Resistant Bacteria, Coagulase-negative Staphylococcus and Staphylococcus aureus were the most isolated bacteria in patients affected by blepharitis in our tertiary eye centre.)]\n",
            "2422: [(19, 33, Antibiotic-Resistant Bacteria, Coagulase-negative Staphylococcus and Staphylococcus aureus were the most isolated bacteria in patients affected by blepharitis in our tertiary eye centre.)]\n",
            "2423: [(32, 44, Antibiotics, Both bacteria were resistant to erythromycin.)]\n",
            "2424: [(52, 66, Microbes, Through the years, it seems that coagulase-negative Staphylococcus gained resistance to penicillin, erythromycin, ciprofloxacin and rifampicin.)]\n",
            "2425: [(88, 98, Antibiotics, Through the years, it seems that coagulase-negative Staphylococcus gained resistance to penicillin, erythromycin, ciprofloxacin and rifampicin.)]\n",
            "2426: [(100, 112, Antibiotics, Through the years, it seems that coagulase-negative Staphylococcus gained resistance to penicillin, erythromycin, ciprofloxacin and rifampicin.)]\n",
            "2427: [(114, 127, Antibiotics, Through the years, it seems that coagulase-negative Staphylococcus gained resistance to penicillin, erythromycin, ciprofloxacin and rifampicin.)]\n",
            "2428: [(132, 142, Antibiotics, Through the years, it seems that coagulase-negative Staphylococcus gained resistance to penicillin, erythromycin, ciprofloxacin and rifampicin.)]\n",
            "2429: [(0, 24, Microbes, Escherichia coli O157:H7-contaminated beef has been implicated in numerous foodborne outbreaks.)]\n",
            "2430: [(100, 107, Antibiotic-Resistant Bacteria, The influence of antibiotic resistance (ABR) on growth rates and cross-tolerance of lettuce isolate E. coli O157:H7 H1730 to LA was evaluated.)]\n",
            "2431: [(0, 10, Antibiotics, Ampicillin resistance was also conferred by plasmid transformation to generate the ampP and ampP strepC strains.)]\n",
            "2432: [(44, 66, Techniques, Ampicillin resistance was also conferred by plasmid transformation to generate the ampP and ampP strepC strains.)]\n",
            "2433: [(92, 111, Antibiotic-Resistant Genes, Ampicillin resistance was also conferred by plasmid transformation to generate the ampP and ampP strepC strains.)]\n",
            "2434: [(83, 87, Antibiotic-Resistant Genes, Ampicillin resistance was also conferred by plasmid transformation to generate the ampP and ampP strepC strains.)]\n",
            "2435: [(210, 224, Microbes, describe which bacteria can be found on lid margins in patients affected with blepharitis, to show their antibiotic susceptibility pattern, and to evaluate the antibiotic resistance trend of coagulase-negative Staphylococcus through time.)]\n",
            "2436: [(133, 140, Antibiotic-Resistant Bacteria, The minimum inhibitory concentration of LA on all the strains evaluated was 0.375% v/v. The lag phase duration of all strains except E. coli O157:H7 ampP strepC increased with increasing concentration of LA.)]\n",
            "2437: [(, , , Consecutive cases of 198 eyes affected with blepharitis between 2012 and 2018 were reviewed.)]\n",
            "2438: [(4, 8, Gene/Proteins, The ampP strepC and ampC strains were most tolerant to 5% LA with declines in the cell population of 2.86 and 2.56 log CFU/mL, respectively (p < 0.05).)]\n",
            "2439: [(20, 24, Gene/Proteins, The ampP strepC and ampC strains were most tolerant to 5% LA with declines in the cell population of 2.86 and 2.56 log CFU/mL, respectively (p < 0.05).)]\n",
            "2440: [(9, 15, Gene/Proteins, The ampP strepC and ampC strains were most tolerant to 5% LA with declines in the cell population of 2.86 and 2.56 log CFU/mL, respectively (p < 0.05).)]\n",
            "2441: [(153, 167, Antibiotic-Resistant Bacteria, The susceptibility of the identified bacteria to common antibiotics was tested. In addition, the antibiotic susceptibility pattern of coagulase-negative Staphylococcus detected from year 2016 to 2018 was compared with that of 4 years before.)]\n",
            "2442: [(0, 15, Microbes, Corynebacterium was resistant to oxacillin and erythromycin. Streptococcus viridans showed resistance to gentamycin and tobramycin.)]\n",
            "2443: [(61, 83, Microbes, Corynebacterium was resistant to oxacillin and erythromycin. Streptococcus viridans showed resistance to gentamycin and tobramycin.)]\n",
            "2444: [(120, 130, Antibiotics, Corynebacterium was resistant to oxacillin and erythromycin. Streptococcus viridans showed resistance to gentamycin and tobramycin.)]\n",
            "2445: [(, , , To limit the emergence of resistance, every effort to reduce and rationalise antibiotic consumption must be made. An increased use of antibiotic stewardship can be greatly effective in this regard.)]\n",
            "2446: [(, , , Over the past century, a multitude of derivatives of structural scaffolds with established antimicrobial potential have been prepared and tested, and a variety of new scaffolds have emerged. The effectiveness of antibiotics, however, is in sharp decline because of the emergence of drug-resistant microorganisms.)]\n",
            "2447: [(, , , The prevalence of drug resistance, both in clinical and community settings, is a consequence of bacterial ingenuity in altering pathways and/or cell morphology, making it a persistent threat to human health.)]\n",
            "2448: [(, , , The fundamental ability of pathogens to survive in a multitude of habitats can be triggered by recognition of chemical signals that warn organisms of exposure to a potentially harmful environment.)]\n",
            "2449: [(, , , Host immune defenses, including reactive oxygen intermediates and antibacterial substances, are among the multitude of chemical signals that can subsequently trigger expression of phenotypes better adapted for survival in that hostile environment.)]\n",
            "2450: [(, , , Thus, resistance development appears to be unavoidable, which leads to the conclusion that developing an alternative perspective for treatment options is vital.)]\n",
            "2451: [(87, 97, Antibiotics, Antibiotic-resistant strain variants were generated by conferring resistance to either ampicillin (ampC) or streptomycin (strepC) or both ampicillin and streptomycin (ampC strepC) through incremental exposure to the antibiotics.)]\n",
            "2452: [(99, 103, Antibiotic-Resistant Genes, Antibiotic-resistant strain variants were generated by conferring resistance to either ampicillin (ampC) or streptomycin (strepC) or both ampicillin and streptomycin (ampC strepC) through incremental exposure to the antibiotics.)]\n",
            "2453: [(108, 120, Antibiotics, Antibiotic-resistant strain variants were generated by conferring resistance to either ampicillin (ampC) or streptomycin (strepC) or both ampicillin and streptomycin (ampC strepC) through incremental exposure to the antibiotics.)]\n",
            "2454: [(122, 128, Antibiotic-Resistant Genes, Antibiotic-resistant strain variants were generated by conferring resistance to either ampicillin (ampC) or streptomycin (strepC) or both ampicillin and streptomycin (ampC strepC) through incremental exposure to the antibiotics.)]\n",
            "2455: [(138, 148, Antibiotics, Antibiotic-resistant strain variants were generated by conferring resistance to either ampicillin (ampC) or streptomycin (strepC) or both ampicillin and streptomycin (ampC strepC) through incremental exposure to the antibiotics.)]\n",
            "2456: [(153, 165, Antibiotics, Antibiotic-resistant strain variants were generated by conferring resistance to either ampicillin (ampC) or streptomycin (strepC) or both ampicillin and streptomycin (ampC strepC) through incremental exposure to the antibiotics.)]\n",
            "2457: [(167, 171, Antibiotic-Resistant Genes, Antibiotic-resistant strain variants were generated by conferring resistance to either ampicillin (ampC) or streptomycin (strepC) or both ampicillin and streptomycin (ampC strepC) through incremental exposure to the antibiotics.)]\n",
            "2458: [(172, 178, Antibiotic-Resistant Genes, Antibiotic-resistant strain variants were generated by conferring resistance to either ampicillin (ampC) or streptomycin (strepC) or both ampicillin and streptomycin (ampC strepC) through incremental exposure to the antibiotics.)]\n",
            "2459: [(, , , A sample was collected by rubbing a swab against the base of the eyelashes of both the eyes of all patients.)]\n",
            "2460: [(, , , The ampP strepC strain was the most tolerant when evaluated by the live/dead viability assay. The addition of the efflux pump inhibitor, carbonyl cyanide m-chlorophenylhydrazone, with 2.5% LA resulted in a significant increase in sensitivity in the no resistance (NR) wild-type and ampC strains, resulting in 6.62 and 6.65 log CFU/mL reduction, respectively, while the highly tolerant ampP strepC strain had a 2.90 log CFU/mL decrease.)]\n",
            "2461: [(, , , We suggest that AMR prevention needs to look beyond the antibiotic as object and the (ir)responsible use binary to engage with the antibiotic effects individuals seek in order to manage infectious diseases.)]\n",
            "2462: [(, , , This antibiotic assemblage orientation is likely to be more meaningful for individuals seeking out methods for promoting their health in the face of common infections.)]\n",
            "2463: [(, , , Goldfish farming gained more attention among the ornamental fishes in aquaculture industry.)]\n",
            "2464: [(, , , The ecological and human health impact of antibiotic use and the related antimicrobial resistance (AMR) in animal husbandry is poorly understood. In many countries, there has been considerable pressure to reduce overall antibiotic use in agriculture or to cease or minimise use of human critical antibiotics.)]\n",
            "2465: [(, , , We have isolated diverse enterobacteriaceae groups which affect the goldfish and identified their response towards 46 antimicrobials of 15 different classes.)]\n",
            "2466: [(, , , Thirteen significant bacterial isolates such as Edwardsiella tarda, Serratia marcescens, Klebsiella aerogenes, Proteus penneri, P. hauseri, Enterobacter cloacae, E. cancerogenus, E. ludwigii, Citrobacter freundii, E. coli, Kluyvera cryocrescens, Plesiomonas shigelloides and Providencia vermicola were recovered from the infected fish with the Shannon-wiener diversity index of 2.556. Multiple antibiotic resistance (MAR) index was found to be maximum for P. penneri (0.87) and minimum for C. freundii and E. cloacae (0.22), highlighting the hyper antibiotic selection pressure in the farm.)]\n",
            "2467: [(, , , Through our analysis, we find that changing the rate at which a slurry tank is emptied may delay the proliferation of multidrug-resistant bacteria by up to five years depending on conditions.)]\n",
            "2468: [(, , , The minimum concentration of antibiotics required to inhibit most of the resistant isolates was found to be > 256 mcg/ml.)]\n",
            "2469: [(, , , These approaches employ a binary opposition of responsible and irresponsible antibiotic use with a focus on the knowledge, behaviours and intentions of the individual.)]\n",
            "2470: [(, , , The occurrence of bacterial infections and further antimicrobial treatment lead to the major crisis of antibiotic resistance in aquaculture.)]\n",
            "2471: [(0, 12, Techniques, Metagenomics was used to determine the relative abundance and mobile potential of antibiotic resistant genes (ARGs), prior to determining the correlation of ARGs with time/incidence of COVID-19.)]\n",
            "2472: [(82, 115, Gene/Proteins, Metagenomics was used to determine the relative abundance and mobile potential of antibiotic resistant genes (ARGs), prior to determining the correlation of ARGs with time/incidence of COVID-19.)]\n",
            "2473: [(164, 178, Others, However, a more nuanced approach would consider the differential impact of use of different antibiotic classes; for example, it is not known whether reduced use of bacteriostatic or bacteriolytic classes of antibiotics would be of greater value.)]\n",
            "2474: [(182, 195, Others, However, a more nuanced approach would consider the differential impact of use of different antibiotic classes; for example, it is not known whether reduced use of bacteriostatic or bacteriolytic classes of antibiotics would be of greater value.)]\n",
            "2475: [(30, 43, Others, We model the chemical fate of bacteriolytic and bacteriostatic antibiotics within the slurry and their effect on a population of bacteria, which are capable of resistance to both types of antibiotic.)]\n",
            "2476: [(48, 62, Others, We model the chemical fate of bacteriolytic and bacteriostatic antibiotics within the slurry and their effect on a population of bacteria, which are capable of resistance to both types of antibiotic.)]\n",
            "2477: [(63, 74, Antibiotics, We model the chemical fate of bacteriolytic and bacteriostatic antibiotics within the slurry and their effect on a population of bacteria, which are capable of resistance to both types of antibiotic.)]\n",
            "2478: [(, , , We also find that, within our model, the development of multidrug resistance is particularly sensitive to the use of bacteriolytic antibiotics, rather than bacteriostatic antibiotics, and this may be cause for controlling the usage of bacteriolytic antibiotics in agriculture.)]\n",
            "2479: [(91, 109, Antibiotic-Resistant Bacteria, This study is the first of its kind to investigate the antimicrobial resistance profile of enterobacteriaceae recovered from goldfish, before and after plasmid curing.)]\n",
            "2480: [(, , , Since the COVID-19 pandemic started, there has been much speculation about how COVID-19 and antimicrobial resistance may be interconnected. In this study, untreated wastewater was sampled from Hospital A designated to treat COVID-19 patients during the first wave of the COVID-19 pandemic alongside Hospital B that did not receive any COVID-19 patients.)]\n",
            "2481: [(, , , Antibiotic-resistant bacteria (ARB) are a serious threat to the health of people and the ecological environment.)]\n",
            "2482: [(, , , With this problem becoming more and more serious, more countries made research on the ARB, and the research number has been sharply increased particularly over the past decade.)]\n",
            "2483: [(, , , Therefore, it is quite necessary to globally retrace relevant researches on the ARB published from 2010 to 2020.)]\n",
            "2484: [(, , , This study highlighted concerns related to the dissemination of antimicrobial resistance (AMR) during the COVID-19 pandemic that may arise from antibiotic use and untreated hospital wastewater.)]\n",
            "2485: [(, , , This will help researchers to understand the current research situation, research trends and research hotspots in this field.)]\n",
            "2486: [(, , , Individual antibiotic use for common infections is a focus for public health efforts seeking to prevent antimicrobial resistance (AMR).)]\n",
            "2487: [(, , , To overcome these unhelpful tendencies and reveal new entry points for AMR prevention, we adopted assemblage theory to analyse personal experience narratives on individual antibiotic use in community settings.)]\n",
            "2488: [(, , , Antibiotic use was irregular, situationally diverse and shaped by factors not always under personal control. Individuals were focussed on preventing, moderating and treating infections that threatened their health.)]\n",
            "2489: [(, , , Our analysis shows that antibiotic assemblages are both cause and effect of individual efforts to manage infections.)]\n",
            "2490: [(, , , Our study performed a statistical analysis of the countries, institutions, journals, authors, research areas, author keywords, Essential Science Indicators (ESI) highly cited papers, and ESI hotspots papers to provide an overview of the ARB field as well as research trends, research hotspots, and future research directions in the field.)]\n",
            "2491: [(, , , Among them, Enterococcus faecalis is an important opportunistic pathogen causing various hospital-acquired infections. A total of 53 E. faecalis isolates were obtained from various infections.)]\n",
            "2492: [(, , , It is also pertinent to create models that can predict the early resistance so that treatment strategies may build up in advance with the evolving resistance.)]\n",
            "2493: [(, , , Modern medicine is threatened by the rising tide of antimicrobial resistance, especially among Gram-negative bacteria, where resistance to β-lactams is most often mediated by β-lactamases.)]\n",
            "2494: [(, , , They were identified by phenotypic and genotypic methods. Determination of antimicrobial resistance patterns was done according to CLSI guidelines.)]\n",
            "2495: [(, , , Detection of antimicrobial resistance genes was performed using standard procedures. According to MDR definition, all of the isolates were MDR (100%).)]\n",
            "2496: [(, , , High-level gentamicin resistance was observed among 50.9% of them (MIC ≥ 500 μg/ml).)]\n",
            "2497: [(, , , The distributions of aac(6')-Ie-aph(2'')-Ia and aph(3')-IIIa genes were 47.2% and 69.8%, respectively.)]\n",
            "2498: [(43, 47, Antibiotic-Resistant Genes, A highly conserved incompatibility complex IncL plasmid scaffold often carries blaOXA-48 and may carry other antimicrobial resistance genes, leaving limited treatment options.)]\n",
            "2499: [(79, 88, Gene/Proteins, A highly conserved incompatibility complex IncL plasmid scaffold often carries blaOXA-48 and may carry other antimicrobial resistance genes, leaving limited treatment options.)]\n",
            "2500: [(4, 8, Gene/Proteins, The ermA and msrA genes were not present in any of the isolates. Our data indicate a high rate of MDR E. faecalis strains.)]\n",
            "2501: [(13, 17, Gene/Proteins, The ermA and msrA genes were not present in any of the isolates. Our data indicate a high rate of MDR E. faecalis strains.)]\n",
            "2502: [(102, 113, Microbes, The ermA and msrA genes were not present in any of the isolates. Our data indicate a high rate of MDR E. faecalis strains.)]\n",
            "2503: [(72, 94, Gene/Proteins, All of high-level gentamicin-resistant isolates carried at least one of aac(6')-Ie-aph(2'')-Ia or aph(3')-IIIa genes.)]\n",
            "2504: [(98, 110, Gene/Proteins, All of high-level gentamicin-resistant isolates carried at least one of aac(6')-Ie-aph(2'')-Ia or aph(3')-IIIa genes.)]\n",
            "2505: [(21, 51, Others, Producers evade most β-lactam-β-lactamase inhibitor combinations, though promising agents have recently been licensed, notably ceftazidime-avibactam and cefiderocol.)]\n",
            "2506: [(16, 20, Antibiotics, Distribution of vanA was notable among the isolates. In addition, ermB and ermC were accountable for resistance to erythromycin.)]\n",
            "2507: [(66, 70, Antibiotics, Distribution of vanA was notable among the isolates. In addition, ermB and ermC were accountable for resistance to erythromycin.)]\n",
            "2508: [(75, 79, Antibiotics, Distribution of vanA was notable among the isolates. In addition, ermB and ermC were accountable for resistance to erythromycin.)]\n",
            "2509: [(115, 127, Antibiotics, Distribution of vanA was notable among the isolates. In addition, ermB and ermC were accountable for resistance to erythromycin.)]\n",
            "2510: [(, , , The objective was to identify the causative organisms for urinary tract infection (UTI) and their sensitivity patterns to antibiotics.)]\n",
            "2511: [(108, 127, Others, The isolates that were non-susceptible to at least one agent in ≥3 antimicrobial categories were defined as multidrug-resistant (MDR).)]\n",
            "2512: [(4, 14, Antibiotics, The penicillin and cephalosporin ascendancies were, in their turn, ended by the proliferation of TEM penicillinases and CTX-M extended-spectrum β-lactamases.)]\n",
            "2513: [(19, 32, Antibiotics, The penicillin and cephalosporin ascendancies were, in their turn, ended by the proliferation of TEM penicillinases and CTX-M extended-spectrum β-lactamases.)]\n",
            "2514: [(97, 115, Gene/Proteins, The penicillin and cephalosporin ascendancies were, in their turn, ended by the proliferation of TEM penicillinases and CTX-M extended-spectrum β-lactamases.)]\n",
            "2515: [(120, 156, Gene/Proteins, The penicillin and cephalosporin ascendancies were, in their turn, ended by the proliferation of TEM penicillinases and CTX-M extended-spectrum β-lactamases.)]\n",
            "2516: [(14, 26, Gene/Proteins, These class A β-lactamases have long been considered the most important. For carbapenems, however, the threat is increasingly from the insidious rise of a class D carbapenemase, OXA-48, and its close relatives.)]\n",
            "2517: [(77, 88, Gene/Proteins, These class A β-lactamases have long been considered the most important. For carbapenems, however, the threat is increasingly from the insidious rise of a class D carbapenemase, OXA-48, and its close relatives.)]\n",
            "2518: [(163, 176, Gene/Proteins, These class A β-lactamases have long been considered the most important. For carbapenems, however, the threat is increasingly from the insidious rise of a class D carbapenemase, OXA-48, and its close relatives.)]\n",
            "2519: [(178, 184, Gene/Proteins, These class A β-lactamases have long been considered the most important. For carbapenems, however, the threat is increasingly from the insidious rise of a class D carbapenemase, OXA-48, and its close relatives.)]\n",
            "2520: [(24, 30, Gene/Proteins, Over the past 20 years, OXA-48 and \"OXA-48-like\" enzymes have proliferated to become the most prevalent enterobacterial carbapenemases across much of Europe, Northern Africa, and the Middle East.)]\n",
            "2521: [(36, 42, Gene/Proteins, Over the past 20 years, OXA-48 and \"OXA-48-like\" enzymes have proliferated to become the most prevalent enterobacterial carbapenemases across much of Europe, Northern Africa, and the Middle East.)]\n",
            "2522: [(0, 6, Gene/Proteins, OXA-48-like enzymes are notoriously difficult to detect because they often cause only low-level in vitro resistance to carbapenems, meaning that the true burden is likely underestimated.)]\n",
            "2523: [(119, 130, Antibiotics, OXA-48-like enzymes are notoriously difficult to detect because they often cause only low-level in vitro resistance to carbapenems, meaning that the true burden is likely underestimated.)]\n",
            "2524: [(83, 93, Antibiotics, The aph(2'')-Ib, aph(2'')-Ic, aph(2'')-Id, and ant(4')-Ia genes were not detected. Vancomycin resistance was found in 45.3% of strains.)]\n",
            "2525: [(39, 49, Antibiotics, Despite this, they are associated with carbapenem treatment failures.)]\n",
            "2526: [(85, 101, Antibiotic-Resistant Bacteria, High conjugation efficiency means that this plasmid is sometimes carried by multiple Enterobacterales in a single patient.)]\n",
            "2527: [(135, 151, Antibiotic-Resistant Bacteria, The molecular machinery enabling global spread, current treatment options, and the development pipeline of potential new therapies for Enterobacterales that produce OXA-48-like β-lactamases form the focus of this review.)]\n",
            "2528: [(165, 189, Gene/Proteins, The molecular machinery enabling global spread, current treatment options, and the development pipeline of potential new therapies for Enterobacterales that produce OXA-48-like β-lactamases form the focus of this review.)]\n",
            "2529: [(, , , One of the major global health care crises in the 21st century is antibiotic resistance.)]\n",
            "2530: [(, , , Rapid spread of multidrug resistant microorganisms is a matter of great concern throughout the glove including Bangladesh.)]\n",
            "2531: [(, , , Almost all clinically used antibiotics have resistance emerging to them.)]\n",
            "2532: [(, , , Antibiotic Resistance can be regarded as the 'Faceless Pandemic' that has enthralled the entire world.)]\n",
            "2533: [(, , , It has become peremptory to develop treatment options as an alternative to antibiotic therapy for combating antibiotic-resistant pathogens.)]\n",
            "2534: [(, , , A clearer understanding of antibiotic resistance is required to prevent the rapid spread of antibiotic-resistant genes and the re-emergence of infections.)]\n",
            "2535: [(, , , published the greatest number of articles; CM Manaia has the most contributions; Environmental Sciences and Ecology is the most popular research area; and \"antibiotic resistance,\" \"antibiotics,\" and \"antibiotic resistance genes\" were the most frequently occurring author keywords. A citation analysis showed that aquatic environment-related antibiotic resistance is a key research area in this field, while antimicrobial nanomaterial-related research is a recent popular topic.)]\n",
            "2536: [(, , , The present review provides an insight into the different classifications and modes of action of antibiotics to understand how the hosts develop resistance to them.)]\n",
            "2537: [(, , , Multiple drug-resistant enterococci are major cause of healthcare-associated infections due to their antibiotic resistance traits.)]\n",
            "2538: [(, , , In addition, the association of genetics in the development of antibiotic resistance and environmental factors has also been discussed, emphasizing developing action plans to counter this \"quiescent pandemic\".)]\n",
            "2539: [(, , , In vitro experimental evolution has complemented clinical studies as an excellent tool to identify genetic changes responsible for the de novo evolution of antimicrobial resistance.)]\n",
            "2540: [(, , , E. coli was the predominant organism responsible for UTI and was resistant to most of the first- and second-line antibiotics.)]\n",
            "2541: [(, , , However, the in vivo context for adaptation contributes to the success of particular evolutionary trajectories, especially in intracellular niches where the adaptive landscape of virulence and resistance are strongly coupled.)]\n",
            "2542: [(, , , In this work, we designed an ex vivo evolution approach to identify evolutionary trajectories responsible for antibiotic resistance in the Live Vaccine Strain (LVS) of Francisella tularensis subsp.)]\n",
            "2543: [(, , , Immediate action is needed to develop empirical guideline for empirical management of UTI and establish surveillance system for monitoring.)]\n",
            "2544: [(, , , Patients with sepsis present across a spectrum of infection sites and severity of illnesses requiring complex decision making at the bedside as to when prompt antibiotics are indicated and which regimen is warranted.)]\n",
            "2545: [(, , , Many hemodynamically stable patients with sepsis and low acuity of illness may benefit from further work up before initiating therapy, whereas patients with septic shock warrant emergent broad-spectrum antibiotics.)]\n",
            "2546: [(, , , Overall, adaptation within macrophages advanced much slower when compared to previous in vitro evolution studies reflecting a limiting capacity for the expansion of adaptive mutations within the macrophage.)]\n",
            "2547: [(, , , The precise empiric regimen is determined by assessing patient and epidemiological risk factors, likely source of infection based on presenting signs and symptoms, and severity of illness.)]\n",
            "2548: [(, , , Hospitals should implement quality improvement measures to aid in the rapid and accurate diagnosis of septic patients and to ensure antibiotics are given to patients in an expedited fashion after antibiotic order.)]\n",
            "2549: [(, , , We are losing our ability to treat infectious diseases for a number of reasons, including antimicrobial resistance (AMR).)]\n",
            "2550: [(66, 76, Microbes, Antibiotic resistome is a dynamic and ever-expanding component in Salmonella.)]\n",
            "2551: [(102, 112, Microbes, The foundation of resistome in Salmonella is laid long before; therefore, the antibiotic resistome of Salmonella is reviewed, discussed, and summarized.)]\n",
            "2552: [(31, 41, Microbes, The foundation of resistome in Salmonella is laid long before; therefore, the antibiotic resistome of Salmonella is reviewed, discussed, and summarized.)]\n",
            "2553: [(109, 119, Microbes, We have searched the literature using PubMed, MEDLINE, and Google Scholar with related key terms (resistome, Salmonella, antibiotics, drug resistance) and prepared this review.)]\n",
            "2554: [(38, 44, Techniques, We have searched the literature using PubMed, MEDLINE, and Google Scholar with related key terms (resistome, Salmonella, antibiotics, drug resistance) and prepared this review.)]\n",
            "2555: [(46, 53, Techniques, We have searched the literature using PubMed, MEDLINE, and Google Scholar with related key terms (resistome, Salmonella, antibiotics, drug resistance) and prepared this review.)]\n",
            "2556: [(27, 84, Antibiotic-Resistant Bacteria, We report the emergence of imipenem-relebactam nonsusceptible Pseudomonas aeruginosa in 5 patients treated for nosocomial pneumonia for 10-28 days.)]\n",
            "2557: [(125, 132, Antibiotic-Resistant Bacteria, Median age of patients was 35 years and 80% were female. The main organisms isolated from urine culture of UTI patients were E. coli (64%), Klebsiella (12%) and Enterococci species (10%).)]\n",
            "2558: [(140, 150, Antibiotic-Resistant Bacteria, Median age of patients was 35 years and 80% were female. The main organisms isolated from urine culture of UTI patients were E. coli (64%), Klebsiella (12%) and Enterococci species (10%).)]\n",
            "2559: [(161, 172, Antibiotic-Resistant Bacteria, Median age of patients was 35 years and 80% were female. The main organisms isolated from urine culture of UTI patients were E. coli (64%), Klebsiella (12%) and Enterococci species (10%).)]\n",
            "2560: [(68, 73, Gene/Proteins, Genome sequence analysis identified treatment-emergent mutations in MexAB-OprM and/or MexEF-OprN efflux operons that arose independently in each patient across distinct P. aeruginosa sequence types.)]\n",
            "2561: [(86, 91, Gene/Proteins, Genome sequence analysis identified treatment-emergent mutations in MexAB-OprM and/or MexEF-OprN efflux operons that arose independently in each patient across distinct P. aeruginosa sequence types.)]\n",
            "2562: [(52, 59, Antibiotic-Resistant Bacteria, Susceptibility to antibiotics was analyzed only for E. coli (n=27) since the number of isolates of other organisms were small.)]\n",
            "2563: [(0, 7, Microbes, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2564: [(93, 104, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2565: [(113, 122, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2566: [(130, 144, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2567: [(192, 203, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2568: [(290, 302, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2569: [(270, 278, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2570: [(230, 243, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2571: [(211, 221, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2572: [(174, 184, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2573: [(152, 166, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2574: [(251, 262, Antibiotics, E. coli was found to be resistant to most of the first- and second-line antibiotics, such as Amoxicillin (100%), Amoxyclav (72%), Co-trimoxazole (89%), Nalidixic acid (78%), Ceftazidim (94%), Ceftriaxone (73%), Cefuroxime (100%), Ciprofloxacin (59%), Cephotaxime (80%), Cefixime (100%) and Moxifloxacin (100%).)]\n",
            "2575: [(110, 121, Others, holarctica while being passaged to increasing ciprofloxacin (CIP) and doxycycline (DOX) concentrations within macrophages.)]\n",
            "2576: [(92, 101, Antibiotics, Longitudinal genomic analysis identified resistance conferring gyrase mutations outside the Quinolone Resistance Determining Region.)]\n",
            "2577: [(12, 18, Gene/Proteins, Strikingly, FupA/B mutations that are uniquely associated with in vitro CIP resistance in Francisella were not observed ex vivo, reflecting the coupling of intracellular survival and resistance during intracellular adaptation.)]\n",
            "2578: [(72, 75, Antibiotics, Strikingly, FupA/B mutations that are uniquely associated with in vitro CIP resistance in Francisella were not observed ex vivo, reflecting the coupling of intracellular survival and resistance during intracellular adaptation.)]\n",
            "2579: [(90, 101, Microbes, Strikingly, FupA/B mutations that are uniquely associated with in vitro CIP resistance in Francisella were not observed ex vivo, reflecting the coupling of intracellular survival and resistance during intracellular adaptation.)]\n",
            "2580: [(, , , Resistome governs essential and separate biological functions to develop resistance against antibiotics.)]\n",
            "2581: [(, , , The results provide evidence of differences in mutational profiles of populations adapted to the same antibiotic in different environments/cellular compartments and underscore the significance of host mediated stress during resistance evolution.)]\n",
            "2582: [(, , , The widespread clinical and nonclinical uses of antibiotics over the years have combined to select antibiotic-resistant determinants and develop resistome in bacteria.)]\n",
            "2583: [(, , , The pipeline for new antibiotics is dry.)]\n",
            "2584: [(, , , Despite the creation of public/private initiatives like Combating Antibiotic Resistant Bacteria Biopharmaceutical Accelerator (Carb-X) and the Antimicrobial Resistance (AMR) Centre, the current focus on 'push-pull' incentives for the pharmaceutical industry still relies on economic return.)]\n",
            "2585: [(, , , We propose a joint, internationally-funded antimicrobial development institute that would fund permanent staff to take on roles previously assigned to pharmaceutical companies.)]\n",
            "2586: [(, , , Of the 60 patients, culture and sensitivity report was available for 42 patients. Therefore, data were further analyzed for these 42 cases.)]\n",
            "2587: [(132, 142, Microbes, At present, the emergence of drug resistance because of resistome is a significant problem faced by clinicians for the treatment of Salmonella infection.)]\n",
            "2588: [(, , , Testing with efflux-inhibitor PAβN restored imipenem-relebactam susceptibility.)]\n",
            "2589: [(, , , This institute would receive ring-fenced, long-term, core funding from participating countries as well as charities, with the aim to focus on transforming the largely dormant antimicrobial pipeline.)]\n",
            "2590: [(, , , In the present study, we elaborate the mechanism underlying the effects of pesticides on bacterial antibiotic resistance acquisition as well as the propagation of antimicrobial resistance.)]\n",
            "2591: [(, , , In an Essity-sponsored symposium, entitled 'Antimicrobial resistance-is wound care prepared?', four key opinion leaders discussed the difficult journey to managing antimicrobial resistance within wound care at the global and local levels, backed up by the latest research.)]\n",
            "2592: [(, , , Pesticide stress enhanced the acquisition of antibiotic resistance in bacteria via various mechanisms, including the activation of efflux pumps, inhibition of outer membrane pores for resistance to antibiotics, and gene mutation induction.)]\n",
            "2593: [(, , , Horizontal gene transfer is a major mechanism whereby pesticides influence the transmission of antibiotic resistance genes (ARGs) in bacteria.)]\n",
            "2594: [(, , , Peter Bradley summarises the main points they presented at the EWMA 2020 Virtual Conference.)]\n",
            "2595: [(, , , This article offers an introduction to the symposium, Antimicrobial resistance-is wound care prepared?, held on 18 November 2020, as part of the EWMA 2020 Virtual Conference.)]\n",
            "2596: [(, , , Pesticides promoted the conjugation transfer of ARGs by increasing cell membrane permeability and increased the proportion of bacterial mobile gene elements, which facilitate the spread of ARGs.)]\n",
            "2597: [(, , , There were four speakers. Claire Kilpatrick, Director, S3 Global and Consultant to the World Health Organization, spoke on 'Preventing infections and managing antibiotic use-tackling a crisis for the health of nations'.)]\n",
            "2598: [(, , , This review can improve our understanding regarding the pesticide-induced generation and spread of ARGs and antibiotic resistant bacteria.)]\n",
            "2599: [(, , , The topic of Karen Ousey, Professor of Skin Integrity, Director for the Institute of Skin Integrity and Infection Prevention, University of Huddersfield, was 'Antimicrobial stewardship in wound management-less words more action'.)]\n",
            "2600: [(, , , Moreover, it can be applied to reduce the ecological risks of ARGs in the future.)]\n",
            "2601: [(, , , Finally, Hadar Lev-Tov, Assistant Professor, and Irena Pastar, Associate Professor, both of the Dr Phillip Frost Department of Dermatology and Cutaneous Surgery, University of Miami, gave a talk on 'Silver-resistant bacteria in venous leg ulcers? First results of an exploratory study'.)]\n",
            "2602: [(, , , The objective of this study was to determine the phenotypic patterns of antibiotic resistance and the epidemiology of drug-resistant Campylobacter spp. from a low-resource setting. A birth cohort of 303 patients was followed until 5 years of age.)]\n",
            "2603: [(, , , AMR is a 'slow-moving' threat, which makes it harder to recognize and address.)]\n",
            "2604: [(41, 51, Microbes, This feature is particularly harmful for nosocomial pathogens that can survive on hospital surfaces.)]\n",
            "2605: [(, , , However, this potential application of CNTs presupposes the absence of toxicity and therefore total safety for patients.)]\n",
            "2606: [(41, 53, Gene/Proteins, Further, it highlights the importance of bacteriocins in lowering the minimum inhibitory concentrations of conventional antibiotics when used in combination.)]\n",
            "2607: [(, , , Research studies have been conducted to evaluate new materials that are able to counteract the microbial growth and the colonization of the hospital environment.)]\n",
            "2608: [(, , , In this context, nanotechnologies have showed encouraging applications. We investigated the antibacterial activity of multi-walled carbon nanotubes (MWCNTs), both pristine (p) and functionalized (f), at concentrations of 50 and 100 μg mL-1, against bacterial strains isolated from hospital-acquired infections, and this activity was correlated with the antibiotic susceptibility of the strains.)]\n",
            "2609: [(, , , The inhibiting effect of MWCNTs occurred for both types and doses tested.)]\n",
            "2610: [(, , , Moreover, f-MWCNTs exerted a greater inhibiting effect, with growth decreases greater than 10% at 24 h and 20% at 48 h compared to p-MWCNTs. Moreover, a lower inhibitory effect of MWCNTs, which was more lasting in Gram-positives resistant to cell wall antibiotics, or temporary in Gram-negatives resistant to nucleic acid and protein synthesis inhibitors, was observed, highlighting the strong relation between antibiotic resistance and MWCNT effect.)]\n",
            "2611: [(, , , Resulting drugs would be sold globally and according to a principle of shared burdens.)]\n",
            "2612: [(, , , In conclusion, an antimicrobial activity was observed especially for f-MWCNTs that could therefore be loaded with bioactive antimicrobial molecules.)]\n",
            "2613: [(, , , Our proposed model for antimicrobial development aims to maximise society's investment, through open science, investment in people, and the sharing of intellectual property.)]\n",
            "2614: [(, , , An increasing rate of antibiotic resistance among Gram-negative bacterial pathogens has created an urgent need to discover novel therapeutic agents to combat infectious diseases.)]\n",
            "2615: [(, , , The spread of microbial antibiotic resistance has seriously threatened public health globally.)]\n",
            "2616: [(, , , Use of bacteriocins as therapeutic agents has immense potential due to their high potency and mode of action different from that of conventional antibiotics.)]\n",
            "2617: [(, , , Non-antibiotic stressors have significantly contributed to the evolution of bacterial antibiotic resistance.)]\n",
            "2618: [(, , , Thus, this study shows the broad spectrum antimicrobial activity of novel enterocin E20c against various MDR pathogens.)]\n",
            "2619: [(, , , Although numerous studies have been conducted on the potential risk of pesticide pollution for bacterial antibiotic resistance, a systematic review of these concerns is still lacking.)]\n",
            "2620: [(, , , However, a more extensive and precise assessment is needed to understand the disease burden of AMR more clearly and enable us to compare these indicators with those published by other countries.)]\n",
            "2621: [(, , , The decrease over time of AMR in APEC likely reflects the efficacy of manifold improvements in broiler breeder production systems)]\n",
            "2622: [(, , , Their study could be considered an important milestone in terms of its thoroughness.)]\n",
            "2623: [(, , , A further reduction in AMR is still desirable.)]\n",
            "2624: [(, , , If we hope to estimate the disease burden of AMR in a more precise manner, age-stratified patient data is needed in conjunction with a surveillance report.)]\n",
            "2625: [(, , , Multidrug resistance decreased from 100% in 2009 to 48% in 2018.Further reduction of antimicrobial resistance in broiler breeders desirable.)]\n",
            "2626: [(, , , Foodborne pathogens such as Salmonella that survive cleaning and disinfection during poultry processing are a public health concern because pathogens that survive disinfectants have greater potential to exhibit resistance to antibiotics and disinfectants after their initial disinfectant challenge.)]\n",
            "2627: [(, , , Cassini and colleagues from the World Health Organization estimated the disease burden of AMR in the European Union as disability-adjusted life years (DALYs) in 2018.)]\n",
            "2628: [(, , , Similarly, the overall AMR of the whole period was calculated.)]\n",
            "2629: [(26, 46, Microbes, Antibiotic resistances in Campylobacter jejuni and non-C. jejuni isolates from surveillance and diarrhea samples were compared, and the association between personal macrolide exposure and subsequent occurrence of a macrolide-resistant Campylobacter spp. was assessed.)]\n",
            "2630: [(55, 64, Microbes, Antibiotic resistances in Campylobacter jejuni and non-C. jejuni isolates from surveillance and diarrhea samples were compared, and the association between personal macrolide exposure and subsequent occurrence of a macrolide-resistant Campylobacter spp. was assessed.)]\n",
            "2631: [(235, 252, Microbes, Antibiotic resistances in Campylobacter jejuni and non-C. jejuni isolates from surveillance and diarrhea samples were compared, and the association between personal macrolide exposure and subsequent occurrence of a macrolide-resistant Campylobacter spp. was assessed.)]\n",
            "2632: [(215, 224, Antibiotics, Antibiotic resistances in Campylobacter jejuni and non-C. jejuni isolates from surveillance and diarrhea samples were compared, and the association between personal macrolide exposure and subsequent occurrence of a macrolide-resistant Campylobacter spp. was assessed.)]\n",
            "2633: [(165, 174, Antibiotics, Antibiotic resistances in Campylobacter jejuni and non-C. jejuni isolates from surveillance and diarrhea samples were compared, and the association between personal macrolide exposure and subsequent occurrence of a macrolide-resistant Campylobacter spp. was assessed.)]\n",
            "2634: [(7, 20, Microbes, Of 917 Campylobacter isolates, 77.4% of C. jejuni isolates and 79.8% of non-C. jejuni isolates were resistant to ciprofloxacin, while 4.9% of C. jejuni isolates and 24.8% of non-C. jejuni isolates were not susceptible to azithromycin.)]\n",
            "2635: [(40, 49, Microbes, Of 917 Campylobacter isolates, 77.4% of C. jejuni isolates and 79.8% of non-C. jejuni isolates were resistant to ciprofloxacin, while 4.9% of C. jejuni isolates and 24.8% of non-C. jejuni isolates were not susceptible to azithromycin.)]\n",
            "2636: [(113, 126, Antibiotics, Of 917 Campylobacter isolates, 77.4% of C. jejuni isolates and 79.8% of non-C. jejuni isolates were resistant to ciprofloxacin, while 4.9% of C. jejuni isolates and 24.8% of non-C. jejuni isolates were not susceptible to azithromycin.)]\n",
            "2637: [(76, 85, Microbes, Of 917 Campylobacter isolates, 77.4% of C. jejuni isolates and 79.8% of non-C. jejuni isolates were resistant to ciprofloxacin, while 4.9% of C. jejuni isolates and 24.8% of non-C. jejuni isolates were not susceptible to azithromycin.)]\n",
            "2638: [(142, 151, Microbes, Of 917 Campylobacter isolates, 77.4% of C. jejuni isolates and 79.8% of non-C. jejuni isolates were resistant to ciprofloxacin, while 4.9% of C. jejuni isolates and 24.8% of non-C. jejuni isolates were not susceptible to azithromycin.)]\n",
            "2639: [(178, 187, Microbes, Of 917 Campylobacter isolates, 77.4% of C. jejuni isolates and 79.8% of non-C. jejuni isolates were resistant to ciprofloxacin, while 4.9% of C. jejuni isolates and 24.8% of non-C. jejuni isolates were not susceptible to azithromycin.)]\n",
            "2640: [(221, 233, Antibiotics, Of 917 Campylobacter isolates, 77.4% of C. jejuni isolates and 79.8% of non-C. jejuni isolates were resistant to ciprofloxacin, while 4.9% of C. jejuni isolates and 24.8% of non-C. jejuni isolates were not susceptible to azithromycin.)]\n",
            "2641: [(, , , Colibacillosis is one of the most important diseases in poultry production.)]\n",
            "2642: [(53, 66, Antibiotic-Resistant Bacteria, Of the 303 children, 33.1% had been diagnosed with a Campylobacter strain nonsusceptible to both azithromycin and ciprofloxacin.)]\n",
            "2643: [(97, 109, Antibiotics, Of the 303 children, 33.1% had been diagnosed with a Campylobacter strain nonsusceptible to both azithromycin and ciprofloxacin.)]\n",
            "2644: [(114, 127, Antibiotics, Of the 303 children, 33.1% had been diagnosed with a Campylobacter strain nonsusceptible to both azithromycin and ciprofloxacin.)]\n",
            "2645: [(105, 120, Antibiotics, Personal macrolide exposure did not affect the risk of macrolide-resistant Campylobacter Amoxicillin and clavulanic acid (94.0%) was one of the antibiotics with the highest rates of susceptibility.)]\n",
            "2646: [(75, 100, Microbes, Personal macrolide exposure did not affect the risk of macrolide-resistant Campylobacter Amoxicillin and clavulanic acid (94.0%) was one of the antibiotics with the highest rates of susceptibility.)]\n",
            "2647: [(55, 64, Antibiotics, Personal macrolide exposure did not affect the risk of macrolide-resistant Campylobacter Amoxicillin and clavulanic acid (94.0%) was one of the antibiotics with the highest rates of susceptibility.)]\n",
            "2648: [(67, 99, Techniques, The APEC isolates were tested against eleven antimicrobials by the Kirby-Bauer disc diffusion test.)]\n",
            "2649: [(4, 8, Antibiotic-Resistant Bacteria, The APEC isolates were tested against eleven antimicrobials by the Kirby-Bauer disc diffusion test.)]\n",
            "2650: [(14, 43, Others, Recently, the AMR Clinical Reference Center (AMR CRC) in Japan estimated the disease burden of AMR in the form of number of deaths attributed to blood-stream infections caused by antimicrobial-resistant organisms.)]\n",
            "2651: [(18, 22, Antibiotics, Only seven (2.7%) APEC strains were fully susceptible to all tested antimicrobials.)]\n",
            "2652: [(84, 91, Antibiotic-Resistant Bacteria, RESEARCH HIGHLIGHTSDecreasing trend of antimicrobial resistance in avian pathogenic E. coli over time.Over 50% of isolates still resistant to amoxicillin, tetracycline and doxycycline.)]\n",
            "2653: [(142, 153, Antibiotics, RESEARCH HIGHLIGHTSDecreasing trend of antimicrobial resistance in avian pathogenic E. coli over time.Over 50% of isolates still resistant to amoxicillin, tetracycline and doxycycline.)]\n",
            "2654: [(155, 167, Antibiotics, RESEARCH HIGHLIGHTSDecreasing trend of antimicrobial resistance in avian pathogenic E. coli over time.Over 50% of isolates still resistant to amoxicillin, tetracycline and doxycycline.)]\n",
            "2655: [(172, 183, Antibiotics, RESEARCH HIGHLIGHTSDecreasing trend of antimicrobial resistance in avian pathogenic E. coli over time.Over 50% of isolates still resistant to amoxicillin, tetracycline and doxycycline.)]\n",
            "2656: [(12, 19, Others, At present, AMR CRC is attempting to establish such data for examination at the national level - a challenging but worthwhile task.)]\n",
            "2657: [(153, 163, Antibiotic-Resistant Bacteria, While the mechanisms conferring antimicrobial resistance (AMR) after exposure to disinfectants is complex, understanding the effects of disinfectants on Salmonella in both their planktonic and biofilm states is becoming increasingly important, as AMR and disinfectant tolerant bacteria are becoming more prevalent in the food chain.)]\n",
            "2658: [(, , , The goal of this review article is to characterize the AMR profiles of Salmonella in both their planktonic and biofilm state that have been challenged with hexadecylpyridinium chloride (HDP), peracetic acid (PAA), sodium hypochlorite (SHY) and trisodium phosphate (TSP) in order to understand the risk of these disinfectants inducing AMR in surviving bacteria that may enter the food chain.)]\n",
            "2659: [(81, 88, Antibiotic-Resistant Bacteria, The use of antimicrobials remains a therapeutic cornerstone for avian pathogenic E. coli (APEC), thereby contributing to the development of antimicrobial resistance (AMR).)]\n",
            "2660: [(, , , The aim of this study was to characterize AMR in broiler breeder flocks reared under commercial conditions)]\n",
            "2661: [(29, 77, Antibiotic-Resistant Bacteria, There is a high incidence of quinolone- and macrolide-resistant Campylobacter infections in infants under 24 months of age.)]\n",
            "2662: [(, , , Data covering 10 years, from 2009 to 2018, were used to evaluate the phenotypic AMR of 264 APEC isolates obtained from 158 broiler breeder flocks of a large company in Portugal.)]\n",
            "2663: [(59, 69, Antibiotics, Given the lack of association between personal exposure to macrolides and a subsequent Campylobacter infection resistant to macrolides, there is a need to evaluate the source of multidrug-resistant (MDR) Campylobacter This study provides compelling evidence to propose amoxicillin/clavulanic acid as a treatment for campylobacteriosis.)]\n",
            "2664: [(87, 100, Antibiotic-Resistant Bacteria, Given the lack of association between personal exposure to macrolides and a subsequent Campylobacter infection resistant to macrolides, there is a need to evaluate the source of multidrug-resistant (MDR) Campylobacter This study provides compelling evidence to propose amoxicillin/clavulanic acid as a treatment for campylobacteriosis.)]\n",
            "2665: [(124, 134, Antibiotics, Given the lack of association between personal exposure to macrolides and a subsequent Campylobacter infection resistant to macrolides, there is a need to evaluate the source of multidrug-resistant (MDR) Campylobacter This study provides compelling evidence to propose amoxicillin/clavulanic acid as a treatment for campylobacteriosis.)]\n",
            "2666: [(204, 217, Antibiotic-Resistant Bacteria, Given the lack of association between personal exposure to macrolides and a subsequent Campylobacter infection resistant to macrolides, there is a need to evaluate the source of multidrug-resistant (MDR) Campylobacter This study provides compelling evidence to propose amoxicillin/clavulanic acid as a treatment for campylobacteriosis.)]\n",
            "2667: [(269, 280, Antibiotics, Given the lack of association between personal exposure to macrolides and a subsequent Campylobacter infection resistant to macrolides, there is a need to evaluate the source of multidrug-resistant (MDR) Campylobacter This study provides compelling evidence to propose amoxicillin/clavulanic acid as a treatment for campylobacteriosis.)]\n",
            "2668: [(281, 296, Antibiotics, Given the lack of association between personal exposure to macrolides and a subsequent Campylobacter infection resistant to macrolides, there is a need to evaluate the source of multidrug-resistant (MDR) Campylobacter This study provides compelling evidence to propose amoxicillin/clavulanic acid as a treatment for campylobacteriosis.)]\n",
            "2669: [(, , , Antimicrobial resistance (AMR) is currently a global health threat.)]\n",
            "2670: [(, , , The annual proportion of AMR was calculated by dividing the number of APEC isolates with phenotypic resistance by the total number of APEC isolated that year.)]\n",
            "2671: [(, , , Many countries have issued their own national action plans following the publication of the Global Action Plan on AMR by the World Health Organization.)]\n",
            "2672: [(, , , The relationship of antimicrobial resistance with time (years) was investigated with a generalized linear model using logistic regression.)]\n",
            "2673: [(, , , The government of Japan established its own National Action Plan on AMR in 2016; however, Japan's AMR countermeasures are still in the developmental stage.)]\n",
            "2674: [(, , , The review also covers different treatment technologies being developed for AMR genes as an environmental contaminant.)]\n",
            "2675: [(, , , Additionally, an abundance of AMR was agreeable between metagenomic and whole genome sequencing analysis for several AMR classes in poultry fecal samples suggesting that metagenomics could be used as an indicator for surveillance of AMR in E. coli isolates and vice versa.)]\n",
            "2676: [(, , , The progress made in the area of AMR gene databases and tools is also reviewed.)]\n",
            "2677: [(, , , In this article I suggest a number of ways in which this can be achieved.)]\n",
            "2678: [(, , , Neisseria gonorrhoeae antimicrobial resistance (AMR) surveillance is essential for tracking the emergence and spread of AMR strains in local, national and international populations.)]\n",
            "2679: [(, , , Besides a brief introduction, the content is categorized into three main sections: i) Occurrence of AMR in the Environment, ii) Treatment technologies for AMR, and iii) AMR databases and tools.)]\n",
            "2680: [(, , , Antimicrobial resistance (AMR) is a global problem to animal and public health.)]\n",
            "2681: [(, , , Elucidation of AMR determinants to β-lactam drugs, is a means of monitoring the development of resistance.)]\n",
            "2682: [(, , , It has drawn the attention of public health experts, stakeholders, and medical science due to the substantial economic loss that it causes to individuals and nation as a whole.)]\n",
            "2683: [(, , , Various cross-sectional studies and some national surveys in developing countries have shown increase in the burden of antimicrobial resistance.)]\n",
            "2684: [(, , , It is imperative to continue with enhanced AMR surveillance and to understand the molecular epidemiology of gonococcal strains circulating in Ghana and other African countries.)]\n",
            "2685: [(, , , Antibiotics underpin modern medicine and are critical for pandemic preparedness.)]\n",
            "2686: [(, , , Improved prioritization and government oversight by funders who allocate public resources are a needed first step.)]\n",
            "2687: [(, , , China's AMR governance has fallen behind at the beginning but recently began to catch up with the global trend.)]\n",
            "2688: [(26, 38, Antibiotics, The isolate with the high cefixime MIC contained a mosaic penA-34 allele and belonged to NG-MAST ST1407, an internationally spreading multidrug-resistant clone that has accounted for most cefixime resistance in many countries.)]\n",
            "2689: [(89, 103, Antibiotic-Resistant Genes, The isolate with the high cefixime MIC contained a mosaic penA-34 allele and belonged to NG-MAST ST1407, an internationally spreading multidrug-resistant clone that has accounted for most cefixime resistance in many countries.)]\n",
            "2690: [(51, 65, Antibiotic-Resistant Genes, The isolate with the high cefixime MIC contained a mosaic penA-34 allele and belonged to NG-MAST ST1407, an internationally spreading multidrug-resistant clone that has accounted for most cefixime resistance in many countries.)]\n",
            "2691: [(188, 196, Antibiotics, The isolate with the high cefixime MIC contained a mosaic penA-34 allele and belonged to NG-MAST ST1407, an internationally spreading multidrug-resistant clone that has accounted for most cefixime resistance in many countries.)]\n",
            "2692: [(, , , The COVID pandemic has changed this. Despite the dire predictions of the effect of AMR in the future, we still have time to change course.)]\n",
            "2693: [(132, 148, Antibiotics, In conclusion, AMR testing, NG-MAST, and sequencing of the AMR determinant penA, revealed high rates of resistance to tetracycline, benzylpenicillin, and ciprofloxacin; as well as a highly diverse population of N. gonorrhoeae in Ghana.)]\n",
            "2694: [(15, 26, Techniques, In conclusion, AMR testing, NG-MAST, and sequencing of the AMR determinant penA, revealed high rates of resistance to tetracycline, benzylpenicillin, and ciprofloxacin; as well as a highly diverse population of N. gonorrhoeae in Ghana.)]\n",
            "2695: [(28, 35, Techniques, In conclusion, AMR testing, NG-MAST, and sequencing of the AMR determinant penA, revealed high rates of resistance to tetracycline, benzylpenicillin, and ciprofloxacin; as well as a highly diverse population of N. gonorrhoeae in Ghana.)]\n",
            "2696: [(41, 79, Techniques, In conclusion, AMR testing, NG-MAST, and sequencing of the AMR determinant penA, revealed high rates of resistance to tetracycline, benzylpenicillin, and ciprofloxacin; as well as a highly diverse population of N. gonorrhoeae in Ghana.)]\n",
            "2697: [(118, 130, Antibiotics, In conclusion, AMR testing, NG-MAST, and sequencing of the AMR determinant penA, revealed high rates of resistance to tetracycline, benzylpenicillin, and ciprofloxacin; as well as a highly diverse population of N. gonorrhoeae in Ghana.)]\n",
            "2698: [(154, 167, Antibiotics, In conclusion, AMR testing, NG-MAST, and sequencing of the AMR determinant penA, revealed high rates of resistance to tetracycline, benzylpenicillin, and ciprofloxacin; as well as a highly diverse population of N. gonorrhoeae in Ghana.)]\n",
            "2699: [(211, 225, Microbes, In conclusion, AMR testing, NG-MAST, and sequencing of the AMR determinant penA, revealed high rates of resistance to tetracycline, benzylpenicillin, and ciprofloxacin; as well as a highly diverse population of N. gonorrhoeae in Ghana.)]\n",
            "2700: [(112, 118, Others, Push funding has revitalized the preclinical antimicrobial resistance (AMR) pipeline and government funding via CARB-X and BARDA, as well as private sector-led investment via the AMR Action Fund, will help several new antibiotics obtain regulatory approval.)]\n",
            "2701: [(123, 128, Others, Push funding has revitalized the preclinical antimicrobial resistance (AMR) pipeline and government funding via CARB-X and BARDA, as well as private sector-led investment via the AMR Action Fund, will help several new antibiotics obtain regulatory approval.)]\n",
            "2702: [(0, 30, Others, Mobile genetic elements (MGEs) also play a central role in facilitating the acquisition and spread of AMR genes.)]\n",
            "2703: [(12, 28, Antibiotic-Resistant Bacteria, We isolated Escherichia coli (n = 627) from fecal samples in respectively 25 poultry, 28 swine, and 15 veal calf herds from 6 European countries to investigate the phylogeny of E. coli at country, animal host and farm levels.)]\n",
            "2704: [(177, 184, Antibiotic-Resistant Bacteria, We isolated Escherichia coli (n = 627) from fecal samples in respectively 25 poultry, 28 swine, and 15 veal calf herds from 6 European countries to investigate the phylogeny of E. coli at country, animal host and farm levels.)]\n",
            "2705: [(4, 11, Microbes, The E. coli isolates in this study indicated no clonality or clustering based on country of origin and genetic markers; AMR, and MGEs. Nonetheless, mobile genetic elements play a role in the acquisition of AMR and virulence genes.)]\n",
            "2706: [(64, 118, Techniques, This is crucial for developing or refining treatment guidelines. N. gonorrhoeae multiantigen sequence typing (NG-MAST) is beneficial for describing the molecular epidemiology of gonococci at national and international levels.)]\n",
            "2707: [(44, 54, Antibiotic-Resistant Bacteria, In Ghana, little is known about the current gonococcal AMR prevalence and no characterization of gonococcal isolates has been previously performed.)]\n",
            "2708: [(97, 107, Antibiotic-Resistant Bacteria, In Ghana, little is known about the current gonococcal AMR prevalence and no characterization of gonococcal isolates has been previously performed.)]\n",
            "2709: [(143, 150, Techniques, In this study, gonococcal isolates (n = 44) collected from five health facilities in Ghana from 2012 to 2015, were examined using AMR testing, NG-MAST and sequencing of penA.)]\n",
            "2710: [(130, 141, Techniques, In this study, gonococcal isolates (n = 44) collected from five health facilities in Ghana from 2012 to 2015, were examined using AMR testing, NG-MAST and sequencing of penA.)]\n",
            "2711: [(169, 173, Gene/Proteins, In this study, gonococcal isolates (n = 44) collected from five health facilities in Ghana from 2012 to 2015, were examined using AMR testing, NG-MAST and sequencing of penA.)]\n",
            "2712: [(, , , The situation has not been helped by the difficulty we have had in seeing the actions and health of everyone on this planet as interconnected.)]\n",
            "2713: [(, , , Advocacy by scientists and health professionals is a powerful tool in this process, but there are pitfalls and it must be used wisely.)]\n",
            "2714: [(, , , The emergence of antimicrobial resistance (AMR) is one of the biggest health threats globally.)]\n",
            "2715: [(, , , In addition, the use of antimicrobial drugs in humans and livestock is considered an important driver of antimicrobial resistance.)]\n",
            "2716: [(, , , Nevertheless, revenues generated by new antibiotics are not considered sufficiently profitable by commercial developers to address unmet need.)]\n",
            "2717: [(, , , The commensal microbiota, and especially the intestinal microbiota, has been shown to have an important role in the emergence of AMR)]\n",
            "2718: [(, , , The question remains: Who could viably fund development and secure global equitable access for new antibiotics? Public health need should be the primary driver for antibiotic development.)]\n",
            "2719: [(, , , In this framework, nonprofit research and development organizations, with support from public funders, and unconstrained by commercial profitability requirements are well positioned to work with public and private actors to viably provide new antibiotics to all in need.)]\n",
            "2720: [(, , , This review su mmarizes selected publications from 2017 highlighting the occurrence of antimicrobial resistance (AMR) genes in the environment with emphasis on the aquatic environment.)]\n",
            "2721: [(, , , In this review, we provide a brief overview of AMR development processes, the worldwide burden and drivers of AMR, current knowledge gaps, monitoring methodologies, and global mitigation measures in the development and spread of AMR in the environment.)]\n",
            "2722: [(, , , The development of such standards alongside a centralised reporting system would provide baseline data to feedback directly into wastewater treatment policies for the Antarctic Treaty Area to help preserve this relatively pristine environment. Video Abstract.)]\n",
            "2723: [(, , , Antimicrobial resistance (AMR) is one of the most challenging threats in public health; thus, there is a growing demand for methods and technologies that enable rapid antimicrobial susceptibility testing (AST))]\n",
            "2724: [(, , , Establishment of a surveillance programme and a national plan for containment of AMR, following the National Antibiotics Treatment Guideline 2014 and generation of awareness among veterinarians, technicians, and medical physicians on prudent use of antimicrobial drugs in Nepal could reduce the burden of AMR)]\n",
            "2725: [(, , , The conventional methods and technologies addressing AMR diagnostics and AST employed in clinical microbiology are tedious, with high turnaround times (TAT), and are usually expensive.)]\n",
            "2726: [(, , , As a result, empirical antimicrobial therapies are prescribed leading to AMR spread, which in turn causes higher mortality rates and increased healthcare costs.)]\n",
            "2727: [(, , , This review describes the developments in current cutting-edge methods and technologies, organized by key enabling research domains, towards fighting the looming AMR menace by employing recent advances in AMR diagnostic tools.)]\n",
            "2728: [(, , , First, we summarize the conventional methods addressing AMR detection, surveillance, and AST.)]\n",
            "2729: [(, , , Nepal is one of the major contributors to the growing burden of AMR due to widespread irrational use of antibiotics along with poor health care systems poor infection control and prevention measures.)]\n",
            "2730: [(, , , In fact, Antarctica is often perceived as the last pristine continent on Earth.)]\n",
            "2731: [(, , , Following, we provide examples of commercially available diagnostic platforms for AST.)]\n",
            "2732: [(, , , Such remote regions, which are assumed to have very low levels of AMR due to limited human activity, represent potential model environments to understand the mechanisms and interactions underpinning the early stages of evolution, de novo development, acquisition and transmission of AMR.)]\n",
            "2733: [(, , , Several pathways, including hospital effluent, agricultural waste, and wastewater treatment facilities, have been identified as potential routes for the spread of resistant bacteria and their resistance genes in soil and surrounding ecosystems.)]\n",
            "2734: [(137, 207, Techniques, Thereafter, we examine more recent non-conventional methods and the advancements in each field, including whole genome sequencing (WGS), matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) spectrometry, Fourier transform infrared (FTIR) spectroscopy, and microfluidics technology.)]\n",
            "2735: [(208, 221, Techniques, Thereafter, we examine more recent non-conventional methods and the advancements in each field, including whole genome sequencing (WGS), matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) spectrometry, Fourier transform infrared (FTIR) spectroscopy, and microfluidics technology.)]\n",
            "2736: [(222, 268, Techniques, Thereafter, we examine more recent non-conventional methods and the advancements in each field, including whole genome sequencing (WGS), matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) spectrometry, Fourier transform infrared (FTIR) spectroscopy, and microfluidics technology.)]\n",
            "2737: [(274, 298, Techniques, Thereafter, we examine more recent non-conventional methods and the advancements in each field, including whole genome sequencing (WGS), matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) spectrometry, Fourier transform infrared (FTIR) spectroscopy, and microfluidics technology.)]\n",
            "2738: [(106, 135, Techniques, Thereafter, we examine more recent non-conventional methods and the advancements in each field, including whole genome sequencing (WGS), matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) spectrometry, Fourier transform infrared (FTIR) spectroscopy, and microfluidics technology.)]\n",
            "2739: [(, , , Available cross sectional reports warn that bacterial pathogens are becoming highly resistant to most first- and some second-line antibiotics)]\n",
            "2740: [(, , , Finally, perspectives on the implementation of emerging concepts towards developing paradigm-changing technologies and methodologies for AMR diagnostics are discussed.)]\n",
            "2741: [(, , , The irrational and injudicious use of high doses of antibiotics for therapy and sub-optimal doses as growth promoters are leading causes of AMR in Nepal.)]\n",
            "2742: [(, , , In addition, there is a need to develop a national laboratory strategic plan to provide guidance and governance to national laboratories.)]\n",
            "2743: [(, , , Antimicrobial resistance (AMR) is a critical issue in health care in terms of mortality, quality of services, and financial damage.)]\n",
            "2744: [(, , , Antarctica, with its defined zones of human colonisation (centred around scientific research stations) and large populations of migratory birds and animals, also has great potential with regard to mapping and understanding the spread of early-stage zoonotic interactions.)]\n",
            "2745: [(, , , In the battle against AMR, it is crucial to recognize the impacts of all four domains, namely, mankind, livestock, agriculture, and the ecosystem.)]\n",
            "2746: [(, , , However, to date, studies of AMR in Antarctica are limited.)]\n",
            "2747: [(, , , Many sociocultural and financial practices that are widespread in the world have made resistance management extremely complicated.)]\n",
            "2748: [(, , , Furthermore, areas where there is little, to no, human activity still appear to be free from anthropogenically introduced AMR.)]\n",
            "2749: [(, , , The overuse of uncontrolled antibiotics and improper treatment and recycled wastewater are among the contributors to AMR.)]\n",
            "2750: [(, , , However, a comprehensive assessment of AMR levels in Antarctica is virtually impossible on current data due to the wide variation in reporting standards and methodologies used and poor geographical coverage.)]\n",
            "2751: [(, , , Health-care organizations have begun to address AMR, although they are currently in the early stages.)]\n",
            "2752: [(, , , Thus, future studies should engage directly with policymakers to promote the implementation of continent-wide AMR reporting standards.)]\n",
            "2753: [(, , , Rapid detection of AMR and level quantification are the prerequisites to control the spread of AMR. Although time-consuming, traditional phenotype-based methods are still the primary methods used in clinical laboratories and are regarded as the gold standard for AMR identification.)]\n",
            "2754: [(, , , Antimicrobial resistance (AMR), especially bacterial AMR, poses a global threat to public health and has become a huge obstacle to the effective control of related infectious diseases.)]\n",
            "2755: [(, , , Many studies have demonstrated the disastrous financial consequences of AMR including extremely high healthcare costs due to an increase in hospital admissions and drug usage.)]\n",
            "2756: [(, , , The literature review, which included articles published after the year 2012, was performed using Scopus, PubMed and Google Scholar with the utilization of keyword searches.)]\n",
            "2757: [(, , , Following the golden age of antimicrobials discovery between the 1940s and 1960s, antimicrobial abuse resulted in the rapid emergence of AMR.)]\n",
            "2758: [(, , , Nowadays, the problem of AMR has become increasingly serious, and some bacteria have reached the brink of no suitable antimicrobials available.)]\n",
            "2759: [(, , , Results indicated that the multifactorial threat of antimicrobial resistance has resulted in different complex issues affecting countries across the globe.)]\n",
            "2760: [(, , , These impacts found in the sources are categorized into three different levels: patient, healthcare, and economic.)]\n",
            "2761: [(, , , Although gaps in knowledge about AMR and areas for improvement are obvious, there is not any clearly understood progress to put an end to the persistent trends of antimicrobial resistance.)]\n",
            "2762: [(, , , We aimed to identify knowledge gaps to help develop research strategies, especially towards mitigating AMR in the environment, the often-neglected part of the One Health triad.)]\n",
            "2763: [(, , , This chapter will describe the advances in the above technologies applied in AMR testing.)]\n",
            "2764: [(, , , The effectiveness of antibiotics continues to erode because of the relentless spread of antimicrobial resistance (AMR).)]\n",
            "2765: [(, , , One of the main drivers of AMR is overprescription of antibiotics, both in human and in veterinary medicine.)]\n",
            "2766: [(, , , Public and private foundations, professional organizations, and international health agencies recognize the threat posed by AMR and have issued calls for action.)]\n",
            "2767: [(102, 126, Techniques, In addition, the spontaneous evolution, mutation of bacteria, and passing the resistant genes through horizontal gene transfer are significant contributors to antimicrobial resistance.)]\n",
            "2768: [(, , , This complex issue requires a multisectoral One Health approach to address the interconnectedness of humans, animals and the natural environment.)]\n",
            "2769: [(, , , The prevalence of AMR in these reservoirs varies widely among countries and thus often requires a country-specific approach.)]\n",
            "2770: [(, , , The One Health concept is a response from a broad group of stakeholders to counter the global health threat posed by AMR)]\n",
            "2771: [(, , , In New Zealand (NZ), AMR and antimicrobial usage in humans are relatively well-monitored and -understood, with high human use of antimicrobials and the frequency of resistant pathogens increasing in hospitals and the community.)]\n",
            "2772: [(, , , In this article, we discuss current trends in AMR and suggest strategies to mitigate its ongoing dissemination.)]\n",
            "2773: [(, , , In contrast, on average, NZ is a low user of antimicrobials in animal husbandry systems with low rates of AMR in food-producing animals.)]\n",
            "2774: [(, , , Antimicrobial resistance (AMR) has been recognized as an emerging and growing problem worldwide.)]\n",
            "2775: [(, , , AMR in New Zealand's environment is little understood, and the role of the natural environment in AMR transmission is unclear.)]\n",
            "2776: [(, , , Knowledge concerning AMR bacteria circulating in wildlife is currently limited, although it could provide important insights into AMR emergence and persistence.)]\n",
            "2777: [(, , , Here, we aimed to provide a summary of the current knowledge on AMR in NZ, addressing all three components of the One Health triad with a particular focus on environmental AMR.)]\n",
            "2778: [(, , , Across Europe, wild boar (Sus scrofa) populations have dramatically increased their distribution and number over the last decades.)]\n",
            "2779: [(, , , In the context of AMR dynamics, wild boar is a perfect model species to unveil the emergence, spread and persistence of AMR at the human-livestock-wildlife interface. Here, we summarize the current knowledge on the importance of wild boar as a reservoir of antimicrobial resistant bacteria, and its possible use as sentinel species for surveillance.)]\n",
            "2780: [(, , , Antimicrobial resistance (AMR) has developed as one of the major urgent threats to public health causing serious issues to successful prevention and treatment of persistent diseases.)]\n",
            "2781: [(, , , Analyses of available data showed a rising interest on this topic in the last years, highlighting the growing concern on wild boar potential role as AMR facilitator and it is foreseen that the importance of antimicrobial resistance research in wild boar will continue to increase in years to come.)]\n",
            "2782: [(, , , In spite of different actions taken in recent decades to tackle this issue, the trends of global AMR demonstrate no signs of slowing down.)]\n",
            "2783: [(, , , Available studieshave been focused on specific bacterial species, particularlyE. coli, Salmonellaspp. andEnterococcusspp., bioindicators of AMR, and have been mainly conducted in three countries: Spain, Portugal and Germany.)]\n",
            "2784: [(, , , Misusing and overusing different antibacterial agents in the health care setting as well as in the agricultural industry are considered the major reasons behind the emergence of antimicrobial resistance.)]\n",
            "2785: [(, , , Overall, accumulated data showed thatwild boar are carriers of antimicrobial resistant bacteria, withvariation in the prevalence of bacterial species and thepercentage of resistance to different antibiotics.)]\n",
            "2786: [(, , , This review summarizes current knowledge on the prevalence of AMR at the interface of animal and agricultural production and discusses the potential implications for human health.)]\n",
            "2787: [(, , , Thelack of harmonized sampling and testing protocols makes it difficult to compare AMR in wild boar.)]\n",
            "2788: [(, , , Soil resistomes are augmented by the application of manure from treated livestock.)]\n",
            "2789: [(, , , The need for the establishment of standardised protocols keen to provide quantitative comparable data is highlighted.)]\n",
            "2790: [(, , , Subsequent transfer of AMR into plant microbiomes may likely play a critical role in human exposure to antibiotic resistance in the environment.)]\n",
            "2791: [(, , , The contribution to AMR from environmental factors is usually only mentioned in large-scale animal production.)]\n",
            "2792: [(, , , We finally suggest the long-term monitoring of wild boar as a sentinel species for AMR surveillance in order to inform public policies on this topic.)]\n",
            "2793: [(, , , The recent advancements in rapid and affordable DNA sequencing technologies have revolutionized diagnostic microbiology and microbial surveillance.)]\n",
            "2794: [(, , , Antimicrobial resistance (AMR) presents serious threats to human and animal health.)]\n",
            "2795: [(, , , The availability of bioinformatics tools and online accessible databases has been a prerequisite for this.)]\n",
            "2796: [(, , , Although AMR of pathogens is often evaluated independently between humans and animals, comparative analysis of AMR between humans and animals is necessary for zoonotic pathogens.)]\n",
            "2797: [(, , , We conducted a scientific literature review and here we present a description of examples of available tools and databases for antimicrobial resistance (AMR) detection and provide future perspectives and recommendations.)]\n",
            "2798: [(, , , High prevalences of AMR exist among Shigella, Campylobacter, and Yersinia, including resistance to antimicrobials important to public health, such as macrolides.)]\n",
            "2799: [(, , , The objective of this review is to provide a complete report of the prevalences of AMR among zoonotic bacteria that present the greatest threats to NHP, occupational, and public health.)]\n",
            "2800: [(112, 137, Others, Based on the knowledge that is currently available we advocate that more attention should be paid to the role of environmental resistomes in the AMR crisis.)]\n",
            "2801: [(42, 60, Microbes, Major surveillance systems monitor AMR of zoonotic pathogens in humans and food animals, but comprehensive AMR data in veterinary medicine is not diligently monitored for most animal species with which humans commonly contact, including NHP.)]\n",
            "2802: [(, , , As a consequence, each tool has strengths and limitations in sensitivity and specificity of detection of AMR determinants and in application, which for some of the tools have been highlighted in benchmarking exercises and scientific articles.)]\n",
            "2803: [(, , , Despite improvements in regulations, standards, policies, practices, and zoonotic awareness, occupational exposures to and illnesses due to zoonotic pathogens continue to be reported and, given the documented prevalences of AMR, constitute an occupational and public health risk.)]\n",
            "2804: [(, , , The identified tools are either available at public genome data centers, from GitHub or can be run locally.)]\n",
            "2805: [(, , , However, published literature is sparse, thus indicating the need for veterinarians to proactively monitor AMR in dangerous zoonotic bacteria, to enable veterinarians to make more informed decisions to maximize antimicrobial therapy and minimize occupational risk.)]\n",
            "2806: [(, , , NCBI and European Nucleotide Archive (ENA) provide possibilities for online submission of both sequencing and accompanying phenotypic antimicrobial susceptibility data, allowing for other researchers to further analyze data, and develop and test new tools.)]\n",
            "2807: [(, , , Antimicrobial resistance (AMR) circulates through humans, animals, and the environments, requiring a One Health approach.)]\n",
            "2808: [(, , , The advancement in whole genome sequencing and the application of online tools for real-time detection of AMR determinants are essential to identify control and prevention strategies to combat the increasing threat of AMR.)]\n",
            "2809: [(, , , Recently, urban sewage has increasingly been suggested as a hotspot for AMR even in high-income countries (HICs), where the water sanitation and hygiene infrastructure are well-developed.)]\n",
            "2810: [(, , , Accessible tools and DNA sequence data are expanding, which will allow establishing global pathogen surveillance and AMR tracking based on genomics. There is however, a need for standardization of pipelines and databases as well as phenotypic predictions based on the data.)]\n",
            "2811: [(, , , With increasing reports on antimicrobial resistance (AMR) in humans, animals and the environment, we are at risk of returning to a pre-antibiotic era.)]\n",
            "2812: [(, , , Therefore, AMR is recognized as one of the major global health threats of this century.)]\n",
            "2813: [(, , , Antibiotics are used extensively in farming systems to treat and prevent infections in food animals or to increase their growth. Besides the risk of a transfer of AMR between the human and the animal sector, there is another yet largely overlooked sector in the One Health triad.)]\n",
            "2814: [(, , , Strikingly, AMR surveillance in wild boar is uneven and still poorly allocated as many wild boar high-density countries do not yet have publications on the topic.)]\n",
            "2815: [(, , , Human-dominated ecosystems such as agricultural soils are a major sink for antibiotics and AMR originating from livestock farming.)]\n",
            "2816: [(, , , To understand the current status of AMR in wastewater in a HIC, we reviewed the epidemiological studies on AMR in the sewage environment in Japan from the published literature.)]\n",
            "2817: [(, , , Our review shows the importance of both monitoring for AMR and antimicrobials in human wastewater and efforts to reduce their contamination load in wastewater.)]\n",
            "2818: [(, , , However, in the race towards developing alternative approaches to combat AMR, it appears that the scientific community is falling behind when pitched against the evolutionary capacity of multi-drug resistant (MDR) bacteria.)]\n",
            "2819: [(, , , During the past 70 years, there has been a continued incremental worsening of antimicrobial resistance (AMR) by a combination of abusing antimicrobials in humans and animals as well as insufficient research and investment for new antimicrobial agents.)]\n",
            "2820: [(, , , The current trend of worsening AMR is likely to result in increased mortality and morbidity, longer stays in hospitals and accelerated health care costs. It is estimated that the global mortality attributed by AMR could reach 10 million per year by 2050, which is a massive increase from the current estimated mortality of 700, 000 per year.)]\n",
            "2821: [(, , , This review will briefly detail the causes and mechanisms of AMR as background, and then provide insights into a novel, future emerging or evolving strategies that are currently being evaluated and which may be developed in the future to tackle the progression of AMR.)]\n",
            "2822: [(, , , The year 2014 was the turning point of more than a half century of AMR history, transforming it from technical issue to a political agenda.)]\n",
            "2823: [(, , , China has great potential for engaging in global actions on antimicrobial resistance (AMR) control.)]\n",
            "2824: [(, , , This study aims to summarize the process of global AMR governance and provide relevant policy recommendations on how China could take more initiative in the global AMR governance.)]\n",
            "2825: [(, , , Major progress includes adoption of the Global Action Plan on AMR at the WHO World Health Assembly in May 2015, followed by completion of a National Action Plan in most parts of the world, enactment of the Global Antimicrobial Surveillance System (GLASS) in October 2015, launch of the World Antibiotic Awareness Week in November 2015, and G20 Leaderscommitment to create the AMR Global Collaboration Hub in July 2017.)]\n",
            "2826: [(, , , We searched for academic articles and official document published or issued before December 2020 in e-journal databases, official websites of major organizations, and the relevant national ministries.)]\n",
            "2827: [(, , , Moreover, a comprehensive program against AMR has been implemented in some countries, such as UK, USA, and Germany.)]\n",
            "2828: [(, , , The strategic approach through coordination led by WHO with relevant international agencies and other entities was one of the key enabling factors for sustained political commitment on AMR.)]\n",
            "2829: [(, , , This review revealed that global action on AMR control has experienced three stages: (1) The beginning stage (1980s and 1990s) when actions were mainly sponsored by high-income countries and AMR surveillance was focused on hospitals; (2) The rapid development stage (2000-2010) when global AMR governance began to concentrate on joint actions in multi-sectors, and developing countries were gradually involved in global actions; (3) The comprehensive stage (2011 to present) when global actions on AMR have covered various fields in different countries.)]\n",
            "2830: [(, , , However, the effect on AMR caused by heavy metals has not been clearly revealed. In this study, we used a microcosm experiment and metagenomics to examine whether common levels of Cu and Zn in pig manure influence AMR transmission in manured soil.)]\n",
            "2831: [(, , , We found that the abundance of 204 ARGs significantly increased after manure application, even though the manure did not contain antibiotic residuals.)]\n",
            "2832: [(, , , However, the combined addition of low Cu and Zn (500 and 1000 mg/kg, respectively) only caused 14 ARGs to significantly increase, and high Cu and Zn (1000 and 3000 mg/kg, respectively) caused 27 ARGs to significantly increase.)]\n",
            "2833: [(, , , The disparity of these numbers suggested that factors within the manure were the primary driving reasons for AMR transmission, rather than metal amendments.)]\n",
            "2834: [(, , , This study offers deeper insights into AMR transmission in relation to the effects of manure application and heavy metals at commonly reported levels.)]\n",
            "2835: [(, , , To fight antimicrobial resistance (AMR), we must recognize and target all its manifestations.)]\n",
            "2836: [(, , , Our review showed that a wide variety of clinically important antibiotic-resistant bacteria (ARB), antibiotic resistance genes (ARGs), and antimicrobial residues are present in human wastewater in Japan.)]\n",
            "2837: [(, , , Our findings recommend that more comprehensive measures in controlling AMR in the pig industry are needed apart from restricting heavy metal additions.)]\n",
            "2838: [(, , , Their concentrations are lower than in low- and middle-income countries (LMICs) and are further reduced by sewage treatment plants (STPs) before discharge.)]\n",
            "2839: [(, , , Antimicrobials are useful compounds intended to eradicate or stop the growth of harmful microorganisms.)]\n",
            "2840: [(, , , Nevertheless, the remaining ARB and ARGs could be an important source of AMR contamination in river water.)]\n",
            "2841: [(, , , The sustained increase in the rates of antimicrobial resistance (AMR) worldwide is worrying and poses a major public health threat.)]\n",
            "2842: [(, , , Furthermore, hospital effluence may be an important reservoir of clinically important ARB. The high concentration of antimicrobial agents commonly prescribed in Japan may contribute to the selection and dissemination of AMR within wastewater.)]\n",
            "2843: [(, , , The development of new antimicrobial agents is one of the critical approaches to overcome AMR.)]\n",
            "2844: [(, , , Developing methods that can detect and predict AMR phenotypes can help to mitigate the spread of AMR by informing clinical decision making and appropriate mitigation strategies.)]\n",
            "2845: [(, , , Antimicrobial resistance (AMR) in rural Latin America is not fully understood.)]\n",
            "2846: [(, , , Many bioinformatic methods have been developed for predicting AMR phenotypes from whole-genome sequences and AMR genes, but recent studies have indicated that predictions can be made from incomplete genome sequence data.)]\n",
            "2847: [(, , , The transmission pathways are partially known since research predominantly focuses on the urban hospital setting.)]\n",
            "2848: [(, , , The novel and most innovative technologies herein reviewed can support this goal and add powerful tools in the arsenal of weapons against AMR.)]\n",
            "2849: [(, , , We started by building models from alignments that were based on a reference chromosome for each species.)]\n",
            "2850: [(, , , We then subsampled each chromosomal alignment and built models for the resulting subalignments, finding that very small regions, representing approximately 0.1 to 0.2% of the chromosome, are predictive.)]\n",
            "2851: [(, , , We used these models to identify regions of the chromosome with high and low predictive signals. Finally, subalignments that retain high accuracy across larger phylogenetic distances were examined in greater detail, revealing genes and intergenic regions with potential links to AMR, virulence, transport, and survival under stress conditions.)]\n",
            "2852: [(82, 86, Gene/Proteins, While their usefulness has been extensively proved against viral pathogens, human mAbs have yet to find their space in treating and preventing infections from AMR-bacteria and fully conquer the field of infectious diseases.)]\n",
            "2853: [(163, 184, Microbes, In order to more systematically understand this, we built random forest-based machine learning classifiers for predicting susceptible and resistant phenotypes for Klebsiella pneumoniae (1,640 strains), Mycobacterium tuberculosis (2,497 strains), and Salmonella enterica (1,981 strains).)]\n",
            "2854: [(202, 228, Microbes, In order to more systematically understand this, we built random forest-based machine learning classifiers for predicting susceptible and resistant phenotypes for Klebsiella pneumoniae (1,640 strains), Mycobacterium tuberculosis (2,497 strains), and Salmonella enterica (1,981 strains).)]\n",
            "2855: [(250, 269, Microbes, In order to more systematically understand this, we built random forest-based machine learning classifiers for predicting susceptible and resistant phenotypes for Klebsiella pneumoniae (1,640 strains), Mycobacterium tuberculosis (2,497 strains), and Salmonella enterica (1,981 strains).)]\n",
            "2856: [(3, 16, Antibiotic-Resistant Bacteria, In K. pneumoniae, M. tuberculosis, and S. enterica, the subalignments are able to predict multiple AMR phenotypes with at least 70% accuracy, even though most do not encode an AMR-related function.)]\n",
            "2857: [(18, 33, Antibiotic-Resistant Bacteria, In K. pneumoniae, M. tuberculosis, and S. enterica, the subalignments are able to predict multiple AMR phenotypes with at least 70% accuracy, even though most do not encode an AMR-related function.)]\n",
            "2858: [(39, 50, Antibiotic-Resistant Bacteria, In K. pneumoniae, M. tuberculosis, and S. enterica, the subalignments are able to predict multiple AMR phenotypes with at least 70% accuracy, even though most do not encode an AMR-related function.)]\n",
            "2859: [(, , , The central government should take a far-fetched view, act decisively and positively towards the global efforts of addressing AMR to play a more active and greater role on the international stage.)]\n",
            "2860: [(, , , Machine learning models are capable of predicting antimicrobial resistance phenotypes from bacterial genome sequence data by identifying resistance genes, mutations, and other correlated features.)]\n",
            "2861: [(, , , Antimicrobial resistance (AMR) is a quickly advancing threat for human health worldwide and almost 5 million deaths are already attributable to this phenomenon every year.)]\n",
            "2862: [(, , , They are also capable of implicating regions of the genome that have not been previously characterized as being involved in resistance.)]\n",
            "2863: [(62, 83, Antibiotic-Resistant Bacteria, In this study, we generated global chromosomal alignments for Klebsiella pneumoniae, Mycobacterium tuberculosis, and Salmonella enterica and systematically searched them for small conserved regions of the genome that enable the prediction of antimicrobial resistance phenotypes.)]\n",
            "2864: [(85, 111, Antibiotic-Resistant Bacteria, In this study, we generated global chromosomal alignments for Klebsiella pneumoniae, Mycobacterium tuberculosis, and Salmonella enterica and systematically searched them for small conserved regions of the genome that enable the prediction of antimicrobial resistance phenotypes.)]\n",
            "2865: [(117, 136, Antibiotic-Resistant Bacteria, In this study, we generated global chromosomal alignments for Klebsiella pneumoniae, Mycobacterium tuberculosis, and Salmonella enterica and systematically searched them for small conserved regions of the genome that enable the prediction of antimicrobial resistance phenotypes.)]\n",
            "2866: [(, , , While their usefulness has been extensively proved against viral pathogens, human mAbs have yet to find their space in treating and preventing infections from AMR-bacteria and fully conquer the field of infectious diseases.)]\n",
            "2867: [(, , , In addition to known antimicrobial resistance genes, this analysis identified genes involved in virulence and transport functions, as well as many genes with no previous implication in antimicrobial resistance.)]\n",
            "2868: [(, , , Antimicrobial resistance (AMR) represents a global threat to both human and animal health and has received increasing attention over the years from different stakeholders.)]\n",
            "2869: [(, , , Antimicrobial resistance (AMR) is a quickly advancing threat for human health worldwide and almost 5 million deaths are already attributable to this phenomenon every year.)]\n",
            "2870: [(, , , Certain AMR bacteria circulate between humans, animals, and the environment, while AMR genes can be found in all ecosystems.)]\n",
            "2871: [(108, 112, Antibiotic-Resistant Genes, Since antibiotics are failing to treat AMR-bacteria, new tools are needed, and human monoclonal antibodies (mAbs) can fill this role. In almost 50 years since the introduction of the first technology that led to mAb discovery, enormous leaps forward have been made to identify and develop extremely potent human mAbs.)]\n",
            "2872: [(312, 316, Antibiotic-Resistant Genes, Since antibiotics are failing to treat AMR-bacteria, new tools are needed, and human monoclonal antibodies (mAbs) can fill this role. In almost 50 years since the introduction of the first technology that led to mAb discovery, enormous leaps forward have been made to identify and develop extremely potent human mAbs.)]\n",
            "2873: [(, , , The aim of the present review was to provide an overview of antimicrobial use in food-producing animals and to document the current status of the role of farm animals in the spread of AMR to humans.)]\n",
            "2874: [(, , , The available body of scientific evidence supported the notion that restricted use of antimicrobials in farm animals was effective in reducing AMR in livestock and, in some cases, in humans.)]\n",
            "2875: [(, , , The novel and most innovative technologies herein reviewed can support this goal and add powerful tools in the arsenal of weapons against AMR.)]\n",
            "2876: [(, , , However, most recent studies have reported that livestock have little contribution to the acquisition of AMR bacteria and/or AMR genes by humans. Overall, strategies applied on farms that target the reduction of all antimicrobials are recommended, as these are apparently associated with notable reduction in AMR (avoiding co-resistance between antimicrobials).)]\n",
            "2877: [(, , , Antimicrobial resistance (AMR) is an important global health threat that impacts millions of people worldwide each year.)]\n",
            "2878: [(, , , The interconnection between human and animal health as well as the environment requires the acceleration of the implementation of the 'One Health' approach to effectively fight AMR while preserving the effectiveness of antimicrobials.)]\n",
            "2879: [(, , , A total of 75 publications were included, of which 19 studies reported AMU data for dairy calves and 68 described AMR profiles of the four most prevalent bacteria that are associated with calf diarrhea and calf pneumonia.)]\n",
            "2880: [(, , , Search terms were \"ANTIMICROBIAL\" OR \"ANTIBIOTIC\" AND \"RESISTANT\" OR \"RESISTANCE\" and \"WILDLIFE\" \"MAMMAL\" \"BIRD\" \"REPTILE\" \"FERAL\" \"FREE RANGE\". A total of 219 articles were obtained, published between 1979 and 2019.)]\n",
            "2881: [(, , , Large variation in AMU was found among herds across different regions. There seems to be a positive association between exposure to antimicrobials and occurrence of resistance.)]\n",
            "2882: [(, , , A rising interest in the last decades towards this topic becomes evident.)]\n",
            "2883: [(, , , Most AMU was accounted for by treatment of diseases, while a small proportion of AMU was prophylactic.)]\n",
            "2884: [(, , , During this period, the scientific literature was distributed among several scientific areas, however it became more multidisciplinary in the last years, focusing on the \"One Health\" paradigm.)]\n",
            "2885: [(, , , To understand the state of the literature on AMR in rural LA, we carried out a scoping review using the One Health (OH) perspective.)]\n",
            "2886: [(, , , AMU was more common in treating calf diarrhea than in treating pneumonia, and the resistance rates in bacteria associated with diarrhea were higher than those in pathogens related to pneumonia.)]\n",
            "2887: [(, , , OH recognises the concomitant contributions and interconnectedness of humans, animal, and the environment, thus, we used the OH perspective to select those articles adopting a holistic view of the problem.)]\n",
            "2888: [(, , , Organic farms used significantly fewer antimicrobials to treat calf disease; however, the antimicrobial resistance rates of bacteria associated with calf diarrhea and pneumonia on both types of farms were comparable.)]\n",
            "2889: [(, , , We searched original articles in English, Spanish, and Portuguese in four peer-reviewed databases and included 21 publications in the analysis.)]\n",
            "2890: [(, , , Feeding waste or pasteurized milk was associated with a higher risk of AMR in pathogens.)]\n",
            "2891: [(, , , We charted data on bibliometrics, design, data collection sources, and instruments.)]\n",
            "2892: [(, , , Altogether, this review summarizes AMU and AMR data for dairy calves and suggests areas for future research, providing evidence for the design of antimicrobial use stewardship programs in dairy calf farming.)]\n",
            "2893: [(, , , High levels of antimicrobial resistance (AMR) in Indonesia are caused by the use of inappropriate antimicrobials (AM) in healthcare services and the livestock and fisheries sector.)]\n",
            "2894: [(, , , Antimicrobial use (AMU) is the major driver of antimicrobial resistance (AMR) among bacteria in dairy herds.)]\n",
            "2895: [(, , , Intensive and non-intensive animal production systems and agricultural practices were the most frequently found human contributions to AMR.)]\n",
            "2896: [(, , , The available data and information about overused antibiotics and the AMR threat in Indonesia are limited.)]\n",
            "2897: [(, , , The aim of the study is to describe the AMR situation in Indonesia based on perceptions of government officials, health professionals, and the community to determine actions needed to develop AMR-related strategy and policy.)]\n",
            "2898: [(, , , We found that only 1 article out of 21 considered the OH approach as a framework for their sampling scheme, whereas 5 out 21 discussed all the three OH components.)]\n",
            "2899: [(, , , The study was done in eight provinces in Indonesia and included reviewing AMR-related policy, collecting antibiotic use reports in primary health care from health offices and hospitals, and conducting in-depth interviews and focus group discussions with informants from health and nonhealth sectors.)]\n",
            "2900: [(, , , There were hardly any descriptions of humans or human waste as reservoirs for AMR in rural locations, and rural health centres or hospitals and wildlife were not represented.)]\n",
            "2901: [(, , , The results of the study show that AM misuse happens not only in healthcare facilities but also in communities.)]\n",
            "2902: [(, , , No studies identified mining as an anthropogenic activity driving AMR.)]\n",
            "2903: [(, , , Medical officers are unfamiliar with AMR-related policy, as are officers in the livestock and fisheries sectors.)]\n",
            "2904: [(, , , More OH-oriented studies, with emphasis on molecular approaches-for identification and comparison of AMR genes-are sorely needed to understand better the existence of a network of interconnected transmission routes in rural Latin America and provide efficient strategies to prevent further AMR emergence.)]\n",
            "2905: [(, , , There is limited coordination between sectors regarding the AMR situation in Indonesia.)]\n",
            "2906: [(, , , The government has to take stronger measures to oversee better implementation of AMR policies)]\n",
            "2907: [(, , , There have been numerous studies on AMU and AMR in dairy cows; however, studies on AMU and AMR in dairy calves are limited.)]\n",
            "2908: [(, , , Antimicrobial resistance (AMR) is a complex and global problem.)]\n",
            "2909: [(, , , A comprehensive overview of the current state of knowledge of AMU and AMR among pathogens in dairy calves is important for the development of scientifically supported and applicable measures to curb antimicrobial use and the increasing risk of AMR.)]\n",
            "2910: [(, , , Despite the growing literature on AMR in the medical and veterinary settings, there is still a lack of knowledge on the wildlife compartment.)]\n",
            "2911: [(, , , Therefore, we performed a systematic review of research on AMU and AMR in dairy calves.)]\n",
            "2912: [(, , , The main aim of this study was to report the global trends in AMR research in wildlife, through a bibliometric study of articles found in the Web of Science database.)]\n",
            "2913: [(, , , Future research is needed in order to understand the full extent of the epidemiology and ecology of AMR in poultry within low-resource settings.)]\n",
            "2914: [(, , , Antimicrobial resistance (AMR) is a serious public health issue.)]\n",
            "2915: [(, , , Antimicrobial resistance (AMR) is a significant and growing public health threat.)]\n",
            "2916: [(, , , Due to resistance to current antibiotics and a low rate of development of new classes of antimicrobials, AMR is a leading cause of death worldwide.)]\n",
            "2917: [(, , , There was a geographical bias in the research outputs: most published documents were from the United States, followed by Spain, Portugal and the United Kingdom.)]\n",
            "2918: [(, , , The most productive institutions in terms of publication number were located in Portugal and Spain.)]\n",
            "2919: [(, , , A key problem in AMR detection, however, is the need for computational approaches detecting potential novel AMR genes or variants, which are not included in the reference databases.)]\n",
            "2920: [(, , , An important level of international collaboration was identified.)]\n",
            "2921: [(, , , We show how known AMR protein variants tend to correspond to exposed residues, while on the contrary their susceptible counterparts tend to be buried.)]\n",
            "2922: [(36, 43, Techniques, Based on these findings, we develop RSA-AMR, a novel relative solvent accessibility-based AMR scoring system.)]\n",
            "2923: [(141, 156, Techniques, The two main limitations of RSA-AMR score is that it is designed on single point changes, and a limited number of variants was available for model learning.)]\n",
            "2924: [(0, 22, Microbes, Listeria monocytogenes is a deadly foodborne pathogen that causes listeriosis for the immunocompromised, the elderly, and pregnant women.)]\n",
            "2925: [(66, 77, Others, Listeria monocytogenes is a deadly foodborne pathogen that causes listeriosis for the immunocompromised, the elderly, and pregnant women.)]\n",
            "2926: [(65, 102, Others, Toward this direction, here we study the relation between AMR and relative solvent accessibility (RSA) of protein variants from an in silico perspective.)]\n",
            "2927: [(72, 79, Antibiotic-Resistant Bacteria, An analysis of the main keywords showed an overall dominance of \"AMR\", \"E. coli\", \"genes\", \"prevalence\", \"bacteria\", \"Salmonella spp.\" and \"wild birds\".)]\n",
            "2928: [(117, 134, Microbes, An analysis of the main keywords showed an overall dominance of \"AMR\", \"E. coli\", \"genes\", \"prevalence\", \"bacteria\", \"Salmonella spp.\" and \"wild birds\".)]\n",
            "2929: [(, , , Given the growth tendency over the last years, it is envisaged that scientific production will expand in the future.)]\n",
            "2930: [(, , , This scoring system can be applied to any protein variant to estimate its propensity of altering the relative solvent accessibility, and potentially conferring (or hindering) AMR. We show how RSA-AMR score can be integrated with existing AMR detection algorithms to expand their range of applicability into detecting potential novel AMR variants, and provide a ten-fold increase in Specificity.)]\n",
            "2931: [(, , , In addition to offering a broad view of the existing research trends, this study identifies research gaps both in terms of geographical incidence and in relation to unexplored subtopics.)]\n",
            "2932: [(, , , Minimizing emissions and determining antimicrobial emission limits are priorities in the containment of environmental AMR development.)]\n",
            "2933: [(, , , Unearthing scientific areas that should be explored in the future is key to designing new strategic research agendas in AMR research in wildlife and to inform funding programs.)]\n",
            "2934: [(, , , The misuse and overuse of antibiotics have resulted in an alarmingly high prevalence of antimicrobial resistance (AMR) in human and animal bacteria.)]\n",
            "2935: [(, , , The emergence, spread, and persistence of antimicrobial resistance (AMR) remain a pressing global health issue.)]\n",
            "2936: [(, , , European monitoring programmes show that AMR occurrence in food animals is lower in Sweden than in most other EU Member States and that the use of antibiotics for animals is among the lowest in Europe.)]\n",
            "2937: [(, , , Animal husbandry, in particular poultry, makes up a substantial portion of the global antimicrobial use.)]\n",
            "2938: [(, , , In this retrospective country case study, we analysed published documents to identify factors contributing to this favourable situation.)]\n",
            "2939: [(, , , Despite the growing body of research evaluating the AMR within industrial farming systems, there is a gap in understanding the emergence of bacterial resistance originating from poultry within resource-limited environments.)]\n",
            "2940: [(, , , A fundamental factor identified was early insight into and sustained awareness of the risks of AMR and the need for the prudent use of antibiotics.)]\n",
            "2941: [(, , , As countries continue to transition from low- to middle income countries (LMICs), there will be an increased demand for quality sources of animal protein.)]\n",
            "2942: [(, , , Early and continuous access to data on antibiotic use and AMR made it possible to focus activities on areas of concern.)]\n",
            "2943: [(, , , Further promotion of intensive poultry farming could address issues of food security, but it may also increase risks of AMR exposure to poultry, other domestic animals, wildlife, and human populations.)]\n",
            "2944: [(, , , Another factor identified was the long-term control and eradication of infectious animal diseases, including coordinated activities against endemic diseases, which reduced the need to use antibiotics.)]\n",
            "2945: [(, , , Given that intensively raised poultry can function as animal reservoirs for AMR, surveillance is needed to evaluate the impacts on humans, other animals, and the environment.)]\n",
            "2946: [(, , , Structures and strategies for that purpose established at the national level have since proven useful in counteracting AMR as an integral part of disease prevention and control, guided by a \"prevention is better than cure\" approach.)]\n",
            "2947: [(, , , Here, we provide a comprehensive review of poultry production within low-resource settings in order to inform future small-scale poultry farming development.)]\n",
            "2948: [(, , , A third factor identified was consensus among stakeholders on the need to address AMR and their cooperation in the design and implementation of measures.)]\n",
            "2949: [(, , , An association between AMR and CDI outbreaks is well documented, however, data is limited to a few ‘epidemic’ strains in specific geographical regions.)]\n",
            "2950: [(, , , Principal component analysis was firstly used to project the multi-dimensional data into two dimensions.)]\n",
            "2951: [(, , , There was a strong association between specific AMR determinants and three major epidemic C. difficile STs: ST1 (clade 2) with fluoroquinolone resistance (mainly T82I substitution in GyrA) (P<0.0001), ST11 (clade 5) with tetracycline resistance (various tet-family genes) (P<0.0001) and ST37 (clade 4) with macrolide-lincosamide-streptogramin B (MLSB) resistance (mainly ermB) (P<0.0001) and MDR (P<0.0001).)]\n",
            "2952: [(, , , This study provides a comprehensive review of AMR in the global C. difficile population which may aid in the early detection of drug-resistant C. difficile strains, and prevention of their dissemination worldwide.)]\n",
            "2953: [(, , , This study aimed to identify and understand the challenges and opportunities for such collaborative endeavors to enhance equitable partnerships.)]\n",
            "2954: [(178, 194, Others, This review summarizes the existing health infrastructure affecting the establishment of AMR surveillance programs, the burden of bacterial infections demonstrating the need for AMR surveillance, and current progress and challenges in AMR surveillance efforts in eight South and Southeast Asian countries.)]\n",
            "2955: [(89, 113, Microbes, Antimicrobial resistance (AMR) plays an important role in the pathogenesis and spread of Clostridioides difficile infection (CDI), the leading healthcare-related gastrointestinal infection in the world.)]\n",
            "2956: [(61, 73, Antibiotic-Resistant Bacteria, Here, through detailed analysis of 10 330 publicly-available C. difficile genomes from strains isolated worldwide (spanning 270 multilocus sequence types (STs) across all known evolutionary clades), this study provides the first species-wide snapshot of AMR genomic epidemiology in C. difficile .)]\n",
            "2957: [(282, 296, Antibiotic-Resistant Bacteria, Here, through detailed analysis of 10 330 publicly-available C. difficile genomes from strains isolated worldwide (spanning 270 multilocus sequence types (STs) across all known evolutionary clades), this study provides the first species-wide snapshot of AMR genomic epidemiology in C. difficile .)]\n",
            "2958: [(240, 243, Others, Of the 10 330 C . difficile genomes, 4532 (43.9 %) in 89 STs across clades 1–5 carried at least one genotypic AMR determinant, with 901 genomes (8.7 %) carrying AMR determinants for three or more antimicrobial classes (multidrug-resistant, MDR).)]\n",
            "2959: [(279, 295, Microbes, Unfortunately, antimicrobial resistance has been reported in L. monocytogenes. This study conducted the first comprehensive statistical analysis of L. monocytogenes isolate data from the National Pathogen Detection Isolate Browser (NPDIB) to identify the trends for AMR genes in L. monocytogenes.)]\n",
            "2960: [(187, 238, Others, Unfortunately, antimicrobial resistance has been reported in L. monocytogenes. This study conducted the first comprehensive statistical analysis of L. monocytogenes isolate data from the National Pathogen Detection Isolate Browser (NPDIB) to identify the trends for AMR genes in L. monocytogenes.)]\n",
            "2961: [(148, 164, Microbes, Unfortunately, antimicrobial resistance has been reported in L. monocytogenes. This study conducted the first comprehensive statistical analysis of L. monocytogenes isolate data from the National Pathogen Detection Isolate Browser (NPDIB) to identify the trends for AMR genes in L. monocytogenes.)]\n",
            "2962: [(61, 77, Microbes, Unfortunately, antimicrobial resistance has been reported in L. monocytogenes. This study conducted the first comprehensive statistical analysis of L. monocytogenes isolate data from the National Pathogen Detection Isolate Browser (NPDIB) to identify the trends for AMR genes in L. monocytogenes.)]\n",
            "2963: [(0, 23, Techniques, Hierarchical clustering was then used to identify the significant AMR genes found in L. monocytogenes samples and to assess changes during the period from 2010 through to 2021.)]\n",
            "2964: [(33, 58, Others, A novel and previously overlooked tetM-positive transposon designated Tn6944 was identified, predominantly among clade 2 strains.)]\n",
            "2965: [(70, 76, Others, A novel and previously overlooked tetM-positive transposon designated Tn6944 was identified, predominantly among clade 2 strains.)]\n",
            "2966: [(44, 49, Antibiotic-Resistant Genes, Statistical analysis of the data identified fosX, lin, abc-f, and tet(M) as the four most common AMR genes found in L. monocytogenes.)]\n",
            "2967: [(50, 53, Antibiotic-Resistant Genes, Statistical analysis of the data identified fosX, lin, abc-f, and tet(M) as the four most common AMR genes found in L. monocytogenes.)]\n",
            "2968: [(55, 60, Antibiotic-Resistant Genes, Statistical analysis of the data identified fosX, lin, abc-f, and tet(M) as the four most common AMR genes found in L. monocytogenes.)]\n",
            "2969: [(65, 72, Antibiotic-Resistant Genes, Statistical analysis of the data identified fosX, lin, abc-f, and tet(M) as the four most common AMR genes found in L. monocytogenes.)]\n",
            "2970: [(, , , AMR surveillance is critical in LMICs due to high burden of bacterial infections; however, conducting AMR surveillance in resource-limited settings is constrained by poorly functioning health systems, scarce financial resources, and lack of skilled personnel.)]\n",
            "2971: [(, , , Antimicrobial Resistance is recognized as a major threat to global health security.)]\n",
            "2972: [(, , , In 2015, the United Nations World Health Assembly endorsed the World Health Organization's Global Action Plan to tackle AMR; thus, several countries are striving to improve their AMR surveillance capacity, including making significant investments and establishing and expanding surveillance networks.)]\n",
            "2973: [(, , , The WHO Southeast Asia region is dubbed a \"global hub for AMR emergence\", as it runs the highest risk for AMR emergence among all WHO regions in Asia.)]\n",
            "2974: [(, , , Initial data generated from AMR surveillance networks in LMICs suggest the high prevalence of resistance, but these data exhibit several shortcomings, such as a lack of representativeness, lack of standardized laboratory practices, and underutilization of microbiology services.)]\n",
            "2975: [(, , , Hence, there is a need for Asia-centric, collaborative AMR research aligned with the true needs and priorities of the region.)]\n",
            "2976: [(, , , Despite significant progress, AMR surveillance networks in LMICs face several challenges in expansion and sustainability due to limited financial resources and technical capacity.)]\n",
            "2977: [(, , , This qualitative study adopted an interpretative approach involving a thematic analysis of 15 semi-structured interviews with AMR experts conducting research in the region.)]\n",
            "2978: [(, , , The study identified several factors influencing research collaborations, such as the multi-dimensional nature of AMR, limited or lack of funds, different AMR research priorities in Asian countries, absence of Asia-centric AMR leadership, lack of trust and, unequal power relationships between researchers, and the negative impact of the COVID-19 pandemic in research collaborations.)]\n",
            "2979: [(, , , It also identified some opportunities, such as the willingness of researchers to collaborate, the formation of a few networks, and the prioritization by many academics of the One Health paradigm for framing AMR research.)]\n",
            "2980: [(, , , Participants reported that the initiation of stronger cross-discipline and cross-country networks, the development of Asia-centric AMR leadership, flexible research agendas with shared priorities, transparent and transferable funds, and support to enhance research capacity in LMICs could assist in developing more equitable collaborative research in Asia.)]\n",
            "2981: [(, , , In this review, we briefly summarize the history that led to recognition of the various manifestations of AMR in bacterial pathogens and the ways in which they interrelate.)]\n",
            "2982: [(, , , Antimicrobial resistance (AMR) is a global concern, pertaining not only to human health but also to the health of industry and the environment.)]\n",
            "2983: [(, , , Antimicrobial resistance (AMR) is the acquired ability of pathogens to withstand antimicrobial treatment.)]\n",
            "2984: [(, , , AMR research has traditionally focused on genetic exchange mechanisms and abiotic environmental constraints, leaving important aspects of microbial ecology unresolved.)]\n",
            "2985: [(, , , To bridge the gap in knowledge for implementing effective and targeted interventions in relation to the AMR in Armenia, we designed this study to explore the performance of AMR diagnostics and the profile of AMR in the Nork Infection Clinical Hospital (NICH) for the period of 2016-2019, particularly to (i) determine the proportions of antimicrobial resistance among all samples tested at the hospital laboratory, (ii) determine the proportion of resistance against specific antimicrobials, and (iii) identify factors associated with AMR.)]\n",
            "2986: [(, , , The genetic and ecological aspects of AMR, however, not only contribute separately to the problem but also are interrelated.)]\n",
            "2987: [(, , , A cross-sectional study was conducted with a secondary data analysis that included all the patients tested for AMR in the laboratory of the NICH for the period of 2016-2019.)]\n",
            "2988: [(, , , For example, mutualistic associations among microbes such as biofilms can both serve as a barrier to antibiotic penetration and a breeding ground for horizontal exchange of antimicrobial resistance genes (ARGs).)]\n",
            "2989: [(, , , For this period, only 107 (0.3%) patients out of 36,528 had their AMR test results available and of them, 87 (81%) had resistance at least to one tested antimicrobial.)]\n",
            "2990: [(, , , In this review, we elucidate how species interactions promote and impede the establishment, maintenance, and spread of ARGs and indicate how management initiatives might benefit from leveraging the principles and tools of community ecology to better understand and manipulate the processes underlying AMR.)]\n",
            "2991: [(, , , This study has provided some valuable information on the AMR situation in Armenia.)]\n",
            "2992: [(, , , Antimicrobial resistance (AMR) is a global health crisis that affects all life on Earth.)]\n",
            "2993: [(, , , The results call for immediate actions to control the access to and the use of antimicrobials, strengthen AMR surveillance, and improve laboratory capacity for the proper and fast identification of drug resistance through a comprehensive system.)]\n",
            "2994: [(, , , In 2015, the World Health Organization developed guidance to combat AMR in accordance with a One Health framework considering human, animal, and environment sectors of planetary health.)]\n",
            "2995: [(, , , Antimicrobial resistance (AMR) is a globally recognized crisis with meaningful engagement across humans, animals, and the environment as in the One Health approach.)]\n",
            "2996: [(, , , The environment is the potential source, reservoir, and transmission route of AMR, and it plays a key role in AMR development from the One Health perspective.)]\n",
            "2997: [(, , , This review found that global and national stewardship primarily focuses on mitigating antibiotic use in the human and animal sectors, overlooking environmental drivers, particularly diverse environmental waters.)]\n",
            "2998: [(, , , Animal farming, hospitals, and the pharmaceutical industry are identified as the main emission sources in the environment.)]\n",
            "2999: [(, , , The findings of this study highlight the need to broaden the scope of water-related AMR concerns beyond water, sanitation, and hygiene (WASH) infrastructure for water supply and wastewater treatment, and account for environmental waters in AMR development and dissemination, particularly in low-income countries where half a billion people rely on environmental waters to meet daily needs.)]\n",
            "3000: [(, , , Equitably accounting for water environments, supplies, and waste in AMR prevention, mitigation, surveillance, and innovation can significantly enhance the integration of environmental objectives in One Health AMR stewardship.)]\n",
            "3001: [(, , , Antimicrobial resistance (AMR) is an urgent public health threat, and continues to be on the rise.)]\n",
            "3002: [(, , , Basic microbiological research is the foundation for addressing knowledge gaps both for the development of new antibiotics, diagnostics and preventives but also to inform strategies to mitigate the transmission of resistance and drug resistant microorganisms.)]\n",
            "3003: [(, , , Translating this research into new products to reinvigorate dwindling pipelines, especially for new antibiotics, is one of the main challenges faced in addressing AMR.)]\n",
            "3004: [(, , , The scientific complexity is compounded by the market uncertainty of any new products leading to a large proportion of pharmaceutical companies exiting the market.)]\n",
            "3005: [(, , , Consequently, a number of initiatives were developed to reinvigorate the AMR research and development (R&D) landscape.)]\n",
            "3006: [(, , , Despite all these efforts, the antibiotic pipeline remains inadequate to keep up with the increasing rates of resistance globally.)]\n",
            "3007: [(, , , We present an integrated view of AMR by reframing it as a spectrum of phenotypes within a continuous three-dimensional space defined by the growth rate, prevalence, and kill rate of cells displaying AMR.)]\n",
            "3008: [(, , , Finally, we reflect on strategies that may help stem the emergence of AMR.)]\n",
            "3009: [(, , , Given the number of individual and multilateral actions, there is an urgent need for a common platform and coordination to ensure that resources are adequately used to address the critical challenges posed by AMR globally hence the founding of the Global AMR R&D Hub to take on this role.)]\n",
            "3010: [(, , , We present the pooled prevalence of AMR for major bacterium-antibiotic combination in different sample types.)]\n",
            "3011: [(, , , The latter includes a description of standard pig farming practices, detection of AMR and clonal analysis of bacteria, and AMR in the food chain and associated environments. Finally, the possibility of using alternatives to antibiotics and improving policies for future strategies in combating AMR in a SEA context are outlined.)]\n",
            "3012: [(, , , The pooled prevalence of AMR in bacteria from food producing live animals was 20%. High estimates of AMR pooled prevalence were found in bacteria identified from milk, food handlers and the environmental samples with 29%, and 28% in meat.)]\n",
            "3013: [(, , , The Pathosystems Resource Integration Center (PATRIC, www.patricbrc.org) is designed to provide researchers with the tools and services that they need to perform genomic and other 'omic' data analyses.)]\n",
            "3014: [(, , , In response to mounting concern over antimicrobial resistance (AMR), the PATRIC team has been developing new tools that help researchers understand AMR and its genetic determinants.)]\n",
            "3015: [(, , , In conclusion, the high estimate of prevalence of AMR observed in bacteria recovered from different sample sources related to the animal-source food chain (food, live animal and environment) can highlight the possible linkage among them.)]\n",
            "3016: [(, , , The MDR levels in several bacteria species are a clear indication that the threat is directed to many antimicrobials.)]\n",
            "3017: [(105, 121, Microbes, Microbes reported having a higher MDR pattern were: Staphylococcus spp. (96%), Salmonella spp. (81%) and Escherichia coli (77%).)]\n",
            "3018: [(52, 71, Microbes, Microbes reported having a higher MDR pattern were: Staphylococcus spp. (96%), Salmonella spp. (81%) and Escherichia coli (77%).)]\n",
            "3019: [(79, 94, Microbes, Microbes reported having a higher MDR pattern were: Staphylococcus spp. (96%), Salmonella spp. (81%) and Escherichia coli (77%).)]\n",
            "3020: [(96, 111, Techniques, To support comparative analyses, we have added AMR phenotype data to over 15 000 genomes in the PATRIC database, often assembling genomes from reads in public archives and collecting their associated AMR panel data from the literature to augment the collection.)]\n",
            "3021: [(113, 124, Others, The present review revealed a high resistance against commonly used drugs for animal and human treatments and/or prophylaxis.)]\n",
            "3022: [(, , , Our review demonstrated that the high overall AMR resistance levels call for effective policy and intervention measures, which best address the problem along the food chain through a One Health approach.)]\n",
            "3023: [(, , , Antimicrobial resistance (AMR) is a critical challenge worldwide as it impacts public health, especially via contamination in the food chain and in healthcare-associated infections.)]\n",
            "3024: [(, , , In relation to farming, the systems used, waste management on farms, and the production line process are all determinants reflecting the risk of AMR emergence and rate of contamination of foodstuffs.)]\n",
            "3025: [(, , , This review focuses on South East Asia (SEA), which contains diverse regions covering 11 countries, each having different levels of development, customs, laws, and regulations.)]\n",
            "3026: [(, , , It is desirable to establish a holistic framework to coordinate international actions on the containment of environmental AMR development.)]\n",
            "3027: [(, , , Routinely, here as elsewhere antimicrobials are still used for three indications: therapy, prevention, and growth promotion, and these are the fundamental drivers of AMR development and persistence.)]\n",
            "3028: [(, , , To establish a community with a shared future for humanity, China should and could play an important role in international cooperation to cope with AMR challenges.)]\n",
            "3029: [(, , , The accuracy of detection of antibiotic resistant bacteria (ARB) and antibiotic resistance genes (ARG) depends on the laboratory standards applicable in the various institutes and countries, and this affects the consistency of regional data.)]\n",
            "3030: [(, , , Antimicrobial resistance (AMR) has been recognized as one of the greatest global threats for human and animal health.)]\n",
            "3031: [(27, 43, Antibiotic-Resistant Bacteria, Enterobacteriaceae such as Escherichia coli and Klebsiella pneumoniae are the standard proxy species used for indicating AMR-associated nosocomial infections and healthcare-associated infections.)]\n",
            "3032: [(48, 69, Antibiotic-Resistant Bacteria, Enterobacteriaceae such as Escherichia coli and Klebsiella pneumoniae are the standard proxy species used for indicating AMR-associated nosocomial infections and healthcare-associated infections.)]\n",
            "3033: [(0, 18, Microbes, Enterobacteriaceae such as Escherichia coli and Klebsiella pneumoniae are the standard proxy species used for indicating AMR-associated nosocomial infections and healthcare-associated infections.)]\n",
            "3034: [(, , , The present review retrieved up to date information on the epidemiology of AMR in the animal-source food chain in Ethiopia focusing on AMR in bacterial species isolated from food handlers, live animals, foods (animal origin and non-animal origin), and in environmental samples.)]\n",
            "3035: [(, , , Pig feces and wastewater have been suspected as one of the hotspots for spread and circulation of ARB and ARG.)]\n",
            "3036: [(, , , Accordingly, pooled prevalence of AMR in the different sources was estimated.)]\n",
            "3037: [(, , , As part of AMR surveillance in a One Health approach, clonal typing is used to identify bacterial clonal transmission from the production process to consumers and patients - although to date there have been few published definitive studies about this in SEA.)]\n",
            "3038: [(, , , For data analysis, we used random effect meta-analysis and in order to avoid exclusion of studies with zero prevalence of antimicrobial resistance, Freeman-Tukey double arcsine transformation was applied.)]\n",
            "3039: [(, , , Various alternatives to antibiotics are available to reduce antibiotic use on farms. Certain of these alternatives together with improved disease prevention methods are essential tools to reduce antimicrobial usage in swine farms and to support global policy.)]\n",
            "3040: [(, , , We identified 152 eligible studies and retrieved 4097 data records (183 in food handlers, 2055 in foods, 1040 in live animals and 819 for environmental samples) which together reported a total of 86,813 AMR tests with 64 different antimicrobial disks for 81 bacteria species.)]\n",
            "3041: [(, , , This review highlights evidence for potential transfer of resistant bacteria from food animals to humans, and awareness and understanding of AMR through a description of the occurrence of AMR in pig farm food chains under SEA management systems.)]\n",
            "3042: [(19, 32, Microbes, A 96% agreement on Campylobacter AMR profiles was achieved between the results from the IoT and the conventional broth microdilution method.)]\n",
            "3043: [(, , , The data collected from the intelligent sensors were transmitted from local computers to a cloud server, facilitating real-time data collection and integration.)]\n",
            "3044: [(, , , A web browser was developed to demonstrate the spatial and temporal AMR trends to end-users.)]\n",
            "3045: [(, , , We have developed an ordinary differential equation model to investigate the effects of farm practice on the spread and persistence of AMR in the dairy slurry tank environment.)]\n",
            "3046: [(, , , We have also been using this collection of AMR metadata to build machine learning-based classifiers that can predict the AMR phenotypes and the genomic regions associated with resistance for genomes being submitted to the annotation service.)]\n",
            "3047: [(, , , This rapid, cost-effective, and portable approach is able to monitor, assess, and mitigate the burden of bacterial AMR in the agri-food chain.)]\n",
            "3048: [(, , , Likewise, we have undertaken a large AMR protein annotation effort by manually curating data from the literature and public repositories.)]\n",
            "3049: [(80, 119, Techniques, To address these challenges, we present the internet of things (IoT), including colorimetric-based microfluidic sensors, a custom-built portable incubator, and machine learning algorithms, to monitor AMR trends in real time.)]\n",
            "3050: [(140, 169, Others, As a top priority microbe that poses risks to human health, Campylobacter was selected as a bacterial model to demonstrate and validate the IoT-assisted AMR surveillance.)]\n",
            "3051: [(60, 73, Antibiotic-Resistant Bacteria, As a top priority microbe that poses risks to human health, Campylobacter was selected as a bacterial model to demonstrate and validate the IoT-assisted AMR surveillance.)]\n",
            "3052: [(76, 89, Microbes, The IoT was used to carry out a small-scale survey study, identifying eight Campylobacter isolates out of 35 chicken samples.)]\n",
            "3053: [(126, 132, Others, This collection of 7370 AMR reference proteins, which contains many protein annotations (functional roles) that are unique to PATRIC and RAST, has been manually curated so that it projects stably across genomes.)]\n",
            "3054: [(137, 141, Others, This collection of 7370 AMR reference proteins, which contains many protein annotations (functional roles) that are unique to PATRIC and RAST, has been manually curated so that it projects stably across genomes.)]\n",
            "3055: [(, , , Finally, the PATRIC Web site has been expanded to enable AMR-based custom page views so that researchers can easily explore AMR data and design experiments based on whole genomes or individual genes.)]\n",
            "3056: [(, , , Forty-seven (94%) products contained bacterial DNA. Of these, 97% contained at least 1 AMR gene, and 82% contained 2 or more.)]\n",
            "3057: [(, , , These results indicate that further evaluation of the risk for transmission of these AMR genes may be warranted.)]\n",
            "3058: [(, , , Antimicrobial resistance (AMR) of foodborne pathogens is a global crisis in public health and economic growth.)]\n",
            "3059: [(, , , A real-time surveillance system is key to track the emergence of AMR bacteria and provides a comprehensive AMR trend from farm to fork.)]\n",
            "3060: [(, , , However, current AMR surveillance systems, which integrate results from multiple laboratories using the conventional broth microdilution method, are labor-intensive and time-consuming.)]\n",
            "3061: [(, , , Image classification with convolution neural network ResNet50 on the colorimetric sensors achieved an accuracy of 99.5% in classifying bacterial growth/inhibition patterns.)]\n",
            "3062: [(, , , Emerging topics included Water and environment, and Sequencing.)]\n",
            "3063: [(80, 84, Others, Whatever the sanitation system used, an important aspect of controlling AMR and HAIs relates to the ability to check any variation of a microbial population rapidly and effectively, thus effective monitoring procedures are also described.)]\n",
            "3064: [(, , , Thus, future studies should engage directly with policymakers to promote the implementation of continent-wide AMR reporting standards.)]\n",
            "3065: [(, , , Further, the causal factors leading to adverse effects of antibiotic-load to river organisms with an elaboration on the current measures to eradicate the ARB, ARGs, and remove antibiotics from the urban river ecosystems are also discussed.)]\n",
            "3066: [(43, 49, Microbes, This manuscript reviews the application of phages in order to prevent food- and water-borne diseases from a One Health perspective.)]\n",
            "3067: [(, , , Antibiotics are the widely used therapeutic options to combat infectious diseases; however, unfortunately, inadequate and irregular antibiotic courses are also major contributing factors in the emergence of AMR.)]\n",
            "3068: [(, , , The escalating threat of antibiotic-resistant pathogens resulting from the rapid emergence of antimicrobial resistance (AMR) is a significant health and economic concern worldwide, exacerbated by the misuse of antibiotics in both humans and livestock.)]\n",
            "3069: [(, , , Unfortunately, the dawn of the antibiotic era has sadly corresponded to the rise of the phenomenon of antimicrobial resistance (AMR), which is a natural process whereby microbes evolve in such a way to withstand the action of drugs.)]\n",
            "3070: [(172, 201, Others, These include the need to implement diagnostic stewardship to assess the global incidence of coinfections and secondary infections in COVID-19 patients, including those by multidrug-resistant pathogens, to identify patients most likely to benefit from antibiotic treatment and identify when antibiotics can be safely withheld, de-escalated or discontinued.)]\n",
            "3071: [(, , , Vaccination is one of the most effective strategies to fight antimicrobial resistance. Vaccines, unlike drugs, are less likely to produce resistance since they are precise to their target illnesses.)]\n",
            "3072: [(, , , In this Review, we outline a theoretical and practical framework for the development of effective antibiotic combinations.)]\n",
            "3073: [(, , , We present some of the initiatives and activities at the national and global level that target the AMR challenge.)]\n",
            "3074: [(115, 140, Others, We discuss the following components for inclusion: development of infectious disease diagnostics and therapeutics; antimicrobial stewardship interventions in long-term care and Indigenous communities; environmental monitoring of AMR; comprehensive antimicrobial use, and AMR surveillance; and support for capacity-building in low and middle-income countries.)]\n",
            "3075: [(271, 287, Others, We discuss the following components for inclusion: development of infectious disease diagnostics and therapeutics; antimicrobial stewardship interventions in long-term care and Indigenous communities; environmental monitoring of AMR; comprehensive antimicrobial use, and AMR surveillance; and support for capacity-building in low and middle-income countries.)]\n",
            "3076: [(, , , Antimicrobial use has been identified as a major driver of AMR, and reductions in use are a focal point of interventions to reduce resistance.)]\n",
            "3077: [(46, 54, Microbes, Finally, we discuss the zoonotic potential of S. canis, highlighting the need for further evidence in this area.)]\n",
            "3078: [(, , , Objectives: Considering the relevance of including the social sciences, this scoping review assesses the state of training on the social dimensions of AMR.)]\n",
            "3079: [(, , , As declared by WHO, antimicrobial resistance (AMR) is a high priority issue with a pressing need to develop impactful technologies to curb it.)]\n",
            "3080: [(72, 96, Techniques, Although antibiotics continue to be essential to treat such infections, non-antibiotic therapies will play an important role in limiting the increase of antibiotic resistant microorganisms.)]\n",
            "3081: [(57, 72, Antibiotics, With 16 and 19 proteins or protein complexes involved in fluoroquinolone and glycopeptide resistances, respectively, and the complexities of bacterial sensing mechanisms that trigger and regulate a wide variety of possible resistance mechanisms, we propose that these antimicrobial resistance mechanisms might be considered complex 'nanomachines' that drive survival of bacterial cells in antibiotic environments.)]\n",
            "3082: [(, , , Bacteriophage-derived endolysins are cell wall hydrolases which could hydrolyze the peptidoglycan layer from within and outside of bacterial pathogens.)]\n",
            "3083: [(17, 46, Others, In this context, outer membrane vesicles (OMV) represent a promising approach, as they recapitulate the complete antigen repertoire that occurs on the surface of Gram-negative bacteria.)]\n",
            "3084: [(162, 184, Microbes, In this context, outer membrane vesicles (OMV) represent a promising approach, as they recapitulate the complete antigen repertoire that occurs on the surface of Gram-negative bacteria.)]\n",
            "3085: [(113, 131, Techniques, In this context, outer membrane vesicles (OMV) represent a promising approach, as they recapitulate the complete antigen repertoire that occurs on the surface of Gram-negative bacteria.)]\n",
            "3086: [(0, 31, Others, Antimicrobial stewardship (AMS), an organizational or system-wide health-care strategy, is designed to promote, improve, monitor, and evaluate the rational use of antimicrobials to preserve their future effectiveness, along with the promotion and protection of public health.)]\n",
            "3087: [(, , , However, a comprehensive assessment of AMR levels in Antarctica is virtually impossible on current data due to the wide variation in reporting standards and methodologies used and poor geographical coverage.)]\n",
            "3088: [(, , , There is evidence to suggest that factors such as the proliferation of adulterated antimicrobials in some developing countries, international travels, issues with healthcare financing, use/misuse by humans, and in agricultural production and climate change are determinants of AMR at various levels of society.)]\n",
            "3089: [(119, 138, Microbes, Niche environments, including plants, animals, and humans appear to be colonized by different phylogroups, among which KpI (K. pneumoniae) is commonly associated with human infection.)]\n",
            "3090: [(, , , Improved diagnostics are urgently needed to improve AMR surveillance and the management of infected patients.)]\n",
            "3091: [(, , , As countries worldwide prepare to embrace a One Health approach to AMR management, which is one that recognizes the interconnectivity between human, animal, and environmental health, increasing attention is being paid to identifying and monitoring key contributing factors and critical control points.)]\n",
            "3092: [(42, 81, Techniques, The sentinel laboratories will use simple antimicrobial susceptibility test (AST) methods such as disk diffusion tests, whereas the reference laboratories should use automated quantitative AST methods as well as advanced molecular methods to explicit AMR emergence mechanisms.)]\n",
            "3093: [(98, 118, Techniques, The sentinel laboratories will use simple antimicrobial susceptibility test (AST) methods such as disk diffusion tests, whereas the reference laboratories should use automated quantitative AST methods as well as advanced molecular methods to explicit AMR emergence mechanisms.)]\n",
            "3094: [(166, 200, Techniques, The sentinel laboratories will use simple antimicrobial susceptibility test (AST) methods such as disk diffusion tests, whereas the reference laboratories should use automated quantitative AST methods as well as advanced molecular methods to explicit AMR emergence mechanisms.)]\n",
            "3095: [(212, 238, Techniques, The sentinel laboratories will use simple antimicrobial susceptibility test (AST) methods such as disk diffusion tests, whereas the reference laboratories should use automated quantitative AST methods as well as advanced molecular methods to explicit AMR emergence mechanisms.)]\n",
            "3096: [(, , , Correlation analyses (odds ratio) and relationships between bacteria density and ARG were determined.)]\n",
            "3097: [(, , , This change represents a 14% total reduction in dispensed scripts after the series of intervention programs began in 2009.)]\n",
            "3098: [(183, 194, Microbes, As our knowledge on factors that increase expression of resistance genes in the absence of IS elements is limited, further research is needed prior to implementing AMR prediction for B. fragilis from whole-genome sequencing.)]\n",
            "3099: [(200, 223, Techniques, As our knowledge on factors that increase expression of resistance genes in the absence of IS elements is limited, further research is needed prior to implementing AMR prediction for B. fragilis from whole-genome sequencing.)]\n",
            "3100: [(63, 94, Antibiotic-Resistant Bacteria, Feeding tylosin for short durations may mitigate the impact on macrolide-resistant enterococci and further studies are encouraged to determine the effect of minimizing or eliminating tylosin use in beef cattle.)]\n",
            "3101: [(78, 100, Microbes, Our work demonstrates a great variability of plasmids in human and veterinary staphylococcal strains and their ability to maintain and express resistance plasmids from other staphylococcal species.)]\n",
            "3102: [(174, 196, Microbes, Our work demonstrates a great variability of plasmids in human and veterinary staphylococcal strains and their ability to maintain and express resistance plasmids from other staphylococcal species.)]\n",
            "3103: [(18, 22, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3104: [(24, 32, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3105: [(34, 45, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3106: [(47, 55, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3107: [(57, 66, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3108: [(68, 71, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3109: [(73, 79, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3110: [(81, 84, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3111: [(86, 92, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3112: [(98, 102, Antibiotic-Resistant Genes, It turns out that aadA, aph(3''), aph(3'')-Ib, aph(6)-I, aph(6)-Id, bla, blaCMY, tet, tet(A), and sul2 constructed the assembly of ten genes that were most commonly involved in antimicrobial resistance in these six states.)]\n",
            "3113: [(15, 22, Microbes, In conclusion, E. coli bacterium can harbor high resistance to different classes of antimicrobials which increases the risk of potential uncontrolled transmission of the multi-drug resistant E. coli bacterium to veterinarians and horse handlers, as well as to the equine population itself.)]\n",
            "3114: [(170, 198, Antibiotic-Resistant Bacteria, In conclusion, E. coli bacterium can harbor high resistance to different classes of antimicrobials which increases the risk of potential uncontrolled transmission of the multi-drug resistant E. coli bacterium to veterinarians and horse handlers, as well as to the equine population itself.)]\n",
            "3115: [(, , , The increasing prevalence of antimicrobial-resistant bacteria drives the need for advanced methods to identify antimicrobial-resistance (AMR) genes in bacterial pathogens.)]\n",
            "3116: [(115, 137, Techniques, Hospital sewage was contaminated with many ARB species, ARGs and residual antibiotics, which can cause a burden on WWTP sewage treatment.)]\n",
            "3117: [(, , , In particular, the extensive use of antibiotics in food animals and agricultural settings could be significantly contributed to the emergence of antibiotic-resistant pathogens.)]\n",
            "3118: [(100, 118, Antibiotic-Resistant Bacteria, The objectives of this study were to estimate the prevalence and antimicrobial-resistance status of Campylobacter spp. in human diarrheal samples collected from Surya Kanta Hospital, Mymensingh, Bangladesh.)]\n",
            "3119: [(0, 21, Antibiotic-Resistant Bacteria, Staphylococcus aureus is a Gram-positive pathogen that causes various infections in humans and domestic animals.)]\n",
            "3120: [(27, 49, Microbes, Staphylococcus aureus is a Gram-positive pathogen that causes various infections in humans and domestic animals.)]\n",
            "3121: [(, , , Other strategies involving manipulation of the microbiome to deplete antibiotic-resistant organisms or to enhance immune responses to vaccines may prove valuable in addressing antimicrobial resistance as well.)]\n",
            "3122: [(25, 36, Antibiotic-Resistant Bacteria, Eventually, 13 different Vibrio spp. were successfully isolated and characterized using the pyrH gene.)]\n",
            "3123: [(92, 101, Gene/Proteins, Eventually, 13 different Vibrio spp. were successfully isolated and characterized using the pyrH gene.)]\n",
            "3124: [(5, 11, Antibiotic-Resistant Genes, Both mef(E) and mel were upregulated > 400-fold, and spxB (encoding an enzyme responsible for production of most H2O2) was upregulated 14.2-fold, whereas transcription of the autolysin (lytA) gene was downregulated when incubated with erythromycin.)]\n",
            "3125: [(16, 19, Antibiotic-Resistant Genes, Both mef(E) and mel were upregulated > 400-fold, and spxB (encoding an enzyme responsible for production of most H2O2) was upregulated 14.2-fold, whereas transcription of the autolysin (lytA) gene was downregulated when incubated with erythromycin.)]\n",
            "3126: [(53, 57, Gene/Proteins, Both mef(E) and mel were upregulated > 400-fold, and spxB (encoding an enzyme responsible for production of most H2O2) was upregulated 14.2-fold, whereas transcription of the autolysin (lytA) gene was downregulated when incubated with erythromycin.)]\n",
            "3127: [(175, 196, Gene/Proteins, Both mef(E) and mel were upregulated > 400-fold, and spxB (encoding an enzyme responsible for production of most H2O2) was upregulated 14.2-fold, whereas transcription of the autolysin (lytA) gene was downregulated when incubated with erythromycin.)]\n",
            "3128: [(235, 247, Antibiotics, Both mef(E) and mel were upregulated > 400-fold, and spxB (encoding an enzyme responsible for production of most H2O2) was upregulated 14.2-fold, whereas transcription of the autolysin (lytA) gene was downregulated when incubated with erythromycin.)]\n",
            "3129: [(82, 90, Antibiotics, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3130: [(129, 139, Antibiotics, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3131: [(141, 144, Antibiotics, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3132: [(146, 152, Antibiotics, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3133: [(154, 157, Antibiotics, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3134: [(159, 162, Antibiotics, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3135: [(92, 97, Antibiotic-Resistant Genes, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3136: [(99, 104, Antibiotic-Resistant Genes, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3137: [(106, 111, Antibiotic-Resistant Genes, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3138: [(112, 117, Antibiotic-Resistant Genes, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3139: [(118, 123, Antibiotic-Resistant Genes, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3140: [(175, 179, Antibiotic-Resistant Genes, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3141: [(207, 217, Antibiotics, The ARG were confirmed for all investigated classes of antimicrobials, except for colistin (mcr-1, mcr-2, mcr-3,mcr-4 mcr-5) and carbapenem (IMP, OXA-48, NDM, KPC), while the vanB gene was only detected for vancomycin.)]\n",
            "3142: [(31, 35, Antibiotic-Resistant Genes, Eighty-five staphylococci were mecA gene-positive, and 40% of them harbored SCCmec IV and V.)]\n",
            "3143: [(76, 91, Gene/Proteins, Eighty-five staphylococci were mecA gene-positive, and 40% of them harbored SCCmec IV and V.)]\n",
            "3144: [(, , , Ubiquitous in nature, antimicrobial resistance (AMR) has existed long before the golden age of antimicrobials.)]\n",
            "3145: [(, , , The introduction of antibiotics revolutionized medicine in the 20th-century permitting the treatment of once incurable infections.)]\n",
            "3146: [(43, 55, Microbes, In this review we analyze the diversity of enterococcal species and their distribution in the intestinal tract of animals.)]\n",
            "3147: [(, , , In rare cases, AST at the level of single-cell visualization was developed. This has not yet led to major changes in routine high-throughput clinical microbiological detection of antimicrobial resistance.)]\n",
            "3148: [(26, 41, Microbes, The isolation rate of the Salmonella spp. is decreasing, reflecting the current socio-economic status of South Korea.)]\n",
            "3149: [(0, 19, Others, Multiple resistance (to four or more antimicrobials) is also common in the poultry-associated pathogens Salmonella virchow and Salmonella hadar, with an increasing number of strains of these serotypes exhibiting decreased susceptibility to ciprofloxacin.)]\n",
            "3150: [(240, 253, Antibiotics, Multiple resistance (to four or more antimicrobials) is also common in the poultry-associated pathogens Salmonella virchow and Salmonella hadar, with an increasing number of strains of these serotypes exhibiting decreased susceptibility to ciprofloxacin.)]\n",
            "3151: [(104, 122, Microbes, Multiple resistance (to four or more antimicrobials) is also common in the poultry-associated pathogens Salmonella virchow and Salmonella hadar, with an increasing number of strains of these serotypes exhibiting decreased susceptibility to ciprofloxacin.)]\n",
            "3152: [(127, 143, Microbes, Multiple resistance (to four or more antimicrobials) is also common in the poultry-associated pathogens Salmonella virchow and Salmonella hadar, with an increasing number of strains of these serotypes exhibiting decreased susceptibility to ciprofloxacin.)]\n",
            "3153: [(, , , This article reviews the current situation of antimicrobial resistance in companion small animals and highlights how important it is for veterinarians to recognize the significance of antimicrobial resistance and to commit to the judicious use of antimicrobial agents.)]\n",
            "3154: [(, , , Coronavirus SARS-CoV-2 causes COVID-19 illness which can progress to severe pneumonia.)]\n",
            "3155: [(, , , The availability of new compounds that show in vitro efficacy against MDR represents a unique opportunity to face the threat of resistance and to optimize the current use of antimicrobials, potentially reducing toxicity.)]\n",
            "3156: [(199, 222, Microbes, These mechanisms of resistance are not exclusive and the interplay of several mechanisms causes high levels of resistance.In this review, we explore the molecular mechanisms underlying resistance in Gram-negative organisms and how these different mechanisms enable them to survive many different stress conditions.)]\n",
            "3157: [(, , , Unlike conventional antibiotics, repeated use of the polymers does not lead to drug resistance.)]\n",
            "3158: [(, , , In addition, our study highlights the potential application of AMPs in treating ocular bacterial infections.)]\n",
            "3159: [(108, 112, Antibiotic-Resistant Bacteria, Among them, compound 5i showed excellent antibacterial activity against Gram-positive bacteria and clinical MRSA isolates (minimum inhibitory concentrations (MICs) = 0.5-2 μg/mL) with low hemolytic and cytotoxic activities and high membrane selectivity.)]\n",
            "3160: [(, , , Cellular entry is by means of diffusion and involves, at least in part, the porin pathway in the outer membrane of gram-negative bacteria.)]\n",
            "3161: [(, , , The development of new antimicrobials has become an urgent priority because of a global challenge emerging from the rise of antimicrobial resistant pathogens.)]\n",
            "3162: [(125, 136, Microbes, Here, we review current known mechanisms of antimicrobial tolerance, and include an in-depth analysis of those identified in enterococci with implications for both the development and prevention of AMR.)]\n",
            "3163: [(30, 42, Antibiotics, Of the antimicrobials tested, enrofloxacin had the best performance, with only one (04.54%) isolate resistant to it, on the other hand, the antimicrobials with the lowest performance were cefotaxime and penicillin with 19 (82.36%) and 18 (81.81%) resistant isolates, respectively.)]\n",
            "3164: [(188, 198, Antibiotics, Of the antimicrobials tested, enrofloxacin had the best performance, with only one (04.54%) isolate resistant to it, on the other hand, the antimicrobials with the lowest performance were cefotaxime and penicillin with 19 (82.36%) and 18 (81.81%) resistant isolates, respectively.)]\n",
            "3165: [(203, 213, Antibiotics, Of the antimicrobials tested, enrofloxacin had the best performance, with only one (04.54%) isolate resistant to it, on the other hand, the antimicrobials with the lowest performance were cefotaxime and penicillin with 19 (82.36%) and 18 (81.81%) resistant isolates, respectively.)]\n",
            "3166: [(, , , Possible strategies to optimize antibiotic use in everyday clinical practice and to reduce the risk of inducing bacterial resistance include: the implementation of rapid microbiological diagnostics for identification and antimicrobial susceptibility testing, the use of inflammation markers to guide initiation and duration of therapies, the reduction of standard durations of antibiotic courses, the individualization of antibiotic therapies and dosing considering pharmacokinetics/pharmacodynamics targets, and avoiding antibiotic classes carrying a higher risk for induction of bacterial resistance.)]\n",
            "3167: [(4, 15, Microbes, For Enterococci, 48 isolates were obtained from fecal samples (25 before treatment and 23 after treatment).)]\n",
            "3168: [(, , , Acute diarrhea (AD) can have significant impacts on military troop readiness.)]\n",
            "3169: [(, , , Literature analysis showed that the emergence of antibiotic resistance is an unavoidable phenomenon closely correlated with the use of antibiotics themselves.)]\n",
            "3170: [(, , , This finding has implications for farming practice and the policies that influence waste management practices.)]\n",
            "3171: [(42, 55, Antibiotics, All the isolates were susceptible towards ciprofloxacin. Plasmid curing and further AMR tests could reveal the location of antibiotic resistance genes mainly as plasmids which determine the large extent of AMR spread through horizontal gene transfer.)]\n",
            "3172: [(111, 123, Antibiotics, The vanA gene was detected in 37.7% of isolates, whereas vanB and vanC1 genes were not observed in any strain. Erythromycin resistance rate was 79.2% and the frequencies of ermB and ermC genes were 88.6% and 69.8%, respectively.)]\n",
            "3173: [(4, 8, Antibiotic-Resistant Genes, The vanA gene was detected in 37.7% of isolates, whereas vanB and vanC1 genes were not observed in any strain. Erythromycin resistance rate was 79.2% and the frequencies of ermB and ermC genes were 88.6% and 69.8%, respectively.)]\n",
            "3174: [(57, 61, Antibiotic-Resistant Genes, The vanA gene was detected in 37.7% of isolates, whereas vanB and vanC1 genes were not observed in any strain. Erythromycin resistance rate was 79.2% and the frequencies of ermB and ermC genes were 88.6% and 69.8%, respectively.)]\n",
            "3175: [(66, 71, Antibiotic-Resistant Genes, The vanA gene was detected in 37.7% of isolates, whereas vanB and vanC1 genes were not observed in any strain. Erythromycin resistance rate was 79.2% and the frequencies of ermB and ermC genes were 88.6% and 69.8%, respectively.)]\n",
            "3176: [(173, 177, Antibiotic-Resistant Genes, The vanA gene was detected in 37.7% of isolates, whereas vanB and vanC1 genes were not observed in any strain. Erythromycin resistance rate was 79.2% and the frequencies of ermB and ermC genes were 88.6% and 69.8%, respectively.)]\n",
            "3177: [(182, 186, Antibiotic-Resistant Genes, The vanA gene was detected in 37.7% of isolates, whereas vanB and vanC1 genes were not observed in any strain. Erythromycin resistance rate was 79.2% and the frequencies of ermB and ermC genes were 88.6% and 69.8%, respectively.)]\n",
            "3178: [(0, 9, Gene/Proteins, Resistome is a cluster of microbial genes encoding proteins with necessary functions to resist the action of antibiotics.)]\n",
            "3179: [(150, 161, Microbes, To our knowledge, this is the first experimental study demonstrating the ability to conduct experimental evolution to antimicrobial resistance within macrophages.)]\n",
            "3180: [(, , , Antimicrobial resistance has spread globally, compromising the treatment of common infections.)]\n",
            "3181: [(108, 121, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3182: [(141, 160, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3183: [(162, 182, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3184: [(184, 202, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3185: [(204, 222, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3186: [(224, 241, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3187: [(243, 258, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3188: [(260, 276, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3189: [(278, 289, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3190: [(294, 315, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3191: [(317, 334, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3192: [(336, 355, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3193: [(361, 396, Antibiotics, Stool samples from asymptomatic children (n = 10,008) and those with diarrhea (n = 3,175) were cultured for Campylobacter Disk diffusion for ciprofloxacin (CIP), nalidixic acid (NAL), erythromycin (ERY), azithromycin (AZM), tetracycline (TE), gentamicin (GM), ampicillin (AMP), amoxicillin and clavulanic acid (AMC), ceftriaxone (CRO), chloramphenicol (C), and trimethoprim-sulfamethoxazole (TMS) was determined.)]\n",
            "3194: [(, , , This review examines the modes of action of various types of disinfectants commonly used during poultry processing (quaternary ammonium, organic acids, chlorine, alkaline detergents) and the mechanisms that may confer tolerance to disinfectants and cross-protection to antibiotics.)]\n",
            "3195: [(44, 56, Antibiotics, High rates of resistance were identified to tetracycline (100%), benzylpenicillin (90.9%), and ciprofloxacin (81.8%). One isolate had a high cefixime MIC (0.75 μg/ml). Twenty-eight NG-MAST sequence types (STs) were identified, seventeen of which were novel.)]\n",
            "3196: [(65, 81, Antibiotics, High rates of resistance were identified to tetracycline (100%), benzylpenicillin (90.9%), and ciprofloxacin (81.8%). One isolate had a high cefixime MIC (0.75 μg/ml). Twenty-eight NG-MAST sequence types (STs) were identified, seventeen of which were novel.)]\n",
            "3197: [(95, 108, Antibiotics, High rates of resistance were identified to tetracycline (100%), benzylpenicillin (90.9%), and ciprofloxacin (81.8%). One isolate had a high cefixime MIC (0.75 μg/ml). Twenty-eight NG-MAST sequence types (STs) were identified, seventeen of which were novel.)]\n",
            "3198: [(141, 153, Antibiotics, High rates of resistance were identified to tetracycline (100%), benzylpenicillin (90.9%), and ciprofloxacin (81.8%). One isolate had a high cefixime MIC (0.75 μg/ml). Twenty-eight NG-MAST sequence types (STs) were identified, seventeen of which were novel.)]\n",
            "3199: [(44, 56, Antibiotics, High rates of resistance were identified to tetracycline (100%), benzylpenicillin (90.9%), and ciprofloxacin (81.8%). One isolate had a high cefixime MIC (0.75 μg/ml). Twenty-eight NG-MAST sequence types (STs) were identified, seventeen of which were novel.)]\n",
            "3200: [(65, 81, Antibiotics, High rates of resistance were identified to tetracycline (100%), benzylpenicillin (90.9%), and ciprofloxacin (81.8%). One isolate had a high cefixime MIC (0.75 μg/ml). Twenty-eight NG-MAST sequence types (STs) were identified, seventeen of which were novel.)]\n",
            "3201: [(95, 108, Antibiotics, High rates of resistance were identified to tetracycline (100%), benzylpenicillin (90.9%), and ciprofloxacin (81.8%). One isolate had a high cefixime MIC (0.75 μg/ml). Twenty-eight NG-MAST sequence types (STs) were identified, seventeen of which were novel.)]\n",
            "3202: [(141, 149, Antibiotics, High rates of resistance were identified to tetracycline (100%), benzylpenicillin (90.9%), and ciprofloxacin (81.8%). One isolate had a high cefixime MIC (0.75 μg/ml). Twenty-eight NG-MAST sequence types (STs) were identified, seventeen of which were novel.)]\n",
            "3203: [(181, 188, Techniques, High rates of resistance were identified to tetracycline (100%), benzylpenicillin (90.9%), and ciprofloxacin (81.8%). One isolate had a high cefixime MIC (0.75 μg/ml). Twenty-eight NG-MAST sequence types (STs) were identified, seventeen of which were novel.)]\n",
            "3204: [(, , , This review was conducted to summarize the situation of AMR in Nepal, determinants of AMR, current government intervention strategies and the way forward to reduce the AMR burden in Nepal.)]\n",
            "3205: [(, , , Antimicrobial resistance (AMR) is an increasing global threat that affects human, animal and, often less acknowledged, environmental health.)]\n",
            "3206: [(, , , At least 47 freely accessible bioinformatics resources for detection of AMR determinants in DNA amino acid sequence data have been developed to date.)]\n",
            "3207: [(, , , Heavy metals have the potential to influence the transmission of antimicrobial resistance (AMR).)]\n",
            "3208: [(, , , IMPORTANCE Antimicrobial resistance causes thousands of deaths annually worldwide. Understanding the regions of the genome that are involved in antimicrobial resistance is important for developing mitigation strategies and preventing transmission.)]\n",
            "3209: [(, , , We identified the human, animal, and environmental contributions to AMR in rural locations, and information gaps on AMR transmission routes and AMR drivers.)]\n",
            "3210: [(, , , This is the first study providing a global overview of the spatial and temporal trends of research related to AMR in wildlife.)]\n",
            "3211: [(, , , It was determined that there was no increase in AMR genes during the studied time period.)]\n",
            "3212: [(63, 78, Techniques, The collection currently projects to 1 610 744 proteins in the PATRIC database.)]\n",
            "3213: [(, , , With high specificity, rapid action, high efficiency, and low risk of resistance development, endolysins are believed to be among the best alternative therapeutic agents to treat multidrug resistant (MDR) bacteria.)]\n",
            "3214: [(16, 37, Microbes, The genotype of Staphylococcus aureus (S. aureus) was ST25, and the dominant genotype of methicillin-resistant Staphylococcus epidermidis (MRSE) was ST59.)]\n",
            "3215: [(111, 137, Microbes, The genotype of Staphylococcus aureus (S. aureus) was ST25, and the dominant genotype of methicillin-resistant Staphylococcus epidermidis (MRSE) was ST59.)]\n",
            "3216: [(39, 48, Microbes, The genotype of Staphylococcus aureus (S. aureus) was ST25, and the dominant genotype of methicillin-resistant Staphylococcus epidermidis (MRSE) was ST59.)]\n",
            "3217: [(0, 23, Techniques, Transcriptomic analysis of bacteria further supports development of resistance to antibiotics but not to the macromolecules after 30 treatments.)]\n",
            "3218: [(269, 283, Antibiotic-Resistant Bacteria, Only five isolates had MAR less than 0.2, being one ocular, one oral, and three nasal, they had multiple resistance index varied between 0.07 and 0.92, with an average of 0.45 and a mode of 0.3. Among the samples with the highest resistance index, a positive coagulase Staphylococcus stood out, being intermediate to gentamicin and resistant to other antibiotics and an intermediate streptococcus to gentamicin, enrofloxacin, and resistant to other antibiotics. No sample was positive to mecA gene.)]\n",
            "3219: [(383, 396, Antibiotic-Resistant Bacteria, Only five isolates had MAR less than 0.2, being one ocular, one oral, and three nasal, they had multiple resistance index varied between 0.07 and 0.92, with an average of 0.45 and a mode of 0.3. Among the samples with the highest resistance index, a positive coagulase Staphylococcus stood out, being intermediate to gentamicin and resistant to other antibiotics and an intermediate streptococcus to gentamicin, enrofloxacin, and resistant to other antibiotics. No sample was positive to mecA gene.)]\n",
            "3220: [(488, 492, Gene/Proteins, Only five isolates had MAR less than 0.2, being one ocular, one oral, and three nasal, they had multiple resistance index varied between 0.07 and 0.92, with an average of 0.45 and a mode of 0.3. Among the samples with the highest resistance index, a positive coagulase Staphylococcus stood out, being intermediate to gentamicin and resistant to other antibiotics and an intermediate streptococcus to gentamicin, enrofloxacin, and resistant to other antibiotics. No sample was positive to mecA gene.)]\n",
            "3221: [(412, 424, Antibiotics, Only five isolates had MAR less than 0.2, being one ocular, one oral, and three nasal, they had multiple resistance index varied between 0.07 and 0.92, with an average of 0.45 and a mode of 0.3. Among the samples with the highest resistance index, a positive coagulase Staphylococcus stood out, being intermediate to gentamicin and resistant to other antibiotics and an intermediate streptococcus to gentamicin, enrofloxacin, and resistant to other antibiotics. No sample was positive to mecA gene.)]\n",
            "3222: [(400, 410, Antibiotics, Only five isolates had MAR less than 0.2, being one ocular, one oral, and three nasal, they had multiple resistance index varied between 0.07 and 0.92, with an average of 0.45 and a mode of 0.3. Among the samples with the highest resistance index, a positive coagulase Staphylococcus stood out, being intermediate to gentamicin and resistant to other antibiotics and an intermediate streptococcus to gentamicin, enrofloxacin, and resistant to other antibiotics. No sample was positive to mecA gene.)]\n",
            "3223: [(48, 57, Others, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder. Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "3224: [(59, 63, Others, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder. Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "3225: [(65, 70, Others, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder. Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "3226: [(72, 79, Others, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder. Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "3227: [(81, 91, Others, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder. Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "3228: [(93, 98, Others, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder. Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "3229: [(100, 114, Others, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder. Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "3230: [(116, 125, Others, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder. Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "3231: [(131, 140, Others, These include, among others but not limited to, ARG-ANNOT, CARD, SRST2, MEGARes, Genefinder, ARIBA, KmerResistance, AMRFinder, and ResFinder. Bioinformatics resources differ for several parameters including type of accepted input data, presence/absence of software for search within a database of AMR determinants that can be specific to a tool or cloned from other resources, and for the search approach employed, which can be based on mapping or on alignment.)]\n",
            "3232: [(154, 167, Antibiotic-Resistant Genes, It was also observed that the number of isolates decreased from 2016 to 2020. This study establishes a baseline for the ongoing monitoring of&amp;nbsp;L. monocytogenes for AMR genes.)]\n",
            "3233: [(130, 168, Others, This paper describes vaccine roles in combatting antimicrobial resistance, quantifies the overall advantages of vaccination as an anti-antimicrobial resistance approach, analyzes existing antimicrobial vaccines and those currently under development, and emphasizes some of the obstacles and prospects of vaccine research and development.)]\n",
            "3234: [(12, 21, Microbes, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3235: [(72, 81, Microbes, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3236: [(48, 59, Antibiotics, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3237: [(209, 220, Antibiotics, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3238: [(376, 380, Gene/Proteins, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3239: [(382, 386, Gene/Proteins, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3240: [(388, 392, Gene/Proteins, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3241: [(398, 402, Gene/Proteins, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3242: [(406, 421, Antibiotics, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3243: [(246, 260, Microbes, Two-hundred S. aureus isolates (43 MRSA and 157 methicillin-susceptible S. aureus [MSSA] isolates) were analyzed to determine 1) the genotypes of the isolates (multilocus sequence, agr, and spa types), 2) the methicillin resistance phenotype and staphylococcal cassette chromosome mec (SCCmec) types, 3) the antimicrobial resistance profiles, and 4) the mutational changes in gyrA, gyrB, parC, and parE in fluoroquinolone-resistant isolates.)]\n",
            "3244: [(76, 91, Gene/Proteins, Bacteria resist antimicrobials biochemically by inactivating the drugs with beta-lactamases, acetylases, adenylases, and phosphorylases; reducing drug access sites of action by virtue of membrane characteristics; altering the drug target so that the antimicrobial no longer binds to it; bypassing the drug's metabolism; and developing tolerance.)]\n",
            "3245: [(93, 103, Gene/Proteins, Bacteria resist antimicrobials biochemically by inactivating the drugs with beta-lactamases, acetylases, adenylases, and phosphorylases; reducing drug access sites of action by virtue of membrane characteristics; altering the drug target so that the antimicrobial no longer binds to it; bypassing the drug's metabolism; and developing tolerance.)]\n",
            "3246: [(105, 115, Gene/Proteins, Bacteria resist antimicrobials biochemically by inactivating the drugs with beta-lactamases, acetylases, adenylases, and phosphorylases; reducing drug access sites of action by virtue of membrane characteristics; altering the drug target so that the antimicrobial no longer binds to it; bypassing the drug's metabolism; and developing tolerance.)]\n",
            "3247: [(121, 135, Gene/Proteins, Bacteria resist antimicrobials biochemically by inactivating the drugs with beta-lactamases, acetylases, adenylases, and phosphorylases; reducing drug access sites of action by virtue of membrane characteristics; altering the drug target so that the antimicrobial no longer binds to it; bypassing the drug's metabolism; and developing tolerance.)]\n",
            "3248: [(, , , Given the global nature of gonorrhea, the high rate of usage of antimicrobials, suboptimal control and monitoring of antimicrobial resistance (AMR) and treatment failures, slow update of treatment guidelines in most geographical settings, and the extraordinary capacity of the gonococci to develop and retain AMR, it is likely that the global problem of gonococcal AMR will worsen in the foreseeable future and that the severe complications of gonorrhea will emerge as a silent epidemic.)]\n",
            "3249: [(86, 98, Microbes, Importantly, high in vivo treatment efficacy of the macromolecules is achieved in MDR A. baumannii-, E. coli-, K. pneumoniae-, methicillin-resistant S. aureus-, cecal ligation and puncture-induced polymicrobial peritonitis, and P. aeruginosa lung infection mouse models while remaining non-toxic (e.g., therapeutic index-ED50/LD50: 1473 for A. baumannii infection).)]\n",
            "3250: [(101, 108, Microbes, Importantly, high in vivo treatment efficacy of the macromolecules is achieved in MDR A. baumannii-, E. coli-, K. pneumoniae-, methicillin-resistant S. aureus-, cecal ligation and puncture-induced polymicrobial peritonitis, and P. aeruginosa lung infection mouse models while remaining non-toxic (e.g., therapeutic index-ED50/LD50: 1473 for A. baumannii infection).)]\n",
            "3251: [(111, 124, Microbes, Importantly, high in vivo treatment efficacy of the macromolecules is achieved in MDR A. baumannii-, E. coli-, K. pneumoniae-, methicillin-resistant S. aureus-, cecal ligation and puncture-induced polymicrobial peritonitis, and P. aeruginosa lung infection mouse models while remaining non-toxic (e.g., therapeutic index-ED50/LD50: 1473 for A. baumannii infection).)]\n",
            "3252: [(127, 138, Antibiotics, Importantly, high in vivo treatment efficacy of the macromolecules is achieved in MDR A. baumannii-, E. coli-, K. pneumoniae-, methicillin-resistant S. aureus-, cecal ligation and puncture-induced polymicrobial peritonitis, and P. aeruginosa lung infection mouse models while remaining non-toxic (e.g., therapeutic index-ED50/LD50: 1473 for A. baumannii infection).)]\n",
            "3253: [(149, 158, Microbes, Importantly, high in vivo treatment efficacy of the macromolecules is achieved in MDR A. baumannii-, E. coli-, K. pneumoniae-, methicillin-resistant S. aureus-, cecal ligation and puncture-induced polymicrobial peritonitis, and P. aeruginosa lung infection mouse models while remaining non-toxic (e.g., therapeutic index-ED50/LD50: 1473 for A. baumannii infection).)]\n",
            "3254: [(228, 241, Microbes, Importantly, high in vivo treatment efficacy of the macromolecules is achieved in MDR A. baumannii-, E. coli-, K. pneumoniae-, methicillin-resistant S. aureus-, cecal ligation and puncture-induced polymicrobial peritonitis, and P. aeruginosa lung infection mouse models while remaining non-toxic (e.g., therapeutic index-ED50/LD50: 1473 for A. baumannii infection).)]\n",
            "3255: [(161, 175, Techniques, Importantly, high in vivo treatment efficacy of the macromolecules is achieved in MDR A. baumannii-, E. coli-, K. pneumoniae-, methicillin-resistant S. aureus-, cecal ligation and puncture-induced polymicrobial peritonitis, and P. aeruginosa lung infection mouse models while remaining non-toxic (e.g., therapeutic index-ED50/LD50: 1473 for A. baumannii infection).)]\n",
            "3256: [(341, 353, Microbes, Importantly, high in vivo treatment efficacy of the macromolecules is achieved in MDR A. baumannii-, E. coli-, K. pneumoniae-, methicillin-resistant S. aureus-, cecal ligation and puncture-induced polymicrobial peritonitis, and P. aeruginosa lung infection mouse models while remaining non-toxic (e.g., therapeutic index-ED50/LD50: 1473 for A. baumannii infection).)]\n",
            "3257: [(389, 401, Antibiotics, The objective of this study was to determine if transferable antimicrobial resistance (AMR) genes are present in commercial animal probiotics. DNA was extracted from 50 probiotics, tested for the presence of bacterial DNA, and analyzed by polymerase chain reaction (PCR) for the presence of 8 transferrable AMR genes, including tetracycline, erythromycin, aminoglycoside, sulfonamide, and trimethoprim. Samples that were positive by PCR were confirmed by genome sequencing.)]\n",
            "3258: [(191, 198, Microbes, Therefore, the current systematic review, following the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines, was conducted to find out the AMR scenarios in E. coli isolates sourced from poultry and poultry environments in Bangladesh between 2010 and 2021.)]\n",
            "3259: [(174, 177, Others, Therefore, the current systematic review, following the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines, was conducted to find out the AMR scenarios in E. coli isolates sourced from poultry and poultry environments in Bangladesh between 2010 and 2021.)]\n",
            "3260: [(37, 75, Antibiotics, Indeed, although the introduction of pneumococcal conjugate vaccines (PCVs) has been associated with an overall decline in pneumococcal AMR, there have been increases in prevalence of potentially disease-causing AMR serotypes not targeted by vaccination.)]\n",
            "3261: [(177, 186, Antibiotics, Nutritional supplements were associated to the eutrophic group. ARG screening showed differences being more frequent among obese, and positive for 27 genetic markers related to β-lactams, tetracyclines, the macrolide lincosamide and streptogramin group, quinolones, sulfonamides, aminoglycosides, and efflux pump.)]\n",
            "3262: [(188, 201, Antibiotics, Nutritional supplements were associated to the eutrophic group. ARG screening showed differences being more frequent among obese, and positive for 27 genetic markers related to β-lactams, tetracyclines, the macrolide lincosamide and streptogramin group, quinolones, sulfonamides, aminoglycosides, and efflux pump.)]\n",
            "3263: [(206, 228, Antibiotics, Nutritional supplements were associated to the eutrophic group. ARG screening showed differences being more frequent among obese, and positive for 27 genetic markers related to β-lactams, tetracyclines, the macrolide lincosamide and streptogramin group, quinolones, sulfonamides, aminoglycosides, and efflux pump.)]\n",
            "3264: [(233, 246, Antibiotics, Nutritional supplements were associated to the eutrophic group. ARG screening showed differences being more frequent among obese, and positive for 27 genetic markers related to β-lactams, tetracyclines, the macrolide lincosamide and streptogramin group, quinolones, sulfonamides, aminoglycosides, and efflux pump.)]\n",
            "3265: [(254, 264, Antibiotics, Nutritional supplements were associated to the eutrophic group. ARG screening showed differences being more frequent among obese, and positive for 27 genetic markers related to β-lactams, tetracyclines, the macrolide lincosamide and streptogramin group, quinolones, sulfonamides, aminoglycosides, and efflux pump.)]\n",
            "3266: [(266, 278, Antibiotics, Nutritional supplements were associated to the eutrophic group. ARG screening showed differences being more frequent among obese, and positive for 27 genetic markers related to β-lactams, tetracyclines, the macrolide lincosamide and streptogramin group, quinolones, sulfonamides, aminoglycosides, and efflux pump.)]\n",
            "3267: [(280, 295, Antibiotics, Nutritional supplements were associated to the eutrophic group. ARG screening showed differences being more frequent among obese, and positive for 27 genetic markers related to β-lactams, tetracyclines, the macrolide lincosamide and streptogramin group, quinolones, sulfonamides, aminoglycosides, and efflux pump.)]\n",
            "3268: [(301, 313, Others, Nutritional supplements were associated to the eutrophic group. ARG screening showed differences being more frequent among obese, and positive for 27 genetic markers related to β-lactams, tetracyclines, the macrolide lincosamide and streptogramin group, quinolones, sulfonamides, aminoglycosides, and efflux pump.)]\n",
            "3269: [(82, 118, Antibiotics, Newer, often novel, agents seem necessary to combat these pathogens. Among these, quinupristin/dalfopristin (Synercid), evernimicin or SCH 27899 (Ziracin), and linezolid (Zyvax) have the highest potency, widest spectrum, and most clinical experience.)]\n",
            "3270: [(121, 131, Antibiotics, Newer, often novel, agents seem necessary to combat these pathogens. Among these, quinupristin/dalfopristin (Synercid), evernimicin or SCH 27899 (Ziracin), and linezolid (Zyvax) have the highest potency, widest spectrum, and most clinical experience.)]\n",
            "3271: [(135, 154, Antibiotics, Newer, often novel, agents seem necessary to combat these pathogens. Among these, quinupristin/dalfopristin (Synercid), evernimicin or SCH 27899 (Ziracin), and linezolid (Zyvax) have the highest potency, widest spectrum, and most clinical experience.)]\n",
            "3272: [(160, 177, Antibiotics, Newer, often novel, agents seem necessary to combat these pathogens. Among these, quinupristin/dalfopristin (Synercid), evernimicin or SCH 27899 (Ziracin), and linezolid (Zyvax) have the highest potency, widest spectrum, and most clinical experience.)]\n",
            "3273: [(150, 161, Antibiotics, Although the present work is a preliminary research, the results encourages the development of further studies to determinate the effect of prolonged doxycycline therapy on antimicrobial resistance.)]\n",
            "3274: [(76, 92, Gene/Proteins, Contamination occurs despite the use of antimicrobial interventions such as lactic acid (LA). In addition, resistance to antibiotics such as ampicillin and streptomycin among isolates has been frequently reported.)]\n",
            "3275: [(141, 151, Antibiotics, Contamination occurs despite the use of antimicrobial interventions such as lactic acid (LA). In addition, resistance to antibiotics such as ampicillin and streptomycin among isolates has been frequently reported.)]\n",
            "3276: [(156, 168, Antibiotics, Contamination occurs despite the use of antimicrobial interventions such as lactic acid (LA). In addition, resistance to antibiotics such as ampicillin and streptomycin among isolates has been frequently reported.)]\n",
            "3277: [(43, 53, Antibiotics, Our findings showed that ARGs resistant to macrolides, sulfonamides, and tetracyclines were positively correlated with time in Hospital A but not in Hospital B. Likewise, minor extended spectrum beta-lactamases (ESBLs) and carbapenemases of classes B and D were positively correlated with time, suggesting the selection of rare and/or carbapenem-resistant genes in Hospital A. Non-carbapenemase blaVEB also positively correlated with both time and intI1 and was copresent with other ARGs including carbapenem-resistant genes in 6 metagenome-assembled genomes (MAGs).)]\n",
            "3278: [(73, 86, Antibiotics, Our findings showed that ARGs resistant to macrolides, sulfonamides, and tetracyclines were positively correlated with time in Hospital A but not in Hospital B. Likewise, minor extended spectrum beta-lactamases (ESBLs) and carbapenemases of classes B and D were positively correlated with time, suggesting the selection of rare and/or carbapenem-resistant genes in Hospital A. Non-carbapenemase blaVEB also positively correlated with both time and intI1 and was copresent with other ARGs including carbapenem-resistant genes in 6 metagenome-assembled genomes (MAGs).)]\n",
            "3279: [(55, 67, Antibiotics, Our findings showed that ARGs resistant to macrolides, sulfonamides, and tetracyclines were positively correlated with time in Hospital A but not in Hospital B. Likewise, minor extended spectrum beta-lactamases (ESBLs) and carbapenemases of classes B and D were positively correlated with time, suggesting the selection of rare and/or carbapenem-resistant genes in Hospital A. Non-carbapenemase blaVEB also positively correlated with both time and intI1 and was copresent with other ARGs including carbapenem-resistant genes in 6 metagenome-assembled genomes (MAGs).)]\n",
            "3280: [(335, 345, Antibiotics, Our findings showed that ARGs resistant to macrolides, sulfonamides, and tetracyclines were positively correlated with time in Hospital A but not in Hospital B. Likewise, minor extended spectrum beta-lactamases (ESBLs) and carbapenemases of classes B and D were positively correlated with time, suggesting the selection of rare and/or carbapenem-resistant genes in Hospital A. Non-carbapenemase blaVEB also positively correlated with both time and intI1 and was copresent with other ARGs including carbapenem-resistant genes in 6 metagenome-assembled genomes (MAGs).)]\n",
            "3281: [(498, 508, Antibiotics, Our findings showed that ARGs resistant to macrolides, sulfonamides, and tetracyclines were positively correlated with time in Hospital A but not in Hospital B. Likewise, minor extended spectrum beta-lactamases (ESBLs) and carbapenemases of classes B and D were positively correlated with time, suggesting the selection of rare and/or carbapenem-resistant genes in Hospital A. Non-carbapenemase blaVEB also positively correlated with both time and intI1 and was copresent with other ARGs including carbapenem-resistant genes in 6 metagenome-assembled genomes (MAGs).)]\n",
            "3282: [(64, 80, Antibiotic-Resistant Bacteria, We first evaluated the fate of ingested antimicrobial-resistant Escherichia coli harboring a plasmid containing antimicrobial-resistance genes (ARGs) throughout the housefly (Musca domestica) life cycle, from adult to the subsequent F1 generation.)]\n",
            "3283: [(32, 45, Antibiotics, In North America, resistance to ciprofloxacin also increased between 2008 and 2017, from 4% to 12%. Data exploring different age groups did not show a consistent relationship with resistance, whereas two studies found that fluoroquinolone resistance was higher in postmenopausal women than premenopausal women.)]\n",
            "3284: [(223, 238, Antibiotics, In North America, resistance to ciprofloxacin also increased between 2008 and 2017, from 4% to 12%. Data exploring different age groups did not show a consistent relationship with resistance, whereas two studies found that fluoroquinolone resistance was higher in postmenopausal women than premenopausal women.)]\n",
            "3285: [(77, 100, Others, A similar trend was found for biocide and metal resistance genes (BMRGs) and mobile genetic elements (MGEs).)]\n",
            "3286: [(42, 72, Antibiotic-Resistant Genes, A similar trend was found for biocide and metal resistance genes (BMRGs) and mobile genetic elements (MGEs).)]\n",
            "3287: [(44, 57, Microbes, Among the positive cases, the prevalence of Campylobacter was higher in the age group 0-5 years (52%) followed by 6-18 years (42.7%), 19-40 years (34.0%), 41-60 years (25.4%), and >60 years (10.5%).)]\n",
            "3288: [(328, 339, Microbes, It is hoped that the application of Codes of Practice for the use of such agents, which have been prepared by the pharmaceutical industry in response to widespread international concern about the development of drug resistance in bacterial pathogens, will now result in a widespread reduction in the incidence of drug-resistant salmonellas in food production animals and humans on an international scale.)]\n",
            "3289: [(208, 219, Antibiotics, The AMR to almost all the antimicrobials evaluated presented an increasing pattern over time, more apparent from 2009 onwards. The antimicrobials with a higher increase in their AMR prevalence over time were clindamycin, gentamycin, and oxacillin. Africa, Asia and Latin America were the continents with higher AMR to most compounds included in this study.)]\n",
            "3290: [(221, 231, Antibiotics, The AMR to almost all the antimicrobials evaluated presented an increasing pattern over time, more apparent from 2009 onwards. The antimicrobials with a higher increase in their AMR prevalence over time were clindamycin, gentamycin, and oxacillin. Africa, Asia and Latin America were the continents with higher AMR to most compounds included in this study.)]\n",
            "3291: [(237, 246, Antibiotics, The AMR to almost all the antimicrobials evaluated presented an increasing pattern over time, more apparent from 2009 onwards. The antimicrobials with a higher increase in their AMR prevalence over time were clindamycin, gentamycin, and oxacillin. Africa, Asia and Latin America were the continents with higher AMR to most compounds included in this study.)]\n",
            "3292: [(, , , For all sectors, reducing the occurrence of faecal microbial contamination of fertilisers, water, feed and the production environment and minimising persistence/recycling of ARB within animal production facilities is a priority.)]\n",
            "3293: [(18, 37, Antibiotic-Resistant Bacteria, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3294: [(42, 49, Antibiotic-Resistant Bacteria, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3295: [(54, 62, Antibiotic-Resistant Bacteria, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3296: [(145, 149, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3297: [(151, 155, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3298: [(157, 160, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3299: [(162, 166, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3300: [(168, 173, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3301: [(175, 182, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3302: [(184, 189, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3303: [(191, 196, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3304: [(198, 208, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3305: [(210, 220, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3306: [(222, 232, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3307: [(234, 243, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3308: [(245, 253, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3309: [(255, 265, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3310: [(267, 271, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3311: [(273, 277, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3312: [(279, 285, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3313: [(291, 297, Antibiotic-Resistant Genes, It was found that Salmonella enterica and E. coli and Shigella were the highly occurring AMR pathogens for both settings. Additionally the genes fosA, oqxB, ble, floR, fosA7, mcr-9.1, aadA1, aadA2, ant(2\")-Ia, aph(3\")-Ib, aph(3')-Ia, aph(6)-Id, blaTEM-1, qacEdelta1, sul1, sul2, tet(A), and tet(B) were mostly detected for both clinical and environmental settings.)]\n",
            "3314: [(243, 253, Antibiotics, We screened 17939 assembled metagenomic samples covering 21 biomes, differing in sequencing quality and depth, unevenly across 46 countries, 6 continents, and 14 years (2005-2019) for clinically crucial ARGs, mobile colistin resistance (mcr), carbapenem resistance (CR), and (extended-spectrum) beta-lactamase (ESBL and BL) genes.)]\n",
            "3315: [(295, 329, Gene/Proteins, We screened 17939 assembled metagenomic samples covering 21 biomes, differing in sequencing quality and depth, unevenly across 46 countries, 6 continents, and 14 years (2005-2019) for clinically crucial ARGs, mobile colistin resistance (mcr), carbapenem resistance (CR), and (extended-spectrum) beta-lactamase (ESBL and BL) genes.)]\n",
            "3316: [(13, 43, Techniques, In addition, horizontal gene transfer (HGT) plays an important role in the spread of resistance genes, for example mobilizing reservoirs of AMR from commensal bacteria into pathogenic ones.)]\n",
            "3317: [(11, 21, Antibiotic-Resistant Bacteria, We obtained G. anatis isolates from bronchoalveolar lavage samples of 10 calves with bronchopneumonia unresponsive to antimicrobial therapy.)]\n",
            "3318: [(118, 140, Techniques, We obtained G. anatis isolates from bronchoalveolar lavage samples of 10 calves with bronchopneumonia unresponsive to antimicrobial therapy.)]\n",
            "3319: [(, , , We emphasize the importance of distinguishing between AMR arising from genetic alterations versus induction of endogenous machinery in response to environmental triggers, including - paradoxically - stresses from host immunity and antimicrobial therapy.)]\n",
            "3320: [(79, 82, Antibiotic-Resistant Genes, Various unique aspects of urban settings that promote spread and resilience of ARGs/ARB are discussed: (i) the role of hospitals and recreational parks as reservoirs; (ii) private and public transportation as carriers of ARGs/ARB; (iii) the role of built environments as a hub for horizontal gene transfer even though they support lower microbial biodiversity than outdoor environments; (iv) the need to employ ecological and evolutionary concepts, such as modeling the fate of a specific ARG/ARB, to gain enhanced health risk assessments.)]\n",
            "3321: [(84, 87, Antibiotic-Resistant Bacteria, Various unique aspects of urban settings that promote spread and resilience of ARGs/ARB are discussed: (i) the role of hospitals and recreational parks as reservoirs; (ii) private and public transportation as carriers of ARGs/ARB; (iii) the role of built environments as a hub for horizontal gene transfer even though they support lower microbial biodiversity than outdoor environments; (iv) the need to employ ecological and evolutionary concepts, such as modeling the fate of a specific ARG/ARB, to gain enhanced health risk assessments.)]\n",
            "3322: [(221, 224, Antibiotic-Resistant Genes, Various unique aspects of urban settings that promote spread and resilience of ARGs/ARB are discussed: (i) the role of hospitals and recreational parks as reservoirs; (ii) private and public transportation as carriers of ARGs/ARB; (iii) the role of built environments as a hub for horizontal gene transfer even though they support lower microbial biodiversity than outdoor environments; (iv) the need to employ ecological and evolutionary concepts, such as modeling the fate of a specific ARG/ARB, to gain enhanced health risk assessments.)]\n",
            "3323: [(226, 229, Antibiotic-Resistant Bacteria, Various unique aspects of urban settings that promote spread and resilience of ARGs/ARB are discussed: (i) the role of hospitals and recreational parks as reservoirs; (ii) private and public transportation as carriers of ARGs/ARB; (iii) the role of built environments as a hub for horizontal gene transfer even though they support lower microbial biodiversity than outdoor environments; (iv) the need to employ ecological and evolutionary concepts, such as modeling the fate of a specific ARG/ARB, to gain enhanced health risk assessments.)]\n",
            "3324: [(489, 492, Antibiotic-Resistant Genes, Various unique aspects of urban settings that promote spread and resilience of ARGs/ARB are discussed: (i) the role of hospitals and recreational parks as reservoirs; (ii) private and public transportation as carriers of ARGs/ARB; (iii) the role of built environments as a hub for horizontal gene transfer even though they support lower microbial biodiversity than outdoor environments; (iv) the need to employ ecological and evolutionary concepts, such as modeling the fate of a specific ARG/ARB, to gain enhanced health risk assessments.)]\n",
            "3325: [(493, 496, Antibiotic-Resistant Bacteria, Various unique aspects of urban settings that promote spread and resilience of ARGs/ARB are discussed: (i) the role of hospitals and recreational parks as reservoirs; (ii) private and public transportation as carriers of ARGs/ARB; (iii) the role of built environments as a hub for horizontal gene transfer even though they support lower microbial biodiversity than outdoor environments; (iv) the need to employ ecological and evolutionary concepts, such as modeling the fate of a specific ARG/ARB, to gain enhanced health risk assessments.)]\n",
            "3326: [(225, 247, Techniques, Wastewater treatment is a double-edged sword that can act as either a pathway for AMR spread or as a barrier to reduce the environmental release of anthropogenic AMR. State of the art ARB identification technologies, such as metagenomic sequencing and fluorescence-activated cell sorting, have enriched ARG/ARB databases, unveiled keystone species in AMR networks, and improved the resolution of AMR dissemination models.)]\n",
            "3327: [(252, 287, Techniques, Wastewater treatment is a double-edged sword that can act as either a pathway for AMR spread or as a barrier to reduce the environmental release of anthropogenic AMR. State of the art ARB identification technologies, such as metagenomic sequencing and fluorescence-activated cell sorting, have enriched ARG/ARB databases, unveiled keystone species in AMR networks, and improved the resolution of AMR dissemination models.)]\n",
            "3328: [(313, 334, Techniques, Bacteroides fragilis constitutes a significant part of the normal human gut microbiota and can also act as an opportunistic pathogen. Antimicrobial resistance (AMR) and the prevalence of AMR genes are increasing, and prediction of antimicrobial susceptibility based on sequence information could support targeted antimicrobial therapy in a clinical setting. Complete identification of insertion sequence (IS) elements carrying promoter sequences upstream of resistance genes is necessary for prediction of AMR.)]\n",
            "3329: [(0, 20, Microbes, Bacteroides fragilis constitutes a significant part of the normal human gut microbiota and can also act as an opportunistic pathogen. Antimicrobial resistance (AMR) and the prevalence of AMR genes are increasing, and prediction of antimicrobial susceptibility based on sequence information could support targeted antimicrobial therapy in a clinical setting. Complete identification of insertion sequence (IS) elements carrying promoter sequences upstream of resistance genes is necessary for prediction of AMR.)]\n",
            "3330: [(113, 117, Gene/Proteins, The mechanism of resistance was examined by sequencing the quinolone resistance determining region (QRDR) of the gyrA (a quinolone target enzyme) gene and by comparing the active efflux mechanisms of two strains isolated from this patient (before hospitalization and after tosufloxacin treatment) and one strain isolated from a patient with secondary infection with that of the standard strain ATCC 29903.)]\n",
            "3331: [(59, 68, Antibiotics, The mechanism of resistance was examined by sequencing the quinolone resistance determining region (QRDR) of the gyrA (a quinolone target enzyme) gene and by comparing the active efflux mechanisms of two strains isolated from this patient (before hospitalization and after tosufloxacin treatment) and one strain isolated from a patient with secondary infection with that of the standard strain ATCC 29903.)]\n",
            "3332: [(172, 195, Techniques, The mechanism of resistance was examined by sequencing the quinolone resistance determining region (QRDR) of the gyrA (a quinolone target enzyme) gene and by comparing the active efflux mechanisms of two strains isolated from this patient (before hospitalization and after tosufloxacin treatment) and one strain isolated from a patient with secondary infection with that of the standard strain ATCC 29903.)]\n",
            "3333: [(273, 285, Antibiotics, The mechanism of resistance was examined by sequencing the quinolone resistance determining region (QRDR) of the gyrA (a quinolone target enzyme) gene and by comparing the active efflux mechanisms of two strains isolated from this patient (before hospitalization and after tosufloxacin treatment) and one strain isolated from a patient with secondary infection with that of the standard strain ATCC 29903.)]\n",
            "3334: [(43, 54, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3335: [(60, 70, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3336: [(78, 90, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3337: [(98, 109, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3338: [(117, 126, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3339: [(134, 142, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3340: [(150, 162, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3341: [(170, 180, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3342: [(188, 202, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3343: [(210, 221, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3344: [(232, 244, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3345: [(310, 321, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3346: [(326, 336, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3347: [(338, 346, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3348: [(348, 358, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3349: [(360, 374, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3350: [(376, 387, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3351: [(392, 404, Antibiotics, The overall AMR of the 10-year period was: amoxicillin 78%, ampicillin 73.5%, tetracycline 63.3%, doxycycline 56.4%, apramycin 34.5%, neomycin 68.2%, enrofloxacin 32.6%, flumequine 39.4%, co-trimoxazole 47.7%, florfenicol 46.6% and lincospectin 66.3%. Over time, a significant decrease in AMR was observed for amoxicillin and ampicillin, neomycin, flumequine, co-trimoxazole, florfenicol and lincospectin. Multidrug resistance (MDR) decreased from 100% in 2009 to 48% in 2018.)]\n",
            "3352: [(232, 239, Antibiotic-Resistant Bacteria, Furthermore, we examine the evolution of AMR in E. coli genomes including an association with virulence genes, plasmids and MGEs. We compared the abundance metrics retrieved from metagenomic sequencing and whole genome sequenced of E. coli isolates from the same fecal samples and farms.)]\n",
            "3353: [(48, 55, Antibiotic-Resistant Bacteria, Furthermore, we examine the evolution of AMR in E. coli genomes including an association with virulence genes, plasmids and MGEs. We compared the abundance metrics retrieved from metagenomic sequencing and whole genome sequenced of E. coli isolates from the same fecal samples and farms.)]\n",
            "3354: [(37, 68, Antibiotics, Indeed, although the introduction of pneumococcal conjugate vaccines (PCVs) has been associated with an overall decline in pneumococcal AMR, there have been increases in prevalence of potentially disease-causing AMR serotypes not targeted by vaccination.)]\n",
            "3355: [(95, 102, Microbes, Recent progress in comparative genomic studies suggests that the genomes of the drug-resistant Vibrios harbor mobile genetic elements like plasmids, integrating conjugative elements, superintegron, transposable elements, and insertion sequences, which are the major carriers of genetic determinants encoding antimicrobial resistance.)]\n",
            "3356: [(139, 147, Others, Recent progress in comparative genomic studies suggests that the genomes of the drug-resistant Vibrios harbor mobile genetic elements like plasmids, integrating conjugative elements, superintegron, transposable elements, and insertion sequences, which are the major carriers of genetic determinants encoding antimicrobial resistance.)]\n",
            "3357: [(149, 181, Others, Recent progress in comparative genomic studies suggests that the genomes of the drug-resistant Vibrios harbor mobile genetic elements like plasmids, integrating conjugative elements, superintegron, transposable elements, and insertion sequences, which are the major carriers of genetic determinants encoding antimicrobial resistance.)]\n",
            "3358: [(183, 196, Others, Recent progress in comparative genomic studies suggests that the genomes of the drug-resistant Vibrios harbor mobile genetic elements like plasmids, integrating conjugative elements, superintegron, transposable elements, and insertion sequences, which are the major carriers of genetic determinants encoding antimicrobial resistance.)]\n",
            "3359: [(198, 219, Others, Recent progress in comparative genomic studies suggests that the genomes of the drug-resistant Vibrios harbor mobile genetic elements like plasmids, integrating conjugative elements, superintegron, transposable elements, and insertion sequences, which are the major carriers of genetic determinants encoding antimicrobial resistance.)]\n",
            "3360: [(225, 244, Others, Recent progress in comparative genomic studies suggests that the genomes of the drug-resistant Vibrios harbor mobile genetic elements like plasmids, integrating conjugative elements, superintegron, transposable elements, and insertion sequences, which are the major carriers of genetic determinants encoding antimicrobial resistance.)]\n",
            "3361: [(76, 87, Others, This study yielded the following findings: (1) a gene for extended-spectrum β-lactamase was transferred from a child to other children and a teacher; (2) a nalidixic acid-resistant isolate was transferred from a teacher to a child; and (3) a virulent bacterium was transferred between children.)]\n",
            "3362: [(156, 170, Antibiotics, This study yielded the following findings: (1) a gene for extended-spectrum β-lactamase was transferred from a child to other children and a teacher; (2) a nalidixic acid-resistant isolate was transferred from a teacher to a child; and (3) a virulent bacterium was transferred between children.)]\n",
            "3363: [(312, 333, Techniques, Although the \"pioneering strategy\" of discovering completely new drugs is a rational approach, the time and effort taken are considerable, the process of drug development could instead be expedited if efforts were concentrated on enhancing the efficacy of existing antimicrobials through: combination therapies; bacteriophage therapy; antimicrobial adjuvants therapy or the application of nanotechnology.)]\n",
            "3364: [(289, 310, Techniques, Although the \"pioneering strategy\" of discovering completely new drugs is a rational approach, the time and effort taken are considerable, the process of drug development could instead be expedited if efforts were concentrated on enhancing the efficacy of existing antimicrobials through: combination therapies; bacteriophage therapy; antimicrobial adjuvants therapy or the application of nanotechnology.)]\n",
            "3365: [(335, 366, Techniques, Although the \"pioneering strategy\" of discovering completely new drugs is a rational approach, the time and effort taken are considerable, the process of drug development could instead be expedited if efforts were concentrated on enhancing the efficacy of existing antimicrobials through: combination therapies; bacteriophage therapy; antimicrobial adjuvants therapy or the application of nanotechnology.)]\n",
            "3366: [(389, 403, Techniques, Although the \"pioneering strategy\" of discovering completely new drugs is a rational approach, the time and effort taken are considerable, the process of drug development could instead be expedited if efforts were concentrated on enhancing the efficacy of existing antimicrobials through: combination therapies; bacteriophage therapy; antimicrobial adjuvants therapy or the application of nanotechnology.)]\n",
            "3367: [(53, 71, Microbes, Our study highlights the ecology and transmission of K. quasipneumoniae (which have the ability to disseminate to other environmental sources including animals) outside the clinical setting and the contribution of water, vegetables, and table surfaces as potential reservoirs of farm-to-fork transmission of disease via local markets in Khartoum, Sudan.)]\n",
            "3368: [(342, 354, Antibiotics, The most common isolated bacterium was coagulase-negative Staphylococcus (89%) and Staphylococcus aureus (28%). Coagulase-negative Staphylococcus showed highest susceptibility to vancomycin (100%), neomycin (94%) and chloramphenicol (91%). Coagulase-negative Staphylococcus and Staphylococcus aureus were the most resistant to penicillin and erythromycin (resistance in 92%, 91% for coagulase-negative Staphylococcus, 86% and 43% of eyes for Staphylococcus aureus).)]\n",
            "3369: [(327, 337, Antibiotics, The most common isolated bacterium was coagulase-negative Staphylococcus (89%) and Staphylococcus aureus (28%). Coagulase-negative Staphylococcus showed highest susceptibility to vancomycin (100%), neomycin (94%) and chloramphenicol (91%). Coagulase-negative Staphylococcus and Staphylococcus aureus were the most resistant to penicillin and erythromycin (resistance in 92%, 91% for coagulase-negative Staphylococcus, 86% and 43% of eyes for Staphylococcus aureus).)]\n",
            "3370: [(278, 299, Antibiotic-Resistant Bacteria, The most common isolated bacterium was coagulase-negative Staphylococcus (89%) and Staphylococcus aureus (28%). Coagulase-negative Staphylococcus showed highest susceptibility to vancomycin (100%), neomycin (94%) and chloramphenicol (91%). Coagulase-negative Staphylococcus and Staphylococcus aureus were the most resistant to penicillin and erythromycin (resistance in 92%, 91% for coagulase-negative Staphylococcus, 86% and 43% of eyes for Staphylococcus aureus).)]\n",
            "3371: [(58, 72, Microbes, The most common isolated bacterium was coagulase-negative Staphylococcus (89%) and Staphylococcus aureus (28%). Coagulase-negative Staphylococcus showed highest susceptibility to vancomycin (100%), neomycin (94%) and chloramphenicol (91%). Coagulase-negative Staphylococcus and Staphylococcus aureus were the most resistant to penicillin and erythromycin (resistance in 92%, 91% for coagulase-negative Staphylococcus, 86% and 43% of eyes for Staphylococcus aureus).)]\n",
            "3372: [(179, 189, Antibiotics, The most common isolated bacterium was coagulase-negative Staphylococcus (89%) and Staphylococcus aureus (28%). Coagulase-negative Staphylococcus showed highest susceptibility to vancomycin (100%), neomycin (94%) and chloramphenicol (91%). Coagulase-negative Staphylococcus and Staphylococcus aureus were the most resistant to penicillin and erythromycin (resistance in 92%, 91% for coagulase-negative Staphylococcus, 86% and 43% of eyes for Staphylococcus aureus).)]\n",
            "3373: [(198, 206, Antibiotics, The most common isolated bacterium was coagulase-negative Staphylococcus (89%) and Staphylococcus aureus (28%). Coagulase-negative Staphylococcus showed highest susceptibility to vancomycin (100%), neomycin (94%) and chloramphenicol (91%). Coagulase-negative Staphylococcus and Staphylococcus aureus were the most resistant to penicillin and erythromycin (resistance in 92%, 91% for coagulase-negative Staphylococcus, 86% and 43% of eyes for Staphylococcus aureus).)]\n",
            "3374: [(217, 232, Antibiotics, The most common isolated bacterium was coagulase-negative Staphylococcus (89%) and Staphylococcus aureus (28%). Coagulase-negative Staphylococcus showed highest susceptibility to vancomycin (100%), neomycin (94%) and chloramphenicol (91%). Coagulase-negative Staphylococcus and Staphylococcus aureus were the most resistant to penicillin and erythromycin (resistance in 92%, 91% for coagulase-negative Staphylococcus, 86% and 43% of eyes for Staphylococcus aureus).)]\n",
            "3375: [(0, 10, Techniques, Sequencing of bacterial isolates is becoming more common, and therefore automatic identification of resistant bacterial strains is of pivotal importance for efficient, wide-spread AMR detection.)]\n",
            "3376: [(114, 124, Antibiotics, In foods of non-animal origin, the prevalence was lower with 13%. In milk, the highest AMR estimate was found for penicillin (69%) followed by amoxicillin (51%). Regarding multi-drug resistance (MDR), the overall pooled prevalence was 74% among AMR positive samples.)]\n",
            "3377: [(142, 154, Antibiotics, In foods of non-animal origin, the prevalence was lower with 13%. In milk, the highest AMR estimate was found for penicillin (69%) followed by amoxicillin (51%). Regarding multi-drug resistance (MDR), the overall pooled prevalence was 74% among AMR positive samples.)]\n",
            "3378: [(239, 266, Techniques, Although lifestyle and physiology in obese individuals are accepted to lead to changes in the intestinal microbiota, uncertainty remains about microbiota dysbiosis, and xenobiotics intake, as a source of selective pressure, independent of antimicrobial chemotherapy.)]\n",
            "3379: [(143, 163, Others, Although lifestyle and physiology in obese individuals are accepted to lead to changes in the intestinal microbiota, uncertainty remains about microbiota dysbiosis, and xenobiotics intake, as a source of selective pressure, independent of antimicrobial chemotherapy.)]\n",
            "3380: [(38, 62, Others, Rapid spreading of mobile elements by horizontal gene transfer such as plasmids and integrative conjugative elements (ICE) carrying multiple resistance genes has dramatically increased the worldwide prevalence of relevant multi drug-resistant human pathogens such as Staphylococcus aureus, Neisseria gonorrhoeae, and Enterobacteriaceae.)]\n",
            "3381: [(84, 122, Others, Rapid spreading of mobile elements by horizontal gene transfer such as plasmids and integrative conjugative elements (ICE) carrying multiple resistance genes has dramatically increased the worldwide prevalence of relevant multi drug-resistant human pathogens such as Staphylococcus aureus, Neisseria gonorrhoeae, and Enterobacteriaceae.)]\n",
            "3382: [(268, 336, Microbes, Rapid spreading of mobile elements by horizontal gene transfer such as plasmids and integrative conjugative elements (ICE) carrying multiple resistance genes has dramatically increased the worldwide prevalence of relevant multi drug-resistant human pathogens such as Staphylococcus aureus, Neisseria gonorrhoeae, and Enterobacteriaceae.)]\n",
            "3383: [(0, 16, Microbes, Enterococcus spp. (n=391) from water/sediment recovered upstream, within and downstream trout tanks, feed, trout (2 aquacultures; no antibiotic use) and marketed trout (8 supermarkets) showed variable resistance to tetracycline, erythromycin, ciprofloxacin, chloramphenicol, quinupristin-dalfopristin, nitrofurantoin or aminoglycosides.)]\n",
            "3384: [(215, 227, Antibiotics, Enterococcus spp. (n=391) from water/sediment recovered upstream, within and downstream trout tanks, feed, trout (2 aquacultures; no antibiotic use) and marketed trout (8 supermarkets) showed variable resistance to tetracycline, erythromycin, ciprofloxacin, chloramphenicol, quinupristin-dalfopristin, nitrofurantoin or aminoglycosides.)]\n",
            "3385: [(229, 241, Antibiotics, Enterococcus spp. (n=391) from water/sediment recovered upstream, within and downstream trout tanks, feed, trout (2 aquacultures; no antibiotic use) and marketed trout (8 supermarkets) showed variable resistance to tetracycline, erythromycin, ciprofloxacin, chloramphenicol, quinupristin-dalfopristin, nitrofurantoin or aminoglycosides.)]\n",
            "3386: [(243, 256, Antibiotics, Enterococcus spp. (n=391) from water/sediment recovered upstream, within and downstream trout tanks, feed, trout (2 aquacultures; no antibiotic use) and marketed trout (8 supermarkets) showed variable resistance to tetracycline, erythromycin, ciprofloxacin, chloramphenicol, quinupristin-dalfopristin, nitrofurantoin or aminoglycosides.)]\n",
            "3387: [(258, 273, Antibiotics, Enterococcus spp. (n=391) from water/sediment recovered upstream, within and downstream trout tanks, feed, trout (2 aquacultures; no antibiotic use) and marketed trout (8 supermarkets) showed variable resistance to tetracycline, erythromycin, ciprofloxacin, chloramphenicol, quinupristin-dalfopristin, nitrofurantoin or aminoglycosides.)]\n",
            "3388: [(275, 287, Antibiotics, Enterococcus spp. (n=391) from water/sediment recovered upstream, within and downstream trout tanks, feed, trout (2 aquacultures; no antibiotic use) and marketed trout (8 supermarkets) showed variable resistance to tetracycline, erythromycin, ciprofloxacin, chloramphenicol, quinupristin-dalfopristin, nitrofurantoin or aminoglycosides.)]\n",
            "3389: [(288, 300, Antibiotics, Enterococcus spp. (n=391) from water/sediment recovered upstream, within and downstream trout tanks, feed, trout (2 aquacultures; no antibiotic use) and marketed trout (8 supermarkets) showed variable resistance to tetracycline, erythromycin, ciprofloxacin, chloramphenicol, quinupristin-dalfopristin, nitrofurantoin or aminoglycosides.)]\n",
            "3390: [(302, 316, Antibiotics, Enterococcus spp. (n=391) from water/sediment recovered upstream, within and downstream trout tanks, feed, trout (2 aquacultures; no antibiotic use) and marketed trout (8 supermarkets) showed variable resistance to tetracycline, erythromycin, ciprofloxacin, chloramphenicol, quinupristin-dalfopristin, nitrofurantoin or aminoglycosides.)]\n",
            "3391: [(320, 335, Antibiotics, Enterococcus spp. (n=391) from water/sediment recovered upstream, within and downstream trout tanks, feed, trout (2 aquacultures; no antibiotic use) and marketed trout (8 supermarkets) showed variable resistance to tetracycline, erythromycin, ciprofloxacin, chloramphenicol, quinupristin-dalfopristin, nitrofurantoin or aminoglycosides.)]\n",
            "3392: [(, , , However, the clinical and economic impact of these infections and their optimal management are challenging, and consequently, there is an increasing awareness of the importance of ExPECs amongst healthcare professionals and the general public alike.)]\n",
            "3393: [(117, 148, Techniques, To offset the limitation of the long turnaround time of phenotype-based methods, molecular detection methods such as polymerase chain reaction (PCR), isothermal amplification, high-throughput sequencing, gene microarray, and mass spectrometry have begun to be widely used and served as important complements to phenotype-based methods.)]\n",
            "3394: [(150, 174, Techniques, To offset the limitation of the long turnaround time of phenotype-based methods, molecular detection methods such as polymerase chain reaction (PCR), isothermal amplification, high-throughput sequencing, gene microarray, and mass spectrometry have begun to be widely used and served as important complements to phenotype-based methods.)]\n",
            "3395: [(176, 202, Techniques, To offset the limitation of the long turnaround time of phenotype-based methods, molecular detection methods such as polymerase chain reaction (PCR), isothermal amplification, high-throughput sequencing, gene microarray, and mass spectrometry have begun to be widely used and served as important complements to phenotype-based methods.)]\n",
            "3396: [(204, 219, Techniques, To offset the limitation of the long turnaround time of phenotype-based methods, molecular detection methods such as polymerase chain reaction (PCR), isothermal amplification, high-throughput sequencing, gene microarray, and mass spectrometry have begun to be widely used and served as important complements to phenotype-based methods.)]\n",
            "3397: [(225, 242, Techniques, To offset the limitation of the long turnaround time of phenotype-based methods, molecular detection methods such as polymerase chain reaction (PCR), isothermal amplification, high-throughput sequencing, gene microarray, and mass spectrometry have begun to be widely used and served as important complements to phenotype-based methods.)]\n",
            "3398: [(264, 284, Antibiotic-Resistant Genes, We describe the recent application of biophysical approaches such as the hydrodynamic technique of analytical ultracentrifugation and circular dichroism spectroscopy to identify the possible molecular effector of the VanS receptor that activates expression of the Van resistance genes; both approaches demonstrated that vancomycin interacts with VanS, suggesting that vancomycin itself (or vancomycin with an accessory factor) may be an effector of vancomycin resistance.)]\n",
            "3399: [(320, 330, Antibiotics, We describe the recent application of biophysical approaches such as the hydrodynamic technique of analytical ultracentrifugation and circular dichroism spectroscopy to identify the possible molecular effector of the VanS receptor that activates expression of the Van resistance genes; both approaches demonstrated that vancomycin interacts with VanS, suggesting that vancomycin itself (or vancomycin with an accessory factor) may be an effector of vancomycin resistance.)]\n",
            "3400: [(99, 129, Techniques, We describe the recent application of biophysical approaches such as the hydrodynamic technique of analytical ultracentrifugation and circular dichroism spectroscopy to identify the possible molecular effector of the VanS receptor that activates expression of the Van resistance genes; both approaches demonstrated that vancomycin interacts with VanS, suggesting that vancomycin itself (or vancomycin with an accessory factor) may be an effector of vancomycin resistance.)]\n",
            "3401: [(134, 165, Techniques, We describe the recent application of biophysical approaches such as the hydrodynamic technique of analytical ultracentrifugation and circular dichroism spectroscopy to identify the possible molecular effector of the VanS receptor that activates expression of the Van resistance genes; both approaches demonstrated that vancomycin interacts with VanS, suggesting that vancomycin itself (or vancomycin with an accessory factor) may be an effector of vancomycin resistance.)]\n",
            "3402: [(217, 230, Others, We describe the recent application of biophysical approaches such as the hydrodynamic technique of analytical ultracentrifugation and circular dichroism spectroscopy to identify the possible molecular effector of the VanS receptor that activates expression of the Van resistance genes; both approaches demonstrated that vancomycin interacts with VanS, suggesting that vancomycin itself (or vancomycin with an accessory factor) may be an effector of vancomycin resistance.)]\n",
            "3403: [(, , , Antimicrobial resistance (AMR) is a serious global health threat and is predicted to cause significant health and economic impacts, particularly in low- and middle-income countries (LMICs).)]\n",
            "3404: [(, , , This study reviewed global guidance and 25 National Action Plans to evaluate thematic priorities in One Health AMR approaches using a novel framework that additionally facilitated the identification of water-related stewardship gaps, as water resources are recognized as the primary environmental AMR reservoir and dissemination pathway.)]\n",
            "3405: [(, , , Although this property has been extensively studied both alone and when combined with antibiotics against antimicrobial-resistant bacterial and fungal species, relatively little attention has been given to their ability to contribute to the emergence of antimicrobial resistance (AMR).)]\n",
            "3406: [(125, 183, Antibiotic-Resistant Bacteria, The World Health Organization has recently published a global priority list of antibiotic-resistant bacteria, which includes extended-spectrum β-lactamase-producing Enterobacteriaceae and carbapenemase-producing Enterobacteriaceae.)]\n",
            "3407: [(188, 230, Antibiotic-Resistant Bacteria, The World Health Organization has recently published a global priority list of antibiotic-resistant bacteria, which includes extended-spectrum β-lactamase-producing Enterobacteriaceae and carbapenemase-producing Enterobacteriaceae.)]\n",
            "3408: [(, , , All affected buffalo (n=99) and cattle (n=37) presented for treatment in a population of 238 large ruminants, from 15 of 136 households, were administered a topical anesthetic pain-relief product (PRP) wound gel by spray-on (10-30 mL per animal) formulation developed for aversive husbandry procedures (Tri-Solfen, Animal Ethics, Australia).)]\n",
            "3409: [(, , , Although the rapid spread of antimicrobial resistance (AMR), particularly in relation to clinical settings, is causing concern in many regions of the globe, remote, extreme environments, such as Antarctica, are thought to be relatively free from the negative impact of human activities.)]\n",
            "3410: [(, , , Poultry, swine, cattle, and fish were the most frequent livestock mentioned as sources of AMR bacteria. Animal carriage and/or transfer of AMR determinants or bacteria was recognised as the primary contribution of livestock to the problem, while water, soil, and farming were predominant environmental contributions.)]\n",
            "3411: [(, , , Since antibiotics are failing to treat AMR-bacteria, new tools are needed, and human monoclonal antibodies (mAbs) can fill this role. In almost 50 years since the introduction of the first technology that led to mAb discovery, enormous leaps forward have been made to identify and develop extremely potent human mAbs.)]\n",
            "3412: [(, , , From the perspectives of environmental management and environmental engineering, some important actions to minimize risks of AMR development are summarized, including the recent progress in enhanced hydrolysis pre-treatment technology to control the development of antibiotic resistance genes (ARGs) during biological wastewater treatment.)]\n",
            "3413: [(, , , The widespread and often injudicious use of antimicrobial drugs or antibiotics in various sectors including human health and hygiene, agriculture, animal husbandry and food industries are leading to the release of antibiotics into the wastewater/sewage and other water bodies, particularly in urban setups and thus leads to the antimicrobial resistance (AMR) in the microbes.)]\n",
            "3414: [(, , , Moreover, bacteria make use of their evolutionary machinery to adapt to the selective pressure exerted by antibiotic treatments, resulting in reduced efficacy of the therapeutic intervention against human and animal infections.)]\n",
            "3415: [(, , , In this review, we summarize the main points that were raised and highlighted during the pre-conference meeting to the 17th European Cystic Fibrosis Society Basic Science Conference, held from 30 March to 2 April, 2022 in Albufeira, Portugal.)]\n",
            "3416: [(93, 97, Antibiotic-Resistant Genes, In M14 and M15, phoQ deleterious mutations (D90N, I122S and G385S) were identified, while an mgrB variant containing a single deletion (C deletion on position 93) leading to the production of a non-functional protein was detected in M50. mgrB complementation studies showed restoration of polymyxin susceptibility (64 to ≤ 0.25 mg/L) as a wild-type mgrB was inserted into the mgrB-deficient M50.)]\n",
            "3417: [(238, 242, Antibiotic-Resistant Genes, In M14 and M15, phoQ deleterious mutations (D90N, I122S and G385S) were identified, while an mgrB variant containing a single deletion (C deletion on position 93) leading to the production of a non-functional protein was detected in M50. mgrB complementation studies showed restoration of polymyxin susceptibility (64 to ≤ 0.25 mg/L) as a wild-type mgrB was inserted into the mgrB-deficient M50.)]\n",
            "3418: [(349, 353, Antibiotic-Resistant Genes, In M14 and M15, phoQ deleterious mutations (D90N, I122S and G385S) were identified, while an mgrB variant containing a single deletion (C deletion on position 93) leading to the production of a non-functional protein was detected in M50. mgrB complementation studies showed restoration of polymyxin susceptibility (64 to ≤ 0.25 mg/L) as a wild-type mgrB was inserted into the mgrB-deficient M50.)]\n",
            "3419: [(376, 380, Antibiotic-Resistant Genes, In M14 and M15, phoQ deleterious mutations (D90N, I122S and G385S) were identified, while an mgrB variant containing a single deletion (C deletion on position 93) leading to the production of a non-functional protein was detected in M50. mgrB complementation studies showed restoration of polymyxin susceptibility (64 to ≤ 0.25 mg/L) as a wild-type mgrB was inserted into the mgrB-deficient M50.)]\n",
            "3420: [(, , , However, there are still many adverse effects, resulting from each topical antimicrobial. This review aims to summarize the possible adverse effects of commonly used antiseptics (biguanide, silver, iodine, chlorine compounds, and other antiseptics), antibiotics (bacitracin, mafenide, mupirocin, neomycin, and silver sulfadiazine), and natural antimicrobials (curcumin and honey).)]\n",
            "3421: [(240, 248, Antibiotic-Resistant Bacteria, In this review, we summarize the status of resistance against antibiotics in S. typhi, highlight the seminal work in the resistome of S. typhi and the genes involved in the antibiotic resistance, and discuss the various methods to identify S. typhi resistome for the proactive identification of this infection and quick diagnosis of the disease.)]\n",
            "3422: [(134, 142, Antibiotic-Resistant Bacteria, In this review, we summarize the status of resistance against antibiotics in S. typhi, highlight the seminal work in the resistome of S. typhi and the genes involved in the antibiotic resistance, and discuss the various methods to identify S. typhi resistome for the proactive identification of this infection and quick diagnosis of the disease.)]\n",
            "3423: [(77, 85, Antibiotic-Resistant Bacteria, In this review, we summarize the status of resistance against antibiotics in S. typhi, highlight the seminal work in the resistome of S. typhi and the genes involved in the antibiotic resistance, and discuss the various methods to identify S. typhi resistome for the proactive identification of this infection and quick diagnosis of the disease.)]\n",
            "3424: [(, , , Here, we survey the current literature focussing on the following: i) Dissection of human-introduced AMR versus naturally occurring AMR, based on the premise that multiple drug resistance and resistance to synthetic antibiotics not yet found in nature are the results of human contamination ii) The potential role of endemic wildlife in AMR spread There is clear evidence for greater concentrations of AMR around research stations, and although data show reverse zoonosis of the characteristic human gut bacteria to endemic wildlife, AMR within birds and seals appears to be very low, albeit on limited samplings.)]\n",
            "3425: [(, , , The results showed that the number of related studies is increasing year by year; the USA is most published in the field of ARB; China is the most active in this field in the recent years; the Chinese Acad Sci published the most articles; Sci. Total Environ.)]\n",
            "3426: [(, , , We also describe the current state of development of vaccines against resistant bacterial pathogens that cause a substantial disease burden both in high-income countries and in low- and medium-income countries, discuss possible obstacles that hinder progress in vaccine development and speculate on the impact of next-generation vaccines against bacterial infectious diseases on AMR.)]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "len(start), len(end), len(label), len(text)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "pUi9w7KD6Tkx",
        "outputId": "35cc96ca-e7de-4333-e802-f9b7692eabb6"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(3427, 3427, 3427, 3427)"
            ]
          },
          "metadata": {},
          "execution_count": 67
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "text[33][183:197]\n",
        "end[33]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-sKn3_GXcifa",
        "outputId": "a513d5ee-6369-4378-ca37-ee4e325bb528"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "82"
            ]
          },
          "metadata": {},
          "execution_count": 68
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#code for printing tagged sentences\n",
        "## -------------- problem is that un-annotated sentences are returning errors instead of tagging O to all words ------------\n",
        "res=[]\n",
        "for i in range(1000):\n",
        "  start_temp=start[i:i+1]\n",
        "  end_temp=end[i:i+1]\n",
        "  label_temp=label[i:i+1]\n",
        "  result=list(zip(start_temp, end_temp, label_temp))\n",
        "  res.append(result)\n",
        "  nlp=English()\n",
        "  offsets = res[i]\n",
        "  doc = nlp(text[i])\n",
        "  try:\n",
        "    tags = offsets_to_biluo_tags(nlp.make_doc(text[i]), offsets)\n",
        "    print([token.text for token in doc])\n",
        "    print(tags)\n",
        "  except:\n",
        "    pass\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "1bIRvHpVzatM",
        "outputId": "c763511b-4694-4d3e-e681-4b7fb0d67994"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['Current', 'limitations', 'in', 'the', 'understanding', 'and', 'control', 'of', 'antimicrobial', 'resistance', '(', 'AMR', ')', 'in', 'Canada', 'are', 'described', 'through', 'a', 'comprehensive', 'review', 'focusing', 'on', ':', '(', '1', ')', 'treatment', 'optimization', ';', '(', '2', ')', 'surveillance', 'of', 'antimicrobial', 'use', 'and', 'AMR', ';', 'and', '(', '3', ')', 'prevention', 'of', 'transmission', 'of', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Several', 'examples', 'of', 'the', 'implementation', 'of', 'DART', '2020', 'are', 'outlined', 'here', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'L-Others', 'O', 'O', 'O', 'O']\n",
            "['We', 'report', 'the', 'use', 'of', 'a', 'novel', 'pain', '-', 'relief', 'therapy', 'for', 'FMD', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O']\n",
            "['Regarding', 'the', 'necessary', 'decrease', 'in', 'the', 'use', 'of', 'antibiotics', ',', 'results', 'taken', 'from', 'the', 'literature', 'indicate', 'that', 'phages', 'are', 'also', 'promising', 'tools', 'to', 'help', 'to', 'address', 'this', 'issue', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Bacteriophages', '(', 'phages)-viruses', 'that', 'infect', 'and', 'kill', 'bacteria', '-', 'represent', 'a', 'promising', 'tool', 'for', 'combatting', 'the', 'global', 'threat', 'of', 'antibiotic', '-', 'resistant', 'pathogens', '.']\n",
            "['U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'revealed', 'that', '18', 'out', 'of', '64', 'districts', 'in', 'Bangladesh', 'reported', 'E.', 'coli', 'in', 'poultry', ',', 'having', 'a', 'higher', 'prevalence', '(', 'combined', 'prevalence', ':', '69.3', '%', ',', '95', '%', 'confidence', 'interval', ',', 'CI', ':', '67.3', '-', '71', '%', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'found', 'that', 'E.', 'coli', 'isolates', 'showed', 'resistance', 'to', '14', 'antimicrobial', 'classes', 'and', '45', 'different', 'antimicrobial', 'agents', ',', 'including', 'the', 'last', '-', 'line', '(', 'reserve', 'group', ')', 'antibiotics', 'and', 'banned', 'antimicrobial', 'categories', 'for', 'the', 'treatment', 'of', 'infections', 'in', 'agricultural', 'animals', '.']\n",
            "['O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"Phenotypic resistance of E. coli against penicilli...\" with entities \"[(237, 248, 'Antibiotics')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"Phenotypic resistance of E. coli against penicilli...\" with entities \"[(286, 296, 'Antibiotics')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"Phenotypic resistance of E. coli against penicilli...\" with entities \"[(309, 321, 'Antibiotics')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', '-', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Phenotypic', 'resistance', 'of', 'E.', 'coli', 'against', 'penicillins', 'and', 'beta', '-', 'lactamase', 'inhibitors', '(', '20.2%-100', '%', ')', ',', 'cephalosporins', '(', '1.9%-100', '%', ')', ',', 'fluoroquinolones', '(', '5.98%-100', '%', ')', ',', 'aminoglycosides', '(', '6%-100', '%', ')', ',', 'tetracyclines', '(', '17.7%-100', '%', ')', ',', 'carbapenems', '(', '13.6%-72.7', '%', ')', ',', 'macrolides', '(', '11.8%-100', '%', ')', ',', 'polymyxins', '(', '7.9%-100', '%', ')', ',', 'phenicols', '(', '20%-97.2', '%', ')', ',', 'sulfa', 'drugs', '(', '44.7%-100', '%', ')', ',', 'cephamycins', '(', '21.4%-48.8', '%', ')', ',', 'nitrofurans', '(', '21.4%-63.2', '%', ')', ',', 'monobactams', '(', '1.2', '%', ')', ',', 'and', 'glycylcyclines', '(', '2.3', '%', ')', 'was', 'recorded', 'in', 'the', 'last', 'decades', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Twenty', '-', 'four', 'different', 'AMR', 'genes', 'encoding', 'resistance', 'to', 'beta', '-', 'lactams', '(', 'bla', 'TEM', ',', 'bla', 'CTX', '-', 'M-1', ',', 'bla', 'CTX', '-', 'M-2', ',', 'bla', 'CTX', '-', 'M-9', ',', 'bla', 'OXA-1', ',', 'bla', 'OXA-47', ',', 'bla', 'SHV', ',', 'and', 'CITM', ')', ',', 'colistin', '(', 'mcr1', 'and', 'mcr3', ')', ',', 'fluoroquinolones', '(', 'qnrB', 'and', 'qnrS', ')', ',', 'tetracyclines', '(', 'tetA', ',', 'tetB', ',', 'and', 'tetC', ')', ',', 'sulfonamides', '(', 'sulI', 'and', 'sulII', ')', ',', 'trimethoprim', '(', 'dfrA1', ')', ',', 'aminoglycosides', '(', 'rmtB', ')', ',', 'streptomycin', '(', 'aadA1', ')', ',', 'gentamicin', '(', 'aac-3', '-', 'IV', ')', ',', 'erythromycin', '(', 'ereA', ')', ',', 'and', 'chloramphenicol', '(', 'catA1', 'and', 'cmlA', ')', 'were', 'detected', 'in', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'suggest', 'a', 'regular', 'antimicrobial', 'surveillance', 'program', 'with', 'a', 'strong', 'One', 'Health', 'approach', 'to', 'lessen', 'the', 'hazardous', 'effects', 'of', 'AMR', 'E.', 'coli', 'in', 'poultry', 'industries', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'suggest', 'a', 'regular', 'antimicrobial', 'surveillance', 'program', 'with', 'a', 'strong', 'One', 'Health', 'approach', 'to', 'lessen', 'the', 'hazardous', 'effects', 'of', 'AMR', 'E.', 'coli', 'in', 'poultry', 'industries', 'in', 'Bangladesh', '.']\n",
            "['O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Moreover', ',', 'the', 'existence', 'of', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Escherichia', 'coli', 'exerts', 'an', 'ominous', 'effect', 'on', 'the', 'poultry', 'sector', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Following', 'the', 'PRISMA', 'guidelines', ',', 'a', 'total', 'of', '17', 'published', 'scientific', 'articles', 'were', 'selected', 'for', 'this', 'systematic', 'review', '.']\n",
            "['O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Also', ',', '14', 'articles', 'reported', 'MDR', 'E.', 'coli', 'in', 'poultry', ',', 'including', 'a', '100', '%', 'MDR', 'in', 'nine', 'articles', 'and', 'a', '92.7', '%', '(', '95', '%', 'CI', ':', '91.2', '-', '94', '%', ')', 'combined', 'percentage', 'of', 'MDR', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Also', ',', '14', 'articles', 'reported', 'MDR', 'E.', 'coli', 'in', 'poultry', ',', 'including', 'a', '100', '%', 'MDR', 'in', 'nine', 'articles', 'and', 'a', '92.7', '%', '(', '95', '%', 'CI', ':', '91.2', '-', '94', '%', ')', 'combined', 'percentage', 'of', 'MDR', 'E.', 'coli', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O']\n",
            "['Besides', 'the', 'conventional', 'approaches', ',', 'some', 'novel', 'approaches', 'such', 'as', 'the', 'use', 'of', 'antimicrobial', 'peptides', ',', 'bacteriophages', ',', 'immunomodulation', ',', 'host', '-', 'directed', 'therapy', 'and', 'antibodies', 'have', 'shown', 'really', 'promising', 'potentials', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Besides', 'the', 'conventional', 'approaches', ',', 'some', 'novel', 'approaches', 'such', 'as', 'the', 'use', 'of', 'antimicrobial', 'peptides', ',', 'bacteriophages', ',', 'immunomodulation', ',', 'host', '-', 'directed', 'therapy', 'and', 'antibodies', 'have', 'shown', 'really', 'promising', 'potentials', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['With', 'the', 'advent', 'of', 'the', 'global', 'antimicrobial', 'resistance', '(', 'AMR', ')', 'crisis', ',', 'our', 'arsenal', 'of', 'effective', 'antibiotics', 'is', 'diminishing', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O']\n",
            "['Therefore', ',', 'understanding', 'the', 'gut', 'microbiota', 'diversity', ',', 'composition', ',', 'and', 'function', 'during', 'various', 'inflammatory', 'conditions', 'and', 'sepsis', 'may', 'provide', 'a', 'comprehensive', 'knowledge', 'of', 'the', 'mechanisms', 'behind', 'the', 'pathogenesis', 'of', 'gut', '-', 'derived', 'infection', 'in', 'diseases', 'and', 'the', 'design', 'of', 'new', 'treatment', 'options', '(', 'e.g.', ',', 'probiotics', 'or', 'fecal', 'microbiota', 'transplantation', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'alternative', 'antibacterial', 'agents', 'could', 'be', 'a', 'key', 'element', 'in', 'our', 'fight', 'against', 'AMR', 'and', 'provide', 'desperately', 'needed', 'options', ',', 'to', 'veterinary', 'and', 'medical', 'clinicians', 'alike', '.']\n",
            "['O', 'O', 'B-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Overuse', 'of', 'antibiotics', 'in', 'humans', ',', 'agriculture', ',', 'and', 'animal', 'husbandry', 'has', 'resulted', 'in', 'the', 'emergence', 'of', 'a', 'wide', 'range', 'of', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'pathogens', 'which', 'are', 'difficult', 'to', 'treat', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Meanwhile', ',', 'to', 'reduce', 'dependence', 'on', 'antibiotics', ',', 'other', 'alternatives', 'such', 'as', 'vaccines', ',', 'antibodies', ',', 'pattern', 'recognition', 'receptors', ',', 'probiotics', ',', 'bacteriophages', ',', 'peptides', ',', 'phytochemicals', ',', 'metals', ',', 'and', 'antimicrobial', 'enzymes', 'are', 'being', 'explored', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Meanwhile', ',', 'to', 'reduce', 'dependence', 'on', 'antibiotics', ',', 'other', 'alternatives', 'such', 'as', 'vaccines', ',', 'antibodies', ',', 'pattern', 'recognition', 'receptors', ',', 'probiotics', ',', 'bacteriophages', ',', 'peptides', ',', 'phytochemicals', ',', 'metals', ',', 'and', 'antimicrobial', 'enzymes', 'are', 'being', 'explored', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Meanwhile', ',', 'to', 'reduce', 'dependence', 'on', 'antibiotics', ',', 'other', 'alternatives', 'such', 'as', 'vaccines', ',', 'antibodies', ',', 'pattern', 'recognition', 'receptors', ',', 'probiotics', ',', 'bacteriophages', ',', 'peptides', ',', 'phytochemicals', ',', 'metals', ',', 'and', 'antimicrobial', 'enzymes', 'are', 'being', 'explored', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Meanwhile', ',', 'to', 'reduce', 'dependence', 'on', 'antibiotics', ',', 'other', 'alternatives', 'such', 'as', 'vaccines', ',', 'antibodies', ',', 'pattern', 'recognition', 'receptors', ',', 'probiotics', ',', 'bacteriophages', ',', 'peptides', ',', 'phytochemicals', ',', 'metals', ',', 'and', 'antimicrobial', 'enzymes', 'are', 'being', 'explored', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Meanwhile', ',', 'to', 'reduce', 'dependence', 'on', 'antibiotics', ',', 'other', 'alternatives', 'such', 'as', 'vaccines', ',', 'antibodies', ',', 'pattern', 'recognition', 'receptors', ',', 'probiotics', ',', 'bacteriophages', ',', 'peptides', ',', 'phytochemicals', ',', 'metals', ',', 'and', 'antimicrobial', 'enzymes', 'are', 'being', 'explored', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'L-Others', 'O', 'O', 'O', 'O']\n",
            "['Meanwhile', ',', 'to', 'reduce', 'dependence', 'on', 'antibiotics', ',', 'other', 'alternatives', 'such', 'as', 'vaccines', ',', 'antibodies', ',', 'pattern', 'recognition', 'receptors', ',', 'probiotics', ',', 'bacteriophages', ',', 'peptides', ',', 'phytochemicals', ',', 'metals', ',', 'and', 'antimicrobial', 'enzymes', 'are', 'being', 'explored', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Hence', 'it', 'is', 'high', 'time', 'for', 'the', 'development', 'of', 'new', 'strategies', 'to', 'fight', 'against', 'dangerous', 'MDR', 'pathogens', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O']\n",
            "['This', 'review', 'highlights', 'the', 'magnitude', 'of', 'the', 'AMR', 'in', 'the', 'pediatric', 'population', 'and', 'calls', 'for', 'the', 'urgent', 'implementation', 'of', 'antimicrobial', 'stewardship', 'programs', 'to', 'save', 'the', 'remaining', 'antimicrobials', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'have', 'recently', 'reported', 'the', 'acquisition', 'of', 'multidrug', 'resistance', 'in', 'Escherichia', 'coli', 'after', 'exposure', 'to', 'gut', '-', 'relevant', 'concentrations', 'of', 'the', 'antipsychotic', 'quetiapine', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Thus', ',', 'there', 'is', 'still', 'a', 'paucity', 'of', 'phage', '-', 'related', 'studies', 'in', 'this', 'region', ',', 'including', 'those', 'focused', 'on', 'therapeutic', 'applications', ',', 'clinical', 'trials', ',', 'biotechnology', ',', 'and', 'biocontrol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Thus', ',', 'there', 'is', 'still', 'a', 'paucity', 'of', 'phage', '-', 'related', 'studies', 'in', 'this', 'region', ',', 'including', 'those', 'focused', 'on', 'therapeutic', 'applications', ',', 'clinical', 'trials', ',', 'biotechnology', ',', 'and', 'biocontrol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O']\n",
            "['This', 'article', 'highlights', 'the', 'importance', 'of', 'bacteriophage', 'research', 'and', 'development', 'and', 'discusses', 'the', 'potential', 'implementation', 'of', 'phage', '-', 'based', 'therapies', 'in', 'Saudi', 'Arabia', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O']\n",
            "['Bloodstream', 'infections', '(', 'BSI', ')', 'due', 'to', 'multidrug', '-', 'resistant', 'organisms', ',', 'especially', 'from', 'pediatric', 'intensive', 'care', 'units', '(', 'PICU', ')', ',', 'are', 'being', 'increasingly', 'reported', 'across', 'the', 'world', '.']\n",
            "['B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Bloodstream', 'infections', '(', 'BSI', ')', 'due', 'to', 'multidrug', '-', 'resistant', 'organisms', ',', 'especially', 'from', 'pediatric', 'intensive', 'care', 'units', '(', 'PICU', ')', ',', 'are', 'being', 'increasingly', 'reported', 'across', 'the', 'world', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', 'on', 'etiology', 'and', 'its', 'antibiogram', 'in', 'community', 'acquired', 'BSI', ',', 'S.', 'typhi', 'followed', 'by', 'S.', 'paratyphi', 'A', 'were', 'the', 'major', 'bacterial', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', 'on', 'etiology', 'and', 'its', 'antibiogram', 'in', 'community', 'acquired', 'BSI', ',', 'S.', 'typhi', 'followed', 'by', 'S.', 'paratyphi', 'A', 'were', 'the', 'major', 'bacterial', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Quinolone', 'resistance', 'of', 'more', 'than', '90', '%', 'in', 'Salmonella', 'is', 'now', 'reported', 'from', 'all', 'over', 'India', '.', 'Ceftriaxone', 'remains', 'the', 'drug', 'of', 'choice', 'for', 'enteric', 'fever', 'due', 'to', 'its', '100', '%', 'susceptibility', '.']\n",
            "['U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Quinolone', 'resistance', 'of', 'more', 'than', '90', '%', 'in', 'Salmonella', 'is', 'now', 'reported', 'from', 'all', 'over', 'India', '.', 'Ceftriaxone', 'remains', 'the', 'drug', 'of', 'choice', 'for', 'enteric', 'fever', 'due', 'to', 'its', '100', '%', 'susceptibility', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Quinolone', 'resistance', 'of', 'more', 'than', '90', '%', 'in', 'Salmonella', 'is', 'now', 'reported', 'from', 'all', 'over', 'India', '.', 'Ceftriaxone', 'remains', 'the', 'drug', 'of', 'choice', 'for', 'enteric', 'fever', 'due', 'to', 'its', '100', '%', 'susceptibility', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Passive', 'immunization', 'through', 'oral', ',', 'nasal', 'and', 'topical', 'administration', 'of', 'egg', 'yolk', '-', 'derived', 'IgY', 'antibodies', 'from', 'immunized', 'chickens', 'were', 'recently', 'shown', 'to', 'be', 'effective', 'for', 'treating', 'bacterial', 'infections', 'in', 'animals', 'and', 'humans', '.']\n",
            "['B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'PICU', 'there', 'is', 'an', 'emergence', 'of', 'candidemia', 'due', 'to', 'non', '-', 'albicans', 'candida', 'which', 'are', 'now', 'predominant', 'isolates', 'at', 'few', 'centers', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Immunization', 'of', 'chickens', 'with', 'specific', 'antigens', 'offers', 'the', 'possibility', 'of', 'creating', 'specific', 'antibodies', 'targeting', 'a', 'wide', 'range', 'of', 'antibiotic', '-', 'resistant', 'bacteria', '.']\n",
            "['U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['BSI', 'due', 'to', 'gram', '-', 'negative', 'bacteria', ',', 'mostly', 'by', 'Klebseilla', 'pneumoniae', 'and', 'gram', '-', 'positive', 'cocci', '(', 'S.', 'aureus', ')', 'are', 'the', 'other', 'major', 'pathogens', 'commonly', 'observed', 'in', 'BSI', 'from', 'PICU', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['BSI', 'due', 'to', 'gram', '-', 'negative', 'bacteria', ',', 'mostly', 'by', 'Klebseilla', 'pneumoniae', 'and', 'gram', '-', 'positive', 'cocci', '(', 'S.', 'aureus', ')', 'are', 'the', 'other', 'major', 'pathogens', 'commonly', 'observed', 'in', 'BSI', 'from', 'PICU', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'we', 'describe', 'the', 'growing', 'global', 'problem', 'of', 'antimicrobial', 'resistance', 'and', 'highlight', 'the', 'promising', 'potential', 'of', 'the', 'use', 'of', 'egg', 'yolk', 'IgY', 'antibodies', 'for', 'the', 'treatment', 'of', 'bacterial', 'infections', ',', 'particularly', 'those', 'listed', 'in', 'the', 'World', 'Health', 'Organization', 'priority', 'list', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['There', 'is', 'a', 'high', 'prevalence', 'of', 'antimicrobial', 'resistance', 'to', 'commonly', 'used', 'antibiotics', 'like', 'ampicillin', '(', '94.9%-90.7', '%', ')', ',', 'cefotaxime', '(', '92.4%-71.4', '%', ')', ',', 'piperacillin', '-', 'tazobactum', '(', '31.2%-27.5', '%', ')', 'and', 'levofloxacin', '(', '42.4%-39.8', '%', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['There', 'is', 'a', 'high', 'prevalence', 'of', 'antimicrobial', 'resistance', 'to', 'commonly', 'used', 'antibiotics', 'like', 'ampicillin', '(', '94.9%-90.7', '%', ')', ',', 'cefotaxime', '(', '92.4%-71.4', '%', ')', ',', 'piperacillin', '-', 'tazobactum', '(', '31.2%-27.5', '%', ')', 'and', 'levofloxacin', '(', '42.4%-39.8', '%', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['There', 'is', 'a', 'high', 'prevalence', 'of', 'antimicrobial', 'resistance', 'to', 'commonly', 'used', 'antibiotics', 'like', 'ampicillin', '(', '94.9%-90.7', '%', ')', ',', 'cefotaxime', '(', '92.4%-71.4', '%', ')', ',', 'piperacillin', '-', 'tazobactum', '(', '31.2%-27.5', '%', ')', 'and', 'levofloxacin', '(', '42.4%-39.8', '%', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['There', 'is', 'a', 'high', 'prevalence', 'of', 'antimicrobial', 'resistance', 'to', 'commonly', 'used', 'antibiotics', 'like', 'ampicillin', '(', '94.9%-90.7', '%', ')', ',', 'cefotaxime', '(', '92.4%-71.4', '%', ')', ',', 'piperacillin', '-', 'tazobactum', '(', '31.2%-27.5', '%', ')', 'and', 'levofloxacin', '(', '42.4%-39.8', '%', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O']\n",
            "['Resistance', 'to', 'carbapenems', ',', 'primarily', 'due', 'to', 'blaNDM', 'is', 'seen', 'in', 'all', 'the', 'centers', 'and', 'the', 'rate', 'varies', 'between', '1%-', '79', '%', 'with', 'K.', 'pneumoniae', 'and', 'A.', 'baumannii', 'showing', 'the', 'maximum', 'resistance', '.']\n",
            "['O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Resistance', 'to', 'carbapenems', ',', 'primarily', 'due', 'to', 'blaNDM', 'is', 'seen', 'in', 'all', 'the', 'centers', 'and', 'the', 'rate', 'varies', 'between', '1%-', '79', '%', 'with', 'K.', 'pneumoniae', 'and', 'A.', 'baumannii', 'showing', 'the', 'maximum', 'resistance', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Resistance', 'to', 'carbapenems', ',', 'primarily', 'due', 'to', 'blaNDM', 'is', 'seen', 'in', 'all', 'the', 'centers', 'and', 'the', 'rate', 'varies', 'between', '1%-', '79', '%', 'with', 'K.', 'pneumoniae', 'and', 'A.', 'baumannii', 'showing', 'the', 'maximum', 'resistance', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Resistance', 'to', 'carbapenems', ',', 'primarily', 'due', 'to', 'blaNDM', 'is', 'seen', 'in', 'all', 'the', 'centers', 'and', 'the', 'rate', 'varies', 'between', '1%-', '79', '%', 'with', 'K.', 'pneumoniae', 'and', 'A.', 'baumannii', 'showing', 'the', 'maximum', 'resistance', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'bacteriophage', '(', 'or', 'phage', 'for', 'short', ')', 'has', 'been', 'used', 'as', 'an', 'antibacterial', 'agent', 'for', 'over', 'a', 'century', 'but', 'was', 'abandoned', 'in', 'most', 'countries', 'after', 'the', 'discovery', 'and', 'broad', 'use', 'of', 'antibiotics', '.']\n",
            "['O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Nowadays', ',', 'phage', '-', 'based', 'technology', 'has', 'been', 'developed', 'for', 'many', 'purposes', ',', 'including', 'those', 'beyond', 'the', 'framework', 'of', 'antibacterial', 'treatment', ',', 'such', 'as', 'to', 'suppress', 'viruses', 'by', 'phages', ',', 'gene', 'therapy', ',', 'vaccine', 'development', ',', 'etc', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['New', 'hope', 'comes', 'in', 'the', 'shape', 'of', 'antibiotic', 'combination', 'therapy', ',', 'which', 'although', 'used', 'by', 'mother', 'nature', ',', 'is', 'under', '-', 'explored', 'and', 'could', 'provide', 'the', 'solution', 'to', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'addition', 'to', 'being', 'a', 'significant', 'clinical', 'problem', 'in', 'its', 'own', 'right', ',', 'K.', 'pneumoniae', 'is', 'the', 'species', 'within', 'which', 'several', 'new', 'AMR', 'genes', 'were', 'first', 'discovered', 'before', 'spreading', 'to', 'other', 'pathogens', '(', 'e.g.', 'carbapenem', '-', 'resistance', 'genes', 'KPC', ',', 'OXA-48', 'and', 'NDM-1', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'addition', 'to', 'being', 'a', 'significant', 'clinical', 'problem', 'in', 'its', 'own', 'right', ',', 'K.', 'pneumoniae', 'is', 'the', 'species', 'within', 'which', 'several', 'new', 'AMR', 'genes', 'were', 'first', 'discovered', 'before', 'spreading', 'to', 'other', 'pathogens', '(', 'e.g.', 'carbapenem', '-', 'resistance', 'genes', 'KPC', ',', 'OXA-48', 'and', 'NDM-1', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O']\n",
            "['In', 'addition', 'to', 'being', 'a', 'significant', 'clinical', 'problem', 'in', 'its', 'own', 'right', ',', 'K.', 'pneumoniae', 'is', 'the', 'species', 'within', 'which', 'several', 'new', 'AMR', 'genes', 'were', 'first', 'discovered', 'before', 'spreading', 'to', 'other', 'pathogens', '(', 'e.g.', 'carbapenem', '-', 'resistance', 'genes', 'KPC', ',', 'OXA-48', 'and', 'NDM-1', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"Whilst K. pneumoniae's contribution to the overall...\" with entities \"[(7, 21, 'Microbes')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['Whilst', 'K.', 'pneumoniae', \"'s\", 'contribution', 'to', 'the', 'overall', 'AMR', 'crisis', 'is', 'impossible', 'to', 'quantify', ',', 'current', 'evidence', 'suggests', 'it', 'has', 'a', 'wider', 'ecological', 'distribution', ',', 'significantly', 'more', 'varied', 'DNA', 'composition', ',', 'greater', 'AMR', 'gene', 'diversity', 'and', 'a', 'higher', 'plasmid', 'burden', 'than', 'other', 'Gram', 'negative', 'opportunists', '.']\n",
            "['O', '-', '-', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'worldwide', 'emergence', 'and', 'high', 'prevalence', 'of', 'antimicrobial', '-', 'resistant', '(', 'AMR', ')', 'bacteria', 'have', 'led', 'to', 'a', 'revival', 'of', 'interest', 'in', 'the', 'long', '-', 'forgotten', 'antibacterial', 'therapy', 'with', 'phages', '(', 'phage', 'therapy', ')', 'as', 'an', 'alternative', 'approach', 'to', 'combatting', 'AMR', 'bacteria', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'worldwide', 'emergence', 'and', 'high', 'prevalence', 'of', 'antimicrobial', '-', 'resistant', '(', 'AMR', ')', 'bacteria', 'have', 'led', 'to', 'a', 'revival', 'of', 'interest', 'in', 'the', 'long', '-', 'forgotten', 'antibacterial', 'therapy', 'with', 'phages', '(', 'phage', 'therapy', ')', 'as', 'an', 'alternative', 'approach', 'to', 'combatting', 'AMR', 'bacteria', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'worldwide', 'emergence', 'and', 'high', 'prevalence', 'of', 'antimicrobial', '-', 'resistant', '(', 'AMR', ')', 'bacteria', 'have', 'led', 'to', 'a', 'revival', 'of', 'interest', 'in', 'the', 'long', '-', 'forgotten', 'antibacterial', 'therapy', 'with', 'phages', '(', 'phage', 'therapy', ')', 'as', 'an', 'alternative', 'approach', 'to', 'combatting', 'AMR', 'bacteria', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Substantial', 'effort', 'and', 'financial', 'support', 'are', 'being', 'spent', 'on', 'identifying', 'resistance', 'mechanisms', 'circulating', 'within', 'the', 'gonococcal', 'population', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O']\n",
            "['IMPORTANCE', 'Contemporary', 'increases', 'in', 'the', 'prevalence', 'of', 'antimicrobial', 'resistance', '(', 'AMR', ')', 'in', 'Neisseria', 'gonorrhoeae', 'populations', 'is', 'a', 'direct', 'threat', 'to', 'global', 'public', 'health', 'and', 'the', 'effective', 'treatment', 'of', 'gonorrhea', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['However', ',', 'these', 'surveys', 'often', 'overlook', 'a', 'known', 'source', 'of', 'resistance', 'for', 'gonococci', '-', 'the', 'commensal', 'Neisseria', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O']\n",
            "['C.', 'difficile', 'ribotype', '(', 'RT', ')', '017', 'has', 'emerged', 'as', 'the', 'major', 'strain', 'of', 'C.', 'difficile', 'in', 'Asia', ',', 'where', 'antimicrobial', 'use', 'is', 'poorly', 'regulated', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['C.', 'difficile', 'ribotype', '(', 'RT', ')', '017', 'has', 'emerged', 'as', 'the', 'major', 'strain', 'of', 'C.', 'difficile', 'in', 'Asia', ',', 'where', 'antimicrobial', 'use', 'is', 'poorly', 'regulated', '.']\n",
            "['B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Here', ',', 'we', 'propose', 'the', 'expansion', 'of', 'established', 'gonococcal', 'surveillance', 'programs', 'to', 'integrate', 'a', 'collection', ',', 'AMR', 'profiling', ',', 'and', 'genomic', 'sequencing', 'pipeline', 'for', 'commensal', 'species', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Here', ',', 'we', 'propose', 'the', 'expansion', 'of', 'established', 'gonococcal', 'surveillance', 'programs', 'to', 'integrate', 'a', 'collection', ',', 'AMR', 'profiling', ',', 'and', 'genomic', 'sequencing', 'pipeline', 'for', 'commensal', 'species', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'strain', 'has', 'also', 'caused', 'CDI', 'outbreaks', 'around', 'the', 'world', 'for', 'almost', '30', 'years', '.', 'Many', 'of', 'these', 'outbreaks', 'were', 'associated', 'with', 'clindamycin', 'and', 'fluoroquinolone', 'resistance', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O']\n",
            "['This', 'proposed', 'expansion', 'will', 'enhance', 'the', 'field', \"'s\", 'ability', 'to', 'identify', 'resistance', 'in', 'and', 'from', 'nonpathogenic', 'reservoirs', 'and', 'anticipate', 'AMR', 'trends', 'in', 'pathogenic', 'Neisseria', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O']\n",
            "['To', 'address', 'these', 'topics', ',', 'reports', 'of', 'outbreaks', 'due', 'to', 'C.', 'difficile', 'RT', '017', ',', 'epidemiologic', 'studies', 'with', 'antimicrobial', 'susceptibility', 'results', ',', 'studies', 'on', 'resistance', 'mechanisms', 'found', 'in', 'C.', 'difficile', 'and', 'related', 'publications', 'available', 'through', 'Pubmed', 'until', 'September', '2019', 'were', 'collated', 'and', 'the', 'findings', 'discussed', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['To', 'address', 'these', 'topics', ',', 'reports', 'of', 'outbreaks', 'due', 'to', 'C.', 'difficile', 'RT', '017', ',', 'epidemiologic', 'studies', 'with', 'antimicrobial', 'susceptibility', 'results', ',', 'studies', 'on', 'resistance', 'mechanisms', 'found', 'in', 'C.', 'difficile', 'and', 'related', 'publications', 'available', 'through', 'Pubmed', 'until', 'September', '2019', 'were', 'collated', 'and', 'the', 'findings', 'discussed', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Antimicrobial', 'resistance', '(', 'AMR', ')', 'is', 'widespread', 'within', 'Neisseria', 'gonorrhoeae', 'populations', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O']\n",
            "['Recent', 'work', 'has', 'highlighted', 'the', 'importance', 'of', 'commensal', 'Neisseria', '(', 'cN', ')', 'as', 'a', 'source', 'of', 'AMR', 'for', 'their', 'pathogenic', 'relatives', 'through', 'horizontal', 'gene', 'transfer', '(', 'HGT', ')', 'of', 'AMR', 'alleles', ',', 'such', 'as', 'mosaic', 'penicillin', 'binding', 'protein', '2', '(', 'penA', ')', ',', 'multiple', 'transferable', 'efflux', 'pump', '(', 'mtr', ')', ',', 'and', 'DNA', 'gyrase', 'subunit', 'A', '(', 'gyrA', ')', 'which', 'impact', 'beta', '-', 'lactam', ',', 'azithromycin', ',', 'and', 'ciprofloxacin', 'susceptibility', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Recent', 'work', 'has', 'highlighted', 'the', 'importance', 'of', 'commensal', 'Neisseria', '(', 'cN', ')', 'as', 'a', 'source', 'of', 'AMR', 'for', 'their', 'pathogenic', 'relatives', 'through', 'horizontal', 'gene', 'transfer', '(', 'HGT', ')', 'of', 'AMR', 'alleles', ',', 'such', 'as', 'mosaic', 'penicillin', 'binding', 'protein', '2', '(', 'penA', ')', ',', 'multiple', 'transferable', 'efflux', 'pump', '(', 'mtr', ')', ',', 'and', 'DNA', 'gyrase', 'subunit', 'A', '(', 'gyrA', ')', 'which', 'impact', 'beta', '-', 'lactam', ',', 'azithromycin', ',', 'and', 'ciprofloxacin', 'susceptibility', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Recent', 'work', 'has', 'highlighted', 'the', 'importance', 'of', 'commensal', 'Neisseria', '(', 'cN', ')', 'as', 'a', 'source', 'of', 'AMR', 'for', 'their', 'pathogenic', 'relatives', 'through', 'horizontal', 'gene', 'transfer', '(', 'HGT', ')', 'of', 'AMR', 'alleles', ',', 'such', 'as', 'mosaic', 'penicillin', 'binding', 'protein', '2', '(', 'penA', ')', ',', 'multiple', 'transferable', 'efflux', 'pump', '(', 'mtr', ')', ',', 'and', 'DNA', 'gyrase', 'subunit', 'A', '(', 'gyrA', ')', 'which', 'impact', 'beta', '-', 'lactam', ',', 'azithromycin', ',', 'and', 'ciprofloxacin', 'susceptibility', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Gene/Proteins', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Recent', 'work', 'has', 'highlighted', 'the', 'importance', 'of', 'commensal', 'Neisseria', '(', 'cN', ')', 'as', 'a', 'source', 'of', 'AMR', 'for', 'their', 'pathogenic', 'relatives', 'through', 'horizontal', 'gene', 'transfer', '(', 'HGT', ')', 'of', 'AMR', 'alleles', ',', 'such', 'as', 'mosaic', 'penicillin', 'binding', 'protein', '2', '(', 'penA', ')', ',', 'multiple', 'transferable', 'efflux', 'pump', '(', 'mtr', ')', ',', 'and', 'DNA', 'gyrase', 'subunit', 'A', '(', 'gyrA', ')', 'which', 'impact', 'beta', '-', 'lactam', ',', 'azithromycin', ',', 'and', 'ciprofloxacin', 'susceptibility', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Recent', 'work', 'has', 'highlighted', 'the', 'importance', 'of', 'commensal', 'Neisseria', '(', 'cN', ')', 'as', 'a', 'source', 'of', 'AMR', 'for', 'their', 'pathogenic', 'relatives', 'through', 'horizontal', 'gene', 'transfer', '(', 'HGT', ')', 'of', 'AMR', 'alleles', ',', 'such', 'as', 'mosaic', 'penicillin', 'binding', 'protein', '2', '(', 'penA', ')', ',', 'multiple', 'transferable', 'efflux', 'pump', '(', 'mtr', ')', ',', 'and', 'DNA', 'gyrase', 'subunit', 'A', '(', 'gyrA', ')', 'which', 'impact', 'beta', '-', 'lactam', ',', 'azithromycin', ',', 'and', 'ciprofloxacin', 'susceptibility', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Recent', 'work', 'has', 'highlighted', 'the', 'importance', 'of', 'commensal', 'Neisseria', '(', 'cN', ')', 'as', 'a', 'source', 'of', 'AMR', 'for', 'their', 'pathogenic', 'relatives', 'through', 'horizontal', 'gene', 'transfer', '(', 'HGT', ')', 'of', 'AMR', 'alleles', ',', 'such', 'as', 'mosaic', 'penicillin', 'binding', 'protein', '2', '(', 'penA', ')', ',', 'multiple', 'transferable', 'efflux', 'pump', '(', 'mtr', ')', ',', 'and', 'DNA', 'gyrase', 'subunit', 'A', '(', 'gyrA', ')', 'which', 'impact', 'beta', '-', 'lactam', ',', 'azithromycin', ',', 'and', 'ciprofloxacin', 'susceptibility', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O']\n",
            "['Recent', 'work', 'has', 'highlighted', 'the', 'importance', 'of', 'commensal', 'Neisseria', '(', 'cN', ')', 'as', 'a', 'source', 'of', 'AMR', 'for', 'their', 'pathogenic', 'relatives', 'through', 'horizontal', 'gene', 'transfer', '(', 'HGT', ')', 'of', 'AMR', 'alleles', ',', 'such', 'as', 'mosaic', 'penicillin', 'binding', 'protein', '2', '(', 'penA', ')', ',', 'multiple', 'transferable', 'efflux', 'pump', '(', 'mtr', ')', ',', 'and', 'DNA', 'gyrase', 'subunit', 'A', '(', 'gyrA', ')', 'which', 'impact', 'beta', '-', 'lactam', ',', 'azithromycin', ',', 'and', 'ciprofloxacin', 'susceptibility', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Recent', 'work', 'has', 'highlighted', 'the', 'importance', 'of', 'commensal', 'Neisseria', '(', 'cN', ')', 'as', 'a', 'source', 'of', 'AMR', 'for', 'their', 'pathogenic', 'relatives', 'through', 'horizontal', 'gene', 'transfer', '(', 'HGT', ')', 'of', 'AMR', 'alleles', ',', 'such', 'as', 'mosaic', 'penicillin', 'binding', 'protein', '2', '(', 'penA', ')', ',', 'multiple', 'transferable', 'efflux', 'pump', '(', 'mtr', ')', ',', 'and', 'DNA', 'gyrase', 'subunit', 'A', '(', 'gyrA', ')', 'which', 'impact', 'beta', '-', 'lactam', ',', 'azithromycin', ',', 'and', 'ciprofloxacin', 'susceptibility', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Gene/Proteins', 'I-Gene/Proteins', 'I-Gene/Proteins', 'L-Gene/Proteins', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Gene/Proteins', 'I-Gene/Proteins', 'L-Gene/Proteins', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Gene/Proteins', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'provides', 'an', 'overview', 'of', 'the', 'current', 'situation', ',', 'mechanism', ',', 'epidemiology', ',', 'detection', 'methods', 'and', 'clinical', 'treatment', 'for', 'antimicrobial', 'resistant', 'genes', 'in', 'clinical', 'important', 'bacteria', 'including', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', '(', 'MRSA', ')', ',', 'vancomycin', '-', 'resistant', 'Enterococcus', '(', 'VRE', ')', ',', 'penicillin', '-', 'resistant', 'Streptococcus', 'pneumoniae', '(', 'PRSP', ')', ',', 'extended', '-', 'spectrum', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'acquired', 'AmpC', 'β', '-', 'lactamase', '-', 'producing', 'Enterobacteriaceae', ',', 'carbapenemase', '-', 'producing', 'Enterobacteriaceae', '(', 'CPE', ')', ',', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'Acinetobacter', 'baumannii', 'and', 'Pseudomonas', 'aeruginosa', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O']\n",
            "['We', 'summarize', 'the', 'current', 'understanding', 'of', 'commensal', 'Neisseria', 'as', 'an', 'AMR', 'reservoir', ',', 'and', 'call', 'to', 'increase', 'research', 'on', 'commensal', 'Neisseria', 'species', ',', 'through', 'expanding', 'established', 'gonococcal', 'surveillance', 'programs', 'to', 'include', 'the', 'collection', ',', 'isolation', ',', 'antimicrobial', 'resistance', 'phenotyping', ',', 'and', 'whole', '-', 'genome', 'sequencing', '(', 'WGS', ')', 'of', 'commensal', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'summarize', 'the', 'current', 'understanding', 'of', 'commensal', 'Neisseria', 'as', 'an', 'AMR', 'reservoir', ',', 'and', 'call', 'to', 'increase', 'research', 'on', 'commensal', 'Neisseria', 'species', ',', 'through', 'expanding', 'established', 'gonococcal', 'surveillance', 'programs', 'to', 'include', 'the', 'collection', ',', 'isolation', ',', 'antimicrobial', 'resistance', 'phenotyping', ',', 'and', 'whole', '-', 'genome', 'sequencing', '(', 'WGS', ')', 'of', 'commensal', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O']\n",
            "['Vaccines', 'against', 'infectious', 'agents', 'such', 'as', 'Streptococcus', 'pneumoniae', 'and', 'Haemophilus', 'influenzae', 'have', 'already', 'been', 'shown', 'to', 'reduce', 'tolerance', 'to', 'antimicrobial', 'medications', ';', 'however', ',', 'vaccines', 'against', 'some', 'antimicrobial', '-', 'resistant', 'pathogens', 'such', 'as', 'Vibrio', 'cholerae', ',', 'Salmonella', 'typhi', ',', 'Escherichia', 'coli', ',', 'nosocomial', 'infections', ',', 'and', 'pulmonary', 'and', 'diarrheal', 'disease', 'viruses', 'require', 'more', 'research', 'and', 'development', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Vaccines', 'against', 'infectious', 'agents', 'such', 'as', 'Streptococcus', 'pneumoniae', 'and', 'Haemophilus', 'influenzae', 'have', 'already', 'been', 'shown', 'to', 'reduce', 'tolerance', 'to', 'antimicrobial', 'medications', ';', 'however', ',', 'vaccines', 'against', 'some', 'antimicrobial', '-', 'resistant', 'pathogens', 'such', 'as', 'Vibrio', 'cholerae', ',', 'Salmonella', 'typhi', ',', 'Escherichia', 'coli', ',', 'nosocomial', 'infections', ',', 'and', 'pulmonary', 'and', 'diarrheal', 'disease', 'viruses', 'require', 'more', 'research', 'and', 'development', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Vaccines', 'against', 'infectious', 'agents', 'such', 'as', 'Streptococcus', 'pneumoniae', 'and', 'Haemophilus', 'influenzae', 'have', 'already', 'been', 'shown', 'to', 'reduce', 'tolerance', 'to', 'antimicrobial', 'medications', ';', 'however', ',', 'vaccines', 'against', 'some', 'antimicrobial', '-', 'resistant', 'pathogens', 'such', 'as', 'Vibrio', 'cholerae', ',', 'Salmonella', 'typhi', ',', 'Escherichia', 'coli', ',', 'nosocomial', 'infections', ',', 'and', 'pulmonary', 'and', 'diarrheal', 'disease', 'viruses', 'require', 'more', 'research', 'and', 'development', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Vaccines', 'against', 'infectious', 'agents', 'such', 'as', 'Streptococcus', 'pneumoniae', 'and', 'Haemophilus', 'influenzae', 'have', 'already', 'been', 'shown', 'to', 'reduce', 'tolerance', 'to', 'antimicrobial', 'medications', ';', 'however', ',', 'vaccines', 'against', 'some', 'antimicrobial', '-', 'resistant', 'pathogens', 'such', 'as', 'Vibrio', 'cholerae', ',', 'Salmonella', 'typhi', ',', 'Escherichia', 'coli', ',', 'nosocomial', 'infections', ',', 'and', 'pulmonary', 'and', 'diarrheal', 'disease', 'viruses', 'require', 'more', 'research', 'and', 'development', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Vaccines', 'against', 'infectious', 'agents', 'such', 'as', 'Streptococcus', 'pneumoniae', 'and', 'Haemophilus', 'influenzae', 'have', 'already', 'been', 'shown', 'to', 'reduce', 'tolerance', 'to', 'antimicrobial', 'medications', ';', 'however', ',', 'vaccines', 'against', 'some', 'antimicrobial', '-', 'resistant', 'pathogens', 'such', 'as', 'Vibrio', 'cholerae', ',', 'Salmonella', 'typhi', ',', 'Escherichia', 'coli', ',', 'nosocomial', 'infections', ',', 'and', 'pulmonary', 'and', 'diarrheal', 'disease', 'viruses', 'require', 'more', 'research', 'and', 'development', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Introduction', ':', 'Antimicrobial', 'resistance', '(', 'AMR', ')', 'played', 'an', 'important', 'role', 'in', 'the', 'initial', 'outbreaks', 'of', 'Clostridium', 'difficile', 'infection', '(', 'CDI', ')', 'in', 'the', '1970s', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Areas', 'covered', ':', 'This', 'narrative', 'review', 'summarizes', 'the', 'prevalence', 'and', 'mechanisms', 'of', 'AMR', 'in', 'C.', 'difficile', 'RT', '017', 'and', 'transmission', 'of', 'these', 'AMR', 'mechanisms', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Areas', 'covered', ':', 'This', 'narrative', 'review', 'summarizes', 'the', 'prevalence', 'and', 'mechanisms', 'of', 'AMR', 'in', 'C.', 'difficile', 'RT', '017', 'and', 'transmission', 'of', 'these', 'AMR', 'mechanisms', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Commensal', 'Neisseria', 'and', 'pathogenic', 'Neisseria', 'frequently', 'share', 'DNA', 'through', 'horizontal', 'gene', 'transfer', ',', 'which', 'has', 'played', 'a', 'large', 'role', 'in', 'rendering', 'antibiotic', 'therapies', 'ineffective', 'in', 'pathogenic', 'Neisseria', 'populations', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'German', 'Antibiotic', 'Resistance', 'Strategy', '(', 'DART', '2020', ')', 'is', 'based', 'on', 'the', 'main', 'objectives', 'of', 'the', 'global', 'action', 'plan', 'and', 'was', 'revised', 'and', 'published', 'in', '2015', '.']\n",
            "['O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'unique', 'genetic', 'makeup', 'and', 'remarkable', 'competency', 'of', 'Vibrio', 'cholerae', 'are', 'the', 'key', 'factors', 'that', 'help', 'the', 'cholera', 'pathogen', 'adapt', 'rapidly', 'to', 'adverse', 'environmental', 'conditions', 'and', 'resist', 'the', 'detrimental', 'effect', 'of', 'antimicrobial', 'agents', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'the', 'last', 'few', 'decades', ',', 'V.', 'cholerae', 'that', 'causes', 'acute', 'watery', 'diarrhoeal', 'disease', 'cholera', 'has', 'emerged', 'as', 'a', 'notorious', 'multidrug', 'resistant', '(', 'MDR', ')', 'enteric', 'pathogen', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Others', 'O', 'O', 'O', 'O']\n",
            "['In', 'the', 'last', 'few', 'decades', ',', 'V.', 'cholerae', 'that', 'causes', 'acute', 'watery', 'diarrhoeal', 'disease', 'cholera', 'has', 'emerged', 'as', 'a', 'notorious', 'multidrug', 'resistant', '(', 'MDR', ')', 'enteric', 'pathogen', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'government', 'of', 'Japan', 'established', 'its', 'own', 'National', 'Action', 'Plan', 'on', 'AMR', 'in', '2016', ';', 'however', ',', 'Japan', \"'s\", 'AMR', 'countermeasures', 'are', 'still', 'in', 'the', 'developmental', 'stage', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Different', 'antimicrobial', 'resistance', 'genes', 'identified', 'in', 'V.', 'cholerae', 'can', 'contribute', 'in', 'antibiotic', 'resistance', 'by', 'facilitating', 'one', 'of', 'the', 'following', 'three', 'mechanisms', ';', '(', 'i', ')', 'reduced', 'permeability', 'or', 'active', 'efflux', 'of', 'the', 'antibiotics', ',', '(', 'ii', ')', 'alteration', 'of', 'the', 'antibiotic', 'targets', 'by', 'introducing', 'post', '-', 'transcriptional', '/', 'translational', 'modifications', 'and', '(', 'iii', ')', 'hydrolysis', 'or', 'chemical', 'modification', 'of', 'antibiotics', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Although', 'chromosomal', 'mutations', 'can', 'contribute', 'to', 'antimicrobial', 'resistance', '(', 'AMR', ')', ',', 'the', 'frequent', 'acquisition', 'of', 'extrachromosomal', 'mobile', 'genetic', 'elements', '(', 'MGEs', ')', 'from', 'closely', '/', 'distantly', 'related', 'bacterial', 'species', 'are', 'major', 'players', 'in', 'V.', 'cholerae', 'drug', 'resistance', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O']\n",
            "['Geographical', 'trends', 'showed', 'prominence', 'of', 'Multidrug', '-', 'resistant', 'tuberculosis', '(', 'MDR', '-', 'TB', ')', 'in', 'the', 'WHO', 'African', 'Region', ',', 'corresponding', 'with', 'the', 'MDR', '-', 'TB', 'burden', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Geographical', 'trends', 'showed', 'prominence', 'of', 'Multidrug', '-', 'resistant', 'tuberculosis', '(', 'MDR', '-', 'TB', ')', 'in', 'the', 'WHO', 'African', 'Region', ',', 'corresponding', 'with', 'the', 'MDR', '-', 'TB', 'burden', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'highest', 'priority', 'ARGs', 'bla', 'CTX', '-M', ',', 'bla', 'VIM', ',', 'bla', 'NDM', ',', 'bla', 'OXA', '-48', '-', 'like', ',', 'bla', 'OXA', '-23', ',', 'mcr', ',', 'armA', ',', 'vanA', ',', 'cfr', 'and', 'optrA', 'were', 'reported', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O']\n",
            "['Whole', '-', 'genome', 'sequencing', '(', 'WGS', ')', 'and', ',', 'more', 'recently', ',', 'metagenomic', 'analyses', 'have', 'greatly', 'enhanced', 'our', 'understanding', 'of', 'these', 'processes', ',', 'and', 'these', 'approaches', 'are', 'informing', 'mitigation', 'strategies', 'for', 'how', 'we', 'better', 'understand', 'and', 'control', 'AMR', '.']\n",
            "['B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Potential', 'AMR', '-', 'specific', 'interventions', 'are', 'in', 'the', 'early', 'stages', 'of', 'development', '.', 'Many', 'data', 'gaps', 'relating', 'to', 'sources', 'and', 'relevance', 'of', 'transmission', 'routes', ',', 'diversity', 'of', 'ARB', 'and', 'ARGs', ',', 'effectiveness', 'of', 'mitigation', 'measures', 'were', 'identified', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'explores', 'how', 'WGS', 'techniques', 'have', 'advanced', 'global', ',', 'national', ',', 'and', 'local', 'AMR', 'surveillance', ',', 'and', 'how', 'this', 'improved', 'understanding', 'is', 'being', 'applied', 'to', 'inform', 'solutions', ',', 'such', 'as', 'novel', 'diagnostic', 'methods', 'that', 'allow', 'antimicrobial', 'use', 'to', 'be', 'optimised', 'and', 'vaccination', 'strategies', 'for', 'better', 'controlling', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Therefore', ',', 'this', 'study', 'uses', 'topic', 'modelling', ',', 'a', 'machine', 'learning', 'approach', ',', 'to', 'reveal', 'the', 'scientific', 'evolution', 'of', 'AMR', 'research', 'and', 'its', 'trends', ',', 'and', 'provides', 'an', 'interactive', 'user', 'interface', 'for', 'further', 'analyses', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Therefore', ',', 'this', 'study', 'uses', 'topic', 'modelling', ',', 'a', 'machine', 'learning', 'approach', ',', 'to', 'reveal', 'the', 'scientific', 'evolution', 'of', 'AMR', 'research', 'and', 'its', 'trends', ',', 'and', 'provides', 'an', 'interactive', 'user', 'interface', 'for', 'further', 'analyses', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Methods', ':', 'Structural', 'topic', 'modelling', '(', 'STM', ')', 'was', 'applied', 'on', 'a', 'text', 'corpus', 'resulting', 'from', 'a', 'PubMed', 'query', 'comprising', 'AMR', 'articles', '(', '1999', '-', '2018', ')', '.']\n",
            "['O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Several', 'ARB', 'of', 'highest', 'priority', 'for', 'public', 'health', ',', 'such', 'as', 'carbapenem', 'or', 'extended', '-', 'spectrum', 'cephalosporin', 'and/or', 'fluoroquinolone', '-', 'resistant', 'Enterobacterales', '(', 'including', 'Salmonella', 'enterica', ')', ',', 'fluoroquinolone', '-', 'resistant', 'Campylobacter', 'spp', '.', ',', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', 'and', 'glycopeptide', '-', 'resistant', 'Enterococcus', 'faecium', 'and', 'E.', 'faecalis', 'were', 'identified', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Several', 'ARB', 'of', 'highest', 'priority', 'for', 'public', 'health', ',', 'such', 'as', 'carbapenem', 'or', 'extended', '-', 'spectrum', 'cephalosporin', 'and/or', 'fluoroquinolone', '-', 'resistant', 'Enterobacterales', '(', 'including', 'Salmonella', 'enterica', ')', ',', 'fluoroquinolone', '-', 'resistant', 'Campylobacter', 'spp', '.', ',', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', 'and', 'glycopeptide', '-', 'resistant', 'Enterococcus', 'faecium', 'and', 'E.', 'faecalis', 'were', 'identified', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Several', 'ARB', 'of', 'highest', 'priority', 'for', 'public', 'health', ',', 'such', 'as', 'carbapenem', 'or', 'extended', '-', 'spectrum', 'cephalosporin', 'and/or', 'fluoroquinolone', '-', 'resistant', 'Enterobacterales', '(', 'including', 'Salmonella', 'enterica', ')', ',', 'fluoroquinolone', '-', 'resistant', 'Campylobacter', 'spp', '.', ',', 'methicillin', '-', 'resistant', 'Staphylococcus', 'aureus', 'and', 'glycopeptide', '-', 'resistant', 'Enterococcus', 'faecium', 'and', 'E.', 'faecalis', 'were', 'identified', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['NCBI', 'and', 'European', 'Nucleotide', 'Archive', '(', 'ENA', ')', 'provide', 'possibilities', 'for', 'online', 'submission', 'of', 'both', 'sequencing', 'and', 'accompanying', 'phenotypic', 'antimicrobial', 'susceptibility', 'data', ',', 'allowing', 'for', 'other', 'researchers', 'to', 'further', 'analyze', 'data', ',', 'and', 'develop', 'and', 'test', 'new', 'tools', '.']\n",
            "['U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['NCBI', 'and', 'European', 'Nucleotide', 'Archive', '(', 'ENA', ')', 'provide', 'possibilities', 'for', 'online', 'submission', 'of', 'both', 'sequencing', 'and', 'accompanying', 'phenotypic', 'antimicrobial', 'susceptibility', 'data', ',', 'allowing', 'for', 'other', 'researchers', 'to', 'further', 'analyze', 'data', ',', 'and', 'develop', 'and', 'test', 'new', 'tools', '.']\n",
            "['O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'conclude', 'by', 'suggesting', 'that', 'priorities', 'be', 'expanded', 'to', 'include', 'greater', 'consideration', 'of', 'host', '-', 'microbial', 'interactions', 'that', 'dictate', 'AMR', ',', 'as', 'well', 'as', 'anthropogenic', 'and', 'environmental', 'systems', 'that', 'promote', 'dissemination', 'of', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'conclude', 'by', 'suggesting', 'that', 'priorities', 'be', 'expanded', 'to', 'include', 'greater', 'consideration', 'of', 'host', '-', 'microbial', 'interactions', 'that', 'dictate', 'AMR', ',', 'as', 'well', 'as', 'anthropogenic', 'and', 'environmental', 'systems', 'that', 'promote', 'dissemination', 'of', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'overall', 'presence', 'of', 'all', 'resistance', 'genes', 'in', 'a', 'specific', 'environment', 'is', 'defined', 'as', 'a', 'resistome', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O']\n",
            "['Traditional', 'themes', 'such', 'as', 'in', 'vitro', 'antibiotic', 'susceptibility', 'testing', 'and', 'its', 'clinical', 'value', ',', 'AMR', 'evolution', 'in', 'persistent', 'Pseudomonas', 'aeruginosa', 'infection', 'and', 'the', 'impact', 'of', 'biofilm', 'on', 'AMR', 'were', 'discussed', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'recent', 'advancements', 'in', 'rapid', 'and', 'affordable', 'DNA', 'sequencing', 'technologies', 'have', 'revolutionized', 'diagnostic', 'microbiology', 'and', 'microbial', 'surveillance', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'recent', 'advancements', 'in', 'rapid', 'and', 'affordable', 'DNA', 'sequencing', 'technologies', 'have', 'revolutionized', 'diagnostic', 'microbiology', 'and', 'microbial', 'surveillance', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O']\n",
            "['Geographical', 'variation', 'in', 'prevalence', 'of', 'MDROs', 'is', 'substantial', ',', 'and', 'local', 'epidemiology', 'should', 'be', 'considered', 'to', 'assess', 'the', 'current', 'threat', 'of', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['At', 'least', '47', 'freely', 'accessible', 'bioinformatics', 'resources', 'for', 'detection', 'of', 'AMR', 'determinants', 'in', 'DNA', 'or', 'amino', 'acid', 'sequence', 'data', 'have', 'been', 'developed', 'to', 'date', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['At', 'least', '47', 'freely', 'accessible', 'bioinformatics', 'resources', 'for', 'detection', 'of', 'AMR', 'determinants', 'in', 'DNA', 'or', 'amino', 'acid', 'sequence', 'data', 'have', 'been', 'developed', 'to', 'date', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'include', ',', 'among', 'others', 'but', 'not', 'limited', 'to', ',', 'ARG', '-', 'ANNOT', ',', 'CARD', ',', 'SRST2', ',', 'MEGARes', ',', 'Genefinder', ',', 'ARIBA', ',', 'KmerResistance', ',', 'AMRFinder', ',', 'and', 'ResFinder', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'include', ',', 'among', 'others', 'but', 'not', 'limited', 'to', ',', 'ARG', '-', 'ANNOT', ',', 'CARD', ',', 'SRST2', ',', 'MEGARes', ',', 'Genefinder', ',', 'ARIBA', ',', 'KmerResistance', ',', 'AMRFinder', ',', 'and', 'ResFinder', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'include', ',', 'among', 'others', 'but', 'not', 'limited', 'to', ',', 'ARG', '-', 'ANNOT', ',', 'CARD', ',', 'SRST2', ',', 'MEGARes', ',', 'Genefinder', ',', 'ARIBA', ',', 'KmerResistance', ',', 'AMRFinder', ',', 'and', 'ResFinder', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'include', ',', 'among', 'others', 'but', 'not', 'limited', 'to', ',', 'ARG', '-', 'ANNOT', ',', 'CARD', ',', 'SRST2', ',', 'MEGARes', ',', 'Genefinder', ',', 'ARIBA', ',', 'KmerResistance', ',', 'AMRFinder', ',', 'and', 'ResFinder', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'include', ',', 'among', 'others', 'but', 'not', 'limited', 'to', ',', 'ARG', '-', 'ANNOT', ',', 'CARD', ',', 'SRST2', ',', 'MEGARes', ',', 'Genefinder', ',', 'ARIBA', ',', 'KmerResistance', ',', 'AMRFinder', ',', 'and', 'ResFinder', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'include', ',', 'among', 'others', 'but', 'not', 'limited', 'to', ',', 'ARG', '-', 'ANNOT', ',', 'CARD', ',', 'SRST2', ',', 'MEGARes', ',', 'Genefinder', ',', 'ARIBA', ',', 'KmerResistance', ',', 'AMRFinder', ',', 'and', 'ResFinder', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'include', ',', 'among', 'others', 'but', 'not', 'limited', 'to', ',', 'ARG', '-', 'ANNOT', ',', 'CARD', ',', 'SRST2', ',', 'MEGARes', ',', 'Genefinder', ',', 'ARIBA', ',', 'KmerResistance', ',', 'AMRFinder', ',', 'and', 'ResFinder', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'include', ',', 'among', 'others', 'but', 'not', 'limited', 'to', ',', 'ARG', '-', 'ANNOT', ',', 'CARD', ',', 'SRST2', ',', 'MEGARes', ',', 'Genefinder', ',', 'ARIBA', ',', 'KmerResistance', ',', 'AMRFinder', ',', 'and', 'ResFinder', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O']\n",
            "['These', 'include', ',', 'among', 'others', 'but', 'not', 'limited', 'to', ',', 'ARG', '-', 'ANNOT', ',', 'CARD', ',', 'SRST2', ',', 'MEGARes', ',', 'Genefinder', ',', 'ARIBA', ',', 'KmerResistance', ',', 'AMRFinder', ',', 'and', 'ResFinder', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O']\n",
            "['Streptococcus', 'canis', 'may', 'be', 'regarded', 'as', 'a', 'neglected', 'pathogen', 'of', 'one', 'health', 'concern', '.', 'Further', 'research', 'is', 'needed', 'for', 'its', 'better', 'understanding', 'and', 'effective', 'control', '.']\n",
            "['B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'UN', 'General', 'Assembly', 'has', 'listed', 'AMR', 'among', 'issues', 'that', 'need', 'urgent', 'address', ';', 'subsequently', 'FAO', '/', 'OIE', '/', 'WHO', 'tripartite', 'took', 'lead', 'and', 'jointly', 'developed', 'Global', 'Action', 'Plan', '(', 'GAP', ')', 'to', 'tackle', 'the', 'problem', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'UN', 'General', 'Assembly', 'has', 'listed', 'AMR', 'among', 'issues', 'that', 'need', 'urgent', 'address', ';', 'subsequently', 'FAO', '/', 'OIE', '/', 'WHO', 'tripartite', 'took', 'lead', 'and', 'jointly', 'developed', 'Global', 'Action', 'Plan', '(', 'GAP', ')', 'to', 'tackle', 'the', 'problem', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'the', 'social', 'science', 'domain', ',', 'institution', 'and', 'policy', 'elements', 'are', 'most', 'frequently', 'covered', ',', 'while', 'transformations', 'are', 'the', 'least', 'covered', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O']\n",
            "['Streptococcus', 'canis', 'is', 'a', 'multi', '-', 'host', 'pathogen', 'that', 'causes', 'disease', 'of', 'varying', 'severity', 'in', 'a', 'wide', 'range', 'of', 'mammals', ',', 'including', 'humans', '.']\n",
            "['B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Dogs', 'and', 'cats', 'appear', 'to', 'be', 'the', 'primary', 'hosts', 'and', 'may', 'play', 'a', 'role', 'in', 'transmitting', 'infection', 'to', 'humans', '.', 'The', 'broader', 'epidemiology', 'of', 'S.', 'canis', ',', 'however', ',', 'is', 'still', 'poorly', 'understood', ',', 'as', 'are', 'its', 'virulence', 'mechanisms', ',', 'antimicrobial', 'resistance', '(', 'AMR', ')', 'and', 'population', 'structure', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Natural', 'products', 'such', 'as', 'plant', 'extracts', 'and', 'essential', 'oils', '(', 'EOs', ')', 'have', 'been', 'viewed', 'as', 'potential', 'candidates', 'to', 'combat', 'AMR', 'due', 'to', 'their', 'complex', 'chemistry', 'that', 'carries', 'inherent', 'pro', '-', 'oxidant', 'and', 'antioxidant', 'properties', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Others', 'O', 'O']\n",
            "['Natural', 'products', 'such', 'as', 'plant', 'extracts', 'and', 'essential', 'oils', '(', 'EOs', ')', 'have', 'been', 'viewed', 'as', 'potential', 'candidates', 'to', 'combat', 'AMR', 'due', 'to', 'their', 'complex', 'chemistry', 'that', 'carries', 'inherent', 'pro', '-', 'oxidant', 'and', 'antioxidant', 'properties', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O']\n",
            "['Natural', 'products', 'such', 'as', 'plant', 'extracts', 'and', 'essential', 'oils', '(', 'EOs', ')', 'have', 'been', 'viewed', 'as', 'potential', 'candidates', 'to', 'combat', 'AMR', 'due', 'to', 'their', 'complex', 'chemistry', 'that', 'carries', 'inherent', 'pro', '-', 'oxidant', 'and', 'antioxidant', 'properties', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'describe', 'the', 'main', 'ecological', 'niches', 'of', 'S.', 'canis', 'in', 'companion', 'animals', ',', 'discuss', 'potential', 'risk', 'factors', 'for', 'infection', 'in', 'humans', 'and', 'propose', 'a', 'multi', '-', 'host', 'transmission', 'cycle', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O']\n",
            "['We', 'describe', 'the', 'main', 'ecological', 'niches', 'of', 'S.', 'canis', 'in', 'companion', 'animals', ',', 'discuss', 'potential', 'risk', 'factors', 'for', 'infection', 'in', 'humans', 'and', 'propose', 'a', 'multi', '-', 'host', 'transmission', 'cycle', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'show', 'that', 'current', 'knowledge', 'on', 'S.', 'canis', 'virulence', 'determinants', 'is', 'limited', 'and', 'sometimes', 'contradictory', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'illustrate', 'the', 'different', 'typing', 'systems', 'proposed', 'to', 'classify', 'S.', 'canis', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O']\n",
            "['States', 'and', 'Governments', 'have', 'developed', 'AMR', 'National', 'Action', 'Plans', '(', 'NAP', ')', ',', 'Policies', 'and', 'Communication', 'Strategies', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Classical', 'approaches', 'for', 'AMR', 'diagnostics', 'like', 'phenotypic', 'methods', ',', 'biochemical', 'assays', ',', 'and', 'molecular', 'techniques', 'are', 'cumbersome', 'and', 'resource', '-', 'intensive', 'and', 'involve', 'a', 'long', 'turnaround', 'time', 'to', 'yield', 'confirmatory', 'results', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Classical', 'approaches', 'for', 'AMR', 'diagnostics', 'like', 'phenotypic', 'methods', ',', 'biochemical', 'assays', ',', 'and', 'molecular', 'techniques', 'are', 'cumbersome', 'and', 'resource', '-', 'intensive', 'and', 'involve', 'a', 'long', 'turnaround', 'time', 'to', 'yield', 'confirmatory', 'results', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Streptococcus', 'pneumoniae', 'is', 'a', 'major', 'cause', 'of', 'pneumonia', ',', 'meningitis', ',', 'and', 'septicaemia', 'worldwide', '.']\n",
            "['B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Here', ',', 'we', 'discuss', 'a', 'variety', 'of', 'evolutionary', 'mechanisms', 'at', 'the', 'host', ',', 'pathogen', ',', 'and', 'environmental', 'levels', 'that', 'may', 'contribute', 'to', 'changes', 'in', 'the', 'prevalence', 'of', 'pneumococcal', 'AMR', 'in', 'the', 'post', '-', 'vaccination', 'era', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'Food', 'and', 'Agriculture', 'Organization', '(', 'FAO', ')', 'of', 'the', 'United', 'Nations', 'plays', 'an', 'important', 'role', 'in', 'promoting', 'and', 'facilitating', 'antimicrobial', 'stewardship', '.']\n",
            "['B-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'relative', 'importance', 'of', 'these', 'factors', 'may', 'vary', 'by', 'population', ',', 'pneumococcal', 'lineage', ',', 'geography', ',', 'and', 'time', ',', 'leading', 'to', 'the', 'complex', 'relationship', 'between', 'vaccination', ',', 'antibiotic', 'use', ',', 'and', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'relative', 'importance', 'of', 'these', 'factors', 'may', 'vary', 'by', 'population', ',', 'pneumococcal', 'lineage', ',', 'geography', ',', 'and', 'time', ',', 'leading', 'to', 'the', 'complex', 'relationship', 'between', 'vaccination', ',', 'antibiotic', 'use', ',', 'and', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Antimicrobial', 'resistance', '(', 'AMR', ')', 'has', 'become', 'a', 'major', 'concern', 'for', 'healthcare', 'workers', 'due', 'to', 'the', 'emergence', 'of', 'new', 'variants', 'of', 'resistant', 'markers', ',', 'especially', 'carbapenemases', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O']\n",
            "['Combinational', 'antibiotic', 'therapy', 'is', 'one', 'of', 'the', 'best', 'and', 'easiest', 'approaches', 'to', 'handle', 'the', 'current', 'situation', 'of', 'AMR', '.', 'Although', 'some', 'antibiotic', 'combinations', 'are', 'already', 'in', 'clinical', 'use', ',', 'they', 'remain', 'to', 'be', 'studied', 'in', 'detail', '.']\n",
            "['B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['For', 'example', ',', 'FAO', 'is', 'developing', 'a', 'platform', 'to', 'collect', 'data', 'on', 'AMR', 'in', 'animals', 'and', 'antimicrobial', 'use', '(', 'AMU', ')', 'in', 'plants', '(', 'InFARM', ')', ',', 'working', 'on', 'a', 'campaign', 'to', 'reduce', 'the', 'need', 'to', 'use', 'antimicrobials', ',', 'studying', 'the', 'use', 'of', 'alternatives', 'to', 'the', 'use', 'of', 'antimicrobials', '(', 'especially', 'those', 'used', 'for', 'growth', 'promotion', ')', 'and', 'actively', 'promoting', 'the', 'implementation', 'of', 'the', 'Codex', 'Alimentarius', 'AMR', 'standards', '.']\n",
            "['O', 'O', 'O', 'U-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Currently', ',', 'no', 'recognized', 'effective', 'treatments', 'are', 'available', ',', 'especially', 'for', 'chronic', 'antibody', '-', 'mediated', 'rejection', '(', 'CAMR', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O']\n",
            "['Donor', '-', 'specific', 'antibodies', '(', 'DSAs', ')', 'secreted', 'by', 'long', '-', 'lived', 'plasma', 'cells', 'and', 'memory', 'B', 'cells', 'are', 'acknowledged', 'as', 'biomarkers', 'of', 'AMR', '.']\n",
            "['B-Gene/Proteins', 'I-Gene/Proteins', 'I-Gene/Proteins', 'I-Gene/Proteins', 'I-Gene/Proteins', 'I-Gene/Proteins', 'L-Gene/Proteins', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Nevertheless', ',', 'it', 'may', 'be', 'too', 'late', 'for', 'the', 'DSA', 'routine', 'examination', 'production', 'since', 'DSAs', 'may', 'have', 'binded', 'to', 'graft', 'vascular', 'endothelial', 'cells', 'through', 'complement', '-', 'dependent', 'or', 'complement', '-', 'independent', 'pathways', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Nevertheless', ',', 'it', 'may', 'be', 'too', 'late', 'for', 'the', 'DSA', 'routine', 'examination', 'production', 'since', 'DSAs', 'may', 'have', 'binded', 'to', 'graft', 'vascular', 'endothelial', 'cells', 'through', 'complement', '-', 'dependent', 'or', 'complement', '-', 'independent', 'pathways', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Nevertheless', ',', 'it', 'may', 'be', 'too', 'late', 'for', 'the', 'DSA', 'routine', 'examination', 'production', 'since', 'DSAs', 'may', 'have', 'binded', 'to', 'graft', 'vascular', 'endothelial', 'cells', 'through', 'complement', '-', 'dependent', 'or', 'complement', '-', 'independent', 'pathways', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O']\n",
            "['Escherichia', 'coli', 'was', 'the', 'most', 'commonly', 'described', 'organism', ',', 'with', 'median', 'resistance', 'rates', 'from', 'human', 'isolates', 'of', '92.8', '%', '(', 'n=6', 'articles', ')', ',', '46.4', '%', '(', 'n=4', ')', ',', '55.4', '%', '(', 'n=8', ')', ',', 'and', '46.4', '%', '(', 'n=5', ')', 'to', 'ampicillin', ',', '3rd', 'generation', 'cephalosporins', ',', 'fluoroquinolones', ',', 'and', 'gentamicin', 'respectively', '.']\n",
            "['B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Escherichia', 'coli', 'was', 'the', 'most', 'commonly', 'described', 'organism', ',', 'with', 'median', 'resistance', 'rates', 'from', 'human', 'isolates', 'of', '92.8', '%', '(', 'n=6', 'articles', ')', ',', '46.4', '%', '(', 'n=4', ')', ',', '55.4', '%', '(', 'n=8', ')', ',', 'and', '46.4', '%', '(', 'n=5', ')', 'to', 'ampicillin', ',', '3rd', 'generation', 'cephalosporins', ',', 'fluoroquinolones', ',', 'and', 'gentamicin', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Escherichia', 'coli', 'was', 'the', 'most', 'commonly', 'described', 'organism', ',', 'with', 'median', 'resistance', 'rates', 'from', 'human', 'isolates', 'of', '92.8', '%', '(', 'n=6', 'articles', ')', ',', '46.4', '%', '(', 'n=4', ')', ',', '55.4', '%', '(', 'n=8', ')', ',', 'and', '46.4', '%', '(', 'n=5', ')', 'to', 'ampicillin', ',', '3rd', 'generation', 'cephalosporins', ',', 'fluoroquinolones', ',', 'and', 'gentamicin', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Escherichia', 'coli', 'was', 'the', 'most', 'commonly', 'described', 'organism', ',', 'with', 'median', 'resistance', 'rates', 'from', 'human', 'isolates', 'of', '92.8', '%', '(', 'n=6', 'articles', ')', ',', '46.4', '%', '(', 'n=4', ')', ',', '55.4', '%', '(', 'n=8', ')', ',', 'and', '46.4', '%', '(', 'n=5', ')', 'to', 'ampicillin', ',', '3rd', 'generation', 'cephalosporins', ',', 'fluoroquinolones', ',', 'and', 'gentamicin', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O']\n",
            "['Escherichia', 'coli', 'was', 'the', 'most', 'commonly', 'described', 'organism', ',', 'with', 'median', 'resistance', 'rates', 'from', 'human', 'isolates', 'of', '92.8', '%', '(', 'n=6', 'articles', ')', ',', '46.4', '%', '(', 'n=4', ')', ',', '55.4', '%', '(', 'n=8', ')', ',', 'and', '46.4', '%', '(', 'n=5', ')', 'to', 'ampicillin', ',', '3rd', 'generation', 'cephalosporins', ',', 'fluoroquinolones', ',', 'and', 'gentamicin', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O']\n",
            "['Conclusions', ':', 'Whilst', 'resistance', 'rates', 'are', 'high', 'for', 'several', 'GLASS', 'organisms', ',', 'there', 'were', 'insufficient', 'data', 'to', 'draw', 'robust', 'conclusions', 'about', 'the', 'AMR', 'situation', 'in', 'Cambodia', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'non', '-', 'antibiotic', 'strategies', ',', 'vaccines', 'and', 'therapeutic', 'monoclonal', 'antibodies', '(', 'mAbs', ')', 'play', 'a', 'strategic', 'role', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Gene/Proteins', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'era', 'of', 'antimicrobial', 'therapy', 'started', 'in', 'the', 'nineteenth', 'century', 'with', 'the', 'identification', 'of', 'chemical', 'compounds', 'with', 'antimicrobial', 'properties', '.']\n",
            "['O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'the', 'current', 'review', 'we', 'therefore', 'propose', 'using', 'antimicrobial', 'resistant', 'Escherichia', 'coli', 'as', 'indicators', 'for', 'monitoring', 'occurrence', 'and', 'levels', 'of', 'AMR', 'in', 'the', 'environment', ',', 'including', 'wildlife', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'we', 'discuss', 'mechanisms', 'of', 'resistance', 'identified', 'in', 'bacterial', 'agents', 'Staphylococcus', 'aureus', 'and', 'the', 'enterococci', 'towards', 'two', 'priority', 'classes', 'of', 'antibiotics', '-', 'the', 'fluoroquinolones', 'and', 'the', 'glycopeptides', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'we', 'discuss', 'mechanisms', 'of', 'resistance', 'identified', 'in', 'bacterial', 'agents', 'Staphylococcus', 'aureus', 'and', 'the', 'enterococci', 'towards', 'two', 'priority', 'classes', 'of', 'antibiotics', '-', 'the', 'fluoroquinolones', 'and', 'the', 'glycopeptides', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"In this review, we discuss mechanisms of resistanc...\" with entities \"[(112, 124, 'Microbes')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['In', 'this', 'review', ',', 'we', 'discuss', 'mechanisms', 'of', 'resistance', 'identified', 'in', 'bacterial', 'agents', 'Staphylococcus', 'aureus', 'and', 'the', 'enterococci', 'towards', 'two', 'priority', 'classes', 'of', 'antibiotics', '-', 'the', 'fluoroquinolones', 'and', 'the', 'glycopeptides', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'we', 'discuss', 'mechanisms', 'of', 'resistance', 'identified', 'in', 'bacterial', 'agents', 'Staphylococcus', 'aureus', 'and', 'the', 'enterococci', 'towards', 'two', 'priority', 'classes', 'of', 'antibiotics', '-', 'the', 'fluoroquinolones', 'and', 'the', 'glycopeptides', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'L-Antibiotics']\n",
            "['In', 'February', '2017', ',', 'in', 'light', 'of', 'increasing', 'antibiotic', 'resistance', ',', 'the', 'WHO', 'published', 'a', 'list', 'of', 'pathogens', 'that', 'includes', 'the', 'pathogens', 'designated', 'by', 'the', 'acronym', 'ESKAPE', '(', 'Enterococcus', 'faecium', ',', 'Staphylococcus', 'aureus', ',', 'Klebsiella', 'pneumoniae', ',', 'Acinetobacter', 'baumannii', ',', 'Pseudomonas', 'aeruginosa', ',', 'and', 'Enterobacter', 'species', ')', 'to', 'which', 'were', 'given', 'the', 'highest', '\"', 'priority', 'status', '\"', 'since', 'they', 'represent', 'the', 'great', 'threat', 'to', 'humans', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'February', '2017', ',', 'in', 'light', 'of', 'increasing', 'antibiotic', 'resistance', ',', 'the', 'WHO', 'published', 'a', 'list', 'of', 'pathogens', 'that', 'includes', 'the', 'pathogens', 'designated', 'by', 'the', 'acronym', 'ESKAPE', '(', 'Enterococcus', 'faecium', ',', 'Staphylococcus', 'aureus', ',', 'Klebsiella', 'pneumoniae', ',', 'Acinetobacter', 'baumannii', ',', 'Pseudomonas', 'aeruginosa', ',', 'and', 'Enterobacter', 'species', ')', 'to', 'which', 'were', 'given', 'the', 'highest', '\"', 'priority', 'status', '\"', 'since', 'they', 'represent', 'the', 'great', 'threat', 'to', 'humans', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'February', '2017', ',', 'in', 'light', 'of', 'increasing', 'antibiotic', 'resistance', ',', 'the', 'WHO', 'published', 'a', 'list', 'of', 'pathogens', 'that', 'includes', 'the', 'pathogens', 'designated', 'by', 'the', 'acronym', 'ESKAPE', '(', 'Enterococcus', 'faecium', ',', 'Staphylococcus', 'aureus', ',', 'Klebsiella', 'pneumoniae', ',', 'Acinetobacter', 'baumannii', ',', 'Pseudomonas', 'aeruginosa', ',', 'and', 'Enterobacter', 'species', ')', 'to', 'which', 'were', 'given', 'the', 'highest', '\"', 'priority', 'status', '\"', 'since', 'they', 'represent', 'the', 'great', 'threat', 'to', 'humans', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'February', '2017', ',', 'in', 'light', 'of', 'increasing', 'antibiotic', 'resistance', ',', 'the', 'WHO', 'published', 'a', 'list', 'of', 'pathogens', 'that', 'includes', 'the', 'pathogens', 'designated', 'by', 'the', 'acronym', 'ESKAPE', '(', 'Enterococcus', 'faecium', ',', 'Staphylococcus', 'aureus', ',', 'Klebsiella', 'pneumoniae', ',', 'Acinetobacter', 'baumannii', ',', 'Pseudomonas', 'aeruginosa', ',', 'and', 'Enterobacter', 'species', ')', 'to', 'which', 'were', 'given', 'the', 'highest', '\"', 'priority', 'status', '\"', 'since', 'they', 'represent', 'the', 'great', 'threat', 'to', 'humans', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'February', '2017', ',', 'in', 'light', 'of', 'increasing', 'antibiotic', 'resistance', ',', 'the', 'WHO', 'published', 'a', 'list', 'of', 'pathogens', 'that', 'includes', 'the', 'pathogens', 'designated', 'by', 'the', 'acronym', 'ESKAPE', '(', 'Enterococcus', 'faecium', ',', 'Staphylococcus', 'aureus', ',', 'Klebsiella', 'pneumoniae', ',', 'Acinetobacter', 'baumannii', ',', 'Pseudomonas', 'aeruginosa', ',', 'and', 'Enterobacter', 'species', ')', 'to', 'which', 'were', 'given', 'the', 'highest', '\"', 'priority', 'status', '\"', 'since', 'they', 'represent', 'the', 'great', 'threat', 'to', 'humans', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'February', '2017', ',', 'in', 'light', 'of', 'increasing', 'antibiotic', 'resistance', ',', 'the', 'WHO', 'published', 'a', 'list', 'of', 'pathogens', 'that', 'includes', 'the', 'pathogens', 'designated', 'by', 'the', 'acronym', 'ESKAPE', '(', 'Enterococcus', 'faecium', ',', 'Staphylococcus', 'aureus', ',', 'Klebsiella', 'pneumoniae', ',', 'Acinetobacter', 'baumannii', ',', 'Pseudomonas', 'aeruginosa', ',', 'and', 'Enterobacter', 'species', ')', 'to', 'which', 'were', 'given', 'the', 'highest', '\"', 'priority', 'status', '\"', 'since', 'they', 'represent', 'the', 'great', 'threat', 'to', 'humans', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['A', 'more', 'efficient', 'removal', 'of', 'antibiotic', 'microbial', 'resistance', 'abundances', 'from', 'wastewater', 'effluents', 'can', 'be', 'achieved', 'by', 'advanced', 'treatment', 'processes', ',', 'including', 'membrane', 'filtration', ',', 'ozonation', ',', 'UV', '-', 'irradiation', 'or', 'chlorination', ',', 'to', 'levels', 'typically', 'observed', 'in', 'urban', 'surface', 'water', 'or', 'low', 'impacted', 'surface', 'water', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['A', 'more', 'efficient', 'removal', 'of', 'antibiotic', 'microbial', 'resistance', 'abundances', 'from', 'wastewater', 'effluents', 'can', 'be', 'achieved', 'by', 'advanced', 'treatment', 'processes', ',', 'including', 'membrane', 'filtration', ',', 'ozonation', ',', 'UV', '-', 'irradiation', 'or', 'chlorination', ',', 'to', 'levels', 'typically', 'observed', 'in', 'urban', 'surface', 'water', 'or', 'low', 'impacted', 'surface', 'water', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"A more efficient removal of antibiotic microbial r...\" with entities \"[(187, 202, 'Techniques')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['A', 'more', 'efficient', 'removal', 'of', 'antibiotic', 'microbial', 'resistance', 'abundances', 'from', 'wastewater', 'effluents', 'can', 'be', 'achieved', 'by', 'advanced', 'treatment', 'processes', ',', 'including', 'membrane', 'filtration', ',', 'ozonation', ',', 'UV', '-', 'irradiation', 'or', 'chlorination', ',', 'to', 'levels', 'typically', 'observed', 'in', 'urban', 'surface', 'water', 'or', 'low', 'impacted', 'surface', 'water', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', '-', '-', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['A', 'more', 'efficient', 'removal', 'of', 'antibiotic', 'microbial', 'resistance', 'abundances', 'from', 'wastewater', 'effluents', 'can', 'be', 'achieved', 'by', 'advanced', 'treatment', 'processes', ',', 'including', 'membrane', 'filtration', ',', 'ozonation', ',', 'UV', '-', 'irradiation', 'or', 'chlorination', ',', 'to', 'levels', 'typically', 'observed', 'in', 'urban', 'surface', 'water', 'or', 'low', 'impacted', 'surface', 'water', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Antimicrobial', 'resistance', '(', 'AMR', ')', 'in', 'Clostridioides', 'difficile', 'remains', 'a', 'significant', 'threat', 'to', 'global', 'healthcare', 'systems', ',', 'not', 'just', 'for', 'the', 'treatment', 'of', 'C.difficile', 'infection', '(', 'CDI', ')', ',', 'but', 'as', 'a', 'reservoir', 'of', 'AMR', 'genes', 'that', 'could', 'be', 'potentially', 'transferred', 'to', 'other', 'pathogens', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Antimicrobial', 'resistance', '(', 'AMR', ')', 'in', 'Clostridioides', 'difficile', 'remains', 'a', 'significant', 'threat', 'to', 'global', 'healthcare', 'systems', ',', 'not', 'just', 'for', 'the', 'treatment', 'of', 'C.difficile', 'infection', '(', 'CDI', ')', ',', 'but', 'as', 'a', 'reservoir', 'of', 'AMR', 'genes', 'that', 'could', 'be', 'potentially', 'transferred', 'to', 'other', 'pathogens', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['ExPEC', 'strains', 'are', 'versatile', 'bacteria', 'that', 'can', 'cause', 'urinary', 'tract', ',', 'bloodstream', ',', 'prostate', ',', 'and', 'other', 'infections', 'at', 'non', '-', 'intestinal', 'sites', '.']\n",
            "['U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'mechanisms', 'of', 'resistance', 'for', 'several', 'antimicrobials', 'such', 'as', 'metronidazole', 'and', 'MLSB', '-', 'class', 'agents', 'are', 'only', 'beginning', 'to', 'be', 'elucidated', ',', 'and', 'increasingly', ',', 'there', 'is', 'evidence', 'that', 'previously', 'unconsidered', 'mechanisms', 'such', 'as', 'plasmid', '-', 'mediated', 'resistance', 'may', 'play', 'an', 'important', 'role', 'in', 'AMR', 'in', 'this', 'bacterium', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'mechanisms', 'of', 'resistance', 'for', 'several', 'antimicrobials', 'such', 'as', 'metronidazole', 'and', 'MLSB', '-', 'class', 'agents', 'are', 'only', 'beginning', 'to', 'be', 'elucidated', ',', 'and', 'increasingly', ',', 'there', 'is', 'evidence', 'that', 'previously', 'unconsidered', 'mechanisms', 'such', 'as', 'plasmid', '-', 'mediated', 'resistance', 'may', 'play', 'an', 'important', 'role', 'in', 'AMR', 'in', 'this', 'bacterium', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'mechanisms', 'of', 'resistance', 'for', 'several', 'antimicrobials', 'such', 'as', 'metronidazole', 'and', 'MLSB', '-', 'class', 'agents', 'are', 'only', 'beginning', 'to', 'be', 'elucidated', ',', 'and', 'increasingly', ',', 'there', 'is', 'evidence', 'that', 'previously', 'unconsidered', 'mechanisms', 'such', 'as', 'plasmid', '-', 'mediated', 'resistance', 'may', 'play', 'an', 'important', 'role', 'in', 'AMR', 'in', 'this', 'bacterium', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'the', 'genetics', 'of', 'AMR', 'in', 'C.', 'difficile', 'will', 'be', 'described', ',', 'along', 'with', 'a', 'discussion', 'of', 'the', 'factors', 'contributing', 'to', 'the', 'difficulty', 'in', 'clearly', 'determining', 'the', 'true', 'burden', 'of', 'AMR', 'in', 'C.', 'difficile', 'and', 'how', 'it', 'affects', 'the', 'treatment', 'of', 'CDI', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'the', 'genetics', 'of', 'AMR', 'in', 'C.', 'difficile', 'will', 'be', 'described', ',', 'along', 'with', 'a', 'discussion', 'of', 'the', 'factors', 'contributing', 'to', 'the', 'difficulty', 'in', 'clearly', 'determining', 'the', 'true', 'burden', 'of', 'AMR', 'in', 'C.', 'difficile', 'and', 'how', 'it', 'affects', 'the', 'treatment', 'of', 'CDI', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Fluoroquinolone', 'resistance', 'mechanisms', 'include', 'efflux', 'pumps', '(', 'MepA', ',', 'NorA', ',', 'NorB', ',', 'NorC', ',', 'MdeA', ',', 'LmrS', 'or', 'SdrM', 'in', 'S.', 'aureus', 'and', 'EfmA', 'or', 'EfrAB', 'in', 'the', 'enterococci', ')', 'for', 'removal', 'of', 'fluoroquinolone', 'from', 'the', 'intracellular', 'environment', 'of', 'bacterial', 'cells', 'and/or', 'protection', 'of', 'the', 'gyrase', 'and', 'topoisomerase', 'IV', 'target', 'sites', 'in', 'Enterococcus', 'faecalis', 'by', 'Qnr', '-', 'like', 'proteins', '.']\n",
            "['U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Fluoroquinolone', 'resistance', 'mechanisms', 'include', 'efflux', 'pumps', '(', 'MepA', ',', 'NorA', ',', 'NorB', ',', 'NorC', ',', 'MdeA', ',', 'LmrS', 'or', 'SdrM', 'in', 'S.', 'aureus', 'and', 'EfmA', 'or', 'EfrAB', 'in', 'the', 'enterococci', ')', 'for', 'removal', 'of', 'fluoroquinolone', 'from', 'the', 'intracellular', 'environment', 'of', 'bacterial', 'cells', 'and/or', 'protection', 'of', 'the', 'gyrase', 'and', 'topoisomerase', 'IV', 'target', 'sites', 'in', 'Enterococcus', 'faecalis', 'by', 'Qnr', '-', 'like', 'proteins', '.']\n",
            "['O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Fluoroquinolone', 'resistance', 'mechanisms', 'include', 'efflux', 'pumps', '(', 'MepA', ',', 'NorA', ',', 'NorB', ',', 'NorC', ',', 'MdeA', ',', 'LmrS', 'or', 'SdrM', 'in', 'S.', 'aureus', 'and', 'EfmA', 'or', 'EfrAB', 'in', 'the', 'enterococci', ')', 'for', 'removal', 'of', 'fluoroquinolone', 'from', 'the', 'intracellular', 'environment', 'of', 'bacterial', 'cells', 'and/or', 'protection', 'of', 'the', 'gyrase', 'and', 'topoisomerase', 'IV', 'target', 'sites', 'in', 'Enterococcus', 'faecalis', 'by', 'Qnr', '-', 'like', 'proteins', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Fluoroquinolone', 'resistance', 'mechanisms', 'include', 'efflux', 'pumps', '(', 'MepA', ',', 'NorA', ',', 'NorB', ',', 'NorC', ',', 'MdeA', ',', 'LmrS', 'or', 'SdrM', 'in', 'S.', 'aureus', 'and', 'EfmA', 'or', 'EfrAB', 'in', 'the', 'enterococci', ')', 'for', 'removal', 'of', 'fluoroquinolone', 'from', 'the', 'intracellular', 'environment', 'of', 'bacterial', 'cells', 'and/or', 'protection', 'of', 'the', 'gyrase', 'and', 'topoisomerase', 'IV', 'target', 'sites', 'in', 'Enterococcus', 'faecalis', 'by', 'Qnr', '-', 'like', 'proteins', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Fluoroquinolone', 'resistance', 'mechanisms', 'include', 'efflux', 'pumps', '(', 'MepA', ',', 'NorA', ',', 'NorB', ',', 'NorC', ',', 'MdeA', ',', 'LmrS', 'or', 'SdrM', 'in', 'S.', 'aureus', 'and', 'EfmA', 'or', 'EfrAB', 'in', 'the', 'enterococci', ')', 'for', 'removal', 'of', 'fluoroquinolone', 'from', 'the', 'intracellular', 'environment', 'of', 'bacterial', 'cells', 'and/or', 'protection', 'of', 'the', 'gyrase', 'and', 'topoisomerase', 'IV', 'target', 'sites', 'in', 'Enterococcus', 'faecalis', 'by', 'Qnr', '-', 'like', 'proteins', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Fluoroquinolone', 'resistance', 'mechanisms', 'include', 'efflux', 'pumps', '(', 'MepA', ',', 'NorA', ',', 'NorB', ',', 'NorC', ',', 'MdeA', ',', 'LmrS', 'or', 'SdrM', 'in', 'S.', 'aureus', 'and', 'EfmA', 'or', 'EfrAB', 'in', 'the', 'enterococci', ')', 'for', 'removal', 'of', 'fluoroquinolone', 'from', 'the', 'intracellular', 'environment', 'of', 'bacterial', 'cells', 'and/or', 'protection', 'of', 'the', 'gyrase', 'and', 'topoisomerase', 'IV', 'target', 'sites', 'in', 'Enterococcus', 'faecalis', 'by', 'Qnr', '-', 'like', 'proteins', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Gene/Proteins', 'O', 'O', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"Resistance to the glycopeptide antibiotic teicopla...\" with entities \"[(41, 53, 'Antibiotics')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['Resistance', 'to', 'the', 'glycopeptide', 'antibiotic', 'teicoplanin', 'occurs', 'via', 'efflux', 'regulated', 'by', 'the', 'TcaR', 'regulator', 'in', 'S.', 'aureus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Resistance', 'to', 'the', 'glycopeptide', 'antibiotic', 'teicoplanin', 'occurs', 'via', 'efflux', 'regulated', 'by', 'the', 'TcaR', 'regulator', 'in', 'S.', 'aureus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Resistance', 'to', 'the', 'glycopeptide', 'antibiotic', 'teicoplanin', 'occurs', 'via', 'efflux', 'regulated', 'by', 'the', 'TcaR', 'regulator', 'in', 'S.', 'aureus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O']\n",
            "['Engineered', 'CRISPR', '-', 'Cas', 'systems', 'are', 'reported', 'to', 'effectively', 'kill', 'bacteria', 'or', 'even', 'revert', 'bacterial', 'resistance', 'to', 'antibiotics', '(', 'resensitizing', 'bacterial', 'cells', 'to', 'antibiotics', ')', '.']\n",
            "['O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Strategies', 'for', 'combating', 'antimicrobial', 'resistance', 'using', 'CRISPR', '(', 'i.e.', ',', 'Cas9', ',', 'Cas12', ',', 'Cas13', ',', 'and', 'Cas14', ')', 'can', 'be', 'of', 'great', 'significance', 'in', 'detecting', 'bacteria', 'and', 'their', 'resistance', 'to', 'antibiotics', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Strategies', 'for', 'combating', 'antimicrobial', 'resistance', 'using', 'CRISPR', '(', 'i.e.', ',', 'Cas9', ',', 'Cas12', ',', 'Cas13', ',', 'and', 'Cas14', ')', 'can', 'be', 'of', 'great', 'significance', 'in', 'detecting', 'bacteria', 'and', 'their', 'resistance', 'to', 'antibiotics', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Strategies', 'for', 'combating', 'antimicrobial', 'resistance', 'using', 'CRISPR', '(', 'i.e.', ',', 'Cas9', ',', 'Cas12', ',', 'Cas13', ',', 'and', 'Cas14', ')', 'can', 'be', 'of', 'great', 'significance', 'in', 'detecting', 'bacteria', 'and', 'their', 'resistance', 'to', 'antibiotics', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Strategies', 'for', 'combating', 'antimicrobial', 'resistance', 'using', 'CRISPR', '(', 'i.e.', ',', 'Cas9', ',', 'Cas12', ',', 'Cas13', ',', 'and', 'Cas14', ')', 'can', 'be', 'of', 'great', 'significance', 'in', 'detecting', 'bacteria', 'and', 'their', 'resistance', 'to', 'antibiotics', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Strategies', 'for', 'combating', 'antimicrobial', 'resistance', 'using', 'CRISPR', '(', 'i.e.', ',', 'Cas9', ',', 'Cas12', ',', 'Cas13', ',', 'and', 'Cas14', ')', 'can', 'be', 'of', 'great', 'significance', 'in', 'detecting', 'bacteria', 'and', 'their', 'resistance', 'to', 'antibiotics', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['As', 'of', 'now', ',', 'endolysins', 'have', 'been', 'applied', 'to', 'diverse', 'aspects', '.', 'In', 'this', 'review', ',', 'we', 'comprehensively', 'introduce', 'the', 'structures', 'and', 'activities', 'of', 'endolysins', 'and', 'summarize', 'the', 'latest', 'application', 'progress', 'of', 'recombinant', 'endolysins', 'in', 'the', 'fields', 'of', 'medical', 'treatment', ',', 'pathogen', 'diagnosis', ',', 'food', 'safety', ',', 'and', 'agriculture', '.']\n",
            "['O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'focuses', 'on', 'the', 'AMR', 'of', 'Staphylococcus', 'aureus', ',', 'a', 'major', 'gram', '-', 'positive', 'pathogen', ',', 'which', 'has', 'the', 'ability', 'to', 'acquire', 'resistance', 'to', 'antimicrobials', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['As', 'a', 'general', 'classification', ',', 'E.', 'coli', 'can', 'be', 'distinguished', 'based', 'on', 'the', 'ability', 'to', 'cause', 'infection', 'of', 'the', 'gastrointestinal', 'system', '(', 'IPEC', ')', 'or', 'outside', 'of', 'it', '(', 'ExPEC', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['As', 'a', 'general', 'classification', ',', 'E.', 'coli', 'can', 'be', 'distinguished', 'based', 'on', 'the', 'ability', 'to', 'cause', 'infection', 'of', 'the', 'gastrointestinal', 'system', '(', 'IPEC', ')', 'or', 'outside', 'of', 'it', '(', 'ExPEC', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"The resistance factors of S. aureus are frequently...\" with entities \"[(25, 35, 'Antibiotic-Resistant Bacteria')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['The', 'resistance', 'factors', 'of', 'S.', 'aureus', 'are', 'frequently', 'found', 'on', 'mobile', 'elements', ',', 'including', 'plasmids', 'and', 'transposons', '.']\n",
            "['O', 'O', 'O', 'O', '-', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'the', 'different', 'pathogens', ',', 'E.', 'coli', 'are', 'becoming', 'of', 'great', 'importance', ',', 'and', 'it', 'has', 'been', 'suggested', 'that', 'ExPEC', 'may', 'harbor', 'resistance', 'genes', 'that', 'may', 'be', 'transferred', 'to', 'pathogenic', 'or', 'opportunistic', 'bacteria', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'the', 'different', 'pathogens', ',', 'E.', 'coli', 'are', 'becoming', 'of', 'great', 'importance', ',', 'and', 'it', 'has', 'been', 'suggested', 'that', 'ExPEC', 'may', 'harbor', 'resistance', 'genes', 'that', 'may', 'be', 'transferred', 'to', 'pathogenic', 'or', 'opportunistic', 'bacteria', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Furthermore', ',', 'outbreaks', 'of', 'methicillin', '-', 'resistant', 'S.', 'aureus', '(', 'MRSA', ')', 'in', 'hospitals', 'and', 'communities', 'have', 'led', 'us', 'to', 'study', 'infection', 'controls', ',', 'including', 'the', 'antiseptic', 'susceptibility', 'evaluation', 'and', 'molecular', 'epidemiology', 'of', 'MRSA', '.']\n",
            "['O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Various', 'antiseptic', 'resistance', 'determinants', ',', 'such', 'as', 'qacA', '/', 'B', 'and', 'smr', ',', 'were', 'identified', 'on', 'plasmids', 'and', 'characterized', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Gene/Proteins', 'I-Gene/Proteins', 'I-Gene/Proteins', 'I-Gene/Proteins', 'L-Gene/Proteins', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'demonstrated', 'that', 'the', 'plasmid', '-', 'mediated', 'efflux', 'pump', 'QacB', 'variant', 'QacIII', 'confers', 'fluoroquinolone', 'efflux', 'ability', 'to', 'S.', 'aureus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'demonstrated', 'that', 'the', 'plasmid', '-', 'mediated', 'efflux', 'pump', 'QacB', 'variant', 'QacIII', 'confers', 'fluoroquinolone', 'efflux', 'ability', 'to', 'S.', 'aureus', '.']\n",
            "['O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'demonstrated', 'that', 'the', 'plasmid', '-', 'mediated', 'efflux', 'pump', 'QacB', 'variant', 'QacIII', 'confers', 'fluoroquinolone', 'efflux', 'ability', 'to', 'S.', 'aureus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O']\n",
            "['We', 'demonstrated', 'that', 'the', 'plasmid', '-', 'mediated', 'efflux', 'pump', 'QacB', 'variant', 'QacIII', 'confers', 'fluoroquinolone', 'efflux', 'ability', 'to', 'S.', 'aureus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Gene/Proteins', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'demonstrated', 'that', 'the', 'plasmid', '-', 'mediated', 'efflux', 'pump', 'QacB', 'variant', 'QacIII', 'confers', 'fluoroquinolone', 'efflux', 'ability', 'to', 'S.', 'aureus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Gene/Proteins', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Studies', 'on', 'MRSA', 'epidemiology', 'had', 'shown', 'that', 'community', '-', 'acquired', 'MRSA', '(', 'CA', '-', 'MRSA', ')', 'was', 'disseminated', 'into', 'hospitals', 'and', 'that', 'an', 'increased', 'use', 'of', 'alcohol', '-', 'based', 'rubs', 'could', 'reduce', 'the', 'incidence', 'of', 'MRSA', 'infections', 'in', 'such', 'institutions', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Studies', 'on', 'MRSA', 'epidemiology', 'had', 'shown', 'that', 'community', '-', 'acquired', 'MRSA', '(', 'CA', '-', 'MRSA', ')', 'was', 'disseminated', 'into', 'hospitals', 'and', 'that', 'an', 'increased', 'use', 'of', 'alcohol', '-', 'based', 'rubs', 'could', 'reduce', 'the', 'incidence', 'of', 'MRSA', 'infections', 'in', 'such', 'institutions', '.']\n",
            "['O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Studies', 'on', 'MRSA', 'epidemiology', 'had', 'shown', 'that', 'community', '-', 'acquired', 'MRSA', '(', 'CA', '-', 'MRSA', ')', 'was', 'disseminated', 'into', 'hospitals', 'and', 'that', 'an', 'increased', 'use', 'of', 'alcohol', '-', 'based', 'rubs', 'could', 'reduce', 'the', 'incidence', 'of', 'MRSA', 'infections', 'in', 'such', 'institutions', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O']\n",
            "['Additionally', ',', 'the', 'study', 'of', 'CA', '-', 'MRSA', 'collected', 'from', 'communities', 'and', 'hospitals', 'showed', 'an', 'increase', 'in', 'Panton', '-', 'Valentine', 'leucocidin', '(', 'PVL)-positive', 'CA', '-', 'MRSA', ',', 'causing', 'severe', 'skin', 'and', 'soft', 'tissue', 'infections', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Additionally', ',', 'the', 'study', 'of', 'CA', '-', 'MRSA', 'collected', 'from', 'communities', 'and', 'hospitals', 'showed', 'an', 'increase', 'in', 'Panton', '-', 'Valentine', 'leucocidin', '(', 'PVL)-positive', 'CA', '-', 'MRSA', ',', 'causing', 'severe', 'skin', 'and', 'soft', 'tissue', 'infections', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'advent', 'of', 'pandrug', '-', 'resistant', 'bacteria', 'makes', 'most', ',', 'if', 'not', 'all', ',', 'available', 'antibiotics', 'invalid', '.']\n",
            "['O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'CRISPR', '-', 'Cas', 'system', 'is', 'an', 'adaptive', 'immune', 'system', 'found', 'in', 'many', 'prokaryotes', 'that', 'presents', 'attractive', 'opportunities', 'to', 'target', 'and', 'edit', 'nucleic', 'acids', 'with', 'high', 'precision', 'and', 'reliability', '.']\n",
            "['O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'addition', ',', 'we', 'summarize', 'the', 'most', 'recent', 'advances', 'in', 'applying', 'the', 'CRISPR', '-', 'Cas', 'system', 'for', 'virulence', 'modulation', 'of', 'bacterial', 'infections', 'and', 'combating', 'antimicrobial', 'resistance', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'determined', 'the', 'complete', 'DNA', 'sequence', 'of', 'the', 'tetracycline', '-', 'resistance', 'plasmid', 'and', 'found', 'that', 'the', 'inducible', 'expression', 'of', 'tetK', 'in', 'S.', 'aureus', 'was', 'regulated', 'by', 'a', 'post', '-', 'transcriptional', 'attenuation', 'mechanism', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'determined', 'the', 'complete', 'DNA', 'sequence', 'of', 'the', 'tetracycline', '-', 'resistance', 'plasmid', 'and', 'found', 'that', 'the', 'inducible', 'expression', 'of', 'tetK', 'in', 'S.', 'aureus', 'was', 'regulated', 'by', 'a', 'post', '-', 'transcriptional', 'attenuation', 'mechanism', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'determined', 'the', 'complete', 'DNA', 'sequence', 'of', 'the', 'tetracycline', '-', 'resistance', 'plasmid', 'and', 'found', 'that', 'the', 'inducible', 'expression', 'of', 'tetK', 'in', 'S.', 'aureus', 'was', 'regulated', 'by', 'a', 'post', '-', 'transcriptional', 'attenuation', 'mechanism', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'I-Antibiotic-Resistant Genes', 'L-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'we', 'present', 'Escherichia', 'coli', 'and', 'Pseudomonas', 'aeruginosa', 'as', 'specific', 'examples', 'of', 'key', 'AMR', 'threats', 'caused', 'by', 'Gram', '-', 'negative', 'bacteria', 'and', 'we', 'discuss', 'the', 'current', 'status', 'of', 'mAbs', 'and', 'vaccine', 'approaches', 'under', 'development', 'as', 'well', 'as', 'how', 'knowledge', 'on', 'OMV', 'could', 'benefit', 'antigen', 'discovery', 'strategies', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'we', 'present', 'Escherichia', 'coli', 'and', 'Pseudomonas', 'aeruginosa', 'as', 'specific', 'examples', 'of', 'key', 'AMR', 'threats', 'caused', 'by', 'Gram', '-', 'negative', 'bacteria', 'and', 'we', 'discuss', 'the', 'current', 'status', 'of', 'mAbs', 'and', 'vaccine', 'approaches', 'under', 'development', 'as', 'well', 'as', 'how', 'knowledge', 'on', 'OMV', 'could', 'benefit', 'antigen', 'discovery', 'strategies', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'we', 'present', 'Escherichia', 'coli', 'and', 'Pseudomonas', 'aeruginosa', 'as', 'specific', 'examples', 'of', 'key', 'AMR', 'threats', 'caused', 'by', 'Gram', '-', 'negative', 'bacteria', 'and', 'we', 'discuss', 'the', 'current', 'status', 'of', 'mAbs', 'and', 'vaccine', 'approaches', 'under', 'development', 'as', 'well', 'as', 'how', 'knowledge', 'on', 'OMV', 'could', 'benefit', 'antigen', 'discovery', 'strategies', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Non', '-', 'typhoid', 'Salmonella', 'spp', '.', 'is', 'one', 'of', 'the', 'leading', 'foodborne', 'pathogens', 'which', 'infect', 'humans', 'worldwide', 'and', 'is', 'associated', 'with', 'food', 'and', 'livestock', '.']\n",
            "['O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Salmonella', 'enterica', 'remains', 'an', 'important', 'foodborne', 'pathogen', 'in', 'all', 'regions', 'of', 'the', 'world', ',', 'with', 'Typhimurium', 'as', 'one', 'of', 'the', 'most', 'frequent', 'serotypes', 'causing', 'foodborne', 'disease', '.']\n",
            "['B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Salmonella', 'enterica', 'remains', 'an', 'important', 'foodborne', 'pathogen', 'in', 'all', 'regions', 'of', 'the', 'world', ',', 'with', 'Typhimurium', 'as', 'one', 'of', 'the', 'most', 'frequent', 'serotypes', 'causing', 'foodborne', 'disease', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'the', 'present', 'review', ',', 'we', 'highlighted', 'data', 'published', 'on', 'antimicrobial', 'resistant', 'Salmonella', 'species', 'and', 'serovars', 'associated', 'with', 'livestock', 'and', 'food', 'chain', 'animals', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Plasmids', ',', 'transposons', ',', 'and', 'insertion', 'sequence', 'elements', 'mediate', 'dissemination', 'of', 'not', 'only', 'AMR', 'genes', 'but', 'also', 'genes', 'for', 'resistance', 'to', 'heavy', 'metals', 'and', 'disinfectants', ',', 'thus', 'limiting', 'the', 'therapeutic', 'options', 'for', 'treatment', 'and', 'control', 'of', 'Salmonella', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O']\n",
            "['Genes', 'for', 'resistance', 'to', 'colistin', '(', 'mcr-1', 'to', 'mcr-9', ')', 'and', 'carbapenem', '(', 'blaVIM-1', ',', 'blaDNM-1', ',', 'and', 'blaNDM-5', ')', 'have', 'been', 'detected', 'from', 'poultry', ',', 'pig', ',', 'and', 'human', 'Salmonella', 'isolates', ',', 'indicating', 'food', 'animal', '-', 'associated', 'AMR', 'which', 'is', 'a', 'threat', 'to', 'human', 'public', 'health', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Genes', 'for', 'resistance', 'to', 'colistin', '(', 'mcr-1', 'to', 'mcr-9', ')', 'and', 'carbapenem', '(', 'blaVIM-1', ',', 'blaDNM-1', ',', 'and', 'blaNDM-5', ')', 'have', 'been', 'detected', 'from', 'poultry', ',', 'pig', ',', 'and', 'human', 'Salmonella', 'isolates', ',', 'indicating', 'food', 'animal', '-', 'associated', 'AMR', 'which', 'is', 'a', 'threat', 'to', 'human', 'public', 'health', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Genes', 'for', 'resistance', 'to', 'colistin', '(', 'mcr-1', 'to', 'mcr-9', ')', 'and', 'carbapenem', '(', 'blaVIM-1', ',', 'blaDNM-1', ',', 'and', 'blaNDM-5', ')', 'have', 'been', 'detected', 'from', 'poultry', ',', 'pig', ',', 'and', 'human', 'Salmonella', 'isolates', ',', 'indicating', 'food', 'animal', '-', 'associated', 'AMR', 'which', 'is', 'a', 'threat', 'to', 'human', 'public', 'health', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Genes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Genes', 'for', 'resistance', 'to', 'colistin', '(', 'mcr-1', 'to', 'mcr-9', ')', 'and', 'carbapenem', '(', 'blaVIM-1', ',', 'blaDNM-1', ',', 'and', 'blaNDM-5', ')', 'have', 'been', 'detected', 'from', 'poultry', ',', 'pig', ',', 'and', 'human', 'Salmonella', 'isolates', ',', 'indicating', 'food', 'animal', '-', 'associated', 'AMR', 'which', 'is', 'a', 'threat', 'to', 'human', 'public', 'health', '.']\n",
            "['O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Genes', 'for', 'resistance', 'to', 'colistin', '(', 'mcr-1', 'to', 'mcr-9', ')', 'and', 'carbapenem', '(', 'blaVIM-1', ',', 'blaDNM-1', ',', 'and', 'blaNDM-5', ')', 'have', 'been', 'detected', 'from', 'poultry', ',', 'pig', ',', 'and', 'human', 'Salmonella', 'isolates', ',', 'indicating', 'food', 'animal', '-', 'associated', 'AMR', 'which', 'is', 'a', 'threat', 'to', 'human', 'public', 'health', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['We', 'are', 'searching', 'for', 'a', 'new', 'method', 'to', 'combat', 'AMR', 'and', 'phages', 'are', 'viruses', 'that', 'can', 'effectively', 'fight', 'bacterial', 'infections', ',', 'which', 'have', 'renewed', 'interest', 'as', 'antibiotic', 'alternatives', 'with', 'their', 'specificity', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Large', 'phage', 'products', 'have', 'been', 'produced', 'in', 'recent', 'years', 'to', 'fight', 'AMR', '.']\n",
            "['O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['By', '2050', ',', 'it', 'is', 'estimated', 'that', 'the', 'leading', 'cause', 'of', 'death', 'will', 'be', 'through', 'untreatable', 'AMR', 'pathogens', '.']\n",
            "['O', 'U-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['For', 'the', 'discovery', 'of', 'new', 'vaccines', 'and', 'mAbs', ',', 'the', 'search', 'for', 'effective', 'antigens', 'that', 'are', 'able', 'to', 'raise', 'protective', 'immune', 'responses', 'is', 'a', 'challenging', 'undertaking', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Due', 'to', 'the', 'lack', 'of', 'discovery', 'of', 'new', 'antibiotics', 'and', 'the', 'pressure', 'exerted', 'by', 'antimicrobial', 'resistance', 'in', 'the', 'pharmaceutical', 'industry', ',', 'this', 'review', 'aimed', 'to', 'address', 'the', 'issue', 'of', 'antibiotic', 'use', 'in', 'livestock', 'which', 'leads', 'to', 'AMR', 'in', 'Salmonella', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O']\n",
            "['The', 'importance', 'of', 'genomic', 'characterization', 'of', 'carbapenem', 'and', 'colistin', 'resistant', 'Salmonella', 'in', 'determining', 'the', 'relationship', 'between', 'human', 'clinical', 'isolates', 'and', 'food', 'animal', 'isolates', 'was', 'also', 'discussed', 'in', 'this', 'review', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'importance', 'of', 'genomic', 'characterization', 'of', 'carbapenem', 'and', 'colistin', 'resistant', 'Salmonella', 'in', 'determining', 'the', 'relationship', 'between', 'human', 'clinical', 'isolates', 'and', 'food', 'animal', 'isolates', 'was', 'also', 'discussed', 'in', 'this', 'review', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'importance', 'of', 'genomic', 'characterization', 'of', 'carbapenem', 'and', 'colistin', 'resistant', 'Salmonella', 'in', 'determining', 'the', 'relationship', 'between', 'human', 'clinical', 'isolates', 'and', 'food', 'animal', 'isolates', 'was', 'also', 'discussed', 'in', 'this', 'review', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O']\n",
            "['Herein', 'we', 'survey', 'resistance', 'to', 'beta', '-', 'lactams', ',', 'monobactams', ',', 'carbapenems', ',', 'aminoglycosides', ',', 'tetracycline', ',', 'polymyxins', ',', 'macrolides', ',', 'rifampin', ',', 'fosmidomycin', ',', 'and', 'fluoroquinolones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O']\n",
            "['We', 'discuss', 'newly', 'developed', 'methods', 'of', 'testing', 'antibiotics', 'against', 'Francisella', 'which', 'involve', 'the', 'intracellular', 'nature', 'of', 'Francisella', 'infection', 'and', 'may', 'better', 'reflect', 'the', 'eventual', 'clinical', 'outcomes', 'for', 'new', 'antibiotic', 'compounds', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Understanding', 'the', 'genetically', 'encoded', 'determinants', 'of', 'AMR', 'in', 'Francisella', 'is', 'key', 'to', 'optimizing', 'the', 'treatment', 'of', 'patients', 'and', 'potentially', 'developing', 'new', 'antimicrobials', 'for', 'this', 'dangerous', 'intracellular', 'pathogen', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Acinetobacter', 'baumannii', 'has', 'become', 'increasingly', 'resistant', 'to', 'leading', 'antimicrobial', 'agents', 'since', 'the', '1970s', '.']\n",
            "['B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['However', ',', 'the', 'past', 'two', 'decades', 'have', 'seen', 'a', 'rapid', 'worldwide', 'emergence', 'of', 'a', 'new', 'Salmonella', 'serotype', ',', 'namely', 'monophasic', 'variant', 'of', 'S.', 'Typhimurium', ',', 'whose', 'antigenic', 'formula', 'is', '1,4,[5],12', ':', 'i:-', '.', 'It', 'has', 'become', 'one', 'of', 'the', '2', '-', '5', 'most', 'common', 'Salmonella', 'serotypes', 'responsible', 'for', 'animal', 'and', 'human', 'infections', 'in', 'different', 'regions', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'unexpected', 'link', 'to', 'swine', 'has', 'escalated', 'monophasic', 'S.', 'Typhimurium', 'infections', 'to', 'the', 'status', 'of', 'a', 'global', 'public', 'health', 'emergency', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['MDRAb', 'gained', 'media', 'attention', 'after', 'being', 'identified', 'in', 'American', 'soldiers', 'returning', 'from', 'Iraq', 'and', 'treated', 'in', 'US', 'military', 'facilities', ',', 'where', 'it', 'was', 'termed', '\"', 'Iraqibacter', '.', '\"']\n",
            "['U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'large', '-', 'scale', 'application', 'of', 'whole', 'genome', 'sequencing', '(', 'WGS', ')', 'in', 'the', 'last', '10', 'years', 'has', 'revealed', 'the', 'phylogenetic', 'associations', 'of', 'the', 'bacterium', 'and', 'its', 'antimicrobial', 'resistance', '(', 'AMR', ')', 'genes', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Tularemia', ',', 'caused', 'by', 'Francisella', 'tularensis', ',', 'is', 'endemic', 'to', 'the', 'northern', 'hemisphere', '.']\n",
            "['O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', 'we', 'highlight', 'the', 'potential', 'causes', 'of', 'antimicrobial', 'resistance', 'by', 'heavy', 'metals', ',', 'with', 'a', 'focus', 'on', 'its', 'emergence', 'in', 'A.', 'baumanni', 'in', 'war', 'zones', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O']\n",
            "['Francisella', 'is', 'unlike', 'many', 'Gram', '-', 'negative', 'organisms', 'in', 'that', 'it', 'does', 'not', 'have', 'significant', 'plasmid', 'mobility', ',', 'and', 'does', 'not', 'express', 'AMR', 'mechanisms', 'on', 'plasmids', ';', 'thus', 'plasmid', '-', 'mediated', 'resistance', 'does', 'not', 'occur', 'naturally', '.']\n",
            "['U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Infections', 'with', 'multidrug', '-', 'resistant', '(', 'MDR', ')', 'bacteria', 'also', 'have', 'substantial', 'implications', 'on', 'clinical', 'and', 'economic', 'outcomes', '.']\n",
            "['O', 'O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'highlights', 'AMR', \"'s\", 'scale', 'and', 'consequences', ',', 'the', 'importance', ',', 'and', 'implications', 'of', 'an', 'antimicrobial', 'stewardship', 'program', '(', 'ASP', ')', 'to', 'fight', 'resistance', 'and', 'protect', 'global', 'health', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Recently', 'attention', 'has', 'focused', 'on', 'an', 'emerging', 'pathogen', ',', 'multi', '-', 'drug', 'resistant', 'A.', 'baumannii', '(', 'MDRAb', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O']\n",
            "['However', ',', 'MDRAb', 'is', 'strongly', 'associated', 'in', 'the', 'literature', 'with', 'war', 'injuries', 'that', 'are', 'heavily', 'contaminated', 'by', 'both', 'environmental', 'debris', 'and', 'shrapnel', 'from', 'weapons', '.']\n",
            "['O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'article', ',', 'after', 'briefly', 'reviewing', 'the', 'most', 'important', 'and', 'clinically', 'relevant', 'multi', '-', 'drug', '-', 'resistant', 'bacteria', 'and', 'their', 'main', 'resistance', 'mechanisms', ',', 'we', 'describe', 'the', 'emerging', 'antimicrobial', 'options', 'for', 'both', 'MDR', 'Gram', '-', 'positive', 'cocci', 'and', 'Gram', '-', 'negative', 'bacilli', ',', 'including', 'recently', 'marketed', 'agents', ',', 'molecules', 'just', 'approved', 'or', 'under', 'evaluation', 'and', 'rediscovered', 'older', 'antibiotics', 'that', 'have', 'regained', 'importance', 'due', 'to', 'their', 'antimicrobial', 'spectrum', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Specifically', ',', 'emerging', 'options', 'for', 'Gram', '-', 'positive', 'cocci', 'we', 'reviewed', 'include', 'ceftaroline', ',', 'ceftobiprole', ',', 'tedizolid', ',', 'dalbavancin', ',', 'and', 'fosfomycin', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Specifically', ',', 'emerging', 'options', 'for', 'Gram', '-', 'positive', 'cocci', 'we', 'reviewed', 'include', 'ceftaroline', ',', 'ceftobiprole', ',', 'tedizolid', ',', 'dalbavancin', ',', 'and', 'fosfomycin', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Specifically', ',', 'emerging', 'options', 'for', 'Gram', '-', 'positive', 'cocci', 'we', 'reviewed', 'include', 'ceftaroline', ',', 'ceftobiprole', ',', 'tedizolid', ',', 'dalbavancin', ',', 'and', 'fosfomycin', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Specifically', ',', 'emerging', 'options', 'for', 'Gram', '-', 'positive', 'cocci', 'we', 'reviewed', 'include', 'ceftaroline', ',', 'ceftobiprole', ',', 'tedizolid', ',', 'dalbavancin', ',', 'and', 'fosfomycin', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O']\n",
            "['Specifically', ',', 'emerging', 'options', 'for', 'Gram', '-', 'positive', 'cocci', 'we', 'reviewed', 'include', 'ceftaroline', ',', 'ceftobiprole', ',', 'tedizolid', ',', 'dalbavancin', ',', 'and', 'fosfomycin', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O']\n",
            "['Specifically', ',', 'emerging', 'options', 'for', 'Gram', '-', 'positive', 'cocci', 'we', 'reviewed', 'include', 'ceftaroline', ',', 'ceftobiprole', ',', 'tedizolid', ',', 'dalbavancin', ',', 'and', 'fosfomycin', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Emerging', 'treatment', 'options', 'for', 'Gram', '-', 'negative', 'bacilli', 'we', 'considered', 'comprise', 'ceftolozane', '-', 'tazobactam', ',', 'ceftazidime', '-', 'avibactam', ',', 'meropenem', '-', 'vaborbactam', ',', 'imipenem', '-', 'relebactam', ',', 'aztreonam', '-', 'avibactam', ',', 'minocycline', ',', 'fosfomycin', ',', 'eravacycline', ',', 'plazomicin', ',', 'and', 'cefiderocol', '.']\n",
            "['O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'ARGs', 'in', 'urban', 'rivers', 'have', 'the', 'potential', 'of', 'being', 'transferred', 'to', 'clinically', 'important', 'pathogens', '.']\n",
            "['O', 'U-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'is', 'followed', 'by', 'a', 'discussion', 'on', 'a', 'chemical', 'series', 'identified', 'by', 'whole', '-', 'cell', 'cross', '-', 'screening', 'against', 'Mtb', ',', 'for', 'which', 'MoA', 'deconvolution', 'studies', 'revealed', 'a', 'pathway', 'that', 'explained', 'the', 'lack', 'of', 'in', 'vivo', 'efficacy', 'in', 'a', 'mouse', 'model', 'of', 'TB', 'and', 'reiterated', 'the', 'importance', 'of', 'selecting', 'an', 'appropriate', 'growth', 'medium', 'during', 'phenotypic', 'screening', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O']\n",
            "['This', 'is', 'followed', 'by', 'a', 'discussion', 'on', 'a', 'chemical', 'series', 'identified', 'by', 'whole', '-', 'cell', 'cross', '-', 'screening', 'against', 'Mtb', ',', 'for', 'which', 'MoA', 'deconvolution', 'studies', 'revealed', 'a', 'pathway', 'that', 'explained', 'the', 'lack', 'of', 'in', 'vivo', 'efficacy', 'in', 'a', 'mouse', 'model', 'of', 'TB', 'and', 'reiterated', 'the', 'importance', 'of', 'selecting', 'an', 'appropriate', 'growth', 'medium', 'during', 'phenotypic', 'screening', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Mycobacterium', 'tuberculosis', '(', 'Mtb', ')', ',', 'an', 'acid', '-', 'fast', 'bacillus', 'that', 'causes', 'tuberculosis', '(', 'TB', ')', ',', 'merits', 'being', 'one', 'of', 'the', 'highest', 'priority', 'pathogens', 'on', 'this', 'list', 'since', 'drug', '-', 'resistant', 'TB', '(', 'DR', '-', 'TB', ')', 'accounts', 'for', '∼29', '%', 'of', 'deaths', 'attributable', 'to', 'AMR', '.']\n",
            "['B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Mycobacterium', 'tuberculosis', '(', 'Mtb', ')', ',', 'an', 'acid', '-', 'fast', 'bacillus', 'that', 'causes', 'tuberculosis', '(', 'TB', ')', ',', 'merits', 'being', 'one', 'of', 'the', 'highest', 'priority', 'pathogens', 'on', 'this', 'list', 'since', 'drug', '-', 'resistant', 'TB', '(', 'DR', '-', 'TB', ')', 'accounts', 'for', '∼29', '%', 'of', 'deaths', 'attributable', 'to', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['However', ',', 'despite', 'the', 'availability', 'of', 'drug', 'regimes', ',', 'the', 'current', 'cure', 'rate', 'for', 'multi', '-', 'drug', '-', 'resistant', 'TB', '(', 'MDR', '-', 'TB', ')', 'and', 'extensively', 'drug', '-', 'resistant', 'TB', '(', 'XDR', '-', 'TB', ')', 'treatment', 'regimens', 'is', '50', '%', 'and', '30', '%', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['However', ',', 'despite', 'the', 'availability', 'of', 'drug', 'regimes', ',', 'the', 'current', 'cure', 'rate', 'for', 'multi', '-', 'drug', '-', 'resistant', 'TB', '(', 'MDR', '-', 'TB', ')', 'and', 'extensively', 'drug', '-', 'resistant', 'TB', '(', 'XDR', '-', 'TB', ')', 'treatment', 'regimens', 'is', '50', '%', 'and', '30', '%', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Our', 'work', 'mainly', 'involves', 'screening', 'of', 'various', 'small', 'molecule', 'chemical', 'libraries', 'in', 'phenotypic', 'whole', '-', 'cell', 'based', 'assays', 'to', 'identify', 'hits', 'for', 'medicinal', 'chemistry', 'optimization', ',', 'with', 'attendant', 'deconvolution', 'of', 'the', 'MoA.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'the', 'concluding', 'section', ',', 'we', 'preview', 'some', 'promising', 'future', 'directions', 'and', 'the', 'challenges', 'inherent', 'in', 'advancing', 'the', 'drug', 'pipeline', 'to', 'address', 'DR', '-', 'TB', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'L-Others', 'O']\n",
            "['This', 'has', 'so', 'far', 'led', 'to', 'the', 'accelerated', 'regulatory', 'approval', 'of', 'bedaquiline', 'and', 'delamanid', 'for', 'the', 'treatment', 'of', 'DR', '-', 'TB', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'has', 'so', 'far', 'led', 'to', 'the', 'accelerated', 'regulatory', 'approval', 'of', 'bedaquiline', 'and', 'delamanid', 'for', 'the', 'treatment', 'of', 'DR', '-', 'TB', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Combinational', 'antibiotic', 'therapy', 'is', 'one', 'of', 'the', 'best', 'and', 'easiest', 'approaches', 'to', 'handle', 'the', 'current', 'situation', 'of', 'AMR', '.']\n",
            "['B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['This', 'review', 'focuses', 'on', 'therapeutic', 'options', 'for', 'AMR', 'mechanisms', 'of', 'resistance', 'in', 'bacteria', 'that', 'can', 'be', 'overcome', 'by', 'combinational', 'therapy', 'and', 'testing', 'methods', 'for', 'synergy', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Pneumococcal', 'antimicrobial', 'resistance', '(', 'AMR', ')', 'has', 'been', 'highlighted', 'by', 'the', 'WHO', 'as', 'an', 'important', 'public', 'health', 'concern', ',', 'with', 'emerging', 'serotypes', 'showing', 'resistance', 'to', 'multiple', 'antibiotics', '.']\n",
            "['B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Here', ',', 'we', 'discuss', 'a', 'variety', 'of', 'evolutionary', 'mechanisms', 'at', 'the', 'host', ',', 'pathogen', ',', 'and', 'environmental', 'levels', 'that', 'may', 'contribute', 'to', 'changes', 'in', 'the', 'prevalence', 'of', 'pneumococcal', 'AMR', 'in', 'the', 'post', '-', 'vaccination', 'era', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'the', 'prevalence', ',', 'reservoirs', ',', 'AMR', ',', 'Raman', 'spectroscopy', 'detection', ',', 'and', 'pathogenicity', 'of', 'K.', 'pneumoniae', 'are', 'summarized', 'and', 'various', 'extraintestinal', 'infection', 'pathways', 'are', 'further', 'narrated', 'to', 'extend', 'our', 'understanding', 'of', 'its', 'adaptation', 'and', 'survival', 'ability', 'in', 'reservoirs', ',', 'and', 'associated', 'disease', 'risks', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'the', 'prevalence', ',', 'reservoirs', ',', 'AMR', ',', 'Raman', 'spectroscopy', 'detection', ',', 'and', 'pathogenicity', 'of', 'K.', 'pneumoniae', 'are', 'summarized', 'and', 'various', 'extraintestinal', 'infection', 'pathways', 'are', 'further', 'narrated', 'to', 'extend', 'our', 'understanding', 'of', 'its', 'adaptation', 'and', 'survival', 'ability', 'in', 'reservoirs', ',', 'and', 'associated', 'disease', 'risks', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Multi', '-', 'drug', 'and', 'extensively', 'drug', 'resistant', 'gonorrhoea', 'strains', 'have', 'been', 'associated', 'with', 'international', 'spread', ',', 'particularly', 'in', 'travelers', 'returning', 'from', 'Southeast', 'Asia', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Chlamydia', 'is', 'the', 'most', 'common', 'bacterial', 'STI', 'worldwide', '.']\n",
            "['U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Although', 'in', 'vitro', 'resistance', 'has', 'been', 'reported', ',', 'surveillance', 'data', 'suggest', 'that', 'clinically', 'significant', 'resistance', 'to', 'macrolides', 'and', 'tetracyclines', 'is', 'rare', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O']\n",
            "['Although', 'in', 'vitro', 'resistance', 'has', 'been', 'reported', ',', 'surveillance', 'data', 'suggest', 'that', 'clinically', 'significant', 'resistance', 'to', 'macrolides', 'and', 'tetracyclines', 'is', 'rare', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O']\n",
            "['Macrolide', 'resistance', 'in', 'syphilis', 'is', 'now', 'endemic', 'in', 'much', 'of', 'the', 'world', 'but', 'there', 'is', 'no', 'documented', 'penicillin', 'resistance', ',', 'which', 'remains', 'first', '-', 'line', 'therapy', '.']\n",
            "['U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Trichomoniasis', 'is', 'the', 'most', 'common', 'non', '-', 'viral', 'STI', 'worldwide', '.']\n",
            "['U-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Beta', '-', 'lactams', 'and', 'quinolones', 'were', 'the', 'most', 'documented', 'drug', '-', 'resistant', 'organisms', ',', 'accounting', 'for', '35', '%', 'and', '31', '%', 'of', 'the', 'overall', 'drug', 'resistance', ',', 'respectively', '.']\n",
            "['B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Beta', '-', 'lactams', 'and', 'quinolones', 'were', 'the', 'most', 'documented', 'drug', '-', 'resistant', 'organisms', ',', 'accounting', 'for', '35', '%', 'and', '31', '%', 'of', 'the', 'overall', 'drug', 'resistance', ',', 'respectively', '.']\n",
            "['O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Although', 'clinical', 'failure', 'after', 'treatment', 'occurs', ',', 'resistance', 'to', 'metronidazole', 'is', 'thought', 'to', 'be', 'uncommon', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Mycoplasma', 'genitalium', 'exhibits', 'intrinsic', 'resistance', 'to', 'many', 'antibiotics', ',', 'and', 'the', 'prevalence', 'of', 'resistance', 'to', 'both', 'first-', 'and', 'second', '-', 'line', 'regimens', '(', 'macrolides', 'and', 'fluoroquinolones', ')', 'is', 'increasing', 'worldwide', ',', 'with', 'limited', 'alternative', 'therapeutic', 'options', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Mycoplasma', 'genitalium', 'exhibits', 'intrinsic', 'resistance', 'to', 'many', 'antibiotics', ',', 'and', 'the', 'prevalence', 'of', 'resistance', 'to', 'both', 'first-', 'and', 'second', '-', 'line', 'regimens', '(', 'macrolides', 'and', 'fluoroquinolones', ')', 'is', 'increasing', 'worldwide', ',', 'with', 'limited', 'alternative', 'therapeutic', 'options', '.']\n",
            "['B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Mycoplasma', 'genitalium', 'exhibits', 'intrinsic', 'resistance', 'to', 'many', 'antibiotics', ',', 'and', 'the', 'prevalence', 'of', 'resistance', 'to', 'both', 'first-', 'and', 'second', '-', 'line', 'regimens', '(', 'macrolides', 'and', 'fluoroquinolones', ')', 'is', 'increasing', 'worldwide', ',', 'with', 'limited', 'alternative', 'therapeutic', 'options', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Raman', 'spectroscopy', 'has', 'the', 'potential', 'to', 'provide', 'for', 'the', 'rapid', 'identification', 'and', 'screening', 'of', 'antimicrobial', 'susceptibility', 'of', 'Klebsiella', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O']\n",
            "['The', 'relative', 'importance', 'of', 'these', 'factors', 'may', 'vary', 'by', 'population', ',', 'pneumococcal', 'lineage', ',', 'geography', ',', 'and', 'time', ',', 'leading', 'to', 'the', 'complex', 'relationship', 'between', 'vaccination', ',', 'antibiotic', 'use', ',', 'and', 'AMR', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Infections', 'with', 'K.', 'pneumoniae', 'can', 'be', 'transmitted', 'through', 'contaminated', 'food', 'or', 'water', 'and', 'can', 'be', 'associated', 'with', 'community', '-', 'acquired', 'infections', 'or', 'between', 'persons', 'and', 'animals', 'involved', 'in', 'hospital', '-', 'acquired', 'infections', '.']\n",
            "['O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Streptococcus', 'pneumoniae', 'is', 'a', 'major', 'cause', 'of', 'pneumonia', ',', 'meningitis', ',', 'and', 'septicaemia', 'worldwide', '.']\n",
            "['B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Because', 'of', 'its', 'specifically', 'target', 'and', 'cleave', 'DNA', 'sequences', 'encoding', 'antibiotic', 'resistance', 'genes', ',', 'CRISPR', '/', 'Cas', 'system', 'has', 'been', 'developed', 'into', 'a', 'new', 'gene', '-', 'editing', 'tool', 'for', 'the', 'prevention', 'and', 'control', 'of', 'bacterial', 'drug', 'resistance', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['CRISPR', '-', 'Cas', 'plays', 'a', 'potentially', 'important', 'role', 'in', 'controlling', 'horizontal', 'gene', 'transfer', 'and', 'limiting', 'the', 'spread', 'of', 'antibiotic', 'resistance', '.']\n",
            "['B-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'we', 'will', 'introduce', 'the', 'structure', 'and', 'working', 'mechanism', 'of', 'CRISPR', '-', 'Cas', 'systems', ',', 'followed', 'by', 'delivery', 'strategies', ',', 'and', 'then', 'focus', 'on', 'the', 'relationship', 'between', 'antimicrobial', 'resistance', 'and', 'CRISPR', '-', 'Cas', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'this', 'review', ',', 'we', 'will', 'introduce', 'the', 'structure', 'and', 'working', 'mechanism', 'of', 'CRISPR', '-', 'Cas', 'systems', ',', 'followed', 'by', 'delivery', 'strategies', ',', 'and', 'then', 'focus', 'on', 'the', 'relationship', 'between', 'antimicrobial', 'resistance', 'and', 'CRISPR', '-', 'Cas', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'L-Techniques', 'O']\n",
            "['Presently', ',', 'AMR', 'sensing', 'technologies', 'have', 'significantly', 'progressed', 'phenotypic', 'antimicrobial', 'susceptibility', 'testing', '(', 'AST', ')', 'and', 'genotypic', 'antimicrobial', 'resistance', 'gene', '(', 'ARG', ')', 'detection', 'in', 'human', 'healthcare', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Presently', ',', 'AMR', 'sensing', 'technologies', 'have', 'significantly', 'progressed', 'phenotypic', 'antimicrobial', 'susceptibility', 'testing', '(', 'AST', ')', 'and', 'genotypic', 'antimicrobial', 'resistance', 'gene', '(', 'ARG', ')', 'detection', 'in', 'human', 'healthcare', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', '-', '-', '-', '-', '-', '-', '-', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"Presently, AMR sensing technologies have significa...\" with entities \"[(123, 169, 'Antibiotic-Resistant Genes')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['For', 'risk', 'assessment', 'purposes', 'related', 'to', 'human', 'health', ',', 'AMR', 'plasmids', 'found', 'in', 'food', 'and', 'bearing', 'genes', 'conferring', 'resistances', 'to', 'critically', 'important', 'antibiotics', 'in', 'human', 'medicine', '-', 'such', 'as', 'to', 'extended', '-', 'spectrum', 'cephalosporins', ',', 'carbapenems', 'or', 'colistin', '-', 'have', 'been', 'under', 'specific', 'scrutiny', 'these', 'last', 'years', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['For', 'risk', 'assessment', 'purposes', 'related', 'to', 'human', 'health', ',', 'AMR', 'plasmids', 'found', 'in', 'food', 'and', 'bearing', 'genes', 'conferring', 'resistances', 'to', 'critically', 'important', 'antibiotics', 'in', 'human', 'medicine', '-', 'such', 'as', 'to', 'extended', '-', 'spectrum', 'cephalosporins', ',', 'carbapenems', 'or', 'colistin', '-', 'have', 'been', 'under', 'specific', 'scrutiny', 'these', 'last', 'years', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['For', 'risk', 'assessment', 'purposes', 'related', 'to', 'human', 'health', ',', 'AMR', 'plasmids', 'found', 'in', 'food', 'and', 'bearing', 'genes', 'conferring', 'resistances', 'to', 'critically', 'important', 'antibiotics', 'in', 'human', 'medicine', '-', 'such', 'as', 'to', 'extended', '-', 'spectrum', 'cephalosporins', ',', 'carbapenems', 'or', 'colistin', '-', 'have', 'been', 'under', 'specific', 'scrutiny', 'these', 'last', 'years', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Fish', 'and', 'other', 'seafood', 'can', 'be', 'contaminated', 'with', 'Vibrio', 'species', ',', 'natural', 'inhabitants', 'of', 'the', 'marine', ',', 'estuarine', ',', 'and', 'freshwater', 'environment', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Pathogenic', 'Vibrios', 'of', 'major', 'public', 'health', 'concerns', 'are', 'Vibrio', 'cholerae', ',', 'Vibrio', 'parahaemolyticus', ',', 'and', 'Vibrio', 'vulnificus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O']\n",
            "['Pathogenic', 'Vibrios', 'of', 'major', 'public', 'health', 'concerns', 'are', 'Vibrio', 'cholerae', ',', 'Vibrio', 'parahaemolyticus', ',', 'and', 'Vibrio', 'vulnificus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'I-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Pathogenic', 'Vibrios', 'of', 'major', 'public', 'health', 'concerns', 'are', 'Vibrio', 'cholerae', ',', 'Vibrio', 'parahaemolyticus', ',', 'and', 'Vibrio', 'vulnificus', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O']\n",
            "['Approximately', '10', '%', 'showed', 'resistance', 'to', 'macrolides', '(', 'azithromycin', ')', ',', 'for', 'which', 'genetic', 'determinants', 'are', 'less', 'clear', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Quality', 'assessment', 'was', 'done', 'using', 'the', 'Newcastle', '-', 'Ottawa', 'Scale', '(', 'NOS', ')', '.', 'Of', '339', 'full', '-', 'text', 'articles', 'assessed', 'for', 'eligibility', ',', 'only', 'four', 'heterogeneous', 'United', 'States', '(', 'US', ')', 'studies', 'conducted', 'in', 'the', 'period', 'between', '1986', 'and', '2015', 'were', 'found', 'eligible', ',', 'and', 'included', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Currently', ',', 'doxycycline', ',', 'azithromycin', ',', 'or', 'ciprofloxacin', 'are', 'commonly', 'used', 'for', 'V.', 'cholerae', ',', 'and', 'doxycycline', 'or', 'quinolone', 'are', 'administered', 'for', 'V.', 'parahaemolyticus', ',', 'whereas', 'doxycycline', 'and', 'a', 'third', '-', 'generation', 'cephalosporin', 'are', 'recommended', 'for', 'V.', 'vulnificus', 'as', 'initial', 'treatment', 'regimen', '.']\n",
            "['O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Currently', ',', 'doxycycline', ',', 'azithromycin', ',', 'or', 'ciprofloxacin', 'are', 'commonly', 'used', 'for', 'V.', 'cholerae', ',', 'and', 'doxycycline', 'or', 'quinolone', 'are', 'administered', 'for', 'V.', 'parahaemolyticus', ',', 'whereas', 'doxycycline', 'and', 'a', 'third', '-', 'generation', 'cephalosporin', 'are', 'recommended', 'for', 'V.', 'vulnificus', 'as', 'initial', 'treatment', 'regimen', '.']\n",
            "['O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"Currently, doxycycline, azithromycin, or ciproflox...\" with entities \"[(40, 54, 'Antibiotics')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"Currently, doxycycline, azithromycin, or ciproflox...\" with entities \"[(203, 217, 'Antibiotics')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['Currently', ',', 'doxycycline', ',', 'azithromycin', ',', 'or', 'ciprofloxacin', 'are', 'commonly', 'used', 'for', 'V.', 'cholerae', ',', 'and', 'doxycycline', 'or', 'quinolone', 'are', 'administered', 'for', 'V.', 'parahaemolyticus', ',', 'whereas', 'doxycycline', 'and', 'a', 'third', '-', 'generation', 'cephalosporin', 'are', 'recommended', 'for', 'V.', 'vulnificus', 'as', 'initial', 'treatment', 'regimen', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Currently', ',', 'doxycycline', ',', 'azithromycin', ',', 'or', 'ciprofloxacin', 'are', 'commonly', 'used', 'for', 'V.', 'cholerae', ',', 'and', 'doxycycline', 'or', 'quinolone', 'are', 'administered', 'for', 'V.', 'parahaemolyticus', ',', 'whereas', 'doxycycline', 'and', 'a', 'third', '-', 'generation', 'cephalosporin', 'are', 'recommended', 'for', 'V.', 'vulnificus', 'as', 'initial', 'treatment', 'regimen', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'emergence', 'of', 'antimicrobial', 'resistance', '(', 'AMR', ')', 'in', 'Vibrios', 'is', 'increasingly', 'common', 'across', 'the', 'globe', 'and', 'a', 'decrease', 'in', 'the', 'effectiveness', 'of', 'commonly', 'available', 'antibiotics', 'poses', 'a', 'global', 'threat', 'to', 'public', 'health', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['These', 'mobile', 'genetic', 'elements', 'are', 'highly', 'dynamic', 'and', 'could', 'potentially', 'propagate', 'to', 'other', 'bacteria', 'through', 'horizontal', 'gene', 'transfer', '(', 'HGT', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'I-Techniques', 'L-Techniques', 'O']\n",
            "['Several', 'other', 'markers', ',', 'including', 'efflux', 'pumps', 'and', 'aminoglycosides', '-', 'resistance', 'were', 'distributed', 'among', 'brands', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Several', 'other', 'markers', ',', 'including', 'efflux', 'pumps', 'and', 'aminoglycosides', '-', 'resistance', 'were', 'distributed', 'among', 'brands', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['From', '2007', 'to', '2009', ',', '104', 'Escherichia', 'coli', 'isolates', 'were', 'obtained', 'from', 'four', 'teachers', 'and', '38', 'children', 'in', 'a', 'child', 'care', 'center', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotic-Resistant Bacteria', 'L-Antibiotic-Resistant Bacteria', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Of', 'these', ',', 'those', 'caused', 'by', 'diarrheagenic', 'Escherichia', 'coli', '(', 'DEC', ')', 'strains', 'present', 'substantial', 'morbidity', 'and', 'mortality', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', '-', '-', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/training/iob_utils.py:149: UserWarning: [W030] Some entities could not be aligned in the text \"Of these, those caused by diarrheagenic Escherichi...\" with entities \"[(39, 56, 'Antibiotic-Resistant Bacteria')]\". Use `spacy.training.offsets_to_biluo_tags(nlp.make_doc(text), entities)` to check the alignment. Misaligned entities ('-') will be ignored during training.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['In', '2009', ',', 'cephem', '-', 'resistant', 'isolates', 'were', 'detected', 'in', 'children', 'as', 'well', 'as', 'a', 'teacher', '.']\n",
            "['O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Pulsed', '-', 'field', 'gel', 'electrophoresis', 'revealed', '100', '%', 'similarity', 'for', 'four', 'isolates', 'in', '2007', 'and', 'one', 'isolate', 'in', '2008', ',', 'and', 'also', 'similarity', 'among', 'seven', 'isolates', 'carrying', 'the', 'virulence', 'gene', '(', 'CNF1', ')', '.']\n",
            "['O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Laboratory', '-', 'based', 'AMR', 'testing', 'and', 'characterization', 'is', 'the', 'key', 'component', 'of', 'an', 'AMR', 'surveillance', 'program', '.']\n",
            "['O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Through', 'a', 'fingerprinting', 'analysis', ',', 'the', 'tested', 'MFCs', 'were', 'brand', '-', 'clustered', ',', 'indicating', 'reproducibility', 'along', 'the', 'production', 'chain', '.']\n",
            "['O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['A', 'common', 'core', 'of', 'resistance', 'markers', 'in', 'all', 'brands', 'evaluated', 'and', 'related', 'antimicrobials', 'such', 'as', 'β', '-', 'lactams', ',', 'tetracyclines', ',', 'quinolones', ',', 'and', 'sulfonamide', 'was', 'detected', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Antibiotics', 'I-Antibiotics', 'L-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['A', 'common', 'core', 'of', 'resistance', 'markers', 'in', 'all', 'brands', 'evaluated', 'and', 'related', 'antimicrobials', 'such', 'as', 'β', '-', 'lactams', ',', 'tetracyclines', ',', 'quinolones', ',', 'and', 'sulfonamide', 'was', 'detected', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['A', 'common', 'core', 'of', 'resistance', 'markers', 'in', 'all', 'brands', 'evaluated', 'and', 'related', 'antimicrobials', 'such', 'as', 'β', '-', 'lactams', ',', 'tetracyclines', ',', 'quinolones', ',', 'and', 'sulfonamide', 'was', 'detected', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['A', 'common', 'core', 'of', 'resistance', 'markers', 'in', 'all', 'brands', 'evaluated', 'and', 'related', 'antimicrobials', 'such', 'as', 'β', '-', 'lactams', ',', 'tetracyclines', ',', 'quinolones', ',', 'and', 'sulfonamide', 'was', 'detected', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O']\n",
            "['Among', 'the', 'Gram', '-', 'negative', 'bacteria', ',', 'carbapenem', 'resistance', 'was', 'found', 'in', '91.6', '%', 'of', 'Acinetobacter', 'baumannii', 'and', '28.5', '%', 'of', 'Klebsiella', 'pneumoniae', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'the', 'Gram', '-', 'negative', 'bacteria', ',', 'carbapenem', 'resistance', 'was', 'found', 'in', '91.6', '%', 'of', 'Acinetobacter', 'baumannii', 'and', '28.5', '%', 'of', 'Klebsiella', 'pneumoniae', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'the', 'Gram', '-', 'negative', 'bacteria', ',', 'carbapenem', 'resistance', 'was', 'found', 'in', '91.6', '%', 'of', 'Acinetobacter', 'baumannii', 'and', '28.5', '%', 'of', 'Klebsiella', 'pneumoniae', 'isolates', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O']\n",
            "['Positive', 'correlation', 'between', 'ARG', 'and', 'BMI', ',', 'caloric', 'intake', ',', 'and', 'intake', 'of', 'xenobiotics', ',', 'was', 'observed', 'for', 'obese', 'individuals', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['A', 'common', 'theory', 'for', 'this', 'behavior', 'is', 'the', 'loss', 'of', 'gravity', '-', 'driven', 'convection', 'processes', 'in', 'the', 'orbital', 'environment', ',', 'resulting', 'in', 'both', 'reduction', 'of', 'extracellular', 'nutrient', 'availability', 'and', 'the', 'accumulation', 'of', 'bacterial', 'byproducts', 'near', 'the', 'cell', '.', 'To', 'further', 'characterize', 'the', 'responses', ',', 'this', 'study', 'investigated', 'the', 'transcriptomic', 'response', 'of', 'Escherichia', 'coli', 'to', 'both', 'microgravity', 'and', 'antibiotic', 'concentration', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'L-Others']\n",
            "['E.', 'coli', 'was', 'grown', 'aboard', 'International', 'Space', 'Station', 'in', 'the', 'presence', 'of', 'increasing', 'concentrations', 'of', 'the', 'antibiotic', 'gentamicin', 'with', 'identical', 'ground', 'controls', 'conducted', 'on', 'Earth', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Furthermore', ',', 'the', 'increased', 'xenobiotic', 'intake', 'associated', 'to', 'obesity', 'may', 'play', 'an', 'important', 'role', 'in', 'the', 'antimicrobial', 'resistance', 'phenomenon', '.']\n",
            "['O', 'O', 'O', 'O', 'U-Antibiotics', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Our', 'studies', 'demonstrated', 'that', '47.0', '%', '(', '222/472', ')', 'of', 'samples', 'contained', 'thermotolerant', 'coliforms', 'above', 'permissive', 'levels', 'whereas', 'E.', 'coli', 'strains', 'were', 'isolated', 'from', '43.6', '%', '(', '206/472', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Our', 'studies', 'demonstrated', 'that', '47.0', '%', '(', '222/472', ')', 'of', 'samples', 'contained', 'thermotolerant', 'coliforms', 'above', 'permissive', 'levels', 'whereas', 'E.', 'coli', 'strains', 'were', 'isolated', 'from', '43.6', '%', '(', '206/472', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'these', 'E.', 'coli', 'isolates', ',', 'DEC', 'strains', 'were', 'identified', 'in', '14', '%', '(', '29/206', ')', 'of', 'samples', 'including', 'in', 'irrigation', 'canal', '(', '26/29', ')', 'and', 'river', 'water', '(', '3/29', ')', 'collected', 'from', 'the', 'northern', '(', '83', '%', ')', 'and', 'central', 'area', '(', '17', '%', ')', '.', 'Isolated', 'DEC', 'strains', 'were', 'classified', 'as', 'enteroaggregative', 'E.', 'coli', '(', 'EAEC', ')', '34.4', '%', '(', '10/29', ')', ',', 'enteropathogenic', 'E.', 'coli', '(', 'EPEC', ')', '31.0', '%', '(', '9/29', ')', ',', 'diffuse', 'adherent', 'E.', 'coli', '(', 'DAEC', ')', '27.5', '%', '(', '8/29', ')', ',', 'and', 'enterotoxigenic', 'E.', 'coli', '(', 'ETEC', ')', '6.8', '%', '(', '2/29', ')', '.']\n",
            "['O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'these', 'E.', 'coli', 'isolates', ',', 'DEC', 'strains', 'were', 'identified', 'in', '14', '%', '(', '29/206', ')', 'of', 'samples', 'including', 'in', 'irrigation', 'canal', '(', '26/29', ')', 'and', 'river', 'water', '(', '3/29', ')', 'collected', 'from', 'the', 'northern', '(', '83', '%', ')', 'and', 'central', 'area', '(', '17', '%', ')', '.', 'Isolated', 'DEC', 'strains', 'were', 'classified', 'as', 'enteroaggregative', 'E.', 'coli', '(', 'EAEC', ')', '34.4', '%', '(', '10/29', ')', ',', 'enteropathogenic', 'E.', 'coli', '(', 'EPEC', ')', '31.0', '%', '(', '9/29', ')', ',', 'diffuse', 'adherent', 'E.', 'coli', '(', 'DAEC', ')', '27.5', '%', '(', '8/29', ')', ',', 'and', 'enterotoxigenic', 'E.', 'coli', '(', 'ETEC', ')', '6.8', '%', '(', '2/29', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'these', 'E.', 'coli', 'isolates', ',', 'DEC', 'strains', 'were', 'identified', 'in', '14', '%', '(', '29/206', ')', 'of', 'samples', 'including', 'in', 'irrigation', 'canal', '(', '26/29', ')', 'and', 'river', 'water', '(', '3/29', ')', 'collected', 'from', 'the', 'northern', '(', '83', '%', ')', 'and', 'central', 'area', '(', '17', '%', ')', '.', 'Isolated', 'DEC', 'strains', 'were', 'classified', 'as', 'enteroaggregative', 'E.', 'coli', '(', 'EAEC', ')', '34.4', '%', '(', '10/29', ')', ',', 'enteropathogenic', 'E.', 'coli', '(', 'EPEC', ')', '31.0', '%', '(', '9/29', ')', ',', 'diffuse', 'adherent', 'E.', 'coli', '(', 'DAEC', ')', '27.5', '%', '(', '8/29', ')', ',', 'and', 'enterotoxigenic', 'E.', 'coli', '(', 'ETEC', ')', '6.8', '%', '(', '2/29', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'these', 'E.', 'coli', 'isolates', ',', 'DEC', 'strains', 'were', 'identified', 'in', '14', '%', '(', '29/206', ')', 'of', 'samples', 'including', 'in', 'irrigation', 'canal', '(', '26/29', ')', 'and', 'river', 'water', '(', '3/29', ')', 'collected', 'from', 'the', 'northern', '(', '83', '%', ')', 'and', 'central', 'area', '(', '17', '%', ')', '.', 'Isolated', 'DEC', 'strains', 'were', 'classified', 'as', 'enteroaggregative', 'E.', 'coli', '(', 'EAEC', ')', '34.4', '%', '(', '10/29', ')', ',', 'enteropathogenic', 'E.', 'coli', '(', 'EPEC', ')', '31.0', '%', '(', '9/29', ')', ',', 'diffuse', 'adherent', 'E.', 'coli', '(', 'DAEC', ')', '27.5', '%', '(', '8/29', ')', ',', 'and', 'enterotoxigenic', 'E.', 'coli', '(', 'ETEC', ')', '6.8', '%', '(', '2/29', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'these', 'E.', 'coli', 'isolates', ',', 'DEC', 'strains', 'were', 'identified', 'in', '14', '%', '(', '29/206', ')', 'of', 'samples', 'including', 'in', 'irrigation', 'canal', '(', '26/29', ')', 'and', 'river', 'water', '(', '3/29', ')', 'collected', 'from', 'the', 'northern', '(', '83', '%', ')', 'and', 'central', 'area', '(', '17', '%', ')', '.', 'Isolated', 'DEC', 'strains', 'were', 'classified', 'as', 'enteroaggregative', 'E.', 'coli', '(', 'EAEC', ')', '34.4', '%', '(', '10/29', ')', ',', 'enteropathogenic', 'E.', 'coli', '(', 'EPEC', ')', '31.0', '%', '(', '9/29', ')', ',', 'diffuse', 'adherent', 'E.', 'coli', '(', 'DAEC', ')', '27.5', '%', '(', '8/29', ')', ',', 'and', 'enterotoxigenic', 'E.', 'coli', '(', 'ETEC', ')', '6.8', '%', '(', '2/29', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Among', 'these', 'E.', 'coli', 'isolates', ',', 'DEC', 'strains', 'were', 'identified', 'in', '14', '%', '(', '29/206', ')', 'of', 'samples', 'including', 'in', 'irrigation', 'canal', '(', '26/29', ')', 'and', 'river', 'water', '(', '3/29', ')', 'collected', 'from', 'the', 'northern', '(', '83', '%', ')', 'and', 'central', 'area', '(', '17', '%', ')', '.', 'Isolated', 'DEC', 'strains', 'were', 'classified', 'as', 'enteroaggregative', 'E.', 'coli', '(', 'EAEC', ')', '34.4', '%', '(', '10/29', ')', ',', 'enteropathogenic', 'E.', 'coli', '(', 'EPEC', ')', '31.0', '%', '(', '9/29', ')', ',', 'diffuse', 'adherent', 'E.', 'coli', '(', 'DAEC', ')', '27.5', '%', '(', '8/29', ')', ',', 'and', 'enterotoxigenic', 'E.', 'coli', '(', 'ETEC', ')', '6.8', '%', '(', '2/29', ')', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'I-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['Demographics', ',', 'intrinsic', 'and', 'extrinsic', 'risk', 'factors', ',', 'information', 'regarding', 'infection', 'and', 'isolated', 'pathogens', 'with', 'antibiogram', 'results', 'were', 'collected', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Techniques', 'O', 'O', 'O', 'O']\n",
            "['Moreover', ',', '90', '%', 'of', 'isolated', 'DEC', 'strains', 'exhibited', 'resistance', 'to', 'at', 'least', 'one', 'commonly', 'prescribed', 'antibiotic', 'in', 'Mexico', 'whereas', '17', '%', 'were', 'multi', '-', 'drug', 'resistant', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'B-Microbes', 'L-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'conclusion', ',', 'the', 'presence', 'of', 'DEC', 'strains', 'in', 'surface', 'water', 'represents', 'a', 'potential', 'source', 'for', 'human', 'infection', ',', 'and', 'thus', 'routine', 'monitoring', 'of', 'DEC', 'in', 'surface', 'water', 'and', 'other', 'indirect', 'affected', 'areas', 'should', 'be', 'considered', 'at', 'northwestern', 'Mexico', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['In', 'conclusion', ',', 'the', 'presence', 'of', 'DEC', 'strains', 'in', 'surface', 'water', 'represents', 'a', 'potential', 'source', 'for', 'human', 'infection', ',', 'and', 'thus', 'routine', 'monitoring', 'of', 'DEC', 'in', 'surface', 'water', 'and', 'other', 'indirect', 'affected', 'areas', 'should', 'be', 'considered', 'at', 'northwestern', 'Mexico', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'U-Microbes', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n",
            "['The', 'aim', 'of', 'this', 'study', 'was', 'to', 'compare', 'the', 'occurrence', 'of', 'antimicrobial', 'resistance', 'genetic', 'markers', '(', 'ARG', ')', 'in', 'faecal', 'specimens', 'of', 'eutrophic', ',', 'overweight', 'and', 'obese', 'individuals', ',', 'and', 'their', 'correlation', 'with', 'xenobiotic', 'intake', 'and', 'gut', 'bacteria', 'density', '.']\n",
            "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'B-Others', 'I-Others', 'I-Others', 'I-Others', 'I-Others', 'I-Others', 'L-Others', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "lm=\"Whilst K. pneumoniae's contribution to the overall AMR crisis is impossible to quantify, current evidence suggests it has a wider ecological distribution, significantly more varied DNA composition, greater AMR gene diversity and a higher plasmid burden than other Gram negative opportunists.\"\n",
        "te= lm[7:21]\n",
        "te"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 36
        },
        "id": "w3l0qI8RljOL",
        "outputId": "819b5589-a55b-4ef3-d514-ced03e1c1e17"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\"K. pneumoniae'\""
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            }
          },
          "metadata": {},
          "execution_count": 78
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "res[950], text[950][12:32]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Mw1CYdbPep0C",
        "outputId": "c99bb9fc-a92b-4968-e8e3-5048e74a2fc7"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "([(13, 32, 'Techniques')], ' gel electrophoresis')"
            ]
          },
          "metadata": {},
          "execution_count": 76
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from spacy.training import offsets_to_biluo_tags\n",
        "from spacy.lang.en import English\n",
        "nlp=English()\n",
        "offsets = res[950]\n",
        "doc = nlp(text[950])\n",
        "tags = offsets_to_biluo_tags(doc, offsets)\n",
        "print([token.text for token in doc])\n",
        "print(tags)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "rBX3OUZMqJZ7",
        "outputId": "931cd8df-6f69-4449-9063-a1a3f87c114d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['Pulsed', '-', 'field', 'gel', 'electrophoresis', 'revealed', '100', '%', 'similarity', 'for', 'four', 'isolates', 'in', '2007', 'and', 'one', 'isolate', 'in', '2008', ',', 'and', 'also', 'similarity', 'among', 'seven', 'isolates', 'carrying', 'the', 'virulence', 'gene', '(', 'CNF1', ')', '.']\n",
            "['O', 'O', 'O', 'B-Techniques', 'L-Techniques', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "labels_data"
      ],
      "metadata": {
        "id": "RyQACdSGQWy-",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 269
        },
        "outputId": "a823fa43-d87d-4d4f-e991-1fa7ab47c6a6"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "   id                           text  prefix_key  suffix_key background_color  \\\n",
              "0  31                  Gene/Proteins         NaN           0          #1A237E   \n",
              "1  32                       Microbes         NaN           1          #76F648   \n",
              "2  33  Antibiotic-Resistant Bacteria         NaN           2          #07360A   \n",
              "3  34     Antibiotic-Resistant Genes         NaN           3          #EFF1EF   \n",
              "4  35                     Techniques         NaN           4          #D51D0F   \n",
              "5  36                    Antibiotics         NaN           5          #159EE1   \n",
              "6  37                         Others         NaN           6          #D4C543   \n",
              "\n",
              "  text_color  \n",
              "0    #ffffff  \n",
              "1    #ffffff  \n",
              "2    #ffffff  \n",
              "3    #ffffff  \n",
              "4    #ffffff  \n",
              "5    #ffffff  \n",
              "6    #ffffff  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-f37d239a-734a-4cf3-8a29-83eb62febfa3\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>id</th>\n",
              "      <th>text</th>\n",
              "      <th>prefix_key</th>\n",
              "      <th>suffix_key</th>\n",
              "      <th>background_color</th>\n",
              "      <th>text_color</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>31</td>\n",
              "      <td>Gene/Proteins</td>\n",
              "      <td>NaN</td>\n",
              "      <td>0</td>\n",
              "      <td>#1A237E</td>\n",
              "      <td>#ffffff</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>32</td>\n",
              "      <td>Microbes</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1</td>\n",
              "      <td>#76F648</td>\n",
              "      <td>#ffffff</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>33</td>\n",
              "      <td>Antibiotic-Resistant Bacteria</td>\n",
              "      <td>NaN</td>\n",
              "      <td>2</td>\n",
              "      <td>#07360A</td>\n",
              "      <td>#ffffff</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>34</td>\n",
              "      <td>Antibiotic-Resistant Genes</td>\n",
              "      <td>NaN</td>\n",
              "      <td>3</td>\n",
              "      <td>#EFF1EF</td>\n",
              "      <td>#ffffff</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>35</td>\n",
              "      <td>Techniques</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "      <td>#D51D0F</td>\n",
              "      <td>#ffffff</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>36</td>\n",
              "      <td>Antibiotics</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5</td>\n",
              "      <td>#159EE1</td>\n",
              "      <td>#ffffff</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>37</td>\n",
              "      <td>Others</td>\n",
              "      <td>NaN</td>\n",
              "      <td>6</td>\n",
              "      <td>#D4C543</td>\n",
              "      <td>#ffffff</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-f37d239a-734a-4cf3-8a29-83eb62febfa3')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-f37d239a-734a-4cf3-8a29-83eb62febfa3 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-f37d239a-734a-4cf3-8a29-83eb62febfa3');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-228b8477-2be1-429c-8394-5aa356754aa1\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-228b8477-2be1-429c-8394-5aa356754aa1')\"\n",
              "            title=\"Suggest charts.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-228b8477-2be1-429c-8394-5aa356754aa1 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ]
          },
          "metadata": {},
          "execution_count": 14
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# dicto=[]\n",
        "# str_one=\" \"\n",
        "# for i in range(2337):\n",
        "#   dicto.append(data_two['text'][i])\n",
        "# listToStr = ' '.join([str(elem) for elem in dicto])\n",
        "# sentences=nltk.sent_tokenize(listToStr)\n",
        "# data_three = [(i+1, nltk.word_tokenize(sentence)) for i, sentence in enumerate(sentences)]\n",
        "# words = [(sentence, word) for sentence, words in data_three for word in words]\n",
        "# df = pd.DataFrame(words, columns=['Sentence', 'Word'])"
      ],
      "metadata": {
        "id": "_afXU905d2YK"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "df"
      ],
      "metadata": {
        "id": "s59UsgEvVdAn"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "word_counts = df.groupby([\"Sentence\"])['Word'].agg(['count'])\n",
        "word_counts.hist(bins=50)"
      ],
      "metadata": {
        "id": "VvYZtk45T61P"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "all_words = list(df['Word'].unique())\n",
        "word2index = {word : idx for idx, word in enumerate(all_words, 2)}\n",
        "word2index['--UNK--']=0\n",
        "word2index['--PAD--']=1\n",
        "word2index"
      ],
      "metadata": {
        "id": "swbx4LxwLVF3"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "lk = \" gene therap\"\n",
        "lk=lk.strip()\n",
        "lk[-1]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 36
        },
        "id": "9i12bk3gLu47",
        "outputId": "00197471-e7b1-4cbc-eda4-a38de1372798"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'p'"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            }
          },
          "metadata": {},
          "execution_count": 20
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "r39tjDibL1LO"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}